<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Hortic Res</span><span class="iso-abbrev" title="iso-abbrev">Hortic Res</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Horticulture Research</span></div><span class="issn-ppub" tagx="issn" title="issn-ppub">2662-6810</span><span class="issn-epub" tagx="issn" title="issn-epub">2052-7276</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Nature Publishing Group UK</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7848005">7848005</a></span><span class="publisher-id" title="publisher-id">458</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1038/s41438-020-00458-y">10.1038/s41438-020-00458-y</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Insights into triterpene synthesis and unsaturated fatty-acid accumulation provided by chromosomal-level genome analysis of <i>Akebia trifoliata</i> subsp. <i>australis</i></div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0003-1212-496X</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">Hui</span></span><a href="#Aff1">1</a><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liang</span><span class="given-names" tagx="given-names" title="given-names">Juan</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tan</span><span class="given-names" tagx="given-names" title="given-names">Qi</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ou</span><span class="given-names" tagx="given-names" title="given-names">Linfeng</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Xiaolin</span></span><a href="#Aff3">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhong</span><span class="given-names" tagx="given-names" title="given-names">Caihong</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">Huilin</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-7919-7787</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">MÃ¸ller</span><span class="given-names" tagx="given-names" title="given-names">Ian Max</span></span><a href="#Aff4">4</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">Xianjin</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Song</span><span class="given-names" tagx="given-names" title="given-names">Songquan</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">sqsong@ibcas.ac.cn</span></span><a href="#Aff1">1</a><a href="#Aff5">5</a></span><span class="citation_author_institution" id="Aff1">[1], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.411401.1</span><span class="institution-id" title="institution-id">0000 0004 1804 2612</span><span class="institution" title="institution">Key Laboratory of Research and Utilization of Ethnomedicinal Plant Resources of Hunan Province, College of Biological and Food Engineering, Huaihua University, </span></div></span><span class="citation_author_institution" id="Aff2">[2], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.9227.e</span><span class="institution-id" title="institution-id">0000000119573309</span><span class="institution" title="institution">Kunming Institute of Botany, </span><span class="institution" title="institution">Chinese Academy of Sciences, </span></div></span><span class="citation_author_institution" id="Aff3">[3], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.410318.f</span><span class="institution-id" title="institution-id">0000 0004 0632 3409</span><span class="institution" title="institution">State Key Laboratory Breeding Base of Dao-di Herbs, National Resource Center for Chinese Materia Medica, China Academy of Chinese Medical Science, </span></div></span><span class="citation_author_institution" id="Aff4">[4], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.7048.b</span><span class="institution-id" title="institution-id">0000 0001 1956 2722</span><span class="institution" title="institution">Department of Molecular Biology and Genetics, </span><span class="institution" title="institution">Aarhus University, Flakkebjerg, </span></div></span><span class="citation_author_institution" id="Aff5">[5], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.9227.e</span><span class="institution-id" title="institution-id">0000000119573309</span><span class="institution" title="institution">Institute of Botany, </span><span class="institution" title="institution">Chinese Academy of Sciences, </span></div></span></div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2021-2-2</span></span><span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2021-2-2</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2021</span></span><span class="volume" tagx="volume" title="volume">8</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">33</span><span class="history" title="history"><span class="received" title="received">received: 2020-4-15</span><span class="rev-recd" title="rev-recd">rev-recd: 2020-11-16</span><span class="accepted" title="accepted">accepted: 2020-11-20</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , </span><span class="license" title="license"><span class="license-p" title="license-p"><b>Open Access</b> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articleâ€™s Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articleâ€™s Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this license, visit <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>.</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p id="Par1"><i>Akebia trifoliata</i> subsp. <i>australis</i> is a well-known medicinal and potential woody oil plant in China. The limited genetic information available for <i>A. trifoliata</i> subsp. <i>australis</i> has hindered its exploitation. Here, a high-quality chromosome-level genome sequence of <i>A. trifoliata</i> subsp. <i>australis</i> is reported. The <i>de novo</i> genome assembly of 682.14â€‰Mb was generated with a scaffold N50 of 43.11â€‰Mb. The genome includes 25,598 protein-coding genes, and 71.18% (485.55â€‰Mb) of the assembled sequences were identified as repetitive sequences. An ongoing massive burst of long terminal repeat (LTR) insertions, which occurred ~1.0 million years ago, has contributed a large proportion of LTRs in the genome of <i>A. trifoliata</i> subsp. <i>australis</i>. Phylogenetic analysis shows that <i>A. trifoliata</i> subsp. <i>australis</i> is closely related to <i>Aquilegia coerulea</i> and forms a clade with <i>Papaver somniferum</i> and <i>Nelumbo nucifera</i>, which supports the well-established hypothesis of a close relationship between basal eudicot species. The expansion of <i>UDP-glucoronosyl</i> and <i>UDP-glucosyl transferase</i> gene families and <i>Î²-amyrin synthase-like</i> genes and the exclusive contraction of <i>terpene synthase</i> gene families may be responsible for the abundant oleanane-type triterpenoids in <i>A. trifoliata</i> subsp. <i>australis</i>. Furthermore, the <i>acyl-ACP desaturase</i> gene family, including 12 <i>stearoyl-acyl-carrier protein desaturase</i> (<i>SAD</i>) genes, has expanded exclusively. A combined transcriptome and fatty-acid analysis of seeds at five developmental stages revealed that homologs of SADs, acyl-lipid desaturase omega fatty acid desaturases (FADs), and oleosins were highly expressed, consistent with the rapid increase in the content of fatty acids, especially unsaturated fatty acids. The genomic sequences of <i>A. trifoliata</i> subsp. <i>australis</i> will be a valuable resource for comparative genomic analyses and molecular breeding.</p> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Subject terms</div> <div class="kwd" title="kwd">
Genomics</div> <div class="kwd" title="kwd">
Lipids</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="institution" title="institution">Natural Science Foundation of Hunan Province (2019JJ50475), Key Scientific Research Projects of Hunan Education Department (18A448), Foundation of Hunan Double First-rate Discipline Construction Projects of Bioengineering and Key Laboratory of Research and Utilization of Ethnomedicinal Plant Resources of Hunan Province (YYZW2018-6)</span></div> </div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">https://doi.org/10.13039/501100001809</span><span class="institution" title="institution">National Natural Science Foundation of China (National Science Foundation of China)</span></div> </div><span class="award-id" tagx="award-id" title="award-id">81874334</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Xiaolin</span></span></span></div> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="issue-copyright-statement" value="Â© The Author(s) 2021" /> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="introduction"> <div class="title" tagx="title" title="title">
Introduction</div> <p id="Par2"><i>Akebia trifoliata</i> (Thumb.) Koidz. subsp. <i>australis</i> is a perennial woody plant that belongs to the genus <i>Akebia</i> (Lardizabalaceae) and is mainly distributed in East Asia<sup><a href="#CR1">1</a></sup>. <i>A. trifoliata</i> subsp. <i>australis</i> (abbreviated as <i>A. trifoliata</i> hereafter) as well as <i>A. trifoliata</i> subsp. <i>trifoliata</i> and <i>A. quinata</i>, which are two other members of the genus <i>Akebia</i>, are listed in the Chinese Pharmacopoeia<sup><a href="#CR2">2</a></sup> and have been used as traditional herbal medicines for &amp;gt;2000 years. An example is â€œAkebiae Fructusâ€�, which is made from the dry fruit of <i>A. trifoliata</i>, including seeds, peel, and flesh<sup><a href="#CR3">3</a>,<a href="#CR4">4</a></sup>. Experimental and clinical studies have demonstrated that <i>A. trifoliata</i> possesses anti-inflammatory, antimicrobial, antioxidative, and anticancer properties<sup><a href="#CR4">4</a>,<a href="#CR5">5</a></sup>. Lu et al.<sup><a href="#CR5">5</a></sup> reported that the ethanol extract of <i>A. trifoliata</i> seeds has antimetastatic potency against hepatocellular carcinoma cells. The pharmacological properties of <i>A. trifoliata</i> are attributed to numerous bioactive compounds, including triterpenoid saponins, triterpenes, and flavonoids, in dried fruits, stems, leaves, and seeds of <i>A. trifoliata</i><sup><a href="#CR6">6</a></sup>. The content of oleanane-type triterpenoids, especially oleanolic acid, is high in the dry fruit of <i>A. trifoliata</i><sup><a href="#CR7">7</a></sup>. These natural bioactive compounds with minimal or no side effects have attracted attention from chemists and pharmacologists due to their complex structural features and multiple biological effects.</p> <p id="Par3">Studies have indicated that the upstream reactions of triterpenoid biosynthesis through the mevalonate (MVA) or methylerythritol phosphate (MEP) pathways are involved in making building blocks and intermediates such as isopentenylpyrophosphate and farnesyl diphosphate (FPP)<sup><a href="#CR8">8</a></sup>. The downstream reactions of triterpenoid biosynthesis start from the condensation of two molecules of FPP, and the triterpenoid backbone undergoes oxidation, substitution, and glycosylation to generate various triterpenoids, involving many key enzymes, including squalene synthase (SQS), squalene monoxidase, oxidosqualene cyclase (OSC), cytochrome P450 monooxygenase (CYP), and uridine diphosphate glycosyltransferase (UGT)<sup><a href="#CR9">9</a></sup>. Among these enzymes, <i>Î²</i>-amyrin synthase (<i>Î²</i>-AS), a kind of OSC, is a unique key enzyme that catalyzes oleanane-type triterpenoid biosynthesis, and its expression is positively correlated with oleanane-type saponin content<sup><a href="#CR10">10</a></sup>.</p> <p id="Par4">As a potential oilseed medicinal plant, <i>A. trifoliata</i> seeds contain up to 39% oil with 77% unsaturated fatty acids<sup><a href="#CR3">3</a></sup>. Owing to the high UFA content, the seed oil of <i>A. trifoliata</i> is used as a quality edible oil and dietary supplement in China. In contrast to saturated fatty acids (SFAs), the dietary potency of UFAs has health benefits by reducing the risk of cardiovascular disease, obesity, and cancer<sup><a href="#CR11">11</a></sup> and promoting the absorption of lipophilic nutritional compounds<sup><a href="#CR12">12</a></sup>. Fatty acids (FAs) are synthesized in plastids from acetyl-CoA up to 18:1<sup>Î"9</sup>, the first desaturation catalyzed by stearoyl-acyl-carrier protein desaturase (SAD). After export to the cytosol, FAs can be elongated and desaturated to produce long-chain and polyunsaturated fatty acids (PUFAs) by various enzymes within the ER and eventually processed into the storage lipid triacylglycerol (TAG)<sup><a href="#CR13">13</a></sup>. TAG is stored in oil bodies that serve as a natural protective system against fatty-acid oxidation and maintain lipid stability<sup><a href="#CR14">14</a></sup>. In olive oil, monounsaturated oleic acid (C18:1) makes up 75% of all TAGs, followed by saturated palmitic acid (C16:0; ~13.5%), polyunsaturated linoleic acid (C18:2; ~5.5%), and Î±-linolenic acid (C18:3; ~0.75%)<sup><a href="#CR15">15</a></sup>. In sesame seed oil, both oleic acid and linoleic acid are more evenly present (~40%)<sup><a href="#CR16">16</a></sup>. The differential accumulation of oleic and linoleic acids in olive compared with sesame is attributed to the functional divergence of oil biosynthesis pathway genes, such as Ï‰-6 <i>fatty-acid desaturase 2</i> (<i>FAD2</i>) and <i>SAD</i>, following duplication<sup><a href="#CR17">17</a></sup>. Oleosin, the most abundant oil body-associated protein, is a major determinant of oil body size<sup><a href="#CR18">18</a></sup> and has been suggested to contribute to the stability of oil bodies and their synthesis and metabolism<sup><a href="#CR19">19</a></sup>. The heterologous expression of castor bean oleosin in <i>Arabidopsis</i> led to a 20% increase in the ricinoleic acid content in TAGs<sup><a href="#CR20">20</a></sup>. Moreover, coexpression of oleosins with other TAG biosynthesis genes increased the oil content<sup><a href="#CR21">21</a></sup>.</p> <p id="Par5">In addition to the usages mentioned above, the cultivated fruit of <i>A. trifoliata</i> is consumed as a delicacy due to its delicious taste and abundance of nutrients. <i>Akebia trifoliata</i> belongs to the genus <i>Akebia</i> (Lardizabalaceae) and, like <i>Aquilegia coerulea</i> and <i>Papaver somniferum</i>, is a member of the basal eudicots, which possess more diverse floral morphologies than the core eudicots and monocots. The identification and investigation of floral genes in basal eudicots can be used as an evolutionary link between core eudicots and grasses<sup><a href="#CR22">22</a></sup>.</p> <p id="Par6">Despite the considerable importance of <i>A. trifoliata</i>, genomic information for the species is limited, which has hindered its study and utilization. Here, we report the sequencing and assembly of a high-quality chromosomal-level genome sequence of <i>A. trifoliata</i> subsp. <i>australis</i>, which is the first sequenced species in the genus <i>Akebia</i>. Furthermore, we identified key genes involved in triterpene synthesis and the accumulation of UFAs. The availability of this genomic information will be helpful for comparative genomic analysis and the molecular breeding and engineering of <i>A. trifoliata</i>.</p> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results and discussion</div> <div class="denovogenomesequencing,assembly,andqualityassessment" title="sec"> <div class="title" tagx="title" title="title"><i>De novo</i> genome sequencing, assembly, and quality assessment</div> <p id="Par7">The genome of <i>A. trifoliata</i> was initially sequenced and assembled using the HiSeq X Ten sequencing platform from Illumina and PacBio single-molecule real-time (SMRT) sequencing technology, and the assembled contigs were anchored to pseudochromosomes using the Hi-C technique. K-mer analysis revealed that the estimated genome size of <i>A. trifoliata</i> was 669.76â€‰Mb with a heterozygosity of 0.89% (Fig. <a href="#MOESM2">S1</a>). Furthermore, the genome size of <i>A. trifoliata</i> was estimated to be 654.34â€‰Mb using flow cytometry (Fig. <a href="#MOESM2">S2</a>). In total, 193.71â€‰Gb of high-quality sequences with a depth of 284-fold of the <i>A. trifoliata</i> genome were used to assemble the genome (Table <a href="#MOESM1">S1</a>). To obtain further chromosomal information about <i>A. trifoliata</i>, the sequences were then scaffolded and corrected using optical mapping data, and the resulting scaffolds were clustered into 16 pseudochromosomes (2<i>n</i>â€‰=â€‰32), accounting for 98.05% (668.89/682.14â€‰Mb) of the genome (Fig. <a href="#Fig1">1</a>, Fig. <a href="#MOESM2">S3</a>). The final chromosome-scale genome was 682.14â€‰Mb in length with 689 contigs (contig N50â€‰=â€‰6.20â€‰Mb) and 109 scaffolds (scaffolds N50â€‰=â€‰43.11â€‰Mb) (Table <a href="#Tab1">1</a>). The final assembled sequence 682.14â€‰Mb for <i>A. trifoliata</i> was close to the calculated estimated size (669.76â€‰Mb) and to the size estimated by flow cytometry (654.34â€‰Mb). The assembled size was marginally larger than the estimated size, probably because of the relatively high content of repetitive sequences (71.18%). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 1</span></a></div>   <div class="title" tagx="title" title="title">
Chromosomal features of <i>Akebia trifoliata</i> subsp. <i>australis</i>.</div> <p>Hi-C map of the <i>A. trifoliata</i> subsp. <i>australis</i> genome showing genome-wide all-by-all interactions (<b>A</b>) and characterization of the <i>A. trifoliata</i> subsp. <i>australis</i> genome (<b>B</b>). <b>a</b> repeat density; <b>b</b> gene density; <b>c</b> chromosomes; <b>d</b> GC content; and <b>e</b> relationship between synteny blocks</p>   </div> <div class="table-wrap_UNKNOWN" id="Tab1"><span class="label" tagx="label" title="label">Table 1</span> <p>Statistics of <i>A. trifoliata</i> subsp. <i>australis</i> genome sequencing, assembly, and annotation</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th>Genomic feature</th> <th><i>A. trifoliata</i> subsp. <i>australis</i></th> </tr> </thead> <tbody> <tr> <td>Assembled genome size (Mb)</td> <td>682.14</td> </tr> <tr> <td>PacBio reads (Gb)</td> <td>65.96 (96.7 X<sup>a</sup>)</td> </tr> <tr> <td>Illumina reads (Gb)</td> <td>24.75 (36.28 X)</td> </tr> <tr> <td>Hi-C (Gb)</td> <td>103.00 (151.03 X)</td> </tr> <tr> <td>Total reads (Gb)</td> <td>193.71 (284.02 X)</td> </tr> <tr> <td>GC content (%)</td> <td>35.02</td> </tr> <tr> <td>Percentage of anchoring</td> <td>98.05%</td> </tr> <tr> <td>Number of contigs</td> <td>689</td> </tr> <tr> <td>Contigs N50 (kb)</td> <td>6,198</td> </tr> <tr> <td>Contigs N90 (kb)</td> <td>1,563</td> </tr> <tr> <td>Number of scaffolds</td> <td>109</td> </tr> <tr> <td>Scaffold N50 (kb)</td> <td>43,106</td> </tr> <tr> <td>Scaffold N90 (kb)</td> <td>30,967</td> </tr> <tr> <td>Longest sequence length (kb)</td> <td>64,151</td> </tr> <tr> <td>Total repetitive sequences (Mb)</td> <td>485.55 (71.18%)</td> </tr> <tr> <td>Total protein-coding genes</td> <td>25,598</td> </tr> <tr> <td>Annotated protein-coding genes</td> <td>25,008</td> </tr> <tr> <td>Average length per gene (kb)</td> <td>6.47</td> </tr> <tr> <td>Total exon length (kb)</td> <td>41.34</td> </tr> <tr> <td>Average exon length (kb)</td> <td>1.62</td> </tr> <tr> <td>Total intron length (kb)</td> <td>124.39</td> </tr> <tr> <td>Average intron length (kb)</td> <td>4.86</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p><sup>a</sup>Sequence coverage</p> </div> </div> <p /> <p id="Par8">In the sequencing and assembly quality assessment, 98.56% of Illumina short-insert reads could be aligned back to the final assembly. Moreover, BUSCO analysis showed that 1518 (94%) of the 1614 orthologs from the Embryophyta database were completely captured in our assembly, and CEGMA analysis showed that the assembled genome completely recalled 424 (93%) of the 458 core eukaryotic genes (CEGs) and 186 (75%) of the 248 highly conserved CEGs (Fig. <a href="#MOESM2">S4</a>).</p> </div> <div class="genomeannotation" title="sec"> <div class="title" tagx="title" title="title">
Genome annotation</div> <p id="Par9">A total of 25,598 protein-coding genes were predicted in the <i>A. trifoliata</i> genome through a combination of <i>ab</i> initio prediction, homology search, and RNA-Seq prediction. Of all the predicted protein-coding genes, 24,814 (97%) were annotated based on homology search and RNA-Seq reads, with only 3.1% deriving solely from <i>ab</i> initio gene prediction, suggesting that the results of the protein-coding gene prediction were high quality (Fig. S<a href="#MOESM2">5a</a>). The <i>A. trifoliata</i> genome had an average gene length of 6,47â€‰kb and average exon and intron lengths of 1,62â€‰kb and 4,86â€‰kb, respectively (Table <a href="#Tab1">1</a>). By similarity search, 25,008 protein-coding genes (98%) had functional annotations in public databases from several species, including <i>N. nucifera</i> (57.51%), <i>V. vinifera</i> (13.91%), <i>Theobroma cacao</i> (3.05%), and others (25.52%) (Fig. S<a href="#MOESM2">5b, c</a> and d, Tables <a href="#MOESM1">S2</a> and <a href="#MOESM1">S3</a>). With regard to nonprotein-coding genes, we identified 431 tRNAs, 97 miRNAs, 222 rRNAs, 94 snRNAs, and 332 snoRNAs in our assembly (Table <a href="#MOESM1">S4</a>).</p> <p id="Par10">Repetitive sequences generally account for a substantial part of plant genomes and have a close relationship with genome size variation and functional adaptation<sup><a href="#CR23">23</a></sup>. A total of 485.55â€‰Mb (71%) of the <i>A. trifoliata</i> genome was identified as repetitive sequences, which is the same as the proportion in <i>P. somniferum</i> (71%)<sup><a href="#CR24">24</a></sup> and higher than that in <i>M. cordata</i> (63%)<sup><a href="#CR25">25</a></sup>, both of which are medicinal plants and basal eudicots. Approximately 87% of <i>A. trifoliata</i> repetitive sequences were classified as transposable elements (TEs) (Table <a href="#MOESM1">S5</a>). TEs occupy a significant fraction of many eukaryotic genomes and play an important role in the increase in genome size among angiosperms<sup><a href="#CR26">26</a></sup>. In our assembly, retrotransposon (Class I) and DNA transposon (Class II) TEs accounted for 72% and 7% of the genome, respectively. Among all TEs, long terminal repeats (LTRs) were the most abundant category of TEs, with 32% <i>gypsy</i> and 10% <i>copia</i>. Interestingly, the proportion of Penelope-like element/large retrotransposon derivative (LARD) elements in <i>A. trifoliata</i> was higher than that in most sequenced plant species, accounting for 21% of the genome. LARDs are considered to be nonautonomous elements and are the remnants of the deletion of autonomous LTR retrotransposons<sup><a href="#CR27">27</a></sup>. Recent lineage-specific radiation of LARDs (13% of the whole genome) in the pomegranate genome is responsible for fruit development, such as coloration, by affecting the expression of putative <i>UDP-glucose:flavonoid glucosyltransferase</i> (<i>UFGT</i>) and <i>MYB</i> genes<sup><a href="#CR27">27</a></sup>. The abundant LARD elements in the <i>A. trifoliata</i> genome might also have a close relationship with many important characteristics, including secondary metabolite accumulation, as in pomegranate.</p> </div> <div class="genefamilyanalysisandphylogenetictreeconstruction" title="sec"> <div class="title" tagx="title" title="title">
Gene family analysis and phylogenetic tree construction</div> <p id="Par11">We examined the evolutionary relationships among <i>A. trifoliata</i>, two eurosid species (<i>Arabidopsis thaliana</i>, <i>Vitis vinifera</i>), three basal eudicot species (<i>Aquilegia coerulea</i>, <i>Papaver somniferum</i>, and <i>Nelumbo nucifera</i>), two core eudicot oilseed species (<i>Sesamum indicum</i>, <i>Olea europaea</i>), a monocot species (<i>Oryza sativa</i>), and <i>Amborella trichopoda. A. trichopoda</i>, one of the basal angiosperm species, represents a sister group to other flowering plants<sup><a href="#CR28">28</a></sup>. By using gene family cluster analysis, we identified 12,831 gene families in <i>A. trifoliata</i>, of which 399 were unique gene families containing 1,028 genes (Fig. <a href="#Fig2">2A</a>, Table <a href="#MOESM1">S6</a>). Of those species<i>, P. somniferum</i> possessed the most genes (62,879) and unique gene families (2,394) owing to a relatively recent whole-genome duplication (WGD) event<sup><a href="#CR24">24</a></sup>. Then, we further compared the gene families among the four basal eudicot species. As shown in Fig. <a href="#Fig2">2B</a>, 10,268 gene families were shared by <i>A. coerulea</i>, <i>P. somniferum</i>, <i>N. nucifera</i>, and <i>A. trifoliata</i>. Compared with two other basal eudicot species, <i>P. somniferum</i> (2,662) and <i>A. coerulea</i> (1,085), <i>A. trifoliata</i> had fewer unique gene families (461). There were more shared gene family clusters between <i>A. trifoliata</i> and <i>A. coerulea</i> (11,611) than with any two of the other three species. Hence, we inferred a relatively close taxonomic relationship between the two species. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 2</span></a></div>   <div class="title" tagx="title" title="title">
Comparative genomics analysis of <i>A. trifoliata</i> subsp. <i>australis</i> and other representative plant species.</div> <p><b>A</b> Classification and statistics of common and lineage-specific genes in <i>A. trifoliata</i> and other representative plant species. <b>B</b> Venn diagram of gene families in <i>A. trifoliata</i> and three other basal eudicot species. <b>C</b> Phylogenetic analysis, gene family expansion/contraction analyses and divergence time estimations. The number of gene family contraction and expansion events are indicated by blue (the blue part of the pie chart) and red (the red part of the pie chart) numbers, respectively. Divergence times (million years ago, Mya) are indicated by the numbers beside the branch nodes. <b>D</b> Distribution of 4DTv among <i>A. trifoliata</i>, <i>N. nucifera</i>, and <i>V. vinifera</i> in intra- and intergenomic comparisons</p>   </div> <p /> <p id="Par12">In total, 197 conserved single-copy orthologs were identified from the 10 species and were used to construct the phylogenetic tree with <i>O. sativa</i> and <i>A. trichopoda</i> serving as outgroups (Fig. <a href="#Fig2">2C</a>). Phylogenetic analysis revealed that <i>A. trifoliata</i> is closely related to <i>A. coerulea</i> and forms a clade with <i>P. somniferum</i> and <i>N. nucifera</i>, which supports the well-established hypothesis of a close relationship between basal eudicot species<sup><a href="#CR25">25</a></sup>. Based on the phylogenetic tree, we determined that 246 and 473 gene families were expanded and contracted, respectively, in <i>A. trifoliata</i> (Fig. <a href="#Fig2">2C</a>, Table <a href="#MOESM1">S7</a>). Gene ontology analysis also revealed that â€œsupramolecular complexâ€� and â€œcell-killingâ€� appeared exclusively in expanded gene families, while â€œextracellular region partâ€�, â€œcell proliferationâ€�, and â€œbiological adhesionâ€� arose exclusively in contracted gene families (Fig. <a href="#MOESM2">S6</a>, Table <a href="#MOESM1">S8</a>).</p> </div> <div class="divergencetimeestimationandwhole-genomeduplicationanalysis" title="sec"> <div class="title" tagx="title" title="title">
Divergence time estimation and whole-genome duplication analysis</div> <p id="Par13">Using MCMCtree, we estimated that the divergence between <i>A. trifoliata</i> and <i>A. coerulea</i>, which occurred ~94.94â€‰Mya, whereas <i>P. somniferum</i> divergence occurred ~108.61â€‰Mya (Fig. <a href="#Fig2">2C</a>). The previous estimates indicate that the divergence between <i>A. coerulea</i> and <i>M. cordata</i> occurred approximately 115.24â€‰Mya<sup><a href="#CR25">25</a></sup>, suggesting a closer relationship between <i>A. coerulea</i> and <i>A. trifoliata</i> than between <i>A. coerulea</i> and <i>M. cordata</i>. The constructed tree provides the overall divergence time of important basal eudicot species. WGD has been found in almost all fundamental lineages of land plants and is considered a driver of diversity and adaptation<sup><a href="#CR29">29</a></sup>. The fourfold degenerate site transversion (4DTv) value revealed one peak at ~0.2 in <i>A. trifoliata</i>, which implies that it may have undergone WGD after divergence from <i>V. vinifera</i> and <i>N. nucifera</i> (Fig. <a href="#Fig2">2D</a>). An ancient paleohexaploidy (Î³) event, occurring ~125â€‰Mya, was detected in the common ancestor of all sequenced eudicot genomes<sup><a href="#CR30">30</a></sup>. According to the phylogenetic analysis and 4DTv values, we inferred that the Î³ event was absent in <i>A. trifoliata</i>, which is a basal eudicot species, as well as in the basal eudicot species lotus and <i>M. cordata</i><sup><a href="#CR25">25</a>,<a href="#CR31">31</a></sup>. The WGD of basal eudicot species should be further analyzed.</p> </div> <div class="ltrinsertiontime" title="sec"> <div class="title" tagx="title" title="title">
LTR insertion time</div> <p id="Par14">LTR retrotransposons play an important role in genome instability and evolution, which affect the expression and profiles of nearby genes and have significant consequences for phenotypic variation<sup><a href="#CR32">32</a></sup>. To investigate the insertion time of LTRs in the <i>A. trifoliata</i> genome, we estimated the intrasequence divergence of identified full-length LTR elements. We discovered that massive recent insertion events of LTRs occurred in <i>A. trifoliata</i> within the last one million years and in <i>A. coerulea</i> and <i>O. sativa</i>, which explains the accumulation of many recent LTRs (Fig. <a href="#Fig3">3A</a>). The LTR amplifications were inferred to have taken place during the Pleistocene epoch in which freezing occurred and there was limited atmospheric CO<sub>2</sub>. Changes in the climate and environment cause serious survival stresses that force evolutionary adaptation by reorganization of genomes, represented by activated TEs<sup><a href="#CR33">33</a></sup>. Such an ongoing amplification of LTRs may have contributed to an especially large proportion of LTRs in <i>A. trifoliata</i>. Furthermore, the estimation of insertion times of LTRs suggests that <i>N. nucifera</i> and <i>O. europaea</i> experienced amplification events 2â€"5 million years earlier than <i>A. trifoliata</i>. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 3</span></a></div>   <div class="title" tagx="title" title="title">
LTR insertion time estimation, enrichment analysis of expanded and contracted genes, and phylogenetic analysis of <i>Î²-amyrin synthase</i> genes.</div> <p><b>A</b> LTR insertion time analysis in <i>A. trifoliata</i> and other representative plant species. <b>B</b>, <b>C</b> KEGG enrichment analysis of expanded and contracted genes in <i>A. trifoliata</i>. Red indicates pathways related to triterpenoid biosynthesis and fatty-acid metabolism. <b>D</b> Phylogenetic analysis of <i>Î²-amyrin synthase</i> genes in <i>A. trifoliata</i> (red), rice, <i>Arabidopsis</i> and grape. The protein sequences of <i>Î²-amyrin synthase</i> in <i>Arabidopsis</i>, rice and grape were downloaded from the UniProt database. The phylogenetic tree was constructed using the ML method with 1000 bootstrap replicates</p>   </div> <p /> </div> <div class="determinationoffunctionalgenesinvolvedinterpenoidbiosynthesis" title="sec"> <div class="title" tagx="title" title="title">
Determination of functional genes involved in terpenoid biosynthesis</div> <p id="Par15">KEGG enrichment analysis indicated that the sesquiterpenoid and triterpenoid biosynthesis pathways and monoterpenoid biosynthesis pathway were significantly enriched in expanded genes (Fig. <a href="#Fig3">3B, C</a>). Notably, the 24 <i>Î²-amyrin synthase-like</i> (<i>AtrÎ²-AS</i>) genes involved in the sesquiterpenoid and triterpenoid biosynthesis pathways catalyzing the conversion of oxidosqualene to <i>Î²</i>-amyrin, the proposed aglycone of oleanane-type saponins<sup><a href="#CR10">10</a></sup>, were exclusively identified in the expanded gene set (Tables <a href="#MOESM1">S7</a> and <a href="#MOESM1">8</a>). Suppression of <i>Î²-AS</i> expression by RNA interference led to a reduced content of <i>Î²</i>-amyrin and oleanane-type saponins<sup><a href="#CR10">10</a></sup>. The systematic identification, prediction, and evolutionary analysis of <i>AtrÎ²-ASs</i> in <i>A. trifoliata</i> have been of great significance to our understanding of the synthesis and regulation of triterpene saponins. Phylogenetic analysis revealed that 24 <i>AtrÎ²-</i><i>AS</i> genes were classified into two clusters, which formed two monophyletic groups. One <i>AtrÎ²-</i><i>AS</i> gene (EMV0021809) was grouped together with three <i>VitÎ²-AS</i> genes (Fig. <a href="#Fig3">3D</a>). The expansion of <i>AtrÎ²-AS</i> genes might be an explanation for the high content of hederasaponin and oleanolic acid in many tissues of <i>A. trifoliata</i>.</p> <p id="Par16">In the <i>A. trifoliata</i> genome, 12 UDP-glucoronosyl and 3 UDP-glucosyltransferase gene families were expanded and contracted, respectively, while 7 and 14 cytochrome P450 gene families were expanded and contracted, respectively. The downstream reactions in the biosynthesis of saponins are believed to include a set of cytochrome P450-dependent hydroxylation/oxidations and several glycosyl transfer reactions catalyzed by glycosyltransferases<sup><a href="#CR34">34</a></sup>. The frequent expansion and contraction of the two gene families indicate a rapid evolution of these gene families, which might be the cause of the great variety of triterpenes in <i>A. trifoliata</i>. Unexpectedly, terpene synthase (TPS) gene families (PF01397, PF03936), which are responsible for the synthesis of various terpene molecules, were contracted in the <i>A. trifoliata</i> genome (Table <a href="#MOESM1">S7</a>). We performed genome-wide identification of the TPS gene family and identified 34 <i>AtrTPS</i> genes (Table <a href="#MOESM1">S9</a>), which is much fewer than that of <i>V. vinifera</i> with 95 <i>TPS</i> genes. Moreover, the <i>TPS</i> genes of <i>A. trifoliata</i> and the other three species were used to construct a phylogenetic tree (Fig. <a href="#MOESM2">S7</a>), and they grouped into four distinct clusters. All <i>AtrTPS</i> genes were distributed in different branches of Cluster I, while Clusters II, III and IV comprised <i>TPS</i> genes of rice, <i>V. vinifera</i> and <i>Arabidopsis</i>, respectively. These results suggest that there is functional diversification among the <i>TPS</i> genes in those species and the <i>AtrTPS</i> genes in <i>A. trifoliata</i>. Evolutionary plasticity is evident in the TPS family, represented by different product profiles, subcellular locations, activities, and substrates.</p> </div> <div class="molecularfoundationforufaaccumulation" title="sec"> <div class="title" tagx="title" title="title">
Molecular foundation for UFA accumulation</div> <p id="Par17">The oil of <i>A. trifoliata</i> seeds has been suggested to have health benefits because of its high UFA content<sup><a href="#CR3">3</a></sup>. To further study FAs metabolism in <i>A. trifoliata</i> seeds, we determined the FAs composition and gene expression profiles at different stages of seed development (Fig. <a href="#Fig4">4A</a>). The total FA content increased 15-fold from May (17.8â€‰mgâ€‰g<sup>âˆ’1</sup>) to September (260.9â€‰mgâ€‰g<sup>âˆ’1</sup>), and FA accumulated dramatically from June (30.4â€‰mgâ€‰g<sup>âˆ’1</sup>) to July (158.3â€‰mgâ€‰g<sup>âˆ’1</sup>) (Fig. <a href="#Fig4">4B</a>). The time courses for the levels of the individual FAs were similar to that of the total FA content (Fig. <a href="#Fig4">4C</a>). A sharp rise (six- to sevenfold) in the contents of oleic acid, linoleic acid, and palmitic acid was also observed from June to July. The linolenic acid levels remained stable at ~2â€‰mgâ€‰g<sup>âˆ’1</sup> during seed development. In May, linoleic acid (32%), palmitic acid (23%), and stearic acid (23%) were the dominant FAs in seeds, whereas oleic acid (3.8%) was much less abundant. With increasing seed maturity, we observed a significant increase in the proportion of oleic acid (34%) and linoleic acid (37%) and a reduced proportion of palmitic acid (19%), stearic acid (7%), and linolenic acid (0.95%) (Fig. <a href="#Fig4">4D</a>). In summary, the FA contents increased with seed development, and UFAs, including linoleic acid and oleic acid, accumulated especially rapidly during the early stages of seed development. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 4</span></a></div>   <div class="title" tagx="title" title="title">
Changes in the seed content and percentage of total fatty acids (FAs) at different developmental stages.</div> <p><b>A</b> Photos of seeds at different developmental stages. The scale bar is in cm. <b>B</b> Changes in the content of total FAs. <b>C</b> Changes in FA contents. <b>D</b> Changes in the percentage of FAs</p>   </div> <p /> <p id="Par18">Transcriptomic analysis revealed that the gene expression profiles of seeds in July and August were closer to each other and obviously differed from those of the three other stages (Fig. <a href="#Fig5">5A</a>). A total of 2420, 1331, 2170, and 3118 upregulated differentially expressed genes (DEGs) and 5877, 3929, 1144, and 3871 downregulated DEGs were identified compared with those of the previous month (Fig. <a href="#Fig5">5B</a>). There were more DEGs between May vs June and August vs September, which agrees with the principal component analysis results, demonstrating more marked changes in gene expression profiles at the early and later stages of seed development. KEGG enrichment analysis indicated that many DEGs were enriched in lipid metabolism pathways (Fig. <a href="#Fig5">5C</a>, Table <a href="#MOESM1">S10</a>). An expression heat map of these DEGs linked to FA metabolism, integrating the changes in FA contents during the development of <i>A. trifoliata</i> seeds, can provide valuable information about the regulation of oil accumulation and UFA synthesis. The reconstruction of the FA and TAG biosynthetic pathways includes <i>de novo</i> formation of acyl chains in the plastid and TAG assembly in the endoplasmic reticulum (Fig. <a href="#Fig6">6</a>). Almost all DEGs involved in the <i>biotin carboxyl carrier protein (BCCP) of acetyl-CoA carboxylase</i> (<i>ACCase</i>) and <i>fatty-acid synthase</i> (<i>FAS</i>) complexes including <i>Î²-ketoacyl-ACP synthase</i> (<i>KAS</i>), <i>Î²-hydroxacyl-ACP dehydratase</i> (<i>HAD</i>), and <i>enoyl-ACP reductase</i> (<i>EAR</i>), showed maximal transcription in June, followed by July. The transcription peak of the core FA biosynthetic machinery coincided with the onset of oil accumulation in the seed at the early developmental stage. <i>ACCase</i> catalyzes the conversion of acetyl-CoA into malonyl-CoA and is the rate-limiting enzyme in fatty-acid biosynthesis<sup><a href="#CR35">35</a>,<a href="#CR36">36</a></sup>. It has been proposed that lipid biosynthesis can be increased by overexpressing <i>ACCase</i><sup><a href="#CR37">37</a>,<a href="#CR38">38</a></sup>. <i>KASI</i> catalyzes the elongation of <i>de novo</i> fatty acids, and <i>KASI</i> mutation results in a significant reduction in FA contents in seeds<sup><a href="#CR39">39</a></sup>. <i>EAR</i> catalyzes a key regulatory step in FA biosynthesis and shows the highest expression during the early stages of seed development<sup><a href="#CR40">40</a></sup>. In the oil palm mesocarp, the key FA biosynthesis genes were highly expressed at 120 days after pollination when oil accumulation began in the mesocarp<sup><a href="#CR41">41</a></sup>. We inferred that the significant upregulation of <i>ACCase</i> and <i>FAS</i> in seeds in June has a close relationship with the rapid increase in the oil content in July. Additionally, three <i>acyl-ACP thioesterases</i> (<i>FATA/B)</i> and three <i>long-chain acyl-CoA synthetases</i> (<i>LACS)</i> were also highly expressed in June. <i>FATA</i> and <i>FATB</i> play an essential role in chain termination during fatty-acid synthesis. LACS mediates FA transport and conversion to acyl-CoA<sup><a href="#CR41">41</a></sup>. However, many DEGs involved in triacylglycerol (TAG) biosynthesis, such as <i>glycerol kinase</i> (<i>GK</i>), <i>glycerol-3-phosphate acyltransferase</i> (<i>GPAT</i>), <i>1-acylglycerol-3-phosphate acyltransferase</i> (<i>LPAAT</i>), <i>phosphatidate phosphatase</i> (<i>PAP</i>), <i>acyl-CoA:diacylglycerol acyltransferase</i> (<i>DGAT</i>), and <i>phospholipid:diacylglycerol acyltransferase</i> (<i>PDAT</i>), were highly expressed throughout seed development. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 5</span></a></div>   <div class="title" tagx="title" title="title">
Transcriptomic analysis of seeds at different developmental stages.</div> <p><b>A</b> PCA of different expression profiles in seeds. <b>B</b> Statistics on the numbers of differentially expressed genes (DEGs) in seeds at different developmental stages. <b>C</b> KEGG enrichment pathways in seeds at different developmental stages. The bar indicates the <i>p</i> values</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 6</span></a></div>   <div class="title" tagx="title" title="title">
The expression levels of enzymes involved in FA synthesis and TAG assembly in developing <i>A. trifoliata</i> seeds.</div> <p>Gene expression levels in May, June, July, August, and September are indicated with colored bars (log2 FPKM). <i>ACCase</i> acetyl-CoA carboxylase, <i>CPT</i> diacylglycerol cholinephosphotransferase, <i>DGAT</i> acyl-CoA:diacylglycerol acyltransferase, <i>EAR</i> enoyl-acyl carrier protein (ACP) reductase, <i>FAD2</i> oleate desaturase, <i>FAD3</i> linoleate desaturase, <i>FATA/B</i> acyl-ACP thioesterase A/B, <i>GK</i> glycerol kinase, <i>GPAT</i> glycerol-3-phosphate acyltransferase, <i>HAD</i> hydroxyacyl-ACP dehydrase, <i>KAR</i> ketoacyl-ACP reductase, <i>KASI</i> ketoacyl-ACP synthase I, <i>KAS II</i> ketoacyl-ACP synthase II, <i>KAS III</i> ketoacyl-ACP synthase III, <i>KCS</i> 3-ketoacyl-CoA synthase, <i>LACS</i> long-chain acyl-CoA synthetase, <i>LPAAT</i> 1-acylglycerol-3-phosphate acyltransferase, <i>PAP</i> phosphatidate phosphatase, <i>LPCAT</i> 1-acylglycerol-3-phosphocholine acyltransferase, <i>MAT</i> malonyl-CoA:ACP malonyltransferase, <i>PDAT</i> phospholipid:diacylglycerol acyltransferase, <i>SAD</i> stearoyl-ACP desaturase. The colored boxes represent log<sub>2</sub> FPKM values of DEGs in developing seeds, and sample names are shown on the lower right-hand side</p>   </div> <p /> <p id="Par19">Comparative genomics and enrichment analyses indicated that linoleic acid metabolism, Î±-linolenic acid metabolism, UFA, and FA biosynthesis were enriched in an expanded gene set (Fig. <a href="#Fig3">3B</a>). The expansion of those genes involved in lipid metabolism may be responsible for the high UFA and oil contents in <i>A. trifoliata</i> seeds. Among those expanded gene families, an acyl-ACP desaturase gene family (PF03405.9), including 12 <i>AtrSAD</i> genes involved in UFA biosynthesis, was expanded in the <i>A. trifoliata</i> genome. Furthermore, two <i>AtrSADs</i> (EVM0019545 and EVM0019590) were highly expressed during seed development and showed significantly increased expression levels in seeds in July (Fig. <a href="#Fig7">7</a>), which may contribute to the high oleic acid content in <i>A. trifoliata</i> seeds. Three omega <i>FAD</i> genes, <i>FAD2</i> (EVM0006409), <i>FAD6</i> (EVM0001669), and <i>FAD7</i> (EVM0000977), also remained at a high expression level during seed development (Fig. <a href="#Fig7">7A, B</a>), especially <i>FAD2. SAD</i> and <i>FAD2</i> play key roles in the synthesis of UFAs<sup><a href="#CR42">42</a></sup>. The suppression of <i>FAD2</i> expression by siRNA leads to a low linoleic acid content (5.5%) in olive oil<sup><a href="#CR17">17</a></sup>. It is noteworthy that homologs of oleosin (EVM0009451, EVM0021590, EVM0024046, EVM0018287, and EVM0012715), encoding oil body proteins that assist with packaging of TAG and determining oil body size, showed a sharp increase and a significantly higher expression in seeds from June to September compared with that in May (Fig. <a href="#Fig7">7A, C</a>). For example, the FPKM values of the two <i>AtrOLEs</i> (EVM0021590 and EVM0018287) in seeds were 441- and 366-fold higher in June than in May, respectively, which was confirmed by qRT-PCR (Fig. <a href="#Fig7">7C</a>). The results indicate that the significant upregulation of <i>ACCase</i> and <i>FAS</i> in FA biosynthesis, the high levels of <i>SAD</i> and <i>FAD2</i> in FA desaturation, and the stabilization of oil bodies by oleosin allow <i>A. trifoliata</i> to accumulate high levels of oil. In <i>Jatropha</i>, increased expression of FA biosynthesis genes and oleosins synergistically results in the accumulation of high levels of oil in kernels (~63%)<sup><a href="#CR43">43</a></sup>. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 7</span></a></div>   <div class="title" tagx="title" title="title">
The expression levels of SADs, FADs, and oleosins in seeds at different developmental stages.</div> <p><b>A</b> The log<sub>2</sub> (FPKM) values. <b>B</b>, <b>C</b> The expression levels during seed development determined by RNA-Seq and qRT-PCR</p>   </div> <p /> <p id="Par20">A total of 25 FADs (acyl-lipid desaturase and acyl-ACP desaturase) in <i>A. trifoliata</i> were identified (Table <a href="#MOESM1">S11</a>). To evaluate their evolutionary relationships and predict their gene functions, all the <i>FAD</i> genes from <i>A. trifoliata</i>, <i>Arabidopsis</i>, rice, <i>Brassica napus</i>, <i>S. indicum</i>, and <i>O. europaea</i> were aligned to construct an unrooted ML phylogenetic tree (Fig. <a href="#Fig8">8</a>). All <i>AtrSADs</i>, except for <i>Atr0019590</i>, formed a well-defined monophyletic group, suggesting that <i>SAD</i> extension occurred after the divergence of <i>A. trifoliata</i> and other species. Notably, all identified <i>AtrFAD2</i> and <i>AtrFAD6</i> genes, as well as one <i>AtrSAD</i> (<i>Atr0019590</i>) gene, clustered with the homologs of <i>S. indicum</i> and <i>O. europaea</i>, which suggests that these genes perform similar functions in the three species. The expansion and neofunctionalization of <i>SADs</i> in oleasters are likely also responsible for the higher oleic acid contents in <i>A. trifoliata</i> than in sesame<sup><a href="#CR17">17</a></sup>. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 8</span></a></div>   <div class="title" tagx="title" title="title">
Phylogenetic relationship of FAD genes from <i>A. trifoliata</i> subsp. <i>australis</i>, <i>Arabidopsis thaliana</i>, <i>Oryza sativa</i>, <i>Sesamum indicum</i>, <i>Olea europaea</i>, and <i>Brassica rapa</i>.</div> <p>An ML phylogenetic tree of all detected FADs was constructed using RAxML with 1000 bootstrap replicates</p>   </div> <p /> <p id="Par21">In summary, genome sequencing of <i>A. trifoliata</i> has provided crucial information for the systematic study of the biosynthesis and metabolism of triterpenes in this medicinally and economically important nonmodel plant species. The <i>A. trifoliata</i> genome represents a useful resource for the genetic improvement of this plant species and for better understanding its genome evolution.</p> </div> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Materials and methods</div> <div class="samplingandsequencing" title="sec"> <div class="title" tagx="title" title="title">
Sampling and sequencing</div> <p id="Par22">Young <i>A. trifoliata</i> subsp. <i>australis</i> leaves were collected from Huaihua, Hunan Province, China (N27Â°33â€™17.95â€�, E109Â°59â€™54.70â€�). A modified cetyl trimethylammonium bromide method<sup><a href="#CR44">44</a></sup> was used for DNA extraction. The DNA was used to construct a 20â€‰kb insert-sized SMRTbell library for PacBio sequencing and 350â€‰bp insert-size paired-end libraries for Illumina short-read sequencing. These libraries were prepared according to the manufacturerâ€™s protocol (PacBio, CA, USA and Illumina, CA, USA). For RNA-seq, total RNA was extracted from the leaves, roots, and stems of <i>A. trifoliata</i>, and a mixture was made by combining an equal amount from each. After removing genomic DNA using DNase I (Takara), mRNAs were obtained using oligo (dT) beads and broken into short fragments, followed by cDNA synthesis. Paired-end sequencing was conducted on the HiSeq X Ten platform (Illumina, CA, USA). All PacBio and Illumina sequencing procedures were performed by the BioMarker Technologies Company (Beijing, China).</p> </div> <div class="genomesizeestimation" title="sec"> <div class="title" tagx="title" title="title">
Genome size estimation</div> <p id="Par23">Flow cytometry was used to estimate the genome size of <i>A. trifoliata</i> according to the method described by Huang et al.<sup><a href="#CR45">45</a></sup>. First, young fresh leaves of <i>A. trifoliata</i> were chopped in nuclear isolation buffer. The supernatant was filtered through a 50-Î¼m CellTrics filter. After being treated with RNase, cell nuclei were stained with propidium iodide in the dark. The fluorescence intensity of the sample was determined using a flow cytometer (BD FACSVerse). Maize B73 was used as an internal standard. The genome size of <i>A. trifoliata</i> was evaluated by k-mer frequency analysis using Illumina short reads. After removing contaminants, the optimal k-mer size was analyzed using KmerGenie<sup><a href="#CR46">46</a></sup>. Then, Jellyfish was used to analyze the k-mer counts, which were used to estimate the genome size and heterozygosity<sup><a href="#CR47">47</a></sup>.</p> </div> <div class="genomeassemblyandevaluationofgenomequality" title="sec"> <div class="title" tagx="title" title="title">
Genome assembly and evaluation of genome quality</div> <p id="Par24">The PacBio Sequel long reads and Illumina short reads were combined to perform a de novo assembly of the <i>A. trifoliata</i> genome. Canu (v1.4) was first used to assemble the genome with the corrected-error-rate parameter<sup><a href="#CR48">48</a></sup>. Then, the corrected reads were independently assembled with WTDBG (v1.2.3)<sup><a href="#CR49">49</a></sup>. The well-assembled Canu and WTDBG results were merged by Quickmerge<sup><a href="#CR50">50</a></sup>. The merged genome was corrected with the Illumina data using Pilon<sup><a href="#CR51">51</a></sup>.</p> <p id="Par25">For the evaluation of assembly coverage, all paired-end short reads were mapped to our assembly using BWA<sup><a href="#CR52">52</a></sup>. RNA-Seq data from leaf, root, and stem tissues were assembled by Trinity<sup><a href="#CR53">53</a></sup>. HISAT2 was used to map all expressed sequence tags generated from RNA-Seq to the assembly with the default settings<sup><a href="#CR54">54</a></sup> to evaluate gene completeness. The completeness of the assembly was assessed by the CEG mapping approach (v2.5) (CEGMA)<sup><a href="#CR55">55</a></sup> and benchmarking universal single-copy ortholog (v4.0) (BUSCO) analysis<sup><a href="#CR56">56</a></sup>.</p> </div> <div class="chromosome-scaleassemblywithhi-cdata" title="sec"> <div class="title" tagx="title" title="title">
Chromosome-scale assembly with Hi-C data</div> <p id="Par26">Hi-C libraries were constructed from DNA extracted from fresh leaf tissue of <i>A. trifoliata</i>, similar to what was used for genome assembly, as previously described<sup><a href="#CR57">57</a></sup>. The purified and enriched DNA was used for sequencing using the Illumina HiSeq X ten platform. A total of 103â€‰Gb of clean data (151-fold the estimated genome size) was obtained and aligned to the PacBio assembly contigs using BOWTIE2<sup><a href="#CR58">58</a></sup>. The valid paired reads required for genome assembly were defined as uniquely mapped paired-end reads. Hi-C unique mapped paired-end reads were then applied to scaffold the assembled genome using the LACHESIS program<sup><a href="#CR59">59</a></sup>. The mapped read pairs were clustered into different chromosomal groups based on agglomerative hierarchical clustering. LACHESIS iterated all the possibilities of scaffold orientation and generated finely oriented scaffolds using a weighted directed acyclic graph.</p> </div> <div class="genomeannotation" title="sec"> <div class="title" tagx="title" title="title">
Genome annotation</div> <p id="Par27">Repeat elements were identified by combining de novo- and homology-based approaches. RepeatModeler<sup><a href="#CR60">60</a></sup>, LTR_Finder<sup><a href="#CR61">61</a></sup>, and RepeatScout<sup><a href="#CR62">62</a></sup> were used to construct a repeat library for de novo prediction. Based on the repeat sequence database, homology prediction was conducted using RepeatProteinMask and RepeatMasker<sup><a href="#CR63">63</a></sup> against the Repbase TE library<sup><a href="#CR64">64</a></sup> and the TE protein database. Noncoding RNA was annotated using tRNAscan-SE<sup><a href="#CR65">65</a></sup> (for tRNAs) or INFERNAL<sup><a href="#CR66">66</a></sup> (for miRNAs and snRNAs). The rRNAs were identified using BLASTN alignment and RNAammer<sup><a href="#CR67">67</a></sup>.</p> <p id="Par28">Multiple gene prediction methods integrating homolog-, de novo-, and transcriptome-based gene prediction were used to annotate protein-coding genes. For homologous prediction, a gene set including proteins from four plant genomes (<i>Arabidopsis thaliana, A. coerulea, Opium poppy</i>, and <i>M. cordata</i>) was mapped to the assembly of the <i>A. trifoliata</i> genome by BLAST<sup><a href="#CR68">68</a></sup>, and then GeneWise software was used to provide an accurate gene structure prediction<sup><a href="#CR69">69</a></sup>. RNA-Seq reads were first aligned to our genome assembly using HISAT2<sup><a href="#CR54">54</a></sup>, and then StringTie<sup><a href="#CR70">70</a></sup> was used to assemble the alignments into gene models in a transcriptome-based prediction. <i>De novo</i> identification was performed using Augustus (v2.5.5)<sup><a href="#CR71">71</a></sup>, GENSCAN (v1.0)<sup><a href="#CR72">72</a></sup>, SNAP<sup><a href="#CR73">73</a></sup>, and GlimmerHMM (v3.0.1)<sup><a href="#CR74">74</a></sup>. All the resulting genes mentioned above were integrated using EVidenceModeler<sup><a href="#CR75">75</a></sup>.</p> <p id="Par29">The protein-coding genes in <i>A. trifoliata</i> were blasted with an <i>E</i> value cutoff of 1.0â€‰Ã—â€‰10<sup>âˆ’5</sup> against SwissProt, NR, and the Kyoto Encyclopedia of Genes and Genomes (KEGG)<sup><a href="#CR76">76</a></sup> for functional annotation. InterProScan (v4.8)<sup><a href="#CR77">77</a></sup> and HMMER (v3.1)<sup><a href="#CR78">78</a></sup> were used to annotate protein domains against the InterPro (v32.0)<sup><a href="#CR79">79</a></sup> and Pfam (v27.0)<sup><a href="#CR80">80</a></sup> databases, respectively. GO<sup><a href="#CR81">81</a></sup> terms were obtained and grouped into three categories based on the results from the InterPro and Pfam entries.</p> </div> <div class="datingofltrretrotransposonelements" title="sec"> <div class="title" tagx="title" title="title">
Dating of LTR retrotransposon elements</div> <p id="Par30">Intact LTR retrotransposons were identified by searching the genomes of <i>A. trifoliata</i> with LTR_Finder<sup><a href="#CR61">61</a></sup> and LTR_STRUC<sup><a href="#CR82">82</a></sup>. According to the sequence divergence, the insertion times of the identified full-length LTR retrotransposons were estimated with Dismat (EMBOSS package)<sup><a href="#CR83">83</a></sup>. The average base substitution rate of 1.3E-08 per site per year was used to calculate the insertion times<sup><a href="#CR84">84</a></sup>.</p> </div> <div class="genefamily,phylogeneticanalysis,anddivergencetimeestimation" title="sec"> <div class="title" tagx="title" title="title">
Gene family, phylogenetic analysis, and divergence time estimation</div> <p id="Par31">We collected the protein sequences from <i>A. trifoliata</i> and nine other plant species, namely, <i>A. thaliana</i>, <i>O. sativa</i>, <i>V. vinifera</i>, <i>A. coerulea</i>, <i>N. nucifera</i>, <i>P. somniferum</i>, <i>O. europaea</i>, <i>S. indicum</i>, and <i>A. trichopoda</i>, for gene family clustering. We conducted all-versus-all protein sequence queries through BLASTP with an <i>E</i> value of 1.0â€‰Ã—â€‰10<sup>âˆ’5</sup>. OrthoMCL<sup><a href="#CR85">85</a></sup> was used to cluster paralogous and orthologous groups. The four basal eudicot species, <i>A. trifoliata</i>, <i>A. coerulea</i>, <i>P. somniferum</i>, and <i>N. nucifera</i>, were further analyzed to explore their species-specific and shared gene families. The expansion and contraction of the gene family were analyzed using CAFE software<sup><a href="#CR86">86</a></sup>.</p> <p id="Par32">Following alignment by MUSCLE alignment software<sup><a href="#CR87">87</a></sup>, all single-copy genes identified and shared by the ten abovementioned species were used for evolutionary analysis and phylogenetic tree (ML Tree) reconstruction using RAxML software<sup><a href="#CR88">88</a></sup>. The divergence time was estimated using the MCMCtree program within the PAML package<sup><a href="#CR89">89</a></sup>. The divergence times were calibrated with the TimeTree database<sup><a href="#CR90">90</a></sup>.</p> </div> <div class="whole-genomeduplicationanalysis" title="sec"> <div class="title" tagx="title" title="title">
Whole-genome duplication analysis</div> <p id="Par33">Protein sequences were aligned against each other with BLASTP with an <i>E</i> value â‰¤â€‰1â€‰Ã—â€‰10<sup>âˆ’5</sup> to identify conserved paralogous and orthologous genes in <i>A. trifoliata, N. nucifera</i>, and <i>Vitis vinifera</i>. The 4DTv values were calculated using the HKY model<sup><a href="#CR23">23</a></sup>. Then, potential WGD events in the genome were evaluated based on the 4DTv value.</p> </div> <div class="transcriptomeanalysisofdevelopingseedsandqrt-pcrverification" title="sec"> <div class="title" tagx="title" title="title">
Transcriptome analysis of developing seeds and qRT-PCR verification</div> <p id="Par34"><i>A. trifoliata</i> seeds were collected at 20, 50, 80, 110, and 140 days after flowering in May, June, July, August, and September, respectively. Library construction and RNA-Seq were performed as mentioned above. Clean reads were mapped to the reference genomes by HISAT2<sup><a href="#CR54">54</a></sup>. The expression level (fragments per kilobase of transcript per million fragments mapped, FPKM value) of unigenes was calculated by StringTie<sup><a href="#CR65">65</a></sup>. DEGs were identified by DESeq2<sup><a href="#CR91">91</a></sup> (adjusted <i>P</i> value, FDRâ€‰<div class="â€‰0.05)._UNKNOWN" href="http://www.genome.jp/kegg">
http://www.genome.jp/kegg). Pathways with <i>P</i>â€‰<div class="â€‰0.05_UNKNOWN"><a href="#CR22">22</a>. The <i>actin</i> gene was used as a reference in all experiments. Primers used for qRT-PCR are listed in Table <a href="#MOESM1">S11</a>. The qRT-PCR results were derived from three repeated reactions for each gene and sample. Fold change was calculated using the formula 2<sup>âˆ’Î"Î"Ct</sup>.</div></div></p> </div> <div class="genome-wideidentificationandphylogeneticanalysisofgenefamilies" title="sec"> <div class="title" tagx="title" title="title">
Genome-wide identification and phylogenetic analysis of gene families</div> <p id="Par35">The genome and protein sequences of <i>A. thaliana, O. sativa, Vitis vinifera, O. europaea, S. indicum</i>, and <i>Brassica napus</i> were downloaded from the NCBI database. The hmmsearch program of HMMER software (version 3.2.1) (<a href="http://hmmer.org/download.html">http://hmmer.org/download.html</a>) was also applied to the identification of TPSs (PF01397, PF03936) and FADs (PF00487, PF03405) in Pfam 32.0 data (<a href="http://pfam.xfam.org/">http://pfam.xfam.org/</a>). To classify and investigate the phylogenetic relationships of the <i>amyrin synthase-like, TPS</i>, and <i>FAD</i> genes, the predicted genes were aligned using MUSCLE. Hence, two ML (maximum-likelihood) phylogenetic trees were constructed using RAxML software. The bootstrap test was performed with 1000 replicates to obtain high reliability of interior branches. The phylogenetic tree was imported to iTOL (<a href="https://itol.embl.de/">https://itol.embl.de/</a>) for visualization<sup><a href="#CR92">92</a></sup>.</p> </div> <div class="assayoffatty-acidcomposition" title="sec"> <div class="title" tagx="title" title="title">
Assay of fatty-acid composition</div> <p id="Par36">The seeds at different developmental stages used in transcriptomic analysis were used to determine the FA composition. FA extraction and analysis were conducted according to Liu et al.<sup><a href="#CR93">93</a></sup>. Fifty milligrams of seed powder (fresh weight) was treated using chloroform-methanol solution (V/Vâ€‰=â€‰2:1). Ultrasonic technology was used for FA extraction. Fatty acyl methyl esters (FAMEs) were prepared by direct transesterification of FA with 1% sulfuric acid in methanol at 80â€‰Â°C for 30â€‰min. The FAMEs were extracted with 1â€‰mL hexane and analyzed by gas chromatography-mass spectrometry (GC-MS) with methyl heptadecanoate as an internal standard. GC-MS analysis was performed on an Agilent 6890â€‰N/5975B (Agilent, USA) equipped with an Agilent HP-INNOWAX column (30â€‰mâ€‰Ã—â€‰0.25â€‰mm IDâ€‰Ã—â€‰0.25â€‰Âµm). The column temperature was raised from 150â€‰Â°C to 230â€‰Â°C at a rate of 10â€‰Â°Câ€‰min<sup>âˆ’1</sup> and then increased to 250â€‰Â°C and maintained for 10â€‰min. Peaks were identified by comparing the retention times with those of the corresponding standards (Sigma), and their identities were also confirmed by comparing mass spectra to the National Institute of Standards and Technology mass spectral library. The concentration of each sample was normalized according to the internal control.</p> <p id="Par37">Supplementary information accompanies the manuscript on the Horticulture Research website <a href="http://www.nature.com/hortres">http://www.nature.com/hortres</a></p> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary information</div> <div class="sec" title="sec"> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="41438_2020_458_MOESM1_ESM.xls">LINK</a> <p>Supplementary table S3 and S8</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="41438_2020_458_MOESM2_ESM.doc">LINK</a> <p>Supplementary figures and tables</p>  </div> </div> </div> </div> </div> <div class="back" title="back"> <div class="supplementaryinformation" title="sec"> <div class="title" tagx="title" title="title">
Supplementary information</div> <p><b>Supplementary Information</b> accompanies this paper at (10.1038/s41438-020-00458-y).</p> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>We are very grateful to professor Jay J. Thelen (University of Missouri, USA) for constructive criticism and helpful suggestions. This work was funded by the Natural Science Foundation of Hunan Province (2019JJ50475), Key Scientific Research Projects of Hunan Education Department (18A448), Foundation of Hunan Double First-rate Discipline Construction Projects of Bioengineering and Key Laboratory of Research and Utilization of Ethnomedicinal Plant Resources of Hunan Province, and the National Science Foundation (81874334).</p> </div> <div class="title" tagx="title" title="title">
Author contributions</div> <p>S.Q.S. and H.H. conceived and designed the experiments. H.H., J.L., Q.T., L.F.O., X.L.L., C.H.Z., H.L.H., and W.X.J. collected materials and performed the experiments. H.H. and I.M.M. analyzed the data. H.H. wrote the paper, which was modified by I.M.M. and S.Q.S. All authors read and approved the final manuscript.</p> <div class="title" tagx="title" title="title">
Data availability</div> <p>The <i>A. trifoliata</i> subsp. <i>australis</i> genome sequences, gene annoataion information, raw sequence data of genome sequencing and RNA-seq have been deposited under BioProject accession number PRJNA685604.</p> <div class="title" tagx="title" title="title">
Conflict of interest</div> <p id="Par38">The authors declare that they have no conflict of interest.</p> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="CR1" /><span class="label" tagx="label" title="label">1.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">GY</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ma</span><span class="given-names" tagx="given-names" title="given-names">SC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zheng</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lin</span><span class="given-names" tagx="given-names" title="given-names">RC</span></span></span><span class="mixed-article-title" title="mixed-article-title">Two new triterpenoid saponins from <i>Akebia quinata</i> (Thunb.) Decne</span><span class="source" tagx="source" title="source">J. Integr. Plant Biol.</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">49</span><span class="fpage" tagx="fpage" title="fpage">196</span><span class="lpage" tagx="lpage" title="lpage">201</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1744-7909.2007.00362.x">10.1111/j.1744-7909.2007.00362.x</a></span></span></li> <li tag="ref"><a name="CR2" /><span class="label" tagx="label" title="label">2.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Chinese Pharmacopoeia Commission. <i>The Pharmacopoeia of the Peopleâ€™s Republic of China. Beijing: China medicine science and technology press</i> (2015).</span></li> <li tag="ref"><a name="CR3" /><span class="label" tagx="label" title="label">3.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Du</span><span class="given-names" tagx="given-names" title="given-names">YX</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Physicochemical and functional properties of the protein isolate and major fractions prepared from <i>Akebia trifoliata</i> var. <i>australis</i> seed</span><span class="source" tagx="source" title="source">Food Chem.</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">133</span><span class="fpage" tagx="fpage" title="fpage">923</span><span class="lpage" tagx="lpage" title="lpage">929</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.foodchem.2012.02.005">10.1016/j.foodchem.2012.02.005</a></span></span></li> <li tag="ref"><a name="CR4" /><span class="label" tagx="label" title="label">4.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">XY</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">The profiling of bioactives in <i>Akebia trifoliata</i> pericarp and metabolites, bioavailability and in vivo anti-inflammatory activities in DSS-induced colitis mice</span><span class="source" tagx="source" title="source">Food Funct.</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">3977</span><span class="lpage" tagx="lpage" title="lpage">3991</span><span class="pub-id"><a href="https://dx.doi.org/10.1039/C9FO00393B">10.1039/C9FO00393B</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31204754">31204754</a></span></span></li> <li tag="ref"><a name="CR5" /><span class="label" tagx="label" title="label">5.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lu</span><span class="given-names" tagx="given-names" title="given-names">WL</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title"><i>Akebia trifoliata</i> (Thunb.) Koidz seed extract inhibits human hepatocellular carcinoma cell migration and invasion in vitro</span><span class="source" tagx="source" title="source">J. Ethnopharmacol.</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">234</span><span class="fpage" tagx="fpage" title="fpage">204</span><span class="lpage" tagx="lpage" title="lpage">215</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jep.2018.11.044">10.1016/j.jep.2018.11.044</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30528882">30528882</a></span></span></li> <li tag="ref"><a name="CR6" /><span class="label" tagx="label" title="label">6.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiang</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shi</span><span class="given-names" tagx="given-names" title="given-names">SP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cao</span><span class="given-names" tagx="given-names" title="given-names">JJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gao</span><span class="given-names" tagx="given-names" title="given-names">QP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tu</span><span class="given-names" tagx="given-names" title="given-names">PF</span></span></span><span class="mixed-article-title" title="mixed-article-title">Triterpene saponins from the fruits of <i>Akebia quinata</i></span><span class="source" tagx="source" title="source">Biochem. Syst. Ecol.</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">36</span><span class="fpage" tagx="fpage" title="fpage">138</span><span class="lpage" tagx="lpage" title="lpage">141</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.bse.2007.06.004">10.1016/j.bse.2007.06.004</a></span></span></li> <li tag="ref"><a name="CR7" /><span class="label" tagx="label" title="label">7.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Antibacterial oleanane-type triterpenoids from pericarps of <i>Akebia trifoliata</i></span><span class="source" tagx="source" title="source">Food Chem.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">168</span><span class="fpage" tagx="fpage" title="fpage">623</span><span class="lpage" tagx="lpage" title="lpage">629</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.foodchem.2014.07.105">10.1016/j.foodchem.2014.07.105</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25172756">25172756</a></span></span></li> <li tag="ref"><a name="CR8" /><span class="label" tagx="label" title="label">8.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vranova</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Coman</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gruissem</span><span class="given-names" tagx="given-names" title="given-names">W</span></span></span><span class="mixed-article-title" title="mixed-article-title">Network analysis of the MVA and MEP pathways for isoprenoid synthesis</span><span class="source" tagx="source" title="source">Annu. Rev. Plant Biol.</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">64</span><span class="fpage" tagx="fpage" title="fpage">665</span><span class="lpage" tagx="lpage" title="lpage">700</span><span class="pub-id"><a href="https://dx.doi.org/10.1146/annurev-arplant-050312-120116">10.1146/annurev-arplant-050312-120116</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23451776">23451776</a></span></span></li> <li tag="ref"><a name="CR9" /><span class="label" tagx="label" title="label">9.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Kushiro, T. &amp;amp; Ebizuka, Y. â€œTriterpenesâ€�. In Mander L., Liu H. W. B. (eds) Comprehensive Natural Products II: Chemistry and Biology. p 673â€"708 (Elsevier, Oxford, 2010).</span></li> <li tag="ref"><a name="CR10" /><span class="label" tagx="label" title="label">10.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Functional analysis of Î²-amyrin synthase gene in ginsenoside biosynthesis by RNA interference</span><span class="source" tagx="source" title="source">Plant Cell Rep.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">1307</span><span class="lpage" tagx="lpage" title="lpage">1315</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s00299-015-1788-7">10.1007/s00299-015-1788-7</a></span></span></li> <li tag="ref"><a name="CR11" /><span class="label" tagx="label" title="label">11.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">MiliÄ‡eviÄ‡</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">The role of total fats, saturated/unsaturated fatty acids and cholesterol content in chicken meat as cardiovascular risk factors</span><span class="source" tagx="source" title="source">Lipids Health Dis.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">13</span><span class="fpage" tagx="fpage" title="fpage">42</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1476-511X-13-42">10.1186/1476-511X-13-42</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24588940">24588940</a></span></span></li> <li tag="ref"><a name="CR12" /><span class="label" tagx="label" title="label">12.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Failla</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chitchumronchokchai</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ferruzzi</span><span class="given-names" tagx="given-names" title="given-names">MG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goltz</span><span class="given-names" tagx="given-names" title="given-names">SR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Campbell</span><span class="given-names" tagx="given-names" title="given-names">WW</span></span></span><span class="mixed-article-title" title="mixed-article-title">Unsaturated fatty acids promote bioaccessibility and basolateral secretion of carotenoids and Î±-tocopherol by Caco-2 cells</span><span class="source" tagx="source" title="source">Food Funct.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">5</span><span class="fpage" tagx="fpage" title="fpage">1101</span><span class="lpage" tagx="lpage" title="lpage">1112</span><span class="pub-id"><a href="https://dx.doi.org/10.1039/C3FO60599J">10.1039/C3FO60599J</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24710065">24710065</a></span></span></li> <li tag="ref"><a name="CR13" /><span class="label" tagx="label" title="label">13.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baud</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lepiniec</span><span class="given-names" tagx="given-names" title="given-names">L</span></span></span><span class="mixed-article-title" title="mixed-article-title">Physiological and developmental regulation of seed oil production</span><span class="source" tagx="source" title="source">Prog. Lipid Res.</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">49</span><span class="fpage" tagx="fpage" title="fpage">235</span><span class="lpage" tagx="lpage" title="lpage">249</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.plipres.2010.01.001">10.1016/j.plipres.2010.01.001</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20102727">20102727</a></span></span></li> <li tag="ref"><a name="CR14" /><span class="label" tagx="label" title="label">14.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shimada</span><span class="given-names" tagx="given-names" title="given-names">TL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shimada</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Takahashi</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fukao</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hara-Nishimura</span><span class="given-names" tagx="given-names" title="given-names">I</span></span></span><span class="mixed-article-title" title="mixed-article-title">A novel role for oleosins in freezing tolerance of oilseeds in Arabidopsisthaliana</span><span class="source" tagx="source" title="source">Plant J.</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">55</span><span class="fpage" tagx="fpage" title="fpage">798</span><span class="lpage" tagx="lpage" title="lpage">809</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1365-313X.2008.03553.x">10.1111/j.1365-313X.2008.03553.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18485063">18485063</a></span></span></li> <li tag="ref"><a name="CR15" /><span class="label" tagx="label" title="label">15.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Rueda, A., et al. Characterization of fatty acid profile of argan oil and other edible vegetable oils by gas chromatography and discriminant analysis. <i>J. Chem</i>. <b>2014</b>, 843908 (2014).</span></li> <li tag="ref"><a name="CR16" /><span class="label" tagx="label" title="label">16.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Genome sequencing of the high oil crop sesame provides insight into oil biosynthesis</span><span class="source" tagx="source" title="source">Genome Biol.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">15</span><span class="fpage" tagx="fpage" title="fpage">R39</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/gb-2014-15-2-r39">10.1186/gb-2014-15-2-r39</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24576357">24576357</a></span></span></li> <li tag="ref"><a name="CR17" /><span class="label" tagx="label" title="label">17.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Unver</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Genome of wild olive and evolution of oil biosynthesis</span><span class="source" tagx="source" title="source">Proc. Natl Acad. Sci. USA</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">9413</span><span class="lpage" tagx="lpage" title="lpage">9422</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.1708621114">10.1073/pnas.1708621114</a></span></span></li> <li tag="ref"><a name="CR18" /><span class="label" tagx="label" title="label">18.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ting</span><span class="given-names" tagx="given-names" title="given-names">JTL</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Oleosin genes in maize kernels having diverse oil contents are constitutively expressed independent of oil contents</span><span class="source" tagx="source" title="source">Planta</span><span class="year" tagx="year" title="year">1996</span><span class="volume" tagx="volume" title="volume">199</span><span class="fpage" tagx="fpage" title="fpage">158</span><span class="lpage" tagx="lpage" title="lpage">165</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/BF00196892">10.1007/BF00196892</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8680304">8680304</a></span></span></li> <li tag="ref"><a name="CR19" /><span class="label" tagx="label" title="label">19.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Frandsen</span><span class="given-names" tagx="given-names" title="given-names">GI</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mundy</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tzen</span><span class="given-names" tagx="given-names" title="given-names">JTC</span></span></span><span class="mixed-article-title" title="mixed-article-title">Oil bodies and their associated proteins, oleosin and caleosin</span><span class="source" tagx="source" title="source">Physiol. Plant.</span><span class="year" tagx="year" title="year">2001</span><span class="volume" tagx="volume" title="volume">112</span><span class="fpage" tagx="fpage" title="fpage">301</span><span class="lpage" tagx="lpage" title="lpage">307</span><span class="pub-id"><a href="https://dx.doi.org/10.1034/j.1399-3054.2001.1120301.x">10.1034/j.1399-3054.2001.1120301.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11473685">11473685</a></span></span></li> <li tag="ref"><a name="CR20" /><span class="label" tagx="label" title="label">20.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lu</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fulda</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wallis</span><span class="given-names" tagx="given-names" title="given-names">JG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Browse</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">A high-throughput screen for genes from castor that boost hydroxy fatty acid accumulation in seed oils of transgenic <i>Arabidopsis</i></span><span class="source" tagx="source" title="source">Plant J.</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">45</span><span class="fpage" tagx="fpage" title="fpage">847</span><span class="lpage" tagx="lpage" title="lpage">856</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1365-313X.2005.02636.x">10.1111/j.1365-313X.2005.02636.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16460516">16460516</a></span></span></li> <li tag="ref"><a name="CR21" /><span class="label" tagx="label" title="label">21.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bhatla</span><span class="given-names" tagx="given-names" title="given-names">SC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kaushik</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yadav</span><span class="given-names" tagx="given-names" title="given-names">MK</span></span></span><span class="mixed-article-title" title="mixed-article-title">Use of oil bodies and oleosins in recombinant protein production and other biotechnological applications</span><span class="source" tagx="source" title="source">Biotechnol. Adv.</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">28</span><span class="fpage" tagx="fpage" title="fpage">293</span><span class="lpage" tagx="lpage" title="lpage">300</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.biotechadv.2010.01.001">10.1016/j.biotechadv.2010.01.001</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20067829">20067829</a></span></span></li> <li tag="ref"><a name="CR22" /><span class="label" tagx="label" title="label">22.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zahn</span><span class="given-names" tagx="given-names" title="given-names">LM</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Comparative transcriptomics among floral organs of the basal eudicot <i>Eschscholzia californica</i> as reference for floral evolutionary developmental studies</span><span class="source" tagx="source" title="source">Genome Biol.</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">11</span><span class="fpage" tagx="fpage" title="fpage">101</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/gb-2010-11-10-r101">10.1186/gb-2010-11-10-r101</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20128931">20128931</a></span></span></li> <li tag="ref"><a name="CR23" /><span class="label" tagx="label" title="label">23.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hasegawa</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kishino</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yano</span><span class="given-names" tagx="given-names" title="given-names">T</span></span></span><span class="mixed-article-title" title="mixed-article-title">Dating of the human-ape splitting by a molecular clock of mitochondrial DNA</span><span class="source" tagx="source" title="source">J. Mol. Evol.</span><span class="year" tagx="year" title="year">1985</span><span class="volume" tagx="volume" title="volume">22</span><span class="fpage" tagx="fpage" title="fpage">160</span><span class="lpage" tagx="lpage" title="lpage">174</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/BF02101694">10.1007/BF02101694</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3934395">3934395</a></span></span></li> <li tag="ref"><a name="CR24" /><span class="label" tagx="label" title="label">24.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guo</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">The opium poppy genome and morphinan production</span><span class="source" tagx="source" title="source">Science</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">362</span><span class="fpage" tagx="fpage" title="fpage">343</span><span class="lpage" tagx="lpage" title="lpage">347</span><span class="pub-id"><a href="https://dx.doi.org/10.1126/science.aat4096">10.1126/science.aat4096</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30166436">30166436</a></span></span></li> <li tag="ref"><a name="CR25" /><span class="label" tagx="label" title="label">25.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">The genome of medicinal plant <i>Macleaya cordata</i> provides new insights into benzylisoquinoline alkaloids metabolism</span><span class="source" tagx="source" title="source">Mol. Plant</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">975</span><span class="lpage" tagx="lpage" title="lpage">989</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.molp.2017.05.007">10.1016/j.molp.2017.05.007</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28552780">28552780</a></span></span></li> <li tag="ref"><a name="CR26" /><span class="label" tagx="label" title="label">26.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tenaillon</span><span class="given-names" tagx="given-names" title="given-names">MI</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hollister</span><span class="given-names" tagx="given-names" title="given-names">JD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gaut</span><span class="given-names" tagx="given-names" title="given-names">BS</span></span></span><span class="mixed-article-title" title="mixed-article-title">A triptych of the evolution of plant transposable elements</span><span class="source" tagx="source" title="source">Trends Plant Sci.</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">15</span><span class="fpage" tagx="fpage" title="fpage">471</span><span class="lpage" tagx="lpage" title="lpage">478</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.tplants.2010.05.003">10.1016/j.tplants.2010.05.003</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20541961">20541961</a></span></span></li> <li tag="ref"><a name="CR27" /><span class="label" tagx="label" title="label">27.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yuan</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">The pomegranate (<i>Punica granatum</i> L.) genome provides insights into fruit quality and ovule developmental biology</span><span class="source" tagx="source" title="source">Plant Biotechnol. J.</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">16</span><span class="fpage" tagx="fpage" title="fpage">1363</span><span class="lpage" tagx="lpage" title="lpage">1374</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/pbi.12875">10.1111/pbi.12875</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29271050">29271050</a></span></span></li> <li tag="ref"><a name="CR28" /><span class="label" tagx="label" title="label">28.</span><span class="element-citation'"><span class="person-group'"><span class="collab'">collab: Amborella Genome Project.</span></span><span class="mixed-article-title" title="mixed-article-title">The <i>Amborella</i> genome and the evolution of flowering plants</span><span class="source" tagx="source" title="source">Science</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">342</span><span class="fpage" tagx="fpage" title="fpage">1241089</span><span class="pub-id"><a href="https://dx.doi.org/10.1126/science.1241089">10.1126/science.1241089</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24357323">24357323</a></span></span></li> <li tag="ref"><a name="CR29" /><span class="label" tagx="label" title="label">29.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tank</span><span class="given-names" tagx="given-names" title="given-names">DC</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Nested radiations and the pulse of angiosperm diversification: increased diversification rates often follow whole genome duplications</span><span class="source" tagx="source" title="source">N. Phytol.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">207</span><span class="fpage" tagx="fpage" title="fpage">454</span><span class="lpage" tagx="lpage" title="lpage">467</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/nph.13491">10.1111/nph.13491</a></span></span></li> <li tag="ref"><a name="CR30" /><span class="label" tagx="label" title="label">30.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vanneste</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baele</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maere</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Van de Peer</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span></span><span class="mixed-article-title" title="mixed-article-title">Analysis of 41 plant genomes supports a wave of successful genome duplications in association with the Cretaceous-Paleogene boundary</span><span class="source" tagx="source" title="source">Genome Res.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">24</span><span class="fpage" tagx="fpage" title="fpage">1334</span><span class="lpage" tagx="lpage" title="lpage">1347</span><span class="pub-id"><a href="https://dx.doi.org/10.1101/gr.168997.113">10.1101/gr.168997.113</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24835588">24835588</a></span></span></li> <li tag="ref"><a name="CR31" /><span class="label" tagx="label" title="label">31.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ming</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Genome of the long-living sacred lotus (<i>Nelumbo nucifera</i> Gaertn.)</span><span class="source" tagx="source" title="source">Genome Biol.</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">14</span><span class="fpage" tagx="fpage" title="fpage">41</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/gb-2013-14-5-r41">10.1186/gb-2013-14-5-r41</a></span></span></li> <li tag="ref"><a name="CR32" /><span class="label" tagx="label" title="label">32.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Casacuberta</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">GonzÃ¡lez</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">The impact of transposable elements inenvironmental adaptation</span><span class="source" tagx="source" title="source">Mol. Ecol.</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">22</span><span class="fpage" tagx="fpage" title="fpage">1503</span><span class="lpage" tagx="lpage" title="lpage">1517</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/mec.12170">10.1111/mec.12170</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23293987">23293987</a></span></span></li> <li tag="ref"><a name="CR33" /><span class="label" tagx="label" title="label">33.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grandbastien</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span></span><span class="mixed-article-title" title="mixed-article-title">Activation of plant retrotransposons under stress conditions</span><span class="source" tagx="source" title="source">Trends Plant Sci.</span><span class="year" tagx="year" title="year">1998</span><span class="volume" tagx="volume" title="volume">3</span><span class="fpage" tagx="fpage" title="fpage">181</span><span class="lpage" tagx="lpage" title="lpage">187</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S1360-1385(98)01232-1">10.1016/S1360-1385(98)01232-1</a></span></span></li> <li tag="ref"><a name="CR34" /><span class="label" tagx="label" title="label">34.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Achnine</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Genomics-based selection and functional characterization of triterpene glycoyltransferases from the model legume <i>Medicago truncatula</i></span><span class="source" tagx="source" title="source">Plant J.</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">41</span><span class="fpage" tagx="fpage" title="fpage">875</span><span class="lpage" tagx="lpage" title="lpage">887</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1365-313X.2005.02344.x">10.1111/j.1365-313X.2005.02344.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15743451">15743451</a></span></span></li> <li tag="ref"><a name="CR35" /><span class="label" tagx="label" title="label">35.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Salie</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lancikova</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thelen</span><span class="given-names" tagx="given-names" title="given-names">JJ</span></span></span><span class="mixed-article-title" title="mixed-article-title">A family of negative regulators targets the committed step of de Novo fatty acid biosynthesis</span><span class="source" tagx="source" title="source">Plant Cell</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">28</span><span class="fpage" tagx="fpage" title="fpage">2312</span><span class="lpage" tagx="lpage" title="lpage">2325</span><span class="pub-id"><a href="https://dx.doi.org/10.1105/tpc.16.00317">10.1105/tpc.16.00317</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27559025">27559025</a></span></span></li> <li tag="ref"><a name="CR36" /><span class="label" tagx="label" title="label">36.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Salie</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thelen</span><span class="given-names" tagx="given-names" title="given-names">JJ</span></span></span><span class="mixed-article-title" title="mixed-article-title">Regulation and structure of the heteromeric acetyl-CoA carboxylase</span><span class="source" tagx="source" title="source">Biochim. Biophys. Acta</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">186</span><span class="fpage" tagx="fpage" title="fpage">1207</span><span class="lpage" tagx="lpage" title="lpage">1213</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.bbalip.2016.04.004">10.1016/j.bbalip.2016.04.004</a></span></span></li> <li tag="ref"><a name="CR37" /><span class="label" tagx="label" title="label">37.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lu</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sheahan</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fu</span><span class="given-names" tagx="given-names" title="given-names">P</span></span></span><span class="mixed-article-title" title="mixed-article-title">Metabolic engineering of algae for fourth generation biofuels production</span><span class="source" tagx="source" title="source">Energy Environ. Sci.</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">4</span><span class="fpage" tagx="fpage" title="fpage">2451</span><span class="lpage" tagx="lpage" title="lpage">2466</span><span class="pub-id"><a href="https://dx.doi.org/10.1039/c0ee00593b">10.1039/c0ee00593b</a></span></span></li> <li tag="ref"><a name="CR38" /><span class="label" tagx="label" title="label">38.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Chaturvedi, S. et al. Overexpression and repression of key rate-limiting enzymes (acetyl CoA carboxylase and HMG reductase to enhance fatty acid production from <i>Rhodotorula mucilaginosa</i>. <i>J. Basic Microbiol</i>. 1â€"11 (2020).</span></li> <li tag="ref"><a name="CR39" /><span class="label" tagx="label" title="label">39.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">GZ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xue</span><span class="given-names" tagx="given-names" title="given-names">HW</span></span></span><span class="mixed-article-title" title="mixed-article-title"><i>Arabidopsis</i> Î²-ketoacyl-[acyl carrier protein] synthase I is crucial for fatty acid synthesis and plays a role in chloroplast division and embryo development</span><span class="source" tagx="source" title="source">Plant Cell</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">22</span><span class="fpage" tagx="fpage" title="fpage">3726</span><span class="lpage" tagx="lpage" title="lpage">3744</span><span class="pub-id"><a href="https://dx.doi.org/10.1105/tpc.110.075564">10.1105/tpc.110.075564</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21081696">21081696</a></span></span></li> <li tag="ref"><a name="CR40" /><span class="label" tagx="label" title="label">40.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">GonzÃ¡lez-Thuillier</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Venegas-CalerÃ³n</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">GarcÃ©s</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wettstein-Knowles</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">MartÃ­nez-Force</span><span class="given-names" tagx="given-names" title="given-names">E</span></span></span><span class="mixed-article-title" title="mixed-article-title">Sunflower (<i>Helianthus annuus</i>) fatty acid synthase complex: enoyl-[acyl carrier protein]-reductase genes</span><span class="source" tagx="source" title="source">Planta</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">241</span><span class="fpage" tagx="fpage" title="fpage">43</span><span class="lpage" tagx="lpage" title="lpage">56</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s00425-014-2162-7">10.1007/s00425-014-2162-7</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25204631">25204631</a></span></span></li> <li tag="ref"><a name="CR41" /><span class="label" tagx="label" title="label">41.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tranbarger</span><span class="given-names" tagx="given-names" title="given-names">TJ</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Regulatory mechanisms underlying oil palm fruit mesocarp maturation, ripening, and functional specialization in lipid and carotenoid metabolism</span><span class="source" tagx="source" title="source">Plant Physiol.</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">156</span><span class="fpage" tagx="fpage" title="fpage">564</span><span class="lpage" tagx="lpage" title="lpage">584</span><span class="pub-id"><a href="https://dx.doi.org/10.1104/pp.111.175141">10.1104/pp.111.175141</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21487046">21487046</a></span></span></li> <li tag="ref"><a name="CR42" /><span class="label" tagx="label" title="label">42.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">HernÃ¡ndez</span><span class="given-names" tagx="given-names" title="given-names">ML</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sicardo</span><span class="given-names" tagx="given-names" title="given-names">MD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alfonso</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">MartÃ­nez-Rivas</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span></span><span class="mixed-article-title" title="mixed-article-title">Transcriptional regulation of stearoyl-acyl carrier protein desaturase genes in response to abiotic stresses leads to changes in the unsaturated fatty acids composition of olive mesocarp</span><span class="source" tagx="source" title="source">Front. Plant Sci.</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">251</span><span class="pub-id"><a href="https://dx.doi.org/10.3389/fpls.2019.00251">10.3389/fpls.2019.00251</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30891055">30891055</a></span></span></li> <li tag="ref"><a name="CR43" /><span class="label" tagx="label" title="label">43.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ha</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Genome sequence of <i>Jatropha curcas</i> L., a non-edible biodiesel plant, provides a resource to improve seed-related traits</span><span class="source" tagx="source" title="source">Plant Biotech. J.</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">17</span><span class="fpage" tagx="fpage" title="fpage">517</span><span class="lpage" tagx="lpage" title="lpage">530</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/pbi.12995">10.1111/pbi.12995</a></span></span></li> <li tag="ref"><a name="CR44" /><span class="label" tagx="label" title="label">44.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tel-Zur</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abbo</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Myslabodski</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mizrahi</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span></span><span class="mixed-article-title" title="mixed-article-title">Modified CTAB procedure for DNA isolation from epiphytic cacti of the genera hylocereus and selenicereus (Cactaceae)</span><span class="source" tagx="source" title="source">Plant Mol. Biol. Rep.</span><span class="year" tagx="year" title="year">1999</span><span class="volume" tagx="volume" title="volume">17</span><span class="fpage" tagx="fpage" title="fpage">249</span><span class="lpage" tagx="lpage" title="lpage">254</span><span class="pub-id"><a href="https://dx.doi.org/10.1023/A:1007656315275">10.1023/A:1007656315275</a></span></span></li> <li tag="ref"><a name="CR45" /><span class="label" tagx="label" title="label">45.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tong</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">QJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gao</span><span class="given-names" tagx="given-names" title="given-names">LZ</span></span></span><span class="mixed-article-title" title="mixed-article-title">Genome size variation among and within <i>Camellia</i> species by using flow cytometric analysis</span><span class="source" tagx="source" title="source">PLoS ONE</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">e64981</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0064981">10.1371/journal.pone.0064981</a></span></span></li> <li tag="ref"><a name="CR46" /><span class="label" tagx="label" title="label">46.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chikhi</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Medvedev</span><span class="given-names" tagx="given-names" title="given-names">P</span></span></span><span class="mixed-article-title" title="mixed-article-title">Informed and automated k-mer size selection for genome assembly</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">30</span><span class="fpage" tagx="fpage" title="fpage">31</span><span class="lpage" tagx="lpage" title="lpage">37</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btt310">10.1093/bioinformatics/btt310</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23732276">23732276</a></span></span></li> <li tag="ref"><a name="CR47" /><span class="label" tagx="label" title="label">47.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">MarÃ§ais</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kingsford</span><span class="given-names" tagx="given-names" title="given-names">C</span></span></span><span class="mixed-article-title" title="mixed-article-title">A fast, lock-free approach for efficient parallel counting of occurrences of k-mers</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">27</span><span class="fpage" tagx="fpage" title="fpage">764</span><span class="lpage" tagx="lpage" title="lpage">770</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btr011">10.1093/bioinformatics/btr011</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21217122">21217122</a></span></span></li> <li tag="ref"><a name="CR48" /><span class="label" tagx="label" title="label">48.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Koren</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Canu: scalable and accurate long-read assembly via adaptive -mer weighting and repeat separation</span><span class="source" tagx="source" title="source">Genome Res.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">27</span><span class="fpage" tagx="fpage" title="fpage">722</span><span class="lpage" tagx="lpage" title="lpage">736</span><span class="pub-id"><a href="https://dx.doi.org/10.1101/gr.215087.116">10.1101/gr.215087.116</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28298431">28298431</a></span></span></li> <li tag="ref"><a name="CR49" /><span class="label" tagx="label" title="label">49.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">WTDBG package. Accessed 10 Jan 2018. available from <a href="https://github.com/ruanjue/wtdbg2">https://github.com/ruanjue/wtdbg2</a>.</span></li> <li tag="ref"><a name="CR50" /><span class="label" tagx="label" title="label">50.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chakraborty</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baldwin-Brown</span><span class="given-names" tagx="given-names" title="given-names">JG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Long</span><span class="given-names" tagx="given-names" title="given-names">AD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Emerson</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">Contiguous and accurate <i>de novo</i> assembly of metazoan genomes with modest long read coverage</span><span class="source" tagx="source" title="source">Nucleic Acids Res.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">44</span><span class="fpage" tagx="fpage" title="fpage">147</span><span class="lpage" tagx="lpage" title="lpage">147</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gkw419">10.1093/nar/gkw419</a></span></span></li> <li tag="ref"><a name="CR51" /><span class="label" tagx="label" title="label">51.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Walker</span><span class="given-names" tagx="given-names" title="given-names">BJ</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Pilon: an integrated tool for comprehensive microbial variant detection and genome assembly improvement</span><span class="source" tagx="source" title="source">PLoS ONE</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">e112963</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0112963">10.1371/journal.pone.0112963</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25409509">25409509</a></span></span></li> <li tag="ref"><a name="CR52" /><span class="label" tagx="label" title="label">52.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Durbin</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="mixed-article-title" title="mixed-article-title">Fast and accurate short read alignment with Burrows-Wheeler transform</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">25</span><span class="fpage" tagx="fpage" title="fpage">1754</span><span class="lpage" tagx="lpage" title="lpage">1760</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btp324">10.1093/bioinformatics/btp324</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19451168">19451168</a></span></span></li> <li tag="ref"><a name="CR53" /><span class="label" tagx="label" title="label">53.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grabherr</span><span class="given-names" tagx="given-names" title="given-names">MG</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Full-length transcriptome assembly from RNA-seq data without a reference genome</span><span class="source" tagx="source" title="source">Nat. Biotechnol.</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">29</span><span class="fpage" tagx="fpage" title="fpage">644</span><span class="lpage" tagx="lpage" title="lpage">652</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nbt.1883">10.1038/nbt.1883</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21572440">21572440</a></span></span></li> <li tag="ref"><a name="CR54" /><span class="label" tagx="label" title="label">54.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paggi</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Park</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bennett</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Salzberg</span><span class="given-names" tagx="given-names" title="given-names">SL</span></span></span><span class="mixed-article-title" title="mixed-article-title">Graph-based genome alignment and genotyping with HISAT2 and HISAT-genotype</span><span class="source" tagx="source" title="source">Nat. Biotechnol.</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">37</span><span class="fpage" tagx="fpage" title="fpage">907</span><span class="lpage" tagx="lpage" title="lpage">915</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41587-019-0201-4">10.1038/s41587-019-0201-4</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31375807">31375807</a></span></span></li> <li tag="ref"><a name="CR55" /><span class="label" tagx="label" title="label">55.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parra</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bradnam</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Korf</span><span class="given-names" tagx="given-names" title="given-names">I</span></span></span><span class="mixed-article-title" title="mixed-article-title">CEGMA: a pipeline to accurately annotate coregenes in eukaryotic genomes</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">23</span><span class="fpage" tagx="fpage" title="fpage">1061</span><span class="lpage" tagx="lpage" title="lpage">1067</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btm071">10.1093/bioinformatics/btm071</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17332020">17332020</a></span></span></li> <li tag="ref"><a name="CR56" /><span class="label" tagx="label" title="label">56.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">SimÃ£o</span><span class="given-names" tagx="given-names" title="given-names">FA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Waterhouse</span><span class="given-names" tagx="given-names" title="given-names">RM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ioannidis</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kriventseva</span><span class="given-names" tagx="given-names" title="given-names">EV</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zdobnov</span><span class="given-names" tagx="given-names" title="given-names">EM</span></span></span><span class="mixed-article-title" title="mixed-article-title">BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">31</span><span class="fpage" tagx="fpage" title="fpage">3210</span><span class="lpage" tagx="lpage" title="lpage">3212</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btv351">10.1093/bioinformatics/btv351</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26059717">26059717</a></span></span></li> <li tag="ref"><a name="CR57" /><span class="label" tagx="label" title="label">57.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lieberman-Aiden</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Comprehensive mapping of long-rangeinteractions reveals folding principles of the human genome</span><span class="source" tagx="source" title="source">Science</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">326</span><span class="fpage" tagx="fpage" title="fpage">289</span><span class="lpage" tagx="lpage" title="lpage">293</span><span class="pub-id"><a href="https://dx.doi.org/10.1126/science.1181369">10.1126/science.1181369</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19815776">19815776</a></span></span></li> <li tag="ref"><a name="CR58" /><span class="label" tagx="label" title="label">58.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Langmead</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Salzberg</span><span class="given-names" tagx="given-names" title="given-names">SL</span></span></span><span class="mixed-article-title" title="mixed-article-title">Fast gapped-read alignment with Bowtie 2</span><span class="source" tagx="source" title="source">Nat. Methods</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">357</span><span class="lpage" tagx="lpage" title="lpage">359</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nmeth.1923">10.1038/nmeth.1923</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22388286">22388286</a></span></span></li> <li tag="ref"><a name="CR59" /><span class="label" tagx="label" title="label">59.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Burton</span><span class="given-names" tagx="given-names" title="given-names">JN</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Chromosome-scale scaffolding of de novogenome assemblies based on chromatin interactions</span><span class="source" tagx="source" title="source">Nat. Biotechnol.</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">31</span><span class="fpage" tagx="fpage" title="fpage">1119</span><span class="lpage" tagx="lpage" title="lpage">1125</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nbt.2727">10.1038/nbt.2727</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24185095">24185095</a></span></span></li> <li tag="ref"><a name="CR60" /><span class="label" tagx="label" title="label">60.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Smit, A. F. A. &amp;amp; Hubley, R. RepeatModeler Open-1.0. <a href="http://www.repeatmasker.org">http://www.repeatmasker.org</a> (2008â€"2015).</span></li> <li tag="ref"><a name="CR61" /><span class="label" tagx="label" title="label">61.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">H</span></span></span><span class="mixed-article-title" title="mixed-article-title">LTR_FINDER: an efficient tool for the prediction of full-length LTR retrotransposons</span><span class="source" tagx="source" title="source">Nucleic Acids Res.</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">35</span><span class="fpage" tagx="fpage" title="fpage">265</span><span class="lpage" tagx="lpage" title="lpage">268</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gkm286">10.1093/nar/gkm286</a></span></span></li> <li tag="ref"><a name="CR62" /><span class="label" tagx="label" title="label">62.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Price</span><span class="given-names" tagx="given-names" title="given-names">AL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jones</span><span class="given-names" tagx="given-names" title="given-names">NC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pevzner</span><span class="given-names" tagx="given-names" title="given-names">PA</span></span></span><span class="mixed-article-title" title="mixed-article-title">De novo identification of repeat families in large genomes</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">21</span><span class="fpage" tagx="fpage" title="fpage">351</span><span class="lpage" tagx="lpage" title="lpage">358</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/bti1018">10.1093/bioinformatics/bti1018</a></span></span></li> <li tag="ref"><a name="CR63" /><span class="label" tagx="label" title="label">63.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Smit, A. F. A., Hubley, R. &amp;amp; Green, P. RepeatMasker Open-4.0. <a href="http://www.repeatmasker.org">http://www.repeatmasker.org</a> (1996â€"2015).</span></li> <li tag="ref"><a name="CR64" /><span class="label" tagx="label" title="label">64.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jurka</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Repbase update, a database of eukaryotic repetitive elements</span><span class="source" tagx="source" title="source">Cytogenet Genome Res.</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">110</span><span class="fpage" tagx="fpage" title="fpage">462</span><span class="lpage" tagx="lpage" title="lpage">467</span><span class="pub-id"><a href="https://dx.doi.org/10.1159/000084979">10.1159/000084979</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16093699">16093699</a></span></span></li> <li tag="ref"><a name="CR65" /><span class="label" tagx="label" title="label">65.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lowe</span><span class="given-names" tagx="given-names" title="given-names">TM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chan</span><span class="given-names" tagx="given-names" title="given-names">PP</span></span></span><span class="mixed-article-title" title="mixed-article-title">tRNAscan-SE On-line: integrating search and context for analysis of transfer RNA genes</span><span class="source" tagx="source" title="source">Nucleic Acids Res.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">44</span><span class="fpage" tagx="fpage" title="fpage">54</span><span class="lpage" tagx="lpage" title="lpage">57</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gkw413">10.1093/nar/gkw413</a></span></span></li> <li tag="ref"><a name="CR66" /><span class="label" tagx="label" title="label">66.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nawrocki</span><span class="given-names" tagx="given-names" title="given-names">EP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eddy</span><span class="given-names" tagx="given-names" title="given-names">SR</span></span></span><span class="mixed-article-title" title="mixed-article-title">Infernal 1.1: 100-fold faster RNA homology searches</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">29</span><span class="fpage" tagx="fpage" title="fpage">2933</span><span class="lpage" tagx="lpage" title="lpage">2935</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btt509">10.1093/bioinformatics/btt509</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24008419">24008419</a></span></span></li> <li tag="ref"><a name="CR67" /><span class="label" tagx="label" title="label">67.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lagesen</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">RNAmmer: consistent and rapid annotation of ribosomal RNA genes</span><span class="source" tagx="source" title="source">Nucleic Acids Res.</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">35</span><span class="fpage" tagx="fpage" title="fpage">3100</span><span class="lpage" tagx="lpage" title="lpage">3108</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gkm160">10.1093/nar/gkm160</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17452365">17452365</a></span></span></li> <li tag="ref"><a name="CR68" /><span class="label" tagx="label" title="label">68.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Altschul</span><span class="given-names" tagx="given-names" title="given-names">SF</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs</span><span class="source" tagx="source" title="source">Nucleic Acids Res.</span><span class="year" tagx="year" title="year">1997</span><span class="volume" tagx="volume" title="volume">25</span><span class="fpage" tagx="fpage" title="fpage">3389</span><span class="lpage" tagx="lpage" title="lpage">3402</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/25.17.3389">10.1093/nar/25.17.3389</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9254694">9254694</a></span></span></li> <li tag="ref"><a name="CR69" /><span class="label" tagx="label" title="label">69.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Birney</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Clamp</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Durbin</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="mixed-article-title" title="mixed-article-title">GeneWise and genomewise</span><span class="source" tagx="source" title="source">Genome Res.</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">14</span><span class="fpage" tagx="fpage" title="fpage">988</span><span class="lpage" tagx="lpage" title="lpage">995</span><span class="pub-id"><a href="https://dx.doi.org/10.1101/gr.1865504">10.1101/gr.1865504</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15123596">15123596</a></span></span></li> <li tag="ref"><a name="CR70" /><span class="label" tagx="label" title="label">70.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pertea</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">StringTie enables improved reconstruction of a transcriptome from RNA-seq reads</span><span class="source" tagx="source" title="source">Nat. Biotech.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">33</span><span class="fpage" tagx="fpage" title="fpage">290</span><span class="lpage" tagx="lpage" title="lpage">295</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nbt.3122">10.1038/nbt.3122</a></span></span></li> <li tag="ref"><a name="CR71" /><span class="label" tagx="label" title="label">71.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stanke</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Steinkamp</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Waack</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morgenstern</span><span class="given-names" tagx="given-names" title="given-names">B</span></span></span><span class="mixed-article-title" title="mixed-article-title">AUGUSTUS: a web server for gene finding in eukaryotes</span><span class="source" tagx="source" title="source">Nucleic Acids Res.</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">32</span><span class="fpage" tagx="fpage" title="fpage">309</span><span class="lpage" tagx="lpage" title="lpage">312</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gkh379">10.1093/nar/gkh379</a></span></span></li> <li tag="ref"><a name="CR72" /><span class="label" tagx="label" title="label">72.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aggarwal</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ramaswamy</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="mixed-article-title" title="mixed-article-title">Ab initio gene identification: prokaryote genome annotation with GeneScan and GLIMMER</span><span class="source" tagx="source" title="source">J. Biosci.</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">27</span><span class="fpage" tagx="fpage" title="fpage">7</span><span class="lpage" tagx="lpage" title="lpage">14</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/BF02703679">10.1007/BF02703679</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11927773">11927773</a></span></span></li> <li tag="ref"><a name="CR73" /><span class="label" tagx="label" title="label">73.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bromberg</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rost</span><span class="given-names" tagx="given-names" title="given-names">B</span></span></span><span class="mixed-article-title" title="mixed-article-title">SNAP: predict effect of non-synonymous polymorphisms on function</span><span class="source" tagx="source" title="source">Nucleic Acids Res.</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">35</span><span class="fpage" tagx="fpage" title="fpage">3823</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gkm238">10.1093/nar/gkm238</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17526529">17526529</a></span></span></li> <li tag="ref"><a name="CR74" /><span class="label" tagx="label" title="label">74.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Majoros</span><span class="given-names" tagx="given-names" title="given-names">WH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pertea</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Salzberg</span><span class="given-names" tagx="given-names" title="given-names">SL</span></span></span><span class="mixed-article-title" title="mixed-article-title">TigrScan and GlimmerHMM: twoopen source ab initio eukaryotic gene-finders</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">20</span><span class="fpage" tagx="fpage" title="fpage">2878</span><span class="lpage" tagx="lpage" title="lpage">2879</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/bth315">10.1093/bioinformatics/bth315</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15145805">15145805</a></span></span></li> <li tag="ref"><a name="CR75" /><span class="label" tagx="label" title="label">75.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haas</span><span class="given-names" tagx="given-names" title="given-names">BJ</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Automated eukaryotic gene structure annotation using EVidenceModeler and the program to assemble spliced alignments</span><span class="source" tagx="source" title="source">Genome Biol.</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/gb-2008-9-1-r7">10.1186/gb-2008-9-1-r7</a></span></span></li> <li tag="ref"><a name="CR76" /><span class="label" tagx="label" title="label">76.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kanehisa</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goto</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="mixed-article-title" title="mixed-article-title">KEGG: kyoto encyclopedia of genes and genomes</span><span class="source" tagx="source" title="source">Nucleic Acids Res.</span><span class="year" tagx="year" title="year">2000</span><span class="volume" tagx="volume" title="volume">28</span><span class="fpage" tagx="fpage" title="fpage">27</span><span class="lpage" tagx="lpage" title="lpage">30</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/28.1.27">10.1093/nar/28.1.27</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10592173">10592173</a></span></span></li> <li tag="ref"><a name="CR77" /><span class="label" tagx="label" title="label">77.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Quevillon</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">InterProScan: protein domains identifier</span><span class="source" tagx="source" title="source">Nucleic Acids Res.</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">33</span><span class="fpage" tagx="fpage" title="fpage">116</span><span class="lpage" tagx="lpage" title="lpage">120</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gki442">10.1093/nar/gki442</a></span></span></li> <li tag="ref"><a name="CR78" /><span class="label" tagx="label" title="label">78.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Finn</span><span class="given-names" tagx="given-names" title="given-names">RD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Clements</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eddy</span><span class="given-names" tagx="given-names" title="given-names">SR</span></span></span><span class="mixed-article-title" title="mixed-article-title">HMMER web server: interactive sequence similarity searching</span><span class="source" tagx="source" title="source">Nucleic Acids Res.</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">39</span><span class="fpage" tagx="fpage" title="fpage">29</span><span class="lpage" tagx="lpage" title="lpage">37</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gkr367">10.1093/nar/gkr367</a></span></span></li> <li tag="ref"><a name="CR79" /><span class="label" tagx="label" title="label">79.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hunter</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">InterPro: the integrative protein signature database</span><span class="source" tagx="source" title="source">Nucleic Acids Res.</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">37</span><span class="fpage" tagx="fpage" title="fpage">211</span><span class="lpage" tagx="lpage" title="lpage">215</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gkn785">10.1093/nar/gkn785</a></span></span></li> <li tag="ref"><a name="CR80" /><span class="label" tagx="label" title="label">80.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Finn</span><span class="given-names" tagx="given-names" title="given-names">RD</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Pfam: the protein families database</span><span class="source" tagx="source" title="source">Nucleic Acids Res.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">42</span><span class="fpage" tagx="fpage" title="fpage">222</span><span class="lpage" tagx="lpage" title="lpage">230</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gkt1223">10.1093/nar/gkt1223</a></span></span></li> <li tag="ref"><a name="CR81" /><span class="label" tagx="label" title="label">81.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ashburner</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Gene ontology: tool for the unification of biology</span><span class="source" tagx="source" title="source">Nat. Genet.</span><span class="year" tagx="year" title="year">2000</span><span class="volume" tagx="volume" title="volume">25</span><span class="fpage" tagx="fpage" title="fpage">25</span><span class="lpage" tagx="lpage" title="lpage">29</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/75556">10.1038/75556</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10802651">10802651</a></span></span></li> <li tag="ref"><a name="CR82" /><span class="label" tagx="label" title="label">82.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McCarthy</span><span class="given-names" tagx="given-names" title="given-names">EM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McDonald</span><span class="given-names" tagx="given-names" title="given-names">JF</span></span></span><span class="mixed-article-title" title="mixed-article-title">LTR_STRUC: a novel search and identification program for LTR retrotransposons</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">19</span><span class="fpage" tagx="fpage" title="fpage">362</span><span class="lpage" tagx="lpage" title="lpage">367</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btf878">10.1093/bioinformatics/btf878</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12584121">12584121</a></span></span></li> <li tag="ref"><a name="CR83" /><span class="label" tagx="label" title="label">83.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">SanMiguel</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gaut</span><span class="given-names" tagx="given-names" title="given-names">BS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tikhonov</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nakajima</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bennetzen</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span></span><span class="mixed-article-title" title="mixed-article-title">The paleontology of intergene retrotransposons of maize</span><span class="source" tagx="source" title="source">Nat. Genet.</span><span class="year" tagx="year" title="year">1998</span><span class="volume" tagx="volume" title="volume">20</span><span class="fpage" tagx="fpage" title="fpage">43</span><span class="lpage" tagx="lpage" title="lpage">45</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/1695">10.1038/1695</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9731528">9731528</a></span></span></li> <li tag="ref"><a name="CR84" /><span class="label" tagx="label" title="label">84.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ma</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bennetzen</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span></span><span class="mixed-article-title" title="mixed-article-title">Rapid recent growth and divergence of rice nuclear genomes</span><span class="source" tagx="source" title="source">Proc. Natl Acad. Sci. USA</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">101</span><span class="fpage" tagx="fpage" title="fpage">12404</span><span class="lpage" tagx="lpage" title="lpage">12410</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.0403715101">10.1073/pnas.0403715101</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15240870">15240870</a></span></span></li> <li tag="ref"><a name="CR85" /><span class="label" tagx="label" title="label">85.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stoeckert</span><span class="given-names" tagx="given-names" title="given-names">CJ</span><span class="suffix" tagx="suffix" title="suffix">Jr</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roos</span><span class="given-names" tagx="given-names" title="given-names">DS</span></span></span><span class="mixed-article-title" title="mixed-article-title">OrthoMCL: identification of ortholog groups for eukaryotic genomes</span><span class="source" tagx="source" title="source">Genome Res.</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">13</span><span class="fpage" tagx="fpage" title="fpage">2178</span><span class="lpage" tagx="lpage" title="lpage">2189</span><span class="pub-id"><a href="https://dx.doi.org/10.1101/gr.1224503">10.1101/gr.1224503</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12952885">12952885</a></span></span></li> <li tag="ref"><a name="CR86" /><span class="label" tagx="label" title="label">86.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Han</span><span class="given-names" tagx="given-names" title="given-names">MV</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thomas</span><span class="given-names" tagx="given-names" title="given-names">GW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lugo-Martinez</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hahn</span><span class="given-names" tagx="given-names" title="given-names">MW</span></span></span><span class="mixed-article-title" title="mixed-article-title">Estimating gene gain and loss rates in the presence of error in genome assembly and annotation using CAFE 3</span><span class="source" tagx="source" title="source">Mol. Biol. Evol.</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">30</span><span class="fpage" tagx="fpage" title="fpage">1987</span><span class="lpage" tagx="lpage" title="lpage">1997</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/mst100">10.1093/molbev/mst100</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23709260">23709260</a></span></span></li> <li tag="ref"><a name="CR87" /><span class="label" tagx="label" title="label">87.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Edgar</span><span class="given-names" tagx="given-names" title="given-names">RC</span></span></span><span class="mixed-article-title" title="mixed-article-title">MUSCLE: multiple sequence alignment with high accuracy and high throughput</span><span class="source" tagx="source" title="source">Nucleic Acids Res.</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">32</span><span class="fpage" tagx="fpage" title="fpage">1792</span><span class="lpage" tagx="lpage" title="lpage">1797</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gkh340">10.1093/nar/gkh340</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15034147">15034147</a></span></span></li> <li tag="ref"><a name="CR88" /><span class="label" tagx="label" title="label">88.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stamatakis</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">30</span><span class="fpage" tagx="fpage" title="fpage">1312</span><span class="lpage" tagx="lpage" title="lpage">1313</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btu033">10.1093/bioinformatics/btu033</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24451623">24451623</a></span></span></li> <li tag="ref"><a name="CR89" /><span class="label" tagx="label" title="label">89.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span></span><span class="mixed-article-title" title="mixed-article-title">PAML 4: phylogenetic analysis by maximum likelihood</span><span class="source" tagx="source" title="source">Mol. Biol. Evol.</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">24</span><span class="fpage" tagx="fpage" title="fpage">1586</span><span class="lpage" tagx="lpage" title="lpage">1591</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/msm088">10.1093/molbev/msm088</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17483113">17483113</a></span></span></li> <li tag="ref"><a name="CR90" /><span class="label" tagx="label" title="label">90.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kumar</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stecher</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suleski</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hedges</span><span class="given-names" tagx="given-names" title="given-names">SB</span></span></span><span class="mixed-article-title" title="mixed-article-title">TimeTree: a resource for timelines, timetrees, and divergence times</span><span class="source" tagx="source" title="source">Mol. Biol. Evol.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">34</span><span class="fpage" tagx="fpage" title="fpage">1812</span><span class="lpage" tagx="lpage" title="lpage">1819</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/msx116">10.1093/molbev/msx116</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28387841">28387841</a></span></span></li> <li tag="ref"><a name="CR91" /><span class="label" tagx="label" title="label">91.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Anders, S. &amp;amp; Huber, W. Differential expression analysis for sequence count data. <i>Genome Biol</i>. <b>11</b>, R106 (2010).</span></li> <li tag="ref"><a name="CR92" /><span class="label" tagx="label" title="label">92.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Letunic</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bork</span><span class="given-names" tagx="given-names" title="given-names">P</span></span></span><span class="mixed-article-title" title="mixed-article-title">Interactive tree of life (iTOL) v3: an online tool for the displayand annotation of phylogenetic and other trees</span><span class="source" tagx="source" title="source">Nucleic Acids Res.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">44</span><span class="fpage" tagx="fpage" title="fpage">242</span><span class="lpage" tagx="lpage" title="lpage">245</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gkw290">10.1093/nar/gkw290</a></span></span></li> <li tag="ref"><a name="CR93" /><span class="label" tagx="label" title="label">93.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mao</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guarnieri</span><span class="given-names" tagx="given-names" title="given-names">MT</span></span></span><span class="mixed-article-title" title="mixed-article-title">Simultaneous production of triacylglycerol and high-value cartenoids by the astaxanthin-producing oleaginous green microalga <i>Chlorella zofingiensis</i></span><span class="source" tagx="source" title="source">Bioresour. Tech.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">214</span><span class="fpage" tagx="fpage" title="fpage">319</span><span class="lpage" tagx="lpage" title="lpage">327</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.biortech.2016.04.112">10.1016/j.biortech.2016.04.112</a></span></span></li> </ul> </div> </div>  </body></html>