<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Int J Microbiol</span><span class="iso-abbrev" title="iso-abbrev">Int J Microbiol</span><span class="publisher-id" title="publisher-id">IJMICRO</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">International Journal of Microbiology</span></div><span class="issn-ppub" tagx="issn" title="issn-ppub">1687-918X</span><span class="issn-epub" tagx="issn" title="issn-epub">1687-9198</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Hindawi</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7864745">7864745</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1155/2020/6639118">10.1155/2020/6639118</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Rhizobacteria Communities of Phytoremediation Plant Species in Petroleum Hydrocarbon Contaminated Soil of the Sudd Ecosystem, South Sudan</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0002-4795-2055</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ruley</span><span class="given-names" tagx="given-names" title="given-names">J. A.</span></span><span class="email" tagx="email" title="email">janenajeb@yahoo.com</span><a href="#I1"><sup>1</sup></a><a href="#I2"><sup>2</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tumuhairwe</span><span class="given-names" tagx="given-names" title="given-names">J. B.</span></span><a href="#I1"><sup>1</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Amoding</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><a href="#I1"><sup>1</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Westengen</span><span class="given-names" tagx="given-names" title="given-names">O. T.</span></span><a href="#I3"><sup>3</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vinje</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><a href="#I4"><sup>4</sup></a></span></div><span class="citation_author_institution" id="I1">[], <sup>1</sup></span><span class="citation_author_institution" id="I2">[], <sup>2</sup></span><span class="citation_author_institution" id="I3">[], <sup>3</sup></span><span class="citation_author_institution" id="I4">[], <sup>4</sup></span><div class="author-notes" title="author-notes"> <div class="fn-type-other" title="other"> <p>Academic Editor: Zhun Li</p> </div> </div><span class="pub-date-collection" title="pub-date-collection">collection: <span>2020</span></span><span class="pub-date-epub" title="pub-date-epub">epub: <span>2020-12-12</span></span><span class="volume" tagx="volume" title="volume">2020</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">6639118</span><span class="history" title="history"><span class="received" title="received">received: 2020-10-23</span><span class="accepted" title="accepted">accepted: 2020-12-10</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , <span class="copyright-year" tagx="copyright-year" title="copyright-year">2020</span></span><span class="license" title="license"><span class="license-p" title="license-p">This is an open access article distributed under the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>The Sudd wetland is one of the oil-rich regions of South Sudan where environmental pollution resulting from oil extraction activities has been unprecedented. Although phytoremediation is the most feasible technique, its efficacy reduces at high TPH concentration in soil. This has made rhizoremediation the most preferred approach. Rhizoremediation involves use of a combination of phytoremediation and biostimulation. The process is catalyzed by the action of rhizobacteria. Therefore, the objective of this study is to characterize rhizobacteria communities prevalent in phytoremediation species growing in hydrocarbon-contaminated soils biostimulated with cattle manure. The treatments studied were plant species only (T1), plant species and hydrocarbons (T2), plant species and manure (T3), and plant species, manure, and hydrocarbons (T4). The rhizobacteria communities were determined using pyrosequencing of 16S rRNA. In the treatment with phytoremediation species, hydrocarbons 75â€‰gâ€‰Â·â€‰kg<sup>âˆ’1</sup>soil, and cattle manure 5â€‰gâ€‰Â·â€‰kg<sup>âˆ’1</sup>soil (T4), there was a significant increase (<i>p</i> &amp;lt; 0.05) in rhizobacteria abundance with the highest ASV observed in <i>H. rufa</i> (4980) and the lowest in <i>S. arundinaceum</i> (3955). In the same treatment, bacteria community diversity was high in <i>H. rufa</i> (Chao1, 10310) and the least in <i>S. arundinaceum</i> (Chao 1, 8260) with Proteobacteria, Firmicutes, and Actinobacteria as the dominant phyla. Similarly, in contaminated soil treated with cattle manure, there was a significant increase (<i>p</i> &amp;lt; 0.05) in abundance of rhizobacteria genera with <i>Pseudomonas</i> dominating across phytoremediation species. <i>H. rufa</i> was dominated by <i>Bacillus, Fusibacter,</i> and <i>Rhodococcus</i>; <i>G. barbadense</i> was mainly associated with <i>Luteimonas</i> and <i>Mycobacterium,</i> and <i>T. diversifolia</i> was inhabited by <i>Bacillus</i> and <i>Luteimonas</i>. The rhizosphere of <i>O. longistaminata</i> was dominated by <i>Bacillus, Fusibacter,</i> and <i>Luteimonas,</i> while <i>S. arundinaceum</i> was largely inhabited by <i>Sphingomonas.</i> These rhizobacteria genera ought to be applied in the Sudd region for bioremediation.</p> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source">
NORAD</div><span class="award-id" tagx="award-id" title="award-id">SSD-13/0021</span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source">
Makerere University</div> </div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source">
Norwegian University of Life Sciences</div> </div> </div> </div> </div> <div class="body" title="body"> <div class="1.introduction" title="sec"> <div class="title" tagx="title" title="title">
1. Introduction</div> <p>Globally, crude oil is a critical resource for national development. The world greatly depends on oil and, as a result, vast amount is used, transported, and stored [<a href="#B1">1</a>]. Crude oil is number one source of energy and a primary raw material for major industries worldwide [<a href="#B2">2</a>].Oil remains an indispensable input for sectors such as manufacturing (as a raw material and fuel), transport (fuel), and trade (as an export commodity), and there has been a steady increase in global demand for crude oil over the last decades [<a href="#B2">2</a>, <a href="#B3">3</a>, <a href="#B4">4</a>]. Over the next two decades, Rada and colleagues [<a href="#B5">5</a>] anticipate that world oil demand could even rise to 170 million barrels per day. This belief has compelled producing countries to devise ways and means of increasing production, transportation, and refining of crude oil in order to meet the purported growth in demand [<a href="#B2">2</a>, <a href="#B4">4</a>], South Sudan inclusive [<a href="#B6">6</a>].</p> <p>South Sudan is one of the famous crude oil producing countries falling 3<sup>rd</sup> after Nigeria and Angola in Africa [<a href="#B6">6</a>] and 83<sup>rd</sup> among the 171 oil producing countries worldwide [<a href="#B7">7</a>]. Oil is the lifeline of its economy for now and over the medium term [<a href="#B6">6</a>]. For example, between 2008 and 2011, oil exports accounted for 98% of government revenue [<a href="#B6">6</a>]. This position has fundamentally changed her economy from agriculture to industrialization. South Sudan has a production capacity of 298,000/390,000 barrels per day [<a href="#B8">8</a>, <a href="#B9">9</a>] including Sudd wetland. Sudd is the largest wetland in the whole world and covers 57,000â€‰Km<sup>2</sup> that makes up approximately 5% of the total land area of the Republic of South Sudan (648,000â€‰Km<sup>2</sup>) [<a href="#B10">10</a>, <a href="#B11">11</a>]. The area of the wetland is larger than countries such as Switzerland, Belgium, the Netherlands, and Singapore [<a href="#B12">12</a>]. Owing to this enormous size, the Sudd ecosystems are of vast socioeconomic, cultural, and biological importance locally, nationally, and internationally. This accounts for why it was designated as a Ramsar site in 2006 [<a href="#B10">10</a>] making it an essential nature conservation area.</p> <p>In all crude oil producing countries around the world, during the process of crude oil production and secondary activities such as transportation and storage, several solids, liquids, and gaseous forms of wastes and pollutants are generated [<a href="#B13">13</a>]. Also, spills and discharges of petroleum hydrocarbons (PHC) in some environments have reportedly been caused by initial activities such as oil field development, transportation activities such as leakages from oil pipelines and haulage tankers, oil well waxing, and at times when refining and petrochemical equipment is being overhauled [<a href="#B14">14</a>â€"<a href="#B16">16</a>] contributing to gross contamination of ecosystems.</p> <p>Soil contamination with PHC is a widespread problem and has hazardous implications on both environment and human health [<a href="#B17">17</a>, <a href="#B18">18</a>]. As earlier stated, Sudd wetland is an oil rich zone. As a result, the Sudd ecosystems are fragile and therefore threatened by oil exploration and extraction activities since the 1980s [<a href="#B19">19</a>â€"<a href="#B21">21</a>] with notable effects on the environment and natives of the area. The commonly reported effects by studies [<a href="#B20">20</a>â€"<a href="#B22">22</a>] include high salt content in water, death of livestock, reduction in vegetation cover, and outbreak of strange diseases hitherto uncommon among people in the local communities.</p> <p>Attempts to remediate hydrocarbon-contaminated soil are a priority in national development plans of many oil-producing countries in order to counteract the harmful effects of PHC [<a href="#B23">23</a>, <a href="#B24">24</a>]. South Sudan is not exceptional. The techniques deployed in remediation of PHC polluted soil are fivefold: chemical, physical, electrical, thermal, and biological [<a href="#B25">25</a>, <a href="#B26">26</a>]. The use of chemical treatment involves chemical precipitation, membrane separation, ion exchange, carbon absorption, aqueous chemical oxidation, and surfactant enhanced recovery [<a href="#B25">25</a>]. With regard to physical treatment, the main treatment measures involve land filling, pumping and treating, dual phase extraction as well as air sparging, and dual phase extraction [<a href="#B25">25</a>, <a href="#B26">26</a>]. For electrical remediation, electrical principles are applied to decontaminate particular sites though only limited to granular type of soil contaminated with heavy metals [<a href="#B25">25</a>]. The thermal methods are largely used in environments where the contaminants are highly volatile and include <i>in situ</i> vitrification, incineration, and electrical pyrolysis. However, the above-mentioned techniques are not largely used at present because they are associated with various shortcomings such as high cost, handling of the generated excess waste, and secondary contamination [<a href="#B25">25</a>, <a href="#B26">26</a>]. This has given way to biological remediation techniques. It is not by surprise therefore that in recent times, in most crude oil producing countries, biological remediation techniques are dominating any efforts for ecorestoration of PHC contaminated soils [<a href="#B27">27</a>], bioremediation inclusive [<a href="#B25">25</a>].</p> <p>Bioremediation-based rhizoremediation is a biological technology with great potential of restoring PHC contaminated soils [<a href="#B28">28</a>, <a href="#B29">29</a>]. Rhizoremediation refers to a process in which the PHC contaminants are degraded by bacteria in the rhizosphere [<a href="#B23">23</a>, <a href="#B24">24</a>] and uses a suite of indigenous microorganisms [<a href="#B30">30</a>]. This approach is nondestructive and is environmentally acceptable [<a href="#B28">28</a>, <a href="#B29">29</a>, <a href="#B31">31</a>] making it a desirable and sustainable technique [<a href="#B31">31</a>]. For example, it does not generate toxic metabolites [<a href="#B29">29</a>]. Relatedly, Shukla and colleagues [32] contend that the mechanism provides a natural corrective solution in which the synergy between roots of phytoremediation species and resident plant growth promoting rhizobacteria (PGPR) boosts secretion of root exudates, production of siderophores, phytohormones, and phosphatases. This accounts for why the approach is regarded an effective natural remedy for ecorestoration of polluted sites [<a href="#B32">32</a>] leading to its popularity as a green technology as reported elsewhere [<a href="#B33">33</a>â€"<a href="#B35">35</a>].</p> <p>Compared to other soil inhabiting microorganisms, bacteria are the most dominant and, therefore, primary microbial communities that play a fundamental role in biodegradation of PHC contaminants. The various genera utilize hydrocarbons as carbon and energy sources [<a href="#B36">36</a>]. The degradation potential of bacteria is harnessed with biostimulation using manure. For example, cattle manure improves soil physicochemical characteristics, hence enabling adaptability of bacteria in contaminated soil [<a href="#B37">37</a>]. Additionally, some manure compounds (such as nitrogen, phosphorus, and potassium) are high-energy electron acceptors and sources of nutrients for bacteria. Although the degradation potential of different bacteria varies [<a href="#B38">38</a>], studies [<a href="#B39">39</a>, <a href="#B40">40</a>] have reported <i>Acinetobacter, Pseudomonas, Gordonia, Rhodococcus, Immundisolibacter, Luteimonas, Alcanivorax, Marinobacter, Mycobacterium, Corynebacterium, Bacillus, Ochrobactrum, Sphingomonas,</i> and <i>Hahella</i> as the most active hydrocarbon degrading genera in rhizosphere. However, the abundance of these genera in the rhizosphere of contaminated soils in the Sudd ecosystems is unknown yet; it is essential for harnessing bioremediation. Furthermore, Mackova and colleagues [<a href="#B41">41</a>] have shown that inoculation has limited efficiency to biodegrade hydrocarbons due to incompatibility between bacteria and plant species. The deployment of efficient PHC biodegrading bacteria in contaminated soils needs to be coupled with the use of phytoremediation species that enhance their survival and growth [<a href="#B42">42</a>, <a href="#B43">43</a>]. Thus, detailed characterization of rhizosphere bacteria is necessary to facilitate selection and use of efficient PHC biodegrading species of bacteria for inoculation during phytoremediation [<a href="#B44">44</a>]. Therefore, the objective of this study was to determine the rhizobacteria communities influenced by phytoremediation species growing in cattle manure biostimulated crude oil contamination soils in the Sudd region using sequences of 16S ribosomal RNA.</p> </div> <div class="2.materialsandmethods" title="sec"> <div class="title" tagx="title" title="title">
2. Materials and Methods</div> <div class="2.1.experimentaldesign" title="sec"> <div class="title" tagx="title" title="title">
2.1. Experimental Design</div> <p>A screen house pot experiment was conducted at Makerere University Agricultural Research Institute, Kabanyolo (MUARIK), from January 2018 to April 2018 (120 days). The soil used in the experiment was collected from nonhydrocarbon contaminated natural undisturbed land in the Sudd region as composite topsoil samples at a depth of 0â€"30â€‰cm. Treatments included two rates of partially decomposed (1.7â€‰:â€‰0.6â€‰:â€‰0.8 NPK) cattle manure, two rates of crude oil, and five phytoremediation plant species arranged in Completely Randomised Design (CRD). Cattle manure was applied at rates of 0 and 5â€‰gÂ·kg<sup>âˆ’1</sup> soil confirmed as economically appropriate [<a href="#B45">45</a>], while crude oil rates 0 and 75â€‰gâ€‰Â·â€‰kg<sup>âˆ’1</sup> soil were used. The phytoremediation plant species were (i) wild cotton (<i>Gossypium barbadense</i>), (ii) Sudan grass (<i>Sorghum arundinaceum</i>), (iii) wild rice (<i>Oryza longistaminata</i>), (iv) false sunflower (<i>Tithonia diversifolia</i>), and (v) thatching grass (<i>Hyparrhenia rufa</i>). These plant species are abundant in the Sudd region [<a href="#B46">46</a>] and were screened and confirmed as suitable for phytoremediation [<a href="#B47">47</a>].</p> <p>In this study, polypropylene plastic-made pots were used. The soil was apportioned into 5â€‰kg pots. To ensure that soil, cattle manure, and crude oil are mixed thoroughly, a metallic sheet was used as a mixing base. The mixture was returned into the pots with perforated bases to allow aeration. Each pot was labeled with the name of the respective treatment. The labeled pots were left for one week before planting. To cater for any PHC losses, pots were placed on their lids. Any water leachate was used to irrigate the respective pot. The lids were also washed after every two days and the wash water used to irrigate respective pots.</p> <p>At 120 days after planting, roots were removed from the pots. The firmly attached soil to the roots was collected from all pots (60 treatments) and transported in an ice cooler to the Biotechnology Laboratory of Makerere University Regional Centre for Crop Improvement at Kabanyolo, Wakiso district, Uganda, and stored at âˆ’80<sup>o</sup>C until extraction of genomic DNA.</p> </div> <div class="2.2.molecularanalysisofbacterialcommunities" title="sec"> <div class="title" tagx="title" title="title">
2.2. Molecular Analysis of Bacterial Communities</div> <p>Genomic DNA was extracted from 0.25â€‰g of the rhizosphere soil sample using DNeasy Power SoilÂ® DNA Isolation Kit (Qiagen Company) following the manufacturer's instructions. Bacterial diversity was analysed using culture independent molecular technique16S rRNA gene PCR. The primers used in PCR reactions were 341F and 785R. Aâ€‰GC clamp was added to forward primer (F). These targeted approximately 300â€‰bp of hypervariable V3 region. All PCR amplifications were performed using ThermoHybaid PCR cycler (Molecular Biology Instrumentation, Massachusetts, USA). PCR mixtures were prepared with 5â€‰<i>Î¼</i>l of Taq buffer 10Ã—, 2.5â€‰mM of MgCl<sub>2</sub>, 200â€‰<i>Î¼</i>mol of each deoxynucleoside triphosphate (dNTP), 20â€‰pmol each primer, 5â€‰<i>Î¼</i>g of bovine serum albumin, 1% of formamide and 2.5U Taq polymerase (Roche Molecular Biochemical, Mannheim, Germany), and sterile filtered Milli-Q water to a final volume of 50â€‰<i>Î¼</i>l. The PCR program was as follows: denaturing step of 94Â°C for 3â€‰min, followed by 35 cycles of 1â€‰min at 94Â°C, annealing for 1â€‰min at 55Â°C, and elongation for 1â€‰min at 72Â°C, followed by a final elongation at 72Â°C for 10â€‰min.</p> <p>The concentrations used for PCR were as follows: total mixture, 25â€‰<i>Âµ</i>l; dNTPs, 50â€‰<i>Âµ</i>M; genomic DNA, 30â€‰ng/<i>Âµ</i>l; and each primer, 10â€‰pmol/<i>Âµ</i>l. The concentration of MgCl<sub>2</sub> in the reaction mixture was maintained at 1.5â€‰mM for effective amplification. MgCl<sub>2</sub> was a cofactor for Taq enzyme and helped in adding correct dNTPs complementary to the sequence in newly synthesizing strand by binding to dNTPs. A second PCR reaction was performed using 5â€‰ml of the first PCR products as template under the same primers (GC clamp attached to the primer U968f) and conditions specified for the first PCR reaction. The PCR reactions were performed in duplicate, in order to obtain adequate DNA amount for electrophoresis. Amplification products were checked in 1.3% agarose gels stained with ethidium bromide (0.1â€‰mg/ml). This was followed by storage at âˆ’20Â°C and then sending to the LGC Genomics Sequencing Centre in Germany. Purified PCR products were pyrosequenced using Illumina MiSeq by the LGC Genomics Sequencing Centre in Germany (<a href="http://www.Ssupport.illumina.com/downloads/bcl2fastqconversion-software-v2-19.html">http://www.support.illumina.com/downloads/bcl2fastqconversion-software-v2-19.html</a>).</p> </div> <div class="2.3.datapreprocessing" title="sec"> <div class="title" tagx="title" title="title">
2.3. Data Preprocessing</div> <p>After sequencing, demultiplexing of all libraries for each sequencing lane was done using Illumina bcl2fastq 2.17.1.14 software (folder RAW). One to two mismatches were allowed in the barcode and read when the barcode distances between all libraries on the lane allowed for it. The sorting of reads by amplicon inline barcodes (folder RAW) was done through one mismatch that was allowed per barcode. The barcode sequence was then clipped from the sequence after sorting and reads with missing barcodes, one-sided barcodes, or conflicting barcode pairs were discarded.</p> <p>Clipping of sequencing adapter remnants from all reads (folder AdapterClipped) was carried out and reads with final lengthâ€‰<div class="â€‰100_UNKNOWN"> </div></p></div> <div class="2.4.bioinformaticsprocessing" title="sec"> <div class="title" tagx="title" title="title">
2.4. Bioinformatics Processing</div> <p>After primer removal and clipping of sequences, read sequences were loaded into R (version 3.6) and run through DADA2 pipeline (version 1.12) [<a href="#B48">48</a>]. The sequences were filtered and trimmed using â€œ<i>filterAndTrim</i>â€� function. The trimming specifications were as follows. First, truncation length (truncLen) was set to 250 bases for both forward and reverse reads. Secondly, cutoff for maximum expected error calculated from the quality score (maxEE) was set to 3 for both forward and reverse reads for quality plots. The remaining parameters were held as default. The error rate was estimated by function â€œ<i>learnError</i>.â€� Thirdly, a dereplication process was conducted with function â€œ<i>derepFastq</i>.â€� All identical sequencing reads were combined into one unique sequence with a corresponding abundance equal to number of reads with that unique sequence.</p> <p>Before merging, core sample inference algorithm was applied to data [<a href="#B49">49</a>]. The forward and reverse reads were then merged together to obtain full denoised sequences with function â€œ<i>mergePairs</i>.â€� As defaults in DADA2, merged sequences were only output if forward and reverse reads overlapped with a minimum of 12 bases. The merged sequences were then rearranged in an Amplicon Sequence Variant (ASV) table [<a href="#B50">50</a>] and cleaned for chimeras with functions â€œ<i>makeSequenceTable</i>â€� and â€œ<i>removeBimeraDenovo,</i>â€� respectively. For taxonomic classification, recommendations of Callahan and colleagues were used [<a href="#B50">50</a>], together with a native implementation of NaÃ¯ve Bayesian classifier method [<a href="#B51">51</a>] using function â€œ<i>assignTaxonomy</i>â€� still in DADA2 package. The ASVs with chimeras were removed from analysis using â€œ<i>subset</i>_<i>taxa</i>â€� function in â€œ<i>Phyloseq</i>â€� package (<a href="https://github.com/joey711/phyloseq">https://github.com/joey711/phyloseq</a>).</p> </div> <div class="2.5.statisticalanalysis" title="sec"> <div class="title" tagx="title" title="title">
2.5. Statistical Analysis</div> <p>All statistical analyses were performed in R software (V2.15.3). To estimate coverage and sampling diversity, rarefaction curves were constructed. â€œ<i>Phyloseq</i>â€� package calculated population diversity (Simpson index), evenness (Shannon index), and richness (Chao1). To test effect of treatments on bacterial community structures, PERMANOVA analysis using â€œ<i>adonis</i>â€� function in â€œ<i>vegan</i>â€� package was performed. To ensure that PERMANOVA results were not affected by in-group dispersions, an analysis of multivariate homogeneity of group dispersion was conducted for different treatments using â€œ<i>betadisper</i>â€� function in â€œ<i>vegan</i>â€� package (Anderson, 2006). Differences in bacterial community dispersion between treatments were assessed using PERMDISP, since a significant PERMANOVA result may indicate either a difference in centroids or an unequal dispersion between treatments. Multivariate analysis using nonmetric multidimensional scaling (nMDS) and principal component analysis (PCA) were used to explore hierarchical structure of bacterial community composition under effects of different treatments. These were calculated from Brayâ€"Curtis matrices using the â€œ<i>metaMDS</i>â€� function from â€œ<i>vegan</i>â€� package.</p> </div> </div> <div class="3.results" title="sec"> <div class="title" tagx="title" title="title">
3. Results</div> <div class="3.1.effectoftreatmentonbacterialcommunityrichness" title="sec"> <div class="title" tagx="title" title="title">
3.1. Effect of Treatment on Bacterial Community Richness</div> <p>A total of 5 million high-quality paired-end reads were generated from Illumina MiSeq platform with an average of 83,333 reads per sample (<i>n</i>â€‰=â€‰60). The tags were obtained with a maximum of 81,480 filtered sequences clustered to 3927 amplicon sequence variants (ASVs) of sixty samples at 3% confidence interval. Rarefaction was conducted to approximate the number of ASVs in random samples. The rarefaction curves (<a href="#fig1">Figure 1</a>) asymptotically approached a plateau, suggesting that the curves accurately reflected microbial community richness and indicated that the sequencing efforts were sufficient for this study.</p> <p>PERMANOVA analysis showed cattle manure and hydrocarbon contamination significantly affected rhizobacteria community but not plant species (<a href="#tab1">Table 1</a>). The interaction of plant species and cattle manure explained 5.1% of ASVs variation in community structure. Similarly, interaction of plant species and hydrocarbon accounted for 9.2% of variation in ASVs. Biostimulation of plant species for bioremediation of TPH contaminated soil with cattle manure explained 13.4 % variation in assemblages of bacterial communities (<a href="#tab1">Table 1</a>).</p> </div> <div class="3.2.taxonomicbacterialcommunitycomposition" title="sec"> <div class="title" tagx="title" title="title">
3.2. Taxonomic Bacterial Community Composition</div> <p>The sequences were classified into 33 phyla, 54 classes, 128 orders, 268 families, and 511 genera of bacteria at 80% bootstrap. Treatments containing plant species and manure (T3) had the highest number of phyla and genera. This was followed by treatments containing plant species, manure, and hydrocarbon (T4), while plant species and hydrocarbon (T2) had the least number of phyla and genera. The most and least abundant rhizobacteria communities were noted in treatments with plant species, manure and hydrocarbon (T4) and plant species and hydrocarbon (T2), respectively. Similarly, most and least diversities were observed in T4 (<i>H. rufa</i> and <i>T. diversifolia</i>) and T2 (<i>S. arundinaceum</i>), respectively (<a href="#tab2">Table 2</a>).</p> </div> <div class="3.3.bacterialcommunityabundanceanddiversity" title="sec"> <div class="title" tagx="title" title="title">
3.3. Bacterial Community Abundance and Diversity</div> <p>The richness and diversity of bacterial communities significantly differed (<i>p</i> &amp;lt; 0.000) between plant species in all tested parameters except Simpson's index (<a href="#tab2">Table 2</a>). In all the five-plant species, there was high abundance of bacterial communities in the treatment with plant only (T1). However, when TPH was added to plant species (T2), there was a significant (<i>p</i> &amp;lt; 0.05) decline in abundance as shown by a drop in Chao 1 values across the five phytoremediation species. Addition of cattle manure (T4) to the treatment (plantâ€‰+â€‰TPH) significantly increased (<i>p</i> &amp;lt; 0.05) bacterial abundance. In <i>S. arundinaceum</i>, the communities quadrupled (from 2097 to 8260); in <i>G. barbadense</i>, there was multiple increase (from 2781 to 9540) and a near multiple increase in <i>O. longistaminata</i> (from 3597 to 9168) and <i>H. rufa</i> (from 4304 to 10,310), while in <i>T. diversifolia</i>, the communities doubled (from 4223 to 9795).</p> <p>The sudden rise in abundance of bacteria communities in the treatment with plant species, hydrocarbon, and manure (T4) was attributed to addition of cattle manure. Largely, manure improves soil physicochemical properties leading to improved conditions for plant and microbial growth. Therefore, the rhizosphere became a hotspot for survival of different bacteria communities thereby accounting for the increased abundance. Moreover, cattle manure contains bacteria strains, which could have enhanced biodegradation of the TPH. In all phytoremediation plant species, there were significant differences (<i>p</i> &amp;lt; 0.05) in diversity in the Shannon index. However, the diversity in the Simpson index was not significant (<i>p</i> &amp;lt; 0.05) across all treatments. The most abundant phylum was Proteobacteria across all plant species with or without manure and the TPH accounting for about 41.6% of all ASVs, followed by Actinobacteria (12.7%) and Firmicutes (9.8%), of all ASVs (<a href="#fig2">Figure 2</a>).</p> <p>Proteobacteria, Firmicutes, and Actinobacteria dominated TPH contaminated soils across all plant species (<a href="#fig2">Figure 2</a>). For example, <i>G. barbadense</i> was mainly associated with Proteobacteria and Actinobacteria. The rhizospheres of <i>T. diversifolia</i> and <i>O. longistaminata</i> were inhabited by Proteobacteria and Firmicutes. Similarly, the rhizosphere of <i>S. arundinaceum</i> was dominated by only Proteobacteria, while the roots of <i>H. rufa</i> had a high abundance of Proteobacteria, Firmicutes, and Actinobacteria.</p> </div> <div class="3.4.rhizobacteriagenera" title="sec"> <div class="title" tagx="title" title="title">
3.4. Rhizobacteria Genera</div> <p>The dominant genera across all phytoremediation species were <i>Pseudomonas</i> (<a href="#fig3">Figure 3</a>). However, it was more dominant in <i>H. rufa</i>. Generally, besides <i>Pseudomonas,</i> other genera observed in all phytoremediation species were <i>Luteimonas, Sphingomonas, Mycobacterium, Bacillus,</i> and <i>Fusibacter.</i> These were relatively more abundant in manure treated hydrocarbon-contaminated soil (T4) (<a href="#fig3">Figure 3</a>). Compared to the rest, <i>H. rufa</i> had more <i>Bacillus</i>, <i>Fusibacter,</i> and <i>Rhodococcus.</i> Plant species <i>G. barbadense</i> was mainly associated with bacteria genera <i>Luteimonas</i> and <i>Mycobacterium,</i> while <i>T. diversifolia</i> was inhabited by <i>Bacillus</i> and <i>Luteimonas</i>. The rhizosphere of <i>O. longistaminata</i> was dominated by <i>Bacillus, Fusibacter,</i> and <i>Luteimonas,</i> while S<i>. arundinaceum</i> was largely inhabited by <i>Sphingomonas.</i> There was an increase in bacterial diversity in manure treated hydrocarbon contaminated soil due to addition of cattle manure (<a href="#fig3">Figure 3</a>).</p> </div> <div class="3.5.environmentalinfluenceoncompositionofbacterialcommunities" title="sec"> <div class="title" tagx="title" title="title">
3.5. Environmental Influence on Composition of Bacterial Communities</div> <p>Brayâ€"Curtis distance nonmetric multidimensional scaling (NMDS) revealed differences in composition of bacterial communities between hydrocarbon and nonhydrocarbon treatments. In two hydrocarbon-contaminated treatments, one was with plant species and hydrocarbon (T2) and the other with plant species, hydrocarbon, and manure (T4), where bacterial communities clustered in groups (see the ring in <a href="#fig4">Figure 4</a>). This was different from nonhydrocarbon treatments: one with plant species only (T1) and the other with plant species and manure (T3) where bacterial communities were scattered (<a href="#fig4">Figure 4</a>).</p> <p>Results from principal component analysis (PCA) also revealed separate clustering of bacterial communities between hydrocarbon and non-hydrocarbon-containing treatments. Irrespective of phytoremediation plant species, bacterial communities in the treatment containing plant species and hydrocarbon (T2) and one containing plant species, hydrocarbon, and manure (T4) clustered separately from one with plant species only (T1) as well as one with plant species and manure (T3) (<a href="#fig5">Figure 5</a>). However, the pattern of clustering was influenced by specific phytoremediation species, a factor explained by differences in percentage variances for PC1 and PC2 (<a href="#fig5">Figure 5</a>).</p> </div> </div> <div class="4.discussion" title="sec"> <div class="title" tagx="title" title="title">
4. Discussion</div> <p>There were significant variations (<i>p</i> &amp;lt; 0.000) between rhizobacterial communities of phytoremediation species with and without hydrocarbon contamination. In the treatments with plant species alone (T1), high bacterial diversity was noted. However, when hydrocarbons were introduced (T2), there was a great reduction in diversity. The bacterial community shifts and eventual decrease in richness resulted from perturbations that normally occur in hydrocarbon contaminated soil. Past studies [<a href="#B52">52</a>, <a href="#B53">53</a>] have proved that introduction of PHC in soil reduces bacterial diversity considerably regardless of the soil matrix type. Nevertheless, in this study, certain bacterial strains resilient to toxicity of PHC existed. These must have used TPH as a source of energy, carbon, or electron receptors for growth. As reported earlier, across all plant species, bacteria genera <i>Luteimonas, Pseudomonas</i>, and <i>Sphingomonas</i> (phylum Proteobacteria)<i>, Mycobacteria</i> and <i>Rhodococcus</i> (phylum Actinobacteria) and genera <i>Bacillus</i> and <i>Fusibacter</i> (phylum Firmicutes) were abundant in the treatment with plant and hydrocarbon (T2) and one with plant species, hydrocarbon, and cattle manure (T4).</p> <p>Similarly, bacterial communities in hydrocarbon-contaminated soil were significantly affected by biostimulation with cattle manure (T4). The inclusion of cattle manure posted both direct and indirect benefits for the survival of bacterial communities. Directly, cattle manure amendments improved soil physicochemical characteristics enabling speedy adaptation by microorganisms. Furthermore, the introduction of cattle manure must have increased on soil fertility by adding soil organic carbon (SOC), total nitrogen (TN), and NPK. This must have improved plant resilience and performance in the PHC contaminated soil. Accumulation of soil organic carbon for example not only results in increased microbial biomass but also affects microbial community structure and functional diversity [<a href="#B54">54</a>]. Therefore, cattle manure indirectly influenced a spectacular increase in the microbial diversity observed. Cattle manure additions also improved soil pH and physical properties (aggregation and porosity), thus creating favorable growth conditions for microbes. Earlier studies [<a href="#B55">55</a>â€"<a href="#B57">57</a>] have shown that addition of organic manure to hydrocarbon-contaminated soil enhances multiplication of bacteria population. Furthermore, addition of cattle manure improves soil fertility, which is vital for sustained plant growth [<a href="#B56">56</a>].</p> <p>In the rhizosphere, the bacterial colonize the root surfaces, compete against other microbes and form synergestic interactions with host plants [<a href="#B58">58</a>]. Phyla Proteobacteria, Actinobacteria, and Firmicutes dominated the treatment with hydrocarbon contaminated soil and plant species (T2) and treatment T4 with plant species and manure treated hydrocarbon contaminated soil (T4). These phyla contain members of organotrophic microorganisms that utilize a wide range of organic substrates perhaps including hydrocarbon. Although bacterial strains survive best in aerobic conditions, the three phyla also thrive well in anaerobic environments. Their survival in anaerobic conditions is guaranteed by secretion of intracellular and extracellular enzymes which help in biodegradation of recalcitrant and organopollutants. These bacteria have enzymes capable of assimilating, degrading, and utilizing different hydrocarbon constituents as sources of carbon and energy [<a href="#B29">29</a>, <a href="#B58">58</a>].</p> <p>Assimilation is a complex biological oxidation process enhanced by supplementation with fixed nitrogen, phosphate, and other nutrients [<a href="#B58">58</a>]. For example, one of the enzymes, oxidoreductases, enables oxidative coupling to take place enabling both phyla to extract energy via energy-yielding biochemical reactions which cleaves chemical bonds, assisting transfer of electrons from a reduced organic substrate (donor) to another chemical compound (acceptor). In this process, contaminants are finally oxidized to harmless compounds. This guarantees survival of the bacteria communities in a less toxic environment. Furthermore, oxidoreductases catalyze humification of various phenolic substances in soil environment through polymerization and copolymerization with other substrates [<a href="#B59">59</a>].</p> <p>The three phyla are also known for secreting oxygenases. In the test samples, oxygenases both monooxygenases and dioxygenases could have been secreted by the phyla. Monooxygenases catalyze desulfurization, dehalogenation, denitrification, ammonification, hydroxylation, biotransformation, and biodegradation of various aromatic and aliphatic compounds, while dioxygenases introduce molecular oxygen into their substrate [<a href="#B60">60</a>]. Therefore, both processes must have aided transformation of aromatic precursors into aliphatic products that are less toxic, creating better living environmental conditions. Furthermore, Protobacteria, Actinobacteria, and Firmicutes are known for secreting lacasses [<a href="#B61">61</a>] that serves as a catalyst for the rapid oxidation of phenolic and aromatic substrates. Besides, lacasses also enhance reduction of molecular oxygen to water [<a href="#B62">62</a>, <a href="#B63">63</a>]. Equally, lacasses decarboxylate phenolic and methoxy-phenolic acids into nutritious compounds for bacteria [<a href="#B64">64</a>].</p> <p>Results from NMDS and PCA showed clustering of bacteria in hydrocarbon contaminated soil with plant species (T2) and one biostimulated by manure (T4), while in nonhydrocarbon contaminated soil with plant species only (T1) and one with plant species and manure (T3), the communities of bacteria were scattered from each other. The clustering could be associated with catabolic potential of dominant bacteria phyla established by this study. Although Sutton [<a href="#B53">53</a>] reasoned that regardless of soil matrix type, clean samples (nonhydrocarbon contaminated) have higher diversity than contaminated soil, results of this study showed more diversity and clustering in hydrocarbon contaminated treatments compared to those without. The clustering must have occurred due to the ability of the treatments with TPH to selectively stimulate bacterial propagation especially through addition of carbon (mixture of aliphatic and aromatic hydrocarbons) that enriches taxa by serving as growth substrates [<a href="#B65">65</a>]. The metabolic capacities of taxa therefore enabled biotransformation of various organic compounds by breaking down their bigger molecules into smaller units either by oxidation to release energy or complete utilization in other anabolic reactions. The versatility of taxa to use both saturated aliphatic and aromatic hydrocarbons played key role in enhancing survival and, consequently, removal of heterogeneous toxic contamination. This scenario has been observed in past studies. For example, Peng and colleagues [<a href="#B66">66</a>] concluded that oil-polluted soils support a cornucopia of bacterial communities due to their richness in organic matter.</p> <p>Addition of cattle manure to treatments with TPH increased clustering of taxa. Biostimulation with cattle manure boosted growth performance of the rhizosphere of phytoremediation plant species. The associated exudates were colonized by taxa leading to increased clustering as observed in treatments with plant species and hydrocarbon (T2) and plant species, hydrocarbon, and cattle manure (T4) (see <a href="#fig4">Figure 4</a>). This concurs with Praeg [<a href="#B56">56</a>] that rhizosphere zones of plants are hotspots for microbial growth, abundance and diversity due to nutrient availability.</p> </div> <div class="5.conclusionandrecommendation" title="sec"> <div class="title" tagx="title" title="title">
5. Conclusion and Recommendation</div> <p>Plant species growing in TPH contaminated soil are inhabited by various strains of rhizobacteria because their roots provide excellent living conditions. In this study, rhizobacteria genera <i>Bacillus, Fusibacter, Luteimonas, Mycobacterium, Pseudomonas Rhodococcus,</i> and <i>Sphingomonas</i> were abundant in rhizosphere of phytoremediation species. In the same vein, the study also established that addition of cattle manure enhanced multiplication of these genera. Therefore, it is concluded that, in order to achieve better bioremediation, TPH contaminated soils should be bio-stimulated with cattle manure to increase rhizobacteria richness. With the exception of <i>Mycobacterium</i> (a genus that includes dangerous pathogens), this study recommends use of the genera listed above as an inoculum during ecorestoration of PHC contaminated soils in Sudd region, South Sudan. <i>Mycobacterium</i> is a carrier of tuberculosis (TB) which is a common cause of death with a prevalence rate of 257 per 100,000 people in South Sudan.</p> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgments</div> <p>The authors acknowledge support of the Ministry of petroleum and Gas, Dar Petroleum Company Ltd., Sudanese Petroleum, and other associated laboratories and colleagues at University of Juba for valuable comments. The authors also acknowledge the support of NORHED Project Implementers, Dr. Busulwa Henry and Dr. Bojoi Moses Tomor, of Makerere University and University of Juba, respectively. This study was funded by NORAD through the Sudd project (NORHED Project no. SSD-13/0021) implemented by University of Juba, Makerere University, and the Norwegian University of Life Sciences, NMBU.</p> </div> <div class="data-availability" title="data-availability"> <div class="title" tagx="title" title="title">
Data Availability</div> <p>Data for the outputs reported in this paper are part of an ongoing Ph.D. study and can only be availed in consultation with the corresponding author reachable at <span class="email" tagx="email" title="email">janenajeb@yahoo.com</span>.</p> </div> <div class="COI-statement" title="COI-statement"> <div class="title" tagx="title" title="title">
Conflicts of Interest</div> <p>The authors declare that there are no conflicts of interest.</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <li tag="ref"><a name="B1" /><span class="label" tagx="label" title="label">1</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jain</span><span class="given-names" tagx="given-names" title="given-names">P. K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gupta</span><span class="given-names" tagx="given-names" title="given-names">V. K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gaur</span><span class="given-names" tagx="given-names" title="given-names">R. K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lowry</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jaroli</span><span class="given-names" tagx="given-names" title="given-names">D. P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chauhan</span><span class="given-names" tagx="given-names" title="given-names">U. K.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Bioremediation of petroleum oil contaminated soil and water</span><span class="source" tagx="source" title="source"><i>Research Journal of Environmental Toxicology</i></span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">5</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">26</span><span class="pub-id"><a href="https://dx.doi.org/10.3923/rjet.2011.1.26">10.3923/rjet.2011.1.26</a></span><span class="pub-id" /></span></li> <li tag="ref"><a name="B2" /><span class="label" tagx="label" title="label">2</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Adesina</span><span class="given-names" tagx="given-names" title="given-names">G. O.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Adelasoye</span><span class="given-names" tagx="given-names" title="given-names">K. A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Effect of crude oil pollution on heavy metal contents, microbial population in soil, and maize and cowpea growth</span><span class="source" tagx="source" title="source"><i>Agricultural Sciences</i></span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">5</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">43</span><span class="lpage" tagx="lpage" title="lpage">50</span><span class="pub-id"><a href="https://dx.doi.org/10.4236/as.2014.51004">10.4236/as.2014.51004</a></span></span></li> <li tag="ref"><a name="B3" /><span class="label" tagx="label" title="label">3</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mukhtar</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Griffin</span><span class="given-names" tagx="given-names" title="given-names">L. D.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Bioremediation-a beneficial use of biotechnology and allied engineering technologies in oil producing regions</span><span class="source" tagx="source" title="source"><i>International Journal of Sustainable Water and Environmental Systems</i></span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">7</span><span class="issue" tagx="issue" title="issue">2</span><span class="fpage" tagx="fpage" title="fpage">83</span><span class="lpage" tagx="lpage" title="lpage">85</span></span></li> <li tag="ref"><a name="B4" /><span class="label" tagx="label" title="label">4</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ojewumi</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Anenih</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Taiwo</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Adekeye</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Awolu</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ojewumi</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span></span><span class="mixed-article-title" title="mixed-article-title">A bioremediation study of raw and treated crude petroleum oil polluted soil with Aspergillus Niger and Pseudomonas aeruginosa</span><span class="source" tagx="source" title="source"><i>Journal of Ecological Engineering</i></span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">19</span><span class="issue" tagx="issue" title="issue">2</span><span class="fpage" tagx="fpage" title="fpage">226</span><span class="lpage" tagx="lpage" title="lpage">235</span><span class="pub-id"><a href="https://dx.doi.org/10.12911/22998993/83564">10.12911/22998993/83564</a></span><span class="pub-id" /></span></li> <li tag="ref"><a name="B5" /><span class="label" tagx="label" title="label">5</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rada</span><span class="given-names" tagx="given-names" title="given-names">E. C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Andreottola</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Istrate</span><span class="given-names" tagx="given-names" title="given-names">I. A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Viotti</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Conti</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Magaril</span><span class="given-names" tagx="given-names" title="given-names">E. R.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Remediation of soil polluted by organic compounds through chemical oxidation and phytoremediation combined with DCT</span><span class="source" tagx="source" title="source"><i>International Journal of Environmental Research and Public Health</i></span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">16</span><span class="issue" tagx="issue" title="issue">17</span><span class="fpage" tagx="fpage" title="fpage">p. 3179</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/ijerph16173179">10.3390/ijerph16173179</a></span><span class="pub-id" /></span></li> <li tag="ref"><a name="B6" /><span class="label" tagx="label" title="label">6</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cust</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harding</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Oil in South Sudan: implications from international experience</span><span class="source" tagx="source" title="source"><i>International Growth Centre (IGC), Working Paper</i></span><span class="year" tagx="year" title="year">2013</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London, UK</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">London School of Economic and Political Science</span><span class="fpage" tagx="fpage" title="fpage">47</span></span></li> <li tag="ref"><a name="B7" /><span class="label" tagx="label" title="label">7</span><span class="element-citation'"><span class="collab'">collab: U.S. Energy Information Administration</span><span class="source" tagx="source" title="source"><i>Production of Crude Oil Including Lease Condensate 2019</i></span><span class="year" tagx="year" title="year">2019</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Washington, DC, USA</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">U.S. Energy Information Administration</span></span></li> <li tag="ref"><a name="B8" /><span class="label" tagx="label" title="label">8</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuch</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bavumiragira</span><span class="given-names" tagx="given-names" title="given-names">J. P.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Impacts of crude oil exploration and production on environment and its implications on human health: South Sudan Review</span><span class="source" tagx="source" title="source"><i>International Journal of Scientific and Research Publications (IJSRP)</i></span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">9</span><span class="issue" tagx="issue" title="issue">4</span><span class="fpage" tagx="fpage" title="fpage">p. p8836</span><span class="pub-id"><a href="https://dx.doi.org/10.29322/ijsrp.9.04.2019.p8836">10.29322/ijsrp.9.04.2019.p8836</a></span></span></li> <li tag="ref"><a name="B9" /><span class="label" tagx="label" title="label">9</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">De Waal</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">When Kleptocracy becomes insolvent: brute causes of the civil war in South Sudan</span><span class="source" tagx="source" title="source"><i>African Affairs</i></span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">113</span><span class="issue" tagx="issue" title="issue">452</span><span class="fpage" tagx="fpage" title="fpage">347</span><span class="lpage" tagx="lpage" title="lpage">369</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/afraf/adu028">10.1093/afraf/adu028</a></span><span class="pub-id" /></span></li> <li tag="ref"><a name="B10" /><span class="label" tagx="label" title="label">10</span><span class="element-citation'"><span class="collab'">collab: Ramsar Convention Secretariat</span><span class="mixed-article-title" title="mixed-article-title">The ramsar strategic plan 2009-2015: goals, strategies and expectations for the ramsar conventionâ€™s implementation for the period 2009 to 2015</span><span class="source" tagx="source" title="source"><i>Ramsar handbooks for the wise use of wetlands</i></span><span class="year" tagx="year" title="year">2010</span><span class="edition" tagx="edition" title="edition">4th</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Gland, Switzerland</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Ramsar Convention Secretariat</span></span></li> <li tag="ref"><a name="B11" /><span class="label" tagx="label" title="label">11</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sosnowski</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ghoneim</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Burke</span><span class="given-names" tagx="given-names" title="given-names">J. J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hines</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Halls</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Remote regions, remote data: a spatial investigation of precipitation, dynamic land covers, and conflict in the Sudd wetland of South Sudan</span><span class="source" tagx="source" title="source"><i>Applied Geography</i></span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">69</span><span class="fpage" tagx="fpage" title="fpage">51</span><span class="lpage" tagx="lpage" title="lpage">64</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.apgeog.2016.02.007">10.1016/j.apgeog.2016.02.007</a></span><span class="pub-id" /></span></li> <li tag="ref"><a name="B12" /><span class="label" tagx="label" title="label">12</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="comment">NationMaster (2020). Geography &amp;gt; Land Area &amp;gt; Square Miles: Countries Compared. <a href="https://www.nationmaster.com/country-info/stats/Geography/Land-area/Square-miles">https://www.nationmaster.com/country-info/stats/Geography/Land-area/Square-miles</a></span></span></li> <li tag="ref"><a name="B13" /><span class="label" tagx="label" title="label">13</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ngene</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tota-Maharaj</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eke</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hills</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Environmental and economic impacts of crude oil and natural gas production in developing countries</span><span class="source" tagx="source" title="source"><i>International Journal of Economy, Energy and Environment</i></span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">1</span><span class="issue" tagx="issue" title="issue">3</span><span class="fpage" tagx="fpage" title="fpage">64</span><span class="lpage" tagx="lpage" title="lpage">73</span></span></li> <li tag="ref"><a name="B14" /><span class="label" tagx="label" title="label">14</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Berger</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schwarzbauer</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Historical deposition of riverine contamination on terrestrial floodplains as revealed by organic indicators from an industrial point source</span><span class="source" tagx="source" title="source"><i>Water, air &amp;amp; Soil Pollution</i></span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">227</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">p. 20</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s11270-015-2708-8">10.1007/s11270-015-2708-8</a></span><span class="pub-id" /></span></li> <li tag="ref"><a name="B15" /><span class="label" tagx="label" title="label">15</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shapiro</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khanna</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ustin</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Vegetation impact and recovery from oil-induced stress on three ecologically distinct wetland sites in the gulf of Mexico</span><span class="source" tagx="source" title="source"><i>Journal of Marine Science and Engineering</i></span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">4</span><span class="issue" tagx="issue" title="issue">2</span><span class="fpage" tagx="fpage" title="fpage">33</span><span class="lpage" tagx="lpage" title="lpage">39</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/jmse4020033">10.3390/jmse4020033</a></span><span class="pub-id" /></span></li> <li tag="ref"><a name="B16" /><span class="label" tagx="label" title="label">16</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ke</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Porous metal filters and membranes for oilâ€"water separation</span><span class="source" tagx="source" title="source"><i>Nanoscale Research Letters</i></span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">13</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">p. 284</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s11671-018-2693-0">10.1186/s11671-018-2693-0</a></span><span class="pub-id" /></span></li> <li tag="ref"><a name="B17" /><span class="label" tagx="label" title="label">17</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fan</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">He</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Heavy metal contamination in soil and Brown rice and human health risk assessment near three mining areas in Central China</span><span class="source" tagx="source" title="source"><i>Journal of Healthcare Engineering</i></span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">2017</span><span class="fpage" tagx="fpage" title="fpage">9</span><span class="pub-id" /><span class="pub-id"><a href="https://dx.doi.org/10.1155/2017/4124302">10.1155/2017/4124302</a></span><span class="pub-id" /></span></li> <li tag="ref"><a name="B18" /><span class="label" tagx="label" title="label">18</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Ecological risk assessment of heavy metals in soils surrounding oil waste disposal areas</span><span class="source" tagx="source" title="source"><i>Environmental Monitoring and Assessment</i></span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">188</span><span class="issue" tagx="issue" title="issue">2</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">10</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s10661-016-5093-x">10.1007/s10661-016-5093-x</a></span><span class="pub-id" /><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26627206">26627206</a></span></span></li> <li tag="ref"><a name="B19" /><span class="label" tagx="label" title="label">19</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mager</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wirkus</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schoepfer</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Impact assessment of oil exploitation in South Sudan using multi-temporal landsat imagery</span><span class="source" tagx="source" title="source"><i>Photogrammetrieâ€"Fernerkundungâ€"Geoinformation</i></span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">2016</span><span class="issue" tagx="issue" title="issue">4</span><span class="fpage" tagx="fpage" title="fpage">211</span><span class="lpage" tagx="lpage" title="lpage">223</span><span class="pub-id"><a href="https://dx.doi.org/10.1127/pfg/2016/0300">10.1127/pfg/2016/0300</a></span><span class="pub-id" /></span></li> <li tag="ref"><a name="B20" /><span class="label" tagx="label" title="label">20</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pragst</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stieglitz</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Runge</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">High concentrations of lead and barium in hair of the rural population caused by water pollution in the Thar Jath oilfields in South Sudan</span><span class="source" tagx="source" title="source"><i>Forensic Science International</i></span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">274</span><span class="fpage" tagx="fpage" title="fpage">99</span><span class="lpage" tagx="lpage" title="lpage">106</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.forsciint.2016.12.022">10.1016/j.forsciint.2016.12.022</a></span><span class="pub-id" /><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28069428">28069428</a></span></span></li> <li tag="ref"><a name="B21" /><span class="label" tagx="label" title="label">21</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rueskamp</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ariki</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stieglitz</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Treskatis</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Effect of oil exploration and production on the salinity of a marginally permeable aquifer system in the Thar Jath-, Mala- and Unity Oilfields, Southern Sudan</span><span class="source" tagx="source" title="source"><i>Zentralblatt fÃ¼r Geologie und PalÃ¤ontologie, Teil I</i></span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">2014</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">95</span><span class="lpage" tagx="lpage" title="lpage">115</span><span class="pub-id"><a href="https://dx.doi.org/10.1127/zgpi/2014/0095-0115">10.1127/zgpi/2014/0095-0115</a></span></span></li> <li tag="ref"><a name="B22" /><span class="label" tagx="label" title="label">22</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tutdel</span><span class="given-names" tagx="given-names" title="given-names">I. Y.</span></span></span><span class="source" tagx="source" title="source"><i>Falling between the Cracks? Prospects for Environmental Litigation Arising from Oil Production in Southern Sudan</i></span><span class="year" tagx="year" title="year">2010</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Johannesburg, South Africa</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Institute of International Affairs</span></span></li> <li tag="ref"><a name="B23" /><span class="label" tagx="label" title="label">23</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bisht</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pandey</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sood</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sharma</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bisht</span><span class="given-names" tagx="given-names" title="given-names">N. S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Biodegradation of naphthalene and anthracene by chemo-tactically active rhizobacteria of Populus deltoides</span><span class="source" tagx="source" title="source"><i>Brazilian Journal of Microbiology</i></span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">41</span><span class="issue" tagx="issue" title="issue">4</span><span class="fpage" tagx="fpage" title="fpage">922</span><span class="lpage" tagx="lpage" title="lpage">930</span><span class="pub-id"><a href="https://dx.doi.org/10.1590/s1517-83822010000400011">10.1590/s1517-83822010000400011</a></span><span class="pub-id" /><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24031572">24031572</a></span></span></li> <li tag="ref"><a name="B24" /><span class="label" tagx="label" title="label">24</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bisht</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pandey</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bhargava</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sharma</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kumar</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sharma</span><span class="given-names" tagx="given-names" title="given-names">K. D.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Bioremediation of polyaromatic hydrocarbons (PAHs) using rhizosphere technology</span><span class="source" tagx="source" title="source"><i>Brazilian Journal of Microbiology</i></span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">46</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">7</span><span class="lpage" tagx="lpage" title="lpage">21</span><span class="pub-id"><a href="https://dx.doi.org/10.1590/s1517-838246120131354">10.1590/s1517-838246120131354</a></span><span class="pub-id" /><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26221084">26221084</a></span></span></li> <li tag="ref"><a name="B25" /><span class="label" tagx="label" title="label">25</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Anih</span><span class="given-names" tagx="given-names" title="given-names">C. E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Okewale</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moses</span><span class="given-names" tagx="given-names" title="given-names">N. O. E.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Effect of nutrients on bioremediation of crude oil-polluted water</span><span class="source" tagx="source" title="source"><i>American Journal of Environmental Science and Engineering</i></span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">3</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">7</span><span class="pub-id"><a href="https://dx.doi.org/10.11648/j.ajese.20190301.11">10.11648/j.ajese.20190301.11</a></span></span></li> <li tag="ref"><a name="B26" /><span class="label" tagx="label" title="label">26</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ossai</span><span class="given-names" tagx="given-names" title="given-names">I. C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ahmed</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hassan</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hamid</span><span class="given-names" tagx="given-names" title="given-names">F. S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Remediation of soil and water contaminated with petroleum hydrocarbon: a review</span><span class="source" tagx="source" title="source"><i>Environmental Technology &amp;amp; Innovation</i></span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">17</span><span class="fpage" tagx="fpage" title="fpage">p. 100526</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.eti.2019.100526">10.1016/j.eti.2019.100526</a></span></span></li> <li tag="ref"><a name="B27" /><span class="label" tagx="label" title="label">27</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shahaby</span><span class="given-names" tagx="given-names" title="given-names">A. F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alharthi</span><span class="given-names" tagx="given-names" title="given-names">A. A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">El Tarras</span><span class="given-names" tagx="given-names" title="given-names">A. E.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Bioremediation of petroleum oil by potential biosurfactant-producing bacteria using gravimetric assay</span><span class="source" tagx="source" title="source"><i>International Journal of Current Microbiology and Applied Sciences</i></span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">4</span><span class="issue" tagx="issue" title="issue">5</span><span class="fpage" tagx="fpage" title="fpage">390</span><span class="lpage" tagx="lpage" title="lpage">403</span></span></li> <li tag="ref"><a name="B28" /><span class="label" tagx="label" title="label">28</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guarino</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zuzolo</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marziano</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Investigation and assessment for an effective approach to the reclamation of polycyclic aromatic hydrocarbon (PAHs) contaminated site: SIN bagnoli, Italy</span><span class="source" tagx="source" title="source"><i>Scientific Reports</i></span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">9</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">12</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41598-019-48005-7">10.1038/s41598-019-48005-7</a></span><span class="pub-id" /><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30626917">30626917</a></span></span></li> <li tag="ref"><a name="B29" /><span class="label" tagx="label" title="label">29</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Niu</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cao</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Effect of soil aeration on root morphology and photosynthetic characteristics of potted tomato plants (Solanum lycopersicum) at different NaCl salinity levels</span><span class="source" tagx="source" title="source"><i>BMC Plant Biology</i></span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">19</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">p. 331</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s12870-019-1927-3">10.1186/s12870-019-1927-3</a></span><span class="pub-id" /></span></li> <li tag="ref"><a name="B30" /><span class="label" tagx="label" title="label">30</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Veerapagu</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jeya</span><span class="given-names" tagx="given-names" title="given-names">K. R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kalaivani</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jeyanthi</span><span class="given-names" tagx="given-names" title="given-names">K. A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Geethanjali</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Screening of hydrocarbon degrading bacteria isolated from oil contaminated soil</span><span class="source" tagx="source" title="source"><i>The Pharma Innovation Journal</i></span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">8</span><span class="issue" tagx="issue" title="issue">6</span><span class="fpage" tagx="fpage" title="fpage">69</span><span class="lpage" tagx="lpage" title="lpage">72</span></span></li> <li tag="ref"><a name="B31" /><span class="label" tagx="label" title="label">31</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Obi</span><span class="given-names" tagx="given-names" title="given-names">L. U.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Atagana</span><span class="given-names" tagx="given-names" title="given-names">H. I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Adeleke</span><span class="given-names" tagx="given-names" title="given-names">R. A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Isolation and characterisation of crude oil sludge degrading bacteria</span><span class="source" tagx="source" title="source"><i>SpringerPlus</i></span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">5</span><span class="fpage" tagx="fpage" title="fpage">p. 1946</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s40064-016-3617-z">10.1186/s40064-016-3617-z</a></span><span class="pub-id" /></span></li> <li tag="ref"><a name="B32" /><span class="label" tagx="label" title="label">32</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shukla</span><span class="given-names" tagx="given-names" title="given-names">K. P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sharma</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Singh</span><span class="given-names" tagx="given-names" title="given-names">N. K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Singh</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bisht</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kumar</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Rhizoremediation: a promising rhizosphere technology</span><span class="source" tagx="source" title="source"><i>Applied Bioremediation Active and Passive Approaches</i></span><span class="year" tagx="year" title="year">2013</span><span class="fpage" tagx="fpage" title="fpage">333</span><span class="lpage" tagx="lpage" title="lpage">352</span></span></li> <li tag="ref"><a name="B33" /><span class="label" tagx="label" title="label">33</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brooijmans</span><span class="given-names" tagx="given-names" title="given-names">R. J. W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pastink</span><span class="given-names" tagx="given-names" title="given-names">M. I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Siezen</span><span class="given-names" tagx="given-names" title="given-names">R. J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Hydrocarbon-degrading bacteria: the oil-spill clean-up crew</span><span class="source" tagx="source" title="source"><i>Microbial Biotechnology</i></span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">2</span><span class="issue" tagx="issue" title="issue">6</span><span class="fpage" tagx="fpage" title="fpage">587</span><span class="lpage" tagx="lpage" title="lpage">594</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1751-7915.2009.00151.x">10.1111/j.1751-7915.2009.00151.x</a></span><span class="pub-id" /><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21255292">21255292</a></span></span></li> <li tag="ref"><a name="B34" /><span class="label" tagx="label" title="label">34</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oyetibo</span><span class="given-names" tagx="given-names" title="given-names">G. O.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miyauchi</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Biotechnological remedies for the estuarine environment polluted with heavy metals and persistent organic pollutants</span><span class="source" tagx="source" title="source"><i>International Biodeterioration &amp;amp; Biodegradation</i></span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">119</span><span class="fpage" tagx="fpage" title="fpage">614</span><span class="lpage" tagx="lpage" title="lpage">625</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ibiod.2016.10.005">10.1016/j.ibiod.2016.10.005</a></span><span class="pub-id" /></span></li> <li tag="ref"><a name="B35" /><span class="label" tagx="label" title="label">35</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prince</span><span class="given-names" tagx="given-names" title="given-names">R. C.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The microbiology of marine oil spill bioremediation</span><span class="source" tagx="source" title="source"><i>Petroleum Microbiology</i></span><span class="year" tagx="year" title="year">2005</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Washington, DC, USA</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">American Society for Microbiology (ASM)</span><span class="fpage" tagx="fpage" title="fpage">317</span><span class="lpage" tagx="lpage" title="lpage">335</span></span></li> <li tag="ref"><a name="B36" /><span class="label" tagx="label" title="label">36</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Das</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chandran</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Microbial degradation of petroleum hydrocarbon contaminants: an overview</span><span class="source" tagx="source" title="source"><i>Biotechnology Research International</i></span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">2011</span><span class="fpage" tagx="fpage" title="fpage">13</span><span class="pub-id" /><span class="pub-id"><a href="https://dx.doi.org/10.4061/2011/941810">10.4061/2011/941810</a></span></span></li> <li tag="ref"><a name="B37" /><span class="label" tagx="label" title="label">37</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abed</span><span class="given-names" tagx="given-names" title="given-names">R. M. M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Al-Sabahi</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Al-Maqrashi</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Al-Habsi</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Al-Hinai</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Characterization of hydrocarbon-degrading bacteria isolated from oil-contaminated sediments in the Sultanate of Oman and evaluation of bioaugmentation and biostimulation approaches in microcosm experiments</span><span class="source" tagx="source" title="source"><i>International Biodeterioration &amp;amp; Biodegradation</i></span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">89</span><span class="fpage" tagx="fpage" title="fpage">58</span><span class="lpage" tagx="lpage" title="lpage">66</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ibiod.2014.01.006">10.1016/j.ibiod.2014.01.006</a></span><span class="pub-id" /></span></li> <li tag="ref"><a name="B38" /><span class="label" tagx="label" title="label">38</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Agamuthu</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tan</span><span class="given-names" tagx="given-names" title="given-names">Y. S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fauziah</span><span class="given-names" tagx="given-names" title="given-names">S. H.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Bioremediation of hydrocarbon contaminated soil using selected organic wastes</span><span class="source" tagx="source" title="source"><i>Procedia Environmental Sciences</i></span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">18</span><span class="fpage" tagx="fpage" title="fpage">694</span><span class="lpage" tagx="lpage" title="lpage">702</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.proenv.2013.04.094">10.1016/j.proenv.2013.04.094</a></span></span></li> <li tag="ref"><a name="B39" /><span class="label" tagx="label" title="label">39</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Siciliano</span><span class="given-names" tagx="given-names" title="given-names">S. D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fortin</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mihoc</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Selection of specific endophytic bacterial genotypes by plants in response to soil contamination</span><span class="source" tagx="source" title="source"><i>Applied and Environmental Microbiology</i></span><span class="year" tagx="year" title="year">2001</span><span class="volume" tagx="volume" title="volume">67</span><span class="issue" tagx="issue" title="issue">6</span><span class="fpage" tagx="fpage" title="fpage">2469</span><span class="lpage" tagx="lpage" title="lpage">2475</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/aem.67.6.2469-2475.2001">10.1128/aem.67.6.2469-2475.2001</a></span><span class="pub-id" /><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11375152">11375152</a></span></span></li> <li tag="ref"><a name="B40" /><span class="label" tagx="label" title="label">40</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hassanshahian</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zeynalipour</span><span class="given-names" tagx="given-names" title="given-names">M. S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Musa</span><span class="given-names" tagx="given-names" title="given-names">F. H.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Isolation and characterization of crude oil degrading bacteria from the Persian Gulf (Khorramshahr provenance)</span><span class="source" tagx="source" title="source"><i>Marine Pollution Bulletin</i></span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">82</span><span class="issue" tagx="issue" title="issue">1-2</span><span class="fpage" tagx="fpage" title="fpage">39</span><span class="lpage" tagx="lpage" title="lpage">44</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.marpolbul.2014.03.027">10.1016/j.marpolbul.2014.03.027</a></span><span class="pub-id" /><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24703768">24703768</a></span></span></li> <li tag="ref"><a name="B41" /><span class="label" tagx="label" title="label">41</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mackova</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dowling</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Macek</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span></span><span class="source" tagx="source" title="source"><i>Phytoremediation and rhizoremediation</i></span><span class="year" tagx="year" title="year">2006</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Berlin, Germany</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Springer Science &amp;amp; Business Media</span></span></li> <li tag="ref"><a name="B42" /><span class="label" tagx="label" title="label">42</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liste</span><span class="given-names" tagx="given-names" title="given-names">H.-H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prutz</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Plant performance, dioxygenase-expressing rhizosphere bacteria, and biodegradation of weathered hydrocarbons in contaminated soil</span><span class="source" tagx="source" title="source"><i>Chemosphere</i></span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">62</span><span class="issue" tagx="issue" title="issue">9</span><span class="fpage" tagx="fpage" title="fpage">1411</span><span class="lpage" tagx="lpage" title="lpage">1420</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.chemosphere.2005.05.018">10.1016/j.chemosphere.2005.05.018</a></span><span class="pub-id" /><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15996713">15996713</a></span></span></li> <li tag="ref"><a name="B43" /><span class="label" tagx="label" title="label">43</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Phillips</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Greer</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Germida</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Culture-based and culture-independent assessment of the impact of mixed and single plant treatments on rhizosphere microbial communities in hydrocarbon contaminated flare-pit soil</span><span class="source" tagx="source" title="source"><i>Soil Biology and Biochemistry</i></span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">38</span><span class="issue" tagx="issue" title="issue">9</span><span class="fpage" tagx="fpage" title="fpage">2823</span><span class="lpage" tagx="lpage" title="lpage">2833</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.soilbio.2006.04.038">10.1016/j.soilbio.2006.04.038</a></span><span class="pub-id" /></span></li> <li tag="ref"><a name="B44" /><span class="label" tagx="label" title="label">44</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Daane</span><span class="given-names" tagx="given-names" title="given-names">L. L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harjono</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zylstra</span><span class="given-names" tagx="given-names" title="given-names">G. J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">HÃ¤ggblom</span><span class="given-names" tagx="given-names" title="given-names">M. M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Isolation and characterization of polycyclic aromatic hydrocarbon-degrading bacteria associated with the rhizosphere of salt marsh plants</span><span class="source" tagx="source" title="source"><i>Applied and Environmental Microbiology</i></span><span class="year" tagx="year" title="year">2001</span><span class="volume" tagx="volume" title="volume">67</span><span class="issue" tagx="issue" title="issue">6</span><span class="fpage" tagx="fpage" title="fpage">2683</span><span class="lpage" tagx="lpage" title="lpage">2691</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/aem.67.6.2683-2691.2001">10.1128/aem.67.6.2683-2691.2001</a></span><span class="pub-id" /><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11375181">11375181</a></span></span></li> <li tag="ref"><a name="B45" /><span class="label" tagx="label" title="label">45</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ruley</span><span class="given-names" tagx="given-names" title="given-names">J. A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Amoding</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tumuhairwe</span><span class="given-names" tagx="given-names" title="given-names">J. B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Basamba</span><span class="given-names" tagx="given-names" title="given-names">T. A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Opolot</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oryem-Origa</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Enhancing the phytoremediation of hydrocarbon-contaminated soils in the Sudd wetlands, South Sudan, using organic manure</span><span class="source" tagx="source" title="source"><i>Applied and Environmental Soil Science</i></span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">2020</span><span class="fpage" tagx="fpage" title="fpage">8</span><span class="pub-id" /><span class="pub-id"><a href="https://dx.doi.org/10.1155/2020/4614286">10.1155/2020/4614286</a></span></span></li> <li tag="ref"><a name="B46" /><span class="label" tagx="label" title="label">46</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ruley</span><span class="given-names" tagx="given-names" title="given-names">J. A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Amoding</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tumuhairwe</span><span class="given-names" tagx="given-names" title="given-names">J. B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Basamba</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oryem-Origa</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Hydrocarbons in crude-oil production sites of Sudd-region South-Sudan: implication on soil fertility and plant species risk</span><span class="source" tagx="source" title="source"><i>International Journal of Current Research</i></span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">9</span><span class="issue" tagx="issue" title="issue">12</span><span class="fpage" tagx="fpage" title="fpage">63070</span><span class="lpage" tagx="lpage" title="lpage">63075</span></span></li> <li tag="ref"><a name="B47" /><span class="label" tagx="label" title="label">47</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ruley</span><span class="given-names" tagx="given-names" title="given-names">J. A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tumuhairwe</span><span class="given-names" tagx="given-names" title="given-names">J. B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Amoding</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Opolot</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oryem-Origa</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Basamba</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Assessment of plants for phytoremediation of hydrocarbon-contaminated soils in the Sudd Wetland of South Sudan</span><span class="source" tagx="source" title="source"><i>Plant, Soil and Environment</i></span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">65</span><span class="issue" tagx="issue" title="issue">9</span><span class="fpage" tagx="fpage" title="fpage">463</span><span class="lpage" tagx="lpage" title="lpage">469</span><span class="pub-id"><a href="https://dx.doi.org/10.17221/322/2019-pse">10.17221/322/2019-pse</a></span><span class="pub-id" /></span></li> <li tag="ref"><a name="B48" /><span class="label" tagx="label" title="label">48</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Callahan</span><span class="given-names" tagx="given-names" title="given-names">B. J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McMurdie</span><span class="given-names" tagx="given-names" title="given-names">P. J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rosen</span><span class="given-names" tagx="given-names" title="given-names">M. J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Han</span><span class="given-names" tagx="given-names" title="given-names">A. W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Johnson</span><span class="given-names" tagx="given-names" title="given-names">A. J. A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holmes</span><span class="given-names" tagx="given-names" title="given-names">S. P.</span></span></span><span class="mixed-article-title" title="mixed-article-title">DADA2: high-resolution sample inference from Illumina amplicon data</span><span class="source" tagx="source" title="source"><i>Nature Methods</i></span><span class="year" tagx="year" title="year">2016a</span><span class="volume" tagx="volume" title="volume">13</span><span class="issue" tagx="issue" title="issue">7</span><span class="fpage" tagx="fpage" title="fpage">581</span><span class="lpage" tagx="lpage" title="lpage">583</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nmeth.3869">10.1038/nmeth.3869</a></span><span class="pub-id" /><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27214047">27214047</a></span></span></li> <li tag="ref"><a name="B49" /><span class="label" tagx="label" title="label">49</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Callahan</span><span class="given-names" tagx="given-names" title="given-names">B. J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sankaran</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fukuyama</span><span class="given-names" tagx="given-names" title="given-names">J. A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McMurdie</span><span class="given-names" tagx="given-names" title="given-names">P. J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holmes</span><span class="given-names" tagx="given-names" title="given-names">S. P.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Bioconductor workflow for microbiome data analysis: from raw reads to community analyses</span><span class="source" tagx="source" title="source"><i>F1000Research</i></span><span class="year" tagx="year" title="year">2016b</span><span class="volume" tagx="volume" title="volume">5</span><span class="fpage" tagx="fpage" title="fpage">p. 1492</span><span class="pub-id"><a href="https://dx.doi.org/10.12688/f1000research.8986.1">10.12688/f1000research.8986.1</a></span><span class="pub-id" /></span></li> <li tag="ref"><a name="B50" /><span class="label" tagx="label" title="label">50</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Callahan</span><span class="given-names" tagx="given-names" title="given-names">B. J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Silva taxonomic training data formatted for dada2 (silva version 128)</span><span class="year" tagx="year" title="year">2017</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Oakville, Canada</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">OpenAire Inc</span><span class="comment">Ph. D thesis</span></span></li> <li tag="ref"><a name="B51" /><span class="label" tagx="label" title="label">51</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Q.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Garrity</span><span class="given-names" tagx="given-names" title="given-names">G. M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tiedje</span><span class="given-names" tagx="given-names" title="given-names">J. M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cole</span><span class="given-names" tagx="given-names" title="given-names">J. R.</span></span></span><span class="mixed-article-title" title="mixed-article-title">NaÃ¯ve bayesian classifier for rapid assignment of rRNA sequences into the new bacterial taxonomy</span><span class="source" tagx="source" title="source"><i>Applied and Environmental Microbiology</i></span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">73</span><span class="issue" tagx="issue" title="issue">16</span><span class="fpage" tagx="fpage" title="fpage">5261</span><span class="lpage" tagx="lpage" title="lpage">5267</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/aem.00062-07">10.1128/aem.00062-07</a></span><span class="pub-id" /><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17586664">17586664</a></span></span></li> <li tag="ref"><a name="B52" /><span class="label" tagx="label" title="label">52</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chikere</span><span class="given-names" tagx="given-names" title="given-names">C. B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Okpokwasili</span><span class="given-names" tagx="given-names" title="given-names">G. C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chikere</span><span class="given-names" tagx="given-names" title="given-names">B. O.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Bacterial diversity in a tropical crude oil-polluted soil undergoing bioremediation</span><span class="source" tagx="source" title="source"><i>African Journal of Biotechnology</i></span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">8</span><span class="issue" tagx="issue" title="issue">11</span></span></li> <li tag="ref"><a name="B53" /><span class="label" tagx="label" title="label">53</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sutton</span><span class="given-names" tagx="given-names" title="given-names">N. B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maphosa</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morillo</span><span class="given-names" tagx="given-names" title="given-names">J. A.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Impact of long-term diesel contamination on soil microbial community structure</span><span class="source" tagx="source" title="source"><i>Applied and Environmental Microbiology</i></span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">79</span><span class="issue" tagx="issue" title="issue">2</span><span class="fpage" tagx="fpage" title="fpage">619</span><span class="lpage" tagx="lpage" title="lpage">630</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/aem.02747-12">10.1128/aem.02747-12</a></span><span class="pub-id" /><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23144139">23144139</a></span></span></li> <li tag="ref"><a name="B54" /><span class="label" tagx="label" title="label">54</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guo</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Han</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Effects of long-term fertilization on soil organic carbon mineralization and microbial community structure</span><span class="source" tagx="source" title="source"><i>PLoS One</i></span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">14</span><span class="issue" tagx="issue" title="issue">1</span><span class="pub-id" /><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0216006">10.1371/journal.pone.0216006</a></span><span class="pub-id" /></span></li> <li tag="ref"><a name="B55" /><span class="label" tagx="label" title="label">55</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ikuesan</span><span class="given-names" tagx="given-names" title="given-names">F. A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boboye</span><span class="given-names" tagx="given-names" title="given-names">B. E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Adetuyi</span><span class="given-names" tagx="given-names" title="given-names">F. C.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Comparative bioremediation of crude oil-contaminated soil samples using activated soil and activated cow dung</span><span class="source" tagx="source" title="source"><i>Sky Journal of Microbiology Research</i></span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">4</span><span class="issue" tagx="issue" title="issue">4</span><span class="fpage" tagx="fpage" title="fpage">21</span><span class="lpage" tagx="lpage" title="lpage">30</span></span></li> <li tag="ref"><a name="B56" /><span class="label" tagx="label" title="label">56</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Olubunmi</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Olatunde</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abiodun</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Helen</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Bacterial degradation of contaminated soil using organic manure</span><span class="source" tagx="source" title="source"><i>International Journal of Environment, Agriculture and Biotechnology</i></span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">4</span><span class="issue" tagx="issue" title="issue">5</span><span class="pub-id"><a href="https://dx.doi.org/10.22161/ijeab.45.52">10.22161/ijeab.45.52</a></span></span></li> <li tag="ref"><a name="B57" /><span class="label" tagx="label" title="label">57</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bhattacharyya</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bakshi</span><span class="given-names" tagx="given-names" title="given-names">U.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mallick</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mukherji</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bera</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ghosh</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Genome-guided insights into plant growth promotion capabilities of physiologically versatile Bacillus aryabhattai strain AB211</span><span class="source" tagx="source" title="source"><i>Frontiers in Microbiology</i></span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">p. 411</span><span class="pub-id"><a href="https://dx.doi.org/10.3389/fmicb.2017.00411">10.3389/fmicb.2017.00411</a></span><span class="pub-id" /></span></li> <li tag="ref"><a name="B58" /><span class="label" tagx="label" title="label">58</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nwinyi</span><span class="given-names" tagx="given-names" title="given-names">O. C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kanu</span><span class="given-names" tagx="given-names" title="given-names">I. A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tunde</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ajanaku</span><span class="given-names" tagx="given-names" title="given-names">K. O.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Characterization of diesel degrading bacterial species from contaminated tropical ecosystem</span><span class="source" tagx="source" title="source"><i>Brazilian Archives of Biology and Technology</i></span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">57</span><span class="issue" tagx="issue" title="issue">5</span><span class="fpage" tagx="fpage" title="fpage">789</span><span class="lpage" tagx="lpage" title="lpage">796</span><span class="pub-id"><a href="https://dx.doi.org/10.1590/s1516-8913201402250">10.1590/s1516-8913201402250</a></span><span class="pub-id" /></span></li> <li tag="ref"><a name="B59" /><span class="label" tagx="label" title="label">59</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kudanga</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Burton</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nyanhongo</span><span class="given-names" tagx="given-names" title="given-names">G. S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guebitz</span><span class="given-names" tagx="given-names" title="given-names">G. M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Versatility of oxidoreductases in the remediation of environmental pollutants</span><span class="source" tagx="source" title="source"><i>Frontiers in Bioscience</i></span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">E4</span><span class="issue" tagx="issue" title="issue">3</span><span class="fpage" tagx="fpage" title="fpage">1127</span><span class="lpage" tagx="lpage" title="lpage">1149</span><span class="pub-id"><a href="https://dx.doi.org/10.2741/e446">10.2741/e446</a></span></span></li> <li tag="ref"><a name="B60" /><span class="label" tagx="label" title="label">60</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Van Hellemond</span><span class="given-names" tagx="given-names" title="given-names">E. W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Janssen</span><span class="given-names" tagx="given-names" title="given-names">D. B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fraaije</span><span class="given-names" tagx="given-names" title="given-names">M. W.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Discovery of a novel styrene monooxygenase originating from the metagenome</span><span class="source" tagx="source" title="source"><i>Applied and Environmental Microbiology</i></span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">73</span><span class="issue" tagx="issue" title="issue">18</span><span class="fpage" tagx="fpage" title="fpage">5832</span><span class="lpage" tagx="lpage" title="lpage">5839</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/aem.02708-06">10.1128/aem.02708-06</a></span><span class="pub-id" /><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17644649">17644649</a></span></span></li> <li tag="ref"><a name="B61" /><span class="label" tagx="label" title="label">61</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ausec</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Van Elsas</span><span class="given-names" tagx="given-names" title="given-names">J. D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mandic-Mulec</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Two- and three-domain bacterial laccase-like genes are present in drained peat soils</span><span class="source" tagx="source" title="source"><i>Soil Biology and Biochemistry</i></span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">43</span><span class="issue" tagx="issue" title="issue">5</span><span class="fpage" tagx="fpage" title="fpage">975</span><span class="lpage" tagx="lpage" title="lpage">983</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.soilbio.2011.01.013">10.1016/j.soilbio.2011.01.013</a></span><span class="pub-id" /></span></li> <li tag="ref"><a name="B62" /><span class="label" tagx="label" title="label">62</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rivera-Hoyos</span><span class="given-names" tagx="given-names" title="given-names">C. M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morales-Ã�lvarez</span><span class="given-names" tagx="given-names" title="given-names">E. D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Poutou-PiÃ±ales</span><span class="given-names" tagx="given-names" title="given-names">R. A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pedroza-RodrÃ­guez</span><span class="given-names" tagx="given-names" title="given-names">A. M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">RodrÃ�guez-VÃ¡zquez</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Delgado-Boada</span><span class="given-names" tagx="given-names" title="given-names">J. M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Fungal laccases</span><span class="source" tagx="source" title="source"><i>Fungal Biology Reviews</i></span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">27</span><span class="issue" tagx="issue" title="issue">3-4</span><span class="fpage" tagx="fpage" title="fpage">67</span><span class="lpage" tagx="lpage" title="lpage">82</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.fbr.2013.07.001">10.1016/j.fbr.2013.07.001</a></span><span class="pub-id" /></span></li> <li tag="ref"><a name="B63" /><span class="label" tagx="label" title="label">63</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chauhan</span><span class="given-names" tagx="given-names" title="given-names">P. S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goradia</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saxena</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Bacterial laccase: recent update on production, properties and industrial applications</span><span class="source" tagx="source" title="source"><i>3 Biotech</i></span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">7</span><span class="issue" tagx="issue" title="issue">5</span><span class="fpage" tagx="fpage" title="fpage">p. 323</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s13205-017-0955-7">10.1007/s13205-017-0955-7</a></span><span class="pub-id" /></span></li> <li tag="ref"><a name="B64" /><span class="label" tagx="label" title="label">64</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gedikli</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aytar</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ãœnal</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">YamaÃ§</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ã‡abuk</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kolankaya</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Enhancement with inducers of lacasse production by some strains and application of enzyme to dechlorination of 2, 4, 5-trichlorophenol</span><span class="source" tagx="source" title="source"><i>Electronic Journal of Biotechnology</i></span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">13</span><span class="issue" tagx="issue" title="issue">6</span><span class="fpage" tagx="fpage" title="fpage">6</span><span class="lpage" tagx="lpage" title="lpage">7</span><span class="pub-id"><a href="https://dx.doi.org/10.2225/vol13-issue6-fulltext-9">10.2225/vol13-issue6-fulltext-9</a></span><span class="pub-id" /></span></li> <li tag="ref"><a name="B65" /><span class="label" tagx="label" title="label">65</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Praeg</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pauli</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Illmer</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Microbial diversity in bulk and rhizosphere soil of Ranunculus glacialis along a high-alpine altitudinal gradient</span><span class="source" tagx="source" title="source"><i>Frontiers in Microbiology</i></span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">p. 1429</span><span class="pub-id"><a href="https://dx.doi.org/10.3389/fmicb.2019.01429">10.3389/fmicb.2019.01429</a></span><span class="pub-id" /></span></li> <li tag="ref"><a name="B66" /><span class="label" tagx="label" title="label">66</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peng</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zi</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Q.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Bacterial community diversity of oil-contaminated soils assessed by high throughput sequencing of 16S rRNA genes</span><span class="source" tagx="source" title="source"><i>International Journal of Environmental Research and Public Health</i></span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">12</span><span class="issue" tagx="issue" title="issue">10</span><span class="fpage" tagx="fpage" title="fpage">12002</span><span class="lpage" tagx="lpage" title="lpage">12015</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/ijerph121012002">10.3390/ijerph121012002</a></span><span class="pub-id" /><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26404329">26404329</a></span></span></li> </ul> </div> </div> <div class="floats-group" title="floats-group"> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1</span></a></div>   <p>Rarefaction curve showing sampling depths across the five plant species with and without compost and petroleum contamination (T1â€‰=â€‰plant species only, T2â€‰=â€‰plant species and hydrocarbon, T3â€‰=â€‰plant species and Manure, and T4â€‰=â€‰plant species, manure, and hydrocarbon).</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2</span></a></div>   <p>Composition and distribution of bacterial phyla with â‰¥3% relative abundance across five plant spp. with and without compost and petroleum contamination (T1â€‰=â€‰plant species only, T2â€‰=â€‰plant species and hydrocarbon, T3â€‰=â€‰plant species and manure, and T4â€‰=â€‰plant species, manure, and hydrocarbon).</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 3</span></a></div>   <p>Variation in bacterial genera within rhizosphere of five phytoremediation species under four different treatments (T1â€‰=â€‰plant species only, T2â€‰=â€‰plant species and hydrocarbon, T3â€‰=â€‰plant species and manure, and T4â€‰=â€‰plant species, manure, and hydrocarbon).</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 4</span></a></div>   <p>Nonmultidimensional scaling (NMDS) showing ASVs between hydrocarbon (T2, T4) and nonhydrocarbon (T1, T3) treatments. T1â€‰=â€‰plant species only (<i>G. barbadense, H rufa T diversifolia, O longistaminata.</i> and <i>S. arundinaceum</i>), T2â€‰=â€‰Plant species and hydrocarbon (0, 75â€‰gÂ·kg<sup>âˆ’1</sup>), T3â€‰=â€‰Plant species and manure (0, 2 tha<sup>âˆ’1</sup>), and T4â€‰=â€‰Plant species, manure, hydrocarbon.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 5</span></a></div>   <p>Principal component analysis of bacterial communities across different phytoremediation species growing under different treatments (T1, plant species only; T2, plant species and hydrocarbon; T3, plant species and manure; T4, plant species, manure, and hydrocarbon).</p>   </div> <div class="table-wrap_UNKNOWN" id="tab1" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 1</span> <p>PERMANOVA analysis of interaction effects of plant species, organic manure, and hydrocarbons on bacterial community based on Brayâ€"Curtis dissimilarity.</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="left" rowspan="1" colspan="1">Factor</th> <th align="center" rowspan="1" colspan="1">Df</th> <th align="center" rowspan="1" colspan="1">SS</th> <th align="center" rowspan="1" colspan="1">MS</th> <th align="center" rowspan="1" colspan="1">F. Model</th> <th align="center" rowspan="1" colspan="1"><i>R</i><sup>2</sup></th> <th align="center" rowspan="1" colspan="1">Pr (&amp;gt;F)</th> </tr> </thead> <tbody> <tr> <td align="left" rowspan="1" colspan="1">Plant species</td> <td align="center" rowspan="1" colspan="1">4</td> <td align="center" rowspan="1" colspan="1">0.5899</td> <td align="center" rowspan="1" colspan="1">0.5987</td> <td align="center" rowspan="1" colspan="1">3.9978</td> <td align="center" rowspan="1" colspan="1">0.02941</td> <td align="center" rowspan="1" colspan="1">0.081</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Plant species: manure</td> <td align="center" rowspan="1" colspan="1">1</td> <td align="center" rowspan="1" colspan="1">0.9233</td> <td align="center" rowspan="1" colspan="1">0.32611</td> <td align="center" rowspan="1" colspan="1">3.0066</td> <td align="center" rowspan="1" colspan="1">0.05130</td> <td align="center" rowspan="1" colspan="1">0.009<sup><i>âˆ—</i></sup></td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Plant species: hydrocarbon</td> <td align="center" rowspan="1" colspan="1">1</td> <td align="center" rowspan="1" colspan="1">0.6157</td> <td align="center" rowspan="1" colspan="1">0.18444</td> <td align="center" rowspan="1" colspan="1">1.5462</td> <td align="center" rowspan="1" colspan="1">0.09255</td> <td align="center" rowspan="1" colspan="1">0.001<sup><i>âˆ—</i></sup></td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Interaction</td> <td align="center" rowspan="1" colspan="1">4</td> <td align="center" rowspan="1" colspan="1">0.9845</td> <td align="center" rowspan="1" colspan="1">0.3217</td> <td align="center" rowspan="1" colspan="1">2.2381</td> <td align="center" rowspan="1" colspan="1">0.13421</td> <td align="center" rowspan="1" colspan="1">0.002<sup><i>âˆ—</i></sup></td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Residuals</td> <td align="center" rowspan="1" colspan="1">95</td> <td align="center" rowspan="1" colspan="1">13.1882</td> <td align="center" rowspan="1" colspan="1">0.13977</td> <td align="center" rowspan="1" colspan="1">â€"</td> <td align="center" rowspan="1" colspan="1">0.80211</td> <td align="center" rowspan="1" colspan="1">â€"</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Total</td> <td align="center" rowspan="1" colspan="1">105</td> <td align="center" rowspan="1" colspan="1">15.3171</td> <td align="center" rowspan="1" colspan="1">â€"</td> <td align="center" rowspan="1" colspan="1">â€"</td> <td align="center" rowspan="1" colspan="1">1</td> <td align="center" rowspan="1" colspan="1">â€"</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p>Dfâ€‰=â€‰degrees of freedom; SSâ€‰=â€‰sum of squares; MSâ€‰=â€‰mean squares; F. Modelâ€‰=â€‰F-test value for model; <i>R</i><sup>2</sup>â€‰=â€‰R-squared; Pr (&amp;gt;F)â€‰=â€‰<i>p</i> value.</p> </div> </div> </div> <div class="table-wrap_UNKNOWN" id="tab2" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 2</span> <p>Rhizobacteria community abundance and diversity in phytoremediation plant species biostimulated with cattle manure in hydrocarbon contaminated soils.</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="center" colspan="3" rowspan="1">Treatments</th> <th align="center" colspan="6" rowspan="1">Parameters</th> </tr> <tr> <th align="left" rowspan="1" colspan="1">Plant species</th> <th align="center" rowspan="1" colspan="1">Compost (t/ha)</th> <th align="center" rowspan="1" colspan="1">TPH (mg kg-1 soil)</th> <th align="center" rowspan="1" colspan="1">ASVs</th> <th align="center" rowspan="1" colspan="1">Number of phyla</th> <th align="center" rowspan="1" colspan="1">Number of genera</th> <th align="center" rowspan="1" colspan="1">Abundance chao 1</th> <th align="center" rowspan="1" colspan="1">Shannon index</th> <th align="center" rowspan="1" colspan="1">Simpson's index</th> </tr> </thead> <tbody> <tr> <td align="left" rowspan="4" colspan="1"><i>Gossypium barbadense</i></td> <td rowspan="2" align="center" colspan="1"><b>0</b></td> <td align="center" rowspan="1" colspan="1"><b>0</b></td> <td align="center" rowspan="1" colspan="1">2625</td> <td align="center" rowspan="1" colspan="1">40</td> <td align="center" rowspan="1" colspan="1">44</td> <td align="center" rowspan="1" colspan="1">5600</td> <td align="center" rowspan="1" colspan="1">4.93</td> <td align="center" rowspan="1" colspan="1">0.87</td> </tr> <tr> <td align="center" rowspan="1" colspan="1"><b>75</b></td> <td align="center" rowspan="1" colspan="1">1215</td> <td align="center" rowspan="1" colspan="1">24</td> <td align="center" rowspan="1" colspan="1">27</td> <td align="center" rowspan="1" colspan="1">2781</td> <td align="center" rowspan="1" colspan="1">4. 11</td> <td align="center" rowspan="1" colspan="1">0.97</td> </tr> <tr> <td rowspan="2" align="center" colspan="1"><b>2</b></td> <td align="center" rowspan="1" colspan="1"><b>0</b></td> <td align="center" rowspan="1" colspan="1">6935</td> <td align="center" rowspan="1" colspan="1">101</td> <td align="center" rowspan="1" colspan="1">81</td> <td align="center" rowspan="1" colspan="1">14226</td> <td align="center" rowspan="1" colspan="1">5.97</td> <td align="center" rowspan="1" colspan="1">0.76</td> </tr> <tr> <td align="center" rowspan="1" colspan="1"><b>75</b></td> <td align="center" rowspan="1" colspan="1">4595</td> <td align="center" rowspan="1" colspan="1">68.</td> <td align="center" rowspan="1" colspan="1">54</td> <td align="center" rowspan="1" colspan="1">9540</td> <td align="center" rowspan="1" colspan="1">5.09</td> <td align="center" rowspan="1" colspan="1">0.84</td> </tr> <tr> <td align="left" colspan="9" rowspan="1"> <hr /> </td> </tr> <tr> <td align="left" rowspan="4" colspan="1"><i>Tithonia diversifolia</i></td> <td rowspan="2" align="center" colspan="1"><b>0</b></td> <td align="center" rowspan="1" colspan="1"><b>0</b></td> <td align="center" rowspan="1" colspan="1">3144</td> <td align="center" rowspan="1" colspan="1">34</td> <td align="center" rowspan="1" colspan="1">57</td> <td align="center" rowspan="1" colspan="1">6638</td> <td align="center" rowspan="1" colspan="1">5.14</td> <td align="center" rowspan="1" colspan="1">0.82</td> </tr> <tr> <td align="center" rowspan="1" colspan="1"><b>75</b></td> <td align="center" rowspan="1" colspan="1">1936</td> <td align="center" rowspan="1" colspan="1">21</td> <td align="center" rowspan="1" colspan="1">34</td> <td align="center" rowspan="1" colspan="1">4223</td> <td align="center" rowspan="1" colspan="1">4.30</td> <td align="center" rowspan="1" colspan="1">0.94</td> </tr> <tr> <td rowspan="2" align="center" colspan="1"><b>2</b></td> <td align="center" rowspan="1" colspan="1"><b>0</b></td> <td align="center" rowspan="1" colspan="1">7770</td> <td align="center" rowspan="1" colspan="1">85</td> <td align="center" rowspan="1" colspan="1">99</td> <td align="center" rowspan="1" colspan="1">15891</td> <td align="center" rowspan="1" colspan="1">6.25</td> <td align="center" rowspan="1" colspan="1">0.73</td> </tr> <tr> <td align="center" rowspan="1" colspan="1"><b>75</b></td> <td align="center" rowspan="1" colspan="1">4723</td> <td align="center" rowspan="1" colspan="1">60</td> <td align="center" rowspan="1" colspan="1">63</td> <td align="center" rowspan="1" colspan="1">9795</td> <td align="center" rowspan="1" colspan="1">5.24</td> <td align="center" rowspan="1" colspan="1">0.81</td> </tr> <tr> <td align="left" colspan="9" rowspan="1"> <hr /> </td> </tr> <tr> <td align="left" rowspan="4" colspan="1"><i>Oryza longistaminata</i></td> <td rowspan="2" align="center" colspan="1"><b>0</b></td> <td align="center" rowspan="1" colspan="1"><b>0</b></td> <td align="center" rowspan="1" colspan="1">2764</td> <td align="center" rowspan="1" colspan="1">49</td> <td align="center" rowspan="1" colspan="1">66</td> <td align="center" rowspan="1" colspan="1">5878</td> <td align="center" rowspan="1" colspan="1">5.09</td> <td align="center" rowspan="1" colspan="1">0.83</td> </tr> <tr> <td align="center" rowspan="1" colspan="1"><b>75</b></td> <td align="center" rowspan="1" colspan="1">1623</td> <td align="center" rowspan="1" colspan="1">30</td> <td align="center" rowspan="1" colspan="1">43</td> <td align="center" rowspan="1" colspan="1">3597</td> <td align="center" rowspan="1" colspan="1">4.16</td> <td align="center" rowspan="1" colspan="1">0.96</td> </tr> <tr> <td rowspan="2" align="center" colspan="1"><b>2</b></td> <td align="center" rowspan="1" colspan="1"><b>0</b></td> <td align="center" rowspan="1" colspan="1">6289</td> <td align="center" rowspan="1" colspan="1">99</td> <td align="center" rowspan="1" colspan="1">80</td> <td align="center" rowspan="1" colspan="1">12944</td> <td align="center" rowspan="1" colspan="1">6.14</td> <td align="center" rowspan="1" colspan="1">0.74</td> </tr> <tr> <td align="center" rowspan="1" colspan="1"><b>75</b></td> <td align="center" rowspan="1" colspan="1">4408</td> <td align="center" rowspan="1" colspan="1">82</td> <td align="center" rowspan="1" colspan="1">58</td> <td align="center" rowspan="1" colspan="1">9168</td> <td align="center" rowspan="1" colspan="1">5.16</td> <td align="center" rowspan="1" colspan="1">0.82</td> </tr> <tr> <td align="left" colspan="9" rowspan="1"> <hr /> </td> </tr> <tr> <td align="left" rowspan="4" colspan="1"><i>Sorghum arundinaceum</i></td> <td rowspan="2" align="center" colspan="1"><b>0</b></td> <td align="center" rowspan="1" colspan="1"><b>0</b></td> <td align="center" rowspan="1" colspan="1">2198</td> <td align="center" rowspan="1" colspan="1">37</td> <td align="center" rowspan="1" colspan="1">37</td> <td align="center" rowspan="1" colspan="1">4745</td> <td align="center" rowspan="1" colspan="1">4.86</td> <td align="center" rowspan="1" colspan="1">0.88</td> </tr> <tr> <td align="center" rowspan="1" colspan="1"><b>75</b></td> <td align="center" rowspan="1" colspan="1">871</td> <td align="center" rowspan="1" colspan="1">23</td> <td align="center" rowspan="1" colspan="1">23</td> <td align="center" rowspan="1" colspan="1">2097</td> <td align="center" rowspan="1" colspan="1">3.98</td> <td align="center" rowspan="1" colspan="1">0.98</td> </tr> <tr> <td rowspan="2" align="center" colspan="1"><b>2</b></td> <td align="center" rowspan="1" colspan="1"><b>0</b></td> <td align="center" rowspan="1" colspan="1">5371</td> <td align="center" rowspan="1" colspan="1">69</td> <td align="center" rowspan="1" colspan="1">65</td> <td align="center" rowspan="1" colspan="1">11095</td> <td align="center" rowspan="1" colspan="1">5.87</td> <td align="center" rowspan="1" colspan="1">0.78</td> </tr> <tr> <td align="center" rowspan="1" colspan="1"><b>75</b></td> <td align="center" rowspan="1" colspan="1">3955</td> <td align="center" rowspan="1" colspan="1">56</td> <td align="center" rowspan="1" colspan="1">43</td> <td align="center" rowspan="1" colspan="1">8260</td> <td align="center" rowspan="1" colspan="1">4.84</td> <td align="center" rowspan="1" colspan="1">0.89</td> </tr> <tr> <td align="left" colspan="9" rowspan="1"> <hr /> </td> </tr> <tr> <td align="left" rowspan="4" colspan="1"><i>Hyparrhenia. rufa</i></td> <td rowspan="2" align="center" colspan="1"><b>0</b></td> <td align="center" rowspan="1" colspan="1"><b>0</b></td> <td align="center" rowspan="1" colspan="1">3583</td> <td align="center" rowspan="1" colspan="1">79</td> <td align="center" rowspan="1" colspan="1">86</td> <td align="center" rowspan="1" colspan="1">7518</td> <td align="center" rowspan="1" colspan="1">5.37</td> <td align="center" rowspan="1" colspan="1">0.8</td> </tr> <tr> <td align="center" rowspan="1" colspan="1"><b>75</b></td> <td align="center" rowspan="1" colspan="1">1979</td> <td align="center" rowspan="1" colspan="1">40</td> <td align="center" rowspan="1" colspan="1">47</td> <td align="center" rowspan="1" colspan="1">4304</td> <td align="center" rowspan="1" colspan="1">4.54</td> <td align="center" rowspan="1" colspan="1">0.92</td> </tr> <tr> <td rowspan="2" align="center" colspan="1"><b>2</b></td> <td align="center" rowspan="1" colspan="1"><b>0</b></td> <td align="center" rowspan="1" colspan="1">9675</td> <td align="center" rowspan="1" colspan="1">172</td> <td align="center" rowspan="1" colspan="1">164</td> <td align="center" rowspan="1" colspan="1">19701</td> <td align="center" rowspan="1" colspan="1">6.55</td> <td align="center" rowspan="1" colspan="1">2.89</td> </tr> <tr> <td align="center" rowspan="1" colspan="1"><b>75</b></td> <td align="center" rowspan="1" colspan="1">4980</td> <td align="center" rowspan="1" colspan="1">140</td> <td align="center" rowspan="1" colspan="1">136</td> <td align="center" rowspan="1" colspan="1">10310</td> <td align="center" rowspan="1" colspan="1">5.64</td> <td align="center" rowspan="1" colspan="1">0.75</td> </tr> <tr> <td align="left" colspan="9" rowspan="1"> <hr /> </td> </tr> <tr> <td align="left" colspan="3" rowspan="1"><b>LSD (0.05) (plant</b><i>âˆ— </i><b>TPH</b><i>âˆ— </i><b>compost)</b></td> <td align="center" rowspan="1" colspan="1">181.1<sup><i>âˆ—âˆ—âˆ—</i></sup></td> <td align="center" rowspan="1" colspan="1"><b>5.2</b><sup><i>âˆ—âˆ—</i></sup></td> <td align="center" rowspan="1" colspan="1"><b>2.1</b><sup><i>âˆ—âˆ—âˆ—</i></sup></td> <td align="center" rowspan="1" colspan="1"><b>365</b><sup><i>âˆ—âˆ—âˆ—</i></sup></td> <td align="center" rowspan="1" colspan="1">0.06<sup><i>âˆ—</i></sup></td> <td align="center" rowspan="1" colspan="1">ns</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p><sup><i>âˆ—âˆ—âˆ—</i></sup>0.000, <sup><i>âˆ—âˆ—</i></sup>0.001, and <sup><i>âˆ—</i></sup>0.05.</p> </div> </div> </div> </div>  </body></html>