{
  "nbformat": 4,
  "nbformat_minor": 0,
  "metadata": {
    "colab": {
      "name": "fun_with_pandas.ipynb",
      "provenance": [],
      "authorship_tag": "ABX9TyOqQOxUNus6r5hrfYARx4pE",
      "include_colab_link": true
    },
    "kernelspec": {
      "name": "python3",
      "display_name": "Python 3"
    },
    "language_info": {
      "name": "python"
    }
  },
  "cells": [
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "view-in-github",
        "colab_type": "text"
      },
      "source": [
        "<a href=\"https://colab.research.google.com/github/petermr/CEVOpen/blob/master/fun_with_pandas.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "#Pandas DataFrame Quick Tutorial\n",
        "This Colab introduces DataFrames, which are the central data structure in the pandas API. This Colab is semi comprehensive DataFrames tutorial. \n",
        "\n",
        "A DataFrame is similar to an in-memory spreadsheet. Like a spreadsheet:\n",
        "\n",
        "A DataFrame stores data in cells.\n",
        "A DataFrame has named columns (usually) and numbered rows."
      ],
      "metadata": {
        "id": "BQMFG4yw0Ga4"
      }
    },
    {
      "cell_type": "markdown",
      "source": [
        "#Installing and Loading pandas\n",
        "\n",
        "If you are new to pandas take a moment read this link [pandas](https://https://pandas.pydata.org/docs/getting_started/install.html) on how to install pandas in your respective enviornmnet.\n",
        "For those who already have python installed in there enviornment please run the code below:"
      ],
      "metadata": {
        "id": "GnDw8i4yTLUy"
      }
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "Quo9F7Ywa4qI",
        "outputId": "d3c628a8-602d-49b1-ae65-bb6e67dac856"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Requirement already satisfied: pandas in /usr/local/lib/python3.7/dist-packages (1.1.5)\n",
            "Requirement already satisfied: pytz>=2017.2 in /usr/local/lib/python3.7/dist-packages (from pandas) (2018.9)\n",
            "Requirement already satisfied: python-dateutil>=2.7.3 in /usr/local/lib/python3.7/dist-packages (from pandas) (2.8.2)\n",
            "Requirement already satisfied: numpy>=1.15.4 in /usr/local/lib/python3.7/dist-packages (from pandas) (1.19.5)\n",
            "Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.7/dist-packages (from python-dateutil>=2.7.3->pandas) (1.15.0)\n"
          ]
        }
      ],
      "source": [
        "!pip install pandas                    # to install the pandas packages in your enviornment do\n",
        "                                       #This line of code will install all the releveent packages along will pandas for sucessful operation.\n",
        "                                       # And this line of code is meant to be run only once. AS it will download all the packages in your system\n",
        "\n"
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "import pandas as pd               #This will import the pandas into you active session. \n",
        "                                  #You need to do this every time you are running a script. So that pandas is called and loaded."
      ],
      "metadata": {
        "id": "L8mXC8rRywaY"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "source": [
        "To get the data run the cell below:"
      ],
      "metadata": {
        "id": "BWraWfNdfSy0"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "import gdown\n",
        "#https://drive.google.com/file/d/11_R7D-EdOm6fgJy0lPuz2YQDVq2z3dyT/view?usp=sharing\n",
        "url = 'https://drive.google.com/uc?id=11_R7D-EdOm6fgJy0lPuz2YQDVq2z3dyT'\n",
        "output = 'files.zip'\n",
        "gdown.download(url, output, quiet=False)\n",
        "!unzip /content/files.zip"
      ],
      "metadata": {
        "id": "ADk4dKqRQhSd",
        "outputId": "54bdc7f7-59a5-4020-98a1-1a8fa102cf0d",
        "colab": {
          "base_uri": "https://localhost:8080/"
        }
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "Downloading...\n",
            "From: https://drive.google.com/uc?id=11_R7D-EdOm6fgJy0lPuz2YQDVq2z3dyT\n",
            "To: /content/files.zip\n",
            "100%|██████████| 2.29k/2.29k [00:00<00:00, 4.37MB/s]"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Archive:  /content/files.zip\n",
            "  inflating: terpene files/Terpene and Other Plant Pathway Figures (3).csv  \n",
            "  inflating: terpene files/Terpene and Other Plant Pathway Figures.csv  \n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "# Reading csv files\n",
        "\n",
        "Through pandas you can read several types of file like csv,xls,hsv etc. More about this can be found [here](https://) \n",
        "\n",
        "Our concerned file type is CSV therefore we have to do read_csv. More information about read_csv can be found [here](https://pandas.pydata.org/docs/reference/api/pandas.read_csv.html)\n",
        "There are lot of attributes for read_csv. Below I have mention few important once:\n",
        "Header: \n",
        "\n",
        "1.   **header**:  If set to None, there will be no header for your dataframe. DEfault is 'infer' which autometically detects the header from csv\n",
        "2.   **index** :  Similar to header if set to None it wont return any indexes of you dataframe.\n",
        "3.   **names**  : List of column names to use. If the file contains a header row, then you should explicitly pass header=0 to override the column names. Duplicates in this list are not allowed.\n",
        "\n",
        "\n"
      ],
      "metadata": {
        "id": "EHaymBxwTQGN"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "df = pd.read_csv('/content/terpene files/Terpene and Other Plant Pathway Figures.csv')          #to read any csv file do\n",
        "df\n"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 569
        },
        "id": "4P6K2EB90FrU",
        "outputId": "c9986e44-f0ed-45c1-8fe7-68cb65ad8065"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/html": [
              "\n",
              "  <div id=\"df-7a0e80de-1b2a-4d55-892d-d90a37ccf8e3\">\n",
              "    <div class=\"colab-df-container\">\n",
              "      <div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>pmcid</th>\n",
              "      <th>paper.title</th>\n",
              "      <th>authors</th>\n",
              "      <th>year</th>\n",
              "      <th>number</th>\n",
              "      <th>figure.title</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>0</th>\n",
              "      <td>PMC6323817</td>\n",
              "      <td>Transcriptomic de novo analysis of pitaya (Hyl...</td>\n",
              "      <td>Min Xu, Cheng-L...</td>\n",
              "      <td>2019</td>\n",
              "      <td>Fig. 6</td>\n",
              "      <td>Differentially expressed genes (DEG...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>PMC6348685</td>\n",
              "      <td>Comparative transcriptome analysis of a long-t...</td>\n",
              "      <td>Luodong Huang, ...</td>\n",
              "      <td>2019</td>\n",
              "      <td>Fig. 4</td>\n",
              "      <td>Putative genes and their expression...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2</th>\n",
              "      <td>PMC6356271</td>\n",
              "      <td>Telomeres in Plants and Humans: Not So Differe...</td>\n",
              "      <td>Petra Procházko...</td>\n",
              "      <td>2019</td>\n",
              "      <td>Figure 2</td>\n",
              "      <td>Terminal telomeric complex</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3</th>\n",
              "      <td>PMC6360084</td>\n",
              "      <td>Identification, expression, and phylogenetic a...</td>\n",
              "      <td>Jipeng Mao, Zid...</td>\n",
              "      <td>2019</td>\n",
              "      <td>Figure 8</td>\n",
              "      <td>Identification, expression, and phy...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4</th>\n",
              "      <td>PMC6374618</td>\n",
              "      <td>Agarwood Induction: Current Developments and F...</td>\n",
              "      <td>Cheng Seng CS T...</td>\n",
              "      <td>2019</td>\n",
              "      <td>FIGURE 4</td>\n",
              "      <td>Schematic relationships between the...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>5</th>\n",
              "      <td>PMC6394206</td>\n",
              "      <td>A chromosomal-scale genome assembly of Tectona...</td>\n",
              "      <td>Dongyan Zhao, J...</td>\n",
              "      <td>2019</td>\n",
              "      <td>Figure 5:</td>\n",
              "      <td>Proposed diterpene pathway based on...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>6</th>\n",
              "      <td>PMC6412517</td>\n",
              "      <td>New Insights on Arabidopsis thaliana Root Adap...</td>\n",
              "      <td>Inmaculada Cole...</td>\n",
              "      <td>2019</td>\n",
              "      <td>Figure 2</td>\n",
              "      <td>Induction of secondary metabolism i...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>7</th>\n",
              "      <td>PMC6423058</td>\n",
              "      <td>Why Algae Release Volatile Organic Compounds—T...</td>\n",
              "      <td>Zhaojiang Zuo</td>\n",
              "      <td>2019</td>\n",
              "      <td>FIGURE 2</td>\n",
              "      <td>Pathway of terpene synthesis</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>8</th>\n",
              "      <td>PMC6437448</td>\n",
              "      <td>Production of ginsenoside aglycone (protopanax...</td>\n",
              "      <td>Yu Shin YS Gwak...</td>\n",
              "      <td>2019</td>\n",
              "      <td>Fig. 1</td>\n",
              "      <td>Biosynthetic pathway engineering fo...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>9</th>\n",
              "      <td>PMC6441073</td>\n",
              "      <td>Transcriptional response of grapevine to infec...</td>\n",
              "      <td>Wei Zhang, Jiye...</td>\n",
              "      <td>2019</td>\n",
              "      <td>Figure 3</td>\n",
              "      <td>DEGs involved in the plant–pathogen...</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "</div>\n",
              "      <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-7a0e80de-1b2a-4d55-892d-d90a37ccf8e3')\"\n",
              "              title=\"Convert this dataframe to an interactive table.\"\n",
              "              style=\"display:none;\">\n",
              "        \n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "       width=\"24px\">\n",
              "    <path d=\"M0 0h24v24H0V0z\" fill=\"none\"/>\n",
              "    <path d=\"M18.56 5.44l.94 2.06.94-2.06 2.06-.94-2.06-.94-.94-2.06-.94 2.06-2.06.94zm-11 1L8.5 8.5l.94-2.06 2.06-.94-2.06-.94L8.5 2.5l-.94 2.06-2.06.94zm10 10l.94 2.06.94-2.06 2.06-.94-2.06-.94-.94-2.06-.94 2.06-2.06.94z\"/><path d=\"M17.41 7.96l-1.37-1.37c-.4-.4-.92-.59-1.43-.59-.52 0-1.04.2-1.43.59L10.3 9.45l-7.72 7.72c-.78.78-.78 2.05 0 2.83L4 21.41c.39.39.9.59 1.41.59.51 0 1.02-.2 1.41-.59l7.78-7.78 2.81-2.81c.8-.78.8-2.07 0-2.86zM5.41 20L4 18.59l7.72-7.72 1.47 1.35L5.41 20z\"/>\n",
              "  </svg>\n",
              "      </button>\n",
              "      \n",
              "  <style>\n",
              "    .colab-df-container {\n",
              "      display:flex;\n",
              "      flex-wrap:wrap;\n",
              "      gap: 12px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert {\n",
              "      background-color: #E8F0FE;\n",
              "      border: none;\n",
              "      border-radius: 50%;\n",
              "      cursor: pointer;\n",
              "      display: none;\n",
              "      fill: #1967D2;\n",
              "      height: 32px;\n",
              "      padding: 0 0 0 0;\n",
              "      width: 32px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert:hover {\n",
              "      background-color: #E2EBFA;\n",
              "      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "      fill: #174EA6;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert {\n",
              "      background-color: #3B4455;\n",
              "      fill: #D2E3FC;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert:hover {\n",
              "      background-color: #434B5C;\n",
              "      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "      fill: #FFFFFF;\n",
              "    }\n",
              "  </style>\n",
              "\n",
              "      <script>\n",
              "        const buttonEl =\n",
              "          document.querySelector('#df-7a0e80de-1b2a-4d55-892d-d90a37ccf8e3 button.colab-df-convert');\n",
              "        buttonEl.style.display =\n",
              "          google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "        async function convertToInteractive(key) {\n",
              "          const element = document.querySelector('#df-7a0e80de-1b2a-4d55-892d-d90a37ccf8e3');\n",
              "          const dataTable =\n",
              "            await google.colab.kernel.invokeFunction('convertToInteractive',\n",
              "                                                     [key], {});\n",
              "          if (!dataTable) return;\n",
              "\n",
              "          const docLinkHtml = 'Like what you see? Visit the ' +\n",
              "            '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
              "            + ' to learn more about interactive tables.';\n",
              "          element.innerHTML = '';\n",
              "          dataTable['output_type'] = 'display_data';\n",
              "          await google.colab.output.renderOutput(dataTable, element);\n",
              "          const docLink = document.createElement('div');\n",
              "          docLink.innerHTML = docLinkHtml;\n",
              "          element.appendChild(docLink);\n",
              "        }\n",
              "      </script>\n",
              "    </div>\n",
              "  </div>\n",
              "  "
            ],
            "text/plain": [
              "        pmcid  ...                            figure.title\n",
              "0  PMC6323817  ...  Differentially expressed genes (DEG...\n",
              "1  PMC6348685  ...  Putative genes and their expression...\n",
              "2  PMC6356271  ...              Terminal telomeric complex\n",
              "3  PMC6360084  ...  Identification, expression, and phy...\n",
              "4  PMC6374618  ...  Schematic relationships between the...\n",
              "5  PMC6394206  ...  Proposed diterpene pathway based on...\n",
              "6  PMC6412517  ...  Induction of secondary metabolism i...\n",
              "7  PMC6423058  ...            Pathway of terpene synthesis\n",
              "8  PMC6437448  ...  Biosynthetic pathway engineering fo...\n",
              "9  PMC6441073  ...  DEGs involved in the plant–pathogen...\n",
              "\n",
              "[10 rows x 6 columns]"
            ]
          },
          "metadata": {},
          "execution_count": 15
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "#Saving csv files"
      ],
      "metadata": {
        "id": "MVtaCeyj7vtN"
      }
    },
    {
      "cell_type": "markdown",
      "source": [
        "To save any csv file, use '***to_csv***' where we pass the file name with which we want to save the file along with the path where we wath so save the file. That is:\n",
        "\n",
        "to_csv(path +'file_name.csv')\n",
        "\n",
        "Here,\n",
        "\n",
        "path = '/content/'\n",
        "\n",
        "file name = 'new_file.csv'\n",
        "\n",
        "It is very importanat to add .csv at the end else the file will get saved in the default format that is .txt.\n",
        "\n",
        "More information on this can be found [here](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.to_csv.html). There are lot of attributes for to_csv. Few of them are mentioned below:\n",
        "\n",
        "\n",
        "1.   **column**: Columns to write.\n",
        "\n",
        "\n",
        "2.   **header**:   Write out the column names. If a list of strings is given it is assumed to be aliases for the column names.(default is True)\n",
        "\n",
        "3.   **index**: Write row names (index).default is True\n",
        "\n",
        "\n"
      ],
      "metadata": {
        "id": "f6Yq5cfl6dxo"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "#to save csv file\n",
        "df.to_csv('/content/new_file.csv')"
      ],
      "metadata": {
        "id": "lbWx5G5JfkX0"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "source": [
        "#Data Visualization\n",
        "\n",
        "Pandas helps us to inspect the data format by looking into the top or bottom values by using the functinality below:\n",
        "\n",
        "**df.head(n)** -----> returns the top n rows of dataframe, where n is a variable.\n",
        "\n",
        "**df.tail(n)**-----> returns the bottom n rows of dataframe, where n is a variable."
      ],
      "metadata": {
        "id": "ytwUoYZJS0Yv"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "df.head(10) # return top 10 values of the dataframe\n"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 569
        },
        "id": "CWKI5i43SZ30",
        "outputId": "155d7ad3-31fc-41bd-a69b-956ae96a14dd"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/html": [
              "\n",
              "  <div id=\"df-29fd231d-55ed-49e1-8885-4ba9addf5c33\">\n",
              "    <div class=\"colab-df-container\">\n",
              "      <div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>pmcid</th>\n",
              "      <th>paper.title</th>\n",
              "      <th>authors</th>\n",
              "      <th>year</th>\n",
              "      <th>number</th>\n",
              "      <th>figure.title</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>0</th>\n",
              "      <td>PMC6323817</td>\n",
              "      <td>Transcriptomic de novo analysis of pitaya (Hyl...</td>\n",
              "      <td>Min Xu, Cheng-L...</td>\n",
              "      <td>2019</td>\n",
              "      <td>Fig. 6</td>\n",
              "      <td>Differentially expressed genes (DEG...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>PMC6348685</td>\n",
              "      <td>Comparative transcriptome analysis of a long-t...</td>\n",
              "      <td>Luodong Huang, ...</td>\n",
              "      <td>2019</td>\n",
              "      <td>Fig. 4</td>\n",
              "      <td>Putative genes and their expression...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2</th>\n",
              "      <td>PMC6356271</td>\n",
              "      <td>Telomeres in Plants and Humans: Not So Differe...</td>\n",
              "      <td>Petra Procházko...</td>\n",
              "      <td>2019</td>\n",
              "      <td>Figure 2</td>\n",
              "      <td>Terminal telomeric complex</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3</th>\n",
              "      <td>PMC6360084</td>\n",
              "      <td>Identification, expression, and phylogenetic a...</td>\n",
              "      <td>Jipeng Mao, Zid...</td>\n",
              "      <td>2019</td>\n",
              "      <td>Figure 8</td>\n",
              "      <td>Identification, expression, and phy...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4</th>\n",
              "      <td>PMC6374618</td>\n",
              "      <td>Agarwood Induction: Current Developments and F...</td>\n",
              "      <td>Cheng Seng CS T...</td>\n",
              "      <td>2019</td>\n",
              "      <td>FIGURE 4</td>\n",
              "      <td>Schematic relationships between the...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>5</th>\n",
              "      <td>PMC6394206</td>\n",
              "      <td>A chromosomal-scale genome assembly of Tectona...</td>\n",
              "      <td>Dongyan Zhao, J...</td>\n",
              "      <td>2019</td>\n",
              "      <td>Figure 5:</td>\n",
              "      <td>Proposed diterpene pathway based on...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>6</th>\n",
              "      <td>PMC6412517</td>\n",
              "      <td>New Insights on Arabidopsis thaliana Root Adap...</td>\n",
              "      <td>Inmaculada Cole...</td>\n",
              "      <td>2019</td>\n",
              "      <td>Figure 2</td>\n",
              "      <td>Induction of secondary metabolism i...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>7</th>\n",
              "      <td>PMC6423058</td>\n",
              "      <td>Why Algae Release Volatile Organic Compounds—T...</td>\n",
              "      <td>Zhaojiang Zuo</td>\n",
              "      <td>2019</td>\n",
              "      <td>FIGURE 2</td>\n",
              "      <td>Pathway of terpene synthesis</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>8</th>\n",
              "      <td>PMC6437448</td>\n",
              "      <td>Production of ginsenoside aglycone (protopanax...</td>\n",
              "      <td>Yu Shin YS Gwak...</td>\n",
              "      <td>2019</td>\n",
              "      <td>Fig. 1</td>\n",
              "      <td>Biosynthetic pathway engineering fo...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>9</th>\n",
              "      <td>PMC6441073</td>\n",
              "      <td>Transcriptional response of grapevine to infec...</td>\n",
              "      <td>Wei Zhang, Jiye...</td>\n",
              "      <td>2019</td>\n",
              "      <td>Figure 3</td>\n",
              "      <td>DEGs involved in the plant–pathogen...</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "</div>\n",
              "      <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-29fd231d-55ed-49e1-8885-4ba9addf5c33')\"\n",
              "              title=\"Convert this dataframe to an interactive table.\"\n",
              "              style=\"display:none;\">\n",
              "        \n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "       width=\"24px\">\n",
              "    <path d=\"M0 0h24v24H0V0z\" fill=\"none\"/>\n",
              "    <path d=\"M18.56 5.44l.94 2.06.94-2.06 2.06-.94-2.06-.94-.94-2.06-.94 2.06-2.06.94zm-11 1L8.5 8.5l.94-2.06 2.06-.94-2.06-.94L8.5 2.5l-.94 2.06-2.06.94zm10 10l.94 2.06.94-2.06 2.06-.94-2.06-.94-.94-2.06-.94 2.06-2.06.94z\"/><path d=\"M17.41 7.96l-1.37-1.37c-.4-.4-.92-.59-1.43-.59-.52 0-1.04.2-1.43.59L10.3 9.45l-7.72 7.72c-.78.78-.78 2.05 0 2.83L4 21.41c.39.39.9.59 1.41.59.51 0 1.02-.2 1.41-.59l7.78-7.78 2.81-2.81c.8-.78.8-2.07 0-2.86zM5.41 20L4 18.59l7.72-7.72 1.47 1.35L5.41 20z\"/>\n",
              "  </svg>\n",
              "      </button>\n",
              "      \n",
              "  <style>\n",
              "    .colab-df-container {\n",
              "      display:flex;\n",
              "      flex-wrap:wrap;\n",
              "      gap: 12px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert {\n",
              "      background-color: #E8F0FE;\n",
              "      border: none;\n",
              "      border-radius: 50%;\n",
              "      cursor: pointer;\n",
              "      display: none;\n",
              "      fill: #1967D2;\n",
              "      height: 32px;\n",
              "      padding: 0 0 0 0;\n",
              "      width: 32px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert:hover {\n",
              "      background-color: #E2EBFA;\n",
              "      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "      fill: #174EA6;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert {\n",
              "      background-color: #3B4455;\n",
              "      fill: #D2E3FC;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert:hover {\n",
              "      background-color: #434B5C;\n",
              "      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "      fill: #FFFFFF;\n",
              "    }\n",
              "  </style>\n",
              "\n",
              "      <script>\n",
              "        const buttonEl =\n",
              "          document.querySelector('#df-29fd231d-55ed-49e1-8885-4ba9addf5c33 button.colab-df-convert');\n",
              "        buttonEl.style.display =\n",
              "          google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "        async function convertToInteractive(key) {\n",
              "          const element = document.querySelector('#df-29fd231d-55ed-49e1-8885-4ba9addf5c33');\n",
              "          const dataTable =\n",
              "            await google.colab.kernel.invokeFunction('convertToInteractive',\n",
              "                                                     [key], {});\n",
              "          if (!dataTable) return;\n",
              "\n",
              "          const docLinkHtml = 'Like what you see? Visit the ' +\n",
              "            '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
              "            + ' to learn more about interactive tables.';\n",
              "          element.innerHTML = '';\n",
              "          dataTable['output_type'] = 'display_data';\n",
              "          await google.colab.output.renderOutput(dataTable, element);\n",
              "          const docLink = document.createElement('div');\n",
              "          docLink.innerHTML = docLinkHtml;\n",
              "          element.appendChild(docLink);\n",
              "        }\n",
              "      </script>\n",
              "    </div>\n",
              "  </div>\n",
              "  "
            ],
            "text/plain": [
              "        pmcid  ...                            figure.title\n",
              "0  PMC6323817  ...  Differentially expressed genes (DEG...\n",
              "1  PMC6348685  ...  Putative genes and their expression...\n",
              "2  PMC6356271  ...              Terminal telomeric complex\n",
              "3  PMC6360084  ...  Identification, expression, and phy...\n",
              "4  PMC6374618  ...  Schematic relationships between the...\n",
              "5  PMC6394206  ...  Proposed diterpene pathway based on...\n",
              "6  PMC6412517  ...  Induction of secondary metabolism i...\n",
              "7  PMC6423058  ...            Pathway of terpene synthesis\n",
              "8  PMC6437448  ...  Biosynthetic pathway engineering fo...\n",
              "9  PMC6441073  ...  DEGs involved in the plant–pathogen...\n",
              "\n",
              "[10 rows x 6 columns]"
            ]
          },
          "metadata": {},
          "execution_count": 17
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "df.tail(10) # return bottom 10 values of the dataframe"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 569
        },
        "id": "RLg8ZPC1UDoN",
        "outputId": "f19fc41e-7e6b-4efe-cab3-5afe130c0a8b"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/html": [
              "\n",
              "  <div id=\"df-ba480f52-b740-4263-a788-4d75a5d77716\">\n",
              "    <div class=\"colab-df-container\">\n",
              "      <div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>pmcid</th>\n",
              "      <th>paper.title</th>\n",
              "      <th>authors</th>\n",
              "      <th>year</th>\n",
              "      <th>number</th>\n",
              "      <th>figure.title</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>0</th>\n",
              "      <td>PMC6323817</td>\n",
              "      <td>Transcriptomic de novo analysis of pitaya (Hyl...</td>\n",
              "      <td>Min Xu, Cheng-L...</td>\n",
              "      <td>2019</td>\n",
              "      <td>Fig. 6</td>\n",
              "      <td>Differentially expressed genes (DEG...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>PMC6348685</td>\n",
              "      <td>Comparative transcriptome analysis of a long-t...</td>\n",
              "      <td>Luodong Huang, ...</td>\n",
              "      <td>2019</td>\n",
              "      <td>Fig. 4</td>\n",
              "      <td>Putative genes and their expression...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2</th>\n",
              "      <td>PMC6356271</td>\n",
              "      <td>Telomeres in Plants and Humans: Not So Differe...</td>\n",
              "      <td>Petra Procházko...</td>\n",
              "      <td>2019</td>\n",
              "      <td>Figure 2</td>\n",
              "      <td>Terminal telomeric complex</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3</th>\n",
              "      <td>PMC6360084</td>\n",
              "      <td>Identification, expression, and phylogenetic a...</td>\n",
              "      <td>Jipeng Mao, Zid...</td>\n",
              "      <td>2019</td>\n",
              "      <td>Figure 8</td>\n",
              "      <td>Identification, expression, and phy...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4</th>\n",
              "      <td>PMC6374618</td>\n",
              "      <td>Agarwood Induction: Current Developments and F...</td>\n",
              "      <td>Cheng Seng CS T...</td>\n",
              "      <td>2019</td>\n",
              "      <td>FIGURE 4</td>\n",
              "      <td>Schematic relationships between the...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>5</th>\n",
              "      <td>PMC6394206</td>\n",
              "      <td>A chromosomal-scale genome assembly of Tectona...</td>\n",
              "      <td>Dongyan Zhao, J...</td>\n",
              "      <td>2019</td>\n",
              "      <td>Figure 5:</td>\n",
              "      <td>Proposed diterpene pathway based on...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>6</th>\n",
              "      <td>PMC6412517</td>\n",
              "      <td>New Insights on Arabidopsis thaliana Root Adap...</td>\n",
              "      <td>Inmaculada Cole...</td>\n",
              "      <td>2019</td>\n",
              "      <td>Figure 2</td>\n",
              "      <td>Induction of secondary metabolism i...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>7</th>\n",
              "      <td>PMC6423058</td>\n",
              "      <td>Why Algae Release Volatile Organic Compounds—T...</td>\n",
              "      <td>Zhaojiang Zuo</td>\n",
              "      <td>2019</td>\n",
              "      <td>FIGURE 2</td>\n",
              "      <td>Pathway of terpene synthesis</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>8</th>\n",
              "      <td>PMC6437448</td>\n",
              "      <td>Production of ginsenoside aglycone (protopanax...</td>\n",
              "      <td>Yu Shin YS Gwak...</td>\n",
              "      <td>2019</td>\n",
              "      <td>Fig. 1</td>\n",
              "      <td>Biosynthetic pathway engineering fo...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>9</th>\n",
              "      <td>PMC6441073</td>\n",
              "      <td>Transcriptional response of grapevine to infec...</td>\n",
              "      <td>Wei Zhang, Jiye...</td>\n",
              "      <td>2019</td>\n",
              "      <td>Figure 3</td>\n",
              "      <td>DEGs involved in the plant–pathogen...</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "</div>\n",
              "      <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-ba480f52-b740-4263-a788-4d75a5d77716')\"\n",
              "              title=\"Convert this dataframe to an interactive table.\"\n",
              "              style=\"display:none;\">\n",
              "        \n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "       width=\"24px\">\n",
              "    <path d=\"M0 0h24v24H0V0z\" fill=\"none\"/>\n",
              "    <path d=\"M18.56 5.44l.94 2.06.94-2.06 2.06-.94-2.06-.94-.94-2.06-.94 2.06-2.06.94zm-11 1L8.5 8.5l.94-2.06 2.06-.94-2.06-.94L8.5 2.5l-.94 2.06-2.06.94zm10 10l.94 2.06.94-2.06 2.06-.94-2.06-.94-.94-2.06-.94 2.06-2.06.94z\"/><path d=\"M17.41 7.96l-1.37-1.37c-.4-.4-.92-.59-1.43-.59-.52 0-1.04.2-1.43.59L10.3 9.45l-7.72 7.72c-.78.78-.78 2.05 0 2.83L4 21.41c.39.39.9.59 1.41.59.51 0 1.02-.2 1.41-.59l7.78-7.78 2.81-2.81c.8-.78.8-2.07 0-2.86zM5.41 20L4 18.59l7.72-7.72 1.47 1.35L5.41 20z\"/>\n",
              "  </svg>\n",
              "      </button>\n",
              "      \n",
              "  <style>\n",
              "    .colab-df-container {\n",
              "      display:flex;\n",
              "      flex-wrap:wrap;\n",
              "      gap: 12px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert {\n",
              "      background-color: #E8F0FE;\n",
              "      border: none;\n",
              "      border-radius: 50%;\n",
              "      cursor: pointer;\n",
              "      display: none;\n",
              "      fill: #1967D2;\n",
              "      height: 32px;\n",
              "      padding: 0 0 0 0;\n",
              "      width: 32px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert:hover {\n",
              "      background-color: #E2EBFA;\n",
              "      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "      fill: #174EA6;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert {\n",
              "      background-color: #3B4455;\n",
              "      fill: #D2E3FC;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert:hover {\n",
              "      background-color: #434B5C;\n",
              "      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "      fill: #FFFFFF;\n",
              "    }\n",
              "  </style>\n",
              "\n",
              "      <script>\n",
              "        const buttonEl =\n",
              "          document.querySelector('#df-ba480f52-b740-4263-a788-4d75a5d77716 button.colab-df-convert');\n",
              "        buttonEl.style.display =\n",
              "          google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "        async function convertToInteractive(key) {\n",
              "          const element = document.querySelector('#df-ba480f52-b740-4263-a788-4d75a5d77716');\n",
              "          const dataTable =\n",
              "            await google.colab.kernel.invokeFunction('convertToInteractive',\n",
              "                                                     [key], {});\n",
              "          if (!dataTable) return;\n",
              "\n",
              "          const docLinkHtml = 'Like what you see? Visit the ' +\n",
              "            '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
              "            + ' to learn more about interactive tables.';\n",
              "          element.innerHTML = '';\n",
              "          dataTable['output_type'] = 'display_data';\n",
              "          await google.colab.output.renderOutput(dataTable, element);\n",
              "          const docLink = document.createElement('div');\n",
              "          docLink.innerHTML = docLinkHtml;\n",
              "          element.appendChild(docLink);\n",
              "        }\n",
              "      </script>\n",
              "    </div>\n",
              "  </div>\n",
              "  "
            ],
            "text/plain": [
              "        pmcid  ...                            figure.title\n",
              "0  PMC6323817  ...  Differentially expressed genes (DEG...\n",
              "1  PMC6348685  ...  Putative genes and their expression...\n",
              "2  PMC6356271  ...              Terminal telomeric complex\n",
              "3  PMC6360084  ...  Identification, expression, and phy...\n",
              "4  PMC6374618  ...  Schematic relationships between the...\n",
              "5  PMC6394206  ...  Proposed diterpene pathway based on...\n",
              "6  PMC6412517  ...  Induction of secondary metabolism i...\n",
              "7  PMC6423058  ...            Pathway of terpene synthesis\n",
              "8  PMC6437448  ...  Biosynthetic pathway engineering fo...\n",
              "9  PMC6441073  ...  DEGs involved in the plant–pathogen...\n",
              "\n",
              "[10 rows x 6 columns]"
            ]
          },
          "metadata": {},
          "execution_count": 18
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "In order to know the column names of your dataframe. You have to run the line of code below:\n",
        "\n",
        "**df.column** --------> returns an object type which contain the list of column names\n",
        "\n",
        "To access the list directly do:\n",
        "\n",
        "**list(df.columns)** --------> return the list of names of the columns\n"
      ],
      "metadata": {
        "id": "rmc6ZYjBNBp5"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "df.columns # returns list of column names\n",
        "list(df.columns)"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "5lHmy9m3Otco",
        "outputId": "33e632dd-c456-4de6-a4d9-dcbecdae20a5"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "['pmcid', 'paper.title', 'authors', 'year', 'number', 'figure.title']"
            ]
          },
          "metadata": {},
          "execution_count": 22
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "Similar to columns is yu want to acces the index names or its range. Please follow the line of code given below:\n",
        "\n",
        "**df.index**  ------> return the range of the index and is the names of index are string then it will return the list of names."
      ],
      "metadata": {
        "id": "d3zlrV-kZ8oz"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "df.index   # return indexes\n"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "S5x6omYrUGxg",
        "outputId": "cc1ea25d-20a0-4601-bf47-72b034754f95"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "RangeIndex(start=0, stop=10, step=1)"
            ]
          },
          "metadata": {},
          "execution_count": 20
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "To get to know the about the no of columns and rows in the dataframe, run the line of code below:\n",
        "\n",
        "**df.shape**   -------> returns a tuple which looks like **(no_of_columns, no_of_rows)**\n",
        "\n",
        "Therefore,\n",
        "\n",
        "No of Rows = df.shape[0]\n",
        "\n",
        "No of Columns = df.shape[1]"
      ],
      "metadata": {
        "id": "YFyL8yfpd3Hs"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "df.shape # return a tuple giving information aout the shape of the file for ex: (30,10) means the csv has 30 rows and 10 columns\n",
        "print('No of rows in the csv file are :{}     and \\nNo of columns are: {}'.format(df.shape[0],df.shape[1]))"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "O3r0EqB1UIYj",
        "outputId": "a98e9bac-d330-4bfb-aab7-9e9545ffea65"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "No of rows in the csv file are :10     and \n",
            "No of columns are: 6\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "# Statistical inference\n",
        "To infer the different statistics of your data like mean, max, median etc you can see the block of code below.\n",
        "Also detailed operations can be found [here](https://pandas.pydata.org/docs/getting_started/intro_tutorials/06_calculate_statistics.html)"
      ],
      "metadata": {
        "id": "5T7iHaG_TI8m"
      }
    },
    {
      "cell_type": "markdown",
      "source": [
        "**df.max()** -------> return the maximnum value of each column.\n",
        "\n",
        "Here you can also find the maximum of an individula column. Lets say we want max for the column named 'year' then we run the line of code:\n",
        "\n",
        "**df['year'].max()** --------> returns max value amongst others values in the column named year"
      ],
      "metadata": {
        "id": "_gmEKZVcgef_"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "df.max()      # return max for each column"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "h9Iwm3_AUiih",
        "outputId": "5b500233-bbed-4dbd-a4e3-83408038cb2a"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "pmcid                                                  PMC6441073\n",
              "paper.title     Why Algae Release Volatile Organic Compounds—T...\n",
              "authors                                             Zhaojiang Zuo\n",
              "year                                                         2019\n",
              "number                                                   Figure 8\n",
              "figure.title                           Terminal telomeric complex\n",
              "dtype: object"
            ]
          },
          "metadata": {},
          "execution_count": 30
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "Similary,\n",
        "**df.min()** --------> returns the min value of eac columns \n",
        "\n",
        "we can aslo find indivual min for a given column following the same method as we did for maximum."
      ],
      "metadata": {
        "id": "sP8IJGzjioDZ"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "df.min()      # return min for each column"
      ],
      "metadata": {
        "id": "aOCIvypeijZu",
        "outputId": "461f0a91-0aec-4373-977c-e8c238061706",
        "colab": {
          "base_uri": "https://localhost:8080/"
        }
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "pmcid                                                  PMC6323817\n",
              "paper.title     A chromosomal-scale genome assembly of Tectona...\n",
              "authors                                        Cheng Seng CS T...\n",
              "year                                                         2019\n",
              "number                                                   FIGURE 2\n",
              "figure.title               Biosynthetic pathway engineering fo...\n",
              "dtype: object"
            ]
          },
          "metadata": {},
          "execution_count": 31
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "We can also extract mean, variance and standard deviation for each column as given in the lines of code below:"
      ],
      "metadata": {
        "id": "FnMKgJiijcLA"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "df.mean()      #return mean for each column"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "qKH5KlFtUnI_",
        "outputId": "556168f6-af67-4ed3-c014-e0795f3ec58a"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "year    2019.0\n",
              "dtype: float64"
            ]
          },
          "metadata": {},
          "execution_count": 32
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "df.var() #return variance for each column"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "xdzwjYXjUxxs",
        "outputId": "6b7691ca-23d6-4c99-edac-770787b1f7a9"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "year    0.0\n",
              "dtype: float64"
            ]
          },
          "metadata": {},
          "execution_count": 33
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "df.std() #return standard deviation for each column"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "nftYAx-MU4ke",
        "outputId": "14945c71-f92d-4e65-c8bf-b1d9ad845ccc"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "year    0.0\n",
              "dtype: float64"
            ]
          },
          "metadata": {},
          "execution_count": 34
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "# Combining two csv\n",
        "When you have several csv and want to combine them, Pandas concat function comes really handy. \n",
        "In the code below we are appending two csv by first converting them to dataframes and then concatenating them with help of concat function.\n",
        "\n",
        "You can find more about **pd.concat** [here](https://)\n"
      ],
      "metadata": {
        "id": "UVc4S8OYTAfF"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "#appending two csv files :\n",
        "\n",
        "df1 = pd.read_csv('/content/Terpene and Other Plant Pathway Figures.csv')\n",
        "df2 = pd.read_csv('/content/Terpene and Other Plant Pathway Figures (3).csv')\n",
        "df_new = pd.concat([df1,df2], axis = 0).reset_index() ##horizontly\n",
        "#df_new = pd.concat([df1,df2], axis = 1) ##vertically"
      ],
      "metadata": {
        "id": "KcW2GFFvNAgI"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "df_new"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 1000
        },
        "id": "hF4OWE8xOsNl",
        "outputId": "1e23048d-964a-493d-8c8a-176ebb1e8c39"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/html": [
              "\n",
              "  <div id=\"df-93d94c84-39e0-4f2e-9baf-d0d43f6de9d3\">\n",
              "    <div class=\"colab-df-container\">\n",
              "      <div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>index</th>\n",
              "      <th>pmcid</th>\n",
              "      <th>paper.title</th>\n",
              "      <th>authors</th>\n",
              "      <th>year</th>\n",
              "      <th>number</th>\n",
              "      <th>figure.title</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>0</th>\n",
              "      <td>0</td>\n",
              "      <td>PMC6323817</td>\n",
              "      <td>Transcriptomic de novo analysis of pitaya (Hyl...</td>\n",
              "      <td>Min Xu, Cheng-L...</td>\n",
              "      <td>2019</td>\n",
              "      <td>Fig. 6</td>\n",
              "      <td>Differentially expressed genes (DEG...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>1</td>\n",
              "      <td>PMC6348685</td>\n",
              "      <td>Comparative transcriptome analysis of a long-t...</td>\n",
              "      <td>Luodong Huang, ...</td>\n",
              "      <td>2019</td>\n",
              "      <td>Fig. 4</td>\n",
              "      <td>Putative genes and their expression...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2</th>\n",
              "      <td>2</td>\n",
              "      <td>PMC6356271</td>\n",
              "      <td>Telomeres in Plants and Humans: Not So Differe...</td>\n",
              "      <td>Petra Procházko...</td>\n",
              "      <td>2019</td>\n",
              "      <td>Figure 2</td>\n",
              "      <td>Terminal telomeric complex</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3</th>\n",
              "      <td>3</td>\n",
              "      <td>PMC6360084</td>\n",
              "      <td>Identification, expression, and phylogenetic a...</td>\n",
              "      <td>Jipeng Mao, Zid...</td>\n",
              "      <td>2019</td>\n",
              "      <td>Figure 8</td>\n",
              "      <td>Identification, expression, and phy...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4</th>\n",
              "      <td>4</td>\n",
              "      <td>PMC6374618</td>\n",
              "      <td>Agarwood Induction: Current Developments and F...</td>\n",
              "      <td>Cheng Seng CS T...</td>\n",
              "      <td>2019</td>\n",
              "      <td>FIGURE 4</td>\n",
              "      <td>Schematic relationships between the...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>5</th>\n",
              "      <td>5</td>\n",
              "      <td>PMC6394206</td>\n",
              "      <td>A chromosomal-scale genome assembly of Tectona...</td>\n",
              "      <td>Dongyan Zhao, J...</td>\n",
              "      <td>2019</td>\n",
              "      <td>Figure 5:</td>\n",
              "      <td>Proposed diterpene pathway based on...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>6</th>\n",
              "      <td>6</td>\n",
              "      <td>PMC6412517</td>\n",
              "      <td>New Insights on Arabidopsis thaliana Root Adap...</td>\n",
              "      <td>Inmaculada Cole...</td>\n",
              "      <td>2019</td>\n",
              "      <td>Figure 2</td>\n",
              "      <td>Induction of secondary metabolism i...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>7</th>\n",
              "      <td>7</td>\n",
              "      <td>PMC6423058</td>\n",
              "      <td>Why Algae Release Volatile Organic Compounds—T...</td>\n",
              "      <td>Zhaojiang Zuo</td>\n",
              "      <td>2019</td>\n",
              "      <td>FIGURE 2</td>\n",
              "      <td>Pathway of terpene synthesis</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>8</th>\n",
              "      <td>8</td>\n",
              "      <td>PMC6437448</td>\n",
              "      <td>Production of ginsenoside aglycone (protopanax...</td>\n",
              "      <td>Yu Shin YS Gwak...</td>\n",
              "      <td>2019</td>\n",
              "      <td>Fig. 1</td>\n",
              "      <td>Biosynthetic pathway engineering fo...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>9</th>\n",
              "      <td>9</td>\n",
              "      <td>PMC6441073</td>\n",
              "      <td>Transcriptional response of grapevine to infec...</td>\n",
              "      <td>Wei Zhang, Jiye...</td>\n",
              "      <td>2019</td>\n",
              "      <td>Figure 3</td>\n",
              "      <td>DEGs involved in the plant–pathogen...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>10</th>\n",
              "      <td>0</td>\n",
              "      <td>PMC6453882</td>\n",
              "      <td>Proteomic analysis reveals novel insights into...</td>\n",
              "      <td>Angela Contrera...</td>\n",
              "      <td>2019</td>\n",
              "      <td>Figure 4</td>\n",
              "      <td>Overview of the proposed biosynthet...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>11</th>\n",
              "      <td>1</td>\n",
              "      <td>PMC6475556</td>\n",
              "      <td>Protective Effect of the Total Triterpenes of ...</td>\n",
              "      <td>Wei Huang, Hui ...</td>\n",
              "      <td>2019</td>\n",
              "      <td>Figure 7</td>\n",
              "      <td>The mechanisms of total triterpenes...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>12</th>\n",
              "      <td>2</td>\n",
              "      <td>PMC6486373</td>\n",
              "      <td>Emerging Antitumor Activities of the Bitter Me...</td>\n",
              "      <td>Evandro Fei EF ...</td>\n",
              "      <td>2019</td>\n",
              "      <td>Fig. (1)</td>\n",
              "      <td>Possible mechanisms of the anti-tum...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>13</th>\n",
              "      <td>3</td>\n",
              "      <td>PMC6509841</td>\n",
              "      <td>Transcriptome and physiological analyses for r...</td>\n",
              "      <td>Xing Yu, Tanchu...</td>\n",
              "      <td>2019</td>\n",
              "      <td>Fig. 8</td>\n",
              "      <td>DEGs relevant to the “plant hormone...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>14</th>\n",
              "      <td>4</td>\n",
              "      <td>PMC6544981</td>\n",
              "      <td>Co-production of farnesol and coenzyme Q10 fro...</td>\n",
              "      <td>Xueduan Chen, X...</td>\n",
              "      <td>2019</td>\n",
              "      <td>Fig. 1</td>\n",
              "      <td>Co-production of farnesol and coenz...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>15</th>\n",
              "      <td>5</td>\n",
              "      <td>PMC6546118</td>\n",
              "      <td>Determination of the Key Resistance Gene Analo...</td>\n",
              "      <td>Ziwei Zhou, Ido...</td>\n",
              "      <td>2019</td>\n",
              "      <td>FIGURE 3</td>\n",
              "      <td>Plant MAPK signaling pathway</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>16</th>\n",
              "      <td>6</td>\n",
              "      <td>PMC6585571</td>\n",
              "      <td>Inside and Beyond Color: Comparative Overview ...</td>\n",
              "      <td>Riadh Ilahy, Im...</td>\n",
              "      <td>2019</td>\n",
              "      <td>FIGURE 1</td>\n",
              "      <td>Schematic Carotenoid Metabolic Path...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>17</th>\n",
              "      <td>7</td>\n",
              "      <td>PMC6589721</td>\n",
              "      <td>Multifaceted role of geminivirus associated be...</td>\n",
              "      <td>Prabu Gnanaseka...</td>\n",
              "      <td>2019</td>\n",
              "      <td>Figure 2</td>\n",
              "      <td>Multifaceted roles of BC1 in the sc...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>18</th>\n",
              "      <td>8</td>\n",
              "      <td>PMC6589721</td>\n",
              "      <td>Multifaceted role of geminivirus associated be...</td>\n",
              "      <td>Prabu Gnanaseka...</td>\n",
              "      <td>2019</td>\n",
              "      <td>Figure 3</td>\n",
              "      <td>Hypothetical working model illustra...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>19</th>\n",
              "      <td>9</td>\n",
              "      <td>PMC6589726</td>\n",
              "      <td>RNAi of the sesquiterpene cyclase gene for phy...</td>\n",
              "      <td>Miki Yoshioka, ...</td>\n",
              "      <td>2019</td>\n",
              "      <td>Figure 1</td>\n",
              "      <td>Scheme of sesquiterpenoid phytoalex...</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "</div>\n",
              "      <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-93d94c84-39e0-4f2e-9baf-d0d43f6de9d3')\"\n",
              "              title=\"Convert this dataframe to an interactive table.\"\n",
              "              style=\"display:none;\">\n",
              "        \n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "       width=\"24px\">\n",
              "    <path d=\"M0 0h24v24H0V0z\" fill=\"none\"/>\n",
              "    <path d=\"M18.56 5.44l.94 2.06.94-2.06 2.06-.94-2.06-.94-.94-2.06-.94 2.06-2.06.94zm-11 1L8.5 8.5l.94-2.06 2.06-.94-2.06-.94L8.5 2.5l-.94 2.06-2.06.94zm10 10l.94 2.06.94-2.06 2.06-.94-2.06-.94-.94-2.06-.94 2.06-2.06.94z\"/><path d=\"M17.41 7.96l-1.37-1.37c-.4-.4-.92-.59-1.43-.59-.52 0-1.04.2-1.43.59L10.3 9.45l-7.72 7.72c-.78.78-.78 2.05 0 2.83L4 21.41c.39.39.9.59 1.41.59.51 0 1.02-.2 1.41-.59l7.78-7.78 2.81-2.81c.8-.78.8-2.07 0-2.86zM5.41 20L4 18.59l7.72-7.72 1.47 1.35L5.41 20z\"/>\n",
              "  </svg>\n",
              "      </button>\n",
              "      \n",
              "  <style>\n",
              "    .colab-df-container {\n",
              "      display:flex;\n",
              "      flex-wrap:wrap;\n",
              "      gap: 12px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert {\n",
              "      background-color: #E8F0FE;\n",
              "      border: none;\n",
              "      border-radius: 50%;\n",
              "      cursor: pointer;\n",
              "      display: none;\n",
              "      fill: #1967D2;\n",
              "      height: 32px;\n",
              "      padding: 0 0 0 0;\n",
              "      width: 32px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert:hover {\n",
              "      background-color: #E2EBFA;\n",
              "      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "      fill: #174EA6;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert {\n",
              "      background-color: #3B4455;\n",
              "      fill: #D2E3FC;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert:hover {\n",
              "      background-color: #434B5C;\n",
              "      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "      fill: #FFFFFF;\n",
              "    }\n",
              "  </style>\n",
              "\n",
              "      <script>\n",
              "        const buttonEl =\n",
              "          document.querySelector('#df-93d94c84-39e0-4f2e-9baf-d0d43f6de9d3 button.colab-df-convert');\n",
              "        buttonEl.style.display =\n",
              "          google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "        async function convertToInteractive(key) {\n",
              "          const element = document.querySelector('#df-93d94c84-39e0-4f2e-9baf-d0d43f6de9d3');\n",
              "          const dataTable =\n",
              "            await google.colab.kernel.invokeFunction('convertToInteractive',\n",
              "                                                     [key], {});\n",
              "          if (!dataTable) return;\n",
              "\n",
              "          const docLinkHtml = 'Like what you see? Visit the ' +\n",
              "            '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
              "            + ' to learn more about interactive tables.';\n",
              "          element.innerHTML = '';\n",
              "          dataTable['output_type'] = 'display_data';\n",
              "          await google.colab.output.renderOutput(dataTable, element);\n",
              "          const docLink = document.createElement('div');\n",
              "          docLink.innerHTML = docLinkHtml;\n",
              "          element.appendChild(docLink);\n",
              "        }\n",
              "      </script>\n",
              "    </div>\n",
              "  </div>\n",
              "  "
            ],
            "text/plain": [
              "    index       pmcid  ...     number                            figure.title\n",
              "0       0  PMC6323817  ...     Fig. 6  Differentially expressed genes (DEG...\n",
              "1       1  PMC6348685  ...     Fig. 4  Putative genes and their expression...\n",
              "2       2  PMC6356271  ...   Figure 2              Terminal telomeric complex\n",
              "3       3  PMC6360084  ...   Figure 8  Identification, expression, and phy...\n",
              "4       4  PMC6374618  ...   FIGURE 4  Schematic relationships between the...\n",
              "5       5  PMC6394206  ...  Figure 5:  Proposed diterpene pathway based on...\n",
              "6       6  PMC6412517  ...   Figure 2  Induction of secondary metabolism i...\n",
              "7       7  PMC6423058  ...   FIGURE 2            Pathway of terpene synthesis\n",
              "8       8  PMC6437448  ...     Fig. 1  Biosynthetic pathway engineering fo...\n",
              "9       9  PMC6441073  ...   Figure 3  DEGs involved in the plant–pathogen...\n",
              "10      0  PMC6453882  ...   Figure 4  Overview of the proposed biosynthet...\n",
              "11      1  PMC6475556  ...   Figure 7  The mechanisms of total triterpenes...\n",
              "12      2  PMC6486373  ...   Fig. (1)  Possible mechanisms of the anti-tum...\n",
              "13      3  PMC6509841  ...     Fig. 8  DEGs relevant to the “plant hormone...\n",
              "14      4  PMC6544981  ...     Fig. 1  Co-production of farnesol and coenz...\n",
              "15      5  PMC6546118  ...   FIGURE 3            Plant MAPK signaling pathway\n",
              "16      6  PMC6585571  ...   FIGURE 1  Schematic Carotenoid Metabolic Path...\n",
              "17      7  PMC6589721  ...   Figure 2  Multifaceted roles of BC1 in the sc...\n",
              "18      8  PMC6589721  ...   Figure 3  Hypothetical working model illustra...\n",
              "19      9  PMC6589726  ...   Figure 1  Scheme of sesquiterpenoid phytoalex...\n",
              "\n",
              "[20 rows x 7 columns]"
            ]
          },
          "metadata": {},
          "execution_count": 43
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "#data cleaning"
      ],
      "metadata": {
        "id": "FqkdDDswSt0A"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "#Now the data mostly we get from the internet or other source are not clean, which means several for the inputs might be empty, or junk\n",
        "#In order to clean the data some of the basic steps are:\n",
        "\n",
        "pd.isna(df) #returns a boolean mask where values are nan.\n",
        "\n"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 357
        },
        "id": "HX0xkL4N1Ogb",
        "outputId": "2b24ecd1-25f5-4959-c08e-a9228a28bb1b"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/html": [
              "\n",
              "  <div id=\"df-c492b6e2-c2a7-4d00-ad18-f7e6d77fff78\">\n",
              "    <div class=\"colab-df-container\">\n",
              "      <div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>pmcid</th>\n",
              "      <th>paper.title</th>\n",
              "      <th>authors</th>\n",
              "      <th>year</th>\n",
              "      <th>number</th>\n",
              "      <th>figure.title</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>0</th>\n",
              "      <td>False</td>\n",
              "      <td>False</td>\n",
              "      <td>False</td>\n",
              "      <td>False</td>\n",
              "      <td>False</td>\n",
              "      <td>False</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>False</td>\n",
              "      <td>False</td>\n",
              "      <td>False</td>\n",
              "      <td>False</td>\n",
              "      <td>False</td>\n",
              "      <td>False</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2</th>\n",
              "      <td>False</td>\n",
              "      <td>False</td>\n",
              "      <td>False</td>\n",
              "      <td>False</td>\n",
              "      <td>False</td>\n",
              "      <td>False</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3</th>\n",
              "      <td>False</td>\n",
              "      <td>False</td>\n",
              "      <td>False</td>\n",
              "      <td>False</td>\n",
              "      <td>False</td>\n",
              "      <td>False</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4</th>\n",
              "      <td>False</td>\n",
              "      <td>False</td>\n",
              "      <td>False</td>\n",
              "      <td>False</td>\n",
              "      <td>False</td>\n",
              "      <td>False</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>5</th>\n",
              "      <td>False</td>\n",
              "      <td>False</td>\n",
              "      <td>False</td>\n",
              "      <td>False</td>\n",
              "      <td>False</td>\n",
              "      <td>False</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>6</th>\n",
              "      <td>False</td>\n",
              "      <td>False</td>\n",
              "      <td>False</td>\n",
              "      <td>False</td>\n",
              "      <td>False</td>\n",
              "      <td>False</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>7</th>\n",
              "      <td>False</td>\n",
              "      <td>False</td>\n",
              "      <td>False</td>\n",
              "      <td>False</td>\n",
              "      <td>False</td>\n",
              "      <td>False</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>8</th>\n",
              "      <td>False</td>\n",
              "      <td>False</td>\n",
              "      <td>False</td>\n",
              "      <td>False</td>\n",
              "      <td>False</td>\n",
              "      <td>False</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>9</th>\n",
              "      <td>False</td>\n",
              "      <td>False</td>\n",
              "      <td>False</td>\n",
              "      <td>False</td>\n",
              "      <td>False</td>\n",
              "      <td>False</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "</div>\n",
              "      <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-c492b6e2-c2a7-4d00-ad18-f7e6d77fff78')\"\n",
              "              title=\"Convert this dataframe to an interactive table.\"\n",
              "              style=\"display:none;\">\n",
              "        \n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "       width=\"24px\">\n",
              "    <path d=\"M0 0h24v24H0V0z\" fill=\"none\"/>\n",
              "    <path d=\"M18.56 5.44l.94 2.06.94-2.06 2.06-.94-2.06-.94-.94-2.06-.94 2.06-2.06.94zm-11 1L8.5 8.5l.94-2.06 2.06-.94-2.06-.94L8.5 2.5l-.94 2.06-2.06.94zm10 10l.94 2.06.94-2.06 2.06-.94-2.06-.94-.94-2.06-.94 2.06-2.06.94z\"/><path d=\"M17.41 7.96l-1.37-1.37c-.4-.4-.92-.59-1.43-.59-.52 0-1.04.2-1.43.59L10.3 9.45l-7.72 7.72c-.78.78-.78 2.05 0 2.83L4 21.41c.39.39.9.59 1.41.59.51 0 1.02-.2 1.41-.59l7.78-7.78 2.81-2.81c.8-.78.8-2.07 0-2.86zM5.41 20L4 18.59l7.72-7.72 1.47 1.35L5.41 20z\"/>\n",
              "  </svg>\n",
              "      </button>\n",
              "      \n",
              "  <style>\n",
              "    .colab-df-container {\n",
              "      display:flex;\n",
              "      flex-wrap:wrap;\n",
              "      gap: 12px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert {\n",
              "      background-color: #E8F0FE;\n",
              "      border: none;\n",
              "      border-radius: 50%;\n",
              "      cursor: pointer;\n",
              "      display: none;\n",
              "      fill: #1967D2;\n",
              "      height: 32px;\n",
              "      padding: 0 0 0 0;\n",
              "      width: 32px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert:hover {\n",
              "      background-color: #E2EBFA;\n",
              "      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "      fill: #174EA6;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert {\n",
              "      background-color: #3B4455;\n",
              "      fill: #D2E3FC;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert:hover {\n",
              "      background-color: #434B5C;\n",
              "      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "      fill: #FFFFFF;\n",
              "    }\n",
              "  </style>\n",
              "\n",
              "      <script>\n",
              "        const buttonEl =\n",
              "          document.querySelector('#df-c492b6e2-c2a7-4d00-ad18-f7e6d77fff78 button.colab-df-convert');\n",
              "        buttonEl.style.display =\n",
              "          google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "        async function convertToInteractive(key) {\n",
              "          const element = document.querySelector('#df-c492b6e2-c2a7-4d00-ad18-f7e6d77fff78');\n",
              "          const dataTable =\n",
              "            await google.colab.kernel.invokeFunction('convertToInteractive',\n",
              "                                                     [key], {});\n",
              "          if (!dataTable) return;\n",
              "\n",
              "          const docLinkHtml = 'Like what you see? Visit the ' +\n",
              "            '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
              "            + ' to learn more about interactive tables.';\n",
              "          element.innerHTML = '';\n",
              "          dataTable['output_type'] = 'display_data';\n",
              "          await google.colab.output.renderOutput(dataTable, element);\n",
              "          const docLink = document.createElement('div');\n",
              "          docLink.innerHTML = docLinkHtml;\n",
              "          element.appendChild(docLink);\n",
              "        }\n",
              "      </script>\n",
              "    </div>\n",
              "  </div>\n",
              "  "
            ],
            "text/plain": [
              "   pmcid  paper.title  authors   year  number  figure.title\n",
              "0  False        False    False  False   False         False\n",
              "1  False        False    False  False   False         False\n",
              "2  False        False    False  False   False         False\n",
              "3  False        False    False  False   False         False\n",
              "4  False        False    False  False   False         False\n",
              "5  False        False    False  False   False         False\n",
              "6  False        False    False  False   False         False\n",
              "7  False        False    False  False   False         False\n",
              "8  False        False    False  False   False         False\n",
              "9  False        False    False  False   False         False"
            ]
          },
          "metadata": {},
          "execution_count": 44
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "new_df = df.dropna()#remove the rows if any cell in them is empty\n",
        "new_df"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 569
        },
        "id": "WCAsntZTbGFF",
        "outputId": "59c1385a-560a-4df7-aa15-cc4bc91fdfaa"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/html": [
              "\n",
              "  <div id=\"df-ae6bbc65-5e8c-496e-bdc7-8801b79a6046\">\n",
              "    <div class=\"colab-df-container\">\n",
              "      <div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>pmcid</th>\n",
              "      <th>paper.title</th>\n",
              "      <th>authors</th>\n",
              "      <th>year</th>\n",
              "      <th>number</th>\n",
              "      <th>figure.title</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>0</th>\n",
              "      <td>PMC6323817</td>\n",
              "      <td>Transcriptomic de novo analysis of pitaya (Hyl...</td>\n",
              "      <td>Min Xu, Cheng-L...</td>\n",
              "      <td>2019</td>\n",
              "      <td>Fig. 6</td>\n",
              "      <td>Differentially expressed genes (DEG...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>PMC6348685</td>\n",
              "      <td>Comparative transcriptome analysis of a long-t...</td>\n",
              "      <td>Luodong Huang, ...</td>\n",
              "      <td>2019</td>\n",
              "      <td>Fig. 4</td>\n",
              "      <td>Putative genes and their expression...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2</th>\n",
              "      <td>PMC6356271</td>\n",
              "      <td>Telomeres in Plants and Humans: Not So Differe...</td>\n",
              "      <td>Petra Procházko...</td>\n",
              "      <td>2019</td>\n",
              "      <td>Figure 2</td>\n",
              "      <td>Terminal telomeric complex</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3</th>\n",
              "      <td>PMC6360084</td>\n",
              "      <td>Identification, expression, and phylogenetic a...</td>\n",
              "      <td>Jipeng Mao, Zid...</td>\n",
              "      <td>2019</td>\n",
              "      <td>Figure 8</td>\n",
              "      <td>Identification, expression, and phy...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4</th>\n",
              "      <td>PMC6374618</td>\n",
              "      <td>Agarwood Induction: Current Developments and F...</td>\n",
              "      <td>Cheng Seng CS T...</td>\n",
              "      <td>2019</td>\n",
              "      <td>FIGURE 4</td>\n",
              "      <td>Schematic relationships between the...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>5</th>\n",
              "      <td>PMC6394206</td>\n",
              "      <td>A chromosomal-scale genome assembly of Tectona...</td>\n",
              "      <td>Dongyan Zhao, J...</td>\n",
              "      <td>2019</td>\n",
              "      <td>Figure 5:</td>\n",
              "      <td>Proposed diterpene pathway based on...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>6</th>\n",
              "      <td>PMC6412517</td>\n",
              "      <td>New Insights on Arabidopsis thaliana Root Adap...</td>\n",
              "      <td>Inmaculada Cole...</td>\n",
              "      <td>2019</td>\n",
              "      <td>Figure 2</td>\n",
              "      <td>Induction of secondary metabolism i...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>7</th>\n",
              "      <td>PMC6423058</td>\n",
              "      <td>Why Algae Release Volatile Organic Compounds—T...</td>\n",
              "      <td>Zhaojiang Zuo</td>\n",
              "      <td>2019</td>\n",
              "      <td>FIGURE 2</td>\n",
              "      <td>Pathway of terpene synthesis</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>8</th>\n",
              "      <td>PMC6437448</td>\n",
              "      <td>Production of ginsenoside aglycone (protopanax...</td>\n",
              "      <td>Yu Shin YS Gwak...</td>\n",
              "      <td>2019</td>\n",
              "      <td>Fig. 1</td>\n",
              "      <td>Biosynthetic pathway engineering fo...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>9</th>\n",
              "      <td>PMC6441073</td>\n",
              "      <td>Transcriptional response of grapevine to infec...</td>\n",
              "      <td>Wei Zhang, Jiye...</td>\n",
              "      <td>2019</td>\n",
              "      <td>Figure 3</td>\n",
              "      <td>DEGs involved in the plant–pathogen...</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "</div>\n",
              "      <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-ae6bbc65-5e8c-496e-bdc7-8801b79a6046')\"\n",
              "              title=\"Convert this dataframe to an interactive table.\"\n",
              "              style=\"display:none;\">\n",
              "        \n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "       width=\"24px\">\n",
              "    <path d=\"M0 0h24v24H0V0z\" fill=\"none\"/>\n",
              "    <path d=\"M18.56 5.44l.94 2.06.94-2.06 2.06-.94-2.06-.94-.94-2.06-.94 2.06-2.06.94zm-11 1L8.5 8.5l.94-2.06 2.06-.94-2.06-.94L8.5 2.5l-.94 2.06-2.06.94zm10 10l.94 2.06.94-2.06 2.06-.94-2.06-.94-.94-2.06-.94 2.06-2.06.94z\"/><path d=\"M17.41 7.96l-1.37-1.37c-.4-.4-.92-.59-1.43-.59-.52 0-1.04.2-1.43.59L10.3 9.45l-7.72 7.72c-.78.78-.78 2.05 0 2.83L4 21.41c.39.39.9.59 1.41.59.51 0 1.02-.2 1.41-.59l7.78-7.78 2.81-2.81c.8-.78.8-2.07 0-2.86zM5.41 20L4 18.59l7.72-7.72 1.47 1.35L5.41 20z\"/>\n",
              "  </svg>\n",
              "      </button>\n",
              "      \n",
              "  <style>\n",
              "    .colab-df-container {\n",
              "      display:flex;\n",
              "      flex-wrap:wrap;\n",
              "      gap: 12px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert {\n",
              "      background-color: #E8F0FE;\n",
              "      border: none;\n",
              "      border-radius: 50%;\n",
              "      cursor: pointer;\n",
              "      display: none;\n",
              "      fill: #1967D2;\n",
              "      height: 32px;\n",
              "      padding: 0 0 0 0;\n",
              "      width: 32px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert:hover {\n",
              "      background-color: #E2EBFA;\n",
              "      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "      fill: #174EA6;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert {\n",
              "      background-color: #3B4455;\n",
              "      fill: #D2E3FC;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert:hover {\n",
              "      background-color: #434B5C;\n",
              "      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "      fill: #FFFFFF;\n",
              "    }\n",
              "  </style>\n",
              "\n",
              "      <script>\n",
              "        const buttonEl =\n",
              "          document.querySelector('#df-ae6bbc65-5e8c-496e-bdc7-8801b79a6046 button.colab-df-convert');\n",
              "        buttonEl.style.display =\n",
              "          google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "        async function convertToInteractive(key) {\n",
              "          const element = document.querySelector('#df-ae6bbc65-5e8c-496e-bdc7-8801b79a6046');\n",
              "          const dataTable =\n",
              "            await google.colab.kernel.invokeFunction('convertToInteractive',\n",
              "                                                     [key], {});\n",
              "          if (!dataTable) return;\n",
              "\n",
              "          const docLinkHtml = 'Like what you see? Visit the ' +\n",
              "            '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
              "            + ' to learn more about interactive tables.';\n",
              "          element.innerHTML = '';\n",
              "          dataTable['output_type'] = 'display_data';\n",
              "          await google.colab.output.renderOutput(dataTable, element);\n",
              "          const docLink = document.createElement('div');\n",
              "          docLink.innerHTML = docLinkHtml;\n",
              "          element.appendChild(docLink);\n",
              "        }\n",
              "      </script>\n",
              "    </div>\n",
              "  </div>\n",
              "  "
            ],
            "text/plain": [
              "        pmcid  ...                            figure.title\n",
              "0  PMC6323817  ...  Differentially expressed genes (DEG...\n",
              "1  PMC6348685  ...  Putative genes and their expression...\n",
              "2  PMC6356271  ...              Terminal telomeric complex\n",
              "3  PMC6360084  ...  Identification, expression, and phy...\n",
              "4  PMC6374618  ...  Schematic relationships between the...\n",
              "5  PMC6394206  ...  Proposed diterpene pathway based on...\n",
              "6  PMC6412517  ...  Induction of secondary metabolism i...\n",
              "7  PMC6423058  ...            Pathway of terpene synthesis\n",
              "8  PMC6437448  ...  Biosynthetic pathway engineering fo...\n",
              "9  PMC6441073  ...  DEGs involved in the plant–pathogen...\n",
              "\n",
              "[10 rows x 6 columns]"
            ]
          },
          "metadata": {},
          "execution_count": 47
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "df_filled = df.fillna(value=5) # filling the missing value with cetain vale here it is int\n",
        "df_filled"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 569
        },
        "id": "texEryfvbI3P",
        "outputId": "4bd225bb-173c-4ffb-80cc-660e2fde5fd0"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/html": [
              "\n",
              "  <div id=\"df-5f0d0da1-daf5-4318-aca6-a91e7657260e\">\n",
              "    <div class=\"colab-df-container\">\n",
              "      <div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>pmcid</th>\n",
              "      <th>paper.title</th>\n",
              "      <th>authors</th>\n",
              "      <th>year</th>\n",
              "      <th>number</th>\n",
              "      <th>figure.title</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>0</th>\n",
              "      <td>PMC6323817</td>\n",
              "      <td>Transcriptomic de novo analysis of pitaya (Hyl...</td>\n",
              "      <td>Min Xu, Cheng-L...</td>\n",
              "      <td>2019</td>\n",
              "      <td>Fig. 6</td>\n",
              "      <td>Differentially expressed genes (DEG...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>PMC6348685</td>\n",
              "      <td>Comparative transcriptome analysis of a long-t...</td>\n",
              "      <td>Luodong Huang, ...</td>\n",
              "      <td>2019</td>\n",
              "      <td>Fig. 4</td>\n",
              "      <td>Putative genes and their expression...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2</th>\n",
              "      <td>PMC6356271</td>\n",
              "      <td>Telomeres in Plants and Humans: Not So Differe...</td>\n",
              "      <td>Petra Procházko...</td>\n",
              "      <td>2019</td>\n",
              "      <td>Figure 2</td>\n",
              "      <td>Terminal telomeric complex</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3</th>\n",
              "      <td>PMC6360084</td>\n",
              "      <td>Identification, expression, and phylogenetic a...</td>\n",
              "      <td>Jipeng Mao, Zid...</td>\n",
              "      <td>2019</td>\n",
              "      <td>Figure 8</td>\n",
              "      <td>Identification, expression, and phy...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4</th>\n",
              "      <td>PMC6374618</td>\n",
              "      <td>Agarwood Induction: Current Developments and F...</td>\n",
              "      <td>Cheng Seng CS T...</td>\n",
              "      <td>2019</td>\n",
              "      <td>FIGURE 4</td>\n",
              "      <td>Schematic relationships between the...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>5</th>\n",
              "      <td>PMC6394206</td>\n",
              "      <td>A chromosomal-scale genome assembly of Tectona...</td>\n",
              "      <td>Dongyan Zhao, J...</td>\n",
              "      <td>2019</td>\n",
              "      <td>Figure 5:</td>\n",
              "      <td>Proposed diterpene pathway based on...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>6</th>\n",
              "      <td>PMC6412517</td>\n",
              "      <td>New Insights on Arabidopsis thaliana Root Adap...</td>\n",
              "      <td>Inmaculada Cole...</td>\n",
              "      <td>2019</td>\n",
              "      <td>Figure 2</td>\n",
              "      <td>Induction of secondary metabolism i...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>7</th>\n",
              "      <td>PMC6423058</td>\n",
              "      <td>Why Algae Release Volatile Organic Compounds—T...</td>\n",
              "      <td>Zhaojiang Zuo</td>\n",
              "      <td>2019</td>\n",
              "      <td>FIGURE 2</td>\n",
              "      <td>Pathway of terpene synthesis</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>8</th>\n",
              "      <td>PMC6437448</td>\n",
              "      <td>Production of ginsenoside aglycone (protopanax...</td>\n",
              "      <td>Yu Shin YS Gwak...</td>\n",
              "      <td>2019</td>\n",
              "      <td>Fig. 1</td>\n",
              "      <td>Biosynthetic pathway engineering fo...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>9</th>\n",
              "      <td>PMC6441073</td>\n",
              "      <td>Transcriptional response of grapevine to infec...</td>\n",
              "      <td>Wei Zhang, Jiye...</td>\n",
              "      <td>2019</td>\n",
              "      <td>Figure 3</td>\n",
              "      <td>DEGs involved in the plant–pathogen...</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "</div>\n",
              "      <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-5f0d0da1-daf5-4318-aca6-a91e7657260e')\"\n",
              "              title=\"Convert this dataframe to an interactive table.\"\n",
              "              style=\"display:none;\">\n",
              "        \n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "       width=\"24px\">\n",
              "    <path d=\"M0 0h24v24H0V0z\" fill=\"none\"/>\n",
              "    <path d=\"M18.56 5.44l.94 2.06.94-2.06 2.06-.94-2.06-.94-.94-2.06-.94 2.06-2.06.94zm-11 1L8.5 8.5l.94-2.06 2.06-.94-2.06-.94L8.5 2.5l-.94 2.06-2.06.94zm10 10l.94 2.06.94-2.06 2.06-.94-2.06-.94-.94-2.06-.94 2.06-2.06.94z\"/><path d=\"M17.41 7.96l-1.37-1.37c-.4-.4-.92-.59-1.43-.59-.52 0-1.04.2-1.43.59L10.3 9.45l-7.72 7.72c-.78.78-.78 2.05 0 2.83L4 21.41c.39.39.9.59 1.41.59.51 0 1.02-.2 1.41-.59l7.78-7.78 2.81-2.81c.8-.78.8-2.07 0-2.86zM5.41 20L4 18.59l7.72-7.72 1.47 1.35L5.41 20z\"/>\n",
              "  </svg>\n",
              "      </button>\n",
              "      \n",
              "  <style>\n",
              "    .colab-df-container {\n",
              "      display:flex;\n",
              "      flex-wrap:wrap;\n",
              "      gap: 12px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert {\n",
              "      background-color: #E8F0FE;\n",
              "      border: none;\n",
              "      border-radius: 50%;\n",
              "      cursor: pointer;\n",
              "      display: none;\n",
              "      fill: #1967D2;\n",
              "      height: 32px;\n",
              "      padding: 0 0 0 0;\n",
              "      width: 32px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert:hover {\n",
              "      background-color: #E2EBFA;\n",
              "      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "      fill: #174EA6;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert {\n",
              "      background-color: #3B4455;\n",
              "      fill: #D2E3FC;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert:hover {\n",
              "      background-color: #434B5C;\n",
              "      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "      fill: #FFFFFF;\n",
              "    }\n",
              "  </style>\n",
              "\n",
              "      <script>\n",
              "        const buttonEl =\n",
              "          document.querySelector('#df-5f0d0da1-daf5-4318-aca6-a91e7657260e button.colab-df-convert');\n",
              "        buttonEl.style.display =\n",
              "          google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "        async function convertToInteractive(key) {\n",
              "          const element = document.querySelector('#df-5f0d0da1-daf5-4318-aca6-a91e7657260e');\n",
              "          const dataTable =\n",
              "            await google.colab.kernel.invokeFunction('convertToInteractive',\n",
              "                                                     [key], {});\n",
              "          if (!dataTable) return;\n",
              "\n",
              "          const docLinkHtml = 'Like what you see? Visit the ' +\n",
              "            '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
              "            + ' to learn more about interactive tables.';\n",
              "          element.innerHTML = '';\n",
              "          dataTable['output_type'] = 'display_data';\n",
              "          await google.colab.output.renderOutput(dataTable, element);\n",
              "          const docLink = document.createElement('div');\n",
              "          docLink.innerHTML = docLinkHtml;\n",
              "          element.appendChild(docLink);\n",
              "        }\n",
              "      </script>\n",
              "    </div>\n",
              "  </div>\n",
              "  "
            ],
            "text/plain": [
              "        pmcid  ...                            figure.title\n",
              "0  PMC6323817  ...  Differentially expressed genes (DEG...\n",
              "1  PMC6348685  ...  Putative genes and their expression...\n",
              "2  PMC6356271  ...              Terminal telomeric complex\n",
              "3  PMC6360084  ...  Identification, expression, and phy...\n",
              "4  PMC6374618  ...  Schematic relationships between the...\n",
              "5  PMC6394206  ...  Proposed diterpene pathway based on...\n",
              "6  PMC6412517  ...  Induction of secondary metabolism i...\n",
              "7  PMC6423058  ...            Pathway of terpene synthesis\n",
              "8  PMC6437448  ...  Biosynthetic pathway engineering fo...\n",
              "9  PMC6441073  ...  DEGs involved in the plant–pathogen...\n",
              "\n",
              "[10 rows x 6 columns]"
            ]
          },
          "metadata": {},
          "execution_count": 48
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "#df.duplicates() # return the no of duplicates present in the csv\n",
        "df_no_duplicate= df.drop_duplicates(inplace = True) # remove duplicate elemments\n",
        "df_no_duplicate"
      ],
      "metadata": {
        "id": "nHXMKweHbPnk"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "source": [
        "it return None which means there are no duplicates in the csv"
      ],
      "metadata": {
        "id": "2sSgFnpIbymg"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "#if you want to update value in any cell follow the comment below:\n",
        "df.loc[7, 'longitude'] = 45 # here we are changing the value of "
      ],
      "metadata": {
        "id": "whOqrVZebRoO"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "source": [
        ""
      ],
      "metadata": {
        "id": "Oe36VHTybPFX"
      }
    }
  ]
}