<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">BMC Genomics</span><span class="iso-abbrev" title="iso-abbrev">BMC Genomics</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">BMC Genomics</span></div><span class="issn-epub" tagx="issn" title="issn-epub">1471-2164</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">BioMed Central</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7849092">7849092</a></span><span class="publisher-id" title="publisher-id">7339</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1186/s12864-020-07339-7">10.1186/s12864-020-07339-7</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Identification of genes associated with the biosynthesis of unsaturated fatty acid and oil accumulation in herbaceous peony â€˜Hangshaoâ€™ (<i>Paeonia lactiflora</i> â€˜Hangshaoâ€™) seeds based on transcriptome analysis</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meng</span><span class="given-names" tagx="given-names" title="given-names">Jia-Song</span></span><a href="#Aff1">1</a><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tang</span><span class="given-names" tagx="given-names" title="given-names">Yu-Han</span></span><a href="#Aff2">2</a><a href="#Aff3">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">Jing</span></span><a href="#Aff1">1</a><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">Da-Qiu</span></span><a href="#Aff1">1</a><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Ke-Liang</span></span><a href="#Aff1">1</a><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tao</span><span class="given-names" tagx="given-names" title="given-names">Jun</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">taojun@yzu.edu.cn</span></span><a href="#Aff1">1</a><a href="#Aff2">2</a></span><span class="citation_author_institution" id="Aff1">[1], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.268415.c</span><span class="institution" title="institution">College of Horticulture and Plant Protection, </span><span class="institution" title="institution">Yangzhou University, </span></div></span><span class="citation_author_institution" id="Aff2">[2], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.268415.c</span><span class="institution" title="institution">Joint International Research Laboratory of Agriculture and Agri-Product Safety, the Ministry of Education of China, </span><span class="institution" title="institution">Yangzhou University, </span></div></span><span class="citation_author_institution" id="Aff3">[3], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.268415.c</span><span class="institution" title="institution">College of Animal Science and Technology, </span><span class="institution" title="institution">Yangzhou University, </span></div></span></div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2021-2-2</span></span><span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2021-2-2</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2021</span></span><span class="volume" tagx="volume" title="volume">22</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">94</span><span class="history" title="history"><span class="received" title="received">received: 2020-6-11</span><span class="accepted" title="accepted">accepted: 2020-12-22</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , </span><span class="license" title="license"><span class="license-p" title="license-p"><b>Open Access</b>This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>. The Creative Commons Public Domain Dedication waiver (<a href="http://creativecommons.org/publicdomain/zero/1.0/">http://creativecommons.org/publicdomain/zero/1.0/</a>) applies to the data made available in this article, unless otherwise stated in a credit line to the data.</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="background" title="sec"> <div class="title" tagx="title" title="title">
Background</div> <p id="Par1"><i>Paeonia lactiflora</i> â€˜Hangshaoâ€™ is widely cultivated in China as a traditional Chinese medicine â€˜Radix Paeoniae Albaâ€™. Due to the abundant unsaturated fatty acids in its seed, it can also be regarded as a new oilseed plant. However, the process of the biosynthesis of unsaturated fatty acids in it has remained unknown. Therefore, transcriptome analysis is helpful to better understand the underlying molecular mechanisms.</p> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <p id="Par2">Five main fatty acids were detected, including stearic acid, palmitic acid, oleic acid, linoleic acid and Î±-linolenic acid, and their absolute contents first increased and then decreased during seed development. A total of 150,156 unigenes were obtained by transcriptome sequencing. There were 15,005 unigenes annotated in the seven functional databases, including NR, NT, GO, KOG, KEGG, Swiss-Prot and InterPro. Based on the KEGG database, 1766 unigenes were annotated in the lipid metabolism. There were 4635, 12,304, and 18,291 DEGs in Group I (60 vs 30 DAF), Group II (90 vs 60 DAF) and Group III (90 vs 30 DAF), respectively. A total of 1480 DEGs were detected in the intersection of the three groups. In 14 KEGG pathways of lipid metabolism, 503 DEGs were found, belonging to 111 enzymes. We screened out 123 DEGs involved in fatty acid biosynthesis (39 DEGs), fatty acid elongation (33 DEGs), biosynthesis of unsaturated fatty acid (24 DEGs), TAG assembly (17 DEGs) and lipid storage (10 DEGs). Furthermore, qRT-PCR was used to analyze the expression patterns of 16 genes, including <i>BBCP</i>, <i>BC</i>, <i>MCAT</i>, <i>KASIII</i>, <i>KASII</i>, <i>FATA</i>, <i>FATB</i>, <i>KCR</i>, <i>SAD</i>, <i>FAD2</i>, <i>FAD3</i>, <i>FAD7</i>, <i>GPAT</i>, <i>DGAT</i>, <i>OLE</i> and <i>CLO</i>, most of which showed the highest expression at 45 DAF, except for <i>DGAT</i>, <i>OLE</i> and <i>CLO</i>, which showed the highest expression at 75 DAF.</p> </div> <div class="conclusions" title="sec"> <div class="title" tagx="title" title="title">
Conclusions</div> <p id="Par3">We predicted that <i>MCAT</i>, <i>KASIII</i>, <i>FATA</i>, <i>SAD</i>, <i>FAD2</i>, <i>FAD3</i>, <i>DGAT</i> and <i>OLE</i> were the key genes in the unsaturated fatty acid biosynthesis and oil accumulation in herbaceous peony seed. This study provides the first comprehensive genomic resources characterizing herbaceous peony seed gene expression at the transcriptional level. These data lay the foundation for elucidating the molecular mechanisms of fatty acid biosynthesis and oil accumulation for herbaceous peony.</p> </div> <div class="supplementaryinformation" title="sec"> <div class="title" tagx="title" title="title">
Supplementary Information</div> <p>The online version contains supplementary material available at 10.1186/s12864-020-07339-7.</p> </div> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Keywords</div> <div class="kwd" title="kwd"><i>Paeonia lactiflora</i> â€˜Hangshaoâ€™</div> <div class="kwd" title="kwd">
Fatty acid biosynthesis</div> <div class="kwd" title="kwd">
Transcriptome</div> <div class="kwd" title="kwd">
Differentially expressed gene</div> <div class="kwd" title="kwd">
qRT-PCR</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/501100001809</span><span class="institution" title="institution">National Natural Science Foundation of China</span></div> </div><span class="award-id" tagx="award-id" title="award-id">32071813</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meng</span><span class="given-names" tagx="given-names" title="given-names">Jia-Song</span></span></span></div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="institution" title="institution">Three-Side Innovation Projects for Agriculture in Jiangsu Province</span></div><span class="award-id" tagx="award-id" title="award-id">SXGC[2017]297</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tao</span><span class="given-names" tagx="given-names" title="given-names">Jun</span></span></span></div> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="issue-copyright-statement" value="Â© The Author(s) 2021" /> </div> </div> </div> <div class="body" title="body"> <div class="background" title="sec"> <div class="title" tagx="title" title="title">
Background</div> <p id="Par51">As a well-known Chinese traditional flower, herbaceous peony (<i>Paeonia lactiflora</i> Pall.) was cultivated more than 4000â€‰years. Owing to the beauty of the color, shape and fragrance, it is regarded as â€˜the Prime-Minister of flowersâ€™ and is known as the â€˜two superb of flowersâ€™ with tree peony (<i>Paeonia suffruticosa</i> Andr.), which is regarded as â€˜the King of flowersâ€™. Herbaceous peony is cultivated as a medicinal and ornamental plant in China. To date, more than 500 cultivars are distributed in different regions of China [<a href="#CR1">1</a>]. The dried roots of two herbaceous peony species are the main raw materials for the traditional Chinese medicine â€˜Radix Paeoniae Albaâ€™ and â€˜Radix Paeoniae Rubraâ€™ [<a href="#CR2">2</a>, <a href="#CR3">3</a>], which have antithrombic [<a href="#CR4">4</a>], antitumor [<a href="#CR5">5</a>], and anti-posttraumatic stress disorder functions [<a href="#CR6">6</a>]; alleviate inflammation [<a href="#CR7">7</a>]; remove pathogenic heat from the blood; treat blood stasis; relieve pain [<a href="#CR8">8</a>], and so on. As an important medicinal plant, herbaceous peony â€˜Hangshaoâ€™, the root of which is the traditional Chinese medicine â€˜Radix Paeoniae Albaâ€™, is mainly cultivated in Zhejiang Province and widely cultivated in China [<a href="#CR9">9</a>]. In addition, it has a single lobe and a higher seed-setting rate. However, the tremendous number of seeds from herbaceous peony â€˜Hangshaoâ€™ are mainly wasted every year, except the handful of seeds that are used for propagation [<a href="#CR10">10</a>]. Previous studies have shown that the seed oil content of â€˜Hangshaoâ€™ is approximately 25%, with more than 90% unsaturated fatty acids (UFAs) [<a href="#CR11">11</a>]. Therefore, herbaceous peony â€˜Hangshaoâ€™ has the potential to be cultivated as a multifunctional plant, not only as an ornamental, medicinal plant but also as an edible oil resource.</p> <p id="Par52">Herbaceous peony â€˜Hangshaoâ€™ seed oil predominantly contains five fatty acids: oleic acid (C18:1<sup>Î"9</sup>, OA), linoleic acid (C18:2<sup>Î"9,12</sup>, LA), Î±-linolenic acid (C18:3<sup>Î"9,12,15</sup>, ALA), palmitic acid (C16:0, PA) and stearic acid (C18:0, SA) [<a href="#CR10">10</a>, <a href="#CR11">11</a>]. OA, LA and ALA are classified as UFAs, which include monounsaturated fatty acids (MUFAs) and polyunsaturated fatty acids (PUFAs). OA, which can improve diastolic heart function, is a MUFA [<a href="#CR12">12</a>]. LA and ALA are PUFAs, which can reduce thrombosis, decrease cardiovascular events and prevent cancer [<a href="#CR13">13</a>]. According to the location of the last double carbon bond relative to the terminal methyl end of the molecule, PUFAs can be divided into the omega-3 (n-3) series and omega-6 (n-6) series. ALA, an important omega-3 PUFA, is an essential fatty acid that cannot be synthesized in the human body and must be obtained via dietary intake; it is the precursor of docosahexaenoic acid (DHA) and eicosapentaenoic acid (EPA), which are essential for the growth and development of the brain and retina [<a href="#CR14">14</a>]. Additionally, LA, an important omega-6 PUFA, is also an essential fatty acid that cannot be produced by the human body; it is the precursor of the omega-6 series of fatty acids such as Î³-linolenic acid (C18:3<sup>Î"6,9,12</sup>, GLA) and arachidonic acid (C18:4<sup>Î"5,8,11,14</sup>, AA) [<a href="#CR15">15</a>]. Nevertheless, high intake of omega-6 PUFAs with low intake of omega-3 PUFAs might promote the pathogenesis of many diseases, including cardiovascular disease, cancer, and inflammatory and autoimmune diseases [<a href="#CR16">16</a>]. Therefore, the Food and Agriculture Organization (FAO) of the United Nations and the World Health Organization (WHO) suggest that the omega-6/3 ratio should be lower than 5:1 [<a href="#CR17">17</a>].</p> <p id="Par53">Previous research has shown that the ratio of omega-6/omega-3 is approximately 1.5:1 in herbaceous peony â€˜Hangshaoâ€™ mature seed oil. However, the ratio of omega-6/omega-3 increases during seed development because the relative content of ALA decreases gradually and the relative content of LA slightly increases [<a href="#CR10">10</a>, <a href="#CR11">11</a>]. Nevertheless, very little research has examined fatty acid accumulation at the molecular level for herbaceous peony seed oil. Fatty acid biosynthesis in plants occurs via a very complicated biological pathway regulated by various enzymes. RNA sequencing (RNA-Seq) has provided a convenient and rapid method for finding key genes in fatty acid biosynthesis and unsaturated fatty acid biosynthesis at the molecular level. De novo transcriptome sequencing had been applied to identify and profile gene expression differences of some herb oilseed plants such as soybean [<a href="#CR18">18</a>], peanut [<a href="#CR19">19</a>], rape [<a href="#CR20">20</a>], and sunflower [<a href="#CR21">21</a>] and some tree oilseed plants such as tree peony [<a href="#CR17">17</a>], olive [<a href="#CR22">22</a>], tea-oil camellia [<a href="#CR23">23</a>], and <i>Hiptage benghalensis</i> [<a href="#CR24">24</a>]. However, research on herbaceous peony has mainly concentrated on thermotolerant-related differentially expressed genes [<a href="#CR25">25</a>], codon usage patterns [<a href="#CR26">26</a>] and anthocyanin biosynthetic genes [<a href="#CR27">27</a>] based on transcriptome sequencing. The fatty acid metabolism of herbaceous peony has remained unknown. Moreover, genetic control of the unsaturated fatty acid accumulation of herbaceous peony is unexplored.</p> <p id="Par54">In this study, we measured the oil content and absolute content of five main fatty acids of herbaceous peony â€˜Hangshaoâ€™ seeds at five developmental stages (30, 45, 60, 75 and 90â€‰days after flowering, DAF) by gas chromatography-mass spectrometry (GC-MS). We identified differentially expressed genes (DEGs) related to the biosynthesis of unsaturated fatty acids and oil accumulation. Furthermore, we validated the expression of 16 DEGs using quantitative real-time polymerase chain reaction (qRT-PCR). Our results reveal the transcriptomic aspects of unsaturated fatty acid biosynthesis and oil accumulation in herbaceous peony seed, and provide a useful reference for fatty acid biosynthesis studies of other plant species.</p> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <div class="changeinoilcontentandabsolutecontentsoffivefattyacids" title="sec"> <div class="title" tagx="title" title="title">
Change in oil content and absolute contents of five fatty acids</div> <p id="Par55">The oil content of herbaceous peony â€˜Hangshaoâ€™ seeds at five different stages was studied. It was approximately 5.06%Â±0.06% at 30 DAF, and it increased constantly, reaching a maximum of 20.92%Â±0.25% at 75 DAF and then slightly decreasing at 90 DAF (Fig.Â <a href="#Fig1">1</a>b). Five main fatty acids, PA, SA, OA, LA and ALA, were detected by gas chromatography-mass spectrometry (GC-MS) analysis of the fatty acid components of the seeds from the herbaceous peony â€˜Hangshaoâ€™. Through quantitative analysis of the contents of five fatty acids (Fig. <a href="#Fig1">1</a>c), it was found that the content of ALA, LA and PA showed a trend toward first rising and then falling during seed development, and the maximum value appeared in the S4 period. In contrast, the content of OA and SA increased consistently during seed development, and the maximum value appeared in the S5 period. However, the absolute contents of the five fatty acids in the five periods showed that the ALA content was highest and the SA content was lowest, and the contents followed the order ALA &amp;gt; LA &amp;gt; OA &amp;gt; PA &amp;gt; SA. The total contents of the five fatty acids from S1 to S5 seeds were 4.49â€‰gÂ·100â€‰g<sup>âˆ’â€‰1</sup>, 12.17â€‰gÂ·100â€‰g<sup>âˆ’â€‰1</sup>, 21.19â€‰gÂ·100â€‰g<sup>âˆ’â€‰1</sup>, 26.11â€‰gÂ·100â€‰g<sup>âˆ’â€‰1</sup> and 24.75â€‰gÂ·100â€‰g<sup>âˆ’â€‰1</sup>. Moreover, the content of ALA exceeded 10â€‰gÂ·100â€‰g<sup>âˆ’â€‰1</sup> in both the S4 and S5 stages. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 1</span></a></div>   <p>Oil content and absolute content of five main fatty acids in seeds of <i>Paeonia lactiflora</i> â€˜Hangshaoâ€™. <b>a</b> Seeds in the collecting period; <b>b</b> Oil content in the developing seed; <b>c</b> Absolute content of five main fatty acids in the developing seed. DAF, days after flowering</p>   </div> <p /> </div> <div class="sequencingreadfilteringanddenovoassembly" title="sec"> <div class="title" tagx="title" title="title">
Sequencing read filtering and de novo assembly</div> <p id="Par56">Nine cDNA libraries containing three biological repeats were created from three stages of developing herbaceous peony â€˜Hangshaoâ€™ seeds (the early, middle and late stages of seed development, that is 30, 60, and 90 DAF, respectively) collected from the same strain and sequenced using the Illumina high-throughput sequencing platform, to explore the molecular mechanism of fatty acid biosynthesis and oil accumulation. A total of 100.07â€‰G paired-end raw reads were produced by transcriptome sequencing. After removing low-quality, adapter-contaminated and high-content unknown base (N) reads, the percentage of clean reads for nine samples was more than 82% (AdditionalÂ fileÂ <a href="#MOESM1">1</a>: Table S1). After read filtering and assembly, a set of 150,156 unigenes was obtained, with a total length, average length, and N50 and GC content of 154,674,160â€‰bp, 1030â€‰bp, 1825â€‰bp and 39.97%, respectively (AdditionalÂ fileÂ <a href="#MOESM2">2</a>: Table S2). The unigene length ranged from 300 to 3000â€‰nt, and the unigene number gradually decreased without obvious separation as the size of the sequences increased (AdditionalÂ fileÂ <a href="#MOESM3">3</a>: Figure S1).</p> </div> <div class="genefunctionalannotation" title="sec"> <div class="title" tagx="title" title="title">
Gene functional annotation</div> <p id="Par57">After assembly, all the unigenes were subjected to BLASTx analysis against seven public databases, including NR, NT, GO, KOG, KEGG, Swiss-Prot and InterPro (TableÂ <a href="#Tab1">1</a>). A total of 73,399 (48.88%) unigenes were annotated in at least one database, whereas 15,005 (9.99%) unigenes were annotated in all databases. The highest annotation rate was obtained in the NR database, which assigned 67,096 (44.68%) unigenes. A total of 45,488 (30.29%), 42,533 (28.33%) and 48,907 (32.57%) unigenes were annotated using the NT, Swiss-Prot and InterPro database, respectively. <div class="table-wrap_UNKNOWN" id="Tab1"><span class="label" tagx="label" title="label">Table 1</span> </div></p><p>Functional annotation of unigenes for seeds of <i>Paeonia lactiflora</i> â€˜Hangshaoâ€™ in a public functional database</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th>Values</th> <th>Number</th> <th>Percentage</th> </tr> </thead> <tbody> <tr> <td>Total unigenes</td> <td>150,156</td> <td>100%</td> </tr> <tr> <td>Nr</td> <td>67,096</td> <td>44.68%</td> </tr> <tr> <td>Nt</td> <td>45,488</td> <td>30.29%</td> </tr> <tr> <td>Swissprot</td> <td>42,533</td> <td>28.33%</td> </tr> <tr> <td>KEGG</td> <td>49,630</td> <td>33.05%</td> </tr> <tr> <td>KOG</td> <td>51,612</td> <td>34.37%</td> </tr> <tr> <td>Interpro</td> <td>48,907</td> <td>32.57%</td> </tr> <tr> <td>GO</td> <td>29,413</td> <td>19.59%</td> </tr> <tr> <td>Intersection</td> <td>15,005</td> <td>9.99%</td> </tr> <tr> <td>Overall</td> <td>73,399</td> <td>48.88%</td> </tr> </tbody> </table>  <p /> <p id="Par58">Based on the NR database function annotation result, the ratio of different species unigene annotation was calculated, and the distribution map was drawn (AdditionalÂ fileÂ <a href="#MOESM4">4</a>:Figure S2A). The unigenes of herbaceous peony â€˜Hangshaoâ€™ seeds were matched to species such as <i>Vitis vinifera</i> (32.79%), <i>Nelumbo nucifera</i> (5.84%), <i>Theobroma cacao</i> (5.45%) and <i>Jatropha curcas</i> (3.66%). In addition, 51,612 (34.37%) unigenes were annotated with 25 KOG terms (Additional file <a href="#MOESM4">4</a>: Figure S2B). Interestingly, 1868 unigenes were annotated in the â€˜Lipid transport and metabolismâ€™ category. Moreover, 29,413 (19.59%) unigenes were annotated with 55 GO terms, including 24 terms related to biological processes, 17 terms related to cellular component and 14 terms related to molecular functions (Additional file <a href="#MOESM4">4</a>:Figure S2C). To further understand the biological functions and interactions of genes, they were also classified into metabolic pathways using KEGG. A total of 49,630 (33.05%) unigenes were assigned to 21 subcategories divided into six categories, including cellular processes, environmental information processing, genetic information processing, human diseases, metabolism and organismal systems (Additional file <a href="#MOESM4">4</a>:Figure S2D). Among the KEGG pathways, the top three subcategories were â€˜global and overview mapsâ€™ (12,390 unigenes), metabolismâ€™ (4905 unigenes) and â€˜translationâ€™ (4321 unigenes). In contrast, 1766 unigenes were annotated to â€˜lipid metabolismâ€™, which will help us understand the fatty acid and oil accumulation in herbaceous peony seeds.</p> </div> <div class="analysisofdifferentiallyexpressedgenesfordevelopmentalherbaceouspeonyâ€˜hangshaoâ€™seeds" title="sec"> <div class="title" tagx="title" title="title">
Analysis of differentially expressed genes for developmental herbaceous peony â€˜Hangshaoâ€™ seeds</div> <p id="Par59">To understand the patterns of seed development in detail and identify key genes in fatty acid biosynthesis, differentially expressed genes (DEGs) were analyzed by comparing the FPKM value of unigenes between the different developmental seeds of the herbaceous peony â€˜Hangshaoâ€™. Based on the gene expression level, DEGs were identified between different groups (Fig.Â <a href="#Fig2">2</a>a). Seeds at 30 DAF served as the control, and 4635 and 18,291 DEGs were identified at 60 and 90 DAF, respectively. Additionally, seeds at 60 DAF served as the control, and 12,304 DEGs were also identified at 90 DAF. There were 2410, 4210 and 7385 upregulated unigenes and 2225, 8094 and 10,906 downregulated unigenes at 30 vs 60 DAF (Group I), 60 vs 90 DAF (Group II) and 30 vs 90 DAF (Group III), respectively. Two hierarchical cluster analyses were conducted based on the FPKM value of these DEGs for the three groups (Fig. <a href="#Fig2">2</a>b). The DEG intersection (Inter) and DEG union of the three different groups were clustered into three clusters. DEGs in the same cluster had identical or similar expression patterns during the seed developmental stage. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 2</span></a></div>   <p>Analysis of DEGs in seeds of <i>Paeonia lactiflora</i> â€˜Hangshaoâ€™<b>. a</b> Gene analysis of DEGs in different stages; <b>b</b> Cluster analysis of DEGs in different stages</p>   </div> <p /> <p id="Par60">The DEGs at different developmental stages were integrated and mapped into the GO database (AdditionalÂ fileÂ <a href="#MOESM5">5</a>: Table S3 and AdditionalÂ fileÂ <a href="#MOESM6">6</a>: Figure S3). Overall, the number of DEGs annotated to the same GO term between the three different groups increased with the developmental stage. The DEGs in Group III were annotated to 23 biological processes, 17 cellular components and 14 molecular functions. In contrast, the DEGs in Group II were annotated to 21 biological processes (except for â€˜biological adhesionâ€™ and â€˜locomotionâ€™), 17 cellular components and 13 molecular functions (except for â€˜metallochaperone activityâ€™). Additionally, the DEGs in Group I were annotated to 21 biological processes (except for â€˜biological adhesionâ€™ and â€˜rhythmic processâ€™), 14 cellular components (except for â€˜nucleoidâ€™, â€˜virionâ€™ and â€˜virion partâ€™) and 12 molecular functions (except for â€˜metallochaperone activityâ€™ and â€˜protein tagâ€™).</p> <p id="Par61">DEGs in the three different groups were annotated to five KEGG categories (including cellular processes, environmental information processing, genetic information processing, metabolism and organismal systems) and 19 KEGG subcategories (AdditionalÂ fileÂ <a href="#MOESM7">7</a>: Table S4). The numbers of DEGs annotated to the KEGG pathway were 1698, 4321 and 6035 for Group I, Group II and Group III, respectively (AdditionalÂ fileÂ <a href="#MOESM8">8</a>: Table S5). Further analysis of the KEGG annotation showed that 28, 43 and 41 pathways were significantly enriched for Group I, Group II and Group III, respectively (AdditionalÂ fileÂ <a href="#MOESM9">9</a>: Table S6). Among the top 25 enriched pathways in each group, 13 pathways were significantly enriched in all three groups (Fig.Â <a href="#Fig3">3</a>a). Among them, glycerolipid metabolism, arachidonic acid metabolism and sphingolipid biosynthesis belong to lipid metabolism. Additionally, 12 pathways were significantly enriched in two groups, including cutin, suberine and wax biosynthesis and steroid biosynthesis, which belong to lipid metabolism. Moreover, 12 pathways were significantly enriched in only one group, including fatty acid biosynthesis and linoleic acid metabolism. Additionally, 14 pathways were annotated to lipid metabolism (Fig. <a href="#Fig3">3</a>b and AdditionalÂ fileÂ <a href="#MOESM10">10</a>:Table S7). Glycerolipid metabolism, fatty acid biosynthesis, arachidonic acid metabolism and sphingolipid metabolism were significantly enriched in all three groups. The number of DEGs in the 4 KEGG pathways mentioned above increased with seed development. Despite no significant enrichment of DEGs in the KEGG pathway of unsaturated fatty acid biosynthesis, the number of DEGs also increased with seed development. These results provide clues regarding the identification of key genes involved in the biosynthesis of unsaturated fatty acids. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 3</span></a></div>   <p>Analysis of DEGs in significantly enriched KEGG pathways. <b>a</b> Number of DEGs in the top 25 significantly enriched KEGG pathways; <b>b</b> Number of DEGs in the l4 lipid metabolism KEGG pathways; * represents a significantly enriched KEGG pathway</p>   </div> <p /> <p id="Par62">In addition, 1480 DEGs were obtained from the intersection analysis of the three groups (Fig.Â <a href="#Fig4">4</a>a), and these 1480 DEGs were annotated to 17 biological processes, 14 cellular components and 12 molecular functions in the GO database (Fig. <a href="#Fig4">4</a>b). In biological process, the largest category was â€˜metabolic processâ€™, followed by â€˜cellular processâ€™ and â€˜single-organism processâ€™. In cellular component, the largest category was â€˜membraneâ€™, followed by â€˜cellâ€™ and â€˜cell partâ€™. In molecular function, the two largest categories were â€˜catalytic activityâ€™ and â€˜bindingâ€™. Moreover, the 1480 DEGs were annotated to 19 KEGG pathways (Fig. <a href="#Fig4">4</a>c). The largest category was â€˜global and overview mapsâ€™, followed by â€˜carbohydrate metabolismâ€™ and â€˜lipid metabolismâ€™. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 4</span></a></div>   <p>Analysis of DEGs obtained from the intersection of three groups. <b>a</b> Venn diagram between DEGs from three groups; <b>b</b> Functional distribution of intersection DEGs annotated by GO; <b>c</b> Functional distribution of intersection DEGs annotated by KEGG</p>   </div> <p /> </div> <div class="analysisofdegsforlipidmetabolism" title="sec"> <div class="title" tagx="title" title="title">
Analysis of DEGs for lipid metabolism</div> <p id="Par63">The analysis of DEGs in 14 KEGG pathways annotated to lipid metabolism found 503 DEGs (AdditionalÂ fileÂ <a href="#MOESM11">11</a>: Table S8) belonging to 111 enzymes. Then, based on the standard FPKM and LOG2 values of different replicates, 111 DEGs were selected for heat map and cluster analysis (AdditionalÂ fileÂ <a href="#MOESM12">12</a>: Table S9 and Fig.Â <a href="#Fig5">5</a>). The results showed that 111 DEGs for lipid metabolism were clustered into four clusters (Fig. <a href="#Fig5">5</a>a). There were 11, 26, 32 and 42 DEGs in Cluster I, II, III and IV, respectively. The DEGs in Cluster I and II were highly expressed, whereas those in Cluster III and IV were expressed at a relatively low level. Additionally, the average log2 FPKM values for the 11 DEGs in Cluster I and the 42 DEGs in Cluster IV showed an increasing pattern; however, those of the 26 DEGs in Cluster II and 32 DEGs in Cluster III showed a declining pattern (Fig. <a href="#Fig5">5</a>b). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 5</span></a></div>   <p>Cluster analysis of DEGs in the lipid metabolism pathway of <i>Paeonia lactiflora</i> â€˜Hangshaoâ€™ seeds. <b>a</b> Hierarchical clustering dendrogram of DEGs in lipid metabolism. The red highlighting indicates genes that were highly expressed, whereas the blue highlighting indicates genes with low expression. <b>b</b> The four cluster groups with different gene expression patterns</p>   </div> <p /> </div> <div class="analysisofdegsforfattyacidbiosynthesis,elongationanddesaturation" title="sec"> <div class="title" tagx="title" title="title">
Analysis of DEGs for fatty acid biosynthesis, elongation and desaturation</div> <p id="Par64">To further uncover the key genes associated with the biosynthesis of unsaturated fatty acids and oil accumulation in the developmental herbaceous peony â€˜Hangshaoâ€™ seed, DEGs in five pathways were analyzed, including fatty acid biosynthesis, fatty acid elongation, biosynthesis of unsaturated fatty acids, triacylglycerol (TAG) biosynthesis and lipid storage (Fig.Â <a href="#Fig6">6</a>). First, the DEGs with a log2 value of the FPKM ration in any two periods greater than 1 or less than âˆ’â€‰1 were selected. Then, sequences of the DEGs selected above belonging to the same enzyme were compared using Bioxm software. Last, those DEGs with the same sequences were removed. Finally, we identified 123 DEGs involved in fatty acid biosynthesis (39 DEGs), fatty acid elongation (33 DEGs), biosynthesis of unsaturated fatty acids (24 DEGs), TAG biosynthesis (17 DEGs) and lipid storage (10 DEGs) (AdditionalÂ fileÂ <a href="#MOESM13">13</a>: Table S10 and Fig. <a href="#Fig6">6</a>). The expression levels of these DEGs at 30, 60 and 90 DAF are summarized in AdditionalÂ fileÂ <a href="#MOESM13">13</a>. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 6</span></a></div>   <p>Proposed gene networks involved in unsaturated fatty acid biosynthesis and oil accumulation in <i>Paeonia lactiflora</i> â€˜Hangshaoâ€™ seeds. The expression levels (represented by the Log<sub>2</sub> FPKM) of the possible candidates are highlighted by color scales (blue to red scale) in <i>Paeonia lactiflora</i> â€˜Hangshaoâ€™ seeds at different development stages (30, 60 and 90 DAF). BC: biotin carboxylase; BCCP: biotin carboxyl carrier protein; Î±-CT: carboxyl transferase subunit alpha; Î²-CT: carboxyl transferase subunit beta; MACT: malonyl-CoA-acyl carrier protein transacylase; KASIII: 3-oxoacyl-ACP synthase III; KAR: 3-oxoacyl-ACP reductase; HAD: 3-hydroxyacyl-ACP dehydratase; EAR: enoyl-ACP reductase I; KASII: 3-oxoacyl-ACP synthase II; FATB: fatty acyl-ACP thioesterase B; FATA: fatty acyl-ACP thioesterase A; SAD: stearoyl-ACP desaturase; FAD: fatty acid desaturase; PCH: palmitoyl-CoA hydrolase; LACS: long-chain acyl-CoA synthetase; PC: phosphatidylcholine LPC: lysophosphatidylcholine; LPCAT: lysophosphatidylcholine acyltransferase; PLA2: phospholipase A2; KCS: 3-ketoacyl-CoA synthase; KCR: very-long-chain 3-oxoacyl-CoA reductase; HCD: very-long-chain (3R)-3-hydroxyacyl-CoA dehydratase; ECR: very-long-chain enoyl-CoA reductase; GPAT: glycerol-3-phosphate acyltransferase; LPAAT: 1-acyl-sn-glycerol-3-phosphate acyltransferase; PAP: phosphatide phosphatase; DGAT: diacylglycerol O-acyltransferase; PDAT: phospholipid:diacylglycerol acyltransferase; PDCT: phosphatidylcholine:diacylglycerol cholinephosphotransferase; OB: oil body; OLE: oleosin; CLO: caleosin. * represented DEGs in all three groups. This model was developed based on the transcriptome data obtained in this study and information from Li et al. [<a href="#CR17">17</a>], Tian et al. [<a href="#CR24">24</a>] and Yang et al. [<a href="#CR28">28</a>]</p>   </div> <p /> <p id="Par65">For fatty acid biosynthesis, the critical steps and key enzymes are shown in Fig. <a href="#Fig6">6</a>. First, acetyl-CoA conversion to malonyl-CoA is catalyzed by the enzyme acetyl-CoA carboxylase (ACCase), which consists of four subunits: biotin carboxylase (BC), carboxyl transferase subunit alpha (Î±-CT), carboxyl transferase subunit beta (Î²-CT) and biotin carboxyl carrier protein (BCCP). We identified 15 DEGs encoding ACCase subunits, including 5â€‰<i>BC</i>, 3 <i>Î±-CT</i> and 7 <i>BCCP</i>. Then, malonyl-CoA was converted to malonyl-ACP, the primary substrate for a subsequent cycle of condensation reactions, by malonyl-CoA ACP transacylase (MCAT). Only 1 DEG was identified as <i>MCAT</i>. Subsequently, malonyl-ACP combined with acetyl-CoA were condensed to Î²-ketobutyryl-ACP by 3-oxoacyl-ACP synthase III (KASIII), and only 1 DEG for <i>KASIII</i> was identified. Then, Î²-ketobutyryl-ACP was converted to Î²-hedroxybutyryl-ACP via reduction by 3-oxoacyl-ACP reductase (KAR), and 8 DEGs were identified as <i>KAR</i>. Next, the Î²-hedroxybutyryl-ACP was dehydrated by 3-hydroxyacyl-ACP dehydratase (HAD) to generate <i>trans</i>-Î"<sup>2</sup>-butenoyl-ACP, and 4 DEGs were identified as <i>HAD</i>. The <i>trans</i>-Î"<sup>2</sup>-butenoyl-ACP was reduced to butyryl-ACP (C4:0-ACP) by enoyl-ACP reductase I (EAR), and 3 unigenes were identified as <i>EAR</i>. Subsequently, after 6â€‰cycles for a series of condensation, reduction, dehydration and reduction reactions by 3-oxoacyl-ACP synthase II (KASII), KAR, HAD and EAR, C4:0-ACP was transformed to palmitoyl-ACP (C16:0-ACP). Moreover, after 7â€‰cycles of those reactions, C4:0-ACP was transformed to stearoyl-ACP (C18:0-ACP). We identified 4 DEGs as <i>KASII</i>. Under the effect of fatty acyl-ACP thioesterase B (FATB) and fatty acyl-ACP thioesterase A (FATA), C16:0-ACP and C18:0-ACP were transformed to palmitic acid (C16:0, PA) and stearic acid (C18:0, SA), respectively. Two DEGs were identified as <i>FATB</i>, but only 1 DEG was identified as <i>FATA</i>.</p> <p id="Par66">Fatty acid elongation occurs via long-chain acyl-CoA (Câ‰¥16) in endoplasmic reticulum. Fatty acids (Câ‰¥16), which are catalyzed by thiolase from fatty acyl-ACP in plastids, are transformed to long-chain acyl-CoA (Câ‰¥16) by long-chain acyl-CoA synthetase (LACS) and transported to the endoplasmic reticulum. Three DEGs identified as <i>LACS</i>. After a series of complicated biochemical reactions, long-chain acyl-CoA (Câ‰¥16) was transformed to long-chain acyl-CoA (C+â€‰2) by 3-ketoacyl-CoA synthase (KCS), very-long-chain 3-oxoacyl-CoA reductase (KCR), very-long-chain (3R)-3-hydroxyacyl-CoA dehydratase (HCD) and very-long-chain enoyl-CoA reductase (ECR), and 13, 4, 3 and 3 DEGs were identified as <i>KCS</i>, <i>KCR</i>, <i>HCD</i> and <i>ECR</i>, respectively.</p> <p id="Par67">The biosynthesis of unsaturated fatty acids occurs via two pathways, the plastid pathway and endoplasmic reticulum pathway, beginning with C18:0-ACP. First, with dehydrogenation by stearoyl-ACP desaturase (SAD), C18:0-ACP is transformed to oleoyl-ACP (C18:1-ACP) in the plastid. Six DEGs were identified as <i>SAD</i>. Then, some C18:1-ACP was dehydrogenized to linoleoyl-ACP (C18:2-ACP) by delta-12 desaturase in the plastid (FAD6). No DEGs were identified as <i>FAD6</i>. Next, C18:2-ACP was dehydrogenized by Delta-15 desaturase in the plastid (FAD7/8) to form alpha-linolenoyl-ACP (C18:3-ACP). Two DEGs each were annotated to <i>FAD7</i> and <i>FAD8</i>. However, some C18:1-ACP was transformed to C18:1 by FATA, which was transformed to C18:1-CoA by LACS, and then C18:1-CoA was transformed in the endoplasmic reticulum. Subsequently, C18:1-CoA and lysophosphatidylcholine (LPC) were transformed to C18:1-PC by lysophosphatidylcholine acyltransferase (LPCAT). No DEGs were identified as <i>LPCAT</i>. Next, C18:1-PC was dehydrogenized to C18:2-PC by delta-12 desaturase in the endoplasmic reticulum (FAD2), and C18:2-PC was dehydrogenized to C18:3-PC by delta-15 desaturase in the endoplasmic reticulum (FAD3). Nine and 5 DEGs were identified as <i>FAD2</i> and <i>FAD3</i>, respectively.</p> </div> <div class="analysisofdegsfortagassemblyandoilaccumulation" title="sec"> <div class="title" tagx="title" title="title">
Analysis of DEGs for TAG assembly and oil accumulation</div> <p id="Par68">Most fatty acids in plants are stored in the form of TAG. The fatty acids synthesized in the plasmid are generally transferred from the plastid to the endoplasmic reticulum in the form of fatty acyl-CoA and become a substrate for TAG assembly. First, under catalysis by glycerol-3-phosphoate acyltransferase (GPAT), the fatty acid carbon chain on the fatty acyl-CoA molecule is transferred to the <i>sn-1</i> position of glycerol-3-phosphate (G3P) to form lysophosphatidic acid (LPA). Then, by catalysis of lyso-phosphatidic acid acyltransferase (LPAAT), phosphatidic acid (PHA) is formed from the <i>sn-2</i> position of LPA binding the fatty acid carbon chain of fatty acyl-CoA. Next, PHA is dephosphorylated by phosphatidic acid phosphatase (PAP) to form diacylglycerol (DAG). Finally, DAG is catalyzed by diacylglycerol acyltransferase (DGAT) to bind the fatty acid carbon chain of fatty acyl-CoA to form TAG at the <i>sn-3</i> position. We identified 4, 4, 5 and 1 DEGs as <i>GPAT</i>, <i>LPAAT</i>, <i>PAP</i> and <i>DGAT</i>, respectively.</p> <p id="Par69">TAG is stored as an oil body (OB), which is the main lipid-storage organelle in plant cells. Additionally, OB contains the hydrophobic core of TAG, which is surrounded by a monolayer of phospholipids and unique proteins called oleosin (OLE), caleosin (CLO) and steroleosin (STE), which play important roles in maintaining the stability of OB, regulating the size of OB, and controlling the amount of OB. We identified 6 and 4 DEGs as <i>OLE</i> and <i>CLO</i>, respectively. We also found that the expression levels of most of those DEGs showed a rising trend.</p> <p id="Par70">The heat maps of the abovementioned DEGs are shown in Fig. <a href="#Fig6">6</a>. We found that most of the DEGs had relatively low expression levels, which appear blue in the heat map, and a small number of them were highly expressed, which appear read in the heat map, including 4 <i>KAR</i>, 1 <i>KCR</i>, 2 <i>SAD</i>, 4 <i>FAD3</i>, 6 <i>OLE</i> and 1 <i>CLO</i>. In addition, we analyzed the number and attribution of DEGs in each group (AdditionalÂ fileÂ <a href="#MOESM13">13</a>: Table S10). In Group I, 76 DEGs, including 17 upregulated and 59 downregulated DEGs, were divided into 32 DEGs in the fatty acid biosynthesis pathway, 15 DEGs in the fatty acid elongation pathway, 13Â DEGs in the biosynthesis of unsaturated fatty acid pathway, 8 DEGs in the TAG assembly pathway, and 8 DEGs in the lipid storage pathway. In Group II, 90 DEGs, including 28 upregulated and 62 downregulated DEGs, were divided into 26 DEGs in the fatty acid biosynthesis pathway, 26 DEGs in the fatty acid elongation pathway, 24 DEGs in the biosynthesis of unsaturated fatty acid pathway, 12 DEGs in the TAG assembly pathway, and 2 DEGs in the lipid storage pathway. In Group III, 118 DEGs, including 32 upregulated and 86 downregulated DEGs, were divided into 39 DEGs in the fatty acid biosynthesis pathway, 30 DEGs in the fatty acid elongation pathway, 24 DEGs in the biosynthesis of unsaturated fatty acid pathway, 16 DEGs in the TAG assembly pathway, and 9 DEGs in the lipid storage pathway. Moreover, we found 47 DEGs in all three groups (Fig. <a href="#Fig6">6</a>, marked as *), among which 33 DEGs were downregulated, including <i>BC</i> (3), <i>Î±-CT</i> (2), <i>BCCP</i> (3), <i>HAD</i> (3), <i>EAR</i> (3), <i>KASII</i> (3), <i>KCS</i> (3), <i>KCR</i> (2), <i>HCD</i> (1), <i>ECR</i> (1), <i>FAD2</i>(5), <i>FAD3</i>(3) and <i>PDCT</i> (1); only 1 DEG was upregulated belonging to <i>KAR</i>, and the other 13 DEGs were upregulated in one group or downregulated in another group.</p> </div> <div class="characteristicsofthebccpgenefamilyinpaeonialactifloraâ€˜hangshaoâ€™" title="sec"> <div class="title" tagx="title" title="title">
Characteristics of the <i>BCCP</i> gene family in <i>Paeonia lactiflora</i> â€˜Hangshaoâ€™</div> <p id="Par71">We identified 17 <i>BCCP</i> genes in <i>Paeonia lactiflora</i> â€˜Hangshaoâ€™ from transcriptome database, and the gene sequences were available in AdditionalÂ fileÂ <a href="#MOESM14">14</a>: Table S11. To further analyze the diversity of BCCP protein in <i>P. lactiflora</i> â€˜Hangshaoâ€™, we used MEME to analyze the conservative motif. Generally speaking, proteins clustered in a family have similar motif composition. The results showed that these 17 BCCP proteins all contained three conserved motifs (Fig.Â <a href="#Fig7">7</a>a). Compared with BCCP protein in <i>Arabidopsis thaliana</i>, there are also three conservative motifs, but there are still differences in the motif locations among them. Subsequently, we used DNAMAN to compare the amino acid sequence of BCCP in <i>P. lactiflora</i> â€˜Hangshaoâ€™ with the amino acid sequence of other plants in NCBI and found that they have certain similarities (Fig. <a href="#Fig7">7</a>b). Then, we constructed the phylogenetic tree with the neighbor-joining method using MEGA 7.0 following multiple alignments of protein sequences with ClustalW (Fig. <a href="#Fig7">7</a>c). The alignment results indicated that the BCCP protein sequence has a consistency in different plants. And all 17 BCCPs were divided into 5 clades. Moreover, we analyzed the FPKM value of 17 BCCP (Fig. <a href="#Fig7">7</a>d), and found that most of them are less than 10 except BCCP1 (CL14917.Contig1_All), BCCP2 (CL14917.Contig2_All) and BCCP17 (Unigene23603_All) in 30 DAF. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 7</span></a></div>   <p>Comparative and phylogenetic analysis and FPKM value of 17 genes putatively encoding BCCP in <i>Paeonia lactiflora</i> â€˜Hangshaoâ€™. <b>a</b> Protein domain analysis of 17 BCCP: MEME prediction resulet; <b>b</b> Amino acid sequence alignment of BCCP family in plant species; <b>c</b> Phylogenetic analysis of the BCCP proteins in <i>Paeonia lactiflora</i> â€˜Hangshaoâ€™ and other plant speices. Other plant species including <i>Gossypium hirsutum</i> (Gh), <i>Gossypium arboreum</i> (Ga), <i>Theobroma cacao</i> (Tc), <i>Juglans regia</i> (Jr), <i>Pistacia vera</i> (Pv), <i>Glycine soja</i> (Gs), <i>Glycine max</i> (Gm), <i>Malus domestica</i> (Md), <i>Vitis vinifera</i> (Vv), <i>Vitis riparia</i> (Vr), <i>Camellia sinensis</i> (Cs-XM 02822298), <i>Prunus persica</i> (Pp), <i>Prunus mume</i> (Pm), <i>Prunus dulcis</i> (Pd), <i>Arabidopsis thaliana</i> (At), <i>Vernicia fordii</i> (Vf), <i>Jatropha curcas</i> (Jc), <i>Ricinus communis</i> (Rc), <i>Citrus sinensis</i> (Cs-XM 006482671), <i>Quercus suber</i> (Qs), <i>Quercus lobata</i> (Ql), <i>Nelumbo nucifera</i> (Nn), <i>Helianthus annuus</i> (Ha), <i>Salvia miltiorrhiza</i> (Sm), <i>Sesamum indicum</i> (Si), <i>Olea europaea</i> var. <i>sylvestris</i> (Oe) and <i>Nicotiana tabacum</i> (Nt). <b>d</b> FPKM value of 17 BCCP in <i>Paeonia lactiflora</i> â€˜Hangshaoâ€™ from transcriptome database</p>   </div> <p /> </div> <div class="experimentalvalidationandanalysisofsixteendegsinvolvedinÂ lipidmetabolism" title="sec"> <div class="title" tagx="title" title="title">
Experimental validation and analysis of sixteen DEGs involved inÂ lipid metabolism</div> <p id="Par72">To validate the DEGs, the relative expression levels of 16 randomly selected DEGs involved in lipid metabolism of KEGG pathway were analyzed by qRT-PCR. Among them, there were 7 DEGs related to fatty acid biosynthesis, namely, <i>BCCP</i>, <i>BC</i>, <i>MCAT</i>, <i>KASII</i>, <i>KASIII</i>, <i>FATA</i> and <i>FATB</i>, and 1 DEG was involved in fatty acid elongation (<i>KCR</i>). Four DEGs were involved in the biosynthesis of unsaturated fatty acids, including <i>SAD</i>, <i>FAD2</i>, <i>FAD3</i> and <i>FAD7</i>, and 2 DEGs, <i>GPAT</i> and <i>DGAT</i>, were involved in TAG assembly and 2 DEGs, <i>OLE</i> and <i>CLO</i>, in lipid storage (Fig.Â <a href="#Fig8">8</a>). Overall, the expression patterns of 16 DEGs verified by qRT-PCR were highly consistent with the transcriptome sequencing results, indicating that the DEG analysis findings were reliable. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 8</span></a></div>   <p>qRT-PCR validation of the expression levels of DEGs related to unsaturated fatty acid biosynthesis and oil accumulation for developmental seeds of <i>Paeonia lactiflora</i> â€˜Hangshaoâ€™</p> <p>The column chart shows the qRT-PCR validations in 5 periods; the broken line graph shows the transcriptome sequencing in 3 periods.</p>   </div> <p /> </div> </div> <div class="discussion" title="sec"> <div class="title" tagx="title" title="title">
Discussion</div> <p id="Par73">Most plants contain accumulated oil (mainly TAG) in their seeds to provide energy for seed germination. Plant oils have great significance in agricultural production, and most are mainly used as edible oil in food processing and preparation [<a href="#CR29">29</a>]. Plant oils contain essential fatty acids, including omega-3 series ALA and omega-6 series LA, which can only be synthesized in plants [<a href="#CR30">30</a>]. The proportion of fatty acids in plant oils, especially the proportion of unsaturated fatty acids, is one of the evaluation criteria for plant oil quality [<a href="#CR31">31</a>]. However, the content of LA and ALA constituting different proportions of essential fatty acids have different effects on human physiological functions [<a href="#CR32">32</a>, <a href="#CR33">33</a>]. At present, many studies have shown that the intake ratio of LA/ALA has the most significant physiological function at 1:1 [<a href="#CR34">34</a>]. In this study, the oil content of mature seed of herbaceous peony â€˜Hangshaoâ€™ was approximately 20%, and five main fatty acids including PA, SA, OA, LA and ALA were detected by GC-MS, which is consistent with the findings for other peonies [<a href="#CR11">11</a>, <a href="#CR17">17</a>]. The absolute contents of the five fatty acid components during seed development were all highest for ALA and lowest for SA. The specific manifestations followed the order ALA&amp;gt;LA&amp;gt;OA&amp;gt;PA&amp;gt;SA. The ratio of LA/ALA was lower during the development of seeds of herbaceous peony â€˜Hangshaoâ€™, maintained between 0.55 and 0.71, which is close to the optimal intake ratio of 1:1. Therefore, herbaceous peony seed oil has potential to become a new resource of nutrition for specific populations.</p> <p id="Par74">Oilseed plants have high contents of seed oil compared with other plants. Despite the similarities of lipid biosynthesis and its components in oilseed plants, the contents of oil and fatty acid vary considerably. However, the regulation of oil content and fatty acid composition is a complex process, which varies in oil biosynthesis phases and accumulation. Nevertheless, the difference in regulatory mechanisms in different species is attributed to the variation in oil content among species [<a href="#CR35">35</a>]. Mainly, the molecular regulators of oil content and fatty acid composition consist of genes involved in fatty acid and TAG assembly [<a href="#CR36">36</a>]. Therefore, the transcriptome was used to understand the genes, gene networks, regulatory factors and their interactions that direct seed oil biosynthesis [<a href="#CR37">37</a>]. In this study, a comparative transcriptome was used for the first time to reveal the high oil content and unsaturated fatty acids of the herbaceous peony â€˜Hangshaoâ€™ seed. Here we de novo assembled the transcript in developing seeds of <i>P. lactiflora</i> â€˜Hangshaoâ€™, resulting in 150,156 unigenes, exceeding the transcriptome data from other oilseed plants such as <i>Malania oleifera</i> [<a href="#CR28">28</a>], <i>Camellia oleifer</i> [<a href="#CR23">23</a>] and <i>Paeonia ostii</i> [<a href="#CR38">38</a>]. We think that the number of unigenes may have a great divergence in different plants. Even in the same plant, the number of unigenes may be different in various tissues and/or at different stages of development. For example, the unigenes in the buds, stems and budding seeds of <i>P. lactiflora</i> were 207,827 [<a href="#CR39">39</a>], 107,929 [<a href="#CR40">40</a>] and 123,577 [<a href="#CR41">41</a>], respectively. Of course, it is undeniable that there may be incomplete and redundant stitching in Trinity based on sequencing by using Illumina HiSeq 4000 system. When the length of one gene is relatively long, it is difficult to assemble the full length of the gene, and it is possible that some unigenes actually belong to the part of one gene. In order to obtain as complete unigenes as possible, we took a series of measures. Firstly, we repeated it three times at three different stages to construct the cDNA libraries and sequenced using the Illumina Hiseq 4000 platform. Secondly, sequence adapters, reads with more than 5% N base and low-quality reads (percentage of base with a quality less than 15 greater than 20% in a read) were removed to obtain clean reads. Lastly, all clean reads were assembled using Trinity software [<a href="#CR42">42</a>], and Tgicl software [<a href="#CR43">43</a>] was used to perform clustering and eliminate redundant data in the assembled transcipts to obtain unique genes. The clustering quality metrics are shown as Additional file Table S2. Moreover, we selected 16 differentially expressed genes (DEGs) to analyze the relative expression by qRT-PCR to validate the transcriptome sequencing results. Overall, we found that the expression patterns of 16 DEGs verified by qRT-PCR were highly consistent with the transcriptome sequencing results, indicating that the DEG analysis findings were reliable (Fig. <a href="#Fig8">8</a>).</p> <p id="Par75">One of main objectives of this study was to identify potential unigenes involved in unsaturated fatty acid biosynthesis and oil accumulation in developing seeds of herbaceous peony. However, oil accumulation mainly involves fatty acid biosynthesis (including fatty acid carbon chain elongation and desaturation), TAG assembly and lipid storage [<a href="#CR44">44</a>]. Overall, to produce a C16 or C18 fatty acid from acetyl-CoA and malonyl-CoA, more than 30 enzymatic reactions are required, involving enzymes such as ACCase (including BC, BCCP, Î±-CT and Î²-CT), MCAT, KAS (including KASI, KASII and KASIII), KAR, HAD, EAR, and FAT (including FATA and FATB), among others [<a href="#CR45">45</a>]. In this study, we screened out many relevant DEGs encoding the above enzymes (Fig. <a href="#Fig6">6</a>). Most of them were downregulated DEGs such as <i>BCCP</i>. Even though some of these DEGs were upregulated, their FPKM values were lower than 10, except for two DEGs that were annotated to <i>KAR</i> (named CL2376.Contig1_All and CL2376.Contig2_All). Interestingly, only 1 unigene each was annotated to MCAT, KASIII and FATA, all of which were downregulated DEGs. MCAT is a key enzyme in the fatty acid synthesis pathway, and its expression level is closely related to lipid accumulation. At present, overexpression of the <i>MCAT</i> gene in rape and other plants has a positive effect on promoting the accumulation of C16 and C18 fatty acids [<a href="#CR46">46</a>]. KASIII is a key enzyme that initiates de novo fatty acid synthesis. At present, <i>KASIII</i> has been cloned from plants such as sunflower [<a href="#CR47">47</a>] and jatropha [<a href="#CR48">48</a>]. Some studies have shown that KASIII can use other receptor substrates to generate different final products of fatty acid synthesis, which indicates that KASIII is one of the main factors determining the carbon chain structure of fatty acids [<a href="#CR49">49</a>, <a href="#CR50">50</a>]. In addition, KASIII, as the initiator of de novo synthesis of fatty acid, plays a second role in determining the length of the carbon chain of fatty acids [<a href="#CR51">51</a>]. Some studies have suggested that the synthesis rate of fatty acid is affected by KASIII enzymatic activity, indicating that it may be one of the rate-limiting enzymes in the fatty acid synthesis pathway [<a href="#CR52">52</a>]. In the present study, the FPKM value of the <i>KASIII</i> gene in this transcriptome decreased with the development of seeds, but its expression by qRT-PCR was highest at 45 DAF (Fig. <a href="#Fig8">8</a>), which was consistent with the accumulation rate of fatty acid in herbaceous peony seeds. FATA is a type of acyl-ACP thioesterase (FAT). De novo synthesis of fatty acid was terminated by FAT, which hydrolyzes the thiol-lipid bond located between acyl and ACP to form fatty acids [<a href="#CR53">53</a>]. The specificity of FAT largely determines the chain length and unsaturated degree of plant fatty acids in glycerolipid and TAG [<a href="#CR54">54</a>]. According to the sequence alignment, FAT can be classified as FATA and FATB, which have different substrate specificities. However, FATA has the highest catalytic activity of C18:1<sup>Î"9</sup>-ACP [<a href="#CR55">55</a>]. In this study, the FPKM value of <i>FATA</i> in this transcriptome also decreased with the development of seeds; however, the tendency of <i>FATA</i> expression by qRT-PCR was consistent with the total accumulation rate of OA, LA and ALA in herbaceous peony seeds.</p> <p id="Par76">UFAs are the main components of plant cell membrane lipids and storage lipids, which play important roles in physiological functions. In most plants, UFAs account for more than 75% of the total fatty acids [<a href="#CR45">45</a>]. In this study, there were more than 95% UFAs, especially 70% PUFAs in herbaceous peony seed oil (Fig. <a href="#Fig1">1</a>c). The first double bond is introduced by the soluble enzyme stearoyl-ACP desaturase (SAD), which catalyzes the desaturation of C18:0-ACP to form C18:1-ACP. There are subsequently two pathways of desaturation by fatty acid desaturases (FADs). In one pathway, C18:1-ACP is successively transformed to C18:2-ACP and C18:3-ACP in plastid by FAD6 and FAD7 or FAD8, respectively. In another pathway, C18:1-ACP undergoes a series of enzymatic reactions to produce C18:1-PC, which is desaturated by FAD2 and FAD3 in the endoplasmic reticulum to form C18:2-PC and C-18:3 PC. SAD controls the total amount of UFAs in plant oils and the ratio of saturated fatty acid to UFA [<a href="#CR56">56</a>]. The content of SA in the <i>SAD</i> mutant was found to increase from 0.8 to 14.3% compared with wild-type <i>Arabidopsis</i>, accompanied by a decrease in OA content [<a href="#CR57">57</a>]. In contrast, upon overexpression of <i>SAD</i> in <i>Arabidopsis</i>, the SA content decreased and OA content increased [<a href="#CR58">58</a>]. The expression of <i>SAD</i> was higher in seeds than in other tissues, and the expression of <i>SAD</i> was bell-shaped with seed development [<a href="#CR59">59</a>]. FADs are all intact membrane proteins, including FAD2, FAD3, FAD6, FAD7 and FAD8, and they catalyze the desaturation of C18 fatty acids. FAD2/FAD6 catalyze the dehydrogenation of OA at the Î"12 position to form LA [<a href="#CR60">60</a>], and LA to form ALA by Î"15 fatty acid dehydrogenase including FAD3, FAD7 and FAD8 [<a href="#CR61">61</a>]. In this process, FADs play a key role in the formation of ALA [<a href="#CR62">62</a>]. FAD2 is a key enzyme that controls the transformation of OA to LA. By inducing the mutation of <i>FAD2</i> or inhibiting its expression by antisense methodologies, the content of OA in soybean [<a href="#CR63">63</a>], sunflower [<a href="#CR64">64</a>] and peanut [<a href="#CR65">65</a>] seed oil can be improved, thus improving the quality of edible oil. The <i>FAD3</i> gene is semidominant, and its overexpression leads to a decrease in LA and an increase in ALA [<a href="#CR66">66</a>, <a href="#CR67">67</a>]. In this study, we identified 6, 9, 5, 2 and 2 DEGs annotated to <i>SAD</i>, <i>FAD2</i>, <i>FAD3</i>, <i>FAD7</i> and <i>FAD8</i>, respectively. The FPKM values of those DEGs at 90 DAF were mostly less than 1, which indicated that the absolute contents of unsaturated fatty acid in herbaceous peony seed oil showed a downward trend at the later stage. Although most of the FPKM values of <i>SAD</i>, <i>FAD2</i> and <i>FAD3</i> based on the transcriptome sequencing showed a decreasing trend with seed development, their expression levels were highest at 45 DAF by qRT-PCR analysis and exhibited a bell shape with seed development (Fig. <a href="#Fig8">8</a>), consistent with the rate of unsaturated fatty acid accumulation (Fig. <a href="#Fig1">1</a>c). Moreover, the FPKM values of <i>FAD7</i> and <i>FAD8</i> were more lower than that of <i>FAD3</i>. Therefore, we inferred that the key genes in herbaceous peony seed oil with high contents of unsaturated fatty acids, such as ALA and LA, are <i>SAD</i>, <i>FAD2</i> and <i>FAD3</i>.</p> <p id="Par77">Most fatty acids in plants are successively transferred to the <i>sn-1</i>, <i>sn-2</i> and <i>sn-3</i> position of glycerol-3-phosphate to form TAG, which is assembled in the endoplasmic reticulum through the Kennedy pathway, and finally formed into an oil body by budding for storage [<a href="#CR68">68</a>]. TAG assembly mainly involves enzymes including GPAT, LPAAT, PAP and DGAT [<a href="#CR69">69</a>]. DGAT catalyzes the final step of TAG assembly in the Kennedy pathway, and it is the only rate-limiting enzyme in this pathway [<a href="#CR70">70</a>]. The expression of single <i>DGAT</i> can restore the accumulation of TAG and the formation of OBs [<a href="#CR71">71</a>]. Four different types of <i>DGATs</i> were found in plants, namely, <i>DGAT1</i>, <i>DGAT2</i>, <i>DGAT3</i> and <i>WS/DGAT</i> [<a href="#CR72">72</a>, <a href="#CR73">73</a>]. Organ expression of <i>DGAT</i> is specific. <i>DGAT1</i> is highly expressed in developing seeds, petals, and flower buds of most dicotyledonous plants, but it is poorly expressed in leaves and stems [<a href="#CR74">74</a>, <a href="#CR75">75</a>]. In this study, there was only 1 DEG annotated to <i>DGAT</i>, the FPKM value of which showed a continuous upward trend and expression by qRT-PCR was highest at 75 DAF, which was consistent with the highest oil content at 75 DAF. TAGs are the most abundant neutral lipids, and most TAGs are partitioned in small spherical particles called oil bodies (OBs) [<a href="#CR68">68</a>]. OBs contain a hydrophobic core of TAG surrounded by a monolayer of phospholipids and unique proteins called oleosin (OLE), caleosin and steroleosin [<a href="#CR76">76</a>]. Among these proteins, OLE is the main protein and covers almost the whole surface of OBs, accounting for approximately 75â€"80% [<a href="#CR77">77</a>]. OLE plays an important role in maintaining the size and stability of the oil body [<a href="#CR78">78</a>]. In addition, Siloto et al. found that the expression level of <i>OLE</i> in <i>Arabidopsis</i> seeds with high oil content was higher than that in seeds with low oil content [<a href="#CR79">79</a>]. The expression of <i>OLE</i> is higher in seeds than in roots, stems, leaves and flowers. Additionally, with the development of seeds, the expression of <i>OLE</i> in seeds first increases and then decreases [<a href="#CR80">80</a>]. In the present study, we identified 6 DEGs annotated to <i>OLE</i>, the FPKM values of which in the transcriptome were very high and increasing; however, the expression of <i>OLE</i> by qRT-PCR was highest at 75 DAF, which was consistent with the oil accumulation. Similarly, the trend of <i>OLE</i> expression in developmental seeds was consistent with other studies. Therefore, we speculated that high oil accumulation in herbaceous peony seed was related to the high expression levels of <i>DGAT</i> and <i>OLE</i>.</p> <p id="Par78">BCCP, or biotin carboxyl carrier protein, is one of the subunits of ACCase, and BC, Î±-CT and Î²-CT constitute four subunits of ACCase which catalyze acetyl-CoA to malonyl-CoA that was the committed step for de novo fatty acid biosynthesis [<a href="#CR81">81</a>]. As an important component of ACCase, BCCP protein is the link and bridge connecting the other three subunits of ACCase, and plays a very important role in the biosynthesis of fatty acid from seeds [<a href="#CR82">82</a>]. Studies have shown that the up-regulation of BCCP protein expression is positively correlated with the specific activity of ACCase and the content of seed oil during the development of <i>Arabidopsis thaliana</i> and rapeseed seeds [<a href="#CR82">82</a>]. We identified 17 unigenes annotated to BCCP from transcriptome database of <i>P. lactiflora</i> â€˜Hangshaoâ€™, and we found that the sequence conservation of BCCP was highly conserved by alignment with the amino acid sequences of family genes of various plants. According to the phylogenetic analysis, 17 BCCPs in <i>P. lactiflora</i> â€˜Hangshaoâ€™ were divided into five clades. Chen et al. found that BCCPs was in four subfamilies, separated by more subtle but statistically significant differences in primary structure [<a href="#CR83">83</a>]. Moreover, Thelen et al. believed that <i>BCCP</i> gene can encode two kinds of BCCP protein, one can be biotinylated, and the other cannot be biotinylated [<a href="#CR84">84</a>]. These two kinds of BCCP proteins can regulate the activity of ACCase by competitive binding to the other subunits of ACCase, so as to realize the regulation of fatty acid synthesis process [<a href="#CR84">84</a>]. From these studies, it has been shown that BCCP has a significant influence on the content of fatty acid of plants.</p> </div> <div class="conclusions" title="sec"> <div class="title" tagx="title" title="title">
Conclusions</div> <p id="Par79">GC-MS analysis revealed that â€˜Hangshaoâ€™ seeds have five main fatty acids, including palmitic acid (PA), stearic acid (SA), oleic acid (OA), linoleic acid (LA) and Î±-linolenic acid (ALA). The total content of the five fatty acids first increased and then decreased, reaching a maximum at 75 DAF. Additionally, the ratio of the content of OA to ALA was close to 1:1. A total of 150,156 unigenes were obtained by transcriptome sequencing, with an average length of 1030â€‰bp. There were 15,005 unigenes annotated in the seven functional databases, including NR, NT, GO, KOG, KEGG, Swiss-Prot and InterPro. Based on the KEGG database, there were 1766 unigenes annotated to lipid metabolism. We also screened out 503 DEGs in lipid metabolism, which were annotated to 111 enzymes. To further uncover the key genes associated with the biosynthesis of unsaturated fatty acid and oil accumulation, we identified 123 DEGs involved in fatty acid biosynthesis (39 DEGs), fatty acid elongation (33 DEGs), biosynthesis of unsaturated fatty acid (24 DEGs), TAG assembly (17 DEGs) and lipid storage (10 DEGs). Furthermore, qRT-PCR was used to analyze the expression patterns of 16 related genes, including <i>BBCP</i>, <i>BC</i>, <i>MCAT</i>, <i>KASII, KASIII, FATA FATB</i>, <i>KCR</i>, <i>SAD, FAD2</i>, <i>FAD3</i>, <i>FAD7</i>, <i>GPAT, DGAT, OLE</i> and <i>CLO,</i> and the results showed that they were highly consistent with the transcriptome results. Moreover, we deduced that the key genes with high contents of unsaturated fatty acids and oil accumulation in herbaceous peony seed oil are <i>MCAT</i>, <i>KASIII</i>, <i>FATA</i>, <i>SAD</i>, <i>FAD2</i>, <i>FAD3</i>, <i>DGAT</i> and <i>OLE</i>.</p> <p id="Par80">In conclusion, we present the first high-quality transcriptome sequence for herbaceous peony seeds. The sequences of genes related to unsaturated fatty acid biosynthesis and oil accumulation were obtained based on large-scale transcriptome data, which will enable further metabolomic and gene functional studies. This species with rich PUFA content, accompanied by an omega-6/omega-3 ratio close to 1:1, was studied to form a more diversified set of PUFAs to eventually increase storage PUFA production and meet the needs of more humans worldwide for intake of an omega-6/omega-3 ratio close to 1:1.</p> </div> <div class="methods" title="sec"> <div class="title" tagx="title" title="title">
Methods</div> <div class="plantmaterials" title="sec"> <div class="title" tagx="title" title="title">
Plant materials</div> <p id="Par81">Seeds of herbaceous peony â€˜Hangshaoâ€™ were collected from the same plant which was introduced from Panâ€™an County Zhejiang Province (traditional Chinese medicine â€˜Radix Paeoniae Albaâ€™ production base), identified by Yang-Qian Cao, a senior agronomist from Heze Peony Research Institure in Shangdong Province of China, and cultivated in the peony germplasm resource garden of Yangzhou University, China (32Â°39â€²N, 119Â°42â€²E) for 8 years. Due to the limited fruit and low oil extraction rate at the early stage, the follicle was hand-collected in the 30â€‰days after flowering (DAF) for the first time and at intervals of 15â€‰days until full maturity (including 30, 45, 60, 75 and 90 DAF) for fatty acid absolute content analysis by GC-MS (Fig. <a href="#Fig1">1</a>a). Seeds used for transcriptome sequencing were collected from the same strain at developmental stages of 30, 60 and 90 DAF. The collected samples were immediately frozen in liquid nitrogen and stored in a âˆ’â€‰80â€‰Â°C cryogenic refrigerator until further use.</p> </div> <div class="oilcontentanalysis" title="sec"> <div class="title" tagx="title" title="title">
Oil content analysis</div> <p id="Par82">Ten grams of powdered seeds were weighed and transferred to a 500-mL round-bottom flask. Then, 130â€‰ml n-hexane at a 1:13 ratio (gÂ·mL<sup>âˆ’â€‰1</sup>) was added to the flask. The flask was connected to a condenser tube and placed in the ultrasonic cleaning machine (KQ-200VDB, Kunshan ultrasonic instrument co., LTD, China) to perform oil extraction at 70â€‰Â°C and 200â€‰W at a fixed frequency of 80 KHz. After extraction for 1.5â€‰h, the mixture was centrifuged at room temperature for 10â€‰min at 4000 rÂ·min<sup>âˆ’â€‰1</sup>. The supernatant was separated and dried under the rotary evaporator (RE52â€"99, Yarong biochemical instrument factory, Shanghai, China) at 55â€‰Â°C for oil separation and n-hexane recovery. The oil was then stored in a refrigerator (4â€‰Â°C) until fatty acid analysis. The oil content was calculated and expressed as % (oil weight/seed weight).</p> </div> <div class="quantitativeanalysisofthemainfattyacidcomposition" title="sec"> <div class="title" tagx="title" title="title">
Quantitative analysis of the main fatty acid composition</div> <p id="Par83">The absolute content of fatty acid was determined according to the national standard (GB 5009.168â€"2016) of the Peopleâ€™s Republic of China. The specific process is as follows.</p> <p id="Par84">Standard solution preparation: 1) 5â€‰mgÂ·mL<sup>âˆ’â€‰1</sup> triundecanoin (C11) internal standard solution: 0.5â€‰g triundecanoin (C11) was added to a beaker and dissolved by the addition of methanol. Then, the solution was transferred into a volumetric flask and increased to a 100-mL constant volume with methanol. 2) Mixed standard solution of 37-component fatty acid methyl: 100â€‰Î¼L of the 37-component fatty acid methyl mixture was added to a 10-mL volumetric flask and increased to a constant volume with n-hexane. The two standard solutions were stored in a âˆ’â€‰4â€‰Â°C refrigerator.</p> <p id="Par85">Sample hydrolysis: A total of 0.5â€‰g of seeds were ground into granules and placed in a flask. Then, 100â€‰Î¼L triundecanoin (C11) internal standard solution, 100â€‰mg pyrogallic acid, several zeolites and 2â€‰mL 95% ethanol were added and mixed. Subsequently, 10â€‰mL hydrochloric acid solution was added and mixed well. Next, the flask was placed in a 70â€‰Â°C water bath for hydrolysis for 40â€‰min, after which it was cooled to room temperature.</p> <p id="Par86">Fat extraction: The hydrolyzed sample was supplemented with 10â€‰mL 95% ethanol and mixed. The hydrolysate in the flask was then transferred to the separation funnel. Then, the flask and stopper were washed with 50â€‰mL of the ethyl ether petroleum ether mixture. Next, the rinse solution was incorporated into the separation funnel, capped, shaken for 5â€‰min, and set aside for 10â€‰min. Subsequently, the extraction solution was collected into a 250-mL flask. The hydrolysate was extracted 3 times as described in the previous steps. Finally, the separation funnel was flushed with the petroleum ether with ether mixture (volume ratio 1:1), and the flushing fluid was collected into a constant weight flash that would be boiled in a water bath and dried at 100â€‰Â°C for 2â€‰h.</p> <p id="Par87">Fatty acid esterification: The 2â€‰mL of 2% NaOH methanol solution was added to the fat extract and heated at 80â€‰Â°C for 30â€‰min. Then, 3â€‰mL 14% trifluoro (methanol) boron solution was added and heated at 80â€‰Â°C for 30â€‰min. Next, the mixture was cooled to room temperature and placed in a centrifuge tube, and 1â€‰mL n-hexane was added. It was extracted by shock for 2â€‰min and then left for 1â€‰h and stratified. One hundred microliters of the supernatant was increased to 1â€‰mL with n-hexane and filtered with a 0.45-Î¼m organic-phase filter into a 2-mL chromatographic sample bottle.</p> <p id="Par88">Chromatographic and mass spectrometric conditions: The conditions have been previously described [<a href="#CR10">10</a>]. The trinonadecanoin internal standard solution, mixed standard solution of 37-component fatty acid methyl and sample solution were successively subjected to GC-MS to determine the chromatographic peak area of each fatty acid response.</p> <p id="Par89">Calculation formula for the fatty acid absolute content: W<sub>i</sub> (gÂ·100â€‰g<sup>âˆ’â€‰1</sup>) =F<sub>i</sub>Ã—(A<sub>i</sub>/A<sub>C11</sub>)Ã—(Ï�<sub>C11</sub>Ã—V<sub>C11</sub>Ã—â€‰1.0067/M)Ã—â€‰100Ã—N<sub>i</sub>, where W<sub>i</sub> is the absolute content of i fatty acid, F<sub>i</sub> is the response factor of fatty acid methyl ester, A<sub>i</sub> is the peak area of fatty acid methyl ester in the sample, A<sub>C11</sub> is the peak area of the triundecanoin (C11) internal standard solution added to the sample, Ï�<sub>C11</sub> is the concentration of triundecanoin (C11), which was 5â€‰mgÂ·mL<sup>âˆ’â€‰1</sup>, V<sub>C11</sub> is the volume of triundecanoin (C11), which was 0.1â€‰mL, 1.0067 is the conversion coefficient of triundecanoin to methyl undecanoate, M is the sample mass (mg), 100 is the coefficient that converts the content to the content per 100â€‰g of sample and N<sub>i</sub> is the conversion coefficient of fatty acid methyl ester to fatty acid. The calculation formula of F<sub>i</sub> is (Ï�<sub>Si</sub>Ã—A<sub>11</sub>)/(Ï�<sub>11</sub>Ã—A<sub>Si</sub>), Ï�<sub>Si</sub> is the concentration of fatty acid methyl ester in the 37-component fatty acid methyl mixture, Ï�<sub>11</sub> is the concentration of methyl undecanoate in the 37-component fatty acid methyl mixture, A<sub>11</sub> is the peak area of methyl undecanoate and A<sub>Si</sub> is the peak area of fatty acid methyl ester.</p> </div> <div class="totalrnaextraction,cdnalibrarypreparationandtranscriptomesequencinganalysis" title="sec"> <div class="title" tagx="title" title="title">
Total RNA extraction, cDNA library preparation and transcriptome sequencing analysis</div> <p id="Par90">Nine seed samples from three stages (30, 60 and 90 DAF, with three biological replicates) were snap-frozen in liquid nitrogen and ground to a fine powder. Total RNA samples were extracted using the Total RNA Isolation System (Takara, Japan) according to the manufacturerâ€™s instructions. The quality of the resulting RNA was verified by agarose gel electrophoresis using a Nanodrop ND-4301000 spectrophotometer and Agilent 2100 Bioanalyzer (Agilent, Santa Clara, CA, USA). After RNA detection, Oligo (dT) was used to isolate mRNA. Mixed with the fragmentation buffer, the mRNA samples were fragmented. Then cDNA was synthesized using the mRNA fragments as templates. Short fragments are purified and resolved with EB buffer for end reparation and single nucleotide A (adenine) addition. After that, the short fragments are connected with adapters and the suitable fragments are selected for the PCR amplification to construct the sample cDNA libraries. The quantification and qualification of the sample cDNA libraries were checked with Agilent 2100 Bioanaylzer and ABI StepOnePlus Real-Time PCR System during the QC steps. Finally, the nine qualified cDNA libraries (HS30d_1, HS30d_2, HS30d_3, HS60d_1, HS60d_2, HS60d_3, HS90d_1, HS90d_2 and HS30d_3) were sequenced using Illumina HiSeq 4000 system at Beijing Genomics Institute Co., Ltd. (Shenzhen, China). Raw data was analyzed through Sickle (<a href="https://github.com/najoshi/sickle">https://github.com/najoshi/sickle</a>) and SeqPrep (<a href="https://github.com/jstjohn/Seq-Prep">https://github.com/jstjohn/Seq-Prep</a>), low-quality reads (percentage of base with a quality less than 15 greater than 20% in a read), adapter-containing reads and poly-N reads (with more than 5% N base) were removed to get high-quality clean reads. Based on the clean reads, the sequence duplication level and Q20, Q30, GC-content were calculated. Trinity software was used to assemble all of the clean reads from the left.fq and right.fq data, the min_kmer_cov was set as 2 and other parameters was set as their default. And Tgicl software [<a href="#CR43">43</a>] was used to perform clustering and eliminate redundant data in the assembled transcipts to obtain unique genes based on the parameters as followed: âˆ’â€‰140 -c -v25 -Oâ€²-repeat_stringency 0.95 -minmatch 35 -minscore 35â€². All clean reads were uploaded to the National Center for Biotechnology Information (NCBI) Sequence Read Archive (SRA) database under accession number SRP148668.</p> </div> <div class="functionalannotationandclassification" title="sec"> <div class="title" tagx="title" title="title">
Functional annotation and classification</div> <p id="Par91">To obtain comprehensive functional information, all Illumina-assembled unigenes were aligned against the NCBI nonredundant protein (Nr) (<a href="http://www.ncbi.nlm.nih.gov">http://www.ncbi.nlm.nih.gov</a>), NCBI nonredundant nucleotide sequence (Nt), cluster of euKaryotic Orthologous Groups (KOG) (<a href="http://www.ncbi.nlm.nih.gov/KOG">http://www.ncbi.nlm.nih.gov/KOG</a>), Kyoto Encyclopedia of Genes and Genomes (KEGG) (<a href="http://www.genome.jp/kegg">http://www.genome.jp/kegg</a>) and Swiss-Prot protein (<a href="http://ftp.ebi.ac.uk/pub/databases/swissprot">http://ftp.ebi.ac.uk/pub/databases/swissprot</a>) databases using BLASTx alignments with an <i>E</i>-value cutoff of 10<sup>âˆ’â€‰5</sup> [<a href="#CR85">85</a>]. With Nr annotation, Gene Ontology (GO) annotation of unigenes was obtained using Blast2GO software (<a href="http://www.geneontology.org">http://www.geneontology.org</a>) [<a href="#CR86">86</a>]. Inter-Pro annotation was obtained using InterProScan5 [<a href="#CR87">87</a>].</p> </div> <div class="analysisofdifferentiallyexpressedgenes(degs)" title="sec"> <div class="title" tagx="title" title="title">
Analysis of differentially expressed genes (DEGs)</div> <p id="Par92">Before performing differential expression analysis of unigenes, we mapped all the clean reads of each sample to the unigenes with Bowtie2 [<a href="#CR88">88</a>], and we calculated the gene expression level with RSEM software based on the assembly results [<a href="#CR89">89</a>]. The FPKM (expected number of fragments per kilobase of transcript sequence per million mapped fragments) value was used to quantify gene expression levels [<a href="#CR86">86</a>]. All expression data sets and assembled unigenes are available at the NCBI Gene Expression Omnibus (GEO) database under accession number GSE151746. Next, we conducted a differential expression analysis of the samples with three biological replications using DESeq2 as described by Michaekl et al. [<a href="#CR90">90</a>]. Genes with a fold change â‰¥ 2.0 and adjusted <i>P</i>-value â‰¤ 0.05 were regarded as differentially expressed. DEG functions were explored through GO and KEGG pathway analysis, and a false discovery rate (FDR) no larger than 0.01 was defined as significant enrichment. This process was performed to identify significantly enriched metabolic pathways.</p> </div> <div class="analysisofthebccpgenefamilycharacteristicsinpaeonialactifloraâ€˜hangshaoâ€™" title="sec"> <div class="title" tagx="title" title="title">
Analysis of the BCCP gene family characteristics in <i>Paeonia lactiflora</i> â€˜Hangshaoâ€™</div> <p id="Par93">To analyze <i>PlBCCP</i> family characteristics, we utilize BLASTp from NCBI non-redundant protein database to obtain its homologous genes. The conserved motifs of BCCP sequences were identified by MEME program (<a href="http://alternate.memesuite.org/tools/mem">http://alternate.memesuite.org/tools/mem</a>), parameters were set as maximum of 10 misifits and an optimum motif width of 6â€"200 amino acid residues [<a href="#CR91">91</a>]. ClustalX was used to align all of full-length BCCP protein sequences. To construct the phylogenetic tree, MEGA7.0 software was introduced with neighbor-joining (NJ) method, bootstrap values was set as 1000 replicates.</p> </div> <div class="geneexpressionanalysisusingquantitativereal-timepcr(qrt-pcr)" title="sec"> <div class="title" tagx="title" title="title">
Gene expression analysis using quantitative real-time PCR (qRT-PCR)</div> <p id="Par94">qRT-PCR was introduced to analyze gene transcript levels with BIO-RAD CFX Connectâ"¢ Optics Module (Bio-Rad, USA). Extracted seeds RNA (1â€‰Î¼g) were reverse-transcribed into cDNA in a 10-Î¼l reaction based on the superscript first-strand synthesis system (PrimeScriptÂ® RT Reagent Kit With gDNA Eraser, TaKaRa, Japan). <i>PlActin</i> (JN105299) was used as an internal reference gene [<a href="#CR92">92</a>]. All specific primers for qRT-PCR were synthesized by Shanghai Sangon Biological Engineering Technology and Services Co., Ltd., sequences were shown in AdditionalÂ fileÂ <a href="#MOESM15">15</a>: Table S12. SYBRÂ® Premix Ex Taqâ"¢ (Perfect Real Time) was used for qRT-PCR (TaKaRa, Japan). The PCR cycles were as follows: 55â€‰Â°C for 2â€‰min, followed by an initial denaturation step at 95â€‰Â°C for 30â€‰s, 40â€‰cycles at 95â€‰Â°C for 5â€‰s, 55â€‰Â°C for 15â€‰s, and 72â€‰Â°C for 30â€‰s. 2<sup>-Î"Î"Ct</sup> comparative threshold cycle (Ct) method was used for calculation of relative expression levels [<a href="#CR93">93</a>].</p> </div> <div class="statisticalanalysis" title="sec"> <div class="title" tagx="title" title="title">
Statistical analysis</div> <p id="Par95">Experiments described in this study were repeated three times through completely randomized design. Variance analysis was using SAS/STAT statistical analysis software (SAS Institute, Cary, NC, USA). Data shown in figures were means Â± SDs (Standard deviation), and the different letters represent for significant differences (<i>P</i> <div class="â€‰0.05)._UNKNOWN" /></p> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary Information</div> <div class="sec" title="sec"> <p> </p><div class="local-data" title="local-data"> <div class="media" title="media"><a href="12864_2020_7339_MOESM1_ESM.docx">LINK</a> <p><b>Additional file 1: Table S1</b>. Clean reads quality metrics</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="12864_2020_7339_MOESM2_ESM.docx">LINK</a> <p><b>Additional file 2: Table S2</b>. Quality metrics of transcripts of seeds of <i>Paeonia lactiflora</i> â€˜Hangshaoâ€™</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="12864_2020_7339_MOESM3_ESM.docx">LINK</a> <p><b>Additional file 3: Figure S1</b>. The length distribution of unigenes for seeds of <i>Paeonia lactiflora</i> â€˜Hangshaoâ€™</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="12864_2020_7339_MOESM4_ESM.docx">LINK</a> <p><b>Additional file 4: Figure S2</b>. Functional distribution of unigenes annotated for seeds of <i>Paeonia lactiflora</i> â€˜Hangshaoâ€™</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="12864_2020_7339_MOESM5_ESM.docx">LINK</a> <p><b>Additional file 5: Table S3</b>. Number of DEGs for GO classification</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="12864_2020_7339_MOESM6_ESM.docx">LINK</a> <p><b>Additional file 6: Figure S3</b>. Functional distribution of GO-annotated DEGs for seeds of <i>Paeonia lactiflora</i> â€˜Hangshaoâ€™</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="12864_2020_7339_MOESM7_ESM.docx">LINK</a> <p><b>Additional file 7: Table S4</b>. Number of DEGs for KEGG annotation</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="12864_2020_7339_MOESM8_ESM.xlsx">LINK</a> <p><b>Additional file 8: Table S5</b>. DEGs for KEGG annotation in Groups I, II and III</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="12864_2020_7339_MOESM9_ESM.docx">LINK</a> <p><b>Additional file 9: Table S6</b>. Number of DEGs for KEGG pathway annotation in Groups I, II and III</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="12864_2020_7339_MOESM10_ESM.docx">LINK</a> <p><b>Additional file 10: Table S7</b>. Number of DEGs in 14 KEGG lipid metabolism pathways</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="12864_2020_7339_MOESM11_ESM.docx">LINK</a> <p><b>Additional file 11: Table S8</b>. All DEGs annotated to lipid metabolism</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="12864_2020_7339_MOESM12_ESM.docx">LINK</a> <p><b>Additional file 12: Table S9</b>. One hundred eleven DEGs used for the cluster analysis</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="12864_2020_7339_MOESM13_ESM.docx">LINK</a> <p><b>Additional file 13: Table S10</b>. DEGs related to fatty acid biosynthesis and oil accumulation in herbaceous peony â€˜Hangshaoâ€™ seeds</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="12864_2020_7339_MOESM14_ESM.docx">LINK</a> <p><b>Additional file 14: Table S11</b>. Gene sequences of 17 BCCP in herbaceous peony â€˜Hangshaoâ€™ from tanscriptome database</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="12864_2020_7339_MOESM15_ESM.docx">LINK</a> <p><b>Additional file 15: Table S12</b>. Gene-specific primer sequence for qRT-PCR detection</p>  </div> </div> <p /> </div> </div> </div> <div class="back" title="back"> <div class="glossary" title="glossary"> <div class="title" tagx="title" title="title">
Abbreviations</div> <div class="def-list" title="def-list"> <div class="def-item" title="def-item"> <div class="term" title="term">
DEG</div> <div class="def" title="def"> <p id="Par4">Differentially expressed gene</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
UFA</div> <div class="def" title="def"> <p id="Par5">Unsaturated fatty acid</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
OA</div> <div class="def" title="def"> <p id="Par6">Oleic acid</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
LA</div> <div class="def" title="def"> <p id="Par7">Linoleic acid</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
ALA</div> <div class="def" title="def"> <p id="Par8">Î±-linolenic acid</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
PA</div> <div class="def" title="def"> <p id="Par9">Palmitic acid</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
SA</div> <div class="def" title="def"> <p id="Par10">Stearic acid</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
MUFA</div> <div class="def" title="def"> <p id="Par11">Monounsaturated fatty acid</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
PUFA</div> <div class="def" title="def"> <p id="Par12">Polyunsaturated fatty acid</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
DAF</div> <div class="def" title="def"> <p id="Par13">Days after flowering</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
qRT-PCR</div> <div class="def" title="def"> <p id="Par14">Quantitative real-time PCR</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
GC-MS</div> <div class="def" title="def"> <p id="Par15">Gas chromatography-mass spectrometry</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
GO</div> <div class="def" title="def"> <p id="Par16">Gene ontology</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
KOG</div> <div class="def" title="def"> <p id="Par17">Clusters of orthologous groups</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
KEGG</div> <div class="def" title="def"> <p id="Par18">Kyoto encyclopedia of genes and genomes</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
BC</div> <div class="def" title="def"> <p id="Par19">Biotin carboxylase</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
BCCP</div> <div class="def" title="def"> <p id="Par20">Biotin carboxyl carrier protein</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
Î±-CT</div> <div class="def" title="def"> <p id="Par21">Carboxyl transferase subunit alpha</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
Î²-CT</div> <div class="def" title="def"> <p id="Par22">Carboxyl transferase subunit beta</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
MACT</div> <div class="def" title="def"> <p id="Par23">Malonyl-CoA-acyl carrier protein transacylase</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
KASIII</div> <div class="def" title="def"> <p id="Par24">3-oxoacyl-ACP synthase III</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
KAR</div> <div class="def" title="def"> <p id="Par25">3-oxoacyl-ACP reductase</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
HAD</div> <div class="def" title="def"> <p id="Par26">3-hydroxyacyl-ACP dehydratase</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
EAR</div> <div class="def" title="def"> <p id="Par27">Enoyl-ACP reductase I</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
KASII</div> <div class="def" title="def"> <p id="Par28">3-oxoacyl-ACP synthase II</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
FATB</div> <div class="def" title="def"> <p id="Par29">Fatty acyl-ACP thioesterase B</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
FATA</div> <div class="def" title="def"> <p id="Par30">Fatty acyl-ACP thioesterase A</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
LACS</div> <div class="def" title="def"> <p id="Par31">Long-chain acyl-CoA synthetase</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
KCS</div> <div class="def" title="def"> <p id="Par32">3-ketoacyl-CoA synthase</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
KCR</div> <div class="def" title="def"> <p id="Par33">Very-long-chain 3-oxoacyl-CoA reductase</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
HCD</div> <div class="def" title="def"> <p id="Par34">Very-long-chain (3R)-3-hydroxyacyl-CoA dehydratase</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
ECR</div> <div class="def" title="def"> <p id="Par35">Very-long-chain enoyl-CoA reductase</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
SAD</div> <div class="def" title="def"> <p id="Par36">Stearoyl-ACP desaturase</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
FAD</div> <div class="def" title="def"> <p id="Par37">Fatty acid desaturase</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
LPC</div> <div class="def" title="def"> <p id="Par38">Lysophosphatidylcholine</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
LPCAT</div> <div class="def" title="def"> <p id="Par39">Lysophosphatidylcholine acyltransferase</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
GPAT</div> <div class="def" title="def"> <p id="Par40">Glycerol-3-phosphate acyltransferase</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
LPAAT</div> <div class="def" title="def"> <p id="Par41">1-acyl-sn-glycerol-3-phosphate acyltransferase</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
PAP</div> <div class="def" title="def"> <p id="Par42">Phosphatide phosphatase</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
DGAT</div> <div class="def" title="def"> <p id="Par43">Diacylglycerol O-acyltransferase</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
PDAT</div> <div class="def" title="def"> <p id="Par44">Phospholipid:diacylglycerol acyltransferase</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
PDCT</div> <div class="def" title="def"> <p id="Par45">Phosphatidylcholine: diacylglycerol cholinephosphotransferase</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
TAG</div> <div class="def" title="def"> <p id="Par46">Triacylglycerol</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
OLE</div> <div class="def" title="def"> <p id="Par47">Oleosin</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
CLO</div> <div class="def" title="def"> <p id="Par48">Caleosin</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
FPKM</div> <div class="def" title="def"> <p id="Par49">Fragments per kilobase of exon model per million mapped fragment</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
OB</div> <div class="def" title="def"> <p id="Par50">Oil body</p> </div> </div> </div> </div> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p><b>Publisherâ€™s Note</b></p> <p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p> </div> </div> <div class="ack" title="ack"> <p>We thank BGI (Beijing Genomics Institute, China) for help with the transcriptome sequencing and for technical assistance.</p> </div> <div class="title" tagx="title" title="title">
Authorsâ€™ contributions</div> <p>JT designed and managed the experiments and organized the manuscript. JSM conducted the sequence data analysis and drafted the manuscript. JSM and YHT performed the experiments. JSM, YHT, JS, DQZ and KLZ contributed to the manuscript revision. The authors have carefully read and approved the final manuscript.</p> <div class="title" tagx="title" title="title">
Funding</div> <p>Funding for this study was provided by the National Natural Science Foundation of China (32071813) and Three-Side Innovation Projects for Agriculture in Jiangsu Province (SXGC [2017]297).</p> <div class="title" tagx="title" title="title">
Availability of data and materials</div> <p>All data generated or analyzed during this study are included in the main paper and supplementary information files. In addition, all clean reads were uploaded to the National Center for Biotechnology Information (NCBI) Sequence Read Archive (SRA) database under accession number SRP148668. And all expression data sets and assembled unigenes are available at the NCBI Gene Expression Omnibus (GEO) database under accession number GSE151746.</p> <div class="title" tagx="title" title="title">
Ethics approval and consent to participate</div> <p id="Par96">Herbaceous peony â€˜Hangshaoâ€™ (<i>Paeonia lactiflora</i> â€˜Hangshaoâ€™) is an important ornamental and medicinal plant, the root of which is the traditional Chinese medicine â€˜Radix Paeoniae Albaâ€™, is mainly cultivated in Zhejiang Province and widely cultivated in China. The seeds of herbaceous peony in this study was collected from the peony germplasm resource garden of Yangzhou University. No specific permits are required for sample collection on herbaceous peony â€˜Hangshaoâ€™.</p> <div class="title" tagx="title" title="title">
Consent for publication</div> <p id="Par97">Not applicable.</p> <div class="title" tagx="title" title="title">
Competing interests</div> <p id="Par98">The authors declare that they have no competing interests.</p> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="CR1" /><span class="label" tagx="label" title="label">1.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qing</span><span class="given-names" tagx="given-names" title="given-names">KJ</span></span></span><span class="source" tagx="source" title="source">Illustrtion of one hundred ornamental flowers bonsai-the herbaceous peony</span><span class="year" tagx="year" title="year">2004</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Beijing</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">China Forestry Press</span></span></li> <li tag="ref"><a name="CR2" /><span class="label" tagx="label" title="label">2.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">MX</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">SN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">You</span><span class="given-names" tagx="given-names" title="given-names">HD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Han</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">ZP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">YX</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Simultaneous determination of paeoniflorin and albiflorin in radix paeoniae Rubra by HPLCâ€"DADâ€"ELSD</span><span class="source" tagx="source" title="source">Acta Chromatogr.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">29</span><span class="issue" tagx="issue" title="issue">2</span><span class="fpage" tagx="fpage" title="fpage">279</span><span class="lpage" tagx="lpage" title="lpage">289</span><span class="pub-id"><a href="https://dx.doi.org/10.1556/1326.2017.29.2.12">10.1556/1326.2017.29.2.12</a></span></span></li> <li tag="ref"><a name="CR3" /><span class="label" tagx="label" title="label">3.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bi</span><span class="given-names" tagx="given-names" title="given-names">XX</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qu</span><span class="given-names" tagx="given-names" title="given-names">TG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mu</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guan</span><span class="given-names" tagx="given-names" title="given-names">PP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qu</span><span class="given-names" tagx="given-names" title="given-names">XD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">ZY</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Anti-inflammatory effects, SAR, and action mechanism of monoterpenoids from radix paeoniae Alba on LPS-stimulated RAW 264.7 cells</span><span class="source" tagx="source" title="source">Molecules</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">22</span><span class="fpage" tagx="fpage" title="fpage">715</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/molecules22050715">10.3390/molecules22050715</a></span></span></li> <li tag="ref"><a name="CR4" /><span class="label" tagx="label" title="label">4.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Xie PY, Cui LL, Shan Y, Kang WY. Antithrombotic effect and mechanism of Radix Paeoniae Rubra. BioMed Res Int. 2017; 10.1155/2017/9475074.</span></li> <li tag="ref"><a name="CR5" /><span class="label" tagx="label" title="label">5.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lin</span><span class="given-names" tagx="given-names" title="given-names">MY</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chiang</span><span class="given-names" tagx="given-names" title="given-names">SY</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">YZ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">MF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">YS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">JY</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Anti-tumor effect of Radix Paeoniae Rubra extract on mice bladder tumors using intravesical therapy</span><span class="source" tagx="source" title="source">Oncol Lett.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">12</span><span class="issue" tagx="issue" title="issue">2</span><span class="fpage" tagx="fpage" title="fpage">904</span><span class="lpage" tagx="lpage" title="lpage">910</span><span class="pub-id"><a href="https://dx.doi.org/10.3892/ol.2016.4698">10.3892/ol.2016.4698</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27446367">27446367</a></span></span></li> <li tag="ref"><a name="CR6" /><span class="label" tagx="label" title="label">6.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qiu</span><span class="given-names" tagx="given-names" title="given-names">ZK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">He</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zeng</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheng</span><span class="given-names" tagx="given-names" title="given-names">JS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nie</span><span class="given-names" tagx="given-names" title="given-names">H</span></span></span><span class="mixed-article-title" title="mixed-article-title">Anti-PTSD-like effects of albiflorin extracted from Radix Paeoniae Alba</span><span class="source" tagx="source" title="source">J Ethnopharmacol.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">198</span><span class="fpage" tagx="fpage" title="fpage">324</span><span class="lpage" tagx="lpage" title="lpage">330</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jep.2016.12.028">10.1016/j.jep.2016.12.028</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27993636">27993636</a></span></span></li> <li tag="ref"><a name="CR7" /><span class="label" tagx="label" title="label">7.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiang</span><span class="given-names" tagx="given-names" title="given-names">DX</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">YS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hou</span><span class="given-names" tagx="given-names" title="given-names">XT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">JF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mu</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">W</span></span></span><span class="mixed-article-title" title="mixed-article-title">Influence of <i>Paeonia lactiflora</i> roots extract on cAMP-phosphodiesterase activity and related anti-inflammatory action</span><span class="source" tagx="source" title="source">J Ethnopharmacol.</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">137</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">914</span><span class="lpage" tagx="lpage" title="lpage">920</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jep.2011.07.020">10.1016/j.jep.2011.07.020</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21782011">21782011</a></span></span></li> <li tag="ref"><a name="CR8" /><span class="label" tagx="label" title="label">8.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">JJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Su</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qi</span><span class="given-names" tagx="given-names" title="given-names">XM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">YG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shen</span><span class="given-names" tagx="given-names" title="given-names">JJ</span></span></span><span class="mixed-article-title" title="mixed-article-title">Effect of total glucosides of paeony on the expression of nephrin in the kidneys from diabetic rats</span><span class="source" tagx="source" title="source">Am J Chin Med.</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">37</span><span class="issue" tagx="issue" title="issue">2</span><span class="fpage" tagx="fpage" title="fpage">295</span><span class="lpage" tagx="lpage" title="lpage">307</span><span class="pub-id"><a href="https://dx.doi.org/10.1142/S0192415X09006850">10.1142/S0192415X09006850</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19507273">19507273</a></span></span></li> <li tag="ref"><a name="CR9" /><span class="label" tagx="label" title="label">9.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">JZ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">YM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">XP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span></span><span class="mixed-article-title" title="mixed-article-title">Study on method of Radix Paeonia Alba producing and concocting integration processing</span><span class="source" tagx="source" title="source">China J Chinese Mater Med.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">39</span><span class="issue" tagx="issue" title="issue">13</span><span class="fpage" tagx="fpage" title="fpage">2504</span><span class="lpage" tagx="lpage" title="lpage">2508</span></span></li> <li tag="ref"><a name="CR10" /><span class="label" tagx="label" title="label">10.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meng</span><span class="given-names" tagx="given-names" title="given-names">JS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tao</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">Fatty acid composition and <i>PlFAD</i>s expression related to a-linolenic acid biosynthesis in herbaceous peony (<i>Paeonia lactiflora</i> Pall.)</span><span class="source" tagx="source" title="source">Acta Physiol Plant</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">39</span><span class="fpage" tagx="fpage" title="fpage">222</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s11738-017-2506-6">10.1007/s11738-017-2506-6</a></span></span></li> <li tag="ref"><a name="CR11" /><span class="label" tagx="label" title="label">11.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ning</span><span class="given-names" tagx="given-names" title="given-names">CL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meng</span><span class="given-names" tagx="given-names" title="given-names">JS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">CH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tao</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">Herbaceous peony seed oil: a rich source of unsaturated fatty acids and Î³-tocopheral</span><span class="source" tagx="source" title="source">Eur J Lipid Sci Technol.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">117</span><span class="issue" tagx="issue" title="issue">4</span><span class="fpage" tagx="fpage" title="fpage">532</span><span class="lpage" tagx="lpage" title="lpage">542</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/ejlt.201400212">10.1002/ejlt.201400212</a></span></span></li> <li tag="ref"><a name="CR12" /><span class="label" tagx="label" title="label">12.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thandapilly</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Raj</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Louis</span><span class="given-names" tagx="given-names" title="given-names">XL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perera</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yamanagedara</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zahradka</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Canola oil rich in oleic acid improves diastolic heart function in diet-induced obese rats</span><span class="source" tagx="source" title="source">J Physiol Sci.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">67</span><span class="fpage" tagx="fpage" title="fpage">425</span><span class="lpage" tagx="lpage" title="lpage">430</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s12576-016-0504-x">10.1007/s12576-016-0504-x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27909891">27909891</a></span></span></li> <li tag="ref"><a name="CR13" /><span class="label" tagx="label" title="label">13.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Patterson E, Wall R, Fitzgerald GF, Ross RP, Stanton C. Health implications of high dietary omega-6 polyunsaturated fatty acids. J Nutr Metab. 2012; 10.1155/2012/539426.</span></li> <li tag="ref"><a name="CR14" /><span class="label" tagx="label" title="label">14.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kulkarni</span><span class="given-names" tagx="given-names" title="given-names">KP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Song</span><span class="given-names" tagx="given-names" title="given-names">JT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bilyeu</span><span class="given-names" tagx="given-names" title="given-names">KD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">JD</span></span></span><span class="mixed-article-title" title="mixed-article-title">Genetic improvement of the fatty acid biosynthesis system to alter the Ï‰-6/Ï‰-3 ratio in the soybean seed</span><span class="source" tagx="source" title="source">J Am Oil Chem Soc.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">94</span><span class="fpage" tagx="fpage" title="fpage">1403</span><span class="lpage" tagx="lpage" title="lpage">1410</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s11746-017-3049-0">10.1007/s11746-017-3049-0</a></span></span></li> <li tag="ref"><a name="CR15" /><span class="label" tagx="label" title="label">15.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Domenichiello</span><span class="given-names" tagx="given-names" title="given-names">AF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kitson</span><span class="given-names" tagx="given-names" title="given-names">AP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">CT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">TrÃ©panier</span><span class="given-names" tagx="given-names" title="given-names">MO</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stavro</span><span class="given-names" tagx="given-names" title="given-names">PM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bazinet</span><span class="given-names" tagx="given-names" title="given-names">RP</span></span></span><span class="mixed-article-title" title="mixed-article-title">The effect of linoleic acid on the whole body synthesis rates of polyunsaturated fatty acids from Î±-linolenic acid and linoleic acid in free-living rats</span><span class="source" tagx="source" title="source">J Nutr Biochem.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">30</span><span class="fpage" tagx="fpage" title="fpage">167</span><span class="lpage" tagx="lpage" title="lpage">176</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jnutbio.2015.11.016">10.1016/j.jnutbio.2015.11.016</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27012633">27012633</a></span></span></li> <li tag="ref"><a name="CR16" /><span class="label" tagx="label" title="label">16.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simopoulos</span><span class="given-names" tagx="given-names" title="given-names">AP</span></span></span><span class="mixed-article-title" title="mixed-article-title">The importance of the ratio of omega-6/mogea-3 essential fatty acids</span><span class="source" tagx="source" title="source">Biomed Pharmacother.</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">56</span><span class="fpage" tagx="fpage" title="fpage">365</span><span class="lpage" tagx="lpage" title="lpage">379</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0753-3322(02)00253-6">10.1016/S0753-3322(02)00253-6</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12442909">12442909</a></span></span></li> <li tag="ref"><a name="CR17" /><span class="label" tagx="label" title="label">17.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">SS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">LS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shu</span><span class="given-names" tagx="given-names" title="given-names">QY</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">LG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shao</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Fatty acid composition of developing tree peony (<i>Paeonia</i> section <i>Moutan</i> DC.) seeds and transcriptome analysis during seed development</span><span class="source" tagx="source" title="source">BMC Genomics.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">16</span><span class="fpage" tagx="fpage" title="fpage">208</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s12864-015-1429-0">10.1186/s12864-015-1429-0</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25887415">25887415</a></span></span></li> <li tag="ref"><a name="CR18" /><span class="label" tagx="label" title="label">18.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goettel</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ramirez</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Upchurch</span><span class="given-names" tagx="given-names" title="given-names">RG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">An</span><span class="given-names" tagx="given-names" title="given-names">YC</span></span></span><span class="mixed-article-title" title="mixed-article-title">Identification and characterization of large DNA deletions affecting oil quality traits in soybean seeds through transcriptome sequencing analysis</span><span class="source" tagx="source" title="source">Theor Appl Genet.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">129</span><span class="fpage" tagx="fpage" title="fpage">1577</span><span class="lpage" tagx="lpage" title="lpage">1593</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s00122-016-2725-z">10.1007/s00122-016-2725-z</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27179525">27179525</a></span></span></li> <li tag="ref"><a name="CR19" /><span class="label" tagx="label" title="label">19.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Yin DM, Wang Y, Zhang XG, Li HM, Lu X, Zhang JS, et al. De novo assembly of the peanut (<i>Arachis hypogaea</i> L.) seed transcriptome revealed candidate unigenes for oil accumulation pathways. PLoS One. 2013; 10.1371/journal.pone.0073767.</span></li> <li tag="ref"><a name="CR20" /><span class="label" tagx="label" title="label">20.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bancroft</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morgan</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fraser</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Higgins</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wells</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Clissold</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Dissecting the genome of the polyploidy crop oilseed rape by transciptome sequencing</span><span class="source" tagx="source" title="source">Nat Biotechnol.</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">29</span><span class="fpage" tagx="fpage" title="fpage">762</span><span class="lpage" tagx="lpage" title="lpage">766</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nbt.1926">10.1038/nbt.1926</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21804563">21804563</a></span></span></li> <li tag="ref"><a name="CR21" /><span class="label" tagx="label" title="label">21.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Livaja</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wieckhorst</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haseneyer</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Seidel</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hahn</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">BSTA: a targeted approach combines bulked segregant analysis with next-generation sequencing and <i>de novo</i> transcriptome assembly for SNP discovery in sunflower</span><span class="source" tagx="source" title="source">BMC Genomics.</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">14</span><span class="fpage" tagx="fpage" title="fpage">628</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1471-2164-14-628">10.1186/1471-2164-14-628</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24330545">24330545</a></span></span></li> <li tag="ref"><a name="CR22" /><span class="label" tagx="label" title="label">22.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Bazakos C, Manioudaki ME, Therios I, Voyiatzis D, Kafetzopouls D, Awada T, et al. Comparative transcriptome analysis of two olive cultivars in response to NaCl-stress. PLoS One. 2012; 10.1371/journal.pone.0042931.</span></li> <li tag="ref"><a name="CR23" /><span class="label" tagx="label" title="label">23.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Xia EH, Jiang JJ, Huang H, Zhang LP, Zhang HB, Gao LZ. Transcriptome analysis of the oil-rich tea plant, <i>Camellia oleifera</i>, reveals candidate genes related to lipid metabolism. PLoS One. 2014; 10.1371/journal.pone.0104150.</span></li> <li tag="ref"><a name="CR24" /><span class="label" tagx="label" title="label">24.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tian</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">TQ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">RL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">GQ</span></span></span><span class="mixed-article-title" title="mixed-article-title">Identification of genes associated with ricinoleic acid accumulation in <i>Hiptage benghalensis</i> via transcriptome analysis</span><span class="source" tagx="source" title="source">Biotechnol Biofuels.</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">12</span><span class="fpage" tagx="fpage" title="fpage">16</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s13068-019-1358-2">10.1186/s13068-019-1358-2</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30679955">30679955</a></span></span></li> <li tag="ref"><a name="CR25" /><span class="label" tagx="label" title="label">25.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hao</span><span class="given-names" tagx="given-names" title="given-names">ZJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wei</span><span class="given-names" tagx="given-names" title="given-names">MR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gong</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">DQ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jun</span><span class="given-names" tagx="given-names" title="given-names">T</span></span></span><span class="mixed-article-title" title="mixed-article-title">Transcriptome and digital gene expression analysis of herbaceous peony (<i>Paeonia lactiflora</i> Pall.) to screen thermo-tolerant related differently expressed genes</span><span class="source" tagx="source" title="source">Genes Genom.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">38</span><span class="fpage" tagx="fpage" title="fpage">1201</span><span class="lpage" tagx="lpage" title="lpage">1215</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s13258-016-0465-8">10.1007/s13258-016-0465-8</a></span></span></li> <li tag="ref"><a name="CR26" /><span class="label" tagx="label" title="label">26.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">YQ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">DQ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tao</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">Analysis of codon usage patterns in herbaceous peony (<i>Paeonia lactiflora</i> Pall.) based on transcriptome data</span><span class="source" tagx="source" title="source">Genes.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">6</span><span class="issue" tagx="issue" title="issue">4</span><span class="fpage" tagx="fpage" title="fpage">1125</span><span class="lpage" tagx="lpage" title="lpage">1139</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/genes6041125">10.3390/genes6041125</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26506393">26506393</a></span></span></li> <li tag="ref"><a name="CR27" /><span class="label" tagx="label" title="label">27.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">DQ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ning</span><span class="given-names" tagx="given-names" title="given-names">CL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meng</span><span class="given-names" tagx="given-names" title="given-names">JS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lin</span><span class="given-names" tagx="given-names" title="given-names">SS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ding</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Transcriptome sequencing of a chimaera reveals coordinated expression of anthocyanin biosynthetic genes mediating yellow formation in herbaceous peony (<i>Paeonia lactiflora</i> Pall.)</span><span class="source" tagx="source" title="source">BMC Genomics.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">15</span><span class="fpage" tagx="fpage" title="fpage">689</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1471-2164-15-689">10.1186/1471-2164-15-689</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25134523">25134523</a></span></span></li> <li tag="ref"><a name="CR28" /><span class="label" tagx="label" title="label">28.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jaworski</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cahoon</span><span class="given-names" tagx="given-names" title="given-names">EB</span></span></span><span class="mixed-article-title" title="mixed-article-title">Industrial oils from transgenic plants</span><span class="source" tagx="source" title="source">Curr Opin Plant Biol.</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">6</span><span class="issue" tagx="issue" title="issue">2</span><span class="fpage" tagx="fpage" title="fpage">178</span><span class="lpage" tagx="lpage" title="lpage">184</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S1369-5266(03)00013-X">10.1016/S1369-5266(03)00013-X</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12667876">12667876</a></span></span></li> <li tag="ref"><a name="CR29" /><span class="label" tagx="label" title="label">29.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Spector</span><span class="given-names" tagx="given-names" title="given-names">AA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">HY</span></span></span><span class="mixed-article-title" title="mixed-article-title">Discovery of essential fatty acids</span><span class="source" tagx="source" title="source">J Lipid Res.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">56</span><span class="fpage" tagx="fpage" title="fpage">11</span><span class="lpage" tagx="lpage" title="lpage">21</span><span class="pub-id"><a href="https://dx.doi.org/10.1194/jlr.R055095">10.1194/jlr.R055095</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25339684">25339684</a></span></span></li> <li tag="ref"><a name="CR30" /><span class="label" tagx="label" title="label">30.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">ZG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">LM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">HL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">LJ</span></span></span><span class="mixed-article-title" title="mixed-article-title">Recent advances in understanding the effects of polyunsaturated fatty acids on brain function</span><span class="source" tagx="source" title="source">Food Sci.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">36</span><span class="issue" tagx="issue" title="issue">21</span><span class="fpage" tagx="fpage" title="fpage">284</span><span class="lpage" tagx="lpage" title="lpage">290</span></span></li> <li tag="ref"><a name="CR31" /><span class="label" tagx="label" title="label">31.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bougnoux</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maillard</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">ChajÃ¨s</span><span class="given-names" tagx="given-names" title="given-names">V</span></span></span><span class="mixed-article-title" title="mixed-article-title">Omega-6/omega-3 polyunsaturated fatty acid ratio and breast cancer</span><span class="source" tagx="source" title="source">World Rev Nutr Diet.</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">94</span><span class="fpage" tagx="fpage" title="fpage">158</span><span class="lpage" tagx="lpage" title="lpage">165</span><span class="pub-id"><a href="https://dx.doi.org/10.1159/000088235">10.1159/000088235</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16145262">16145262</a></span></span></li> <li tag="ref"><a name="CR32" /><span class="label" tagx="label" title="label">32.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Russo</span><span class="given-names" tagx="given-names" title="given-names">GL</span></span></span><span class="mixed-article-title" title="mixed-article-title">Dietary n-6 and n-3 polyunsaturated fatty acids: from biochemistry to clinical implications in cardiovascular prevention</span><span class="source" tagx="source" title="source">Biochem Pharmacol.</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">77</span><span class="fpage" tagx="fpage" title="fpage">937</span><span class="lpage" tagx="lpage" title="lpage">946</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.bcp.2008.10.020">10.1016/j.bcp.2008.10.020</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19022225">19022225</a></span></span></li> <li tag="ref"><a name="CR33" /><span class="label" tagx="label" title="label">33.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jin</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lu</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pang</span><span class="given-names" tagx="given-names" title="given-names">TT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">TT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yuan</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Effects of dietary n-3 LC-PUFA/n-6 C<sub>18</sub> PUFA ratio on growth, feed utilization, fatty acid composition and lipid metabolism related gene expression in black seabream, <i>Acanthopagrus schlegelil</i></span><span class="source" tagx="source" title="source">Aquaculture.</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">500</span><span class="fpage" tagx="fpage" title="fpage">521</span><span class="lpage" tagx="lpage" title="lpage">531</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.aquaculture.2018.10.056">10.1016/j.aquaculture.2018.10.056</a></span></span></li> <li tag="ref"><a name="CR34" /><span class="label" tagx="label" title="label">34.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guschina</span><span class="given-names" tagx="given-names" title="given-names">IA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Everard</span><span class="given-names" tagx="given-names" title="given-names">JD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kinney</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Quant</span><span class="given-names" tagx="given-names" title="given-names">PA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harwood</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span></span><span class="mixed-article-title" title="mixed-article-title">Studies on the regulation of lipid biosynthesis in plants: application of control analysis to soybean</span><span class="source" tagx="source" title="source">Biochim Biophys Acta.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">1838</span><span class="fpage" tagx="fpage" title="fpage">1488</span><span class="lpage" tagx="lpage" title="lpage">1500</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.bbamem.2014.02.008">10.1016/j.bbamem.2014.02.008</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24565795">24565795</a></span></span></li> <li tag="ref"><a name="CR35" /><span class="label" tagx="label" title="label">35.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chandran</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sankararamasubramanian</span><span class="given-names" tagx="given-names" title="given-names">HM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kumar</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parida</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">Differential expression analysis of transcripts related to oil metabolism in maturing seeds of <i>Jatropha curcas</i> L</span><span class="source" tagx="source" title="source">Physiol Mol Biol Plants.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">20</span><span class="fpage" tagx="fpage" title="fpage">181</span><span class="lpage" tagx="lpage" title="lpage">190</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s12298-013-0216-0">10.1007/s12298-013-0216-0</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24757322">24757322</a></span></span></li> <li tag="ref"><a name="CR36" /><span class="label" tagx="label" title="label">36.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Savadi</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lambani</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kashyap</span><span class="given-names" tagx="given-names" title="given-names">PL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bisht</span><span class="given-names" tagx="given-names" title="given-names">DS</span></span></span><span class="mixed-article-title" title="mixed-article-title">Genetic engineering approaches to enhance oil content in oilseed crops</span><span class="source" tagx="source" title="source">Plant Growth Regul.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">83</span><span class="fpage" tagx="fpage" title="fpage">207</span><span class="lpage" tagx="lpage" title="lpage">222</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s10725-016-0236-1">10.1007/s10725-016-0236-1</a></span></span></li> <li tag="ref"><a name="CR37" /><span class="label" tagx="label" title="label">37.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">TQ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">ZH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">AZ</span></span></span><span class="mixed-article-title" title="mixed-article-title">Transcriptome analysis reveals crucial genes involved in the biosynthesis of nervonic acid in woody <i>Malania oleifera</i> oilseeds</span><span class="source" tagx="source" title="source">BMC Plant Biol.</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">18</span><span class="fpage" tagx="fpage" title="fpage">247</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s12870-018-1463-6">10.1186/s12870-018-1463-6</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30340521">30340521</a></span></span></li> <li tag="ref"><a name="CR38" /><span class="label" tagx="label" title="label">38.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">QY</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">DY</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rahman</span><span class="given-names" tagx="given-names" title="given-names">MM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xie</span><span class="given-names" tagx="given-names" title="given-names">LH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">JY</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Comparative transcriptome analysis reveals an efficient mechanism for Î±-linolenic acid synthesis in tree peony seeds</span><span class="source" tagx="source" title="source">Int J Mol Sci.</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">20</span><span class="fpage" tagx="fpage" title="fpage">65</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/ijms20010065">10.3390/ijms20010065</a></span></span></li> <li tag="ref"><a name="CR39" /><span class="label" tagx="label" title="label">39.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">GQ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xia</span><span class="given-names" tagx="given-names" title="given-names">YP</span></span></span><span class="mixed-article-title" title="mixed-article-title">Transcriptomic analysis of the underground renewal buds during dormancy transition and release in 'Hangbaishao' peony (<i>Paeonia lactiflora</i>)</span><span class="source" tagx="source" title="source">PLoS One.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">10</span><span class="issue" tagx="issue" title="issue">3</span><span class="fpage" tagx="fpage" title="fpage">e0119118</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0119118">10.1371/journal.pone.0119118</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25790307">25790307</a></span></span></li> <li tag="ref"><a name="CR40" /><span class="label" tagx="label" title="label">40.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">DQ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tang</span><span class="given-names" tagx="given-names" title="given-names">YH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xia</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meng</span><span class="given-names" tagx="given-names" title="given-names">JS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shang</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Integration of transcriptome, proteome, and metabolome provides insights into how calcium enhances the mechanical strength of herbaceous peony inflorescence stems</span><span class="source" tagx="source" title="source">Cells.</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">102</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/cells8020102">10.3390/cells8020102</a></span></span></li> <li tag="ref"><a name="CR41" /><span class="label" tagx="label" title="label">41.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ma</span><span class="given-names" tagx="given-names" title="given-names">YL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cui</span><span class="given-names" tagx="given-names" title="given-names">JQ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lu</span><span class="given-names" tagx="given-names" title="given-names">XJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">LJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">ZJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fei</span><span class="given-names" tagx="given-names" title="given-names">RW</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Transcirptome analysis of two different developmental stages of <i>Paeonia lactiflora</i> seeds</span><span class="source" tagx="source" title="source">Int J Genomics.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">2017</span><span class="fpage" tagx="fpage" title="fpage">8027626</span><span class="pub-id"><a href="https://dx.doi.org/10.1155/2017/8027626">10.1155/2017/8027626</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28848760">28848760</a></span></span></li> <li tag="ref"><a name="CR42" /><span class="label" tagx="label" title="label">42.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grabherr</span><span class="given-names" tagx="given-names" title="given-names">MG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haas</span><span class="given-names" tagx="given-names" title="given-names">BJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yassour</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Levin</span><span class="given-names" tagx="given-names" title="given-names">JZ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thompson</span><span class="given-names" tagx="given-names" title="given-names">DA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Amit</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Full-length transcriptome assembly from RNA-Seq data without a reference genome</span><span class="source" tagx="source" title="source">Nat Biotechnol.</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">29</span><span class="issue" tagx="issue" title="issue">7</span><span class="fpage" tagx="fpage" title="fpage">644</span><span class="lpage" tagx="lpage" title="lpage">652</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nbt.1883">10.1038/nbt.1883</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21572440">21572440</a></span></span></li> <li tag="ref"><a name="CR43" /><span class="label" tagx="label" title="label">43.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pertea</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">XQ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liang</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Antonescu</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sultana</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Karamycheva</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">TIGR gene indices clustering tools (TGICL): a software system for fast clustering of large EST datasets</span><span class="source" tagx="source" title="source">Bioinformatics.</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">19</span><span class="issue" tagx="issue" title="issue">5</span><span class="fpage" tagx="fpage" title="fpage">651</span><span class="lpage" tagx="lpage" title="lpage">652</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btg034">10.1093/bioinformatics/btg034</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12651724">12651724</a></span></span></li> <li tag="ref"><a name="CR44" /><span class="label" tagx="label" title="label">44.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zafar</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">YL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">NN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">KM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tan</span><span class="given-names" tagx="given-names" title="given-names">XL</span></span></span><span class="mixed-article-title" title="mixed-article-title">Recent advances in enhancement of oil content in oilseed plants</span><span class="source" tagx="source" title="source">J Biotechnol.</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">301</span><span class="fpage" tagx="fpage" title="fpage">35</span><span class="lpage" tagx="lpage" title="lpage">44</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jbiotec.2019.05.307">10.1016/j.jbiotec.2019.05.307</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31158409">31158409</a></span></span></li> <li tag="ref"><a name="CR45" /><span class="label" tagx="label" title="label">45.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ohlrogge</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Browse</span><span class="given-names" tagx="given-names" title="given-names">H</span></span></span><span class="mixed-article-title" title="mixed-article-title">Lipid biosynthesis</span><span class="source" tagx="source" title="source">Plant Cell.</span><span class="year" tagx="year" title="year">1995</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">957</span><span class="lpage" tagx="lpage" title="lpage">970</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7640528">7640528</a></span></span></li> <li tag="ref"><a name="CR46" /><span class="label" tagx="label" title="label">46.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simon</span><span class="given-names" tagx="given-names" title="given-names">JW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Slabas</span><span class="given-names" tagx="given-names" title="given-names">AR</span></span></span><span class="mixed-article-title" title="mixed-article-title">cDNA cloning of <i>Brassica napus</i> malonyl-CoA:ACP transacylase (MCAT) (<i>fab</i> D) and complementation of an <i>E. coli</i> MCAT mutant</span><span class="source" tagx="source" title="source">FEBS Lett</span><span class="year" tagx="year" title="year">1998</span><span class="volume" tagx="volume" title="volume">435</span><span class="fpage" tagx="fpage" title="fpage">204</span><span class="lpage" tagx="lpage" title="lpage">206</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0014-5793(98)01055-2">10.1016/S0014-5793(98)01055-2</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9762909">9762909</a></span></span></li> <li tag="ref"><a name="CR47" /><span class="label" tagx="label" title="label">47.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">GonzÃ¡lez-Mellado</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Westtstein-Knowles</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">GarcÃ©s</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">MartÃ­nez-Force</span><span class="given-names" tagx="given-names" title="given-names">E</span></span></span><span class="mixed-article-title" title="mixed-article-title">The role of Î²-ketoacyl-acyl carrier protein synthase III in the condensation steps of fatty acid biosynthesis in sunflower</span><span class="source" tagx="source" title="source">Planta.</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">231</span><span class="fpage" tagx="fpage" title="fpage">1277</span><span class="lpage" tagx="lpage" title="lpage">1289</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s00425-010-1131-z">10.1007/s00425-010-1131-z</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20221630">20221630</a></span></span></li> <li tag="ref"><a name="CR48" /><span class="label" tagx="label" title="label">48.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">MR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">PZ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tian</span><span class="given-names" tagx="given-names" title="given-names">CE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiang</span><span class="given-names" tagx="given-names" title="given-names">HW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">GJ</span></span></span><span class="mixed-article-title" title="mixed-article-title">Molecular cloning and expression analysis of a gene encoding a putative beta-ketoacyl-acyl carrier protein (ACP) synthase III (KAS III) from <i>Jatropha curcas</i></span><span class="source" tagx="source" title="source">Tree Physiol.</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">28</span><span class="fpage" tagx="fpage" title="fpage">921</span><span class="lpage" tagx="lpage" title="lpage">927</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/treephys/28.6.921">10.1093/treephys/28.6.921</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18381272">18381272</a></span></span></li> <li tag="ref"><a name="CR49" /><span class="label" tagx="label" title="label">49.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Choi</span><span class="given-names" tagx="given-names" title="given-names">KH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heath</span><span class="given-names" tagx="given-names" title="given-names">RJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rock</span><span class="given-names" tagx="given-names" title="given-names">CO</span></span></span><span class="mixed-article-title" title="mixed-article-title">Î²-Ketoacyl-acyl carrier protein synthase III (FabH) is a determining factor in branched-chain fatty acid biosynthesis</span><span class="source" tagx="source" title="source">J Bacteriol.</span><span class="year" tagx="year" title="year">2000</span><span class="volume" tagx="volume" title="volume">182</span><span class="fpage" tagx="fpage" title="fpage">365</span><span class="lpage" tagx="lpage" title="lpage">370</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/JB.182.2.365-370.2000">10.1128/JB.182.2.365-370.2000</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10629181">10629181</a></span></span></li> <li tag="ref"><a name="CR50" /><span class="label" tagx="label" title="label">50.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abbadi</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brummel</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">SchÃ¼tt</span><span class="given-names" tagx="given-names" title="given-names">BS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Slabaugh</span><span class="given-names" tagx="given-names" title="given-names">MB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schuch</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Spener</span><span class="given-names" tagx="given-names" title="given-names">F</span></span></span><span class="mixed-article-title" title="mixed-article-title">Reaction mechanism of recombinant 3-oxoacyl-(acyl-carrier-protein) synthase III from <i>Cuphea wrightii</i> embryo, a fatty acid synthase tyepe II condensing enzyme</span><span class="source" tagx="source" title="source">Biochem J.</span><span class="year" tagx="year" title="year">2000</span><span class="volume" tagx="volume" title="volume">345</span><span class="fpage" tagx="fpage" title="fpage">153</span><span class="lpage" tagx="lpage" title="lpage">160</span><span class="pub-id"><a href="https://dx.doi.org/10.1042/bj3450153">10.1042/bj3450153</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10600651">10600651</a></span></span></li> <li tag="ref"><a name="CR51" /><span class="label" tagx="label" title="label">51.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">BrÃ¼ck</span><span class="given-names" tagx="given-names" title="given-names">FM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brummel</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schuch</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Spener</span><span class="given-names" tagx="given-names" title="given-names">F</span></span></span><span class="mixed-article-title" title="mixed-article-title">In-vitro evidence for feed-back regulation of Î²-ketoacyl-acyl carrier protein synthase III in medium-chain fatty acid biosynthesis</span><span class="source" tagx="source" title="source">Planta.</span><span class="year" tagx="year" title="year">1996</span><span class="volume" tagx="volume" title="volume">198</span><span class="fpage" tagx="fpage" title="fpage">271</span><span class="lpage" tagx="lpage" title="lpage">278</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/BF00206253">10.1007/BF00206253</a></span></span></li> <li tag="ref"><a name="CR52" /><span class="label" tagx="label" title="label">52.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dehesh</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tai</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Edwards</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Byrne</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jaworski</span><span class="given-names" tagx="given-names" title="given-names">JG</span></span></span><span class="mixed-article-title" title="mixed-article-title">Overexpression of 3-ketoacyl-acyl-carrier protein synthase IIIs in plants reduces the rate of lipid synthesis</span><span class="source" tagx="source" title="source">Plant Physiol.</span><span class="year" tagx="year" title="year">2001</span><span class="volume" tagx="volume" title="volume">125</span><span class="fpage" tagx="fpage" title="fpage">1103</span><span class="lpage" tagx="lpage" title="lpage">1114</span><span class="pub-id"><a href="https://dx.doi.org/10.1104/pp.125.2.1103">10.1104/pp.125.2.1103</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11161065">11161065</a></span></span></li> <li tag="ref"><a name="CR53" /><span class="label" tagx="label" title="label">53.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ohlrogge</span><span class="given-names" tagx="given-names" title="given-names">JB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jaworski</span><span class="given-names" tagx="given-names" title="given-names">JG</span></span></span><span class="mixed-article-title" title="mixed-article-title">Regulation of fatty acid synthesis</span><span class="source" tagx="source" title="source">Annu Rev Plant Biol.</span><span class="year" tagx="year" title="year">1997</span><span class="volume" tagx="volume" title="volume">48</span><span class="fpage" tagx="fpage" title="fpage">109</span><span class="lpage" tagx="lpage" title="lpage">136</span><span class="pub-id"><a href="https://dx.doi.org/10.1146/annurev.arplant.48.1.109">10.1146/annurev.arplant.48.1.109</a></span></span></li> <li tag="ref"><a name="CR54" /><span class="label" tagx="label" title="label">54.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gibson</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Falcone</span><span class="given-names" tagx="given-names" title="given-names">DL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Browse</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Somerville</span><span class="given-names" tagx="given-names" title="given-names">C</span></span></span><span class="mixed-article-title" title="mixed-article-title">Use of transgenic plants and mutants to study the regulation and function of lipid composition</span><span class="source" tagx="source" title="source">Plant Cell Environ.</span><span class="year" tagx="year" title="year">1994</span><span class="volume" tagx="volume" title="volume">17</span><span class="fpage" tagx="fpage" title="fpage">627</span><span class="lpage" tagx="lpage" title="lpage">637</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1365-3040.1994.tb00154.x">10.1111/j.1365-3040.1994.tb00154.x</a></span></span></li> <li tag="ref"><a name="CR55" /><span class="label" tagx="label" title="label">55.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Salas</span><span class="given-names" tagx="given-names" title="given-names">JJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ohlrogge</span><span class="given-names" tagx="given-names" title="given-names">JB</span></span></span><span class="mixed-article-title" title="mixed-article-title">Characterization of substrate specificity of plant FatA and FatB acyl-ACP thioesterases</span><span class="source" tagx="source" title="source">Arch Biochem Biophys.</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">403</span><span class="fpage" tagx="fpage" title="fpage">25</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0003-9861(02)00017-6">10.1016/S0003-9861(02)00017-6</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12061798">12061798</a></span></span></li> <li tag="ref"><a name="CR56" /><span class="label" tagx="label" title="label">56.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yukawa</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Takaiwa</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shoji</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yamada Kyoji</span><span class="given-names" tagx="given-names" title="given-names">MK</span></span></span><span class="mixed-article-title" title="mixed-article-title">Structure and expression of two seed-specific cDNA clones encoding stearoyl-acyl carrier protein desaturase from sesame (<i>Sesamum indicum</i> L)</span><span class="source" tagx="source" title="source">Plant Cell Physiol.</span><span class="year" tagx="year" title="year">1996</span><span class="volume" tagx="volume" title="volume">37</span><span class="fpage" tagx="fpage" title="fpage">201</span><span class="lpage" tagx="lpage" title="lpage">205</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/oxfordjournals.pcp.a028932">10.1093/oxfordjournals.pcp.a028932</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8665096">8665096</a></span></span></li> <li tag="ref"><a name="CR57" /><span class="label" tagx="label" title="label">57.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kachroo</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lapchyk</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fukushige</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hildebrand</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Klessig</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kachroo</span><span class="given-names" tagx="given-names" title="given-names">P</span></span></span><span class="mixed-article-title" title="mixed-article-title">Plastidial fatty acid signaling modulates salicylic acid- and jasmonic acid-mediated defense pathways in the <i>Arabidopsis ssi2</i> mutant</span><span class="source" tagx="source" title="source">Plant Cell.</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">15</span><span class="fpage" tagx="fpage" title="fpage">2952</span><span class="lpage" tagx="lpage" title="lpage">2965</span><span class="pub-id"><a href="https://dx.doi.org/10.1105/tpc.017301">10.1105/tpc.017301</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14615603">14615603</a></span></span></li> <li tag="ref"><a name="CR58" /><span class="label" tagx="label" title="label">58.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Klinkenberg</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Faist</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saupe</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lambertz</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krischke</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stingl</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Two fatty acid desaturases, STEAROYL-ACYL CARRIER PROTEIN â–³<sup>9</sup>-DESATURASE6 and FATTY ACID DESATURASE3, are involved in drought and hypoxia stress signaling in <i>Arabidopis</i> crown galls</span><span class="source" tagx="source" title="source">Plant Physiol.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">164</span><span class="fpage" tagx="fpage" title="fpage">570</span><span class="lpage" tagx="lpage" title="lpage">583</span><span class="pub-id"><a href="https://dx.doi.org/10.1104/pp.113.230326">10.1104/pp.113.230326</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24368335">24368335</a></span></span></li> <li tag="ref"><a name="CR59" /><span class="label" tagx="label" title="label">59.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xia</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qin</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Identification and expression of a stearoyl-ACP desaturase gene responsible for oleic acid accumulation in <i>Xanthoceras sorbifolia</i> seeds</span><span class="source" tagx="source" title="source">Plant Physiol Biochem.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">87</span><span class="fpage" tagx="fpage" title="fpage">9</span><span class="lpage" tagx="lpage" title="lpage">16</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.plaphy.2014.12.009">10.1016/j.plaphy.2014.12.009</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25528221">25528221</a></span></span></li> <li tag="ref"><a name="CR60" /><span class="label" tagx="label" title="label">60.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Okuley</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lightner</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Feldmann</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yadav</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lark</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Browse</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">Arabidopsis FAD2 gene encodes the enzyme that is essential for polyunsaturated lipid synthesis</span><span class="source" tagx="source" title="source">Plant Cell.</span><span class="year" tagx="year" title="year">1994</span><span class="volume" tagx="volume" title="volume">6</span><span class="fpage" tagx="fpage" title="fpage">147</span><span class="lpage" tagx="lpage" title="lpage">158</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7907506">7907506</a></span></span></li> <li tag="ref"><a name="CR61" /><span class="label" tagx="label" title="label">61.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hitz</span><span class="given-names" tagx="given-names" title="given-names">WD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carlson</span><span class="given-names" tagx="given-names" title="given-names">TJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Booth</span><span class="given-names" tagx="given-names" title="given-names">JR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kinney</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stecca</span><span class="given-names" tagx="given-names" title="given-names">KL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yadav</span><span class="given-names" tagx="given-names" title="given-names">NS</span></span></span><span class="mixed-article-title" title="mixed-article-title">Cloning of a higher-plant plastid Ï‰-6 fatty acid desaturase cDNA and its expression in a cyanobacterium</span><span class="source" tagx="source" title="source">Plant Physiol.</span><span class="year" tagx="year" title="year">1994</span><span class="volume" tagx="volume" title="volume">105</span><span class="fpage" tagx="fpage" title="fpage">635</span><span class="lpage" tagx="lpage" title="lpage">641</span><span class="pub-id"><a href="https://dx.doi.org/10.1104/pp.105.2.635">10.1104/pp.105.2.635</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8066133">8066133</a></span></span></li> <li tag="ref"><a name="CR62" /><span class="label" tagx="label" title="label">62.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Iba</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gibson</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nishiuchi</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fuse</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nishimura</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arondel</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">A gene encoding a chloroplast Ï‰-3 fatty acid desaturase complements alterations in fatty acid desaturation and chloroplast copy number of the fad7 mutant of <i>Arabidopsis thaliana</i></span><span class="source" tagx="source" title="source">J Biol Chem.</span><span class="year" tagx="year" title="year">1993</span><span class="volume" tagx="volume" title="volume">268</span><span class="fpage" tagx="fpage" title="fpage">24099</span><span class="lpage" tagx="lpage" title="lpage">24105</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0021-9258(20)80498-0">10.1016/S0021-9258(20)80498-0</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8226956">8226956</a></span></span></li> <li tag="ref"><a name="CR63" /><span class="label" tagx="label" title="label">63.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martin</span><span class="given-names" tagx="given-names" title="given-names">BA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rinne</span><span class="given-names" tagx="given-names" title="given-names">RW</span></span></span><span class="mixed-article-title" title="mixed-article-title">A comparison of oleic acid metabolism in the soybean (<i>Glycine max</i> [L.] Merr.) genotypes Williams and A5, a mutant with decreased linoleic acid in the seed</span><span class="source" tagx="source" title="source">Plant Physiol.</span><span class="year" tagx="year" title="year">1986</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">41</span><span class="lpage" tagx="lpage" title="lpage">44</span><span class="pub-id"><a href="https://dx.doi.org/10.1104/pp.81.1.41">10.1104/pp.81.1.41</a></span></span></li> <li tag="ref"><a name="CR64" /><span class="label" tagx="label" title="label">64.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miller</span><span class="given-names" tagx="given-names" title="given-names">JF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zimmerman</span><span class="given-names" tagx="given-names" title="given-names">DC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vick</span><span class="given-names" tagx="given-names" title="given-names">BA</span></span></span><span class="mixed-article-title" title="mixed-article-title">Genetic control of high oleic acid content in sunflower oil</span><span class="source" tagx="source" title="source">Crop Sci.</span><span class="year" tagx="year" title="year">1987</span><span class="volume" tagx="volume" title="volume">27</span><span class="fpage" tagx="fpage" title="fpage">923</span><span class="lpage" tagx="lpage" title="lpage">926</span><span class="pub-id"><a href="https://dx.doi.org/10.2135/cropsci1987.0011183X002700050019x">10.2135/cropsci1987.0011183X002700050019x</a></span></span></li> <li tag="ref"><a name="CR65" /><span class="label" tagx="label" title="label">65.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Patel</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jung</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moore</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Powell</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ainsworth</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abbott</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">High-oleate peanut mutants results from a MITE insertion in to FAD2 gene</span><span class="source" tagx="source" title="source">Theor Appl Genet.</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">108</span><span class="fpage" tagx="fpage" title="fpage">1492</span><span class="lpage" tagx="lpage" title="lpage">1502</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s00122-004-1590-3">10.1007/s00122-004-1590-3</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14968307">14968307</a></span></span></li> <li tag="ref"><a name="CR66" /><span class="label" tagx="label" title="label">66.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shah</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xin</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Browse</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">Overexpression of the <i>FAD3</i> desaturase gene in a mutant of <i>Arabidopsis</i></span><span class="source" tagx="source" title="source">Plant Physiol.</span><span class="year" tagx="year" title="year">1997</span><span class="volume" tagx="volume" title="volume">114</span><span class="fpage" tagx="fpage" title="fpage">1533</span><span class="lpage" tagx="lpage" title="lpage">1539</span><span class="pub-id"><a href="https://dx.doi.org/10.1104/pp.114.4.1533">10.1104/pp.114.4.1533</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9276960">9276960</a></span></span></li> <li tag="ref"><a name="CR67" /><span class="label" tagx="label" title="label">67.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">HL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yin</span><span class="given-names" tagx="given-names" title="given-names">ZJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xiao</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">YN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qu</span><span class="given-names" tagx="given-names" title="given-names">LQ</span></span></span><span class="mixed-article-title" title="mixed-article-title">Identification and evaluation of Ï‰-3 fatty acid desaturase genes for hyperfortifying Î±-linolenic acid in transgenic rice seed</span><span class="source" tagx="source" title="source">J Exp Bot.</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">63</span><span class="fpage" tagx="fpage" title="fpage">3279</span><span class="lpage" tagx="lpage" title="lpage">3287</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/jxb/ers051">10.1093/jxb/ers051</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22378946">22378946</a></span></span></li> <li tag="ref"><a name="CR68" /><span class="label" tagx="label" title="label">68.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">AHC</span></span></span><span class="mixed-article-title" title="mixed-article-title">Oil bodies and oleosins in seeds</span><span class="source" tagx="source" title="source">Annu Rev Plant Physiol Plant Mol Biol.</span><span class="year" tagx="year" title="year">1992</span><span class="volume" tagx="volume" title="volume">43</span><span class="fpage" tagx="fpage" title="fpage">177</span><span class="lpage" tagx="lpage" title="lpage">200</span><span class="pub-id"><a href="https://dx.doi.org/10.1146/annurev.pp.43.060192.001141">10.1146/annurev.pp.43.060192.001141</a></span></span></li> <li tag="ref"><a name="CR69" /><span class="label" tagx="label" title="label">69.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gibellini</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">TK</span></span></span><span class="mixed-article-title" title="mixed-article-title">The Kennedy pathway-<i>De novo</i> synthesis of phosphatidylethanolamine and phosphatidylcholine</span><span class="source" tagx="source" title="source">IUBMB Life.</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">62</span><span class="fpage" tagx="fpage" title="fpage">414</span><span class="lpage" tagx="lpage" title="lpage">428</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/iub.354">10.1002/iub.354</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20503434">20503434</a></span></span></li> <li tag="ref"><a name="CR70" /><span class="label" tagx="label" title="label">70.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Settlage</span><span class="given-names" tagx="given-names" title="given-names">SB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kwanyuen</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wilson</span><span class="given-names" tagx="given-names" title="given-names">RF</span></span></span><span class="mixed-article-title" title="mixed-article-title">Relation between diacylglycerol acyltransferase activity and oil concentrationin soybean</span><span class="source" tagx="source" title="source">J Am Oil Chem Soc.</span><span class="year" tagx="year" title="year">1998</span><span class="volume" tagx="volume" title="volume">75</span><span class="fpage" tagx="fpage" title="fpage">775</span><span class="lpage" tagx="lpage" title="lpage">781</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s11746-998-0225-2">10.1007/s11746-998-0225-2</a></span></span></li> <li tag="ref"><a name="CR71" /><span class="label" tagx="label" title="label">71.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">GajdoÅ¡</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ledesma-Amaro</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nicaud</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">ÄŒertÃ­k</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rossignol</span><span class="given-names" tagx="given-names" title="given-names">T</span></span></span><span class="mixed-article-title" title="mixed-article-title">Overexpression of diacylglycerol acyltransferase in Yarrowia lipolytica affects lipid body size, number and distribution</span><span class="source" tagx="source" title="source">FEMS Yeast Res</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">16</span><span class="fpage" tagx="fpage" title="fpage">fow062</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/femsyr/fow062">10.1093/femsyr/fow062</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27506614">27506614</a></span></span></li> <li tag="ref"><a name="CR72" /><span class="label" tagx="label" title="label">72.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saha</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Enugutti</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rajakumari</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rajasekharan</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="mixed-article-title" title="mixed-article-title">Cytosolic triacylglycerol biosynthetic pathway in oilseeds. Molecular cloning and expression of peanut cytosolic diacylglycerol acyltransferase</span><span class="source" tagx="source" title="source">Plant Physiol.</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">141</span><span class="fpage" tagx="fpage" title="fpage">1533</span><span class="lpage" tagx="lpage" title="lpage">1543</span><span class="pub-id"><a href="https://dx.doi.org/10.1104/pp.106.082198">10.1104/pp.106.082198</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16798944">16798944</a></span></span></li> <li tag="ref"><a name="CR73" /><span class="label" tagx="label" title="label">73.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lehner</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuksis</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">Biosynthesis of triacylglycerols</span><span class="source" tagx="source" title="source">Prog Lipid Res.</span><span class="year" tagx="year" title="year">1996</span><span class="volume" tagx="volume" title="volume">35</span><span class="fpage" tagx="fpage" title="fpage">169</span><span class="lpage" tagx="lpage" title="lpage">201</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/0163-7827(96)00005-7">10.1016/0163-7827(96)00005-7</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8944226">8944226</a></span></span></li> <li tag="ref"><a name="CR74" /><span class="label" tagx="label" title="label">74.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">HW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">JS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gai</span><span class="given-names" tagx="given-names" title="given-names">JY</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="mixed-article-title" title="mixed-article-title">Cloning and comparative analysis of the gene encoding diacylglycerol acyltransferase from wild type and cultivated soybean</span><span class="source" tagx="source" title="source">Ther Appl Genet.</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">112</span><span class="fpage" tagx="fpage" title="fpage">1086</span><span class="lpage" tagx="lpage" title="lpage">1097</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s00122-006-0210-9">10.1007/s00122-006-0210-9</a></span></span></li> <li tag="ref"><a name="CR75" /><span class="label" tagx="label" title="label">75.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">KS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">RZ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hatanaka</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hildebrand</span><span class="given-names" tagx="given-names" title="given-names">D</span></span></span><span class="mixed-article-title" title="mixed-article-title">Cloning and functional analysis of two type 1 diacylglycerol acyltransferases from <i>Vernonia galamensis</i></span><span class="source" tagx="source" title="source">Phytochemistry.</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">69</span><span class="fpage" tagx="fpage" title="fpage">1119</span><span class="lpage" tagx="lpage" title="lpage">1127</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.phytochem.2007.11.015">10.1016/j.phytochem.2007.11.015</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18179805">18179805</a></span></span></li> <li tag="ref"><a name="CR76" /><span class="label" tagx="label" title="label">76.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chapman</span><span class="given-names" tagx="given-names" title="given-names">KD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dyer</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mullen</span><span class="given-names" tagx="given-names" title="given-names">RT</span></span></span><span class="mixed-article-title" title="mixed-article-title">Biogenesis and functions of lipid droplets in plants thematic review series: lipid droplet synthesis and metabolism: from yeast to man</span><span class="source" tagx="source" title="source">J Lipid Res.</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">53</span><span class="fpage" tagx="fpage" title="fpage">215</span><span class="lpage" tagx="lpage" title="lpage">226</span><span class="pub-id"><a href="https://dx.doi.org/10.1194/jlr.R021436">10.1194/jlr.R021436</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22045929">22045929</a></span></span></li> <li tag="ref"><a name="CR77" /><span class="label" tagx="label" title="label">77.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jolivet</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boulard</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bellamy</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">LarrÃ©</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barre</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rogniaux</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Protein composition of oil bodies from mature <i>Brassica napus</i> seeds</span><span class="source" tagx="source" title="source">Proteomics.</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">3268</span><span class="lpage" tagx="lpage" title="lpage">3284</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/pmic.200800449">10.1002/pmic.200800449</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19562800">19562800</a></span></span></li> <li tag="ref"><a name="CR78" /><span class="label" tagx="label" title="label">78.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Idogawa</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abe</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abe</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fuji</span><span class="given-names" tagx="given-names" title="given-names">T</span></span></span><span class="mixed-article-title" title="mixed-article-title">Effect of oleosins on the stability of oil bodies in soymilk</span><span class="source" tagx="source" title="source">Food Sci Technol Res.</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">24</span><span class="fpage" tagx="fpage" title="fpage">677</span><span class="lpage" tagx="lpage" title="lpage">685</span><span class="pub-id"><a href="https://dx.doi.org/10.3136/fstr.24.677">10.3136/fstr.24.677</a></span></span></li> <li tag="ref"><a name="CR79" /><span class="label" tagx="label" title="label">79.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Siloto</span><span class="given-names" tagx="given-names" title="given-names">RMP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Findlay</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lopez-Villalobos</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yeung</span><span class="given-names" tagx="given-names" title="given-names">EC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nykiforuk</span><span class="given-names" tagx="given-names" title="given-names">CL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moloney</span><span class="given-names" tagx="given-names" title="given-names">MM</span></span></span><span class="mixed-article-title" title="mixed-article-title">The accumulation of oleosins determines the size of seed oil bodies in <i>Arabidopsis</i></span><span class="source" tagx="source" title="source">Plant Cell.</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">18</span><span class="fpage" tagx="fpage" title="fpage">1961</span><span class="lpage" tagx="lpage" title="lpage">1974</span><span class="pub-id"><a href="https://dx.doi.org/10.1105/tpc.106.041269">10.1105/tpc.106.041269</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16877495">16877495</a></span></span></li> <li tag="ref"><a name="CR80" /><span class="label" tagx="label" title="label">80.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cao</span><span class="given-names" tagx="given-names" title="given-names">HP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tan</span><span class="given-names" tagx="given-names" title="given-names">XF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Long</span><span class="given-names" tagx="given-names" title="given-names">HX</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shockey</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span></span><span class="mixed-article-title" title="mixed-article-title">Identification, classification and differetial expression of oleosin gene in tung tree (<i>Vernicia fordii</i>)</span><span class="source" tagx="source" title="source">PLoS One.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">e88409</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0088409">10.1371/journal.pone.0088409</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24516650">24516650</a></span></span></li> <li tag="ref"><a name="CR81" /><span class="label" tagx="label" title="label">81.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thelen</span><span class="given-names" tagx="given-names" title="given-names">JJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mekhedov</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ohlrogge</span><span class="given-names" tagx="given-names" title="given-names">JB</span></span></span><span class="mixed-article-title" title="mixed-article-title">Biotin carboxyl carrier protein isoforms in Brassicaceae oilseeds</span><span class="source" tagx="source" title="source">Biochem Soc Trans.</span><span class="year" tagx="year" title="year">2000</span><span class="volume" tagx="volume" title="volume">28</span><span class="issue" tagx="issue" title="issue">6</span><span class="fpage" tagx="fpage" title="fpage">595</span><span class="lpage" tagx="lpage" title="lpage">598</span><span class="pub-id"><a href="https://dx.doi.org/10.1042/bst0280595">10.1042/bst0280595</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11171138">11171138</a></span></span></li> <li tag="ref"><a name="CR82" /><span class="label" tagx="label" title="label">82.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Focks</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beenning</span><span class="given-names" tagx="given-names" title="given-names">C</span></span></span><span class="mixed-article-title" title="mixed-article-title"><i>Winkled1</i>: a novel, low-seed-oil mutant of <i>Arabidopsis</i> with a deficiency inthe seed-specific regulation of carbohydrate metabolism</span><span class="source" tagx="source" title="source">Plant Physiol.</span><span class="year" tagx="year" title="year">1998</span><span class="volume" tagx="volume" title="volume">118</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">91</span><span class="lpage" tagx="lpage" title="lpage">101</span><span class="pub-id"><a href="https://dx.doi.org/10.1104/pp.118.1.91">10.1104/pp.118.1.91</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9733529">9733529</a></span></span></li> <li tag="ref"><a name="CR83" /><span class="label" tagx="label" title="label">83.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">YF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elizondon-Noriega</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cantu</span><span class="given-names" tagx="given-names" title="given-names">DC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reilly</span><span class="given-names" tagx="given-names" title="given-names">PJ</span></span></span><span class="mixed-article-title" title="mixed-article-title">Structural classification of biotin carboxyl carrier proteins</span><span class="source" tagx="source" title="source">Biotechnol Lett.</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">34</span><span class="fpage" tagx="fpage" title="fpage">1869</span><span class="lpage" tagx="lpage" title="lpage">1875</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s10529-012-0978-4">10.1007/s10529-012-0978-4</a></span></span></li> <li tag="ref"><a name="CR84" /><span class="label" tagx="label" title="label">84.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thelen</span><span class="given-names" tagx="given-names" title="given-names">JJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ohlrogge</span><span class="given-names" tagx="given-names" title="given-names">JB</span></span></span><span class="mixed-article-title" title="mixed-article-title">Both antisense and sense expression of biotin carboxyl carrier protein isoform 2 inactivates the plastid acetyl-coenzyme a carboxlylase in Arabidopsis thaliana</span><span class="source" tagx="source" title="source">Plant J.</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">34</span><span class="issue" tagx="issue" title="issue">4</span><span class="fpage" tagx="fpage" title="fpage">419</span><span class="lpage" tagx="lpage" title="lpage">431</span><span class="pub-id"><a href="https://dx.doi.org/10.1046/j.1365-313X.2002.01435.x">10.1046/j.1365-313X.2002.01435.x</a></span></span></li> <li tag="ref"><a name="CR85" /><span class="label" tagx="label" title="label">85.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Altschul</span><span class="given-names" tagx="given-names" title="given-names">SF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gish</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miller</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Myers</span><span class="given-names" tagx="given-names" title="given-names">EW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lipman</span><span class="given-names" tagx="given-names" title="given-names">DG</span></span></span><span class="mixed-article-title" title="mixed-article-title">Basic local alignment search tool</span><span class="source" tagx="source" title="source">J Mol Biol.</span><span class="year" tagx="year" title="year">1990</span><span class="volume" tagx="volume" title="volume">215</span><span class="issue" tagx="issue" title="issue">3</span><span class="fpage" tagx="fpage" title="fpage">403</span><span class="lpage" tagx="lpage" title="lpage">410</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0022-2836(05)80360-2">10.1016/S0022-2836(05)80360-2</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2231712">2231712</a></span></span></li> <li tag="ref"><a name="CR86" /><span class="label" tagx="label" title="label">86.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Conesa</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">GÃ¶tz</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">GarcÃ­a-GÃ³mez</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Terol</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">TalÃ³n</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Robles</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">Blast2GO: a universal tool for annotation, visualization and analysis in functional genomics research</span><span class="source" tagx="source" title="source">Bioinformatics.</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">21</span><span class="fpage" tagx="fpage" title="fpage">3674</span><span class="lpage" tagx="lpage" title="lpage">3676</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/bti610">10.1093/bioinformatics/bti610</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16081474">16081474</a></span></span></li> <li tag="ref"><a name="CR87" /><span class="label" tagx="label" title="label">87.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Quevillon</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Silventoinen</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pillai</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harte</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mulder</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Apweiler</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">InterProScan: protein domains identifier</span><span class="source" tagx="source" title="source">Nucleic Acids Res.</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">33</span><span class="fpage" tagx="fpage" title="fpage">116</span><span class="lpage" tagx="lpage" title="lpage">120</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gki442">10.1093/nar/gki442</a></span></span></li> <li tag="ref"><a name="CR88" /><span class="label" tagx="label" title="label">88.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Langmead</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Salzberg</span><span class="given-names" tagx="given-names" title="given-names">SL</span></span></span><span class="mixed-article-title" title="mixed-article-title">Fast gapped-read alignment with bowtie</span><span class="source" tagx="source" title="source">Nat Methods.</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">357</span><span class="lpage" tagx="lpage" title="lpage">359</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nmeth.1923">10.1038/nmeth.1923</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22388286">22388286</a></span></span></li> <li tag="ref"><a name="CR89" /><span class="label" tagx="label" title="label">89.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dewey</span><span class="given-names" tagx="given-names" title="given-names">CNRSEM</span></span></span><span class="mixed-article-title" title="mixed-article-title">accurate transcript quantification from RNA-Seq data with or without a reference genome</span><span class="source" tagx="source" title="source">BMC Bioinformatics.</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">12</span><span class="fpage" tagx="fpage" title="fpage">323</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1471-2105-12-323">10.1186/1471-2105-12-323</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21816040">21816040</a></span></span></li> <li tag="ref"><a name="CR90" /><span class="label" tagx="label" title="label">90.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Michael</span><span class="given-names" tagx="given-names" title="given-names">IL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huber</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Anders</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="mixed-article-title" title="mixed-article-title">Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2</span><span class="source" tagx="source" title="source">Genome Biol.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">15</span><span class="fpage" tagx="fpage" title="fpage">550</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s13059-014-0550-8">10.1186/s13059-014-0550-8</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25516281">25516281</a></span></span></li> <li tag="ref"><a name="CR91" /><span class="label" tagx="label" title="label">91.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">DD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">RQ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kong</span><span class="given-names" tagx="given-names" title="given-names">NN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">CH</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Genome-wide analysis of the potato <i>Hsp20</i> gene family: identification, genomic organization and expression profiles in response to heat stess</span><span class="source" tagx="source" title="source">BMC Genomics.</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">19</span><span class="fpage" tagx="fpage" title="fpage">61</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s12864-018-4443-1">10.1186/s12864-018-4443-1</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29347912">29347912</a></span></span></li> <li tag="ref"><a name="CR92" /><span class="label" tagx="label" title="label">92.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">DQ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tao</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Han</span><span class="given-names" tagx="given-names" title="given-names">CX</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ge</span><span class="given-names" tagx="given-names" title="given-names">JT</span></span></span><span class="mixed-article-title" title="mixed-article-title"><i>Actin</i> as an alternative internal control gene for gene expression analysis in herbaceous peony (<i>Paeonia lactiflora</i> Pall.)</span><span class="source" tagx="source" title="source">Afr J Agric Res.</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">2153</span><span class="lpage" tagx="lpage" title="lpage">2159</span></span></li> <li tag="ref"><a name="CR93" /><span class="label" tagx="label" title="label">93.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schmittgen</span><span class="given-names" tagx="given-names" title="given-names">TD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Livak</span><span class="given-names" tagx="given-names" title="given-names">KJ</span></span></span><span class="mixed-article-title" title="mixed-article-title">Analyzing real-time PCR data by the comparative CT method</span><span class="source" tagx="source" title="source">Nat Protoc.</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">3</span><span class="fpage" tagx="fpage" title="fpage">1101</span><span class="lpage" tagx="lpage" title="lpage">1108</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nprot.2008.73">10.1038/nprot.2008.73</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18546601">18546601</a></span></span></li> </ul> </div> </div>  </body></html>