<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">BMC Genomics</span><span class="iso-abbrev" title="iso-abbrev">BMC Genomics</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">BMC Genomics</span></div><span class="issn-epub" tagx="issn" title="issn-epub">1471-2164</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">BioMed Central</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7788892">7788892</a></span><span class="publisher-id" title="publisher-id">7266</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1186/s12864-020-07266-7">10.1186/s12864-020-07266-7</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Transcriptional response of <i>Asarum heterotropoides</i> Fr. Schmidt var. <i>mandshuricum</i> (Maxim.) Kitag. leaves grown under full and partial daylight conditions</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Zhiqing</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">wangzhiqing96@sohu.com</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ma</span><span class="given-names" tagx="given-names" title="given-names">Haiqin</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">mahaiqin@caas.cn</span></span><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Min</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">915752334@qq.com</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Ziqing</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">2043946385@qq.com</span></span><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tian</span><span class="given-names" tagx="given-names" title="given-names">Yixin</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">y.x.tian2003@163.com</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Wei</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">liwei7727@126.com</span></span><a href="#Aff3">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Yingping</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">yingpingw@126.com</span></span><a href="#Aff3">3</a></span><span class="citation_author_institution" id="Aff1">[1], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.464353.3</span><span class="institution-id" title="institution-id">0000 0000 9888 756X</span><span class="institution" title="institution">Laboratory of Cultivation and Breeding of Medicinal Plants, National Administration of Traditional Chinese Medicine, College of Chinese Medicinal Materials, </span><span class="institution" title="institution">Jilin Agricultural University, </span></div></span><span class="citation_author_institution" id="Aff2">[2], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.464373.1</span><span class="institution" title="institution">Institute of Special Wild Economic Animals and Plants, Chinese Academy of Agriculture Sciences, </span></div></span><span class="citation_author_institution" id="Aff3">[3], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.464353.3</span><span class="institution-id" title="institution-id">0000 0000 9888 756X</span><span class="institution" title="institution">State &amp;amp; Local Joint Engineering Research Center of Ginseng Breeding and Application, College of Chinese Medicinal Materials, </span><span class="institution" title="institution">Jilin Agricultural University, </span></div></span></div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2021-1-1</span></span><span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2021-1-1</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2021</span></span><span class="volume" tagx="volume" title="volume">22</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">16</span><span class="history" title="history"><span class="received" title="received">received: 2020-5-22</span><span class="accepted" title="accepted">accepted: 2020-11-23</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , </span><span class="license" title="license"><span class="license-p" title="license-p"><b>Open Access</b>This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>. The Creative Commons Public Domain Dedication waiver (<a href="http://creativecommons.org/publicdomain/zero/1.0/">http://creativecommons.org/publicdomain/zero/1.0/</a>) applies to the data made available in this article, unless otherwise stated in a credit line to the data.</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="background" title="sec"> <div class="title" tagx="title" title="title">
Background</div> <p id="Par1"><i>Asarum heterotropides</i> Fr. Schmidt var. <i>mandshuricum</i> (Maxim.) Kitag. is an important medicinal and industrial plant, which is used in the treatment of various diseases. The main bioactive ingredient is the volatile oil having more than 82 identified components of which methyleugenol, safrole, myristicin, and toluene account for about 70% of the total volume. As a sciophyte plant, the amount of light it absorbs through leaves is an important factor for growth and metabolism.</p> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <p id="Par2">We grew <i>Asarum</i> plants under full, 50, 28, and 12% sunlight conditions to investigate the effect of different light irradiances on the four major volatile oil components. We employed de novo transcriptome sequencing to understand the transcriptional behavior of <i>Asarum</i> leaves regarding the biosynthetic pathways of the four volatile oil components, photosynthesis and biomass accumulation, and hormone signaling. Our results demonstrated that the increasing light conditions promoted higher percent of the four components. Under full sunlight conditions, cinnamyl alcohol dehydrogenase and cytochrome p450719As were upregulated and led the increased methyleugenol, safrole, and myristicin. The transcriptomic data also showed that <i>Asarum</i> leaves, under full sunlight conditions, adjust their photosynthesis-antenna proteins as a photoprotective response with the help of carotenoids. Plant hormone-signaling related genes were also differentially expressed between full sunlight and low light conditions.</p> </div> <div class="conclusions" title="sec"> <div class="title" tagx="title" title="title">
Conclusions</div> <p id="Par3">High light induces accumulation of major bioactive ingredients <i>A. heterotropides</i> volatile oil and this is ascribed to upregulation of key genes such as cinnamyl alcohol dehydrogenase and cytochrome p450719As. The transcriptome data presented here lays the foundation of further understanding of light responses in sciophytes and provides guidance for increasing bioactive molecules in <i>Asarum</i>.</p> </div> <div class="supplementaryinformation" title="sec"> <div class="title" tagx="title" title="title">
Supplementary Information</div> <p>The online version contains supplementary material available at 10.1186/s12864-020-07266-7.</p> </div> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Keywords</div> <div class="kwd" title="kwd">
Hormone signaling</div> <div class="kwd" title="kwd">
Herbal plant</div> <div class="kwd" title="kwd">
Photosynthesis</div> <div class="kwd" title="kwd">
Sciophyte</div> <div class="kwd" title="kwd">
Transcriptome</div> <div class="kwd" title="kwd">
Volatile oil</div> <div class="kwd" title="kwd">
Bioactive component</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="institution" title="institution">National Natural Science Foundation of China </span></div><span class="award-id" tagx="award-id" title="award-id">81403052</span></div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="institution" title="institution">Science and Technology Department Project from Jilin Province of China</span></div><span class="award-id" tagx="award-id" title="award-id">20190304017YY</span></div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="institution" title="institution">National key research special project of China</span></div><span class="award-id" tagx="award-id" title="award-id">2019YFC1710070</span></div> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="issue-copyright-statement" value="Â© The Author(s) 2021" /> </div> </div> </div> <div class="body" title="body"> <div class="background" title="sec"> <div class="title" tagx="title" title="title">
Background</div> <p id="Par44"><i>Asarum heterotropoides</i> Fr. Schmidt var. <i>mandshuricum</i> (Maxim.) Kitag., a perennial herb endemic to China, has been exploited as a traditional medicinal herb due to its anti-inflammatory, anti-bacterial, anti-pyretic, anticancer, fungistatic and analgesic properties [<a href="#CR1">1</a>, <a href="#CR2">2</a>]. This species has a wide geographical distribution and grows in shady habitats and mountainous wetlands. The main producing areas are Jilin, Liaoning, and Heilongjiang in China [<a href="#CR3">3</a>]. Previous studies have documented that the main bioactive ingredient is the essential oil for which more than 82 components have been identified [<a href="#CR4">4</a>, <a href="#CR5">5</a>]. As a sciophyte, different studies have demonstrated that growing in different solar irradiance levels affects the leaf mass to per unit area of the plant, chlorophyll content, and net photosynthetic rate [<a href="#CR6">6</a>]. Two recent studies demonstrated that light intensities affect photosynthesis and chlorophyll content but the content of <i>Asarum</i> volatile oil did not change among different groups [<a href="#CR6">6</a>, <a href="#CR7">7</a>]. However, the study by Wang et al., [<a href="#CR7">7</a>] used GC-MS to determine the composition of the oil and reported variation in oil composition under different light treatments suggesting that light treatments somehow affect the regulation of the pathways involved in volatile oil biosynthesis. The main components of <i>Asarum</i> essential oil used in the pharmaceutical industry are phenylpropane compounds including methyleugenol, safrole, myristicin, 1,3-benzodioxole, 4-methoxy-6- (2-propenyl)-, 3,5-dimethoxytoluene, 2-Hydroxy-4,5- methylenedioxypropiophenone, etc., these compounds account for about 70% of the total volatile oil content [<a href="#CR4">4</a>]. In addition to the above mentioned recent studies, a previous investigation reported that the content of major components was subjected to seasonal variation [<a href="#CR8">8</a>]. In other species, the effect of light intensities on growth and accumulation of essential oils and secondary metabolites has been established e.g. manipulation of the light affected the secondary metabolite contents in <i>Glycyrrhiza uralensis</i> Fisch [<a href="#CR9">9</a>]. Similarly, solar irradiance levels altered volatile oil contents in basil (<i>Ocimum basilicum</i> L.), <i>Myrtus communis</i> L., <i>Ocimum gratissimum</i>, damask rose (<i>Rosa damascena</i> Mill.), and other aromatic plants [<a href="#CR10">10</a>â€"<a href="#CR13">13</a>]. These contradicting reports suggest that a deeper understanding is a prerequisite for establishing an optimal irradiation protocol for <i>Asarum</i> growth, which can provide high yield of volatile oil and its major bioactive components for industrial scale volatile oils.</p> <p id="Par45">Methyleugenol is a common phenylpropanoid found in many medicinal plant species. It is derived from eugenol that is a product of phenylalanine through the reaction of cinnamic acid, ferulic acid, coniferyl alcohol, and coniferyl acetate. Methyleugenol is further converted into myristicin [<a href="#CR14">14</a>â€"<a href="#CR17">17</a>]. Several reports have identified and characterized this pathway-related genes e.g. coniferyl alcohol acyl transferase (CAAT) in apple fruit, eugenol synthase genes (EGS) in rose, Ocimum, and Gymnadenia, genes encoding O-methyltransferases (OMT) in loquat, cinnamyl alcohol dehydrogenase (CAD) in Arabidopsis and many other plant species [<a href="#CR18">18</a>â€"<a href="#CR24">24</a>]. Another important constituent of the volatile oil in <i>Asarum</i> is safrole. It has been suggested that it is possibly biosynthesized from eugenol through the formation of the methylenedioxy bridge and shares a common precursor coniferyl alcohol [<a href="#CR25">25</a>]. Apart from methyleugenol, myristicin, and safrole, toluene (3,5-dimethoxytoluene) is another major component of volatile oil in many aromatic medicinal plants including <i>Asarum</i> species. In roses, a number of OMT genes have been identified to convert orcinol to 3-methoxyl-5-hydroxytoluene, and then to toluene [<a href="#CR26">26</a>, <a href="#CR27">27</a>]. Because, these four components are the main components in the <i>Asarum</i> volatile oil, it is important to understand their possible regulation under different light intensities.</p> <p id="Par46">The amount of solar radiation directly impacts on the photosynthetic characteristics of <i>Asarum</i> [<a href="#CR6">6</a>] and it is known that the biomass accumulation is associated with the rate of photosynthesis in plants. Therefore, it is essential to understand the regulation of genes involved in plant biomass accumulation and photosynthetic efficiency in <i>Asarum</i> together with the impact of irradiance on volatile oil components [<a href="#CR28">28</a>]. In plants, photosynthesis is a complex, multistep process involving electron transport chain, Calvin-Benson cycle, and subsequent steps involving assimilation, transport, and utilization of photoassimilates. These distinct yet overlapping processes require the product of hundreds of proteins and genes associated either with the nucleus or chloroplasts [<a href="#CR29">29</a>]. Similarly, biomass accumulation in plants is a complex process involving photosynthetic pathways, cell architecture, plant growth regulators, sugar transport and accumulation, metabolism, and regulation of transcription [<a href="#CR30">30</a>]. Since both processes involve a high number of genes and pathways, studying them in an individual genetic characterization project or working with a single pathway is not possible and demands large-scale transcriptome analyses. Recent developments in transcriptomics have enabled the understanding of complex pathways in medicinal plants including <i>Asarum</i> [<a href="#CR25">25</a>, <a href="#CR31">31</a>â€"<a href="#CR33">33</a>]. The low light irradiance levels lead plant leaves to adapt in shade conditions through various mechanisms involving photosynthetic machinery, adjustment in cell growth, stomatal conductance, and hormone-signaling [<a href="#CR13">13</a>, <a href="#CR34">34</a>â€"<a href="#CR36">36</a>]. Therefore, a transcriptome will enable the understanding of the differential changes in the expression of genes involved in these pathways.</p> <p id="Par47">Efforts have been made to optimize the extraction of volatile oil from this medicinally important plant [<a href="#CR37">37</a>]. The possible ways to increase the volatile oil production are 1) to understand the effect of different light intensities on the photosynthate and in turn on the volatile oil biosynthesis [<a href="#CR38">38</a>], 2) understand the volatile oil and its componentsâ€™ biosynthesis-related pathways, and based on this knowledge, 3) develop high volatile oil yielding <i>A. heterotropoides</i> genotypes for large scale extraction. In this study, transcriptome sequencing of <i>Asarum</i> leaves grown under four different light treatments was performed to uncover the effect of light treatments on genes involved in pathways associated with the biosynthesis of methyleugenol, myristicin, safrole, and 3,5-dimethoxytoluene. Furthermore, we also studied the effect of light on the genes related to photosynthesis, which in turn influence biomass accumulation under different light irradiances. Additionally, we also explored the differential regulation of plant hormone signaling related genes.</p> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <div class="effectofshadetreatmentsonimportantvolatileoilconstituents" title="sec"> <div class="title" tagx="title" title="title">
Effect of shade treatments on important volatile oil constituents</div> <p id="Par48"><i>Asarum</i> plants were grown under four light irradiances including, full sunlight (L1), 50% sunlight (L2), 28% sunlight (L3), and 12% sunlight (L4). Because leaf is the plant organ, where light is directly absorbed and the main photosynthate is produced and processed, therefore, we focused on the essential oil changes and transcriptional responses as adopted in a previous studies [<a href="#CR6">6</a>]. The percent yield of four important <i>Asarum</i> volatile oil components under various light treatments are shown in Fig.Â <a href="#Fig1">1</a>. The oil component with the highest percentage among the four major volatile oil constituents was methyleugenol followed by safrole, toluene, and myristicin. Metyleugenol content was highest under full sunlight conditions and decreased with the decrease in the light intensity while it did not differ significantly between 28 and 12% light conditions. Myristicin showed an almost similar pattern where the highest content was recorded in full sunlight grown leaves and it decreased with the reduction in the light intensity. It decreased 9.75, 8.58, and 7.6 fold when treated with 50, 28, and 12% light, respectively. Safrole content also showed a reducing pattern with the decrease in the light intensity where the highest content was 21.4% in full sunlight conditions followed by 19.21, 15.26, and 14.70% when leaves were grown under 50, 28, and 12% sunlight conditions, respectively. The toluene percent content in full and 50% sunlight light grown leaves did not differ significantly while a further reduction in sunlight intensity (28% light) resulted in 12.8 fold decrease. Under low light conditions i.e. 28 and 12% light, the toluene content did not change significantly, however, a reducing trend was still noticeable (Fig.Â <a href="#Fig2">2</a>). Together these observations suggest that <i>Asarum</i> leaves grown under higher light conditions result in volatile oil higher enriched in the four compounds. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 1</span></a></div>   <p>Effect of light treatments on methyleugenol, myristicin, safrole, and toluene percent in <i>Asarum</i> essential oil. Error bar represents SD from triplicate data. Means with the same letter are not significantly different from each other (<i>P</i>â€‰<div class="â€‰0.05)._UNKNOWN">   </div></p></div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 2</span></a></div>   <p><b>a</b> Unigene database functional annotation statistics. <b>b</b> Distribution of gene expression in four light irradiation treatments in <i>Asarum</i> leaves. L1, L2, L3 and L4 represent full sunlight, 50% sunlight, 28% sunlight and 12% sunlight, respectively</p>   </div> <p /> </div> <div class="overviewoftranscriptomeanalyses" title="sec"> <div class="title" tagx="title" title="title">
Overview of Transcriptome analyses</div> <p id="Par49">The cDNA libraries constructed from light treated <i>A. heterotropoides</i> leaves were sequenced with Illumina HiseqTM high-throughput sequencing platform. After filtering low quality reads and adapter sequences, a total of 97.33 Gb clean data was obtained consisting of Illumina reads ranging from 41,987,144 to 65,408,518 million/sample (average 54,067,052) (Additional Table <a href="#MOESM1">1</a>). Trinity assembly tool was used to de novo assemble the transcriptome. After data processing, 106,982 unigene sequences were included, and the N50 was 1375â€‰bp long. The summary of the unigene sequence size ranges is shown in Additional Fig. <a href="#MOESM2">1</a>.</p> <p id="Par50">Functional annotation of all unigenes as blasted the non-redundant (NR) (38.47%), Nucleotide (NT) (19.48%), Kyoto encyclopedia of genes and genomes (KEGG) (15.7%), Swiss-Port (27.24%), Pfam (28.96%), Gene Ontology (GO) (28.96%), Ortholog Groups (KOG) (9.13%) databases is presented in Fig. <a href="#Fig2">2</a>a; a total of 106,982 unigenes was annotated. The Fragments Per Kilobase of Transcript per Million Fragments Mapped (FPKM) gene expression levels in the four treatments are shown in Fig. <a href="#Fig2">2</a>b. Pearson correlations between replicates of the four irradiation treatments in <i>Asarum</i> leaves ranged from 0.771 to 0.858 (Fig.Â <a href="#Fig3">3</a>a). Differential expressed genes (DEGs) expressed under different treatment comparisons are shown in Fig. <a href="#Fig3">3</a>b and Additional Fig. <a href="#MOESM2">2</a>. Graphical representation of the KEGG enrichment scatter plot of DEGs between different treatment comparisons is shown in Additional Fig. <a href="#MOESM2">3</a>. We used the Richfactor, Q-value, and number of genes enriched in specific pathways to show the degree of KEGG enrichment. The KEGG pathway enrichment showed that the most common significantly enriched pathways under the tested conditions were phenylpropanoid pathway, plant hormone signaling-transduction, photosynthesis-antenna proteins, protein processing in the endoplasmic reticulum, and flavonoid metabolism (Additional Fig. <a href="#MOESM2">3</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 3</span></a></div>   <p><b>a</b> Pearson correlations between replicates of four irradiation treatments in <i>Asarum</i> leaves, and <b>b)</b> differential gene heat map; the abscissa represents the sample name and hierarchical clustering results and the ordinate represents the differential genes and hierarchical clustering results. L1, L2, L3 and L4 represent full sunlight, 50% sunlight, 28% sunlight and 12% sunlight, respectively</p>   </div> <p /> </div> <div class="transcriptomicresponseofasarumleavestolighttreatments" title="sec"> <div class="title" tagx="title" title="title">
Transcriptomic response of <i>Asarum</i> leaves to light treatments</div> <div class="differentialregulationofvolatileoilbiosynthesisrelatedgenes" title="sec"> <div class="title" tagx="title" title="title">
Differential regulation of volatile oil biosynthesis related genes</div> <p id="Par51">Previous studies have demonstrated that the volatile oil content is affected by light/shade conditions [<a href="#CR6">6</a>, <a href="#CR9">9</a>, <a href="#CR13">13</a>]. Therefore, we searched for the DEGs associated with volatile oil biosynthesis in our comparative transcriptome data between different light treated <i>Asarum</i> leaves. Between the full sunlight and 50% light conditions, a gene (<i>Cluster-24,085.27155</i>) was upregulated in <i>Asarum</i> leaves. This gene is annotated as CAD and the KEGG pathway mapping suggested its involvement in controlling the important steps in the formation of caffeyl-alchol and coniferyl alcohol, which are intermediates in the formation of methyleugenol (Additional Table <a href="#MOESM1">2</a>). The upregulation suggested full sunlight conditions have a beneficial impact on the biosynthesis of methyleugenol through the upregulation of CAD gene. Two other genes involved in the same pathway i.e. the final steps of methyleugenol biosynthesis were also upregulated in full sunlight grown <i>Asarum</i> leaves as compared to 28% light conditions. One of the two genes was annotated as CAD (<i>Cluster-24,085.41570</i>) while the second gene was annotated as peroxidase 25-like (<i>Cluster-24,085.21345</i>) (Additional Table <a href="#MOESM1">3</a>). This second gene is involved in the final step of lignin formation. Another gene (<i>Cluster-24,085.10149</i>) was upregulated in <i>Asarum</i> leaves grown in 50% light conditions when compared with 28% light conditions. This gene is involved in the formation of phenyl-ethylamine, which is an important intermediate of phenylpropanoid pathway (Additional Table <a href="#MOESM1">4</a>; Table <a href="#Tab1">1</a>). <div class="table-wrap_UNKNOWN" id="Tab1"><span class="label" tagx="label" title="label">Table 1</span> </div></p><p>DEGs related to Phenylpropanoid pathway associated with volatile oil biosynthesis in <i>Asarum</i> leaves grown under different light intensities. L1, L2, L3 and L4 represent full sunlight, 50% sunlight, 28% sunlight and 12% sunlight, respectively</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th>Treatment</th> <th>Gene ID</th> <th>Log2 fold change</th> <th>P value adjusted</th> <th>Description</th> </tr> </thead> <tbody> <tr> <td colspan="5">Phenylpropanoid Pathway</td> </tr> <tr> <td>â€ƒL1 to L2</td> <td><i>Cluster-24,085.27155</i></td> <td>2.657</td> <td>0.038336</td> <td>Cinnamyl alcohol dehydrogenase 3</td> </tr> <tr> <td rowspan="2">â€ƒL1 to L3</td> <td><i>Cluster-24,085.21345</i></td> <td>2.2109</td> <td>0.020262</td> <td>Peroxidase 25-like</td> </tr> <tr> <td><i>Cluster-24,085.41570</i></td> <td>2.5763</td> <td>0.020982</td> <td>Cinnamyl alcohol dehydrogenase</td> </tr> <tr> <td>â€ƒL1 to L4</td> <td><i>Cluster-24,085.23909</i></td> <td>2.2421</td> <td>0.00372</td> <td>trans-resveratrol O-methyltransferase</td> </tr> <tr> <td>â€ƒL2 to L3</td> <td><i>Cluster-24,085.10149</i></td> <td>2.7642</td> <td>0.012531</td> <td>aromatic-L-amino-acid decarboxylase-like</td> </tr> </tbody> </table>  <p /> </div> <div class="differentialregulationofphotosynthesisandbiomassaccumulationrelatedgenes" title="sec"> <div class="title" tagx="title" title="title">
Differential regulation of photosynthesis and biomass accumulation related genes</div> <p id="Par52">As it is previously established that morphological, physiological, and biochemical changes occur in plants grown under sunlight versus shade conditions and the process of photosynthesis is affected [<a href="#CR34">34</a>], we searched for genes in our transcriptome that are associated directly or indirectly with photosynthesis. An important gene (<i>Cluster-24,085.63483</i>) implicated in porphyrin and chlorophyll metabolism pathway was upregulated (log2foldchangeâ€‰=â€‰3.2) under full sunlight grown <i>Asarum</i> leaves as compared to 50% light conditions (Additional Table <a href="#MOESM1">2</a>; Table <a href="#Tab2">2</a>). One photosynthesis-antenna protein related to light-harvesting complexes (<i>Cluster-24,085.41612</i>, Lhcb1) was downregulated in full sunlight conditions as compared to 50% light. This gene was also differentially expressed between full sunlight versus very low light intensity (12%) conditions (Additional Table <a href="#MOESM1">4</a>). The downregulation under full sunlight conditions compared to both 50 and 12% light intensities suggests that under the influence of increased light intensity, <i>Asarum</i> optimizes their light-harvesting antenna. Because there are several photosynthesis-antenna proteins, we searched our DEGs for other antenna proteins and found the downregulation of one antenna protein (<i>Cluster-24,085.76692</i>, Lhcb2) in full sunlight versus 28% light conditions. Interestingly, five antenna proteins were downregulated in full versus 12% sunlight grown <i>Asarum</i> leaves (Table <a href="#Tab3">3</a>). These results further confirmed the fact that like other plants, <i>Asarum</i> leaves manipulate antenna proteins to control the light capture rate under natural full sunlight conditions. <div class="table-wrap_UNKNOWN" id="Tab2"><span class="label" tagx="label" title="label">Table 2</span> </div></p><p>List of DEGs related to photosynthesis in <i>Asarum</i> leaves grown under different light intensities. L1, L2, L3 and L4 represent full sunlight, 50% sunlight, 28% sunlight and 12% sunlight, respectively</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th>Treatment</th> <th>Gene ID</th> <th>Log2 fold change</th> <th>Pvalue adjusted</th> <th>Description</th> </tr> </thead> <tbody> <tr> <td colspan="5">Carotenoid biosynthesis</td> </tr> <tr> <td>â€ƒL1 to L2</td> <td><i>Cluster-9588.0</i></td> <td>4.514</td> <td>0.034696</td> <td>Beta-carotene 3-hydroxylase</td> </tr> <tr> <td>â€ƒL1 to L4</td> <td><i>Cluster-9588.0</i></td> <td>4.299</td> <td>0.016687</td> <td>Beta-carotene 3-hydroxylase</td> </tr> <tr> <td rowspan="2">â€ƒL2 to L4</td> <td><i>Cluster-24,085.17524</i></td> <td>âˆ’1.9954</td> <td>0.001628</td> <td>9-Cis-epoxycarotenoid dioxygenase</td> </tr> <tr> <td><i>Cluster-24,085.21006</i></td> <td>âˆ’4.7706</td> <td>3.46E-11</td> <td>Abscisic acid 8â€²-hydroxylase</td> </tr> <tr> <td colspan="5">Photosynthesis - antenna proteins</td> </tr> <tr> <td>â€ƒL1 to L2</td> <td><i>Cluster-24,085.41612</i></td> <td>âˆ’1.291</td> <td>0.000785</td> <td>Chlorophyll a-b binding protein</td> </tr> <tr> <td>â€ƒL1 to L3</td> <td><i>Cluster-24,085.76692</i></td> <td>âˆ’4.6487</td> <td>0.023195</td> <td>Chlorophyll a-b binding protein</td> </tr> <tr> <td rowspan="5">â€ƒL1 to L4</td> <td><i>Cluster-24,085.41606</i></td> <td>âˆ’1.7841</td> <td>0.023063</td> <td>Chlorophyll a-b binding protein</td> </tr> <tr> <td><i>Cluster-24,085.76692</i></td> <td>âˆ’5.3381</td> <td>0.022416</td> <td>Chlorophyll a-b binding protein</td> </tr> <tr> <td><i>Cluster-15,166.0</i></td> <td>âˆ’1.5438</td> <td>0.023391</td> <td>Chlorophyll a-b binding protein</td> </tr> <tr> <td><i>Cluster-24,085.41612</i></td> <td>âˆ’1.9349</td> <td>0.0097455</td> <td>Chlorophyll a-b binding protein</td> </tr> <tr> <td><i>Cluster-24,085.40718</i></td> <td>âˆ’1.8981</td> <td>0.0008757</td> <td>Chlorophyll a-b binding protein</td> </tr> <tr> <td colspan="5">Porphyrin and chlorophyll metabolism</td> </tr> <tr> <td>â€ƒL1 to L2</td> <td><i>Cluster-24,085.63483</i></td> <td>3.2081</td> <td>0.003801</td> <td>UDP-glycosyltransferase 76F1-like</td> </tr> </tbody> </table>  <div class="table-wrap_UNKNOWN" id="Tab3"><span class="label" tagx="label" title="label">Table 3</span> <p>List of DEGs related to biomass accumulation in <i>Asarum</i> leaves grown under different light intensities. L1, L2, L3 and L4 represent full sunlight, 50% sunlight, 28% sunlight and 12% sunlight, respectively</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th>Treatment</th> <th>Gene ID</th> <th>Log2 fold change</th> <th>Pvalue adjusted</th> <th>Description</th> </tr> </thead> <tbody> <tr> <td colspan="5">Ascorbate and aldarate metabolism</td> </tr> <tr> <td rowspan="2">â€ƒL1 to L2</td> <td><i>Cluster-24,085.74455</i></td> <td>âˆ’1.6322</td> <td>0.0091074</td> <td>L-ascorbate oxidase</td> </tr> <tr> <td><i>Cluster-24,085.63483</i></td> <td>3.2081</td> <td>0.003801</td> <td>UDP-glycosyltransferase 76F1-like</td> </tr> <tr> <td rowspan="2">â€ƒL1 to L4</td> <td><i>Cluster-24,085.74455</i></td> <td>âˆ’3.9989</td> <td>2.04E-09</td> <td>L-ascorbate oxidase</td> </tr> <tr> <td><i>Cluster-24,085.74454</i></td> <td>âˆ’3.9441</td> <td>3.30E-06</td> <td>L-ascorbate oxidase</td> </tr> <tr> <td rowspan="2">â€ƒL2 to L4</td> <td><i>Cluster-24,085.74455</i></td> <td>âˆ’2.3647</td> <td>0.001596</td> <td>L-ascorbate oxidase</td> </tr> <tr> <td><i>Cluster-24,085.50783</i></td> <td>âˆ’1.8271</td> <td>0.016912</td> <td>UDP-glycosyltransferase 76F1-like</td> </tr> <tr> <td>â€ƒL3to L4</td> <td><i>Cluster-24,085.74455</i></td> <td>âˆ’2.6414</td> <td>0.0095753</td> <td>L-ascorbate oxidase</td> </tr> <tr> <td colspan="5">Carbon fixation in photosynthetic organisms</td> </tr> <tr> <td>â€ƒL1 to L2</td> <td><i>Cluster-17,798.0</i></td> <td>6.3318</td> <td>0.0006952</td> <td>Phosphoenolpyruvate carboxykinase (ATP)</td> </tr> <tr> <td rowspan="2">â€ƒL2 to L3</td> <td><i>Cluster-20,520.0</i></td> <td>âˆ’6.3406</td> <td>0.03053</td> <td>Ribulose bisphosphate carboxylase small chain</td> </tr> <tr> <td><i>Cluster-23,656.0</i></td> <td>âˆ’6.1474</td> <td>0.0089179</td> <td>Ribulose-bisphosphate carboxylase small chain</td> </tr> <tr> <td>â€ƒL2 to L4</td> <td><i>Cluster-164.0</i></td> <td>âˆ’7.439</td> <td>0.0009329</td> <td>Ribulose bisphosphate carboxylase small chain</td> </tr> <tr> <td rowspan="2">â€ƒL3 to L4</td> <td><i>Cluster-23,656.0</i></td> <td>5.8291</td> <td>0.034387</td> <td>Ribulose-bisphosphate carboxylase small chain</td> </tr> <tr> <td><i>Cluster-164.0</i></td> <td>âˆ’7.3099</td> <td>0.0052386</td> <td>Ribulose bisphosphate carboxylase small chain</td> </tr> <tr> <td colspan="5">Starch and sucrose metabolism</td> </tr> <tr> <td>â€ƒL1 to L2</td> <td><i>Cluster-24,085.63483</i></td> <td>3.2081</td> <td>0.003801</td> <td>UDP-glycosyltransferase 76F1-like</td> </tr> <tr> <td rowspan="4">â€ƒL1 to L4</td> <td><i>Cluster-24,085.9268</i></td> <td>âˆ’2.4712</td> <td>7.81E-05</td> <td>Alpha,alpha-trehalose-phosphate synthase</td> </tr> <tr> <td><i>Cluster-24,085.20752</i></td> <td>4.2334</td> <td>1.38E-05</td> <td>Glucose-1-phosphate adenylyltransferase</td> </tr> <tr> <td><i>Cluster-24,085.45906</i></td> <td>2.3643</td> <td>2.53E-05</td> <td>1,4-alpha-glucan-branching enzyme</td> </tr> <tr> <td><i>Cluster-24,085.42146</i></td> <td>2.0882</td> <td>0.005112</td> <td>Beta-amylase 3, chloroplastic-like</td> </tr> <tr> <td rowspan="5">â€ƒL2to L4</td> <td><i>Cluster-24,085.48054</i></td> <td>âˆ’1.4951</td> <td>0.0051263</td> <td>UDP-glucuronate 4-epimerase 6</td> </tr> <tr> <td><i>Cluster-24,085.9268</i></td> <td>âˆ’1.877</td> <td>0.0049111</td> <td>Alpha,alpha-trehalose-phosphate synthase</td> </tr> <tr> <td><i>Cluster-24,085.77617</i></td> <td>âˆ’6.2549</td> <td>0.026686</td> <td>UDP-glucuronate decarboxylase</td> </tr> <tr> <td><i>Cluster-24,085.43359</i></td> <td>âˆ’2.5661</td> <td>0.049914</td> <td>Growth-regulating factor 1-like</td> </tr> <tr> <td><i>Cluster-24,085.42146</i></td> <td>2.4966</td> <td>0.0002086</td> <td>Beta-amylase 3, chloroplastic-like</td> </tr> </tbody> </table> </div> <p /> <p id="Par53">We searched our transcriptome for DEGs related to the carotenoid pathway and found one gene (<i>Cluster-9588.0</i>) that was upregulated under full sunlight conditions as compared to 50% as well as 12% light conditions. Additionally, we found two more unigenes (<i>Cluster-24,085.17524</i> and <i>Cluster-24,085.21006</i>) that were actually downregulated under full sunlight conditions as compared to 12% light. These genes are involved in the abscisic acid biosynthesis part of the carotenoid biosynthesis where the first gene controls the final step of xanthoxin formation while the second gene converts abscisate to dihydroxy-phaseic acid (Table <a href="#Tab2">2</a>; Additional Table <a href="#MOESM1">2</a>; Table <a href="#Tab4">4</a>). <div class="table-wrap_UNKNOWN" id="Tab4"><span class="label" tagx="label" title="label">Table 4</span> </div></p><p>DEGs enriched in plant hormone signal transduction pathway in <i>Asarum</i> leaves grown under different light intensities. L1, L2, L3 and L4 represent full sunlight, 50% sunlight, 28% sunlight and 12% sunlight, respectively</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th>Treatment</th> <th>Gene ID</th> <th>Log2 fold change</th> <th>Pvalue adjusted</th> <th>Description</th> </tr> </thead> <tbody> <tr> <td colspan="5">Plant hormone signal transduction</td> </tr> <tr> <td>â€ƒL1 to L2</td> <td><i>Cluster-24,085.2657</i></td> <td>3.4089</td> <td>0.035652</td> <td>Ethylene-responsive TF 1B-like</td> </tr> <tr> <td rowspan="6">â€ƒL1 to L4</td> <td><i>Cluster-24,085.21051</i></td> <td>âˆ’1.6807</td> <td>0.0015999</td> <td>Auxin responsive SAUR protein</td> </tr> <tr> <td><i>Cluster-24,085.68791</i></td> <td>âˆ’4.5239</td> <td>3.15E-05</td> <td>Xyloglucan:xyloglucosyl transferase TCH4</td> </tr> <tr> <td><i>Cluster-24,085.68790</i></td> <td>âˆ’3.7161</td> <td>0.0008913</td> <td>Xyloglucan:xyloglucosyl transferase TCH4</td> </tr> <tr> <td><i>Cluster-19,269.0</i></td> <td>âˆ’6.1483</td> <td>0.011149</td> <td>Auxin-responsive protein SAUR72</td> </tr> <tr> <td><i>Cluster-24,085.34259</i></td> <td>1.9218</td> <td>0.015288</td> <td>Histidine-containing phosphotransfer protein 4</td> </tr> <tr> <td><i>Cluster-24,085.18269</i></td> <td>âˆ’3.1107</td> <td>0.015652</td> <td>Xyloglucan:xyloglucosyl transferase TCH4</td> </tr> <tr> <td rowspan="14">â€ƒL2 to L4</td> <td><i>Cluster-24,085.21051</i></td> <td>âˆ’1.7653</td> <td>1.48E-05</td> <td>Auxin responsive SAUR protein</td> </tr> <tr> <td><i>Cluster-24,085.68791</i></td> <td>âˆ’5.2357</td> <td>4.67E-09</td> <td>Xyloglucan:xyloglucosyl transferase TCH4</td> </tr> <tr> <td><i>Cluster-24,085.68792</i></td> <td>âˆ’4.9808</td> <td>3.62E-08</td> <td>Xyloglucan:xyloglucosyl transferase TCH4</td> </tr> <tr> <td><i>Cluster-24,085.68790</i></td> <td>âˆ’2.8511</td> <td>0.014699</td> <td>Xyloglucan:xyloglucosyl transferase TCH4</td> </tr> <tr> <td><i>Cluster-19,269.0</i></td> <td>âˆ’4.8215</td> <td>0.014835</td> <td>Auxin-responsive protein SAUR72</td> </tr> <tr> <td><i>Cluster-24,085.51860</i></td> <td>âˆ’1.4164</td> <td>0.017439</td> <td>Auxin-responsive protein IAA4</td> </tr> <tr> <td><i>Cluster-24,085.34259</i></td> <td>2.3245</td> <td>0.0024764</td> <td>Histidine-containing phosphotransfer protein 4-like</td> </tr> <tr> <td><i>Cluster-24,085.3665</i></td> <td>âˆ’2.2309</td> <td>9.73E-09</td> <td>Protein phosphatase 2C 37-like</td> </tr> <tr> <td><i>Cluster-24,085.25146</i></td> <td>âˆ’1.5467</td> <td>0.002965</td> <td>TF MYC2</td> </tr> <tr> <td><i>Cluster-24,085.41514</i></td> <td>âˆ’1.4164</td> <td>0.017439</td> <td>Auxin-responsive protein IAA4</td> </tr> <tr> <td><i>Cluster-24,085.77156</i></td> <td>âˆ’1.886</td> <td>0.032</td> <td>Auxin early response protein SAUR41</td> </tr> <tr> <td><i>Cluster-24,085.18269</i></td> <td>âˆ’4.2914</td> <td>7.80E-28</td> <td>Xyloglucan:xyloglucosyl transferase TCH4</td> </tr> <tr> <td><i>Cluster-24,085.14689</i></td> <td>âˆ’3.7539</td> <td>0.0021302</td> <td>Auxin-responsive protein IAA25</td> </tr> <tr> <td><i>Cluster-24,085.60271</i></td> <td>âˆ’1.8161</td> <td>0.0065038</td> <td>Jasmonate ZIM domain-containing protein</td> </tr> <tr> <td rowspan="4">â€ƒL3 to L4</td> <td><i>Cluster-24,085.68791</i></td> <td>âˆ’2.8209</td> <td>0.022986</td> <td>Xyloglucan:xyloglucosyl transferase TCH4</td> </tr> <tr> <td><i>Cluster-24,085.34259</i></td> <td>2.624</td> <td>0.011739</td> <td>Histidine-containing phosphotransfer protein 4-like</td> </tr> <tr> <td><i>Cluster-24,085.18269</i></td> <td>âˆ’2.3918</td> <td>0.0027324</td> <td>Xyloglucan endotransglucosylase/hydrolase protein 22</td> </tr> <tr> <td><i>Cluster-24,085.2318</i></td> <td>âˆ’1.912</td> <td>0.036532</td> <td>Cyclin-D3â€"2</td> </tr> </tbody> </table>  <p /> <p id="Par54">Our results demonstrated that ascorbate and aldarate metabolism was a significantly enriched pathway under the studied light conditions. Therefore, we searched for DEGs related to this pathway and found that two genes annotated as L-ascorbate oxidase were downregulated in plants grown under full sunlight conditions (Table <a href="#Tab3">3</a>). The first gene (<i>Cluster-24,085.74455</i>) was downregulated in <i>Asarum</i> leaves grown under full to 50%, full to 12, 50 to 12%, and 28 to 12% sunlight conditions. This gene controls the conversion of L-ascorbate to L-dehydro ascorbate. The second gene (<i>Cluster-24,085.74454</i>) was the only differentially expressed gene between full to 12% sunlight conditions and plays the same role as the first gene (Table <a href="#Tab3">3</a>; Additional Table <a href="#MOESM1">4</a>). A UDP-glycosyltransferase 76F1-like gene (<i>Cluster-24,085.63483</i>) was upregulated in full sunlight grown <i>Asarum</i> leaves as compared to the ones grown under 50% light conditions. This gene converts UDP-D-glucuronate to D-glucuronate [<a href="#CR39">39</a>].</p> <p id="Par55">Carbon fixation is an important process in photosynthetic organisms, which affects carbon acquisition and biomass allocation. Light intensity has been reported to be an important factor in this regard [<a href="#CR40">40</a>]. We found that carbon fixation in photosynthetic organisms was one of the significantly enriched pathways under the studied light conditions (Additional Fig. <a href="#MOESM2">3</a>). The upregulation of a phosphoenolpyruvate carboxykinase (ATP) gene (<i>Cluster-17,798.0</i>) in full sunlight grown leaves as compared to those grown in 50% sunlight conditions was consistent with the findings of a previous study in soybean [<a href="#CR41">41</a>]. Three genes annotated as ribulose-bisphosphate carboxylase small chain were also differentially regulated under low-light intensities i.e. <i>Cluster-20,520.0</i> in 50 to 28%, <i>Cluster-23,656.0</i> in 50 to 28% and 28 to 12%, and <i>Cluster-164.0</i> in 50 to 28% and 28 to 12% light conditions (Table <a href="#Tab3">3</a>). However, these three genes did not differentially express under full sunlight conditions.</p> <p id="Par56">Another key pathway i.e. starch and sucrose metabolism pathway has an important role in overall plant development and biomass accumulation [<a href="#CR42">42</a>]. Several genes involved in this pathway such as UDP-glycosyltransferase 76F1-like, Glucose-1-phosphate adenylyltransferase, 1,4-alpha-glucan-branching enzyme, and Î²-amylase 3, chloroplastic-like, were upregulated in higher light intensities. Some genes such as alpha,alpha-trehalose-phosphate synthase, UDP-glucuronate 4-epimerase 6, UDP-glucuronate decarboxylase, and growth-regulating factor 1-like were downregulated in high light intensity to low intensity. Only one of these four genes (alpha,alpha-trehalose-phosphate synthase) was differentially regulated in full sunlight conditions while the other three were differentially regulated between 50 to 12% light conditions (Table <a href="#Tab3">3</a>; Additional Table <a href="#MOESM1">5</a>).</p> <p id="Par57">The processes of photosynthesis and biomass accumulation are affected by several pathways. Therefore, we searched for DEGs that have been reported in this regard. Stay-green genes regulate chlorophyll degradation during dark-induced senescence [<a href="#CR43">43</a>]. We noticed that one gene predicted as Stay-green protein (<i>Cluster-24,085.51159</i>) was upregulated in <i>Asarum</i> leaves grown under full sunlight grown leaves as compared to those grown in 50 and 12% light. Another predicted Stay-green gene (<i>Cluster-24,085.56669</i>) was upregulated between full to 12% sunlight conditions (Additional Table <a href="#MOESM1">2</a>; Table <a href="#Tab4">4</a>). Among other DEGs, we observed differential regulation of ATP-dependent DNA helicase Q-like genes, cytochrome P450, oxaloacetate decarboxylase, ZINC INDUCED FACILITATOR-LIKE 1-like genes, subtilisin-like proteases, BURP domain protein RD22, protein trichome birefringence-like 38, and receptor-like protein kinases [<a href="#CR44">44</a>â€"<a href="#CR46">46</a>].</p> </div> <div class="differentialregulationofgenesrelatedtohormones" title="sec"> <div class="title" tagx="title" title="title">
Differential regulation of genes related to hormones</div> <p id="Par58">We searched for DEGs related to hormone signaling pathways [<a href="#CR47">47</a>]. A gene (<i>Cluster-24,085.2657</i>) related to ethylene responsive factor in plant hormone signal transduction pathway was upregulated in full sunlight grown <i>Asarum</i> leaves as compared to 50% light treated leaves (Additional Table <a href="#MOESM1">2</a>). A histidine-containing phosphotransfer protein (<i>Cluster-24,085.34259</i>) was also upregulated in these light conditions. This gene was also upregulated in treatment in low light treatment comparisons i.e. 50 to 12% and 28 to 12% light conditions (Table <a href="#Tab4">4</a>). On the other hand, we noticed the downregulation of two auxin responsive SAUR proteins (<i>Cluster-24,085.21051</i> and <i>Cluster-19,269.0</i>) and three xyloglucan:xyloglucosyl transferase TCH4s (<i>Cluster-24,085.68791</i>, <i>Cluster-24,085.68790</i>, and <i>Cluster-24,085.18269</i>) in full sunlight grown <i>Asarum</i> leaves as compared to those grown in 12% light conditions (Additional Table <a href="#MOESM1">4</a>; Table <a href="#Tab4">4</a>). Both auxin responsive SAUR genes were also downregulated in <i>Asarum</i> leaves grown under 50% sunlight as compared to 12% sunlight conditions (Table <a href="#Tab4">4</a>; Additional Table <a href="#MOESM1">6</a>). A similar pattern was observed for the expression of xyloglucan:xyloglucosyl transferase TCH4s. We noticed that a relatively higher number of genes were differentially expressed between 50 and 12% sunlight conditions as compared to full sunlight versus other three light conditions. Those genes were annotated as auxin-responsive protein IAA4, protein phosphatase 2C 37-like, TF MYC2, auxin-responsive protein IAA4, auxin early response protein SAUR41, auxin-responsive protein IAA25, and jasmonate ZIM domain-containing protein (Table <a href="#Tab4">4</a>; Additional Table <a href="#MOESM1">6</a>).</p> </div> <div class="transcriptionfactorsactiveinregulationofgeneexpressionunderdifferentlightintensitiesinasarumleaves" title="sec"> <div class="title" tagx="title" title="title">
Transcription factors active in regulation of gene expression under different light intensities in Asarum leaves</div> <p id="Par59">Our comparative transcriptome data showed the upregulation of only one TF i.e. AP2/ERF-ERF (<i>Cluster-24,085.2657</i>) in full sunlight grown <i>Asarum</i> leaves as compared to 50% light grown leaves. By comparing the full sunlight and 28% light conditions, three TFs i.e. two AP2/ERF-ERFs (<i>Cluster-24,085.12486</i> and <i>Cluster-24,085.17619</i>) and one WRKY (<i>Cluster-24,085.14452</i>) were downregulated while, one WRKY, one MYB, one, TAZ, and one OFP were upregulated. The AP2/EFF-ERF TF (<i>Cluster-24,085.12486</i>) was also downregulated in full to 28%, full to 12, 50 to 28%, and 50 to 12% sunlight conditions. Similarly, the WRKY TF (<i>Cluster-24,085.14452</i>) was also downregulated in 50 to 12% light conditions. Eight AP2/ERF-ERFs, one AP2/ERF-RAV, one C2H2, one MBF1, and one NAC were downregulated between 50 and 28% light conditions while only one bHLH was upregulated. A relatively larger number of TFs belonging to 24 families were differentially regulated. Finally, the transcriptome comparison between 28 and 12% light conditions showed the downregulation of TFs belonging to AP2/ERF-EFF, C2C2-DOF, HSF, ZF-HD, and MYB-related gene families (Additional Table <a href="#MOESM1">7</a>). Interestingly, full sunlight comparisons with the three low light intensities resulted in the upregulation of some TFs while the comparisons between the low light conditions did not show upregulation of TFs except one bHLH (in 50 to 28% and 50 to 12%) (Table <a href="#Tab5">5</a>). <div class="table-wrap_UNKNOWN" id="Tab5"><span class="label" tagx="label" title="label">Table 5</span> </div></p><p>List of differentially regulated transcription factors in <i>Asarum</i> leaves grown under different light intensities. L1, L2, L3 and L4 represent full sunlight, 50% sunlight, 28% sunlight and 12% sunlight, respectively</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th>Treatment</th> <th>Gene_id</th> <th>Log2<br /><br />Fold<br /><br />Change</th> <th>Pvalue adjusted</th> <th>TF family</th> </tr> </thead> <tbody> <tr> <td>L1 to L2</td> <td><i>Cluster-24,085.2657</i></td> <td>3.4089</td> <td>0.035652</td> <td>AP2/ERF-ERF</td> </tr> <tr> <td rowspan="7">L1 to L3</td> <td><i>Cluster-24,085.12486</i></td> <td>âˆ’4.3794</td> <td>3.59E-05</td> <td>AP2/ERF-ERF</td> </tr> <tr> <td><i>Cluster-24,085.14452</i></td> <td>âˆ’3.184</td> <td>0.0020601</td> <td>WRKY</td> </tr> <tr> <td><i>Cluster-24,085.20439</i></td> <td>2.7652</td> <td>0.0096977</td> <td>WRKY</td> </tr> <tr> <td><i>Cluster-24,085.17619</i></td> <td>âˆ’2.7775</td> <td>0.018342</td> <td>AP2/ERF-ERF</td> </tr> <tr> <td><i>Cluster-13,720.1</i></td> <td>4.1745</td> <td>0.020535</td> <td>MYB</td> </tr> <tr> <td><i>Cluster-24,085.66840</i></td> <td>3.2079</td> <td>0.020558</td> <td>TAZ</td> </tr> <tr> <td><i>Cluster-24,085.18987</i></td> <td>3.438</td> <td>0.028837</td> <td>OFP</td> </tr> <tr> <td rowspan="3">L1 to L4</td> <td><i>Cluster-24,085.12486</i></td> <td>âˆ’6.3006</td> <td>6.07E-15</td> <td>AP2/ERF-ERF</td> </tr> <tr> <td><i>Cluster-24,085.10041</i></td> <td>âˆ’3.3336</td> <td>6.31E-13</td> <td>AP2/ERF-ERF</td> </tr> <tr> <td><i>Cluster-24,085.27017</i></td> <td>âˆ’5.5571</td> <td>6.31E-13</td> <td>HSF</td> </tr> <tr> <td rowspan="15">L2 to L3</td> <td><i>Cluster-24,085.12486</i></td> <td>âˆ’4.0405</td> <td>1.08E-09</td> <td>AP2/ERF-ERF</td> </tr> <tr> <td><i>Cluster-24,085.1987</i></td> <td>âˆ’3.8493</td> <td>8.26E-07</td> <td>AP2/ERF-ERF</td> </tr> <tr> <td><i>Cluster-24,085.17619</i></td> <td>âˆ’2.7933</td> <td>5.62E-06</td> <td>AP2/ERF-ERF</td> </tr> <tr> <td><i>Cluster-24,085.21290</i></td> <td>âˆ’2.1018</td> <td>2.75E-05</td> <td>AP2/ERF-ERF</td> </tr> <tr> <td><i>Cluster-24,085.2466</i></td> <td>âˆ’2.34</td> <td>3.19E-05</td> <td>MADS-MIKC</td> </tr> <tr> <td><i>Cluster-24,085.20260</i></td> <td>âˆ’2.7511</td> <td>4.87E-05</td> <td>C2H2</td> </tr> <tr> <td><i>Cluster-24,085.20026</i></td> <td>âˆ’2.5195</td> <td>6.62E-05</td> <td>AP2/ERF-ERF</td> </tr> <tr> <td><i>Cluster-24,085.10041</i></td> <td>âˆ’2.0953</td> <td>0.00012223</td> <td>AP2/ERF-ERF</td> </tr> <tr> <td><i>Cluster-24,085.42603</i></td> <td>âˆ’2.0689</td> <td>0.00017711</td> <td>AP2/ERF-ERF</td> </tr> <tr> <td><i>Cluster-24,085.37030</i></td> <td>âˆ’1.6079</td> <td>0.00026526</td> <td>MBF1</td> </tr> <tr> <td><i>Cluster-24,085.23378</i></td> <td>âˆ’1.7439</td> <td>0.0004069</td> <td>AP2/ERF-ERF</td> </tr> <tr> <td><i>Cluster-24,085.23170</i></td> <td>âˆ’1.6109</td> <td>0.0019938</td> <td>NAC</td> </tr> <tr> <td><i>Cluster-24,085.20351</i></td> <td>âˆ’1.6613</td> <td>0.0060609</td> <td>AP2/ERF-ERF</td> </tr> <tr> <td><i>Cluster-24,085.19577</i></td> <td>âˆ’1.0273</td> <td>0.012246</td> <td>AP2/ERF-RAV</td> </tr> <tr> <td><i>Cluster-24,085.52171</i></td> <td>2.1066</td> <td>0.041305</td> <td>bHLH</td> </tr> <tr> <td rowspan="68">L2 to L4</td> <td><i>Cluster-19,135.0</i></td> <td>âˆ’5.9595</td> <td>8.57E-05</td> <td>AP2/ERF-ERF</td> </tr> <tr> <td><i>Cluster-24,085.10041</i></td> <td>âˆ’3.8815</td> <td>3.67E-37</td> <td>AP2/ERF-ERF</td> </tr> <tr> <td><i>Cluster-24,085.10373</i></td> <td>âˆ’2.9536</td> <td>5.46E-08</td> <td>AP2/ERF-ERF</td> </tr> <tr> <td><i>Cluster-24,085.12486</i></td> <td>âˆ’5.961</td> <td>1.15E-30</td> <td>WRKY</td> </tr> <tr> <td><i>Cluster-24,085.1287</i></td> <td>âˆ’9.4367</td> <td>0.00089839</td> <td>AP2/ERF-ERF</td> </tr> <tr> <td><i>Cluster-24,085.14452</i></td> <td>âˆ’3.4296</td> <td>0.00013736</td> <td>WRKY</td> </tr> <tr> <td><i>Cluster-24,085.14689</i></td> <td>âˆ’3.7539</td> <td>0.0021302</td> <td>AUX/IAA</td> </tr> <tr> <td><i>Cluster-24,085.14837</i></td> <td>âˆ’1.0913</td> <td>0.025532</td> <td>AP2/ERF-ERF</td> </tr> <tr> <td><i>Cluster-24,085.15903</i></td> <td>âˆ’2.7791</td> <td>0.0008352</td> <td>MYB</td> </tr> <tr> <td><i>Cluster-24,085.16202</i></td> <td>âˆ’2.3493</td> <td>1.96E-05</td> <td>MYB</td> </tr> <tr> <td><i>Cluster-24,085.1707</i></td> <td>âˆ’3.3558</td> <td>6.79E-05</td> <td>AP2/ERF-ERF</td> </tr> <tr> <td><i>Cluster-24,085.17343</i></td> <td>âˆ’2.382</td> <td>8.54E-07</td> <td>AP2/ERF-ERF</td> </tr> <tr> <td><i>Cluster-24,085.17619</i></td> <td>âˆ’3.8718</td> <td>3.91E-20</td> <td>AP2/ERF-ERF</td> </tr> <tr> <td><i>Cluster-24,085.18098</i></td> <td>âˆ’2.036</td> <td>0.011069</td> <td>C2H2</td> </tr> <tr> <td><i>Cluster-24,085.18531</i></td> <td>âˆ’2.2255</td> <td>0.0014194</td> <td>MYB-related</td> </tr> <tr> <td><i>Cluster-24,085.19218</i></td> <td>âˆ’2.5795</td> <td>0.00041875</td> <td>NAC</td> </tr> <tr> <td><i>Cluster-24,085.19220</i></td> <td>âˆ’2.3685</td> <td>3.27E-10</td> <td>AP2/ERF-ERF</td> </tr> <tr> <td><i>Cluster-24,085.19364</i></td> <td>âˆ’3.1888</td> <td>0.039496</td> <td>GARP-G2-like</td> </tr> <tr> <td><i>Cluster-24,085.19396</i></td> <td>âˆ’1.7638</td> <td>3.92E-10</td> <td>C2C2-GATA</td> </tr> <tr> <td><i>Cluster-24,085.19455</i></td> <td>âˆ’3.2023</td> <td>7.25E-08</td> <td>WRKY</td> </tr> <tr> <td><i>Cluster-24,085.19577</i></td> <td>âˆ’1.6049</td> <td>3.33E-05</td> <td>AP2/ERF-RAV</td> </tr> <tr> <td><i>Cluster-24,085.19626</i></td> <td>âˆ’2.1474</td> <td>0.02618</td> <td>AP2/ERF-ERF</td> </tr> <tr> <td><i>Cluster-24,085.1987</i></td> <td>âˆ’5.5105</td> <td>5.08E-16</td> <td>AP2/ERF-ERF</td> </tr> <tr> <td><i>Cluster-24,085.20026</i></td> <td>âˆ’4.059</td> <td>9.32E-25</td> <td>AP2/ERF-ERF</td> </tr> <tr> <td><i>Cluster-24,085.20260</i></td> <td>âˆ’4.4895</td> <td>7.77E-25</td> <td>C2H2</td> </tr> <tr> <td><i>Cluster-24,085.20351</i></td> <td>âˆ’2.9081</td> <td>1.19E-16</td> <td>AP2/ERF-ERF</td> </tr> <tr> <td><i>Cluster-24,085.20438</i></td> <td>âˆ’1.8845</td> <td>0.0029452</td> <td>MYB</td> </tr> <tr> <td><i>Cluster-24,085.21290</i></td> <td>âˆ’2.9988</td> <td>6.58E-25</td> <td>AP2/ERF-ERF</td> </tr> <tr> <td><i>Cluster-24,085.21518</i></td> <td>âˆ’3.4887</td> <td>0.001799</td> <td>AP2/ERF-ERF</td> </tr> <tr> <td><i>Cluster-24,085.21683</i></td> <td>âˆ’1.3931</td> <td>0.0078483</td> <td>Others</td> </tr> <tr> <td><i>Cluster-24,085.21894</i></td> <td>âˆ’2.4352</td> <td>2.32E-06</td> <td>WRKY</td> </tr> <tr> <td><i>Cluster-24,085.21936</i></td> <td>âˆ’2.0129</td> <td>0.0056915</td> <td>WRKY</td> </tr> <tr> <td><i>Cluster-24,085.23378</i></td> <td>âˆ’3.1779</td> <td>3.20E-31</td> <td>AP2/ERF-ERF</td> </tr> <tr> <td><i>Cluster-24,085.23829</i></td> <td>âˆ’1.1127</td> <td>0.0333</td> <td>bHLH</td> </tr> <tr> <td><i>Cluster-24,085.2477</i></td> <td>âˆ’1.6509</td> <td>0.018848</td> <td>C2C2-Dof</td> </tr> <tr> <td><i>Cluster-24,085.25146</i></td> <td>âˆ’1.5467</td> <td>0.002965</td> <td>bHLH</td> </tr> <tr> <td><i>Cluster-24,085.2521</i></td> <td>âˆ’3.6674</td> <td>1.43E-10</td> <td>WRKY</td> </tr> <tr> <td><i>Cluster-24,085.27017</i></td> <td>âˆ’5.3399</td> <td>1.20E-11</td> <td>HSF</td> </tr> <tr> <td><i>Cluster-24,085.27420</i></td> <td>âˆ’2.0865</td> <td>0.0003884</td> <td>LOB</td> </tr> <tr> <td><i>Cluster-24,085.32540</i></td> <td>âˆ’1.9852</td> <td>0.0004087</td> <td>C2H2</td> </tr> <tr> <td><i>Cluster-24,085.3305</i></td> <td>âˆ’1.9823</td> <td>1.62E-12</td> <td>HB-HD-ZIP</td> </tr> <tr> <td><i>Cluster-24,085.35957</i></td> <td>âˆ’2.2082</td> <td>7.30E-11</td> <td>WRKY</td> </tr> <tr> <td><i>Cluster-24,085.36255</i></td> <td>âˆ’1.0905</td> <td>0.044329</td> <td>HB-HD-ZIP</td> </tr> <tr> <td><i>Cluster-24,085.37030</i></td> <td>âˆ’2.9731</td> <td>5.77E-06</td> <td>MBF1</td> </tr> <tr> <td><i>Cluster-24,085.38470</i></td> <td>âˆ’1.3752</td> <td>0.013487</td> <td>GRAS</td> </tr> <tr> <td><i>Cluster-24,085.41221</i></td> <td>âˆ’1.4224</td> <td>0.0035887</td> <td>C3H</td> </tr> <tr> <td><i>Cluster-24,085.4136</i></td> <td>âˆ’1.7074</td> <td>0.0025093</td> <td>C2C2-GATA</td> </tr> <tr> <td><i>Cluster-24,085.41514</i></td> <td>âˆ’1.4164</td> <td>0.017439</td> <td>AUX/IAA</td> </tr> <tr> <td><i>Cluster-24,085.42603</i></td> <td>âˆ’3.2229</td> <td>6.46E-18</td> <td>AP2/ERF-ERF</td> </tr> <tr> <td><i>Cluster-24,085.44733</i></td> <td>âˆ’3.4652</td> <td>6.20E-11</td> <td>AP2/ERF-ERF</td> </tr> <tr> <td><i>Cluster-24,085.45927</i></td> <td>âˆ’1.6676</td> <td>0.038415</td> <td>HSF</td> </tr> <tr> <td><i>Cluster-24,085.4609</i></td> <td>âˆ’1.6993</td> <td>0.018848</td> <td>GNAT</td> </tr> <tr> <td><i>Cluster-24,085.51860</i></td> <td>âˆ’1.4164</td> <td>0.017439</td> <td>AUX/IAA</td> </tr> <tr> <td><i>Cluster-24,085.52171</i></td> <td>2.0002</td> <td>0.035938</td> <td>bHLH</td> </tr> <tr> <td><i>Cluster-24,085.546</i></td> <td>âˆ’3.6975</td> <td>3.41E-15</td> <td>zf-HD</td> </tr> <tr> <td><i>Cluster-24,085.55958</i></td> <td>âˆ’1.5396</td> <td>0.0008922</td> <td>C2C2-GATA</td> </tr> <tr> <td><i>Cluster-24,085.60271</i></td> <td>âˆ’1.8161</td> <td>0.0065038</td> <td>Tify</td> </tr> <tr> <td><i>Cluster-24,085.60939</i></td> <td>âˆ’2.6318</td> <td>0.00011965</td> <td>NAC</td> </tr> <tr> <td><i>Cluster-24,085.61360</i></td> <td>âˆ’2.8417</td> <td>5.75E-13</td> <td>HSF</td> </tr> <tr> <td><i>Cluster-24,085.64005</i></td> <td>âˆ’1.1783</td> <td>0.028667</td> <td>LIM</td> </tr> <tr> <td><i>Cluster-24,085.72259</i></td> <td>âˆ’2.2086</td> <td>0.0072876</td> <td>B3-ARF</td> </tr> <tr> <td><i>Cluster-24,085.77102</i></td> <td>âˆ’1.3448</td> <td>0.032949</td> <td>TCP</td> </tr> <tr> <td><i>Cluster-24,085.77333</i></td> <td>âˆ’4.1789</td> <td>0.007682</td> <td>AP2/ERF-ERF</td> </tr> <tr> <td><i>Cluster-24,085.77948</i></td> <td>âˆ’6.5356</td> <td>4.01E-11</td> <td>AP2/ERF-ERF</td> </tr> <tr> <td><i>Cluster-24,085.809</i></td> <td>âˆ’6.9567</td> <td>1.55E-10</td> <td>AP2/ERF-ERF</td> </tr> <tr> <td><i>Cluster-24,085.995</i></td> <td>âˆ’5.6805</td> <td>0.0042634</td> <td>OFP</td> </tr> <tr> <td><i>Cluster-24,730.0</i></td> <td>âˆ’4.3569</td> <td>0.022909</td> <td>AP2/ERF-ERF</td> </tr> <tr> <td><i>Cluster-4248.0</i></td> <td>âˆ’5.3527</td> <td>0.039977</td> <td>MYB-related</td> </tr> <tr> <td rowspan="9">L3 to L4</td> <td><i>Cluster-24,085.10041</i></td> <td>âˆ’1.7807</td> <td>0.00096256</td> <td>AP2/ERF-ERF</td> </tr> <tr> <td><i>Cluster-24,085.10095</i></td> <td>âˆ’5.628</td> <td>0.0075127</td> <td>C2C2-Dof</td> </tr> <tr> <td><i>Cluster-24,085.17343</i></td> <td>âˆ’1.7345</td> <td>0.0064897</td> <td>AP2/ERF-ERF</td> </tr> <tr> <td><i>Cluster-24,085.19626</i></td> <td>âˆ’2.0562</td> <td>0.045191</td> <td>AP2/ERF-ERF</td> </tr> <tr> <td><i>Cluster-24,085.27017</i></td> <td>âˆ’3.4274</td> <td>0.00081077</td> <td>HSF</td> </tr> <tr> <td><i>Cluster-24,085.44733</i></td> <td>âˆ’2.0545</td> <td>0.0023397</td> <td>AP2/ERF-ERF</td> </tr> <tr> <td><i>Cluster-24,085.546</i></td> <td>âˆ’2.2331</td> <td>0.024219</td> <td>zf-HD</td> </tr> <tr> <td><i>Cluster-24,085.809</i></td> <td>âˆ’3.2375</td> <td>0.031137</td> <td>AP2/ERF-ERF</td> </tr> <tr> <td><i>Cluster-4248.0</i></td> <td>âˆ’7.0589</td> <td>0.014273</td> <td>MYB-related</td> </tr> </tbody> </table>  <p /> </div> </div> <div class="quantitativereal-timepcr(qrt-pcr)analysis" title="sec"> <div class="title" tagx="title" title="title">
Quantitative real-time PCR (qRT-PCR) analysis</div> <p id="Par60">We validated the expression profiles of ten <i>Asarum</i> genes (Fig.Â <a href="#Fig4">4</a>). The selection of the genes was random to achieve the objective of this analysis i.e. validation of the expression results observed in transcriptome analysis. The <i>Actin</i> gene was used as the internal control to standardize the data. Among the tested genes, five were upregulated and five were downregulated in full sunlight conditions as compared to low light conditions. The results of the qRT-PCR showed similar expression patterns as recorded in the transcriptome analysis confirming the reliability of our RNA-Seq data. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 4</span></a></div>   <p>qRT-PCR validation of the selected differentially expressed genes (DEGs) in Asarum leaves. L1, L2, L3 and L4 represent full sunlight, 50% sunlight, 28% sunlight and 12% sunlight, respectively</p>   </div> <p /> </div> </div> <div class="discussion" title="sec"> <div class="title" tagx="title" title="title">
Discussion</div> <div class="effectoflightonvolatileoilbiosynthesisinasarumleaves" title="sec"> <div class="title" tagx="title" title="title">
Effect of light on volatile oil biosynthesis in Asarum leaves</div> <p id="Par61"><i>Asarum</i> is a sciophyte that grows in habitats with reduced sunlight. Volatile oil is an essential component of its extracts that have importance as antifungal, antibacterial, antidepressant, and anxiolytic effects. Despite its medicinal importance and availability, only a limited knowledge exists about the effect of light conditions on the biosynthesis of volatile oil [<a href="#CR4">4</a>]. Hence, it is important to understand if the <i>Asarum</i> volatile oil componentsâ€™ are affected by growing this plant in full sunlight conditions. Recent work has shown that the composition of volatile oil detected through LC-MS was different under different light conditions but the total volatile oil content did not significantly differ among the <i>Asarum</i> plants [<a href="#CR7">7</a>]. Our results demonstrated that the individual percent content of each of the four components showed the positive effect of full sunlight conditions as compared to reduced light intensities. To understand the effect of light on the phenylpropanoid pathway, we searched for DEGs that were significantly enriched under the tested conditions. The established phenylpropanoid pathway has shown that coniferyl aldehyde is converted into coniferyl alcohol by the reaction of CAD which is an important step for the subsequent formation of methyleugenol, safrole, and myristicin [<a href="#CR25">25</a>] (Fig.Â <a href="#Fig5">5</a>). Two CADs were highly expressed in full sunlight conditions; one in full to 50% and one in full to 28% sunlight conditions. Therefore, the higher accumulation of these three components in the full sunlight conditions suggests that higher light intensities affect the volatile oil formation by increasing the CAD expression. Alternatively, the coniferyl alcohol can also be converted to lignin by the action of peroxidases. Therefore, the upregulation of CADs in <i>Asarum</i> leaves grown under higher light intensities (full sunlight) would lead to higher lignin content. In this regard, the upregulation of a peroxidase responsible for the lignin formation suggested that under higher light intensities, higher lignin content is produced in the <i>Asarum</i> leaves. This observation is similar with those of heliophytes where it is known that leaf lignin content increases with the increasing light intensities [<a href="#CR48">48</a>]. Once, the methyleugenol is formed, it can further be converted into safrole and then into myristicin. The formation of safrole is under the control of cytochrome p450s (specifically CYP719As) [<a href="#CR25">25</a>]. We found the upregulation of two CYP719As in higher light intensity (50%) to lower light intensity (12%) (Additional Table <a href="#MOESM1">5</a>) which may explain the higher safrole content and subsequently myristicin content in high light intensity. The fourth major component i.e. toluene is synthesized from orcinol by the action of orcinol-OMTs in rose. However, very little is known about the biosynthesis of orcinol, which is hydroxylated in position 3 and 5 [<a href="#CR27">27</a>]. Some studies have reported that orcinol is biosynthesized by the monolignol biosynthetic pathway [<a href="#CR49">49</a>]. A recent study on <i>Rhododendron dauricum</i> L. demonstrated that orcinol is produced from tetraketide non-enzymatically by decarboxylative aldol condensation [<a href="#CR50">50</a>]. In our results, no OMT was differentially regulated between full sunlight and 50% or 28% light conditions. However, two OMTs (one protein L-isoaspartyl methyltransferase (PIMT) and one trans-resveratrol di-O-methyltransferase (ROMT)) were differentially regulated between full and 12% sunlight conditions (Additional Table <a href="#MOESM1">4</a>). PIMT was downregulated while the ROMT was upregulated. ROMT is implicated in stilbene pathway where it catalyses the biosynthesis of pterostilbene from resveratrol in grapevine and our KEGG pathway mapping was also in agreement with this study [<a href="#CR51">51</a>]. Based on the percent content observations, it is found that somehow light affects toluene content but the pathway and the genes involved in this increase needs specific studies or a different approach. Altogether, these results propose that the light treatment had a positive effect on the contents of the four volatile oil component. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 5</span></a></div>   <p>Important steps of the phenylpropanoid pathway in the formation of methyleugenol, safrole, and myristicin and the formation of toluene. Where CADâ€‰=â€‰cinnamyl alcohol dehydrogenase, CAATâ€‰=â€‰coniferyl alcohol acyl transferase, PER25â€‰=â€‰peroxidase 25, IGSâ€‰=â€‰isoeugenol synthase, EGSâ€‰=â€‰eugenol synthase, EOMTâ€‰=â€‰eugenol O-methyltransferase, and CYP719Asâ€‰=â€‰cytochrome-450719As. Red colour shows the upregulation of DEGs in <i>Asarum</i> leaves in full sunlight conditions</p>   </div> <p /> </div> <div class="effectoffullsunlightconditionsonphotosynthesisandbiomassaccumulation" title="sec"> <div class="title" tagx="title" title="title">
Effect of full sunlight conditions on photosynthesis and biomass accumulation</div> <p id="Par62">Light intensity and quality are the most critical environmental factors for crop physiology and biochemistry. <i>Asarum</i> being a sciophyte grows in shady environment and there have been some studies which tried to address the effect of full sunlight on photosynthesis and related parameters. Our study also discusses the transcriptional changes within <i>Asarum</i> leaves related to photosynthesis and biomass accumulation. It is known that higher plantsâ€™ chloroplasts convert light into biological energy by employing electron transport chain and Calvin-Benson cycle. This process uses genes and proteins of both nuclear and chloroplast origins and their expression is highly dynamic and is primarily influenced by light [<a href="#CR29">29</a>]. When <i>Asarum</i> leaves were grown under full sunlight conditions, the photosynthesis-antenna proteins were downregulated as compared to the studied low light treatments i.e. 50, 28, and 12% light conditions (Additional Tables <a href="#MOESM1">2</a>, <a href="#MOESM1">3</a>, and <a href="#MOESM1">4</a>; Table <a href="#Tab2">2</a>). Previous research on photosynthesis has shown that photosynthetic electron transport rates are light saturated at approximately 1/4th of full sunlight intensity. This is due to large optical cross section of light harvesting antenna complexes that capture photons relatively faster (~â€‰10-fold faster) than the rate-limiting step in electron transport. Therefore, this downregulation of several antenna proteins in full sunlight conditions as compared to low light intensities is possibly due to this phenomenon.</p> <p id="Par63">Carotenoids play essential role in photosynthesis and photoprotection and studies have demonstrated that carotenoid biosynthesis is regulated by light [<a href="#CR52">52</a>, <a href="#CR53">53</a>]. Transcriptome comparison showed the upregulation of a Î²-carotene 3-hydroxylase in full sunlight conditions which converts Î±-carotene to zeinoxanthin, which is finally converted into O-lutein. It also directly converts Î±-carotene to O-lutein with the help of another enzyme LUT1 [<a href="#CR54">54</a>]. It is the most abundant carotenoid and the only ligand for site L1 in light-harvesting complex proteins. The upregulation of this gene might be the increasing demand to quench the antenna complex in <i>Asarum</i> leaves under full sunlight conditions similar to previous report in Arabidopsis [<a href="#CR55">55</a>]. In addition to this, the downregulation of one gene controlling the final step in the xanthoxin formation in full sunlight versus 12% sunlight conditions might also be the part of <i>Asarum</i> leavesâ€™ photoprotective strategy under full sunlight conditions (Table <a href="#Tab2">2</a>; Additional Table <a href="#MOESM1">2</a>; Table <a href="#Tab4">4</a>). So, the transcriptomic observations under full sunlight conditions related to antenna proteins as well as carotenoids suggest that <i>Asarum</i> leaves develop a photoprotective strategy under excess light conditions.</p> <p id="Par64">Because a moderate reduction in antenna size results in increased photosynthesis and resultantly higher biomass accumulation, so the observation of different genes related to biomass accumulation under full sunlight conditions in <i>Asarum</i> was expected. In this regard, the downregulation of L-ascrobate oxidases in full sunlight conditions could be important because it is previously known that high light intensity induces higher ascorbate levels in the leaves. It could be possibly due to low ascorbate mutation and needs further specific investigation. Nevertheless, It is documented that ascorbate mediates regulation of plant growth and plays important role in biomass accumulation in plants [<a href="#CR56">56</a>]. The upregulation of phosphoenylpryrovate carboxykinase (ATP) in full sunlight is quite interesting because it is reported that this enzyme is less active during the day time largely by dephosphorylation of the enzyme, which makes it less sensitive to stimulatory metabolites and more sensitive to inhibitory metabolites, such as malate. So further studies on this gene and its regulation in day time will be needed to understand its function in <i>Asarum</i> leaves [<a href="#CR57">57</a>]. Apart from these adaptations, plants also use various strategies to assimilate carbohydrates in vegetative organs. In this regard, one important strategy is to stay-green. This was recently discovered in maize where maize plants stay green for a longer time [<a href="#CR36">36</a>]. Furthermore, it is known that stay-green proteins regulate chlorophyll degradation in dark-induced senescence [<a href="#CR43">43</a>]. The upregulation of two stay-green genes in full sunlight conditions suggests that <i>Asarum</i> leaves, when provided with full sunlight, tend to accumulate higher biomass.</p> </div> <div class="effectoffullsunlightconditionsonhormone-signaltransduction" title="sec"> <div class="title" tagx="title" title="title">
Effect of full sunlight conditions on hormone-signal transduction</div> <p id="Par65">Light is vital for plant growth and development; it provides energy for photosynthesis, but also affects overall plant growth and development through the regulation of endogenous plant hormones [<a href="#CR58">58</a>]. Studies in Arabidopsis have reported that a retrograde signaling is associated with the redox and hormonal pathways. The study demonstrated that AP2/ERFs show a quick upregulation in plants moved from dark to light. However, this response is for a very short time and in full sunlight conditions, the expression is reduced subsequently. Therefore, when the immediate response is finished for the transition from dark to light, under regular sunlight, the expression of these TFs should decrease. The downregulation of AP2/ERFs in higher light intensities as compared to the lower light intensities suggests that a similar response exists in <i>Asarum</i> leaves (Table <a href="#Tab5">5</a>) [<a href="#CR59">59</a>]. However, we noticed that one AP2/ERF (<i>Cluster-24,085.2657</i>) was upregulated in full compared to 50% sunlight conditions. This was quite different from overall TFs of the same family and needs further specific investigation. The histidine-containing phosphotransfer protein which was upregulated under higher light conditions as compared to lower light conditions is implicated in cytokinin pathway where it plays an important role in cell division and shoot initiation [<a href="#CR60">60</a>]. Thus, this gene might be increasing the biomass through increasing cell division and should be studied further. Concerning this observation, the downregulation of xyloglucan:xyloglucosyl transferase TCH4s is interesting because cell wall modifications are needed for cell division [<a href="#CR61">61</a>]. Therefore, enzymes involved in xyloglucan modification may affect cell wall characteristics and, ultimately, cell shape and plant form [<a href="#CR62">62</a>, <a href="#CR63">63</a>]. Under low light conditions, plants experience increased auxin biosynthesis and transport as well as auxin sensitivity are enhanced [<a href="#CR64">64</a>]. That means, when sufficient light is available for normal growth of the plant, it should do the reverse. In our transcriptome data of low light and full sunlight conditions, the downregulation of auxin responsive SAUR genes suggests that <i>Asarum</i> leaves, when grow under low light conditions, increase auxin sensitivity by upregulating these genes and when the light is sufficient (full sunlight conditions), auxin sensitivity is no more needed [<a href="#CR65">65</a>]. Following the above statement that the auxin sensitivity is not needed under full sunlight conditions, the downregulation of AUX/IAAs (Auxin/indole-3-acetic acid) further confirms this adaptation in <i>Asarum</i> leaves [<a href="#CR64">64</a>].</p> </div> </div> <div class="conclusion" title="sec"> <div class="title" tagx="title" title="title">
Conclusion</div> <p id="Par66">In this study, high-throughput sequencing of <i>Asarum</i> transcriptome was performed under different light conditions. In-depth transcriptome analysis allowed us to identify the expression levels of key genes involved in the volatile oil biosynthesis. Specifically, we targeted genes involved in the biosynthesis of four major volatile oil components i.e. methyleugeol, safrol, myristicin, and toluene biosynthesis. The variation of the major genes in addition to the biochemical data of the four components demonstrated that high light conditions enrich <i>Asarum</i> volatile oil with these four components. We further studied the expression of genes related to photosynthesis under full sunlight conditions, which revealed complexity of gene expression in the studied sciophyte. We also found the expression of hormone-signaling pathway related genes in the light of previous studies. Our transcriptome data represent the first genomic resource of <i>Asarum</i> under different light conditions and lays the foundation for further research aiming at enriching important bioactive metabolites in medicinal plants using genetics, genomics, and biotechnology methods. We conclude that full sunlight conditions help <i>Asarum</i> to increase volatile oil biosynthesis. The light intensity under full sunlight conditions is managed by the regulation of antenna proteins and <i>Asarum</i> leaves accumulate biomass when grown in full sunlight. Based on these conclusions, it could be recommended that <i>A. heterotropoides</i> can be grown in full sunlight conditions for increased biomass and volatile oil biosynthesis.</p> </div> <div class="methods" title="sec"> <div class="title" tagx="title" title="title">
Methods</div> <div class="plantmaterial" title="sec"> <div class="title" tagx="title" title="title">
Plant material</div> <p id="Par67"><i>Asarum</i> [<i>A. heterotropoides</i> Fr. Schmidt var. <i>mandshuricum</i> (Maxim.) Kitag.] cultivar zhongnong xixin-1 was used as plant material in this study. For plant material, no voucher has been deposited in a genebank. The formal identification of the plant materials was undertaken by the corresponding author of this article (Zhiqing Wang). The plants were originally collected from the wild in June 2018 and kept at the Medicinal Herbs Garden of Jilin Agricultural University. No permissions are necessary to collect such samples. In October 2019, we transplanted ~â€‰250 four-years old dormant seedlings into 30â€‰cm diameter and 40â€‰cm tall pots filled with humus (pHâ€‰=â€‰6.28). All pots were then placed at Medicinal Herbs Garden of Jilin Agricultural University (43.80â€²N, 125.42â€²E) under normal growing condition. In the next year, soon after the leaves appeared (secondâ€‰week of May 2019), we subjected the plants to different light treatments for 21â€‰days; it is the time when <i>A. heterotropoides</i> leaves expand and it is known that the diurnal net photosynthetic rate increases when grown under variable light conditions [<a href="#CR38">38</a>]. The daytime temperature during 21â€‰days treatment was 27.4â€"31â€‰Â°C. Treatments consisted of full sunlight (L1), 50% sunlight; one layer of black nylon net providing 50% shade (L2), 28% sunlight; two layers of nylon net each providing ~â€‰50% shade (L3), and 12% sunlight; three layers of black nylon net each providing ~â€‰12% shade (L4), and were conducted in net houses (Fig.Â <a href="#Fig6">6</a>). Use of nets for the shade treatment was followed as reported and established in other studies [<a href="#CR66">66</a>, <a href="#CR67">67</a>]. Average day length for the experimental period was 15â€‰Â±â€‰1.5â€‰h. Each treatment involved 24 pots with three replications and each pot contained 4â€"5 plants. For each replication, 6â€"8 leaves were randomly collected for volatile oil constituent analyses. For RNA extraction the 3â€"4 randomly selected leaves were harvested from each treatment, washed and stored immediately at âˆ’â€‰80â€‰Â°C. We used only leaves which were free from worms or any pathogen. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 6</span></a></div>   <p>Light treatments <b>a)</b> full sunlight (L1), <b>b)</b> 50% sunlight (L2), <b>c)</b> 28% sunlight (L3), <b>d)</b> and 12% sunlight (L4)</p>   </div> <p /> </div> <div class="volatileoilconstituentanalysisbygaschromatography-massspectrometry" title="sec"> <div class="title" tagx="title" title="title">
Volatile oil constituent analysis by gas chromatography-mass spectrometry</div> <p id="Par68">After treatment, the leaves were harvested, cleaned, and placed in the shade with good ventilation. The dried leaves were ground into powder and subjected to hydrodistillation for 6â€‰h. 10â€‰Î¼L of volatile oil was drawn from the solution and diluted 50 times using petroleum ether. 1â€‰Î¼L of this diluted sample was analyzed by gas chromatography-mass spectrometry. A HP-5MS highly polar capillary column (30â€‰mâ€‰Ã—â€‰0.32â€‰mmâ€‰Ã—â€‰0.25â€‰Î¼m, Hewlett-Packart, Palo Alto, CA, USA) coated with a 100% polyethylene glycol stationary phase was used. The following oven temperatures and times were used: 40â€‰Â°C (held for 2â€‰min), raised from 40â€‰Â°C to 160â€‰Â°C at a rate of 2.5â€‰Â°C/min, from 160â€‰Â°C to 280â€‰Â°C at a rate of 8â€‰Â°C/min, finally, held at 280â€‰Â°C for 10â€‰min. The injection temperature was set at 280â€‰Â°C. Helium was used as the carrier gas with a flow rate of 1.0â€‰mL/min. An injection volume of 1â€‰Î¼L was used with a split ratio of 100:1. The mass spectrometer was operated under a mode of electron impact (EI) at 70â€‰eV with the scan ranges were between 30 and 550â€‰amu. The ion source temperature was maintained at 230â€‰Â°C and the quadrupole at 150â€‰Â°C. Compound identification was done by comparing the NIST library data of the peaks with those reported in literature, and mass spectra was compared with peaks from literature data. Percentage composition was computed from GC peak areas on with DB-5â€‰ms column without applying correction factors.</p> </div> <div class="rnaisolationandlibrarypreparation" title="sec"> <div class="title" tagx="title" title="title">
RNA isolation and library preparation</div> <p id="Par69">Total RNA from the 6â€"8 leaves was extracted using Qiagen RNeasy mini kit (Qiagen, USA) following the manufacturerâ€™s protocol and was pooled and considered as one replicate. For each treatment, three replicates were processed separately. The concentration of the RNA was measured by using a NanoPhotometer spectrophotometer (IMPLEN, CA, USA) and the purity was assessed by using RNA Nano 6000 Assay Kit of the Agilent Bioanalyzer 2100 system (Agilent Technologies, CA, USA). Sequencing libraries were created using NEB Next Ultra RNA Library Prep Kit (NEB, USA) following manufacturerâ€™s instructions. Briefly, the mRNA was purified from 1.5â€‰Î¼g total RNA of each of three replicate using poly-T oligo-attached magnetic beads. Subsequently, the fragmentation buffer was used to break the RNA into short fragments, and the short-fragment RNA was used as a template to synthesize the first strand cDNA with random hexamer primer and M-MuLV Reverse Transcriptase (RNase H<sup>âˆ’</sup>), followed by buffer, dNTPs (dUTP, dATP, dGTP, and dCTP). Second strand cDNA synthesis was subsequently performed using DNA Polymerase I and RNase H and the double-stranded cDNA was purified using AMPure XP beads (Beckman Coulter, Beverly, USA). The purified double-stranded cDNA was subjected to terminal repair, A tail was added, and the sequencing linker was ligated, and then AMPure XP beads were used for fragment size selection, and finally PCR enrichment was performed to obtain a final cDNA library. Library quality was initially quantified using Qubit 2.0 using the 2100 to test the insert size of the library followed by accurately quantifying the effective concentration of the library (&amp;gt;â€‰2â€‰nM) by q-PCR. Finally, twelve paired-end cDNA libraries with an insert size of 300â€‰bp were constructed for transcriptome sequencing and sequenced on Illumina HiSeq platform (Illumina Inc., San Diego, CA, USA) by Novogene, Beijing, China (<a href="https://en.novogene.com/">https://en.novogene.com/</a>).</p> </div> <div class="denovoassembly,functionalannotation,classificationandmetabolicpathwayanalysis" title="sec"> <div class="title" tagx="title" title="title">
De novo assembly, functional annotation, classification and metabolic pathway analysis</div> <p id="Par70">The raw reads were filtered to obtain clean reads. Adapter sequences were trimmed, low quality reads containing &amp;gt;â€‰50% bases with a Phred quality scoreâ€‰â‰¤â€‰20 were removed, and reads with unknown nucleotides (more than 10% ambiguous residues N) using the FastQC tool (<a href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/">http://www.bioinformatics.babraham.ac.uk/projects/fastqc/</a>) [<a href="#CR68">68</a>]. To stitch clean reads, Trinity was used (Version r20140717, [<a href="#CR69">69</a>]) and Trinity spliced transcript sequences were used as reference sequences for subsequent analyses. For hierarchical clustering, Corset was used (<a href="https://code.google.com/p/corset-project/">https://code.google.com/p/corset-project/</a>). The assembled unigenes were then aligned with various databases such as KEGG [<a href="#CR70">70</a>], GO [<a href="#CR71">71</a>], Clusters of Orthologous Groups (COG) [<a href="#CR72">72</a>], PfAM (protein family), Swissprot [<a href="#CR73">73</a>], NR (NCBI non-redundant protein sequences) [<a href="#CR74">74</a>], KOG [<a href="#CR75">75</a>], and NT (NCBI non-redundant nucleotide sequences). The software KAAS (Moriya et al., 2007) was employed to get the unigene KEGG orthology with a threshold of E-valueâ€‰=â€‰1.0â€‰Ã—â€‰10<sup>âˆ’â€‰10</sup>. For Swiss-Prot (E-valueâ€‰=â€‰1.0â€‰Ã—â€‰10<sup>âˆ’â€‰5</sup>), KOG/COG (E-valueâ€‰=â€‰1.0â€‰Ã—â€‰10<sup>âˆ’â€‰3</sup>), and NR (E-valueâ€‰=â€‰1.0â€‰Ã—â€‰10<sup>âˆ’â€‰5</sup>) annotation we employed diamond V 0.8.22, respectively. For Nt annotation, NCBI BLAST 2.2.28+ was used with a threshold of E-valueâ€‰=â€‰1.0â€‰Ã—â€‰10<sup>âˆ’â€‰5</sup>. The analogs of the unigene amino acid sequences were searched against the Pfam database [<a href="#CR76">76</a>] using HMMER tool [<a href="#CR77">77</a>] with a threshold of E-valueâ€‰=â€‰0.01. For GO annotation was performed using Blast2GO V2.5 with a threshold E-valueâ€‰=â€‰1.0â€‰Ã—â€‰10<sup>âˆ’â€‰6</sup>. The sequenced reads were compared with the unigene library using Bowtie2 with default parameters [<a href="#CR78">78</a>], and the level of expression was estimated in combination with RSEM [<a href="#CR79">79</a>]. The gene expression level was determined according to the Fragments Per Kilobase of transcript sequence per Millions base pairs sequenced (FPKM). For overall RNA-seq quality assessment, Pearson correlation check between samples (and treatments) was performed.</p> </div> <div class="differentialexpressionandenrichmentanalysis" title="sec"> <div class="title" tagx="title" title="title">
Differential expression and enrichment analysis</div> <p id="Par71">The read count was normalized and DESeq R package (1.10.1) [<a href="#CR80">80</a>] was used to determine the differential expression genes (DEGs) between the treatments (I-IV) using a model based on the negative binomial distribution. The resulting <i>P</i> values were adjusted using the Benjamini and Hochbergâ€™s approach for controlling the false discovery rate [<a href="#CR81">81</a>] and FDR correction set at <i>p</i>â€‰<div class="â€‰0.05._UNKNOWN">
pâ€‰<div class="â€‰0.05_UNKNOWN" href="#CR82">
82]. KEGG pathway enrichment analysis of the DEGs was done using KOBAS2.0 [<a href="#CR83">83</a>]. The FDR correction was employed (<i>P</i>â€‰<div class="â€‰0.05)_UNKNOWN"> </div></div></div></p></div> <div class="identificationoftranscriptionfactors" title="sec"> <div class="title" tagx="title" title="title">
Identification of transcription factors</div> <p id="Par72">To perform the identification of transcription factors (TFs) in <i>A. heterotropoides</i> transcriptome data, we used iTAK software. The basic principal is to use the TF family and rules defined in the database to identify TF by hmmscan. TF identification and classification methods were followed as described earlier [<a href="#CR84">84</a>, <a href="#CR85">85</a>].</p> </div> <div class="quantitativert-pcranalysis" title="sec"> <div class="title" tagx="title" title="title">
Quantitative RT-PCR analysis</div> <p id="Par73">Ten DEGs, characterized by interesting expression profiles in different shade treatments were selected for qRT-PCR. First strand cDNAs was synthesized from 100â€‰ng of total RNA using the High Capacity cDNA Reverse Transcription Kit (Applied Biosystem). Primers were designed using Primer3 Software (<a href="http://frodo.wi.mit.edu/primer3/">http://frodo.wi.mit.edu/primer3/</a>; Additional Table <a href="#MOESM1">8</a>) and the specificity was checked by blasting their sequences in the NCBI database. The <i>Actin</i> constitutively expressed gene was used as reference gene [5]. All qRT-PCR reactions were carried out on a Rotor-Gene 6000 machine (Qiagen) with the following thermal cycling profile: 50â€‰Â°C for 2â€‰min and 95â€‰Â°C for 2â€‰min, followed by 40â€‰cycles at 95â€‰Â°C for 3â€‰s and 60â€‰Â°C for 30â€‰s. Melting curve analysis was performed to verify single product amplification with temperature ranging from 55 to 95â€‰Â°C by increasing of 1â€‰Â°C every step. All reactions were performed in a total volume of 10â€‰Î¼l containing 30â€‰ng of cDNA, 5â€‰Î¼l 1â€‰Ã—â€‰SYBRÂ® Select Master Mix (Applied Biosystem) and 0.2â€‰Î¼l (20â€‰Î¼M) of each primer. For each sample, two biological replicates were analyzed in independent runs and a no-template control was included for each gene. Intra-assay variation was evaluated by performing all reactions in triplicate. The quantification cycle (Cq) was automatically determined using Rotor-Gene 6000 Series Software, version 1.7 as reported earlier [<a href="#CR86">86</a>].</p> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary Information</div> <div class="sec" title="sec"> <p> </p><div class="local-data" title="local-data"> <div class="media" title="media"><a href="12864_2020_7266_MOESM1_ESM.xlsx">LINK</a> <p><b>Additional file 1: Table 1</b>. Summary of de novo transcriptome sequencing output. L1, L2, L3 and L4 represent full sunlight, 50% sunlight, 28% sunlight and 12% sunlight, respectively. <b>Table 2</b>. Differentially expressed genes in <i>Asarum</i> leaves grown under full and 50% sunlight conditions. <b>Table 3</b>. Differentially expressed genes in <i>Asarum</i> leaves grown under full and 28% sunlight conditions. <b>Table 4</b>. Differentially expressed genes in <i>Asarum</i> leaves grown under full and 12% sunlight conditions. <b>Table 5</b>. Differentially expressed genes in <i>Asarum</i> leaves grown under 50 and 28% sunlight conditions. <b>TableÂ 6</b>. Differentially expressed genes in <i>Asarum</i> leaves grown under 50 and 12% sunlight conditions. <b>TableÂ 7</b>. Differentially expressed genes in <i>Asarum</i> leaves grown under 28 and 12% sunlight conditions. <b>TableÂ 8</b>. List of primers used for qRT-PCR analysis.</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="12864_2020_7266_MOESM2_ESM.docx">LINK</a> <p><b>Additional file 2: Figure 1</b>. Overview of transcriptome assembly data showing the size distribution of transcripts. <b>Figure 2</b>. Volcano plots of DEGs between treatments a) I vs II b) I vs III, c) I vs IV, d) II vs III, e) II vs IV, and f) III vs IV. I, II, III and IV represent full sunlight, 50% sunlight, 28% sunlight and 12% sunlight, respectively. <b>Figure 3</b>. KEGG enrichment analysis of DEGs between treatments a) I vs II b) I vs III, c) I vs IV, d) II vs III, e) II vs IV, and f) III vs IV. I, II, III and IV represent full sunlight, 50% sunlight, 28% sunlight and 12% sunlight, respectively.</p>  </div> </div> <p /> </div> </div> </div> <div class="back" title="back"> <div class="glossary" title="glossary"> <div class="title" tagx="title" title="title">
Abbreviations</div> <div class="def-list" title="def-list"> <div class="def-item" title="def-item"> <div class="term" title="term">
ATP</div> <div class="def" title="def"> <p id="Par4">Adenosine triphosphate</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
bHLH</div> <div class="def" title="def"> <p id="Par5">Basic helix-loop-helix</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
C2H2</div> <div class="def" title="def"> <p id="Par6">Cys<sub>2</sub>His<sub>2</sub> zinc finger motifs</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
CAAT</div> <div class="def" title="def"> <p id="Par7">Coniferyl alcohol acyl transferase</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
CAD</div> <div class="def" title="def"> <p id="Par8">Cinnamyl alcohol dehydrogenase</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
cDNA</div> <div class="def" title="def"> <p id="Par9">Complementary DNA</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
CYP719As</div> <div class="def" title="def"> <p id="Par10">Cytochrome-450719As</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
DEG</div> <div class="def" title="def"> <p id="Par11">Differentially expressed gene</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
EGS</div> <div class="def" title="def"> <p id="Par12">Eugenol synthase</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
EOMT</div> <div class="def" title="def"> <p id="Par13">Eugenol O-methyltransferase</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
ERF</div> <div class="def" title="def"> <p id="Par14">Ethylene response factor</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
FPKM</div> <div class="def" title="def"> <p id="Par15">Fragments per kilobase of transcript per million fragments mapped</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
Gb</div> <div class="def" title="def"> <p id="Par16">Giga bite</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
GC-MS</div> <div class="def" title="def"> <p id="Par17">Gas chromatography-mass spectrometery</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
GO</div> <div class="def" title="def"> <p id="Par18">Gene ontology</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
HSF</div> <div class="def" title="def"> <p id="Par19">Heat shock factors</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
IAA</div> <div class="def" title="def"> <p id="Par20">Indole acetic acid</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
IGS</div> <div class="def" title="def"> <p id="Par21">Isoeugenol synthase</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
KEGG</div> <div class="def" title="def"> <p id="Par22">Kyoto encyclopedia of genes and genomes</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
KOG</div> <div class="def" title="def"> <p id="Par23">Ortholog groups</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
L1</div> <div class="def" title="def"> <p id="Par24">Full sunlight</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
L2</div> <div class="def" title="def"> <p id="Par25">50% sunlight</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
L3</div> <div class="def" title="def"> <p id="Par26">28% sunlight</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
L4</div> <div class="def" title="def"> <p id="Par27">12% sunlight</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
LC-MS</div> <div class="def" title="def"> <p id="Par28">Liquid chromatography-mass spectrometry</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
LUT1</div> <div class="def" title="def"> <p id="Par29">Lutein 1</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
MBF</div> <div class="def" title="def"> <p id="Par30">Mlu1-binding factor</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
NAC</div> <div class="def" title="def"> <p id="Par31">NAM, ATAF, AND CUC transcription factor</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
NR</div> <div class="def" title="def"> <p id="Par32">Non-redundant</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
NT</div> <div class="def" title="def"> <p id="Par33">Nucleotide</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
OMT</div> <div class="def" title="def"> <p id="Par34">O-methyltransferases</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
PCR</div> <div class="def" title="def"> <p id="Par35">Polymerase chain reaction</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
PER25</div> <div class="def" title="def"> <p id="Par36">Peroxidase 25</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
PIMT</div> <div class="def" title="def"> <p id="Par37">Protein L-isoaspartyl methyltransferase</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
ROMT</div> <div class="def" title="def"> <p id="Par38">Trans-resveratrol di-O-methyltransferase</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
SAUR</div> <div class="def" title="def"> <p id="Par39">Small auxin up-regulated RNA</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
SD</div> <div class="def" title="def"> <p id="Par40">Standard deviation</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
TCH4</div> <div class="def" title="def"> <p id="Par41">Touch 4, xyloglucan endotransglycosylase</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
TF</div> <div class="def" title="def"> <p id="Par42">Transcription factor</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
ZF-HD</div> <div class="def" title="def"> <p id="Par43">Zinc-finger-homeodomain</p> </div> </div> </div> </div> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p><b>Publisherâ€™s Note</b></p> <p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p> </div> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>Not applicable.</p> </div> <div class="title" tagx="title" title="title">
Authorsâ€™ contributions</div> <p>Z W: design the experiment, conducted the experiment, data analysis and wrote the manuscript. H M, M Z, Z W, Y T, W L, Y W: performed the experiment, data analysis. All authors have read and approved the final version of this manuscript.</p> <div class="title" tagx="title" title="title">
Funding</div> <p>This study was funded by the National Natural Science Foundation of China (81403052), Science and Technology Department Project from Jilin Province of China (20190304017YY), National key research special project of China (2019YFC1710070). The Funder has no role in the study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p> <div class="title" tagx="title" title="title">
Availability of data and materials</div> <p>The RNA-seq data has been submitted to NCBI SRA: PRJNA603253.</p> <div class="title" tagx="title" title="title">
Ethics approval and consent to participate</div> <p id="Par74">Not applicable.</p> <div class="title" tagx="title" title="title">
Consent for publication</div> <p id="Par75">Not applicable.</p> <div class="title" tagx="title" title="title">
Competing interests</div> <p id="Par76">The authors declare that they have no competing interests.</p> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="CR1" /><span class="label" tagx="label" title="label">1.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Li Y, Han L, Huang C, Dai W, Tian G, Huang F, Li J, Liu J, Wang Q, Zhou Z. New contributions to Asarum powder on immunology related toxicity effects in lung. Evid Based Complement Alternat Med. 2018;2018:1-14.</span></li> <li tag="ref"><a name="CR2" /><span class="label" tagx="label" title="label">2.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dan</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">H-Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gao</span><span class="given-names" tagx="given-names" title="given-names">W-W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">S-L</span></span></span><span class="mixed-article-title" title="mixed-article-title">Activities of essential oils from Asarum heterotropoides var. mandshuricum against five phytopathogens</span><span class="source" tagx="source" title="source">Crop Prot</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">29</span><span class="fpage" tagx="fpage" title="fpage">295</span><span class="lpage" tagx="lpage" title="lpage">299</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.cropro.2009.12.007">10.1016/j.cropro.2009.12.007</a></span></span></li> <li tag="ref"><a name="CR3" /><span class="label" tagx="label" title="label">3.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheng, C.-y.; Yang, C.-s.</span></span></span><span class="mixed-article-title" title="mixed-article-title">A synopsis of the Chinese species of Asarum (Aristolochiaceae)</span><span class="source" tagx="source" title="source">J Arnold Arboretum</span><span class="year" tagx="year" title="year">1983</span><span class="volume" tagx="volume" title="volume">64</span><span class="fpage" tagx="fpage" title="fpage">565</span><span class="lpage" tagx="lpage" title="lpage">597</span></span></li> <li tag="ref"><a name="CR4" /><span class="label" tagx="label" title="label">4.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haque</span><span class="given-names" tagx="given-names" title="given-names">AT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moon</span><span class="given-names" tagx="given-names" title="given-names">JN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saravana</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tilahun</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chun</span><span class="given-names" tagx="given-names" title="given-names">B-S</span></span></span><span class="mixed-article-title" title="mixed-article-title">Composition of <i>Asarum heterotropoides</i> var. mandshuricum radix oil from different extraction methods and activities against human body odor-producing bacteria</span><span class="source" tagx="source" title="source">J Food Drug Anal</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">24</span><span class="fpage" tagx="fpage" title="fpage">813</span><span class="lpage" tagx="lpage" title="lpage">821</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jfda.2016.04.006">10.1016/j.jfda.2016.04.006</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28911620">28911620</a></span></span></li> <li tag="ref"><a name="CR5" /><span class="label" tagx="label" title="label">5.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hui</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="mixed-article-title" title="mixed-article-title">Deep sequencing and transcriptome analysis to identify genes related to biosynthesis of aristolochic acid in Asarum heterotropoides</span><span class="source" tagx="source" title="source">Sci Rep</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">14</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41598-017-17765-5">10.1038/s41598-017-17765-5</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29311619">29311619</a></span></span></li> <li tag="ref"><a name="CR6" /><span class="label" tagx="label" title="label">6.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fang</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ma</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tian</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span></span><span class="mixed-article-title" title="mixed-article-title">Photosynthetic characteristics and active ingredients differences of Asarum heterotropoides var. mandshuricum under different light irradiance</span><span class="source" tagx="source" title="source">Zhongguo Zhong Yao Za Zhi</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">44</span><span class="fpage" tagx="fpage" title="fpage">2753</span><span class="lpage" tagx="lpage" title="lpage">2761</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31359687">31359687</a></span></span></li> <li tag="ref"><a name="CR7" /><span class="label" tagx="label" title="label">7.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zheng</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pang</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guan</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guo</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span></span><span class="mixed-article-title" title="mixed-article-title">Effect of light intensity on growth and quality of Asarum heterotropoides var. mandshuricum</span><span class="source" tagx="source" title="source">Zhongguo Zhong Yao Za Zhi</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">36</span><span class="fpage" tagx="fpage" title="fpage">1558</span><span class="lpage" tagx="lpage" title="lpage">1567</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22007533">22007533</a></span></span></li> <li tag="ref"><a name="CR8" /><span class="label" tagx="label" title="label">8.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang, X.-j.; Xia, X.-h.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Analysis of season variation of methyleugenol and safrole in Asarum heterotropoides by gas chromatography</span><span class="source" tagx="source" title="source">Se Pu</span><span class="year" tagx="year" title="year">1997</span><span class="volume" tagx="volume" title="volume">15</span><span class="fpage" tagx="fpage" title="fpage">85</span><span class="lpage" tagx="lpage" title="lpage">86</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15739449">15739449</a></span></span></li> <li tag="ref"><a name="CR9" /><span class="label" tagx="label" title="label">9.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hou, J.-l.; Li, W.-d.; Zheng, Q.-y.; Wang, W.-q.; Xiao, B.; Xing, D.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Effect of low light intensity on growth and accumulation of secondary metabolites in roots of Glycyrrhiza uralensis Fisch</span><span class="source" tagx="source" title="source">Biochem Syst Ecol</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">38</span><span class="fpage" tagx="fpage" title="fpage">160</span><span class="lpage" tagx="lpage" title="lpage">168</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.bse.2009.12.026">10.1016/j.bse.2009.12.026</a></span></span></li> <li tag="ref"><a name="CR10" /><span class="label" tagx="label" title="label">10.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chang</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alderson</span><span class="given-names" tagx="given-names" title="given-names">PG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wright</span><span class="given-names" tagx="given-names" title="given-names">CJ</span></span></span><span class="mixed-article-title" title="mixed-article-title">Solar irradiance level alters the growth of basil (Ocimum basilicum L.) and its content of volatile oils</span><span class="source" tagx="source" title="source">Environ Exp Bot</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">63</span><span class="fpage" tagx="fpage" title="fpage">216</span><span class="lpage" tagx="lpage" title="lpage">223</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.envexpbot.2007.10.017">10.1016/j.envexpbot.2007.10.017</a></span></span></li> <li tag="ref"><a name="CR11" /><span class="label" tagx="label" title="label">11.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fernandes</span><span class="given-names" tagx="given-names" title="given-names">VF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Almeida</span><span class="given-names" tagx="given-names" title="given-names">LBD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">FeijÃ³</span><span class="given-names" tagx="given-names" title="given-names">EVDS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Silva</span><span class="given-names" tagx="given-names" title="given-names">DDC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oliveira</span><span class="given-names" tagx="given-names" title="given-names">RAD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mielke</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Costa</span><span class="given-names" tagx="given-names" title="given-names">LCDB</span></span></span><span class="mixed-article-title" title="mixed-article-title">Light intensity on growth, leaf micromorphology and essential oil production of <i>Ocimum gratissimum</i></span><span class="source" tagx="source" title="source">Rev Bras</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">23</span><span class="fpage" tagx="fpage" title="fpage">419</span><span class="lpage" tagx="lpage" title="lpage">424</span></span></li> <li tag="ref"><a name="CR12" /><span class="label" tagx="label" title="label">12.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fadil</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Farah</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ihssane</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haloui</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lebrazi</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zghari</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rachiq</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="mixed-article-title" title="mixed-article-title">Chemometric investigation of light-shade effects on essential oil yield and morphology of Moroccan Myrtus communis L</span><span class="source" tagx="source" title="source">SpringerPlus</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">5</span><span class="fpage" tagx="fpage" title="fpage">1062</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s40064-016-2749-5">10.1186/s40064-016-2749-5</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27462510">27462510</a></span></span></li> <li tag="ref"><a name="CR13" /><span class="label" tagx="label" title="label">13.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Thakur M, Bhatt V, Kumar R. Effect of shade level and mulch type on growth, yield and essential oil composition of damask rose (Rosa damascena mill.) under mid hill conditions of Western Himalayas. PLoS One. 2019;14:e0214672.</span></li> <li tag="ref"><a name="CR14" /><span class="label" tagx="label" title="label">14.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Herrmann</span><span class="given-names" tagx="given-names" title="given-names">KM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weaver</span><span class="given-names" tagx="given-names" title="given-names">LM</span></span></span><span class="mixed-article-title" title="mixed-article-title">The shikimate pathway</span><span class="source" tagx="source" title="source">Annu Rev Plant Biol</span><span class="year" tagx="year" title="year">1999</span><span class="volume" tagx="volume" title="volume">50</span><span class="fpage" tagx="fpage" title="fpage">473</span><span class="lpage" tagx="lpage" title="lpage">503</span><span class="pub-id"><a href="https://dx.doi.org/10.1146/annurev.arplant.50.1.473">10.1146/annurev.arplant.50.1.473</a></span></span></li> <li tag="ref"><a name="CR15" /><span class="label" tagx="label" title="label">15.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Tan KH, Nishida R. Methyl eugenol: its occurrence, distribution, and role in nature, especially in relation to insect behavior and pollination. J Insect Sci. 2012;12.</span></li> <li tag="ref"><a name="CR16" /><span class="label" tagx="label" title="label">16.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bohannon</span><span class="given-names" tagx="given-names" title="given-names">MB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kleiman</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="mixed-article-title" title="mixed-article-title">Myristicin. The major volatile component in mature seed ofPortenschlagia ramosissima</span><span class="source" tagx="source" title="source">Lipids</span><span class="year" tagx="year" title="year">1977</span><span class="volume" tagx="volume" title="volume">12</span><span class="fpage" tagx="fpage" title="fpage">321</span><span class="lpage" tagx="lpage" title="lpage">323</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/BF02533357">10.1007/BF02533357</a></span></span></li> <li tag="ref"><a name="CR17" /><span class="label" tagx="label" title="label">17.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ferrer</span><span class="given-names" tagx="given-names" title="given-names">J-L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Austin</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stewart</span><span class="given-names" tagx="given-names" title="given-names">C</span><span class="suffix" tagx="suffix" title="suffix">Jr</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Noel</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">Structure and function of enzymes involved in the biosynthesis of phenylpropanoids</span><span class="source" tagx="source" title="source">Plant Physiol Biochem</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">46</span><span class="fpage" tagx="fpage" title="fpage">356</span><span class="lpage" tagx="lpage" title="lpage">370</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.plaphy.2007.12.009">10.1016/j.plaphy.2007.12.009</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18272377">18272377</a></span></span></li> <li tag="ref"><a name="CR18" /><span class="label" tagx="label" title="label">18.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pichersky</span><span class="given-names" tagx="given-names" title="given-names">E</span></span></span><span class="mixed-article-title" title="mixed-article-title">Characterization ofS-Adenosyl-l-methionine:(Iso) eugenolO-methyltransferase involved in floral scent production inClarkia breweri</span><span class="source" tagx="source" title="source">Arch Biochem Biophys</span><span class="year" tagx="year" title="year">1998</span><span class="volume" tagx="volume" title="volume">349</span><span class="fpage" tagx="fpage" title="fpage">153</span><span class="lpage" tagx="lpage" title="lpage">160</span><span class="pub-id"><a href="https://dx.doi.org/10.1006/abbi.1997.0452">10.1006/abbi.1997.0452</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9439593">9439593</a></span></span></li> <li tag="ref"><a name="CR19" /><span class="label" tagx="label" title="label">19.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">S-J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">M-R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bedgar</span><span class="given-names" tagx="given-names" title="given-names">DL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moinuddin</span><span class="given-names" tagx="given-names" title="given-names">SG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cardenas</span><span class="given-names" tagx="given-names" title="given-names">CL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Davin</span><span class="given-names" tagx="given-names" title="given-names">LB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kang</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lewis</span><span class="given-names" tagx="given-names" title="given-names">NG</span></span></span><span class="mixed-article-title" title="mixed-article-title">Functional reclassification of the putative cinnamyl alcohol dehydrogenase multigene family in Arabidopsis</span><span class="source" tagx="source" title="source">Proc Natl Acad Sci</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">101</span><span class="fpage" tagx="fpage" title="fpage">1455</span><span class="lpage" tagx="lpage" title="lpage">1460</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.0307987100">10.1073/pnas.0307987100</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14745009">14745009</a></span></span></li> <li tag="ref"><a name="CR20" /><span class="label" tagx="label" title="label">20.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Koeduka</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fridman</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gang</span><span class="given-names" tagx="given-names" title="given-names">DR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vassao</span><span class="given-names" tagx="given-names" title="given-names">DG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jackson</span><span class="given-names" tagx="given-names" title="given-names">BL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kish</span><span class="given-names" tagx="given-names" title="given-names">CM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Orlova</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Spassova</span><span class="given-names" tagx="given-names" title="given-names">SM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lewis</span><span class="given-names" tagx="given-names" title="given-names">NG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Noel</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span></span><span class="mixed-article-title" title="mixed-article-title">Eugenol and isoeugenol, characteristic aromatic constituents of spices, are biosynthesized via reduction of a coniferyl alcohol ester</span><span class="source" tagx="source" title="source">Proc Natl Acad Sci</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">103</span><span class="fpage" tagx="fpage" title="fpage">10128</span><span class="lpage" tagx="lpage" title="lpage">10133</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.0603732103">10.1073/pnas.0603732103</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16782809">16782809</a></span></span></li> <li tag="ref"><a name="CR21" /><span class="label" tagx="label" title="label">21.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gupta</span><span class="given-names" tagx="given-names" title="given-names">AK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schauvinhold</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pichersky</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schiestl</span><span class="given-names" tagx="given-names" title="given-names">FP</span></span></span><span class="mixed-article-title" title="mixed-article-title">Eugenol synthase genes in floral scent variation in Gymnadenia species</span><span class="source" tagx="source" title="source">Funct Integr Genomics</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">14</span><span class="fpage" tagx="fpage" title="fpage">779</span><span class="lpage" tagx="lpage" title="lpage">788</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s10142-014-0397-9">10.1007/s10142-014-0397-9</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25239559">25239559</a></span></span></li> <li tag="ref"><a name="CR22" /><span class="label" tagx="label" title="label">22.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Anand</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jayaramaiah</span><span class="given-names" tagx="given-names" title="given-names">RH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beedkar</span><span class="given-names" tagx="given-names" title="given-names">SD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Singh</span><span class="given-names" tagx="given-names" title="given-names">PA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Joshi</span><span class="given-names" tagx="given-names" title="given-names">RS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mulani</span><span class="given-names" tagx="given-names" title="given-names">FA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dholakia</span><span class="given-names" tagx="given-names" title="given-names">BB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Punekar</span><span class="given-names" tagx="given-names" title="given-names">SA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gade</span><span class="given-names" tagx="given-names" title="given-names">WN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thulasiram</span><span class="given-names" tagx="given-names" title="given-names">HV</span></span></span><span class="mixed-article-title" title="mixed-article-title">Comparative functional characterization of eugenol synthase from four different Ocimum species: Implications on eugenol accumulation</span><span class="source" tagx="source" title="source">Biochim Biophys Acta-Proteins Proteomic</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">1864</span><span class="fpage" tagx="fpage" title="fpage">1539</span><span class="lpage" tagx="lpage" title="lpage">1547</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.bbapap.2016.08.004">10.1016/j.bbapap.2016.08.004</a></span></span></li> <li tag="ref"><a name="CR23" /><span class="label" tagx="label" title="label">23.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Koeduka</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kajiyama</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Furuta</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suzuki</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tsuge</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Matsui</span><span class="given-names" tagx="given-names" title="given-names">K</span></span></span><span class="mixed-article-title" title="mixed-article-title">Characterization of an O-methyltransferase specific to guaiacol-type benzenoids from the flowers of loquat (Eriobotrya japonica)</span><span class="source" tagx="source" title="source">J Biosci Bioeng</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">122</span><span class="fpage" tagx="fpage" title="fpage">679</span><span class="lpage" tagx="lpage" title="lpage">684</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jbiosc.2016.06.012">10.1016/j.jbiosc.2016.06.012</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27473288">27473288</a></span></span></li> <li tag="ref"><a name="CR24" /><span class="label" tagx="label" title="label">24.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yan</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baudino</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Caissard</span><span class="given-names" tagx="given-names" title="given-names">J-C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">NicolÃ¨</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tang</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lu</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="mixed-article-title" title="mixed-article-title">Functional characterization of the eugenol synthase gene (RcEGS1) in rose</span><span class="source" tagx="source" title="source">Plant Physiol Biochem</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">129</span><span class="fpage" tagx="fpage" title="fpage">21</span><span class="lpage" tagx="lpage" title="lpage">26</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.plaphy.2018.05.015">10.1016/j.plaphy.2018.05.015</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29787935">29787935</a></span></span></li> <li tag="ref"><a name="CR25" /><span class="label" tagx="label" title="label">25.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Jinjie Liu , C.X., Honglei Zhang, Fawang Liu, Dongming Ma, Zhong Liu. Comparative transcriptomics analysis for gene mining and identification of a cinnamyl alcohol dehydrogenase involved in methyleugenol biosynthesis from <i>Asarum sieboldii</i> Miq. Molecules 2018, 23, doi:https://doi.org/10.3390/molecules23123184.</span></li> <li tag="ref"><a name="CR26" /><span class="label" tagx="label" title="label">26.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lavid</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shalit</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guterman</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bar</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beuerle</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Menda</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shafir</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zamir</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Adam</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span></span><span class="mixed-article-title" title="mixed-article-title">O-methyltransferases involved in the biosynthesis of volatile phenolic derivatives in rose petals</span><span class="source" tagx="source" title="source">Plant Physiol</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">129</span><span class="fpage" tagx="fpage" title="fpage">1899</span><span class="lpage" tagx="lpage" title="lpage">1907</span><span class="pub-id"><a href="https://dx.doi.org/10.1104/pp.005330">10.1104/pp.005330</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12177504">12177504</a></span></span></li> <li tag="ref"><a name="CR27" /><span class="label" tagx="label" title="label">27.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Scalliet</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Journot</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jullien</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baudino</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Magnard</span><span class="given-names" tagx="given-names" title="given-names">J-L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">ChanneliÃ¨re</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vergne</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dumas</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bendahmane</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cock</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span></span><span class="mixed-article-title" title="mixed-article-title">Biosynthesis of the major scent components 3, 5-dimethoxytoluene and 1, 3, 5-trimethoxybenzene by novel rose O-methyltransferases</span><span class="source" tagx="source" title="source">FEBS Lett</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">523</span><span class="fpage" tagx="fpage" title="fpage">113</span><span class="lpage" tagx="lpage" title="lpage">118</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0014-5793(02)02956-3">10.1016/S0014-5793(02)02956-3</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12123815">12123815</a></span></span></li> <li tag="ref"><a name="CR28" /><span class="label" tagx="label" title="label">28.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">ManhÃ£es</span><span class="given-names" tagx="given-names" title="given-names">AP</span></span></span><span class="mixed-article-title" title="mixed-article-title">Veiga-JÃºnior, V.F.d.; Wiedemann, L.S.M.; Fernandes, K.S.; Sampaio, P.d.T.B. biomass production and essential oil yield from leaves, fine stems and resprouts using pruning the crown of Aniba canelilla (HBK)(Lauraceae) in the Central Amazon</span><span class="source" tagx="source" title="source">Acta Amazon</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">42</span><span class="fpage" tagx="fpage" title="fpage">355</span><span class="lpage" tagx="lpage" title="lpage">362</span><span class="pub-id"><a href="https://dx.doi.org/10.1590/S0044-59672012000300007">10.1590/S0044-59672012000300007</a></span></span></li> <li tag="ref"><a name="CR29" /><span class="label" tagx="label" title="label">29.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Berry</span><span class="given-names" tagx="given-names" title="given-names">JO</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yerramsetty</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zielinski</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mure</span><span class="given-names" tagx="given-names" title="given-names">CM</span></span></span><span class="mixed-article-title" title="mixed-article-title">Photosynthetic gene expression in higher plants</span><span class="source" tagx="source" title="source">Photosynth Res</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">117</span><span class="fpage" tagx="fpage" title="fpage">91</span><span class="lpage" tagx="lpage" title="lpage">120</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s11120-013-9880-8">10.1007/s11120-013-9880-8</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23839301">23839301</a></span></span></li> <li tag="ref"><a name="CR30" /><span class="label" tagx="label" title="label">30.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Freitas Lima</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eloy</span><span class="given-names" tagx="given-names" title="given-names">NB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Siqueira</span><span class="given-names" tagx="given-names" title="given-names">JAB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">InzÃ©</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hemerly</span><span class="given-names" tagx="given-names" title="given-names">AS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ferreira</span><span class="given-names" tagx="given-names" title="given-names">PCG</span></span></span><span class="mixed-article-title" title="mixed-article-title">Molecular mechanisms of biomass increase in plants</span><span class="source" tagx="source" title="source">Biotechnol Res Innov</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">1</span><span class="fpage" tagx="fpage" title="fpage">14</span><span class="lpage" tagx="lpage" title="lpage">25</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.biori.2017.08.001">10.1016/j.biori.2017.08.001</a></span></span></li> <li tag="ref"><a name="CR31" /><span class="label" tagx="label" title="label">31.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zheng</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhong</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">L-A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">Transcriptome analysis and identification of genes related to terpenoid biosynthesis in Cinnamomum camphora</span><span class="source" tagx="source" title="source">BMC Genomics</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">19</span><span class="fpage" tagx="fpage" title="fpage">550</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s12864-018-4941-1">10.1186/s12864-018-4941-1</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30041601">30041601</a></span></span></li> <li tag="ref"><a name="CR32" /><span class="label" tagx="label" title="label">32.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Howyzeh</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Noori</span><span class="given-names" tagx="given-names" title="given-names">SAS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shariati</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Amiripour</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">Comparative transcriptome analysis to identify putative genes involved in thymol biosynthesis pathway in medicinal plant Trachyspermum ammi L</span><span class="source" tagx="source" title="source">Sci Rep</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">19</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41598-017-17765-5">10.1038/s41598-017-17765-5</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29311619">29311619</a></span></span></li> <li tag="ref"><a name="CR33" /><span class="label" tagx="label" title="label">33.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Fu S, Lei M, Zhang Y, Deng Z, Shi J, Hao D. De novo transcriptome analysis of Tibetan medicinal plant Dysphania schraderiana. Genet Mol Biol. 2019;42:480-7.</span></li> <li tag="ref"><a name="CR34" /><span class="label" tagx="label" title="label">34.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mathur</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jain</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jajoo</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">Photosynthetic efficiency in sun and shade plants</span><span class="source" tagx="source" title="source">Photosynthetica</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">56</span><span class="fpage" tagx="fpage" title="fpage">354</span><span class="lpage" tagx="lpage" title="lpage">365</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s11099-018-0767-y">10.1007/s11099-018-0767-y</a></span></span></li> <li tag="ref"><a name="CR35" /><span class="label" tagx="label" title="label">35.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Matthews</span><span class="given-names" tagx="given-names" title="given-names">JS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vialet-Chabrand</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lawson</span><span class="given-names" tagx="given-names" title="given-names">T</span></span></span><span class="mixed-article-title" title="mixed-article-title">Acclimation to fluctuating light impacts the rapidity of response and diurnal rhythm of stomatal conductance</span><span class="source" tagx="source" title="source">Plant Physiol</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">176</span><span class="fpage" tagx="fpage" title="fpage">1939</span><span class="lpage" tagx="lpage" title="lpage">1951</span><span class="pub-id"><a href="https://dx.doi.org/10.1104/pp.17.01809">10.1104/pp.17.01809</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29371250">29371250</a></span></span></li> <li tag="ref"><a name="CR36" /><span class="label" tagx="label" title="label">36.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fengler</span><span class="given-names" tagx="given-names" title="given-names">KA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Van Hemert</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gupta</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mongar</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Allen</span><span class="given-names" tagx="given-names" title="given-names">WB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weers</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mo</span><span class="given-names" tagx="given-names" title="given-names">H</span></span></span><span class="mixed-article-title" title="mixed-article-title">Identification and characterization of a novel stay-green QTL that increases yield in maize</span><span class="source" tagx="source" title="source">Plant Biotechnol J</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">17</span><span class="fpage" tagx="fpage" title="fpage">2272</span><span class="lpage" tagx="lpage" title="lpage">2285</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/pbi.13139">10.1111/pbi.13139</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31033139">31033139</a></span></span></li> <li tag="ref"><a name="CR37" /><span class="label" tagx="label" title="label">37.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peng</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhen</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">He</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">X</span></span></span><span class="mixed-article-title" title="mixed-article-title">Orthogonal test design for optimization of the extraction of essential oil from Asarum heterotropoides var. Mandshuricum and evaluation of its antibacterial activity against periodontal pathogens. <i>3</i></span><span class="source" tagx="source" title="source">Biotech</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">473</span></span></li> <li tag="ref"><a name="CR38" /><span class="label" tagx="label" title="label">38.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xiao</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ma</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tian</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hou</span><span class="given-names" tagx="given-names" title="given-names">W</span></span></span><span class="mixed-article-title" title="mixed-article-title">Effects of light irradiation on essential oil biosynthesis in the medicinal plant Asarum heterotropoides Fr. Schmidt var. mandshuricum (maxim) Kitag</span><span class="source" tagx="source" title="source">PLoS One</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">15</span><span class="fpage" tagx="fpage" title="fpage">e0237952</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0237952">10.1371/journal.pone.0237952</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32970685">32970685</a></span></span></li> <li tag="ref"><a name="CR39" /><span class="label" tagx="label" title="label">39.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shirley</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lahnstein</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fincher</span><span class="given-names" tagx="given-names" title="given-names">GB</span></span></span><span class="mixed-article-title" title="mixed-article-title">Characterization and expression patterns of UDP-D-glucuronate decarboxylase genes in barley</span><span class="source" tagx="source" title="source">Plant Physiol</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">138</span><span class="fpage" tagx="fpage" title="fpage">131</span><span class="lpage" tagx="lpage" title="lpage">141</span><span class="pub-id"><a href="https://dx.doi.org/10.1104/pp.104.057869">10.1104/pp.104.057869</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15849307">15849307</a></span></span></li> <li tag="ref"><a name="CR40" /><span class="label" tagx="label" title="label">40.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Irving</span><span class="given-names" tagx="given-names" title="given-names">LJ</span></span></span><span class="mixed-article-title" title="mixed-article-title">Carbon assimilation, biomass partitioning and productivity in grasses</span><span class="source" tagx="source" title="source">Agriculture</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">5</span><span class="fpage" tagx="fpage" title="fpage">1116</span><span class="lpage" tagx="lpage" title="lpage">1134</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/agriculture5041116">10.3390/agriculture5041116</a></span></span></li> <li tag="ref"><a name="CR41" /><span class="label" tagx="label" title="label">41.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Feng</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Raza</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khalid</span><span class="given-names" tagx="given-names" title="given-names">MHB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Du</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Song</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">L</span></span></span><span class="mixed-article-title" title="mixed-article-title">The influence of light intensity and leaf movement on photosynthesis characteristics and carbon balance of soybean</span><span class="source" tagx="source" title="source">Front Plant Sci</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">1952</span><span class="pub-id"><a href="https://dx.doi.org/10.3389/fpls.2018.01952">10.3389/fpls.2018.01952</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30687355">30687355</a></span></span></li> <li tag="ref"><a name="CR42" /><span class="label" tagx="label" title="label">42.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Anur</span><span class="given-names" tagx="given-names" title="given-names">RM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mufithah</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sawitri</span><span class="given-names" tagx="given-names" title="given-names">WD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sakakibara</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sugiharto</span><span class="given-names" tagx="given-names" title="given-names">B</span></span></span><span class="mixed-article-title" title="mixed-article-title">Overexpression of sucrose phosphate synthase enhanced sucrose content and biomass production in transgenic sugarcane</span><span class="source" tagx="source" title="source">Plants</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">200</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/plants9020200">10.3390/plants9020200</a></span></span></li> <li tag="ref"><a name="CR43" /><span class="label" tagx="label" title="label">43.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Balazadeh</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="mixed-article-title" title="mixed-article-title">Stay-green not always stays green</span><span class="source" tagx="source" title="source">Mol Plant</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">1264</span><span class="lpage" tagx="lpage" title="lpage">1266</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/mp/ssu076">10.1093/mp/ssu076</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24996917">24996917</a></span></span></li> <li tag="ref"><a name="CR44" /><span class="label" tagx="label" title="label">44.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">He</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gou</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Powell</span><span class="given-names" tagx="given-names" title="given-names">RA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yuan</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guo</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">Receptor-like protein kinases, BAK1 and BKK1, regulate a light-dependent cell-death control pathway</span><span class="source" tagx="source" title="source">Plant Signal Behav</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">3</span><span class="fpage" tagx="fpage" title="fpage">813</span><span class="lpage" tagx="lpage" title="lpage">815</span><span class="pub-id"><a href="https://dx.doi.org/10.4161/psb.3.10.5890">10.4161/psb.3.10.5890</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19704566">19704566</a></span></span></li> <li tag="ref"><a name="CR45" /><span class="label" tagx="label" title="label">45.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wayne</span><span class="given-names" tagx="given-names" title="given-names">LL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wallis</span><span class="given-names" tagx="given-names" title="given-names">JG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kumar</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Markham</span><span class="given-names" tagx="given-names" title="given-names">JE</span></span></span><span class="mixed-article-title" title="mixed-article-title">Cytochrome b5 reductase encoded by CBR1 is essential for a functional male gametophyte in Arabidopsis</span><span class="source" tagx="source" title="source">Plant Cell</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">25</span><span class="fpage" tagx="fpage" title="fpage">3052</span><span class="lpage" tagx="lpage" title="lpage">3066</span><span class="pub-id"><a href="https://dx.doi.org/10.1105/tpc.113.113324">10.1105/tpc.113.113324</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23995085">23995085</a></span></span></li> <li tag="ref"><a name="CR46" /><span class="label" tagx="label" title="label">46.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sinclair</span><span class="given-names" tagx="given-names" title="given-names">SA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Larue</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bonk</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khan</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Castillo-Michel</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stein</span><span class="given-names" tagx="given-names" title="given-names">RJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grolimund</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Begerow</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Neumann</span><span class="given-names" tagx="given-names" title="given-names">U</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haydon</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span></span><span class="mixed-article-title" title="mixed-article-title">Etiolated seedling development requires repression of photomorphogenesis by a small cell-wall-derived dark signal</span><span class="source" tagx="source" title="source">Curr Biol</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">27</span><span class="fpage" tagx="fpage" title="fpage">3403</span><span class="lpage" tagx="lpage" title="lpage">3418</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.cub.2017.09.063">10.1016/j.cub.2017.09.063</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29103938">29103938</a></span></span></li> <li tag="ref"><a name="CR47" /><span class="label" tagx="label" title="label">47.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ruberti</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sessa</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ciolfi</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Possenti</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carabelli</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morelli</span><span class="given-names" tagx="given-names" title="given-names">G</span></span></span><span class="mixed-article-title" title="mixed-article-title">Plant adaptation to dynamically changing environment: the shade avoidance response</span><span class="source" tagx="source" title="source">Biotechnol Adv</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">30</span><span class="fpage" tagx="fpage" title="fpage">1047</span><span class="lpage" tagx="lpage" title="lpage">1058</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.biotechadv.2011.08.014">10.1016/j.biotechadv.2011.08.014</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21888962">21888962</a></span></span></li> <li tag="ref"><a name="CR48" /><span class="label" tagx="label" title="label">48.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Falcioni</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moriwaki</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Oliveira</span><span class="given-names" tagx="given-names" title="given-names">DM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Andreotti</span><span class="given-names" tagx="given-names" title="given-names">GC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Souza</span><span class="given-names" tagx="given-names" title="given-names">LA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">dos Santos</span><span class="given-names" tagx="given-names" title="given-names">WD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bonato</span><span class="given-names" tagx="given-names" title="given-names">CM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Antunes</span><span class="given-names" tagx="given-names" title="given-names">WC</span></span></span><span class="mixed-article-title" title="mixed-article-title">Increased gibberellins and light levels promotes cell wall thickness and enhance lignin deposition in xylem fibers</span><span class="source" tagx="source" title="source">Front Plant Sci</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">1391</span><span class="pub-id"><a href="https://dx.doi.org/10.3389/fpls.2018.01391">10.3389/fpls.2018.01391</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30294339">30294339</a></span></span></li> <li tag="ref"><a name="CR49" /><span class="label" tagx="label" title="label">49.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dixon</span><span class="given-names" tagx="given-names" title="given-names">RA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guo</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parvathi</span><span class="given-names" tagx="given-names" title="given-names">K</span></span></span><span class="mixed-article-title" title="mixed-article-title">The biosynthesis of monolignols: a â€œmetabolic gridâ€�, or independent pathways to guaiacyl and syringyl units?</span><span class="source" tagx="source" title="source">Phytochemistry</span><span class="year" tagx="year" title="year">2001</span><span class="volume" tagx="volume" title="volume">57</span><span class="fpage" tagx="fpage" title="fpage">1069</span><span class="lpage" tagx="lpage" title="lpage">1084</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0031-9422(01)00092-9">10.1016/S0031-9422(01)00092-9</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11430980">11430980</a></span></span></li> <li tag="ref"><a name="CR50" /><span class="label" tagx="label" title="label">50.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Taura</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Iijima</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yamanaka</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Takahashi</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kenmoku</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saeki</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morimoto</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Asakawa</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kurosaki</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morita</span><span class="given-names" tagx="given-names" title="given-names">H</span></span></span><span class="mixed-article-title" title="mixed-article-title">A novel class of plant type III polyketide synthase involved in orsellinic acid biosynthesis from Rhododendron dauricum</span><span class="source" tagx="source" title="source">Front Plant Sci</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">1452</span><span class="pub-id"><a href="https://dx.doi.org/10.3389/fpls.2016.01452">10.3389/fpls.2016.01452</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27729920">27729920</a></span></span></li> <li tag="ref"><a name="CR51" /><span class="label" tagx="label" title="label">51.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schmidlin</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Poutaraud</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Claudel</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mestre</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prado</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Santos-Rosa</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wiedemann-Merdinoglu</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Karst</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Merdinoglu</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hugueney</span><span class="given-names" tagx="given-names" title="given-names">P</span></span></span><span class="mixed-article-title" title="mixed-article-title">A stress-inducible resveratrol O-methyltransferase involved in the biosynthesis of pterostilbene in grapevine</span><span class="source" tagx="source" title="source">Plant Physiol</span><span class="year" tagx="year" title="year">1630</span><span class="volume" tagx="volume" title="volume">2008</span><span class="fpage" tagx="fpage" title="fpage">148</span></span></li> <li tag="ref"><a name="CR52" /><span class="label" tagx="label" title="label">52.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yuan</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cao</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yazdani</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tadmor</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">L</span></span></span><span class="mixed-article-title" title="mixed-article-title">Carotenoid metabolism in plants: the role of plastids</span><span class="source" tagx="source" title="source">Mol Plant</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">11</span><span class="fpage" tagx="fpage" title="fpage">58</span><span class="lpage" tagx="lpage" title="lpage">74</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.molp.2017.09.010">10.1016/j.molp.2017.09.010</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28958604">28958604</a></span></span></li> <li tag="ref"><a name="CR53" /><span class="label" tagx="label" title="label">53.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lado</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">AlÃ³s</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Manzi</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cronje</span><span class="given-names" tagx="given-names" title="given-names">PJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">GÃ³mez-Cadenas</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rodrigo</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">ZacarÃ­as</span><span class="given-names" tagx="given-names" title="given-names">L</span></span></span><span class="mixed-article-title" title="mixed-article-title">Light regulation of carotenoid biosynthesis in the peel of mandarin and sweet orange fruits</span><span class="source" tagx="source" title="source">Front Plant Sci</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">1288</span><span class="pub-id"><a href="https://dx.doi.org/10.3389/fpls.2019.01288">10.3389/fpls.2019.01288</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31681382">31681382</a></span></span></li> <li tag="ref"><a name="CR54" /><span class="label" tagx="label" title="label">54.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Galpaz</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ronen</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khalfa</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zamir</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hirschberg</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">A chromoplast-specific carotenoid biosynthesis pathway is revealed by cloning of the tomato white-flower locus</span><span class="source" tagx="source" title="source">Plant Cell</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">18</span><span class="fpage" tagx="fpage" title="fpage">1947</span><span class="lpage" tagx="lpage" title="lpage">1960</span><span class="pub-id"><a href="https://dx.doi.org/10.1105/tpc.105.039966">10.1105/tpc.105.039966</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16816137">16816137</a></span></span></li> <li tag="ref"><a name="CR55" /><span class="label" tagx="label" title="label">55.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dall'Osto</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lico</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alric</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Giuliano</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Havaux</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bassi</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="mixed-article-title" title="mixed-article-title">Lutein is needed for efficient chlorophyll triplet quenching in the major LHCII antenna complex of higher plants and effective photoprotection in vivo under strong light</span><span class="source" tagx="source" title="source">BMC Plant Biol</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">6</span><span class="fpage" tagx="fpage" title="fpage">32</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1471-2229-6-32">10.1186/1471-2229-6-32</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17192177">17192177</a></span></span></li> <li tag="ref"><a name="CR56" /><span class="label" tagx="label" title="label">56.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Plumb</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Townsend</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rasool</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alomrani</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Razak</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Karpinska</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ruban</span><span class="given-names" tagx="given-names" title="given-names">AV</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Foyer</span><span class="given-names" tagx="given-names" title="given-names">CH</span></span></span><span class="mixed-article-title" title="mixed-article-title">Ascorbate-mediated regulation of growth, photoprotection, and photoinhibition in Arabidopsis thaliana</span><span class="source" tagx="source" title="source">J Exp Bot</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">69</span><span class="fpage" tagx="fpage" title="fpage">2823</span><span class="lpage" tagx="lpage" title="lpage">2835</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/jxb/ery170">10.1093/jxb/ery170</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29726917">29726917</a></span></span></li> <li tag="ref"><a name="CR57" /><span class="label" tagx="label" title="label">57.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leegood</span><span class="given-names" tagx="given-names" title="given-names">RC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Walker</span><span class="given-names" tagx="given-names" title="given-names">RP</span></span></span><span class="mixed-article-title" title="mixed-article-title">Regulation and roles of phosphoenolpyruvate carboxykinase in plants</span><span class="source" tagx="source" title="source">Arch Biochem Biophys</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">414</span><span class="fpage" tagx="fpage" title="fpage">204</span><span class="lpage" tagx="lpage" title="lpage">210</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0003-9861(03)00093-6">10.1016/S0003-9861(03)00093-6</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12781772">12781772</a></span></span></li> <li tag="ref"><a name="CR58" /><span class="label" tagx="label" title="label">58.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Halliday</span><span class="given-names" tagx="given-names" title="given-names">KJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">MartÃ­nez-GarcÃ­a</span><span class="given-names" tagx="given-names" title="given-names">JF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Josse</span><span class="given-names" tagx="given-names" title="given-names">E-M</span></span></span><span class="mixed-article-title" title="mixed-article-title">Integration of light and auxin signaling</span><span class="source" tagx="source" title="source">Cold Spring Harb Perspect Biol</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">1</span><span class="fpage" tagx="fpage" title="fpage">a001586</span><span class="pub-id"><a href="https://dx.doi.org/10.1101/cshperspect.a001586">10.1101/cshperspect.a001586</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20457562">20457562</a></span></span></li> <li tag="ref"><a name="CR59" /><span class="label" tagx="label" title="label">59.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moore</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vogel</span><span class="given-names" tagx="given-names" title="given-names">MO</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dietz</span><span class="given-names" tagx="given-names" title="given-names">K-J</span></span></span><span class="mixed-article-title" title="mixed-article-title">The acclimation response to high light is initiated within seconds as indicated by upregulation of AP2/ERF transcription factor network in Arabidopsis thaliana</span><span class="source" tagx="source" title="source">Plant Signal Behav</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">976479</span><span class="pub-id"><a href="https://dx.doi.org/10.4161/15592324.2014.976479">10.4161/15592324.2014.976479</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25482793">25482793</a></span></span></li> <li tag="ref"><a name="CR60" /><span class="label" tagx="label" title="label">60.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hutchison</span><span class="given-names" tagx="given-names" title="given-names">CE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Argueso</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gonzalez</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lewis</span><span class="given-names" tagx="given-names" title="given-names">MW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maxwell</span><span class="given-names" tagx="given-names" title="given-names">BB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perdue</span><span class="given-names" tagx="given-names" title="given-names">TD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schaller</span><span class="given-names" tagx="given-names" title="given-names">GE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alonso</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span></span><span class="mixed-article-title" title="mixed-article-title">The Arabidopsis histidine phosphotransfer proteins are redundant positive regulators of cytokinin signaling</span><span class="source" tagx="source" title="source">Plant Cell</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">18</span><span class="fpage" tagx="fpage" title="fpage">3073</span><span class="lpage" tagx="lpage" title="lpage">3087</span><span class="pub-id"><a href="https://dx.doi.org/10.1105/tpc.106.045674">10.1105/tpc.106.045674</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17122069">17122069</a></span></span></li> <li tag="ref"><a name="CR61" /><span class="label" tagx="label" title="label">61.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">EklÃ¶f</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brumer</span><span class="given-names" tagx="given-names" title="given-names">H</span></span></span><span class="mixed-article-title" title="mixed-article-title">The XTH gene family: an update on enzyme structure, function, and phylogeny in xyloglucan remodeling</span><span class="source" tagx="source" title="source">Plant Physiol</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">153</span><span class="fpage" tagx="fpage" title="fpage">456</span><span class="lpage" tagx="lpage" title="lpage">466</span><span class="pub-id"><a href="https://dx.doi.org/10.1104/pp.110.156844">10.1104/pp.110.156844</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20421457">20421457</a></span></span></li> <li tag="ref"><a name="CR62" /><span class="label" tagx="label" title="label">62.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Campbell</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Braam</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">In vitro activities of four xyloglucan endotransglycosylases from Arabidopsis</span><span class="source" tagx="source" title="source">Plant J</span><span class="year" tagx="year" title="year">1999</span><span class="volume" tagx="volume" title="volume">18</span><span class="fpage" tagx="fpage" title="fpage">371</span><span class="lpage" tagx="lpage" title="lpage">382</span><span class="pub-id"><a href="https://dx.doi.org/10.1046/j.1365-313X.1999.00459.x">10.1046/j.1365-313X.1999.00459.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10406121">10406121</a></span></span></li> <li tag="ref"><a name="CR63" /><span class="label" tagx="label" title="label">63.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nawaz</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rehman</span><span class="given-names" tagx="given-names" title="given-names">HM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Imtiaz</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baloch</span><span class="given-names" tagx="given-names" title="given-names">FS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">JD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">SH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">SI</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chung</span><span class="given-names" tagx="given-names" title="given-names">G</span></span></span><span class="mixed-article-title" title="mixed-article-title">Systems identification and characterization of cell wall reassembly and degradation related genes in <i>Glycine max</i> (L.) Merill, a bioenergy legume</span><span class="source" tagx="source" title="source">Sci Rep</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">10862</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41598-017-11495-4">10.1038/s41598-017-11495-4</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28883533">28883533</a></span></span></li> <li tag="ref"><a name="CR64" /><span class="label" tagx="label" title="label">64.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Procko</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Burko</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jaillais</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ljung</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Long</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chory</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">The epidermis coordinates auxin-induced stem growth in response to shade</span><span class="source" tagx="source" title="source">Genes Dev</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">30</span><span class="fpage" tagx="fpage" title="fpage">13</span><span class="pub-id"><a href="https://dx.doi.org/10.1101/gad.283234.116">10.1101/gad.283234.116</a></span></span></li> <li tag="ref"><a name="CR65" /><span class="label" tagx="label" title="label">65.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">L</span></span></span><span class="mixed-article-title" title="mixed-article-title">Hormonal regulation in shade avoidance</span><span class="source" tagx="source" title="source">Front Plant Sci</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">1527</span><span class="pub-id"><a href="https://dx.doi.org/10.3389/fpls.2017.01527">10.3389/fpls.2017.01527</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28928761">28928761</a></span></span></li> <li tag="ref"><a name="CR66" /><span class="label" tagx="label" title="label">66.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Shao Q, Wang H, Guo H, Zhou A, Huang Y, Sun Y, Li M. Effects of shade treatments on photosynthetic characteristics, chloroplast ultrastructure, and physiology of Anoectochilus roxburghii. PLoS One. 2014;9.</span></li> <li tag="ref"><a name="CR67" /><span class="label" tagx="label" title="label">67.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">LÃ³pez-MarÃ­n J, GÃ¡lvez A, GonzÃ¡lez A, Egea-Gilabert C, Fernandez J. Effect of shade on yield, quality and photosynthesis-related parameters of sweet pepper plants. International Society for Horticultural Science. 2014;956:545â€"52.</span></li> <li tag="ref"><a name="CR68" /><span class="label" tagx="label" title="label">68.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Van der Auwera</span><span class="given-names" tagx="given-names" title="given-names">GA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carneiro</span><span class="given-names" tagx="given-names" title="given-names">MO</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hartl</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Poplin</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Del Angel</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Levy-Moonshine</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jordan</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shakir</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roazen</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thibault</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">From FastQ data to high-confidence variant calls: the genome analysis toolkit best practices pipeline</span><span class="source" tagx="source" title="source">Curr Protoc Bioinformatics</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">43</span><span class="fpage" tagx="fpage" title="fpage">11</span><span class="lpage" tagx="lpage" title="lpage">10</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/0471250953.bi1110s43">10.1002/0471250953.bi1110s43</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25431634">25431634</a></span></span></li> <li tag="ref"><a name="CR69" /><span class="label" tagx="label" title="label">69.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Grabherr, M.G.; Haas, B.J.; Yassour, M.; Levin, J.Z.; Thompson, D.A.; Amit, I.; Adiconis, X.; Fan, L.; Raychowdhury, R.; Zeng, Q., et al. Full-length transcriptome assembly from RNA-Seq data without a reference genome. Nat Biotech 2011, 29, 644, doi:https://doi.org/10.1038/nbt.1883.</span></li> <li tag="ref"><a name="CR70" /><span class="label" tagx="label" title="label">70.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kanehisa</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goto</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kawashima</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Okuno</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hattori</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">The KEGG resource for deciphering the genome</span><span class="source" tagx="source" title="source">Nucleic Acids Res</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">32</span><span class="fpage" tagx="fpage" title="fpage">D277</span><span class="lpage" tagx="lpage" title="lpage">D280</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gkh063">10.1093/nar/gkh063</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14681412">14681412</a></span></span></li> <li tag="ref"><a name="CR71" /><span class="label" tagx="label" title="label">71.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Ashburner, M.; Ball, C.A.; Blake, J.A.; Botstein, D.; Butler, H.; Cherry, J.M.; Davis, A.P.; Dolinski, K.; Dwight, S.S.; Eppig, J.T., et al. Gene ontology: tool for the unification of biology. Nat Genet 2000, 25, 25, doi:https://doi.org/10.1038/75556.</span></li> <li tag="ref"><a name="CR72" /><span class="label" tagx="label" title="label">72.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tatusov</span><span class="given-names" tagx="given-names" title="given-names">RL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Galperin</span><span class="given-names" tagx="given-names" title="given-names">MY</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Natale</span><span class="given-names" tagx="given-names" title="given-names">DA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Koonin</span><span class="given-names" tagx="given-names" title="given-names">EV</span></span></span><span class="mixed-article-title" title="mixed-article-title">The COG database: a tool for fenome-scale analysis of protein functions and evolution</span><span class="source" tagx="source" title="source">Nucleic Acids Res</span><span class="year" tagx="year" title="year">2000</span><span class="volume" tagx="volume" title="volume">28</span><span class="fpage" tagx="fpage" title="fpage">33</span><span class="lpage" tagx="lpage" title="lpage">36</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/28.1.33">10.1093/nar/28.1.33</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10592175">10592175</a></span></span></li> <li tag="ref"><a name="CR73" /><span class="label" tagx="label" title="label">73.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Apweiler</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bairoch</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">CH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barker</span><span class="given-names" tagx="given-names" title="given-names">WC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boeckmann</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ferro</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gasteiger</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lopez</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Magrane</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">UniProt: the universal protein knowledgebase</span><span class="source" tagx="source" title="source">Nucleic Acids Res</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">32</span><span class="fpage" tagx="fpage" title="fpage">D115</span><span class="lpage" tagx="lpage" title="lpage">D119</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gkw1099">10.1093/nar/gkw1099</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14681372">14681372</a></span></span></li> <li tag="ref"><a name="CR74" /><span class="label" tagx="label" title="label">74.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Deng</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">He</span><span class="given-names" tagx="given-names" title="given-names">F</span></span></span><span class="mixed-article-title" title="mixed-article-title">Integrated nr database in protein annotation system and its localization</span><span class="source" tagx="source" title="source">Comput Eng</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">32</span><span class="fpage" tagx="fpage" title="fpage">71</span><span class="lpage" tagx="lpage" title="lpage">72</span></span></li> <li tag="ref"><a name="CR75" /><span class="label" tagx="label" title="label">75.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Koonin</span><span class="given-names" tagx="given-names" title="given-names">EV</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fedorova</span><span class="given-names" tagx="given-names" title="given-names">ND</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jackson</span><span class="given-names" tagx="given-names" title="given-names">JD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jacobs</span><span class="given-names" tagx="given-names" title="given-names">AR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krylov</span><span class="given-names" tagx="given-names" title="given-names">DM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Makarova</span><span class="given-names" tagx="given-names" title="given-names">KS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mazumder</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mekhedov</span><span class="given-names" tagx="given-names" title="given-names">SL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nikolskaya</span><span class="given-names" tagx="given-names" title="given-names">AN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rao</span><span class="given-names" tagx="given-names" title="given-names">BS</span></span></span><span class="mixed-article-title" title="mixed-article-title">A comprehensive evolutionary classification of proteins encoded in complete eukaryotic genomes</span><span class="source" tagx="source" title="source">Genome Biol</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">5</span><span class="fpage" tagx="fpage" title="fpage">R7</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/gb-2004-5-2-r7">10.1186/gb-2004-5-2-r7</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14759257">14759257</a></span></span></li> <li tag="ref"><a name="CR76" /><span class="label" tagx="label" title="label">76.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Finn</span><span class="given-names" tagx="given-names" title="given-names">RD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bateman</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Clements</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Coggill</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eberhardt</span><span class="given-names" tagx="given-names" title="given-names">RY</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eddy</span><span class="given-names" tagx="given-names" title="given-names">SR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heger</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hetherington</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holm</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mistry</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">Pfam: the protein families database</span><span class="source" tagx="source" title="source">Nucleic Acids Res</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">42</span><span class="fpage" tagx="fpage" title="fpage">D222</span><span class="lpage" tagx="lpage" title="lpage">D230</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gkt1223">10.1093/nar/gkt1223</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24288371">24288371</a></span></span></li> <li tag="ref"><a name="CR77" /><span class="label" tagx="label" title="label">77.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Eddy, S.R. Profile hidden markov models. Bioinformatics 1998, 14, 755â€"763, doi:https://doi.org/10.1093/bioinformatics/14.9.755.</span></li> <li tag="ref"><a name="CR78" /><span class="label" tagx="label" title="label">78.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Langmead, B.; Trapnell, C.; Pop, M.; Salzberg, S.L. Ultrafast and memory-efficient alignment of short DNA sequences to the human genome. Genome Biol 2009, 10, R25, doi:https://doi.org/10.1186/gb-2009-10-3-r25.</span></li> <li tag="ref"><a name="CR79" /><span class="label" tagx="label" title="label">79.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dewey</span><span class="given-names" tagx="given-names" title="given-names">CN</span></span></span><span class="mixed-article-title" title="mixed-article-title">RSEM: accurate transcript quantification from RNA-Seq data with or without a reference genome</span><span class="source" tagx="source" title="source">BMC Bioinformatics</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">12</span><span class="fpage" tagx="fpage" title="fpage">323</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1471-2105-12-323">10.1186/1471-2105-12-323</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21816040">21816040</a></span></span></li> <li tag="ref"><a name="CR80" /><span class="label" tagx="label" title="label">80.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Feng</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">X</span></span></span><span class="mixed-article-title" title="mixed-article-title">DEGseq: an R package for identifying differentially expressed genes from RNA-seq data</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">26</span><span class="fpage" tagx="fpage" title="fpage">136</span><span class="lpage" tagx="lpage" title="lpage">138</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btp612">10.1093/bioinformatics/btp612</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19855105">19855105</a></span></span></li> <li tag="ref"><a name="CR81" /><span class="label" tagx="label" title="label">81.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Anders, S.; McCarthy, D.J.; Chen, Y.; Okoniewski, M.; Smyth, G.K.; Huber, W.; Robinson, M.D. Count-based differential expression analysis of RNA sequencing data using R and bioconductor. Nat Protocol 2013, 8, 1765, doi:https://doi.org/10.1038/nprot.2013.099.</span></li> <li tag="ref"><a name="CR82" /><span class="label" tagx="label" title="label">82.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Young</span><span class="given-names" tagx="given-names" title="given-names">MD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wakefield</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smyth</span><span class="given-names" tagx="given-names" title="given-names">GK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oshlack</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">Gene ontology analysis for RNA-seq: accounting for selection bias</span><span class="source" tagx="source" title="source">Genome Biol</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">11</span><span class="fpage" tagx="fpage" title="fpage">R14</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/gb-2010-11-2-r14">10.1186/gb-2010-11-2-r14</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20132535">20132535</a></span></span></li> <li tag="ref"><a name="CR83" /><span class="label" tagx="label" title="label">83.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xie</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mao</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ding</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dong</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kong</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gao</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">C-Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wei</span><span class="given-names" tagx="given-names" title="given-names">L</span></span></span><span class="mixed-article-title" title="mixed-article-title">KOBAS 2.0: a web server for annotation and identification of enriched pathways and diseases</span><span class="source" tagx="source" title="source">Nucleic Acids Res</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">39</span><span class="fpage" tagx="fpage" title="fpage">W316</span><span class="lpage" tagx="lpage" title="lpage">W322</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gkr483">10.1093/nar/gkr483</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21715386">21715386</a></span></span></li> <li tag="ref"><a name="CR84" /><span class="label" tagx="label" title="label">84.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">PÃ©rez-RodrÃ­guez</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Riano-Pachon</span><span class="given-names" tagx="given-names" title="given-names">DM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">CorrÃªa</span><span class="given-names" tagx="given-names" title="given-names">LGG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rensing</span><span class="given-names" tagx="given-names" title="given-names">SA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kersten</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mueller-Roeber</span><span class="given-names" tagx="given-names" title="given-names">B</span></span></span><span class="mixed-article-title" title="mixed-article-title">PlnTFDB: updated content and new features of the plant transcription factor database</span><span class="source" tagx="source" title="source">Nucleic Acids Res</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">38</span><span class="fpage" tagx="fpage" title="fpage">D822</span><span class="lpage" tagx="lpage" title="lpage">D827</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gkp805">10.1093/nar/gkp805</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19858103">19858103</a></span></span></li> <li tag="ref"><a name="CR85" /><span class="label" tagx="label" title="label">85.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jin</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kong</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gao</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luo</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">PlantTFDB 3.0: a portal for the functional and evolutionary study of plant transcription factors</span><span class="source" tagx="source" title="source">Nucleic Acids Res</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">42</span><span class="fpage" tagx="fpage" title="fpage">D1182</span><span class="lpage" tagx="lpage" title="lpage">D1187</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gkt1016">10.1093/nar/gkt1016</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24174544">24174544</a></span></span></li> <li tag="ref"><a name="CR86" /><span class="label" tagx="label" title="label">86.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Silvia Sebastiani M, Bagnaresi P, Sestili S, Biselli C, Zechini A, OrrÃ¹ L, Cattivelli L, Ficcadenti N. Transcriptome analysis of the melon-Fusarium oxysporum f. sp. melonis race 1.2 pathosystem in susceptible and resistant plants. Front Plant Sci. 2017;(8):362.</span></li> </ul> </div> </div>  </body></html>