<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">PLoS One</span><span class="iso-abbrev" title="iso-abbrev">PLoS One</span><span class="publisher-id" title="publisher-id">plos</span><span class="pmc" title="pmc">plosone</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">PLoS ONE</span></div><span class="issn-epub" tagx="issn" title="issn-epub">1932-6203</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Public Library of Science</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">San Francisco, CA USA</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7703983">7703983</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1371/journal.pone.0242776">10.1371/journal.pone.0242776</a></span><span class="publisher-id" title="publisher-id">PONE-D-19-24237</span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Physical Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Chemistry</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Chemical Compounds</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Organic Compounds</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Carbohydrates</span>: <span class="subject" title="subject">Starches</span></div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Physical Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Chemistry</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Organic Chemistry</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Organic Compounds</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Carbohydrates</span>: <span class="subject" title="subject">Starches</span></div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Agriculture</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Agronomy</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Plant Products</span>: <span class="subject" title="subject">Essential Oils</span></div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Agriculture</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Crop Science</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Plant Products</span>: <span class="subject" title="subject">Essential Oils</span></div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biochemistry</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Metabolism</span>: <span class="subject" title="subject">Protein Metabolism</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biochemistry</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Glycobiology</span>: <span class="subject" title="subject">Polysaccharides</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genetics</span>: <span class="subject" title="subject">Gene Expression</span></div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biochemistry</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Metabolism</span>: <span class="subject" title="subject">Metabolic Pathways</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biochemistry</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Metabolism</span>: <span class="subject" title="subject">Metabolic Processes</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and life sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Molecular biology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Molecular biology techniques</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Sequencing techniques</span>: <span class="subject" title="subject">RNA sequencing</span></div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Research and analysis methods</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Molecular biology techniques</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Sequencing techniques</span>: <span class="subject" title="subject">RNA sequencing</span></div> </div> </div> </div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Transcriptome analysis of <i>Curcuma wenyujin</i> from Haikou and Wenzhou, and a comparison of the main constituents and related genes of Rhizoma Curcumae</div> <div class="alt-title" title="alt-title">
A comparison of the main substances and related genes of Rhizoma Curcumae</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0003-4317-0029</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lu</span><span class="given-names" tagx="given-names" title="given-names">Lilan</span></span><span class="role" tagx="role" title="role">Conceptualization</span><span class="role" tagx="role" title="role">Data curation</span><span class="role" tagx="role" title="role">Writing â€" original draft</span><a href="#aff001"><sup>1</sup></a><a href="#aff002"><sup>2</sup></a><a href="#cor001">*</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Peiwei</span></span><span class="role" tagx="role" title="role">Data curation</span><span class="role" tagx="role" title="role">Validation</span><a href="#aff001"><sup>1</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">Yanfang</span></span><span class="role" tagx="role" title="role">Writing â€" review &amp;amp; editing</span><a href="#aff003"><sup>3</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Yuxiu</span></span><span class="role" tagx="role" title="role">Supervision</span><a href="#aff001"><sup>1</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Caixia</span></span><span class="role" tagx="role" title="role">Supervision</span><a href="#aff004"><sup>4</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Feng</span><span class="given-names" tagx="given-names" title="given-names">Jian</span></span><span class="role" tagx="role" title="role">Supervision</span><a href="#aff001"><sup>1</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wei</span><span class="given-names" tagx="given-names" title="given-names">Jianhe</span></span><span class="role" tagx="role" title="role">Writing â€" review &amp;amp; editing</span><a href="#aff001"><sup>1</sup></a><a href="#cor001">*</a></span></div><span class="citation_author_institution" id="aff001">[1], <span class="addr-line" title="addr-line">Haikou Provincial Key Laboratory of Resources Conservation and Development of Southern Medicine, Haikou Branch of the Institute of Medicinal Plant Development, Chinese Academy of Medical Sciences and Peking Union Medical College, Haikou, China</span></span><span class="citation_author_institution" id="aff002">[2], <span class="addr-line" title="addr-line">Hainan Key Laboratory of Tropical Oil Crops Biology/Coconut Research Institute, Chinese Academy of Tropical Agricultural Sciences, Wenchang, China</span></span><span class="citation_author_institution" id="aff003">[3], <span class="addr-line" title="addr-line">State Key Laboratory of Tree Genetics and Breeding, Key Laboratory of Tree Breeding and Cultivation of State Forestry Administration, The Research Institute of Forestry, Chinese Academy of Forestry, Beijing, China</span></span><span class="citation_author_institution" id="aff004">[4], <span class="addr-line" title="addr-line">Institute of Agricultural Environment and Soil, Hainan Academy of Agricultural Sciences, Haikou, China</span></span><div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">SchÃ¶nbach</span><span class="given-names" tagx="given-names" title="given-names">Christian</span></span><span class="role" tagx="role" title="role">Editor</span><a href="#edit1" /></span></div><span class="citation_author_institution" id="edit1">[], <span class="addr-line" title="addr-line">Nazarbayev University, KAZAKHSTAN</span></span><div class="author-notes" title="author-notes"> <div class="fn-type-COI-statement" title="COI-statement"> <p><b>Competing Interests: </b>The authors have declared that no competing interests exist.</p> </div> <div class="corresp" title="corresp">
* E-mail: <span class="email" tagx="email" title="email">lulilan1234@163.com</span> (LL); <span class="email" tagx="email" title="email">wjianh@263.net</span> (JW)</div> </div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2020-11-11</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2020</span></span><span class="volume" tagx="volume" title="volume">15</span><span class="issue" tagx="issue" title="issue">11</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e0242776</span><span class="history" title="history"><span class="received" title="received">received: 2019-8-28</span><span class="accepted" title="accepted">accepted: 2020-11-10</span></span><div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Lu et al</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2020</span></span><span class="license" title="license"><span class="license-p" title="license-p">This is an open access article distributed under the terms of the <a href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</span></span></div><a class="self-uri" href="pone.0242776.pdf" title="self-uri">pone.0242776.pdf</a><div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>For more than a thousand years, Rhizoma Curcumae (known as <i>E zhu</i>), a Chinese herbal medicine, has been used to eradicate blood stasis and relieve aches. The plant <i>Curcuma wenyujin</i>, which is grown primarily in Wenzhou, China, is considered the best source of Rhizoma Curcumae. In this study, we sought to ascertain differences in transcript profiles of <i>C</i>. <i>wenyujin</i> grown in traditional (Wenzhou) and recently established (Haikou) production areas based on Illumina and RNA (RNA-seq) sequencing. We also examined differences in the main components of the volatile oil terpene; curcumin, polysaccharide, and starch constituents and related genes in the corresponding pathways, in <i>C</i>. <i>wenyujin</i> cultivated in the two production areas. We accordingly found that the essential oil (2.05%), curcumin (1.46%), and polysaccharide (8.90%) content in Wenzhou rhizomes was higher than that in the rhizomes of plants from Haikou (1.60%, 0.91%, and 6.15%, respectively). In contrast, the starch content of Wenzhou rhizomes (17.0%) was lower than that of Haikou rhizomes (23.8%). Furthermore, we detected significant differences in the oil components of Haikou and Wenzhou rhizomes, with curzerene (32.34%), curdione (21.35%), and germacrene B (9.39%) being the primary components of the essential oil derived from Wenzhou rhizomes, and curzerene (20.13%), curdione (14.73%), and cineole (9.76%) being the main constituents in Haikou rhizomes. Transcriptome and qPCR analyses revealed considerable differences in gene expression between Wenzhou and Haikou rhizomes. The expression of terpene, curcumin, and polysaccharide pathway-related genes in Wenzhou rhizomes was significantly up-regulated, whereas the expression of starch-associated genes was significantly down-regulated, compared with those in Haikou rhizomes. Difference in the content of terpene, curcumin, polysaccharides, and starch in rhizomes from the two production areas could be explained in terms of differences in expression of the related genes.</p> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="funding-statement" title="funding-statement">
YES, Hainan province Key Scientific and Technological Research and Development Special Project (Grant No: ZDKJ2016006), The funders had a role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</div> </div> <div class="counts" title="counts"><span class="fig-count" title="fig-count">fig-count: </span><span class="table-count" title="table-count">table-count: </span><span class="page-count" title="page-count">page-count: </span></div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="Data Availability" value="All relevant data are within the manuscript and its Supporting information files." /> </div> </div> <div class="title" tagx="title" title="title">
Data Availability</div> <p>All relevant data are within the manuscript and its <a href="#sec026">Supporting information</a> files.</p> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
Introduction</div> <p><i>Curcuma</i> is a plant genus comprising more than 70 rhizome-producing species worldwide. In China, there are approximately 20 <i>Curcuma</i> species, a small number of which have been used in Chinese herbal medicines and food products for more than a thousand years. Rhizoma Curcuma is the dried rhizome of <i>Curcuma kwangsiensis</i> S.G. Lee &amp;amp; C.F. Liang, <i>Curcuma phaeocaulis</i> Valeton, and <i>Curcuma wenyujin</i> Y.H. Chen &amp;amp; C. Ling [<a href="#pone.0242776.ref001">1</a>]. The essential oils of <i>Curcuma</i> are considered the active ingredients of these plants and have been demonstrated to have antiviral and antitumor properties [<a href="#pone.0242776.ref002">2</a>â€"<a href="#pone.0242776.ref005">5</a>]. The main constituents of the essential oils in Curcumae rhizomes are curcumol, germacrone, and curdione, which account for most of the known pharmacological effects, and are typically used as quality control markers for the essential oils [<a href="#pone.0242776.ref006">6</a>â€"<a href="#pone.0242776.ref008">8</a>]. <i>C</i>. <i>wenyujin</i> (â€œ<i>wen e zhu</i>â€� in Chinese) from Wenzhou, Zhejiang Province, China, is widely considered to be the best source of Rhizoma Curcumae (â€œ<i>e zhu</i>â€� [<a href="#pone.0242776.ref009">9</a>â€"<a href="#pone.0242776.ref011">11</a>]), on account of its distinctive regional characteristics and high medicinal value, and is mainly produced in Ruian, Wenzhou. The main active constituents of <i>C</i>. <i>wenyujin</i> are volatile oil and curcumin [<a href="#pone.0242776.ref012">12</a>, <a href="#pone.0242776.ref013">13</a>], the former of which is composed of terpenoids and sesquiterpene derivatives, which have been reported to have anti-inflammatory, antitumor, lipid-regulating, antioxidant, antiviral, antibacterial, and other therapeutic effects [<a href="#pone.0242776.ref014">14</a>â€"<a href="#pone.0242776.ref017">17</a>].</p> <p><i>C</i>. <i>wenyujin</i> is mainly cultivated in China, wherein the quality of the rhizomes varies substantially depending on the variety, cultivation region, cultivation techniques, and extraction methods [<a href="#pone.0242776.ref011">11</a>, <a href="#pone.0242776.ref018">18</a>, <a href="#pone.0242776.ref019">19</a>]. <i>C</i>. <i>wenyujin</i> cultivated in different regions, for example, the traditional and non-traditional producing areas, can differ notably with respect to oil components. Given that different environmental conditions can promote changes in the active constituents and alter the transcriptional regulation of genes that control the formation of secondary metabolites, the same <i>C</i>. <i>wenyujin</i> genotype can produce rhizomes with differing qualities depending on the region in and conditions under which the plants are cultivated, and consequently this can potentially present difficulties with respect to quality control.</p> <p>A notable characteristic of plants is their plasticity in response to unfavorable environmental conditions [<a href="#pone.0242776.ref020">20</a>, <a href="#pone.0242776.ref021">21</a>]. Plants typically initiate a series of specific biochemical responses to environmental stressors associated with diverse aspects of their biology, including development, evolution, molecular biology, anatomy, physiology, genetics, and biochemistry [<a href="#pone.0242776.ref022">22</a>, <a href="#pone.0242776.ref023">23</a>], which are of particular importance from the perspectives of predicting changes in community composition, distribution, and crop productivity in response to continuously changing environments [<a href="#pone.0242776.ref024">24</a>, <a href="#pone.0242776.ref025">25</a>].</p> <p>A range of environmental factors (both biotic and abiotic) can affect (often adversely) medicinal plants during their growth, thereby influencing the formation and accumulation of terpenoids, ketones, phenols, alkaloids, flavonoid, flavonols, volatile oils, and other secondary metabolites [<a href="#pone.0242776.ref026">26</a>, <a href="#pone.0242776.ref027">27</a>]. For example, differences in the chemical and physical properties of soils (determined by the parent material) can influence the growth and quality of plants, and in this regard, it has been found that the oil content of <i>C</i>. <i>wenyujin</i> rhizomes cultivated in alkaline soils is higher than that of rhizomes cultivated in acidic red soils [<a href="#pone.0242776.ref028">28</a>].</p> <p>Isopentenyl pyrophosphate (IPP) is an important intermediate in the biosynthesis of terpenoids and steroids in plants, and steroidal alkaloids can also be formed through other pathways. IPP is formed as a common intermediate in both the mevalonate (MVA) and 2-methyl- <div class="sc_UNKNOWN">
d</div>-erythritol-4-phosphoric acid (MEP) pathways, and can pass through cell walls to transfer between these two pathways [<a href="#pone.0242776.ref029">29</a>]. Abiotic environmental factors (including light, temperature, moisture, and nutrition) affect metabolism and the synthesis of metabolites, predominantly by influencing gene expression and transcriptional factors in the associated metabolic pathways (e.g., MVA and MEP) of secondary metabolites, such as terpenoids, flavonoids and flavonols [<a href="#pone.0242776.ref030">30</a>â€"<a href="#pone.0242776.ref038">38</a>]. Thus, it is speculated that variations in natural environmental conditions in the Wenzhou and Haikou regions, which are characterized by subtropical monsoon and tropical monsoon climates, respectively, may contribute to notable differences in the secondary metabolite profiles of <i>C</i>. <i>wenyujin</i> rhizomes sourced from these two regions.</p> <p>In recent years, drug companies (such as Haikou Bikai Pharmaceutical Co. Ltd) in Haikou, Hainan Province, have introduced the local cultivation of <i>C</i>. <i>wenyujin</i> using plant material originally sourced from Ruian, Wenzhou, Zhejiang Province [<a href="#pone.0242776.ref039">39</a>]. <i>C</i>. <i>wenyujin</i> is cultivated in Haikou for the production of medicines derived from this plant, with the goal of meeting the growing demand for <i>C</i>. <i>wenyujin</i> constituents (volatile oils, crude extracts, and raw materials). In this study, we sought to determine the main constituents and genetic characteristics of <i>C</i>. <i>wenyujin</i> rhizomes sourced from Wenzhou and Haikou, with the aim of assessing the qualities of the respective <i>C</i>. <i>wenyujin</i> rhizomes. The main objective was to analyze the content of essential oils (such as terpene), curcumin, polysaccharides, and starch in <i>C</i>. <i>wenyujin</i> rhizomes obtained from the traditional production areas (Wenzhou) and newly established production areas (Haikou). In this regard, although there have been a few transcriptome studies on plants in the ginger family (e.g., <i>Curcuma zedoaria</i> (Christm.) Roscoe) [<a href="#pone.0242776.ref040">40</a>â€"<a href="#pone.0242776.ref042">42</a>], to date there have been no studies that have examined genetic variations in the metabolic pathways of <i>C</i>. <i>wenyujin</i>. Moreover, there have been only a limited number of studies on the transcriptome of <i>C</i>. <i>wenyujin</i> and the mechanisms underlying interspecific differences in the primary and secondary metabolites that accumulate in the rhizomes of <i>C</i>. <i>wenyujin</i> rhizomes cultivated in traditional and non-traditional production areas.</p> <p>In this study, using Illumina and RNA (RNA-seq) sequencing, we examined the transcript profiles of <i>C</i>. <i>wenyujin</i> rhizomes produced in the traditional production area (Wenzhou) and those derived from an area in which the plant has recently been introduced (Haikou). We also sought to determine variations in the content of the main components of the volatile oil terpene, curcumin, polysaccharides, and starch, and related genes in the corresponding pathways. The data, thus, obtained provide a basis for evaluating the quality of <i>C</i>. <i>wenyujin</i> and elucidating the molecular mechanisms underlying the biosynthetic pathways in plants cultivated in these two production areas, and hence will enable us to assess the potential medicinal value, development, and utilization of <i>C</i>. <i>wenyujin</i> in more agriculturally productive areas, such as Haikou.</p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Materials and methods</div> <div class="plantmaterials" title="sec"> <div class="title" tagx="title" title="title">
Plant materials</div> <p>Samples of the traditionally cultivated (Wenzhou) <i>C</i>. <i>wenyujin</i> (WZ) were purchased from Taoshan Town, Ruian, Wenzhou, Zhejiang Province, on December 5, 2017, whereas those of the introduced (Haikou) <i>C</i>. <i>wenyujin</i> (HK) were obtained on the same date from cultivation fields at the Haikou branch of the Institute of Medicinal Plants. The cultivation management of <i>C</i>. <i>wenyujin</i> in the two areas is similar, and details of the environmental and weather conditions of these areas are presented in <a href="#pone.0242776.s008">S1 Table</a>. The <i>Curcuma</i> samples were subsequently identified as <i>C</i>. <i>wenyujin</i> Y.H. Chen &amp;amp; C. Ling. by Li-Rongtao, Associate Professor at the Institute of Medicinal Plants, Chinese Academy of Medical Sciences. For the purposes of the present study, we selected the rhizomes of the <i>C</i>. <i>wenyujin</i> plants as the experimental materials.</p> </div> <div class="rnaisolation,quantification,andqualification" title="sec"> <div class="title" tagx="title" title="title">
RNA isolation, quantification, and qualification</div> <p>Total RNA was isolated from each sample (rhizomes) using a Quick RNA isolation kit (Bioteke Corporation, Beijing, China). Three biological replicates per experimental group were used for sequencing. Agarose gel (1%) electrophoresis was used to assess the contamination and degradation of RNA. The purity and concentration of the isolated RNA were determined using a Nano spectrophotometer (Photometer<sup>Â®</sup>; IMPLEN, CA, USA) and a Qubit<sup>Â®</sup>2.0 Fluorometer Kit (RNA Assay, Life Technologies, CA, USA), respectively, whereas an Agilent 2100 Bioanalyzer system (Nano 6000; Agilent Technologies, CA, USA) was used to evaluate RNA integrity.</p> </div> <div class="librarypreparationandsequencing" title="sec"> <div class="title" tagx="title" title="title">
Library preparation and sequencing</div> <p>An RNA library was prepared using a total of 3 Î¼g RNA per sample, which was analyzed using high-throughput Illumina sequencing technology at the Beijing BioMarker Technologies Company (Beijing, China). An RNA Library Prep Kit (NEB for Illumina, USA) was used for generating sequencing libraries, in accordance with the manufacturerâ€™s recommendations, and the sequences of each sample were attributed using index codes. mRNA was extracted from the total RNA using Poly-T oligo-attached magnetic beads. Fragmentation of the sequences was facilitated by crushing the samples in the presence of divalent cations in a NEBNext first-chain synthetic buffer (5Ã—) at an elevated temperature. First-chain cDNA synthesis was performed using RNase-H (Invitrogen, USA), and Polymerase I (Invitrogen, USA) and RNase-H (Invitrogen, USA) were used for second-chain cDNA synthesis. The remaining overhangs were blunt-ended based on polymerase or exonuclease reactions. Subsequent to adenylation of the DNA fragments (3Ê¹ ends), the junction (NEBNext), with a hairpin loop structure, was ligated in preparation for hybridization. To preferentially select cDNA fragments of 150â€"200 bp in length, we used an XP system (AMP, Beckman Coulter, Beverly, USA) to separate the DNA fragments. Prior to PCR, 3 Î¼L USER enzyme (NEB, USA) was added to the samples containing the size-selected adaptor-ligated cDNA, which were heated at 37Â°C for 15 min, followed by 95Â°C for 5 min. PCR was carried out using DNA polymerase (Phusion High-Fidelity), Index (X), and Universal Primers (PCR), and the XP system (AMP, Beckman Coulter, Beverly, USA) was applied to purify the PCR products. Library quality was evaluated using an Agilent 2100 Bioanalyzer system (Nano 6000; Agilent Technologies, CA, USA).</p> </div> <div class="dataanalysisandassembly" title="sec"> <div class="title" tagx="title" title="title">
Data analysis and assembly</div> <p>An internal Perl script was initially used to handle the raw read data in fastq (fq) format. Clean reads were acquired by deleting reads including adapters, poly-N sequences, and low-quality reads from the raw data. Moreover, we statistically assessed the sequence repetition, GC content, and Q20 and Q30 levels of clean read data. For all library samples, the left read1 and right read2 files were placed in two larger files (left fq and right fq, respectively). Assembly of the transcriptome was performed based on left/right fq data using the Trinity method, with min_kmer_cov set to 2 and all other parameters automatically set to default [<a href="#pone.0242776.ref043">43</a>, <a href="#pone.0242776.ref044">44</a>].</p> </div> <div class="sequenceannotationandfunctionalclassification" title="sec"> <div class="title" tagx="title" title="title">
Sequence annotation and functional classification</div> <p>Genes with unique sequences (unigenes) were annotated and functionally classified based on Clusters of Orthologous Groups of proteins (COG), KOG, and eggNOG protein sequence data libraries for manual annotation and reviewed against Swiss-Prot. Gene Ontology (GO), and Kyoto Encyclopedia of Genes and Genomes (KEGG) databases [<a href="#pone.0242776.ref045">45</a>].</p> <div class="quantificationofgeneexpression" title="sec"> <div class="title" tagx="title" title="title">
Quantification of gene expression</div> <p>RSEM was applied to analyze the levels of gene expression in each sample [<a href="#pone.0242776.ref046">46</a>]. Clean read data were mapped onto the assembled transcriptome, and the read counts per gene were obtained from the transcriptome database. For the analysis of differential gene expression, we used the DESeq package of R (1.10.1). <i>P</i>-value results were assessed using the Benjamin and Hochberg method (1995) to control for the false discovery rate (FDR), and genes with a <i>P</i>-value &amp;lt; 0.05 were defined as being differentially expressed (DEGs) [<a href="#pone.0242776.ref045">45</a>, <a href="#pone.0242776.ref047">47</a>].</p> </div> <div class="goenrichmentanalysis" title="sec"> <div class="title" tagx="title" title="title">
GO enrichment analysis</div> <p>For analysis of DEG enrichment, we used the R package topGO, based on the Kolmogorovâ€"Smirnov test [<a href="#pone.0242776.ref045">45</a>, <a href="#pone.0242776.ref048">48</a>].</p> </div> <div class="keggpathwayenrichmentanalysis" title="sec"> <div class="title" tagx="title" title="title">
KEGG pathway enrichment analysis</div> <p>KEGG pathway enrichment with DEGs was determined using KOBAS 2.0 software [<a href="#pone.0242776.ref047">47</a>, <a href="#pone.0242776.ref049">49</a>].</p> </div> </div> <div class="samplepreparationandgaschromatography-massspectrometry(gc-ms)analysis" title="sec"> <div class="title" tagx="title" title="title">
Sample preparation and Gas Chromatography-Mass Spectrometry (GC-MS) analysis</div> <p>Sample materials were obtained from large rhizomes of <i>C</i>. <i>wenyujin</i>, which had been boiled for 1 h at 100Â°C and 0.1 MPa, and then cut into 1â€"2 cm slices and dried for 24 h at 50Â°C. The volatile oil content of samples was then extracted in accordance with Chinese Pharmacopoeia guidelines [<a href="#pone.0242776.ref001">1</a>]. For high-performance liquid chromatography (HPLC) we used an Agilent 7890A gas chromatograph under the following chromatographic conditions: DB-5MS column (0.25 mm Ã— 30 m, 0.25 Î¼m), helium carrier gas, EI ion ionization method, 200Â°C source temperature, 70 eV electron energy, 250Â°C interface temperature, 150 Î¼A emission current, 35 to 455 mass range, and a 0.4 s scan cycle. Xcalibur 1.2 was used as a data processing system, and the map library was NIST (Version 1.7). The heating program used was as follows: an initial temperature of 60Â°C held for 1 min, followed by an increase from 60Â°C to 100Â°C at 4Â°CÂ·min<sup>-1</sup>, from 100Â°C to 120Â°C at 2Â°CÂ·min<sup>-1</sup>, from 120Â°C to 180Â°C at 1Â°CÂ·min<sup>-1</sup>, and from 180Â°C to 230Â°C at 23Â°CÂ·min<sup>-1</sup>, where the temperature was held for 10 min. Other parameters were as follows: 53 kPa column pressure, 1.04Â°CÂ·min<sup>-1</sup> flow rate, and 0.2 Î¼L sample volume. A 20 Î¼L volume of eugenol:<i>n</i>-hexane (20:1) was added to a 2 mL brown gas-phase flask, to which was added 1000 Î¼L of <i>n</i>-hexane. The solution was mixed by vortexing for 1 min and then filtered. To obtain the total ion chromatograms of different volatile oils, we used an Agilent 7890A gas chromatograph coupled to a 5975 C quadrupole mass spectrometer. The relative value (percentage) of each chromatogram peak was calculated using the peak area normalization method, and following a computer search and manual analysis, the compounds were identified with reference to the literature.</p> </div> <div class="quantitativereverse-transcriptionpcr(qrt-pcr)analysis" title="sec"> <div class="title" tagx="title" title="title">
Quantitative reverse-transcription PCR (qRT-PCR) analysis</div> <p>qPCR analysis was performed using an IQ5 Multicolor (CFX96) PCR detection system (Bio-Rad, Bole, USA) and a Power PCR Master Mix (SYBR green) (Takara, Beijing, China). RNase-free DNase (TaKaRa, Beijing, China) was used to extract the total RNA from the <i>C</i>. <i>wenyujin</i> rhizomes, and 15 primers (Oligod-T) and a SuperScript (III) RT kit (Invitrogen, USA) were used for the reverse transcription (RT) reaction. For each RT reaction, we used 20 Î¼L reaction mixtures containing 10 Î¼L of master mix reagent (2Ã— SYBR Green Master) (TaKaRa, Beijing, China), 10 ng of sample (cDNA), and 200 nM of gene-specific primers. The cycle conditions were as follows: an initial denaturation at 95Â°C for 30 s, followed by 40 cycles at 95Â°C for 5 s and 60Â°C for 34 s. The amplification program was then followed by a 55Â°Câ€"85Â°C melting curve analysis, with each temperature maintained for 5 s. For PCR amplification, we used gene-specific primers (<a href="#pone.0242776.s009">S2 Table</a>), the annealing efficiency of which was assessed using the primer program (PRIMER 3.0) [<a href="#pone.0242776.ref050">50</a>]. As an internal control gene, we used c121677 against which the average amplification of three replicate samples was normalized. The relative expression values of the target genes were calculated by comparing the target gene cycle thresholds (CTs) with those of the c121677 housekeeping gene using the 2<sup>-Î"Î"CT</sup> method [<a href="#pone.0242776.ref051">51</a>, <a href="#pone.0242776.ref052">52</a>].</p> </div> <div class="determinationofcurcumincontent" title="sec"> <div class="title" tagx="title" title="title">
Determination of curcumin content</div> <p>For HPLC analysis of curcumin, we used an Agilent Extend C18 column (4.6 Ã— 250 mm, 5 Î¼m) under the following chromatographic conditions: a mobile phase of 4% acetonitrile and glacial acetic acid solution (45:55), a flow rate of 1.0 mLÂ·min<sup>-1</sup>, a detection wavelength of 420 nm, a column temperature of 25Â°C, and an injection volume of 10 Î¼L. The number of theoretical plates was not less than 4,000. The curcumin reference standard was accurately weighed (6.35 mg) after ensuring that it had been dried to a constant weight [CAS: 110823â€"201706; China National Institute of Food and Drug Control (CNIFDC), China], and was then placed into a 25 mL brown volumetric flask, dissolved with methanol to 25 mL, shaken, and filtered as the reference stock solution. A curcumin control solution (25.4 Î¼gÂ·mL<sup>-1</sup>) was prepared by diluting a 1 mL aliquot of this stock solution with methanol in a 10 mL brown volumetric flask. Dried powdered rhizome (0.5 g) was placed into a 50 mL conical flask to which 20 mL methanol was added. The mixture was sonicated for 1 h, and then centrifuged. To the resulting supernatant, 15 mL of methanol was added, followed by sonication for 40 min and subsequent filtering. The residual material was rinsed with a small amount of methanol and refiltered. The filtrate, thus, obtained was added to the initial extraction supernatant solution, and the combined filtrate was then made up to 20 mL using methanol at a temperature below 20Â°C.</p> </div> <div class="determinationofpolysaccharidecontent" title="sec"> <div class="title" tagx="title" title="title">
Determination of polysaccharide content</div> <p>The polysaccharide content of rhizomes was determined in accordance with the methodology of the â€œChinese Pharmacopoeiaâ€� (Appendix VA; 2015) [<a href="#pone.0242776.ref001">1</a>]. The extracted polysaccharides were analyzed using a UV1601 UV-Vis spectrophotometer (Shimadzu, Japan) at a wavelength of 490 nm, with an appropriate amount of anhydrous glucose being used as the reference material (reference <div class="sc_UNKNOWN">
d</div>-anhydroglucose; CAS: 0833â€"9501; CNIFDC, China). The analytical chemicals, apparatus, conditions, and preparation of standard and sample solutions have previously been described in detail by Chen [<a href="#pone.0242776.ref053">53</a>].</p> </div> <div class="determinationofstarchcontent" title="sec"> <div class="title" tagx="title" title="title">
Determination of starch content</div> <p>Starch content was analyzed using a UV1601 UV-Vis spectrophotometer (Shimadzu, Japan) at a wavelength of 600 nm, with a soluble starch standard (chromatographic grade, CAS: 140602; CNIFDC, China). The analytical chemicals, apparatus, analytical conditions, standard preparation, starch extraction, and preparation of <i>C</i>. <i>wenyujin</i> rhizomes samples have previously been described in detail by Zhang et al. [<a href="#pone.0242776.ref054">54</a>].</p> </div> <div class="statisticalanalysis" title="sec"> <div class="title" tagx="title" title="title">
Statistical analysis</div> <p>Average values of each parameter were calculated from three replicate analyses, and standard errors of the mean values were obtained. Univariate analysis (analysis of variance) of variance was applied to determine the significance of the results among different treatments, with a <i>P</i>-value &amp;lt; 0.05 being considered significant. SPSS V.13 was used for statistical analyses.</p> </div> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="analysisofrhizomeconstituentandvolatileoilcompositions" title="sec"> <div class="title" tagx="title" title="title">
Analysis of rhizome constituent and volatile oil compositions</div> <p>Our analyses revealed notable differences in the rhizome (<i>E zhu</i>) structure and volatile oil color and content of the <i>Curcuma wenyujin</i> samples obtained from the two different regions. The volatile oil content of <i>C</i>. <i>wenyujin</i> rhizomes derived from Wenzhou was found to be higher than that in the rhizomes from Haikou, and these oils were characterized by bright and dark yellow colors, respectively (<a href="#pone.0242776.g001">Fig 1</a>). Paraffin sections of rhizome tissue, stained using the periodic acid-Schiff (PAS) reaction and potassium iodide, revealed the cellular localization of oil and the quantity of starch granules, respectively (<a href="#pone.0242776.g002">Fig 2</a>). The oil (2.05%), curcumin (1.46%), and polysaccharide (8.90%) content of <i>C</i>. <i>wenyujin</i> rhizomes from Wenzhou was significantly higher than that of rhizomes from Haikou (1.60%, 0.91%, and 6.15%, respectively), whereas the starch content of rhizomes from Wenzhou (17.0%) was significantly lower than that of rhizomes from Haikou (23.8%) (<a href="#pone.0242776.g003">Fig 3</a>). We also detected significant differences in the compositions of the volatile oils derived from the rhizomes of plants obtained from the two cultivation regions, with the content of curzerene and curdione being higher in those from Wenzhou (<a href="#pone.0242776.s010">S3 Table</a>). Curzerene (32.34%), curdione (21.35%), and germacrene B (9.39%) were found to be the major components of the essential oil of rhizomes from Wenzhou, whereas in the rhizomes harvested from Haikou, curzerene (20.13%), curdione (14.73%), and cineole (9.76%) were identified as the main essential oil constituents. Thus, the chemical constituents of essential oils in the rhizomes of Wenzhou- and Haikou-cultivated <i>C</i>. <i>wenyujin</i> appear to be quite different.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pone.0242776.g001"><span class="label" tagx="label" title="label">Fig 1</span></a></div>   <div class="title" tagx="title" title="title">
The rhizomes and volatile oils of <i>Curcuma wenyujin</i> from Wenzhou and Haikou.</div>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pone.0242776.g002"><span class="label" tagx="label" title="label">Fig 2</span></a></div>   <div class="title" tagx="title" title="title">
Paraffin sections of <i>Curcuma wenyujin</i> rhizomes showing starch (potassium iodide staining) (a, b) and oil cells (PAS reaction) (c, d) in rhizomes from Wenzhou (b, d) and Haikou (a, c) observed under a biological microscope (a, b scale = 50 Î¼m; c, d scale = 100 Î¼m).</div>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pone.0242776.g003"><span class="label" tagx="label" title="label">Fig 3</span></a></div>   <div class="title" tagx="title" title="title">
The content of oil, curcumin, polysaccharide, and starch in <i>Curcuma wenyujin</i> rhizomes produced in Wenzhou and Haikou.</div>   </div> </div> <div class="mrnasequencing,assembly,functionalannotation,andclassification" title="sec"> <div class="title" tagx="title" title="title">
mRNA sequencing, assembly, functional annotation, and classification</div> <p>To analyze the dynamics of mRNA expression in <i>C</i>. <i>wenyujin</i> from different production areas, we constructed and sequenced mRNA-seq libraries for <i>C</i>. <i>wenyujin</i> derived from Haikou (introduced cultivation) (HK) and Wenzhou (traditional cultivation) (WZ). A comparison of three replicates of the two experimental groups verified that they do not differ significantly at the sequence level (<a href="#pone.0242776.s011">S4</a> and <a href="#pone.0242776.s013">S6</a> Tables). Sequencing of a total of six transcriptome samples (HK1, HK2, HK3, WZ1, WZ2, and WZ3) yielded a total of 41.74 Gbp of clean data (reaching up to at least 6.37 Gbp for each sample), with Q30 percentages (number of samples with Phred quality score higher than 30) of more than 89.15% in the mRNA-seq libraries of the six samples (<a href="#pone.0242776.s011">S4 Table</a>). In total, we acquired 185,006 transcripts, representing 99,942 unigenes, with N50 values of 1,796 and 1,406 bp for transcripts and unigenes, respectively, and 28,375 unigenes being larger than 1 kb (<a href="#pone.0242776.s012">S5</a> and <a href="#pone.0242776.s013">S6</a> Tables). Searches for all unique sequences in public databases using BLASTX (E values &amp;lt; 1.00<sup>âˆ’5</sup>) [<a href="#pone.0242776.ref055">55</a>] and subsequent functional annotation implemented based on the NR, Swiss-Prot, COG, GO, and KEGG database, enabled us to confirm the identity of 51,609 unigenes (51.6%), whereas we were unable to similarly identify the remaining 48,333 unique sequences (48.4%) (<a href="#pone.0242776.s014">S7 Table</a>, <a href="#pone.0242776.s001">S1 Fig</a>).</p> <p>The functions of the unique sequences extracted from <i>C</i>. <i>wenyujin</i> were classified based on the three GO terms cellular component (CC), biological process (BP), and molecular function (MF). Compared with HK, 15,151 unigenes and 924 DEGs in WZ were classified as CC; 19,906 unigenes and 1,276 DEGs were classified as BP; and 21,535 unigenes and 1,456 DEGs were classified as MF. The DEGs were mainly assigned to the categories â€œcell, go: 0005623,â€� â€œcell part, go: 0044464,â€� â€œorganelle, go: 0043226,â€� â€œmembrane, go: 0016020,â€� â€œcellular process, go: 0009987,â€� â€œmetabolic process, go: 0008152,â€� â€œsingle-organism process, go: 0044699,â€� â€œbinding, go: 0005488,â€� and â€œcatalytic activity, go: 0003824â€� (<a href="#pone.0242776.g004">Fig 4</a>). The majority of the GO terms identified in the present study are in accordance with the GO categories defined previously [<a href="#pone.0242776.ref052">52</a>, <a href="#pone.0242776.ref056">56</a>].</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pone.0242776.g004"><span class="label" tagx="label" title="label">Fig 4</span></a></div>   <div class="title" tagx="title" title="title">
A Gene Ontology (GO) histogram of the classification of annotated unigenes that were differentially expressed between Wenzhou (WZ) and Haikou (HK) rhizomes.</div> <p>The red, green, and blue histograms represent the differentially expressed genes (DEGs) in the different sub-categories, whereas the light red, green, and blue histograms represent the annotated unigenes. The right-hand Y-axis represents the number of annotated genes or DEGs in the main categories, and the left-hand Y-axis represents the percentage of annotated unigenes or DEGs in the main categories.</p>   </div> <p>With respect to KEGG pathways, we identified 1,597 unique sequences (<a href="#pone.0242776.g005">Fig 5</a>). The predominantly enriched pathways appeared to include the following: endocytosis (36; 3.5%) and phagosome (35; 3.4%) in cellular processes; endoplasmic reticulum protein processing (70; 7.09%), spliceosome (77; 8.0%), ribosome (89; 9.09%), and RNA transport (57; 5.5%) in genetic information processing; and carbon metabolism (72; 7.12%), and biosynthesis of amino acids (58; 5.6%) in metabolism.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pone.0242776.g005"><span class="label" tagx="label" title="label">Fig 5</span></a></div>   <div class="title" tagx="title" title="title">
Annotation of unique sequences (1,597) based on KEGG classification.</div>   </div> </div> <div class="differentialgeneexpression" title="sec"> <div class="title" tagx="title" title="title">
Differential gene expression</div> <p>We used the fragments per kilobase per million reads mapped (FPKM) procedure to calculate the expression level of genes [<a href="#pone.0242776.ref057">57</a>], and DESeq (R package, version 1.10.1) was used for DEG analysis, The latter revealed that 4,620 genes were significantly differentially expressed in WZ compared with those in HK, of which 3,978 were up-regulated and 642 were down-regulated (<a href="#pone.0242776.g006">Fig 6</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pone.0242776.g006"><span class="label" tagx="label" title="label">Fig 6</span></a></div>   <div class="title" tagx="title" title="title">
Differential expression of the unique genes in <i>Curcuma wenyujin</i> rhizomes obtained from Wenzhou (WZ) and Haikou (HK).</div> <p>Green and red dots denote genes with significantly different expression (FDR &amp;lt; 0.01), with green indicating down-regulated genes, red indicating up-regulated genes, and black indicating those genes showing no significantly different expression.</p>   </div> <p>To determine similarities and differences in the transcriptional changes of genes in WZ and HK <i>C</i>. <i>wenyujin</i>, we performed hierarchical clustering, which accordingly revealed differences in the expression patterns of genes in the two sources of <i>C</i>. <i>wenyujin</i> (<a href="#pone.0242776.g007">Fig 7</a>). A subset of transcripts, together with their respective fold changes and false discovery rate-corrected <i>P</i>-values, for both clusters, are presented in <a href="#pone.0242776.s015">S8 Table</a>. Among these, genes related to terpene, curcumin, and polysaccharide pathways were found to be up-regulated in <i>C</i>. <i>wenyujin</i> from Wenzhou, where genes associated with starch and sugar pathways were up-regulated in <i>C</i>. <i>wenyujin</i> from Haikou.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pone.0242776.g007"><span class="label" tagx="label" title="label">Fig 7</span></a></div>   <div class="title" tagx="title" title="title">
Cluster analysis of genes that were differentially expressed (4,620) between Wenzhou (WZ) and HK (HK) rhizomes.</div> <p>Red and green represent up- and down-regulated genes, respectively, in Wenzhou (WZ) rhizomes compared with those of Haikou (HK). The heatmap was produced based on fragments per kilobase per million (FPKM) data.</p>   </div> </div> <div class="functionalenrichmentanalysisofdegs" title="sec"> <div class="title" tagx="title" title="title">
Functional enrichment analysis of DEGs</div> <p>In the present study, we used topGO to analyze the functional enrichment of DEGs with respect to the three GO functional categories, CC, BP, and MF (enrichment significance, K<sub>S</sub> &amp;lt; 0.05), the details of which are shown in <a href="#pone.0242776.s016">S9 Table</a>. Among all DEGs, plastid and chloroplast (CC); methylerythritol 4-phosphate (MEP) pathway, isopentenyl diphosphate biosynthetic process, and thylakoid membrane organization (BP); and protein serine/threonine kinase activity and protein kinase activity (MF) were enriched in WZ rhizomes compared with that in HK rhizomes. Among the up-regulated DEGs, the GO terms pollen tube tip (CC); leaf morphogenesis (BP); and polyamine oxidase activity, isomerase activity, ion channel activity, and phospholipase activity (MF) were significantly enriched in WZ compared with those in HK, whereas for down-regulated DEGs, cell wall (CC); defense responses to fungus (BP); and peroxidase activity, copper ion binding, sucrose synthase activity, heat shock protein binding, and catalytic activity (MF) were significantly enriched in WZ compared with those in HK (<a href="#pone.0242776.s019">S1 File</a>).</p> <p><a href="#pone.0242776.g008">Fig 8</a> shows the results of our COG functional classifications of the consensus sequences of genes that were differentially expressed between WZ and HK. The 1,698 genes that were differentially expressed between the rhizomes obtained from these two sources can primarily be divided into the following eight categories: general function prediction only (R) (481, 28.33%); transcription (K) (230, 13.55%); replication, recombination, and repair (L) (208, 12.25%); signal transduction mechanisms (T) (190, 11.19%); translation ribosomal structure and biogenesis (J) (189, 11.13%); posttranslational modification protein turnover chaperones (O) (178, 10.48%); amino acid transport and metabolism (E) (153, 9.01%); and carbohydrate transport and metabolism (G) (140, 8.24%). The corresponding numbers and percentages of the annotated genes in these eight categories are as follows: R (2,427, 19.82%), K (415, 55.42%), L (512, 40.63%), T (405, 46.91%), J (1,426, 13.25%), O (1,373, 12.96%), E (892, 17.15%), and G (765, 18.30%). Thus, the genes associated with these eight categories can be assumed play vital roles in the growth of <i>C</i>. <i>wenyujin</i> in the Wenzhou traditional production area.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pone.0242776.g008"><span class="label" tagx="label" title="label">Fig 8</span></a></div>   <div class="title" tagx="title" title="title">
Clusters of Orthologous Group (COG) functional classification of the consensus sequences of genes that are differentially expressed between Wenzhou (WZ) and Haikou (HK) rhizomes.</div>   </div> <p>Similarly, functional analyses of the DEGs based on the KEGG classification of enriched pathways revealed that with respect to all genes differentially expressed between WZ and HK, 179 pathways showed enrichment, among which there were 117 and 62 pathways associated with up- and down-regulated DEGs, respectively (<a href="#pone.0242776.s017">S10 Table</a>). For these pathways, we analyzed the significance of the enrichment factor and Q values, and in <a href="#pone.0242776.g009">Fig 9</a>, we present the first 20 minimal Q value pathways. In total, 980 DEGs were assigned to KEGG pathways, among which, 844 and 136 were up- and down-regulated, respectively. The dominant and significantly enriched pathways are listed in <a href="#pone.0242776.s017">S10 Table</a>. With respect to all DEGs, the predominantly enriched KEGG pathways associated with the following functional areas: ribosome, spliceosome, endoplasmic reticulum protein processing, ubiquitin-mediated proteolysis, RNA transport, phagosome, glyoxylate and dicarboxylate metabolism, and glutathione metabolism. Pathways predominantly enriched with the up-regulated DEGs included spliceosome, carbon metabolism, ubiquitin-mediated proteolysis, proteasome, citrate cycle (TCA cycle), aminoacyl-tRNA biosynthesis, arginine and proline metabolism, and glutathione metabolism, whereas those significantly enriched the down-regulated DEGs included ribosome, protein export, riboflavin metabolism, endoplasmic reticulum protein processing, plant hormone signal transduction, starch and sucrose metabolism.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pone.0242776.g009"><span class="label" tagx="label" title="label">Fig 9</span></a></div>   <div class="title" tagx="title" title="title">
Enrichment of KEGG pathways with genes that were differentially expressed between Wenzhou (WZ) and Haikou (HK) rhizomes.</div> <p>Pathway name and enrichment intensity are shown in the right-hand legend. The Q value is the corrected <i>P</i>-value (false discovery rate). The enrichment factor is the ratio of the number of differentially expressed genes (DEGs) in a pathway to the number of all genes in that pathway. (a, all DEGs; b, up-regulated DEGs; c, down-regulated DEGs).</p>   </div> </div> <div class="verificationofdegs" title="sec"> <div class="title" tagx="title" title="title">
Verification of DEGs</div> <p>To establish the reliability of the FPKM procedure, we selected the sequences of 15 of the DEGs that are involved in terpene, curcumin, polysaccharide, and starch pathways for qRT-PCR analysis. Our findings that the overall correlation coefficient of the linear regression analysis between FPKM and qRT-PCR was 0.959 (R = 0.959) indicated a good correlation between the transcript abundance detected by RT-PCR and the transcription spectrum of RNA sequence data analysis (<a href="#pone.0242776.s002">S2 Fig</a>). Data obtained from RNA-seq and qRT-PCR analyses revealed that those genes associated with terpene (LPS, CDS/LIS, ISPS, and TPS06), curcumin (CHS2, CHS1, and TT4), and polysaccharide (TSTA, GALE, and GH) pathways were significantly up-regulated in the rhizomes of <i>C</i>. <i>wenyujin</i> cultivated in Wenzhou, whereas genes associated with polysaccharide (LOC103985749) and starch (GAE, PME, BGLU, and SUS) pathways were significantly up-regulated in the rhizomes of <i>C</i>. <i>wenyujin</i> obtained from Haikou. These differences in gene expression patterns were consistent with those obtained based on transcriptome FPKM analysis (<a href="#pone.0242776.g010">Fig 10</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pone.0242776.g010"><span class="label" tagx="label" title="label">Fig 10</span></a></div>   <div class="title" tagx="title" title="title">
qRT-PCR verification of genes that were differentially expressed between Wenzhou (WZ) and Haikou (HK) rhizomes (a) and the corresponding verification based on RNA-seq (b).</div> <p>Using c121677 as a control/reference gene, the relative expression of genes was calculated using the 2-<sup>Î"Î"Ct</sup> method. Asterisks denote significant differences (*, <i>P</i> &amp;lt; 0.05; **, <i>P</i> &amp;lt; 0.01; ***, <i>P</i> &amp;lt; 0.001). Error bars represent the standard deviations of mean values (n = 6). (CHS2: chalcone synthase2; CHS1: chalcone synthase1; TT4: chalcone and stilbene synthase family protein; LPS: levopimaradiene synthase; CDS/LIS: 3S-linalool synthase; ISPS: isoprene synthase; TPS06: dolabella-3,7-dien-18-ol synthase/sesquiterpene synthase 6; GAE: UDP-glucuronate 4-epimerase; PME: pectin methylesterase; BGLU: beta-glucosidase; SUS: sucrose synthase; TSTA: GDP- <div class="sc_UNKNOWN">
l</div>-fucose synthase; GALE: UDP-glucose 4-epimerase; GH: glycoside hydrolase; LOC103985749: uncharacterized protein LOC103985749).</p>   </div> </div> <div class="transcriptionfactoranalysis" title="sec"> <div class="title" tagx="title" title="title">
Transcription factor analysis</div> <p>Transcription factor analysis revealed that 57 classes of transcription factors showed dynamic changes in Haikou rhizomes compared with those from Wenzhou. Compared with the former, 52 types of transcription factor were up-regulated in Wenzhou-derived rhizomes, of which the expression levels of genes in the <i>RLK</i>, <i>C2H2</i>, <i>Zn-clus</i>, <i>TKL</i>, <i>bZIP</i>, <i>CAMK</i>, <i>AP2/ERF</i>, <i>AGC</i>, <i>STE</i>, <i>C2C2</i>, <i>CMGC</i>, <i>C3H</i>, <i>SNF2</i>, <i>WRKY</i>, <i>bHLH</i>, <i>GNAT</i>, <i>MYB</i>, <i>HSF</i>, and <i>NAC</i> families were significantly up-regulated. Among these, the gene expression of certain types of transcription factor only showed up-regulation, whereas a few genes in the <i>RLK</i>, <i>bZIP</i>, <i>AP2/ERF</i>, <i>WRKY</i>, <i>bHLH</i>, <i>MYB</i>, <i>HSF</i>, and <i>NAC</i> classes also showed down-regulation in <i>C</i>. <i>wenyujin</i> rhizomes from WZ compared with those from HK (<a href="#pone.0242776.s003">S3 Fig</a>, <a href="#pone.0242776.s018">S11 Table</a>).</p> </div> </div> <div class="conclusions" title="conclusions"> <div class="title" tagx="title" title="title">
Discussion</div> <p>Environmental factors play a prominent role in determining the growth of plants, and environmental quality and climatic conditions in different growing areas are the most important factors contributing to variations in plant primary and secondary metabolites [<a href="#pone.0242776.ref018">18</a>, <a href="#pone.0242776.ref019">19</a>].</p> <p>Previous studies that have conducted chemical component and transcriptome analyses of <i>Curcuma</i> species have reported large variations in the content and composition of volatile oils and the content of curcumin and polysaccharide among different <i>Curcuma</i> species grown in different areas, as well as in the corresponding Rhizoma Curcumae [<a href="#pone.0242776.ref058">58</a>â€"<a href="#pone.0242776.ref061">61</a>]. Consistently, the findings of the present revealed large differences in the content and composition of volatile oils and the content of curcumin, polysaccharide, and starch in the rhizomes of <i>C</i>. <i>wenyujin</i> sourced from the traditional production areas in Wenzhou and the more recently established production areas in Haikou. Notably in this regard, we detected significantly higher content of the volatile oil terpene, curcumin, and polysaccharides in <i>C</i>. <i>wenyujin</i> rhizomes from Wenzhou compared with that in the rhizomes of plants cultivated in the Haikou area, whereas the content of starch was significantly higher in the Haikou rhizomes. Additionally, we detected certain differences between Wenzhou and Haikou rhizomes with respect to the composition of the volatile oil, which is consistent with the observations of Huang et al., who found that the volatile oil content of <i>C</i>. <i>wenyujin</i> rhizomes from Haikou (2.9%) was slightly lower than that of rhizomes from Zhejiang (3.0%), and also reported considerable differences in the volatile oil constituents of plants growing in different habitats [<a href="#pone.0242776.ref039">39</a>]. Further studies have also shown differences in the chemical constituents of volatile oils in <i>C</i>. <i>wenyujin</i> from different planting areas, which were found to be associated with differences in the quality of the corresponding medicinal materials [<a href="#pone.0242776.ref017">17</a>, <a href="#pone.0242776.ref062">62</a>, <a href="#pone.0242776.ref063">63</a>]. It has similarly been demonstrated that volatile oil content may vary depending upon the origin of the variety, cultivation area, cultivation techniques, and extraction method [<a href="#pone.0242776.ref008">8</a>], and Cao et al. have reported that the content of volatile oil and curcumin differed substantially in different experimental areas, with the curcumin content varying from 1.5% to 5% and volatile oil content ranging from 0.4% to 0.7% [<a href="#pone.0242776.ref063">63</a>]. In the present study, RT-PCR and transcriptome RNA-seq analyses revealed that genes involved in terpene, curcumin, and polysaccharide production were significantly up-regulated in <i>C</i>. <i>wenyujin</i> rhizomes from Wenzhou, thereby indicating that these genes are closely associated with observed differences in the profiles of these metabolites in <i>C</i>. <i>wenyujin</i> cultivated in different areas. In contrast, we observed that starch-related pathways were significantly up-regulated in the rhizomes of <i>C</i>. <i>wenyujin</i> cultivated in Haikou. These results are consistent with the findings of previous studies that have indicated that <i>C</i>. <i>wenyujin</i> rhizomes from Wenzhou have higher content of the volatile oil, terpene, curcumin, polysaccharide but lower starch content than those in <i>C</i>. <i>wenyujin</i> rhizomes from Haikou. We also found that the gene expression patterns were consistent with those determined based on transcriptome FPKM analysis (Figs <a href="#pone.0242776.g003">3</a> and <a href="#pone.0242776.g010">10</a>). Huang et al. have previously suggested that whereas differences in the starch content of <i>C</i>. <i>wenyujin</i> rhizomes from different producing areas are not appreciably influenced by latitude and longitude, they may be related to local soil, temperature, and light conditions, as well cultivation measures [<a href="#pone.0242776.ref064">64</a>]. Starch formation is a photosynthesis-related dark reaction involving a series of enzymatic reactions within the chloroplast matrix, which is influenced to varying degrees by temperature, CO<sub>2</sub> concentration, and pH. It can therefore be speculated that differences in these environmental factors between Wenzhou and Haikou contribute to the observed differences in the content of starch and/or other substances in <i>C</i>. <i>wenyujin</i> rhizomes. Similarly, gene expression analysis has revealed that the curcumin content of <i>C</i>. <i>wenyujin</i> rhizomes is markedly affected by environmental factors and changes in nutritional conditions [<a href="#pone.0242776.ref019">19</a>].</p> <p>Transcriptomics is concerned with analysis of the gene expression levels and diversity profiles of different tissues in animals and plants [<a href="#pone.0242776.ref065">65</a>â€"<a href="#pone.0242776.ref067">67</a>], one of the important focuses of which is the study of the biosynthesis of metabolites in non-model medicinal plants [<a href="#pone.0242776.ref068">68</a>, <a href="#pone.0242776.ref069">69</a>]. In the present study, we adopted hierarchical clustering to obtain complete transcriptional profiles of <i>C</i>. <i>wenyujin</i> cultivated in the Wenzhou and Haikou production areas, and accordingly found that the rhizomes of <i>C</i>. <i>wenyujin</i> grown in Wenzhou showed significant differences with respect to the transcriptional profiles of the co-expressed transcripts when compared with the rhizomes of plants grown in Haikou. Among these co-expressed transcripts, we found that when compared with those in Haikou rhizomes, the number of genes up-regulated in Wenzhou rhizomes was greater than those down-regulated. Functional analysis indicated that the enriched KEGG pathways appeared to be primarily associated with endocytosis and phagosome among cellular processes; ribosome, endoplasmic reticulum protein processing, RNA transport and spliceosome in genetic information processing; and carbon metabolism and biosynthesis of amino acids with respect to metabolism, thereby implying that the associated regulatory genes are significantly differentially expressed between the <i>C</i>. <i>wenyujin</i> rhizomes obtained from Wenzhou and Haikou.</p> <p>Overall, we found that 980 DEGs were allocated to KEGG pathways, among which those associated with spliceosome, endoplasmic reticulum protein processing, ribosome, ubiquitin-mediated proteolysis, RNA transport, phagosome, glyoxylate and dicarboxylate metabolism, and glutathione metabolism were predominantly enriched. With respect to up-regulated DEGs, pathways associated with spliceosome, carbon metabolism, ubiquitin-mediated proteolysis, proteasome, citrate cycle (TCA cycle), aminoacyl-tRNA biosynthesis, arginine and proline metabolism, and glutathione metabolism were predominantly enriched, whereas for the down-regulated DEGs, pathways relating to ribosome, starch and sucrose metabolism, endoplasmic reticulum protein processing, plant hormone signal transduction, protein export, and riboflavin metabolism were significantly enriched. These findings are thus indicative of the active biosynthesis of not only secondary but also primary metabolites. Conversely, we noted that plant hormone signal transduction and starch and sucrose metabolism were inhibited and down-regulated in the rhizomes of <i>C</i>. <i>wenyujin</i> cultivated in Wenzhou. These DEGs annotations will provide a valuable resource for characterizing the relative importance of different signal transduction pathways and specific genes in the rhizomes of <i>C</i>. <i>wenyujin</i> grown in both the traditional and recently established production areas [<a href="#pone.0242776.ref052">52</a>], which in turn could provide useful information for examining biosynthesis mechanisms of interest in <i>C</i>. <i>wenyujin</i> (<a href="#pone.0242776.s004">S4</a>â€"<a href="#pone.0242776.s007">S7</a> Figs).</p> <p>In the present study, we found that among all genes that were differentially expressed between Wenzhou and Haikou rhizomes, the GO terms of plastid and chloroplast (CC); methylerythritol 4-phosphate (MEP) pathway, isopentenyl diphosphate biosynthetic process, and thylakoid membrane organization (BP), and protein kinase activity and protein serine/threonine kinase activity (MF) were predominantly enriched in the rhizomes from Wenzhou compared with those from Haikou. Among up-regulated DEGs, the GO terms of pollen tube tip (CC), leaf morphogenesis (BP), and polyamine oxidase activity, isomerase activity, ion channel activity, and phospholipase activity (MF) were significantly enriched in Wenzhou rhizomes, whereas for down-regulated DEGs, the GO terms cell wall (CC), defense response to fungus (BP), and peroxidase activity, copper ion binding, sucrose synthase activity, heat shock protein binding, and catalytic activity (MF) were significantly enriched in Wenzhou rhizomes. We assume that these differences reflect the fact that when grown under different environmental conditions, <i>C</i>. <i>wenyujin</i> requires different metabolites and levels of energy to respond to the different stimuli associated with the prevailing conditions.</p> <p>When exposed to abiotic stresses, such as drought and elevated temperatures, plants respond by expressing a range transcription factors, induced via a series of signaling pathways. The stimulated transcription factors bind to corresponding <i>cis</i>-acting elements to initiate specific patterns of gene expression to counter the adverse effects of abiotic stress. Plant transcription factors known to respond to abiotic stress mainly include those in the <i>AP2/EREB</i>P, <i>MYB</i>, <i>WRKY</i>, <i>bZIP</i>, and <i>HSFs</i> [<a href="#pone.0242776.ref070">70</a>â€"<a href="#pone.0242776.ref075">75</a>]. <i>AP2/EREBP</i>, <i>MYB</i>, <i>WRKY</i>, <i>bZIP</i>, <i>HSFs</i>, <i>RLK</i>, <i>bHLH</i>, <i>NAC</i>, and <i>CAMK</i> families, the regulation of which is determined by a diverse range of biotic and abiotic pressures. plant growth and material metabolic processes, including trichome and seed coat development, embryogenesis leaf senescence, biosynthesis pathways, and the regulation of hormonal signals [<a href="#pone.0242776.ref076">76</a>â€"<a href="#pone.0242776.ref080">80</a>]. <i>C2H2</i> and <i>C2HC</i> zinc finger structures are characteristic features of the <i>WRKY</i> family of transcription factor from which the <i>WRKY</i> family can be explained [<a href="#pone.0242776.ref073">73</a>]. In the present study, we detected significantly up-regulated expression of transcription factors in the <i>RLK</i>, <i>C2H2</i>, <i>Zn-clus</i>, <i>TKL</i>, <i>bZIP</i>, <i>CAMK</i>, <i>AP2/ERF</i>, <i>AGC</i>, <i>STE</i>, <i>C2C2</i>, <i>CMGC</i>, <i>C3H</i>, <i>SNF2</i>, <i>WRKY</i>, <i>bHLH</i>, <i>GNAT</i>, <i>MYB</i>, <i>HSF</i>, and <i>NAC</i> families. Thus, on the basis of our findings that the genes of transcription factors known to respond to abiotic environmental factors were significantly up-regulated, we can speculate that observed differences in the compositions and gene regulation of <i>C</i>. <i>wenyujin</i> rhizomes derived from the traditional (Wenzhou) and recently established (Haikou) production areas, are probably attributable to differences in environmental factors specific to these two regions.</p> </div> <div class="conclusions" title="conclusions"> <div class="title" tagx="title" title="title">
Conclusions</div> <p>In summary, our findings indicate that the content of volatile oil, curcumin, and polysaccharides in the rhizomes of <i>C</i>. <i>wenyujin</i> cultivated in the Wenzhou production region is higher than those in the rhizomes of <i>C</i>. <i>wenyujin</i> grown in Haikou, whereas the content of starch is lower. We also detected significant differences in the constituents of the volatile oils of rhizomes obtained from the two production areas. Transcriptome analysis revealed notable differences in the gene expression patterns of rhizomes sourced from the Wenzhou and Haikou regions. We found that the expression of genes in pathways related to volatile oil, curcumin, and polysaccharide was significantly up-regulated in Wenzhou rhizomes, whereas that of starch-associated genes was significantly down-regulated, compared with those in Haikou rhizomes. Moreover, we identified the significantly up-regulated expression of transcription factors (including those in the <i>RLK</i>, <i>C2H2</i>, <i>bZIP</i>, <i>CAMK</i>, <i>AP2/ERF</i>, <i>WRKY</i>, <i>bHLH</i>, <i>MYB</i>, <i>HSF</i>, and <i>NAC</i> families) that are typically associated with responses to abiotic environmental stress. Collectively, the findings of this study provide important insights into the molecular mechanisms underlying differences in the metabolite profiles of <i>C</i>. <i>wenyujin</i>, which are assumed to reflect differences in environmental factor characterizing the two examined production regions. We also describe new methodologies that may prove beneficial with respect to evaluating the authenticity of Chinese herbal medicines.</p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supporting information</div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S1 Fig</span> <div class="title" tagx="title" title="title"><i>Curcuma</i> unigene length distribution.</div> <p>(TIF)</p>  <div class="media" title="media"><a href="pone.0242776.s001.tif">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S2 Fig</span> <div class="title" tagx="title" title="title">
Comparison of RNA-seq and quantitative reverse-transcription PCR (qRT-PCR) analyses of 15 selected genes.</div> <p>log2Fold Change [Wenzhou (WZ)/Haikou (HK)].</p> <p>(TIF)</p>  <div class="media" title="media"><a href="pone.0242776.s002.tif">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S3 Fig</span> <div class="title" tagx="title" title="title">
Transcription factor expression for Wenzhou (WZ) versus Haikou (HK).</div> <p>(TIF)</p>  <div class="media" title="media"><a href="pone.0242776.s003.tif">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S4 Fig</span> <div class="title" tagx="title" title="title">
Terpenoid backbone synthesis: KEGG enrichment structure for Wenzhou (WZ) versus Haikou (HK).</div> <p>(TIF)</p>  <div class="media" title="media"><a href="pone.0242776.s004.tif">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S5 Fig</span> <div class="title" tagx="title" title="title">
Flavonoid biosynthesis: KEGG enrichment structure for Wenzhou (WZ) versus Haikou (HK).</div> <p>(TIF)</p>  <div class="media" title="media"><a href="pone.0242776.s005.tif">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S6 Fig</span> <div class="title" tagx="title" title="title">
Amino sugar and nucleotide sugar metabolism: KEGG enrichment structure for Wenzhou (WZ) versus Haikou (HK).</div> <p>(TIF)</p>  <div class="media" title="media"><a href="pone.0242776.s006.tif">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S7 Fig</span> <div class="title" tagx="title" title="title">
Starch and sucrose metabolism: KEGG enrichment structure for Wenzhou (WZ) versus Haikou (HK).</div> <p>(TIF)</p>  <div class="media" title="media"><a href="pone.0242776.s007.tif">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S1 Table</span> <div class="title" tagx="title" title="title">
Environmental and weather conditions in Wenzhou and Haikou.</div> <p>(XLS)</p>  <div class="media" title="media"><a href="pone.0242776.s008.xls">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S2 Table</span> <div class="title" tagx="title" title="title">
RNA-seq verification using RT-PCR primers in <i>Curcuma wenyujin</i>.</div> <p>(XLS)</p>  <div class="media" title="media"><a href="pone.0242776.s009.xls">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S3 Table</span> <div class="title" tagx="title" title="title">
GC-MS detection results of the main volatile oil constituents in <i>Curcuma wenyujin</i> from Wenzhou (WZ) and Haikou (HK).</div> <p>(XLS)</p>  <div class="media" title="media"><a href="pone.0242776.s010.xls">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S4 Table</span> <div class="title" tagx="title" title="title">
Sample sequencing data evaluation statistics.</div> <p>(XLS)</p>  <div class="media" title="media"><a href="pone.0242776.s011.xls">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S5 Table</span> <div class="title" tagx="title" title="title">
Assembly result statistics.</div> <p>(XLS)</p>  <div class="media" title="media"><a href="pone.0242776.s012.xls">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S6 Table</span> <div class="title" tagx="title" title="title">
Comparison of sequencing data and assembly results.</div> <p>(XLS)</p>  <div class="media" title="media"><a href="pone.0242776.s013.xls">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S7 Table</span> <div class="title" tagx="title" title="title">
Unigene annotation statistics.</div> <p>(XLS)</p>  <div class="media" title="media"><a href="pone.0242776.s014.xls">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S8 Table</span> <div class="title" tagx="title" title="title">
Expression data of enriched key DEGs in different assemblies for Wenzhou (WZ) versus Haikou (HK).</div> <p>(XLS)</p>  <div class="media" title="media"><a href="pone.0242776.s015.xls">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S9 Table</span> <div class="title" tagx="title" title="title">
Details of topGO enrichment for Wenzhou (WZ) versus Haikou (HK).</div> <p>(XLS)</p>  <div class="media" title="media"><a href="pone.0242776.s016.xls">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S10 Table</span> <div class="title" tagx="title" title="title">
Details of KEGG enrichment for Wenzhou (WZ) versus Haikou (HK).</div> <p>(XLS)</p>  <div class="media" title="media"><a href="pone.0242776.s017.xls">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S11 Table</span> <div class="title" tagx="title" title="title">
Transcription factor expression for Wenzhou (WZ) versus Haikou (HK).</div> <p>(XLS)</p>  <div class="media" title="media"><a href="pone.0242776.s018.xls">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S1 File</span> <div class="title" tagx="title" title="title">
Enrichment analysis of responsive genes and transcripts using GO terms for Wenzhou (WZ) versus Haikou (HK).</div> <p>The node size is proportional to the number of targets in the GO category. Node color represents enriched significance; a deeper color represents a higher significance [a1, a2, and a3: total (topGO_BP, topGO_CC, and topGO_MF); b1, b2, and b3: down-regulated (topGO_BP, topGO_CC, and topGO_MF); c1, c2, and c3: up-regulated (topGO_BP, topGO_CC, and topGO_MF)].</p> <p>(ZIP)</p>  <div class="media" title="media"><a href="pone.0242776.s019.zip">LINK</a> <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <p>We would like to thank Editage (<a href="http://www.editage.cn">www.editage.cn</a>) for editing the English language in this manuscript.</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="pone.0242776.ref001" /><span class="label" tagx="label" title="label">1</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Committee</span><span class="given-names" tagx="given-names" title="given-names">CP</span></span>. <span class="source" tagx="source" title="source">Pharmacopoeia of the Peopleâ€™s Republic of China</span>. <span class="volume" tagx="volume" title="volume">Vol. I</span> ed. <span class="publisher-name" tagx="publisher-name" title="publisher-name">China Chemical Industry Press</span>: <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Beijing</span><span class="year" tagx="year" title="year">2015</span>; pp, <span class="fpage" tagx="fpage" title="fpage">234</span>â€"<span class="lpage" tagx="lpage" title="lpage">239</span>.</span></li> <li tag="ref"><a name="pone.0242776.ref002" /><span class="label" tagx="label" title="label">2</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nie</span><span class="given-names" tagx="given-names" title="given-names">XH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ao</span><span class="given-names" tagx="given-names" title="given-names">ZH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yin</span><span class="given-names" tagx="given-names" title="given-names">GY</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tao</span><span class="given-names" tagx="given-names" title="given-names">WY</span></span>. <span class="mixed-article-title" title="mixed-article-title">Effect of Extraction Techniqueson the Components and Antitumor Activity in Vitro of Volatile Oil from <i>Curcuma Wenyujin</i></span>. <span class="source" tagx="source" title="source">Pharmaceutical Biotechnology</span>. <span class="year" tagx="year" title="year">2003</span>; <span class="volume" tagx="volume" title="volume">10</span>: <span class="fpage" tagx="fpage" title="fpage">152</span>â€"<span class="lpage" tagx="lpage" title="lpage">154</span>.</span></li> <li tag="ref"><a name="pone.0242776.ref003" /><span class="label" tagx="label" title="label">3</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hong</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bi</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>. <span class="mixed-article-title" title="mixed-article-title">GC-MS method for determination and pharmacokinetic study of seven volatile constituents in rat plasma after oral administration of the essential oil of <i>Rhizoma Curcumae</i></span>. <span class="source" tagx="source" title="source">Journal of Pharmaceutical &amp;amp; Biomedical Analysis</span>. <span class="year" tagx="year" title="year">2017</span>; <span class="volume" tagx="volume" title="volume">149</span>: <span class="fpage" tagx="fpage" title="fpage">577</span>â€"<span class="lpage" tagx="lpage" title="lpage">585</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jpba.2017.11.058">10.1016/j.jpba.2017.11.058</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29197300">29197300</a></span></span></li> <li tag="ref"><a name="pone.0242776.ref004" /><span class="label" tagx="label" title="label">4</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lobo</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prabhu</span><span class="given-names" tagx="given-names" title="given-names">KS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shirwaikar</span><span class="given-names" tagx="given-names" title="given-names">DA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shirwaikar</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>. <span class="mixed-article-title" title="mixed-article-title"><i>Curcuma zedoaria Rosc</i>. (white turmeric): a review of its chemical, pharmacological and ethnomedicinal properties</span>. <span class="source" tagx="source" title="source">J Pharm Pharmacol</span>. <span class="year" tagx="year" title="year">2009</span>; <span class="volume" tagx="volume" title="volume">61</span>(<span class="issue" tagx="issue" title="issue">1</span>): <span class="fpage" tagx="fpage" title="fpage">13</span>â€"<span class="lpage" tagx="lpage" title="lpage">21</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1211/jpp/61.01.0003">10.1211/jpp/61.01.0003</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19126292">19126292</a></span></span></li> <li tag="ref"><a name="pone.0242776.ref005" /><span class="label" tagx="label" title="label">5</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dang</span><span class="given-names" tagx="given-names" title="given-names">YY</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">XC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">QW</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">SP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">YT</span></span>. <span class="mixed-article-title" title="mixed-article-title">Preparative isolation and purification of six volatile compounds from essential oil of <i>Curcuma wenyujin</i> using high-performance centrifugal partition chromatography</span>. <span class="source" tagx="source" title="source">J Sep Sci</span>. <span class="year" tagx="year" title="year">2010</span>; <span class="volume" tagx="volume" title="volume">33</span>(<span class="issue" tagx="issue" title="issue">11</span>): <span class="fpage" tagx="fpage" title="fpage">1658</span>â€"<span class="lpage" tagx="lpage" title="lpage">1664</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1002/jssc.200900453">10.1002/jssc.200900453</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20405479">20405479</a></span></span></li> <li tag="ref"><a name="pone.0242776.ref006" /><span class="label" tagx="label" title="label">6</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zheng</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lu</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Initial study on naturally occurring products from traditional Chinese herbs and vegetables for chemoprevention</span>. <span class="source" tagx="source" title="source">J Cell Biochem Suppl</span>. <span class="year" tagx="year" title="year">1997</span>; <span class="volume" tagx="volume" title="volume">67</span>(<span class="issue" tagx="issue" title="issue">S27</span>): <span class="fpage" tagx="fpage" title="fpage">106</span>â€"<span class="lpage" tagx="lpage" title="lpage">112</span>. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9591200">9591200</a></span></span></li> <li tag="ref"><a name="pone.0242776.ref007" /><span class="label" tagx="label" title="label">7</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lv</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cao</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dong</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lou</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chai</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>. <span class="mixed-article-title" title="mixed-article-title">Analysis and pharmacokinetic study of curdione in <i>Rhizoma Curcumae</i> by UPLC/QTOF/MS</span>. <span class="source" tagx="source" title="source">Biomed Chromatogr</span>. <span class="year" tagx="year" title="year">2014</span>; <span class="volume" tagx="volume" title="volume">28</span>(<span class="issue" tagx="issue" title="issue">6</span>): <span class="fpage" tagx="fpage" title="fpage">782</span>â€"<span class="lpage" tagx="lpage" title="lpage">787</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1002/bmc.3118">10.1002/bmc.3118</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24861744">24861744</a></span></span></li> <li tag="ref"><a name="pone.0242776.ref008" /><span class="label" tagx="label" title="label">8</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Quan</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kui</span><span class="given-names" tagx="given-names" title="given-names">JZ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">GH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ping</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tina</span><span class="given-names" tagx="given-names" title="given-names">TXD</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shao</span><span class="given-names" tagx="given-names" title="given-names">PL</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Molecular genetic and chemical assessment of Rhizoma Curcumae in China</span>. <span class="source" tagx="source" title="source">J Agric Food Chem</span>. <span class="year" tagx="year" title="year">2005</span>; <span class="volume" tagx="volume" title="volume">53</span>(<span class="issue" tagx="issue" title="issue">15</span>): <span class="fpage" tagx="fpage" title="fpage">6019</span>â€"<span class="lpage" tagx="lpage" title="lpage">6026</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1021/jf0508495">10.1021/jf0508495</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16028990">16028990</a></span></span></li> <li tag="ref"><a name="pone.0242776.ref009" /><span class="label" tagx="label" title="label">9</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xie</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Song</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">WP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">HR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tian</span><span class="given-names" tagx="given-names" title="given-names">YX</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Two traditional Chinese medicines Curcumae radix and Curcumae rhizoma: an ethnopharmacology, phytochemistry, and pharmacology review</span>. <span class="source" tagx="source" title="source">Evid. Based Complement. Alternat. Med</span>. <span class="year" tagx="year" title="year">2016</span>; <span class="volume" tagx="volume" title="volume">3</span>: <span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">30</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1155/2016/4973128">10.1155/2016/4973128</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27057197">27057197</a></span></span></li> <li tag="ref"><a name="pone.0242776.ref010" /><span class="label" tagx="label" title="label">10</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Variation in Essential Oil and Bioactive Compounds of <i>Curcuma kwangsiensis</i> Collected from Natural Habitats</span>. <span class="source" tagx="source" title="source">Chemistry &amp;amp; Biodiversity</span>. <span class="year" tagx="year" title="year">2017</span>; <span class="volume" tagx="volume" title="volume">14</span>(<span class="issue" tagx="issue" title="issue">7</span>): <span class="fpage" tagx="fpage" title="fpage">e1700020</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/cbdv.201700020">10.1002/cbdv.201700020</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28398606">28398606</a></span></span></li> <li tag="ref"><a name="pone.0242776.ref011" /><span class="label" tagx="label" title="label">11</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wei</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Su</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pan</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Contrastive analysis of chemical composition of essential oil from twelve <i>Curcuma</i> species distributed in China</span>. <span class="source" tagx="source" title="source">Industrial Crops &amp;amp; Products</span>. <span class="year" tagx="year" title="year">2017</span>; <span class="volume" tagx="volume" title="volume">108</span>: <span class="fpage" tagx="fpage" title="fpage">17</span>â€"<span class="lpage" tagx="lpage" title="lpage">25</span>.</span></li> <li tag="ref"><a name="pone.0242776.ref012" /><span class="label" tagx="label" title="label">12</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">YT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">ZQ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">SC</span></span>. <span class="mixed-article-title" title="mixed-article-title">Determination of Î²-element, curcumol, germacrone and curdione of Zedoary turmeric oil by GC</span>. <span class="source" tagx="source" title="source">Chinese Journal of Pharmaceutical Analysis</span>. <span class="year" tagx="year" title="year">2009</span>; <span class="volume" tagx="volume" title="volume">29</span>: <span class="fpage" tagx="fpage" title="fpage">1832</span>â€"<span class="lpage" tagx="lpage" title="lpage">1836</span>.</span></li> <li tag="ref"><a name="pone.0242776.ref013" /><span class="label" tagx="label" title="label">13</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mao</span><span class="given-names" tagx="given-names" title="given-names">CQ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lu</span><span class="given-names" tagx="given-names" title="given-names">TL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiang</span><span class="given-names" tagx="given-names" title="given-names">GF</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Su</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cai</span><span class="given-names" tagx="given-names" title="given-names">BC</span></span>. <span class="mixed-article-title" title="mixed-article-title">Determination of curdione, curcumol, germacrone, and Î²-elemene in processed <i>Curcumae Rhizoma</i> from different habitats</span>. <span class="source" tagx="source" title="source">Chinese Traditional &amp;amp; Herbal Drugs</span>. <span class="year" tagx="year" title="year">2013</span>; <span class="volume" tagx="volume" title="volume">44</span>(<span class="issue" tagx="issue" title="issue">3</span>): <span class="fpage" tagx="fpage" title="fpage">305</span>â€"<span class="lpage" tagx="lpage" title="lpage">308</span>.</span></li> <li tag="ref"><a name="pone.0242776.ref014" /><span class="label" tagx="label" title="label">14</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Manzan</span><span class="given-names" tagx="given-names" title="given-names">AC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Toniolo</span><span class="given-names" tagx="given-names" title="given-names">FS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bredow</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Povh</span><span class="given-names" tagx="given-names" title="given-names">NP</span></span>. <span class="mixed-article-title" title="mixed-article-title">Extraction of essential oil and pigments from <i>Curcuma longa</i> L. by steam distillation and extraction with volatile solvents</span>. <span class="source" tagx="source" title="source">J. Agric. Food Chem</span>. <span class="year" tagx="year" title="year">2003</span>; <span class="volume" tagx="volume" title="volume">51</span>, <span class="fpage" tagx="fpage" title="fpage">6802</span>â€"<span class="lpage" tagx="lpage" title="lpage">6807</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1021/jf030161x">10.1021/jf030161x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14582978">14582978</a></span></span></li> <li tag="ref"><a name="pone.0242776.ref015" /><span class="label" tagx="label" title="label">15</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mehla</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reeta</span><span class="given-names" tagx="given-names" title="given-names">KH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gupta</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gupta</span><span class="given-names" tagx="given-names" title="given-names">YK</span></span>. <span class="mixed-article-title" title="mixed-article-title">Protective effect of curcumin against seizures and cognitive impairment in a pentylenetetrazole-kindled epileptic rat model</span>. <span class="source" tagx="source" title="source">Life Sci</span>. <span class="year" tagx="year" title="year">2010</span>; <span class="volume" tagx="volume" title="volume">87</span>(<span class="issue" tagx="issue" title="issue">19â€"22</span>): <span class="fpage" tagx="fpage" title="fpage">596</span>â€"<span class="lpage" tagx="lpage" title="lpage">603</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.lfs.2010.09.006">10.1016/j.lfs.2010.09.006</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20840851">20840851</a></span></span></li> <li tag="ref"><a name="pone.0242776.ref016" /><span class="label" tagx="label" title="label">16</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">FQ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">YT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">SP</span></span>. <span class="mixed-article-title" title="mixed-article-title">Simultaneous determination of 11 characteristic components in three species of <i>Curcuma rhizomes</i> using pressurized liquid extraction and high-performance liquid chromatography</span>. <span class="source" tagx="source" title="source">Journal of Chromatography A</span>. <span class="year" tagx="year" title="year">2006</span>; <span class="volume" tagx="volume" title="volume">1134</span>(<span class="issue" tagx="issue" title="issue">1</span>): <span class="fpage" tagx="fpage" title="fpage">226</span>â€"<span class="lpage" tagx="lpage" title="lpage">231</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.chroma.2006.09.048">10.1016/j.chroma.2006.09.048</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17018231">17018231</a></span></span></li> <li tag="ref"><a name="pone.0242776.ref017" /><span class="label" tagx="label" title="label">17</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">ZL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">NN</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">CM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Du</span><span class="given-names" tagx="given-names" title="given-names">SS</span></span>. <span class="mixed-article-title" title="mixed-article-title">Identification of insecticidal constituents of the essential oil of <i>Curcuma wenyujin</i> rhizomes active against <i>Liposcelis bostrychophila Badonnel</i></span>. <span class="source" tagx="source" title="source">Molecules</span>. <span class="year" tagx="year" title="year">2012</span>; <span class="volume" tagx="volume" title="volume">17</span>, <span class="fpage" tagx="fpage" title="fpage">12049</span>â€"<span class="lpage" tagx="lpage" title="lpage">12060</span>. <span class="pub-id"><a href="https://dx.doi.org/10.3390/molecules171012049">10.3390/molecules171012049</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23085655">23085655</a></span></span></li> <li tag="ref"><a name="pone.0242776.ref018" /><span class="label" tagx="label" title="label">18</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ono</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sato</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sugiura</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Altaf-Ul-Amin</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ohta</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Targeted Integration of RNA-Seq and Metabolite Data to Elucidate Curcuminoid Biosynthesis in Four Curcuma Species</span>. <span class="source" tagx="source" title="source">Plant Cell Physiol</span>. <span class="year" tagx="year" title="year">2015</span>; <span class="volume" tagx="volume" title="volume">56</span>(<span class="issue" tagx="issue" title="issue">5</span>): <span class="fpage" tagx="fpage" title="fpage">843</span>â€"<span class="lpage" tagx="lpage" title="lpage">851</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/pcp/pcv008">10.1093/pcp/pcv008</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25637373">25637373</a></span></span></li> <li tag="ref"><a name="pone.0242776.ref019" /><span class="label" tagx="label" title="label">19</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Inavolu</span><span class="given-names" tagx="given-names" title="given-names">SS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sanghamitra</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sujata</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>. <span class="mixed-article-title" title="mixed-article-title">Differential expression of <i>curcumin</i> synthase at nine agroclimatic zones as influenced by environment and nutritional variation</span>. <span class="source" tagx="source" title="source">Journal of Biotechnology</span>. <span class="year" tagx="year" title="year">2016</span>; <span class="volume" tagx="volume" title="volume">231</span>: <span class="fpage" tagx="fpage" title="fpage">S37</span>â€"<span class="lpage" tagx="lpage" title="lpage">S37</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jbiotec.2016.05.147">10.1016/j.jbiotec.2016.05.147</a></span></span></li> <li tag="ref"><a name="pone.0242776.ref020" /><span class="label" tagx="label" title="label">20</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Givnish</span><span class="given-names" tagx="given-names" title="given-names">TJ</span></span>. <span class="mixed-article-title" title="mixed-article-title">Ecological constraints on the evolution of plasticity in plants</span>. <span class="source" tagx="source" title="source">Evolutionary Ecology</span>. <span class="year" tagx="year" title="year">2002</span>; <span class="volume" tagx="volume" title="volume">16</span>(<span class="issue" tagx="issue" title="issue">3</span>): <span class="fpage" tagx="fpage" title="fpage">213</span>â€"<span class="lpage" tagx="lpage" title="lpage">242</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1023/A:1019676410041">10.1023/A:1019676410041</a></span></span></li> <li tag="ref"><a name="pone.0242776.ref021" /><span class="label" tagx="label" title="label">21</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Des Marais</span><span class="given-names" tagx="given-names" title="given-names">DL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Juenger</span><span class="given-names" tagx="given-names" title="given-names">TE</span></span>. <span class="mixed-article-title" title="mixed-article-title">Pleiotropy, plasticity, and the evolution of plant abiotic stress tolerance</span>. <span class="source" tagx="source" title="source">Ann N Y Acad Sci</span>. <span class="year" tagx="year" title="year">2010</span>; <span class="volume" tagx="volume" title="volume">1206</span>: <span class="fpage" tagx="fpage" title="fpage">56</span>â€"<span class="lpage" tagx="lpage" title="lpage">79</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1749-6632.2010.05703.x">10.1111/j.1749-6632.2010.05703.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20860683">20860683</a></span></span></li> <li tag="ref"><a name="pone.0242776.ref022" /><span class="label" tagx="label" title="label">22</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sultan</span><span class="given-names" tagx="given-names" title="given-names">SE</span></span>. <span class="mixed-article-title" title="mixed-article-title">Phenotypic plasticity for plant development, function and life history</span>. <span class="source" tagx="source" title="source">Trends Plant Sci</span>. <span class="year" tagx="year" title="year">2000</span>; <span class="volume" tagx="volume" title="volume">5</span>: <span class="fpage" tagx="fpage" title="fpage">537</span>â€"<span class="lpage" tagx="lpage" title="lpage">542</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/s1360-1385(00)01797-0">10.1016/s1360-1385(00)01797-0</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11120476">11120476</a></span></span></li> <li tag="ref"><a name="pone.0242776.ref023" /><span class="label" tagx="label" title="label">23</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shao</span><span class="given-names" tagx="given-names" title="given-names">HB</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guo</span><span class="given-names" tagx="given-names" title="given-names">QJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chu</span><span class="given-names" tagx="given-names" title="given-names">LY</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">XN</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Su</span><span class="given-names" tagx="given-names" title="given-names">ZL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">YC</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Understanding molecular mechanism of higher plant plasticity under abiotic stress</span>. <span class="source" tagx="source" title="source">Colloids Surf B Biointerfaces</span>. <span class="year" tagx="year" title="year">2007</span>; <span class="volume" tagx="volume" title="volume">54</span>(<span class="issue" tagx="issue" title="issue">1</span>): <span class="fpage" tagx="fpage" title="fpage">37</span>â€"<span class="lpage" tagx="lpage" title="lpage">45</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.colsurfb.2006.07.002">10.1016/j.colsurfb.2006.07.002</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16914294">16914294</a></span></span></li> <li tag="ref"><a name="pone.0242776.ref024" /><span class="label" tagx="label" title="label">24</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">CX</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">MQ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gao</span><span class="given-names" tagx="given-names" title="given-names">XX</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">LN</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">XF</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">JH</span></span>. <span class="mixed-article-title" title="mixed-article-title">Metabolic response of maize plants to multi-factorial abiotic stresses</span>. <span class="source" tagx="source" title="source">Plant Biology</span>. <span class="year" tagx="year" title="year">2015</span>; <span class="volume" tagx="volume" title="volume">18</span>(<span class="issue" tagx="issue" title="issue">Suppl 1</span>): <span class="fpage" tagx="fpage" title="fpage">120</span>â€"<span class="lpage" tagx="lpage" title="lpage">9</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/plb.12305">10.1111/plb.12305</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25622534">25622534</a></span></span></li> <li tag="ref"><a name="pone.0242776.ref025" /><span class="label" tagx="label" title="label">25</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nicotra</span><span class="given-names" tagx="given-names" title="given-names">AB</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Atkin</span><span class="given-names" tagx="given-names" title="given-names">OK</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bonser</span><span class="given-names" tagx="given-names" title="given-names">SP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Davidson</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Finnegan</span><span class="given-names" tagx="given-names" title="given-names">EJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mathesius</span><span class="given-names" tagx="given-names" title="given-names">U</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Plant phenotypic plasticity in a changing climate</span>. <span class="source" tagx="source" title="source">Trends Plant Sci</span>. <span class="year" tagx="year" title="year">2010</span>; <span class="volume" tagx="volume" title="volume">15</span>, <span class="fpage" tagx="fpage" title="fpage">684</span>â€"<span class="lpage" tagx="lpage" title="lpage">692</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.tplants.2010.09.008">10.1016/j.tplants.2010.09.008</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20970368">20970368</a></span></span></li> <li tag="ref"><a name="pone.0242776.ref026" /><span class="label" tagx="label" title="label">26</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lin</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shao</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luo</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>. <span class="mixed-article-title" title="mixed-article-title">Modelling the effects of soil water potential on growth and quality of cut chrysanthemum (Chrysanthemum morifolium)</span>. <span class="source" tagx="source" title="source">Scientia Horticulturae</span>. <span class="year" tagx="year" title="year">2011</span>; <span class="volume" tagx="volume" title="volume">130</span>(<span class="issue" tagx="issue" title="issue">1</span>): <span class="fpage" tagx="fpage" title="fpage">275</span>â€"<span class="lpage" tagx="lpage" title="lpage">288</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.scienta.2011.06.008">10.1016/j.scienta.2011.06.008</a></span></span></li> <li tag="ref"><a name="pone.0242776.ref027" /><span class="label" tagx="label" title="label">27</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mekapogu</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sohn</span><span class="given-names" tagx="given-names" title="given-names">HB</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">YY</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Park</span><span class="given-names" tagx="given-names" title="given-names">HM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jin</span><span class="given-names" tagx="given-names" title="given-names">YI</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Effect of Light Quality on the Expression of Glycoalkaloid Biosynthetic Genes Contributing to Steroidal Glycoalkaloid Accumulation in Potato</span>. <span class="source" tagx="source" title="source">American Journal of Potato Research</span>. <span class="year" tagx="year" title="year">2016</span>; <span class="volume" tagx="volume" title="volume">93</span>(<span class="issue" tagx="issue" title="issue">3</span>): <span class="fpage" tagx="fpage" title="fpage">264</span>â€"<span class="lpage" tagx="lpage" title="lpage">277</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1007/s12230-016-9502-z">10.1007/s12230-016-9502-z</a></span></span></li> <li tag="ref"><a name="pone.0242776.ref028" /><span class="label" tagx="label" title="label">28</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">He</span><span class="given-names" tagx="given-names" title="given-names">XY</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cao</span><span class="given-names" tagx="given-names" title="given-names">JH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lu</span><span class="given-names" tagx="given-names" title="given-names">MG</span></span>. <span class="mixed-article-title" title="mixed-article-title">Experimentation of <i>Curcuma wenyujin</i> in different soil conditions</span>. <span class="source" tagx="source" title="source">Chinese Journal of Eco-Agriculture</span>. <span class="year" tagx="year" title="year">2007</span>; <span class="volume" tagx="volume" title="volume">15</span>: <span class="fpage" tagx="fpage" title="fpage">98</span>â€"<span class="lpage" tagx="lpage" title="lpage">101</span></span></li> <li tag="ref"><a name="pone.0242776.ref029" /><span class="label" tagx="label" title="label">29</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dick</span><span class="given-names" tagx="given-names" title="given-names">CA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Buenrostro</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Butler</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carlson</span><span class="given-names" tagx="given-names" title="given-names">ML</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kliebenstein</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Whittall</span><span class="given-names" tagx="given-names" title="given-names">JB</span></span>. <span class="mixed-article-title" title="mixed-article-title">Arctic mustard flower color polymorphism controlled by petal-specific downregulation at the threshold of the anthocyanin biosynthetic pathway</span>. <span class="source" tagx="source" title="source">PLoS One</span>. <span class="year" tagx="year" title="year">2011</span>; <span class="volume" tagx="volume" title="volume">6</span>(<span class="issue" tagx="issue" title="issue">4</span>): <span class="fpage" tagx="fpage" title="fpage">e18230</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0018230">10.1371/journal.pone.0018230</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21490971">21490971</a></span></span></li> <li tag="ref"><a name="pone.0242776.ref030" /><span class="label" tagx="label" title="label">30</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gershenzon</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>. <span class="chapter-title" tagx="chapter-title" title="chapter-title">Changes in the Levels of Plant Secondary Metabolites Under Water and Nutrient Stress</span> In: <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Timmermann</span><span class="given-names" tagx="given-names" title="given-names">B.N.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Steelink</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Loewus</span><span class="given-names" tagx="given-names" title="given-names">F.A</span></span>. (eds) <span class="source" tagx="source" title="source">Phytochemical Adaptations to Stress. Recent Advances in Phytochemistry (Proceedings of the Phytochemical Society of North America)</span>, <span class="publisher-name" tagx="publisher-name" title="publisher-name">Springer</span>, <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Boston, MA</span><span class="year" tagx="year" title="year">1984</span>; <span class="volume" tagx="volume" title="volume">18</span>: <span class="fpage" tagx="fpage" title="fpage">273</span>â€"<span class="lpage" tagx="lpage" title="lpage">320</span></span></li> <li tag="ref"><a name="pone.0242776.ref031" /><span class="label" tagx="label" title="label">31</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Winkel-Shirley</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>. <span class="mixed-article-title" title="mixed-article-title">Flavonoid Biosynthesis. A Colorful Model for Genetics, biochemistry, cell biology, and biotechnology</span>. <span class="source" tagx="source" title="source">Plant Physiol</span>. <span class="year" tagx="year" title="year">2001</span>; <span class="volume" tagx="volume" title="volume">126</span>: <span class="fpage" tagx="fpage" title="fpage">485</span>â€"<span class="lpage" tagx="lpage" title="lpage">493</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1104/pp.126.2.485">10.1104/pp.126.2.485</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11402179">11402179</a></span></span></li> <li tag="ref"><a name="pone.0242776.ref032" /><span class="label" tagx="label" title="label">32</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lillo</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lea</span><span class="given-names" tagx="given-names" title="given-names">US</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ruoff</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>. <span class="mixed-article-title" title="mixed-article-title">Nutrient depletion as a key factor for manipulating gene expression and product formation in different branches of the flavonoid pathway</span>. <span class="source" tagx="source" title="source">Plant Cell Environ</span>. <span class="year" tagx="year" title="year">2008</span>; <span class="volume" tagx="volume" title="volume">31</span>(<span class="issue" tagx="issue" title="issue">5</span>): <span class="fpage" tagx="fpage" title="fpage">587</span>â€"<span class="lpage" tagx="lpage" title="lpage">601</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1365-3040.2007.01748.x">10.1111/j.1365-3040.2007.01748.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18031469">18031469</a></span></span></li> <li tag="ref"><a name="pone.0242776.ref033" /><span class="label" tagx="label" title="label">33</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Misson</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Raghothama</span><span class="given-names" tagx="given-names" title="given-names">KG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jain</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jouhet</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Block</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bligny</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">A genome-wide transcriptional analysis using Arabidopsis thaliana Affymetrix gene chips determined plant responses to phosphate deprivation</span>. <span class="source" tagx="source" title="source">Proc Natl Acad Sci USA</span>. <span class="year" tagx="year" title="year">2005</span>; <span class="volume" tagx="volume" title="volume">102</span>, <span class="fpage" tagx="fpage" title="fpage">11934</span>â€"<span class="lpage" tagx="lpage" title="lpage">11939</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.0505266102">10.1073/pnas.0505266102</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16085708">16085708</a></span></span></li> <li tag="ref"><a name="pone.0242776.ref034" /><span class="label" tagx="label" title="label">34</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vanderauwera</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zimmermann</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rombauts</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vandenabeele</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Langebartels</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gruissem</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Genome-wide analysis of hydrogen peroxide-regulated gene expression in Arabidopsis reveals a high light-induced transcriptional cluster involved in anthocyanin biosynthesis</span>. <span class="source" tagx="source" title="source">Plant Physiol</span>. <span class="year" tagx="year" title="year">2005</span>; <span class="volume" tagx="volume" title="volume">139</span>(<span class="issue" tagx="issue" title="issue">2</span>): <span class="fpage" tagx="fpage" title="fpage">806</span>â€"<span class="lpage" tagx="lpage" title="lpage">821</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1104/pp.105.065896">10.1104/pp.105.065896</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16183842">16183842</a></span></span></li> <li tag="ref"><a name="pone.0242776.ref035" /><span class="label" tagx="label" title="label">35</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Koyama</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ikeda</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Poudel</span><span class="given-names" tagx="given-names" title="given-names">PR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goto-Yamamoto</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>. <span class="mixed-article-title" title="mixed-article-title">Light quality affects flavonoid biosynthesis in young berries of Cabernet Sauvignon grape</span>. <span class="source" tagx="source" title="source">Phytochemistry</span>. <span class="year" tagx="year" title="year">2012</span>; <span class="volume" tagx="volume" title="volume">78</span>: <span class="fpage" tagx="fpage" title="fpage">54</span>â€"<span class="lpage" tagx="lpage" title="lpage">64</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.phytochem.2012.02.026">10.1016/j.phytochem.2012.02.026</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22455871">22455871</a></span></span></li> <li tag="ref"><a name="pone.0242776.ref036" /><span class="label" tagx="label" title="label">36</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Feng</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">SS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">WJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">LY</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">JY</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Regulation of Different Growth Temperatures on Flavonoids Content and Their Metabolic Pathways in Tobacco Leaves</span>. <span class="source" tagx="source" title="source">Genomics &amp;amp; Applied Biology</span>. <span class="year" tagx="year" title="year">2016</span>; <span class="volume" tagx="volume" title="volume">12</span>: <span class="fpage" tagx="fpage" title="fpage">3527</span>â€"<span class="lpage" tagx="lpage" title="lpage">3537</span></span></li> <li tag="ref"><a name="pone.0242776.ref037" /><span class="label" tagx="label" title="label">37</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiang</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>. <span class="source" tagx="source" title="source">Effects of water factors on flavonoid metabolism of Astragalus</span>. <span class="publisher-name" tagx="publisher-name" title="publisher-name">Jilin Agricultural University</span><span class="year" tagx="year" title="year">2013</span>; pp, <span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">125</span></span></li> <li tag="ref"><a name="pone.0242776.ref038" /><span class="label" tagx="label" title="label">38</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lea</span><span class="given-names" tagx="given-names" title="given-names">US</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Slimestad</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smedvig</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lillo</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>. <span class="mixed-article-title" title="mixed-article-title">Nitrogen deficiency enhances expression of specific MYB and bHLH transcription factors and accumulation of end products in the flavonoid pathway</span>. <span class="source" tagx="source" title="source">Planta</span>. <span class="year" tagx="year" title="year">2007</span>; <span class="volume" tagx="volume" title="volume">225</span>(<span class="issue" tagx="issue" title="issue">5</span>): <span class="fpage" tagx="fpage" title="fpage">1245</span>â€"<span class="lpage" tagx="lpage" title="lpage">1253</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1007/s00425-006-0414-x">10.1007/s00425-006-0414-x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17053893">17053893</a></span></span></li> <li tag="ref"><a name="pone.0242776.ref039" /><span class="label" tagx="label" title="label">39</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pan</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">CT</span></span>. <span class="mixed-article-title" title="mixed-article-title">Composition analysis of essential oil of <i>Curcuma wenyujin</i> Rhizoma from Hainan Province</span>. <span class="source" tagx="source" title="source">Hainan Medical Journal</span>. <span class="year" tagx="year" title="year">2014</span>; <span class="volume" tagx="volume" title="volume">24</span>(<span class="issue" tagx="issue" title="issue">20</span>): <span class="fpage" tagx="fpage" title="fpage">3024</span>â€"<span class="lpage" tagx="lpage" title="lpage">3027</span>.</span></li> <li tag="ref"><a name="pone.0242776.ref040" /><span class="label" tagx="label" title="label">40</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Annadurai</span><span class="given-names" tagx="given-names" title="given-names">RS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Neethiraj</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jayakumar</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Damodaran</span><span class="given-names" tagx="given-names" title="given-names">AC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rao</span><span class="given-names" tagx="given-names" title="given-names">SN</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Katta</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">De Novo Transcriptome Assembly (NGS) of Curcuma longa L. Rhizome Reveals Novel Transcripts Related to Anticancer and Antimalarial Terpenoids</span>. <span class="source" tagx="source" title="source">PLoS ONE</span>, <span class="year" tagx="year" title="year">2013</span>; <span class="volume" tagx="volume" title="volume">8</span>(<span class="issue" tagx="issue" title="issue">2</span>): <span class="fpage" tagx="fpage" title="fpage">e56217</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0056217">10.1371/journal.pone.0056217</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23468859">23468859</a></span></span></li> <li tag="ref"><a name="pone.0242776.ref041" /><span class="label" tagx="label" title="label">41</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prasath</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Karthika</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Habeeba</span><span class="given-names" tagx="given-names" title="given-names">NT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suraby</span><span class="given-names" tagx="given-names" title="given-names">EJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rosana</span><span class="given-names" tagx="given-names" title="given-names">OB</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shaji</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Comparison of the transcriptomes of ginger (<i>Zingiber officinale Rosc</i>.) and mango ginger (<i>Curcuma amada</i> Roxb.) in response to the bacterial wilt infection</span>. <span class="source" tagx="source" title="source">PLoS One</span>. <span class="year" tagx="year" title="year">2014</span>; <span class="volume" tagx="volume" title="volume">9</span>(<span class="issue" tagx="issue" title="issue">6</span>): <span class="fpage" tagx="fpage" title="fpage">e99731</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0099731">10.1371/journal.pone.0099731</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24940878">24940878</a></span></span></li> <li tag="ref"><a name="pone.0242776.ref042" /><span class="label" tagx="label" title="label">42</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sahoo</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kar</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sahoo</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ray</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nayak</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>. <span class="mixed-article-title" title="mixed-article-title">Transcriptome profiling of <i>Curcuma longa</i> L. cv. Suvarna</span>. <span class="source" tagx="source" title="source">Genom Data</span>. <span class="year" tagx="year" title="year">2016</span>; <span class="volume" tagx="volume" title="volume">10</span>: <span class="fpage" tagx="fpage" title="fpage">33</span>â€"<span class="lpage" tagx="lpage" title="lpage">34</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.gdata.2016.09.001">10.1016/j.gdata.2016.09.001</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27668184">27668184</a></span></span></li> <li tag="ref"><a name="pone.0242776.ref043" /><span class="label" tagx="label" title="label">43</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grabherr</span><span class="given-names" tagx="given-names" title="given-names">MG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haas</span><span class="given-names" tagx="given-names" title="given-names">BJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yassour</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Levin</span><span class="given-names" tagx="given-names" title="given-names">JZ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thompson</span><span class="given-names" tagx="given-names" title="given-names">DA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Amit</span><span class="given-names" tagx="given-names" title="given-names">I</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Full-length transcriptome assembly from RNA-Seq data without a reference genome</span>. <span class="source" tagx="source" title="source">Nat Biotechnol</span>. <span class="year" tagx="year" title="year">2011</span>; <span class="volume" tagx="volume" title="volume">29</span>(<span class="issue" tagx="issue" title="issue">7</span>): <span class="fpage" tagx="fpage" title="fpage">644</span>â€"<span class="lpage" tagx="lpage" title="lpage">652</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/nbt.1883">10.1038/nbt.1883</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21572440">21572440</a></span></span></li> <li tag="ref"><a name="pone.0242776.ref044" /><span class="label" tagx="label" title="label">44</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheng</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qiang</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qiang</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dai</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hong</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jian</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Transcriptional profiling provides new insights into the role of nitric oxide in enhancing Ganoderma oregonense resistance to heat stress</span>. <span class="source" tagx="source" title="source">Sci Rep</span>. <span class="year" tagx="year" title="year">2017</span>; <span class="volume" tagx="volume" title="volume">7</span>(<span class="issue" tagx="issue" title="issue">1</span>): <span class="fpage" tagx="fpage" title="fpage">15694</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41598-017-15340-6">10.1038/s41598-017-15340-6</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29146915">29146915</a></span></span></li> <li tag="ref"><a name="pone.0242776.ref045" /><span class="label" tagx="label" title="label">45</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bao</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xing</span><span class="given-names" tagx="given-names" title="given-names">YN</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiang</span><span class="given-names" tagx="given-names" title="given-names">HB</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">XD</span></span>. <span class="mixed-article-title" title="mixed-article-title">Identification of immune-related genes in gills of chinese mitten crabs (eriocheir sinensis) during adaptation to air exposure stress</span>. <span class="source" tagx="source" title="source">Fish &amp;amp; Shellfish Immunology</span>. <span class="year" tagx="year" title="year">2019</span>; <span class="volume" tagx="volume" title="volume">84</span>, <span class="fpage" tagx="fpage" title="fpage">885</span>â€"<span class="lpage" tagx="lpage" title="lpage">893</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.fsi.2018.10.085">10.1016/j.fsi.2018.10.085</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30391295">30391295</a></span></span></li> <li tag="ref"><a name="pone.0242776.ref046" /><span class="label" tagx="label" title="label">46</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">B</span></span> and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Colin</span><span class="given-names" tagx="given-names" title="given-names">ND</span></span>. <span class="mixed-article-title" title="mixed-article-title">RSEM_ accurate transcript quantification from RNA Seq data with or without a reference genome</span>. <span class="source" tagx="source" title="source">BMC Bioinformatics</span>. <span class="year" tagx="year" title="year">2011</span>; <span class="volume" tagx="volume" title="volume">12</span>: <span class="fpage" tagx="fpage" title="fpage">323</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1471-2105-12-323">10.1186/1471-2105-12-323</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21816040">21816040</a></span></span></li> <li tag="ref"><a name="pone.0242776.ref047" /><span class="label" tagx="label" title="label">47</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Anders</span><span class="given-names" tagx="given-names" title="given-names">S</span></span> and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huber</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>. <span class="mixed-article-title" title="mixed-article-title">Differential expression analysis for sequence count data</span>. <span class="source" tagx="source" title="source">Genome Biol</span>. <span class="year" tagx="year" title="year">2010</span>; <span class="volume" tagx="volume" title="volume">11</span>: <span class="fpage" tagx="fpage" title="fpage">R106</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/gb-2010-11-10-r106">10.1186/gb-2010-11-10-r106</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20979621">20979621</a></span></span></li> <li tag="ref"><a name="pone.0242776.ref048" /><span class="label" tagx="label" title="label">48</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Alexa A and Rahnenfuhrer J. topGO Enrichment analysis for Gene Ontology. R package version 2.10.0. 2010</span></li> <li tag="ref"><a name="pone.0242776.ref049" /><span class="label" tagx="label" title="label">49</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mao</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cai</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Olyarchuk</span><span class="given-names" tagx="given-names" title="given-names">JG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wei</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>. <span class="mixed-article-title" title="mixed-article-title">Automated Genome Annotation and Pathway Identification Using the KEGG Orthology (KO) As a Controlled Vocabulary</span>. <span class="source" tagx="source" title="source">Bioinformatics</span><span class="year" tagx="year" title="year">2005</span>; <span class="volume" tagx="volume" title="volume">21</span>: <span class="fpage" tagx="fpage" title="fpage">3787</span>â€"<span class="lpage" tagx="lpage" title="lpage">3793</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/bti430">10.1093/bioinformatics/bti430</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15817693">15817693</a></span></span></li> <li tag="ref"><a name="pone.0242776.ref050" /><span class="label" tagx="label" title="label">50</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rozen</span><span class="given-names" tagx="given-names" title="given-names">S</span></span> and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Skaletsky</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>. <span class="mixed-article-title" title="mixed-article-title">Primer3 on the WWW for General Users and for Biologist Programmers</span>. <span class="source" tagx="source" title="source">Methods Mol. Biol</span>. <span class="year" tagx="year" title="year">2000</span>; <span class="volume" tagx="volume" title="volume">132</span>: <span class="fpage" tagx="fpage" title="fpage">365</span>â€"<span class="lpage" tagx="lpage" title="lpage">386</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1385/1-59259-192-2:365">10.1385/1-59259-192-2:365</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10547847">10547847</a></span></span></li> <li tag="ref"><a name="pone.0242776.ref051" /><span class="label" tagx="label" title="label">51</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kenneth</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thomas</span><span class="given-names" tagx="given-names" title="given-names">DS</span></span>. <span class="mixed-article-title" title="mixed-article-title">Analysis of Relative Gene Expression Data Using Real-Time Quantitative PCR and the 2<sup>âˆ’Î"Î"CT</sup> Method</span>. <span class="source" tagx="source" title="source">Methods</span>. <span class="year" tagx="year" title="year">2001</span>; <span class="volume" tagx="volume" title="volume">25</span>(<span class="issue" tagx="issue" title="issue">4</span>): <span class="fpage" tagx="fpage" title="fpage">402</span>â€"<span class="lpage" tagx="lpage" title="lpage">408</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1006/meth.2001.1262">10.1006/meth.2001.1262</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11846609">11846609</a></span></span></li> <li tag="ref"><a name="pone.0242776.ref052" /><span class="label" tagx="label" title="label">52</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hongmei</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">YJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Song</span><span class="given-names" tagx="given-names" title="given-names">JY</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">LJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Transcriptional data mining of <i>Salvia miltiorrhiza</i> in response to methyl jasmonate to examine the mechanism of bioactive compound biosynthesis and regulation</span>. <span class="source" tagx="source" title="source">Physiol Plant</span>. <span class="year" tagx="year" title="year">2014</span>; <span class="volume" tagx="volume" title="volume">152</span>(<span class="issue" tagx="issue" title="issue">2</span>): <span class="fpage" tagx="fpage" title="fpage">241</span>â€"<span class="lpage" tagx="lpage" title="lpage">55</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/ppl.12193">10.1111/ppl.12193</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24660670">24660670</a></span></span></li> <li tag="ref"><a name="pone.0242776.ref053" /><span class="label" tagx="label" title="label">53</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">HH</span></span>. <span class="mixed-article-title" title="mixed-article-title">Methodology for the determination of polysaccharides in Mongolian astragalus</span>. <span class="source" tagx="source" title="source">Guide of China Medicine</span>. <span class="year" tagx="year" title="year">2018</span>; <span class="volume" tagx="volume" title="volume">16</span> (<span class="issue" tagx="issue" title="issue">6</span>): <span class="fpage" tagx="fpage" title="fpage">16</span>â€"<span class="lpage" tagx="lpage" title="lpage">17</span>.</span></li> <li tag="ref"><a name="pone.0242776.ref054" /><span class="label" tagx="label" title="label">54</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">JS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jia</span><span class="given-names" tagx="given-names" title="given-names">CX</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mao</span><span class="given-names" tagx="given-names" title="given-names">DB</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">WY</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">YQ</span></span>. <span class="mixed-article-title" title="mixed-article-title">Determination of starch content in tobacco by iodine colorimetry</span>. <span class="source" tagx="source" title="source">Tobacco Science &amp;amp;Technology/Tobacco Chemistry</span>. <span class="year" tagx="year" title="year">2004</span>; <span class="volume" tagx="volume" title="volume">05</span>: <span class="fpage" tagx="fpage" title="fpage">24</span>â€"<span class="lpage" tagx="lpage" title="lpage">26</span>.</span></li> <li tag="ref"><a name="pone.0242776.ref055" /><span class="label" tagx="label" title="label">55</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xiao</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liang</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yingshuai</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guren</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>. <span class="mixed-article-title" title="mixed-article-title">Transcriptome response to temperature stress in the wolf spider Pardosa pseudoannulata (Araneae: <i>Lycosidae</i>)</span>. <span class="source" tagx="source" title="source">Ecol Evol</span>. <span class="year" tagx="year" title="year">2016</span>; <span class="volume" tagx="volume" title="volume">6</span>(<span class="issue" tagx="issue" title="issue">11</span>): <span class="fpage" tagx="fpage" title="fpage">3540</span>â€"<span class="lpage" tagx="lpage" title="lpage">3554</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1002/ece3.2142">10.1002/ece3.2142</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27127612">27127612</a></span></span></li> <li tag="ref"><a name="pone.0242776.ref056" /><span class="label" tagx="label" title="label">56</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hua</span><span class="given-names" tagx="given-names" title="given-names">WP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Song</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>. <span class="mixed-article-title" title="mixed-article-title"><i>De novo</i> transcriptome sequencing in <i>Salvia miltiorrhiza</i> to identify genes involved in the biosynthesis of active ingredients</span>. <span class="source" tagx="source" title="source">Genomics</span>. <span class="year" tagx="year" title="year">2011</span>; <span class="volume" tagx="volume" title="volume">98</span>: <span class="fpage" tagx="fpage" title="fpage">272</span>â€"<span class="lpage" tagx="lpage" title="lpage">279</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ygeno.2011.03.012">10.1016/j.ygeno.2011.03.012</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21473906">21473906</a></span></span></li> <li tag="ref"><a name="pone.0242776.ref057" /><span class="label" tagx="label" title="label">57</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mortazavi</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Williams</span><span class="given-names" tagx="given-names" title="given-names">BA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McCue</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schaeffer</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wold</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>. <span class="mixed-article-title" title="mixed-article-title">Mapping and quantifying mammalian transcriptomes by RNA-Seq</span>. <span class="source" tagx="source" title="source">Nat. Methods</span>. <span class="year" tagx="year" title="year">2008</span>; <span class="volume" tagx="volume" title="volume">5</span>: <span class="fpage" tagx="fpage" title="fpage">621</span>â€"<span class="lpage" tagx="lpage" title="lpage">628</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/nmeth.1226">10.1038/nmeth.1226</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18516045">18516045</a></span></span></li> <li tag="ref"><a name="pone.0242776.ref058" /><span class="label" tagx="label" title="label">58</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chand</span><span class="given-names" tagx="given-names" title="given-names">SK</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nanda</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rout</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mohanty</span><span class="given-names" tagx="given-names" title="given-names">JN</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mishra</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Joshi</span><span class="given-names" tagx="given-names" title="given-names">RK</span></span>. <span class="mixed-article-title" title="mixed-article-title">De novo sequencing and characterization of defense transcriptome responsive to Pythium aphanidermatum infection in <i>Curcuma longa</i> L</span>. <span class="source" tagx="source" title="source">Physiological and Molecular Plant Pathology</span>. <span class="year" tagx="year" title="year">2016</span>; <span class="volume" tagx="volume" title="volume">94</span>: <span class="fpage" tagx="fpage" title="fpage">27</span>â€"<span class="lpage" tagx="lpage" title="lpage">37</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.pmpp.2016.03.008">10.1016/j.pmpp.2016.03.008</a></span></span></li> <li tag="ref"><a name="pone.0242776.ref059" /><span class="label" tagx="label" title="label">59</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Santhi</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sheeja</span><span class="given-names" tagx="given-names" title="given-names">TE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krishnamurthy</span><span class="given-names" tagx="given-names" title="given-names">KS</span></span>. <span class="mixed-article-title" title="mixed-article-title">Transcriptome deep sequencing, identification of novel microRNAs and validation under drought stress in turmeric (<i>Curcuma longa</i> L.)</span>. <span class="source" tagx="source" title="source">Plant Biotechnology Reports</span>. <span class="year" tagx="year" title="year">2016</span>; <span class="volume" tagx="volume" title="volume">10</span>(<span class="issue" tagx="issue" title="issue">4</span>): <span class="fpage" tagx="fpage" title="fpage">227</span>â€"<span class="lpage" tagx="lpage" title="lpage">240</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1007/s11816-016-0399-2">10.1007/s11816-016-0399-2</a></span></span></li> <li tag="ref"><a name="pone.0242776.ref060" /><span class="label" tagx="label" title="label">60</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sheeja</span><span class="given-names" tagx="given-names" title="given-names">TE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Deepa</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Santhi</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sasikumar</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>. <span class="mixed-article-title" title="mixed-article-title">Comparative Transcriptome Analysis of Two Species of Curcuma Contrasting in a High-Value Compound Curcumin: Insights into Genetic Basis and Regulation of Biosynthesis</span>. <span class="source" tagx="source" title="source">Plant Molecular Biology Reporter</span>. <span class="year" tagx="year" title="year">2015</span>; <span class="volume" tagx="volume" title="volume">33</span>(<span class="issue" tagx="issue" title="issue">6</span>): <span class="fpage" tagx="fpage" title="fpage">1825</span>â€"<span class="lpage" tagx="lpage" title="lpage">1836</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1007/s11105-015-0878-6">10.1007/s11105-015-0878-6</a></span></span></li> <li tag="ref"><a name="pone.0242776.ref061" /><span class="label" tagx="label" title="label">61</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Deepa</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sheeja</span><span class="given-names" tagx="given-names" title="given-names">TE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rosana</span><span class="given-names" tagx="given-names" title="given-names">OB</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Srinivasan</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krishnamurthy</span><span class="given-names" tagx="given-names" title="given-names">KS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sasikumar</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>. <span class="mixed-article-title" title="mixed-article-title">Highly conserved sequence of ClPKS11 encodes a novel polyketide synthase involved in curcumin biosynthesis in turmeric (<i>Curcuma longa</i> L.)</span>. <span class="source" tagx="source" title="source">Industrial Crops and Products</span>. <span class="year" tagx="year" title="year">2017</span>; <span class="volume" tagx="volume" title="volume">97</span>: <span class="fpage" tagx="fpage" title="fpage">229</span>â€"<span class="lpage" tagx="lpage" title="lpage">241</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.indcrop.2016.12.003">10.1016/j.indcrop.2016.12.003</a></span></span></li> <li tag="ref"><a name="pone.0242776.ref062" /><span class="label" tagx="label" title="label">62</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xiang</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">XQ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gao</span><span class="given-names" tagx="given-names" title="given-names">HC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">ML</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zeng</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>. <span class="mixed-article-title" title="mixed-article-title">GC-MS and HPLC metabolic profiling studies of <i>Curcuma wenyujin</i> rhizomes obtained at different harvest times</span>. <span class="source" tagx="source" title="source">Analytical Letters</span>. <span class="year" tagx="year" title="year">2012</span>; <span class="volume" tagx="volume" title="volume">45</span>(<span class="issue" tagx="issue" title="issue">1</span>): <span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">14</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1080/00032719.2011.565447">10.1080/00032719.2011.565447</a></span></span></li> <li tag="ref"><a name="pone.0242776.ref063" /><span class="label" tagx="label" title="label">63</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cao</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qi</span><span class="given-names" tagx="given-names" title="given-names">ML</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shao</span><span class="given-names" tagx="given-names" title="given-names">QL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fu</span><span class="given-names" tagx="given-names" title="given-names">RN</span></span>. <span class="mixed-article-title" title="mixed-article-title">Analysis of volatile compounds in <i>Curcuma wenyujin</i> Y.H Chen et C Ling by headspace solvent microextraction-gas chromatography-mass spectrometry</span>. <span class="source" tagx="source" title="source">Analytica Chimica Acta</span>. <span class="year" tagx="year" title="year">2006</span>; <span class="volume" tagx="volume" title="volume">561</span>(<span class="issue" tagx="issue" title="issue">1</span>): <span class="fpage" tagx="fpage" title="fpage">88</span>â€"<span class="lpage" tagx="lpage" title="lpage">95</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.aca.2006.01.040">10.1016/j.aca.2006.01.040</a></span></span></li> <li tag="ref"><a name="pone.0242776.ref064" /><span class="label" tagx="label" title="label">64</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">LH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">HL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">XH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>. <span class="mixed-article-title" title="mixed-article-title">Comparative Study of Starch Content in Ginseng from Different Regions, Years and Growth Periods</span>. <span class="source" tagx="source" title="source">Liaoning Journal of Traditional Chinese Medicine</span>. <span class="year" tagx="year" title="year">2013</span>; <span class="volume" tagx="volume" title="volume">6</span>: <span class="fpage" tagx="fpage" title="fpage">1206</span>â€"<span class="lpage" tagx="lpage" title="lpage">1209</span>.</span></li> <li tag="ref"><a name="pone.0242776.ref065" /><span class="label" tagx="label" title="label">65</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Logacheva</span><span class="given-names" tagx="given-names" title="given-names">MD</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kasianov</span><span class="given-names" tagx="given-names" title="given-names">AS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vinogradov</span><span class="given-names" tagx="given-names" title="given-names">DV</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Samigullin</span><span class="given-names" tagx="given-names" title="given-names">TH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gelfand</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Makeev</span><span class="given-names" tagx="given-names" title="given-names">VJ</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">De novo sequencing and characterization of floral transcriptome in two species of buckwheat (<i>Fagopyrum</i>)</span>. <span class="source" tagx="source" title="source">BMC Genomics</span>. <span class="year" tagx="year" title="year">2011</span>; <span class="volume" tagx="volume" title="volume">12</span>: <span class="fpage" tagx="fpage" title="fpage">30</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1471-2164-12-30">10.1186/1471-2164-12-30</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21232141">21232141</a></span></span></li> <li tag="ref"><a name="pone.0242776.ref066" /><span class="label" tagx="label" title="label">66</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Swarbreck</span><span class="given-names" tagx="given-names" title="given-names">SM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lindquist</span><span class="given-names" tagx="given-names" title="given-names">EA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ackerly</span><span class="given-names" tagx="given-names" title="given-names">DD</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Andersen</span><span class="given-names" tagx="given-names" title="given-names">GL</span></span>. <span class="mixed-article-title" title="mixed-article-title">Analysis of leaf and root transcriptomes of soil-grown Avena barbata plants</span>. <span class="source" tagx="source" title="source">Plant Cell Physiol</span>. <span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">52</span>(<span class="issue" tagx="issue" title="issue">2</span>): <span class="fpage" tagx="fpage" title="fpage">317</span>â€"<span class="lpage" tagx="lpage" title="lpage">332</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/pcp/pcq188">10.1093/pcp/pcq188</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21310848">21310848</a></span></span></li> <li tag="ref"><a name="pone.0242776.ref067" /><span class="label" tagx="label" title="label">67</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">XW</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luan</span><span class="given-names" tagx="given-names" title="given-names">JB</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Su</span><span class="given-names" tagx="given-names" title="given-names">YL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xia</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">SS</span></span>. <span class="mixed-article-title" title="mixed-article-title">Transcriptome analysis and comparison reveal divergence between two invasive whitefly cryptic species</span>. <span class="source" tagx="source" title="source">BMC Genomics</span>. <span class="year" tagx="year" title="year">2011</span>; <span class="volume" tagx="volume" title="volume">12</span>: <span class="fpage" tagx="fpage" title="fpage">458</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1471-2164-12-458">10.1186/1471-2164-12-458</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21939539">21939539</a></span></span></li> <li tag="ref"><a name="pone.0242776.ref068" /><span class="label" tagx="label" title="label">68</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xiao</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">EJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barber</span><span class="given-names" tagx="given-names" title="given-names">CJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chakrabarty</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Transcriptome analysis based on next-generation sequencing of non-model plants producing specialized metabolites of biotechnological interest</span>. <span class="source" tagx="source" title="source">J Biotechnol</span>. <span class="year" tagx="year" title="year">2013</span>; <span class="volume" tagx="volume" title="volume">166</span>(<span class="issue" tagx="issue" title="issue">3</span>): <span class="fpage" tagx="fpage" title="fpage">122</span>â€"<span class="lpage" tagx="lpage" title="lpage">134</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jbiotec.2013.04.004">10.1016/j.jbiotec.2013.04.004</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23602801">23602801</a></span></span></li> <li tag="ref"><a name="pone.0242776.ref069" /><span class="label" tagx="label" title="label">69</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sui</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">JS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wei</span><span class="given-names" tagx="given-names" title="given-names">JH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jin</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">YH</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Comparison of root transcriptomes and expressions of genes involved in main medicinal secondary metabolites from Bupleurum chinense and Bupleurum scorzonerifolium, the two Chinese official Radix bupleuri source species</span>. <span class="source" tagx="source" title="source">Physiol Plant</span>. <span class="year" tagx="year" title="year">2015</span>; <span class="volume" tagx="volume" title="volume">153</span>(<span class="issue" tagx="issue" title="issue">2</span>): <span class="fpage" tagx="fpage" title="fpage">230</span>â€"<span class="lpage" tagx="lpage" title="lpage">42</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/ppl.12254">10.1111/ppl.12254</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25117935">25117935</a></span></span></li> <li tag="ref"><a name="pone.0242776.ref070" /><span class="label" tagx="label" title="label">70</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Du</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhai</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Deng</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yan</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Closely related NAC transcription factors of tomato differentially regulate stomatal closure and reopening during pathogen attack</span>. <span class="source" tagx="source" title="source">Plant Cell</span>. <span class="year" tagx="year" title="year">2014</span>; <span class="volume" tagx="volume" title="volume">26</span>(<span class="issue" tagx="issue" title="issue">7</span>): <span class="fpage" tagx="fpage" title="fpage">3167</span>â€"<span class="lpage" tagx="lpage" title="lpage">3184</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1105/tpc.114.128272">10.1105/tpc.114.128272</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25005917">25005917</a></span></span></li> <li tag="ref"><a name="pone.0242776.ref071" /><span class="label" tagx="label" title="label">71</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ko</span><span class="given-names" tagx="given-names" title="given-names">SS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun-Ben</span><span class="given-names" tagx="given-names" title="given-names">KM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ho</span><span class="given-names" tagx="given-names" title="given-names">YC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lin</span><span class="given-names" tagx="given-names" title="given-names">YJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chuang</span><span class="given-names" tagx="given-names" title="given-names">MH</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">The <i>bHLH142</i> Transcription Factor Coordinates with TDR1 to Modulate the Expression of EAT1 and Regulate Pollen Development in Rice</span>. <span class="source" tagx="source" title="source">Plant Cell</span>. <span class="year" tagx="year" title="year">2014</span>; <span class="volume" tagx="volume" title="volume">26</span>(<span class="issue" tagx="issue" title="issue">6</span>): <span class="fpage" tagx="fpage" title="fpage">2486</span>â€"<span class="lpage" tagx="lpage" title="lpage">2504</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1105/tpc.114.126292">10.1105/tpc.114.126292</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24894043">24894043</a></span></span></li> <li tag="ref"><a name="pone.0242776.ref072" /><span class="label" tagx="label" title="label">72</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vermeirssen</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">De Clercq</span><span class="given-names" tagx="given-names" title="given-names">I</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Van Parys</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Van</span><span class="given-names" tagx="given-names" title="given-names">BF</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Van de Peer</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>. <span class="mixed-article-title" title="mixed-article-title">Arabidopsis ensemble reverse-engineered gene regulatory network discloses interconnected transcription factors in oxidative stress</span>. <span class="source" tagx="source" title="source">Plant Cell</span>. <span class="year" tagx="year" title="year">2014</span>; <span class="volume" tagx="volume" title="volume">26</span>(<span class="issue" tagx="issue" title="issue">12</span>): <span class="fpage" tagx="fpage" title="fpage">4656</span>â€"<span class="lpage" tagx="lpage" title="lpage">4679</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1105/tpc.114.131417">10.1105/tpc.114.131417</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25549671">25549671</a></span></span></li> <li tag="ref"><a name="pone.0242776.ref073" /><span class="label" tagx="label" title="label">73</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">ZY</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">SY</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hyeon</span><span class="given-names" tagx="given-names" title="given-names">DY</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">DH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dong</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Park</span><span class="given-names" tagx="given-names" title="given-names">YM</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">The Arabidopsis NAC transcription factor ANAC096 cooperates with <i>bZIP</i>-type transcription factors in dehydration and osmotic stress responses</span>. <span class="source" tagx="source" title="source">Plant Cell</span>. <span class="year" tagx="year" title="year">2013</span>; <span class="volume" tagx="volume" title="volume">25</span>(<span class="issue" tagx="issue" title="issue">11</span>): <span class="fpage" tagx="fpage" title="fpage">4708</span>â€"<span class="lpage" tagx="lpage" title="lpage">4724</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1105/tpc.113.119099">10.1105/tpc.113.119099</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24285786">24285786</a></span></span></li> <li tag="ref"><a name="pone.0242776.ref074" /><span class="label" tagx="label" title="label">74</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pautler</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eveland</span><span class="given-names" tagx="given-names" title="given-names">AL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">LaRue</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weeks</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lunde</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">FASCIATED EAR4 encodes a <i>bZIP</i> transcription factor that regulates shoot meristem size in maize</span>. <span class="source" tagx="source" title="source">Plant Cell</span>. <span class="year" tagx="year" title="year">2015</span>; <span class="volume" tagx="volume" title="volume">27</span>(<span class="issue" tagx="issue" title="issue">1</span>): <span class="fpage" tagx="fpage" title="fpage">104</span>â€"<span class="lpage" tagx="lpage" title="lpage">20</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1105/tpc.114.132506">10.1105/tpc.114.132506</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25616871">25616871</a></span></span></li> <li tag="ref"><a name="pone.0242776.ref075" /><span class="label" tagx="label" title="label">75</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mol</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cornish</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mason</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Koes</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>. <span class="mixed-article-title" title="mixed-article-title">Novel coloured flowers</span>. <span class="source" tagx="source" title="source">Curr. Opin. Biotechnol</span>. <span class="year" tagx="year" title="year">1999</span>; <span class="volume" tagx="volume" title="volume">10</span>: <span class="fpage" tagx="fpage" title="fpage">198</span>â€"<span class="lpage" tagx="lpage" title="lpage">201</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/S0958-1669(99)80035-4">10.1016/S0958-1669(99)80035-4</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10209139">10209139</a></span></span></li> <li tag="ref"><a name="pone.0242776.ref076" /><span class="label" tagx="label" title="label">76</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pandey</span><span class="given-names" tagx="given-names" title="given-names">SP</span></span> and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Somssich</span><span class="given-names" tagx="given-names" title="given-names">IE</span></span>. <span class="mixed-article-title" title="mixed-article-title">The role of <i>WRKY</i> transcription factors in plant immunity</span>. <span class="source" tagx="source" title="source">Plant Physiol</span>. <span class="year" tagx="year" title="year">2009</span>; <span class="volume" tagx="volume" title="volume">150</span>(<span class="issue" tagx="issue" title="issue">4</span>): <span class="fpage" tagx="fpage" title="fpage">1648</span>â€"<span class="lpage" tagx="lpage" title="lpage">1655</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1104/pp.109.138990">10.1104/pp.109.138990</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19420325">19420325</a></span></span></li> <li tag="ref"><a name="pone.0242776.ref077" /><span class="label" tagx="label" title="label">77</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bakshi</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">OelmÃ¼ller</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>. <span class="mixed-article-title" title="mixed-article-title"><i>WRKY</i> transcription factor: jack of many trades in plants</span>. <span class="source" tagx="source" title="source">Plant Signaling &amp;amp; Behavior</span>. <span class="year" tagx="year" title="year">2014</span>; <span class="volume" tagx="volume" title="volume">9</span>(<span class="issue" tagx="issue" title="issue">2</span>): <span class="fpage" tagx="fpage" title="fpage">e27700</span><span class="pub-id"><a href="https://dx.doi.org/10.4161/psb.27700">10.4161/psb.27700</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24492469">24492469</a></span></span></li> <li tag="ref"><a name="pone.0242776.ref078" /><span class="label" tagx="label" title="label">78</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rushton</span><span class="given-names" tagx="given-names" title="given-names">PJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Somssich</span><span class="given-names" tagx="given-names" title="given-names">IE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ringler</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shen</span><span class="given-names" tagx="given-names" title="given-names">QJ</span></span>. <span class="mixed-article-title" title="mixed-article-title"><i>WRKY</i> transcription factors</span>. <span class="source" tagx="source" title="source">Trends Plant Sci</span>. <span class="year" tagx="year" title="year">2010</span>; <span class="volume" tagx="volume" title="volume">15</span>: <span class="fpage" tagx="fpage" title="fpage">247</span>â€"<span class="lpage" tagx="lpage" title="lpage">258</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.tplants.2010.02.006">10.1016/j.tplants.2010.02.006</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20304701">20304701</a></span></span></li> <li tag="ref"><a name="pone.0242776.ref079" /><span class="label" tagx="label" title="label">79</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiang</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>. <span class="mixed-article-title" title="mixed-article-title"><i>WRKY22</i> transcription factor mediates dark-induced leaf senescence in Arabidopsis</span>. <span class="source" tagx="source" title="source">Mol Cells</span>. <span class="year" tagx="year" title="year">2011</span>; <span class="volume" tagx="volume" title="volume">31</span>(<span class="issue" tagx="issue" title="issue">4</span>): <span class="fpage" tagx="fpage" title="fpage">303</span>â€"<span class="lpage" tagx="lpage" title="lpage">313</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1007/s10059-011-0047-1">10.1007/s10059-011-0047-1</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21359674">21359674</a></span></span></li> <li tag="ref"><a name="pone.0242776.ref080" /><span class="label" tagx="label" title="label">80</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>. <span class="source" tagx="source" title="source">Functional study of maize microtubule-associated protein ZmMAP65-1a and transcription factor ZmNAC in abiotic stress</span>. <span class="publisher-name" tagx="publisher-name" title="publisher-name">Nanjing Agricultural University</span><span class="year" tagx="year" title="year">2014</span>; pp, <span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">180</span>.</span></li> </ul> </div> </div>  </body></html>