<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Front Genet</span><span class="iso-abbrev" title="iso-abbrev">Front Genet</span><span class="publisher-id" title="publisher-id">Front. Genet.</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Frontiers in Genetics</span></div><span class="issn-epub" tagx="issn" title="issn-epub">1664-8021</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Frontiers Media S.A.</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7689033">7689033</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.3389/fgene.2020.598714">10.3389/fgene.2020.598714</a></span><div class="article-categories" title="article-categories"> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genetics</span>: <span class="subject" title="subject">Original Research</span></div> </div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Transcriptional Analysis of Metabolic Pathways and Regulatory Mechanisms of Essential Oil Biosynthesis in the Leaves of <i>Cinnamomum camphora</i> (L.) Presl</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hou</span><span class="given-names" tagx="given-names" title="given-names">Jiexi</span></span><a href="#aff1"><sup>1</sup></a><a href="#c001"><sup>*</sup></a><span class="uri" title="uri"><a href="http://loop.frontiersin.org/people/1065906/overview" /></span></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Jie</span></span><a href="#aff1"><sup>1</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Beihong</span></span><a href="#aff1"><sup>1</sup></a><a href="#aff2"><sup>2</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jin</span><span class="given-names" tagx="given-names" title="given-names">Xiaofang</span></span><a href="#aff1"><sup>1</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Haiyan</span></span><a href="#aff1"><sup>1</sup></a><a href="#c002"><sup>*</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jin</span><span class="given-names" tagx="given-names" title="given-names">Zhinong</span></span><a href="#aff1"><sup>1</sup></a><a href="#c003"><sup>*</sup></a><span class="uri" title="uri"><a href="http://loop.frontiersin.org/people/1115151/overview" /></span></span></div><span class="citation_author_institution" id="aff1">[], <sup>1</sup><span class="institution" title="institution">Jiangxi Provincial Engineering Research Center for Seed-Breeding and Utilization of Camphor Trees, The School of Hydraulic and Ecological Engineering, Nanchang Institute of Technology</span><span class="addr-line" title="addr-line">Nanchang</span><span class="country" tagx="country" title="country">China</span></span><span class="citation_author_institution" id="aff2">[], <sup>2</sup><span class="institution" title="institution">Key Laboratory of Silviculture, Co-Innovation Center of Jiangxi Typical Trees Cultivation and Utilization, College of Forestry, Jiangxi Agricultural University</span><span class="addr-line" title="addr-line">Nanchang</span><span class="country" tagx="country" title="country">China</span></span><div class="author-notes" title="author-notes"> <div class="fn-type-edited-by" title="edited-by"> <p>Edited by: Hongjian Wan, ZheJiang Academy of Agricultural Sciences, China</p> </div> <div class="fn-type-edited-by" title="edited-by"> <p>Reviewed by: Changjun Ding, Chinese Academy of Forestry, China; Duc Quan Nguyen, The University of Newcastle, Australia</p> </div> <div class="corresp" title="corresp">
*Correspondence: Jiexi Hou, <span class="email" tagx="email" title="email">jxhou@cctnit.com</span></div> <div class="corresp" title="corresp">
Haiyan Zhang, <span class="email" tagx="email" title="email">HYZhang@cctnit.com</span></div> <div class="corresp" title="corresp">
Zhinong Jin, <span class="email" tagx="email" title="email">agroking@cctnit.com</span></div> <div class="fn-type-other" title="other"> <p>This article was submitted to Plant Genomics, a section of the journal Frontiers in Genetics</p> </div> </div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2020-11-11</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2020</span></span><span class="volume" tagx="volume" title="volume">11</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">598714</span><span class="history" title="history"><span class="received" title="received">received: 2020-8-25</span><span class="accepted" title="accepted">accepted: 2020-10-14</span></span><div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Hou, Zhang, Zhang, Jin, Zhang and Jin</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2020</span></span><span class="license" title="license"><span class="license-p" title="license-p">This is an open-access article distributed under the terms of the Creative Commons Attribution License (CC BY). The use, distribution or reproduction in other forums is permitted, provided the original author(s) and the copyright owner(s) are credited and that the original publication in this journal is cited, in accordance with accepted academic practice. No use, distribution or reproduction is permitted which does not comply with these terms.</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>The roots, bark, and leaves of <i>Cinnamomum camphora</i> are rich in essential oils, which mainly comprised monoterpenes and sesquiterpenes. Although the essential oils obtained from <i>C. camphora</i> have been widely used in pharmaceutical, medicinal, perfume, and food industries, the molecular mechanisms underlying terpenoid biosynthesis are poorly understood. To address this lack of knowledge, we performed transcriptome analysis to investigate the key regulatory genes involved in terpenoid biosynthesis in <i>C. camphora</i>. High-oil-yield trees of linalool type and low-oil-yield trees were used to assemble a <i>de novo</i> transcriptome of <i>C. camphora</i>. A total of 121,285 unigenes were assembled, and the total length, average length, N50, and GC content of unigenes were 87,869,987, 724, 1,063, and 41.1%, respectively. Comparison of the transcriptome profiles of linalool-type <i>C. camphora</i> with trees of low oil yield resulted in a total of 3,689 differentially expressed unigenes, among them 31 candidate genes had annotations associated with metabolism of terpenoids and polyketides, including four in the monoterpenoid biosynthesis pathway and three in the terpenoid backbone biosynthesis pathway. Collectively, this genome-wide transcriptome provides a valuable tool for future identification of genes related to essential oil biosynthesis. Additionally, the identification of a cohort of genes in the biosynthetic pathways of terpenoids provides a theoretical basis for metabolic engineering of essential oils in <i>C. camphora</i>.</p> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
transcriptional analysis</div> <div class="kwd" title="kwd">
terpenoid biosynthesis</div> <div class="kwd" title="kwd">
chemotypes</div> <div class="kwd" title="kwd">
leaves</div> <div class="kwd" title="kwd"><i>Cinnamomum camphora</i></div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source">
National Natural Science Foundation of China<span class="named-content" title="named-content">10.13039/501100001809</span></div> </div> </div> <div class="counts" title="counts"><span class="fig-count" title="fig-count">fig-count: </span><span class="table-count" title="table-count">table-count: </span><span class="equation-count" title="equation-count">equation-count: </span><span class="ref-count" title="ref-count">ref-count: </span><span class="page-count" title="page-count">page-count: </span><span class="word-count" title="word-count">word-count: </span></div> </div> </div> <div class="body" title="body"> <div class="introduction" title="sec"> <div class="title" tagx="title" title="title">
Introduction</div> <p>As a member of the Lauraceae family, <i>Cinnamomum camphora</i> is an evergreen tree widely distributed around the southern portions of the Yantze River in China, with the largest numbers in Jiangxi, Guangdong, Fujian, Zhejiang, Yunnan, and Sichuan. Many parts of the camphor tree, including roots, stems, leaves, flowers, fruits, and bark, are rich in essential oils that have a wide range of uses in the pest control, spice, cosmetics, and pharmaceutical industries (<a href="#B16">Guo et al., 2016</a>; <a href="#B19">Jiang et al., 2016</a>). In order to achieve sustainable production, large-scale plantations have been established in China. Under this model, camphor trees are maintained as bushes, and the leaves and twigs are harvested every year for essential oil production. <i>C. camphora</i> plantations are characterized by fast growth and high afforestation density, and therefore, they can provide a large number of raw materials for the essential oil industry.</p> <p>Essential oils in the camphor tree primarily consist of monoterpenes and sesquiterpenes. Depending on the principal chemical component of their essential oils, camphor trees can be divided into different chemical types, such as linalool, borneol, camphor, cineol, and iso-nerolidol (<a href="#B17">Guo et al., 2017</a>). With the gradual improvements in <i>C. camphora</i> cultivation and essential oil extraction techniques, use of <i>C. camphora</i> as a source of natural linalool, which has long been preferred over its synthetic form for the perfume industry, is becoming more popular. However, the molecular mechanisms underlying the terpenoid biosynthesis in <i>C. camphora</i> are still poorly understood, which impedes the progress of identifying â€œsuperiorâ€� <i>C. camphora</i> clones that could increase the efficiency of using <i>C. camphora</i> for essential oil production.</p> <p>The biosynthetic pathway of terpenoids in plants starts with the synthesis of the universal terpenoid precursors isopentenyl diphosphate (IPP) and dimethylallyl diphosphate (DMAPP) through the mevalonate (MVA) or the methylerythritol 4-phosphate (MEP) pathways. Isoprenyl diphosphate synthases (IDSs) catalyze reactions involving IPP and DMAPP, forming the intermediate phosphate precursors, geranyl diphosphate (GPP), geranylgeranyl diphosphate (GGPP), and farnesyl diphosphate, which are then catalyzed by terpene synthases (TPSs) to form diverse terpenoids (<a href="#B7">Chen et al., 2003</a>; <a href="#B9">Cheng et al., 2007</a>; <a href="#B27">Ma et al., 2012</a>). The enzymes in the MVA pathway have been shown to be located in cytosol or peroxisomes, whereas all the enzymes involved in the MEP pathway are found in plastids (<a href="#B31">Reumann et al., 2007</a>; <a href="#B38">Simkin et al., 2011</a>). In contrast, the localizations of IDSs and TPSs are more diverse and are often at the site of terpenoid biosynthesis (<a href="#B27">Ma et al., 2012</a>). Although characterizations of the genes involved in terpenoid biosynthesis have been reported in many plant species, including peppermint, <i>Arabidopsis</i>, conifers, <i>Hevea brasiliensis</i>, and <i>Salvia miltiorrhiza</i> (<a href="#B4">Battaile and Loomis, 1961</a>; <a href="#B21">Lange and Ghassemian, 2003</a>; <a href="#B35">Sando et al., 2008a,b</a>; <a href="#B46">Zulak and Bohlmann, 2010</a>; <a href="#B27">Ma et al., 2012</a>), many of the enzyme-encoding genes have yet to be elucidated because of the complexity of the terpenoid biosynthesis pathway, especially in the species without a whole-genome sequence.</p> <p>In the studies of <i>C. camphora</i>, transcriptome analyses for leaves of camphor trees of linalool-, borneol-, camphor-, cineol-, and iso-nerolidol types led to the identification of 424 unigenes in the biosynthetic pathways of terpenoids (<a href="#B20">Jiang et al., 2014</a>) and transcriptome profiling of linalool and borneol types of <i>C. camphora</i> identified three monoterpene synthase genes possibly involved in the biosynthesis of borneol (<a href="#B6">Chen et al., 2018</a>). Although these two transcriptome studies have provided valuable insights into the terpenoid biosynthesis pathway, comparisons between only high-oil-yielding individuals could exclude the identification of genes upstream of the biosynthetic pathway of terpenoids. In this study, we used RNA-Seq to identify key enzyme-encoding genes putatively associated with terpenoid biosynthesis in <i>C. camphora</i>, which may partially determine its essential oil composition. Differentially expressed genes (DEGs) between camphor trees of linalool-type and low-oil-yield camphor trees were identified. In addition to four unigenes in the â€œmonoterpenoid biosynthesisâ€� pathway, this analysis also identified three unigenes in the â€œterpenoid backbone biosynthesisâ€� pathway. Overall, these results contribute to the general understanding of terpenoid biosynthesis in <i>C. camphora</i> and provide a theoretical basis for metabolic engineering of terpenoids in this species.</p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Materials and Methods</div> <div class="plantmaterials" title="sec"> <div class="title" tagx="title" title="title">
Plant Materials</div> <p>Mature leaves of linalool-type and low-oil-yield <i>C. camphora</i> were harvested from 3-year-old clones grown at the experimental garden of NanChang Institute of Technology, China, in October. The clones were propagated from mother trees through cutting propagation. For each biological replicate (<i>n</i> = 3), leaves from at least five tree clones, which were cloned from the same mother tree, were collected and mixed. The three replicates each for the low-oil-yield variety and linalool-type <i>C. camphora</i> were termed N- (1â€"3) and LI- (1â€"3) separately. All the samples were harvested and snap-frozen in liquid nitrogen and then stored at âˆ’80Â°C until RNA extraction. Meanwhile, leaves were also collected for oil extraction and chemical component determination.</p> </div> <div class="essentialoilextractionanddeterminationofessentialoilyield" title="sec"> <div class="title" tagx="title" title="title">
Essential Oil Extraction and Determination of Essential Oil Yield</div> <p>Fresh leaves were harvested and processed for oil extraction immediately after harvesting. Leaf samples were hydrodistilled in a modified Clevenger-type apparatus for 4 h. Extracted essential oils were weighed, and oil yields were calculated using the formula:</p> <div class="disp-formula" title="disp-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
Essential oil yield (%)</div> <div class="mo_UNKNOWN">
=</div> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
W1</div> <div class="mo_UNKNOWN">
/</div> <div class="mi_UNKNOWN">
W2</div> </div> <div class="mo_UNKNOWN">
Ã—</div> <div class="mn_UNKNOWN">
100</div> </div> </div></div></div> <p>where W1 is the weight of extracted essential oil, and W2 is the weight of fresh leaves.</p> </div> <div class="gaschromatographyâ€" title="sec"> <div class="title" tagx="title" title="title">
Gas Chromatographyâ€"Mass Spectrometry Analysis</div> <p>Analyses of essential oils were performed on a gas chromatography system (Agilent 7890B, United States) equipped with a TG-5MS capillary column (30 m Ã— 0.25-mm internal diameter Ã— 0.25-Î¼m film thickness). Helium with a flow rate of 1 mL/min was used as the carrier gas. The injector was maintained at 220Â°C. The initial temperature of the column was set at 40Â°C and increased to 280â€"300Â°C at a rate of 6.5Â°C/min. The mass spectrometer was set to scan in the range of 50â€"650 m/z with a scan rate of 0.5 scans/s. Compounds were determined based on their relative retention time and mas spectra data. Details were described in our previous study (<a href="#B45">Zhang et al., 2019</a>).</p> </div> <div class="cdnalibrarypreparationandrnasequencing" title="sec"> <div class="title" tagx="title" title="title">
cDNA Library Preparation and RNA Sequencing</div> <p>Total RNA was extracted from samples using the RNeasy Plant Mini Kit (Qiagen, United States) following the manufacturerâ€™s instructions. Next, gDNA was removed using DNase I digestion (Qiagen, United States). The integrity of extracted RNA was determined by the RNA Nano 6000 Assay Kit of the Agilent 2100 Bioanalyzer (Agilent Technologies, United States). Extracted RNA was sent to Gene <i>Denovo</i> Biotechnology Co. (Guangzhou, China) for cDNA library preparation and sequencing. Up to 1 Î¼g of total RNA from each sample was used for library construction using the NEB #7530 kit for Illumina<sup>Â®</sup> (New England Biolabs, United States), as described by the manufacturer. mRNA was first enriched by oligo(dT) beads and fragmented into small fragments using fragmentation buffer and reverse-transcribed into cDNA with ProtoScript II Reverse Transcriptase and random primers. Second-strand cDNA was synthesized by DNA polymerase I along with RNase H, dNTP, and buffer. Next, cDNA fragments were purified with 1.8 Ã— Agencourt AMPure XP Beads, end-repaired, and ligated to Illumina sequencing adapters after addition of poly(A) tails. Ligated fragments were subjected to size selection by gel electrophoresis and polymerase chain reaction (PCR) amplified. The resulting cDNA library was sequenced using Illumina HiSeq<sup>TM</sup> 4000.</p> </div> <div class="denovoassemblyandfunctionalannotation" title="sec"> <div class="title" tagx="title" title="title"><i>De novo</i> Assembly and Functional Annotation</div> <p>Raw reads from each sample were filtered by removing reads containing more than 10% of unknown nucleotides, low-quality reads that contained more than 40% of low-quality (<i>Q</i> value â‰¤ 20) bases, and adapter sequences according to the Illumina adapter list. Residual ribosome RNA (rRNA) reads were identified by mapping the filtered reads to rRNA, and the identified sequences were then removed. The resulting 87.9 M reads were assembled using short read assembler, Trinity (<a href="#B15">Grabherr et al., 2011</a>), and further clustered to obtain non-redundant unigenes by Corset (<a href="#B11">Davidson and Oshlack, 2014</a>). The completeness of the transcriptomic assembly was assessed using BUSCO (<a href="#B37">Simao et al., 2015</a>) by comparing the 1,440 embryophyta-specific genes to our unigenes. Unigenes were annotated by BLASTx (with an <i>E</i>-value threshold of 1e<sup>â€"5</sup>) to the Swiss-Prot protein database (<a href="#B3">Bairoch and Apweiler, 2000</a>) and NCBI non-redundant protein (Nr) database (<a href="#B29">Oâ€™Leary et al., 2016</a>). Nr annotation results of unigenes were used for Gene Ontology (GO) annotation by Blast2GO software (<a href="#B10">Conesa et al., 2005</a>), and functional classification of unigenes was performed using WEGO software (<a href="#B44">Ye et al., 2006</a>).</p> </div> <div class="identificationofdifferentiallyexpressedgenes" title="sec"> <div class="title" tagx="title" title="title">
Identification of Differentially Expressed Genes</div> <p>Expression values of each gene were calculated as reads per kilobase (of exon) per million (RPKM) mapped reads. Biological samples with low and high oil contents were grouped, and comparison was undertaken to identify DEGs between the two experimental groups. Principal component analysis (PCA) was also performed as a quality control. DEGs with a fold change â‰¥ 2 and <i>p</i> &amp;lt; 0.01 across the two experimental groups were identified using the edgeR package (<a href="#B32">Robinson et al., 2010</a>).</p> </div> <div class="quantitativereversetranscriptaseâ€" title="sec"> <div class="title" tagx="title" title="title">
Quantitative Reverse Transcriptaseâ€"PCR</div> <p>Leaves from five randomly selected 3-year-old tree clones of each chemotype were collected for quantitative reverse transcriptase (RT)â€"PCR analysis. Total RNA was extracted and reverse-transcribed into cDNA using the HiScript II Q RT SuperMix for quantitative PCR (qPCR) (Vazyme, China). The PCRs were performed using ChamQTM SYBR<sup>Â®</sup> qPCR Master Mix (Vazyme, China) according to the manufacturerâ€™s instructions on an ABI Step One Plus System with three replicates. Sequences of the gene-specific primers are listed in <a href="#TS1">Supplementary Table S1</a>. Relative gene expression was calculated using the 2<sup>â€"â–³â–³<i>Ct</i></sup> method, and <i>ACTIN</i> was chosen as the internal reference gene to normalize expression levels.</p> </div> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <div class="chemicalcompositionofleafessentialoilsfromc.camphoraofdifferentoilcontents" title="sec"> <div class="title" tagx="title" title="title">
Chemical Composition of Leaf Essential Oils From <i>C. camphora</i> of Different Oil Contents</div> <p>In general, there were no major differences in morphology between camphor trees of different chemotypes, and the trees were therefore initially classified by odor alone. Fresh leaves of the six samples were harvested, and the essential oils were obtained by water distillation. The average oil outputs of the samples from linalool- and low-oil-yield chemotypes were 1.3 and 0.3%, respectively. Chemical composition of the extracted essential oils was determined by gas chromatographyâ€"mass spectrometry (GC-MS). <a href="#T1">Table 1</a> lists the major components detected, 1â€"15 are monoterpenes, and the remaining are sesquiterpenes. In the essential oils of the linalool type, Î²-linalool was the principal constituent, making up âˆ¼90% in average. Among the various constituents of the essential oils extracted from the different chemotypes of camphor trees, the major difference was observed for the monoterpene content. Therefore, transcriptome analyses between trees of linalool type and trees with low oil yield were performed to investigate the molecular mechanisms underlying the biosynthesis of monoterpenes.</p> <div class="table-wrap_UNKNOWN" id="T1" position="float"><span class="label" tagx="label" title="label">TABLE 1</span> <p>Chemical composition of the essential oils from leaf extracts of <i>C. camphora</i>.</p>  <table frame="hsides" rules="groups" cellspacing="5" cellpadding="5"> <thead> <tr> <td valign="top" align="left" rowspan="1" colspan="1">No.</td> <td valign="top" align="center" rowspan="1" colspan="1">Compound</td> <td valign="top" align="center" rowspan="1" colspan="1">N-1 (%)</td> <td valign="top" align="center" rowspan="1" colspan="1">N-2 (%)</td> <td valign="top" align="center" rowspan="1" colspan="1">N-3 (%)</td> <td valign="top" align="center" rowspan="1" colspan="1">LI-1(%)</td> <td valign="top" align="center" rowspan="1" colspan="1">LI-2(%)</td> <td valign="top" align="center" rowspan="1" colspan="1">LI-3 (%)</td> </tr> </thead> <tbody> <tr> <td valign="top" align="left" rowspan="1" colspan="1">1</td> <td valign="top" align="center" rowspan="1" colspan="1">Î±-Thujene</td> <td valign="top" align="center" rowspan="1" colspan="1">1.51</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> <td valign="top" align="center" rowspan="1" colspan="1">0.1</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">2</td> <td valign="top" align="center" rowspan="1" colspan="1">Pinene</td> <td valign="top" align="center" rowspan="1" colspan="1">8.28</td> <td valign="top" align="center" rowspan="1" colspan="1">4.63</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> <td valign="top" align="center" rowspan="1" colspan="1">0.36</td> <td valign="top" align="center" rowspan="1" colspan="1">0.19</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">3</td> <td valign="top" align="center" rowspan="1" colspan="1">Camphene</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> <td valign="top" align="center" rowspan="1" colspan="1">1.49</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> <td valign="top" align="center" rowspan="1" colspan="1">0.14</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">4</td> <td valign="top" align="center" rowspan="1" colspan="1">Phellandrene</td> <td valign="top" align="center" rowspan="1" colspan="1">26.19</td> <td valign="top" align="center" rowspan="1" colspan="1">0.88</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> <td valign="top" align="center" rowspan="1" colspan="1">0.19</td> <td valign="top" align="center" rowspan="1" colspan="1">0.12</td> <td valign="top" align="center" rowspan="1" colspan="1">0.47</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">5</td> <td valign="top" align="center" rowspan="1" colspan="1">Î²-Myrcene</td> <td valign="top" align="center" rowspan="1" colspan="1">2.26</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> <td valign="top" align="center" rowspan="1" colspan="1">0.16</td> <td valign="top" align="center" rowspan="1" colspan="1">0.16</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">6</td> <td valign="top" align="center" rowspan="1" colspan="1"> <div class="sc_UNKNOWN">
D</div>-Limonene</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> <td valign="top" align="center" rowspan="1" colspan="1">0.15</td> <td valign="top" align="center" rowspan="1" colspan="1">0.26</td> <td valign="top" align="center" rowspan="1" colspan="1">0.26</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">7</td> <td valign="top" align="center" rowspan="1" colspan="1">Eucalyptol</td> <td valign="top" align="center" rowspan="1" colspan="1">37.7</td> <td valign="top" align="center" rowspan="1" colspan="1">2.14</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> <td valign="top" align="center" rowspan="1" colspan="1">0.59</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> <td valign="top" align="center" rowspan="1" colspan="1">0.46</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">8</td> <td valign="top" align="center" rowspan="1" colspan="1">Î²-Ocimene</td> <td valign="top" align="center" rowspan="1" colspan="1">0.56</td> <td valign="top" align="center" rowspan="1" colspan="1">4.87</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> <td valign="top" align="center" rowspan="1" colspan="1">0.68</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">9</td> <td valign="top" align="center" rowspan="1" colspan="1">Terpinene</td> <td valign="top" align="center" rowspan="1" colspan="1">3.48</td> <td valign="top" align="center" rowspan="1" colspan="1" /> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">10</td> <td valign="top" align="center" rowspan="1" colspan="1">Î²-Linalool</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> <td valign="top" align="center" rowspan="1" colspan="1">2.06</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> <td valign="top" align="center" rowspan="1" colspan="1">90.38</td> <td valign="top" align="center" rowspan="1" colspan="1">86.8</td> <td valign="top" align="center" rowspan="1" colspan="1">89.16</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">11</td> <td valign="top" align="center" rowspan="1" colspan="1">Borneol acetate</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> <td valign="top" align="center" rowspan="1" colspan="1">1.95</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">12</td> <td valign="top" align="center" rowspan="1" colspan="1">Camphor</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> <td valign="top" align="center" rowspan="1" colspan="1">1.36</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">13</td> <td valign="top" align="center" rowspan="1" colspan="1">Terpineol</td> <td valign="top" align="center" rowspan="1" colspan="1">10.57</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> <td valign="top" align="center" rowspan="1" colspan="1">0.13</td> <td valign="top" align="center" rowspan="1" colspan="1">0.07</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">14</td> <td valign="top" align="center" rowspan="1" colspan="1">Citronellol</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> <td valign="top" align="center" rowspan="1" colspan="1">0.13</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">15</td> <td valign="top" align="center" rowspan="1" colspan="1">Citral</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> <td valign="top" align="center" rowspan="1" colspan="1">0.43</td> <td valign="top" align="center" rowspan="1" colspan="1">1.16</td> <td valign="top" align="center" rowspan="1" colspan="1">0.54</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">16</td> <td valign="top" align="center" rowspan="1" colspan="1">Caryophyllene</td> <td valign="top" align="center" rowspan="1" colspan="1">2.2</td> <td valign="top" align="center" rowspan="1" colspan="1">13.16</td> <td valign="top" align="center" rowspan="1" colspan="1">8.58</td> <td valign="top" align="center" rowspan="1" colspan="1">2.07</td> <td valign="top" align="center" rowspan="1" colspan="1">1.71</td> <td valign="top" align="center" rowspan="1" colspan="1">1.03</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">17</td> <td valign="top" align="center" rowspan="1" colspan="1">Alloaromadendrene</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> <td valign="top" align="center" rowspan="1" colspan="1">1.24</td> <td valign="top" align="center" rowspan="1" colspan="1">0.18</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">18</td> <td valign="top" align="center" rowspan="1" colspan="1">Germacrene D</td> <td valign="top" align="center" rowspan="1" colspan="1">1.37</td> <td valign="top" align="center" rowspan="1" colspan="1">1.28</td> <td valign="top" align="center" rowspan="1" colspan="1">3.11</td> <td valign="top" align="center" rowspan="1" colspan="1">1.26</td> <td valign="top" align="center" rowspan="1" colspan="1">0.24</td> <td valign="top" align="center" rowspan="1" colspan="1">0.59</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">19</td> <td valign="top" align="center" rowspan="1" colspan="1">Naphthalene</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> <td valign="top" align="center" rowspan="1" colspan="1">1.36</td> <td valign="top" align="center" rowspan="1" colspan="1" /> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">20</td> <td valign="top" align="center" rowspan="1" colspan="1">Humulene</td> <td valign="top" align="center" rowspan="1" colspan="1">1.76</td> <td valign="top" align="center" rowspan="1" colspan="1">3.41</td> <td valign="top" align="center" rowspan="1" colspan="1">5.49</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> <td valign="top" align="center" rowspan="1" colspan="1">0.59</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">21</td> <td valign="top" align="center" rowspan="1" colspan="1">Ã§-Elemene</td> <td valign="top" align="center" rowspan="1" colspan="1">1.25</td> <td valign="top" align="center" rowspan="1" colspan="1">15.01</td> <td valign="top" align="center" rowspan="1" colspan="1">4.43</td> <td valign="top" align="center" rowspan="1" colspan="1">0.86</td> <td valign="top" align="center" rowspan="1" colspan="1">3.36</td> <td valign="top" align="center" rowspan="1" colspan="1">1.18</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">22</td> <td valign="top" align="center" rowspan="1" colspan="1">Nerolidol</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> <td valign="top" align="center" rowspan="1" colspan="1">22.2</td> <td valign="top" align="center" rowspan="1" colspan="1">44.95</td> <td valign="top" align="center" rowspan="1" colspan="1">0.11</td> <td valign="top" align="center" rowspan="1" colspan="1">0.14</td> <td valign="top" align="center" rowspan="1" colspan="1">0.24</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">23</td> <td valign="top" align="center" rowspan="1" colspan="1">(âˆ’)-Spathulenol</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> <td valign="top" align="center" rowspan="1" colspan="1">1.76</td> <td valign="top" align="center" rowspan="1" colspan="1">3.72</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> <td valign="top" align="center" rowspan="1" colspan="1">0.94</td> <td valign="top" align="center" rowspan="1" colspan="1">0.41</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">24</td> <td valign="top" align="center" rowspan="1" colspan="1">Nerolidyl acetate</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> <td valign="top" align="center" rowspan="1" colspan="1">1.29</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">25</td> <td valign="top" align="center" rowspan="1" colspan="1">Guaiol</td> <td valign="top" align="center" rowspan="1" colspan="1">0.9</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> <td valign="top" align="center" rowspan="1" colspan="1">0.24</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">26</td> <td valign="top" align="center" rowspan="1" colspan="1">Caryophyllene oxide</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> <td valign="top" align="center" rowspan="1" colspan="1">1.17</td> <td valign="top" align="center" rowspan="1" colspan="1">1.1</td> <td valign="top" align="center" rowspan="1" colspan="1">0.28</td> <td valign="top" align="center" rowspan="1" colspan="1">0.34</td> <td valign="top" align="center" rowspan="1" colspan="1">0.58</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">27</td> <td valign="top" align="center" rowspan="1" colspan="1">Others and Unknown</td> <td valign="top" align="center" rowspan="1" colspan="1">1.97</td> <td valign="top" align="center" rowspan="1" colspan="1">20.1</td> <td valign="top" align="center" rowspan="1" colspan="1">28.2</td> <td valign="top" align="center" rowspan="1" colspan="1">3.52</td> <td valign="top" align="center" rowspan="1" colspan="1">2.69</td> <td valign="top" align="center" rowspan="1" colspan="1">3.41</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"><span class="attrib" tagx="attrib" title="attrib"><i><i>N: C. camphora with low oil yield; LI: C. camphora of linalool type; â€œâ€"â€� represents a compound not detected or in trace amounts (&amp;lt; 0.1%).</i></i></span></div> </div> </div> <div class="denovoassemblyofthec.camphoratranscriptome" title="sec"> <div class="title" tagx="title" title="title"><i>De novo</i> Assembly of the <i>C. camphora</i> Transcriptome</div> <p>To construct a genome-wide transcriptome of <i>C. camphora</i>, total RNA of the experimental samples was isolated. This pooled total RNA was sequenced and a total of 87.9 M 150 bp paired-end reads remained after raw reads were processed with sequence filtering and quality control (<a href="#TS2">Supplementary Table S2</a>). <i>De novo</i> assembly was carried out with short reads using Trinity (see section â€œMaterials and Methodsâ€�). A total of 121,285 unigenes were assembled with a high N50 value (1063), as well as high values for both the maximum unigene length (15,713) and percentage of mapped reads (&amp;gt;98%), indicating a good assembly quality (<a href="#T2">Table 2</a>). In addition, BUSCO analysis was performed to assess the completeness of the final genome assembly. Based on the 1,440 embryophytic-specfic genes, 79.8% were identified as complete genes, including 76.3% complete and single-copy genes and 3.5% complete and duplicated genes, and 8.8% were identified as fragmented genes, indicating good completeness of the genome assembly (<a href="#FS1">Supplementary Figure S1</a>).</p> <div class="table-wrap_UNKNOWN" id="T2" position="float"><span class="label" tagx="label" title="label">TABLE 2</span> <p>Statistics for <i>de novo</i> assembly of the <i>C. camphora</i> transcriptome.</p>  <table frame="hsides" rules="groups" cellspacing="5" cellpadding="5"> <tbody> <tr> <td valign="top" align="left" rowspan="1" colspan="1">Total assembled bases</td> <td valign="top" align="center" rowspan="1" colspan="1">87.9M</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">Total no. of unigenes</td> <td valign="top" align="center" rowspan="1" colspan="1">121,285</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">N50 (bp)</td> <td valign="top" align="center" rowspan="1" colspan="1">1,063</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">Shortest unigene length (bp)</td> <td valign="top" align="center" rowspan="1" colspan="1">201</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">Longest unigene length (bp)</td> <td valign="top" align="center" rowspan="1" colspan="1">15,713</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">Average unigene length (bp)</td> <td valign="top" align="center" rowspan="1" colspan="1">724</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">Mapped reads (%)</td> <td valign="top" align="center" rowspan="1" colspan="1">&amp;gt;98%</td> </tr> </tbody> </table> </div> </div> <div class="functionalannotationofunigenes" title="sec"> <div class="title" tagx="title" title="title">
Functional Annotation of Unigenes</div> <p>Functional annotation of 121,285 <i>C. camphora</i> unigenes was performed by comparing unigene sequences to the protein databases Nr and Swissprot with an <i>E</i>-value threshold of 1e<sup>â€"5</sup> using BLASTx analysis. This analysis yielded a total of 50,545 annotated genes, with 34,789 unigenes in common between two databases (<a href="#F1">Figure 1A</a>). A total of 49,295 unigenes had significant hits in the Nr database. Further GO annotation analysis assigned the 49,295 unigenes to three main categories, including biological process, cellular component, and molecular function (<a href="#F1">Figure 1B</a>). Within the biological process, â€œmetabolic processâ€� was highly represented (29.5%) followed by â€œcellular processâ€� (24.1%). Within the cellular component, â€œcellâ€� (23.7%), â€œcell partâ€� (23.7%), and â€œorganelleâ€� (16.0%) were most represented. Similarly, â€œcatalytic activityâ€� and â€œbindingâ€� were the most dominant terms under molecular function, accounting for 55.4 and 36.1%, respectively (<a href="#TS3">Supplementary Table S3</a>). To systematically analyze the metabolic pathways of unigene products in cells and predict the function of these unigene products, Kyoto Encyclopedia of Genes and Genomes (KEGG) analysis was performed, and 11,111 unigenes were annotated and classified into 19 metabolic pathways. The category â€œGlobal and Overviewâ€� included the largest number of unigenes (<a href="#F1">Figure 1C</a>), among which â€œmetabolic pathwaysâ€� and â€œbiosynthesis of secondary metabolitesâ€� were dominant, accounting for 40.8 and 22.2%, respectively (<a href="#TS4">Supplementary Table S4</a>). Additionally, 359 unigenes were placed into the category â€œmetabolism of terpenoids and polyketides,â€� which included â€œterpenoid backbone biosynthesisâ€� (32.3%), â€œmonoterpenoid biosynthesisâ€� (5.0%), â€œditerpenoid biosynthesisâ€� (13.4%), â€œsesquiterpenoid and triterpenoid biosynthesisâ€� (5.0%), â€œcarotenoid biosynthesisâ€� (15.3%), â€œbrassinosteroid biosynthesisâ€� (8.9%), â€œzeatin biosynthesisâ€� (8.4%), and â€œlimonene and pinene degradationâ€� (11.7%) (<a href="#TS4">Supplementary Table S4</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIGURE 1</span></a></div>   <p>Functional annotation and GO classification of assembled unigenes in <i>C. camphora</i>. <b>(A)</b> Functional annotation of assembled unigenes by BLASTx against Nr and Swissprot databases with an <i>E</i>-value threshold of 1e<sup>â€"5</sup>. A total of 34,789 unigenes were commonly annotated in the two databases. <b>(B)</b> GO terms of <i>C. camphora</i> unigenes determined by Blast2GO fell into the three categories of biological process, cellular component, and molecular function, where metabolic process, cell part, and catalytic activity were the three most prevalent subcategories, respectively. <b>(C)</b> KEGG classification of annotated unigenes. The <i>x-</i>axis shows the number of unigenes annotated, and the <i>y</i> axis shows the KEGG metabolic pathways the unigenes fell into, including organismal system (O), metabolism (M), genetic information processing (G), environmental information processing (E), and cellular processes (C).</p>   </div> </div> <div class="rna-seqanalysesofdegsbetweenc.camphoraleaveswithhighandlowoilcontents" title="sec"> <div class="title" tagx="title" title="title">
RNA-Seq Analyses of DEGs Between <i>C. camphora</i> Leaves With High and Low Oil Contents</div> <p>To get a better understanding of the dynamic processes of terpenoid biosynthesis in <i>C. camphora</i>, we performed transcriptome analyses of camphor leaves with high and low oil contents based on the oil yield and GC-MS results (<a href="#T1">Table 1</a>). Trimmed reads were mapped to the <i>de novo</i> transcriptome assembly of <i>C. camphora</i>. PCA demonstrated a good similarity between samples in the high- and low-oil-content groups (<a href="#FS2">Supplementary Figure S2A</a>). Similarly, the range of Pearson correlation coefficient between two samples in each experimental group is 0.93â€"0.96 (<a href="#FS2">Supplementary Figure S2B</a>), also indicating that the samples in the same groups are highly correlated. Further transcriptome analysis identified 3,689 DEGs with <i>p</i> &amp;lt; 0.01 and |log2 fold_change| &amp;gt; 2, which comprised 2,120 up-regulated and 1,569 down-regulated unigenes. The GO terms â€œmetabolic processâ€� in the â€œbiological processâ€� category and â€œcatalytic activityâ€� in the â€œmolecular functionâ€� category were the two most highly represented within DEGs (<a href="#FS3">Supplementary Figure S3</a>). In accordance with the GO analysis, KEGG enrichment analysis revealed that most of the 3,689 DEGs were in â€œmetabolismâ€� class (<a href="#F2">Figure 2A</a>), which included â€œbiosynthesis of secondary metabolites,â€� â€œphenylpropanoid biosynthesis,â€� â€œmonoterpenoid biosynthesis,â€� â€œterpenoid backbone biosynthesis,â€� and other pathways (<a href="#TS5">Supplementary Table S5</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIGURE 2</span></a></div>   <p>Transcriptional cascade of DEGs in terpenoid backbone biosynthesis in linalool-type <i>C. camphora</i>. <b>(A)</b> KEGG pathway annotations of DEGs. The <i>x-</i>axis indicates the number of unigenes annotated in each corresponding pathway. The five categories of KEGG pathways are divided and color-coded. <b>(B)</b> DEGs in terpenoid backbone biosynthesis. The transcriptional cascade of terpenoid backbone biosynthesis is shown in the form of protein-coding enzymes or nodes annotated by KEGG pathway analysis. Enzyme expression patterns are indicated with logRPKM values, and DEGs are in azure. The left three and right three columns represent low oil content type (N) and linalool type (LI), respectively. AACT, acetyl-CoA C-acetyltransferase; DXS, 1-deoxy- <div class="sc_UNKNOWN">
D</div>-xylulose-5-phosphate synthase; HMGS, hydroxymethylglutaryl-CoA synthase; PMK, phosphomevalonate kinase; MVD, mevalonate diphosphate decarboxylase; DXR, 1-deoxy-d-xylulose-5-phosphate reductoisomerase; HDS, hydroxymethylbutenyl 4-diphosphate synthase; HDR, (E)-4-hydroxy-3-methyl-but-2-enyl diphosphate reductase; IDI, isopentenyl-diphosphate delta-isomerase; GPPS, geranyl diphosphate synthase; FPPS, farnesyl pyrophosphate synthase; GGPPS, geranylgeranyl pyrophosphate synthase; LIS, linalool synthase; MNR1, (+)-neomenthol dehydrogenase-like isoform; GerS, geraniol synthase; GerD, (âˆ’)-germacrene D synthase; CLS, ent-copalyl diphosphate synthase; CYP88D6, cytochrome P450 88D6; GA2OX, Gibberellin 2-beta-dioxygenase. <b>(C)</b> qRT-PCR validation of RNA-Seq data. The black bars represent relative gene expression determined by qRT-PCR (left <i>y</i>-axis) and the dotted gray bars represent gene expression levels detected by RNA-Seq (right <i>y-</i>axis). Standard errors of the three technical replicates of each sample of the qRT-PCR analysis and three biological replicates in the two experimental groups of the RNA-Seq analysis are indicated by error bars.</p>   </div> </div> <div class="putativegenesinvolvedinterpenoidbackboneandterpenoidbiosynthesisinc.camphora" title="sec"> <div class="title" tagx="title" title="title">
Putative Genes Involved in Terpenoid Backbone and Terpenoid Biosynthesis in <i>C. camphora</i></div> <p>Annotated protein-coding enzymes were mapped into the terpenoid backbone biosynthesis pathway, and gene expression data of each enzyme in the two experimental groups were indicated with logRPKM values (<a href="#F2">Figure 2B</a> and <a href="#TS6">Supplementary Table S6</a>). Two unigenes encoding key enzymes, <i>AACT-like</i> in the MVA pathway and <i>DXS-like</i> in the MEP pathway, exhibited a higher expression level in the groups with high oil contents. Because both MVA and MEP pathways generate the precursors IPP and DMAPP for the final production of terpenoids, increased expression of these two genes could explain the higher amount of terpenoids we observed in this group (<a href="#T1">Table 1</a>). Additionally, the <i>GGPPS-like</i> gene also had a higher expression level in the leaves of linalool type. Among the DEGs, 14 unigenes were annotated as terpenoid synthaseâ€"encoding genes, including four in monoterpenoid biosynthesis, four in sesquiterpenoid and triterpenoid biosynthesis, and six in diterpenoid biosynthesis (<a href="#F2">Figure 2B</a> and <a href="#TS6">Supplementary Table S6</a>). One <i>LIS-like</i> and one <i>GerS-like</i> unigene were up-regulated in the monoterpenoid biosynthesis pathway, indicating that increased availability of terpenoid precursors, along with increased levels of monoterpene synthases, might be the reason for the higher oil yield and higher accumulation of linalool and other monoterpenes in the trees of linalool type. Besides monoterpenoid synthases, four sesquiterpenoid synthase unigenes encoding GerD and four diterpenoid synthase unigenes encoding CLS, CYP88D6, GA2OX1, and GA2OX8 were also up-regulated. In general, the upregulation of terpenoid synthase unigenes in the trees of linalool-type <i>C. camphor</i> is consistent with their increased accumulation of terpenoids, suggesting that the presence of various terpenoid synthases might be the reason for the occurrence of varied terpenoids in the essential oils of <i>C. camphora</i>. To validate the reliability of our RNA-Seq data, four DEGs were chosen for qRT-PCR examination. As shown in <a href="#F2">Figure 2C</a>, the expression profiles of these unigenes observed by qRT-PCR are identical to their profiles identified in the RNA-Seq analysis.</p> </div> </div> <div class="discussion" title="sec"> <div class="title" tagx="title" title="title">
Discussion</div> <p><i>C. camphora</i> oils have a wide variety of uses, including ornamental, medicinal, and industrial, which makes understanding the regulation of essential oil biosynthesis in camphor tree highly important (<a href="#B16">Guo et al., 2016</a>; <a href="#B19">Jiang et al., 2016</a>; <a href="#B6">Chen et al., 2018</a>). Because of earlier unregulated harvesting, many natural sources of camphor trees have been exhausted. In response to this, camphor tree plantations have been established in many countries including China and Japan. These camphor trees are maintained as bushes and are harvested for twigs and leaves before essential oil extraction. Camphor trees can be divided into different chemotypes based on their main terpenoid components. In this study, we identified camphor trees of linalool type and low oil contents without a dominant terpene component. A total of 26 main components were identified by GC-MS. We further demonstrated that the chemical components in the essential oils of <i>C. camphora</i> leaf extracts contained mainly monoterpenes and sesquiterpenes with very minor quantities of diterpenes (<a href="#T1">Table 1</a>). Moreover, linalool took up âˆ¼90 of the total essential oil contents in linalool type, indicating that monoterpene levels account for the major difference between the essential oils of different chemotypes.</p> <p>Terpenoid biosynthesis and its regulation are extensively documented in many plant species. Despite having high structural diversity, all terpenes are derived from two isomeric basic backbone molecules, IPP and DMAPP, which are synthesized either through the MVA or MEP pathways (<a href="#B34">Sacchettini and Poulter, 1997</a>). Although many members of the Lauraceae family are rich in terpenoids and are economically important tree species, limited studies are available analyzing genes involved in its terpenoid biosynthesis pathway (<a href="#B43">Yang et al., 2005</a>; <a href="#B23">Lin et al., 2014</a>; <a href="#B6">Chen et al., 2018</a>; <a href="#B8">Chen et al., 2020</a>). This study was focused on identifying genes involved in terpenoid biosynthesis and key enzymes involved in the biosynthesis of the backbone IPP and DMAPP molecules in <i>C. camphora</i>. A transcriptome of <i>C. camphora</i> was also generated to provide a reference for future RNA-Seqâ€"based expression profiling of terpenoid biosynthesis. Functional annotation analysis showed significant enrichment in genes associated with â€œmetabolic pathwaysâ€� and â€œbiosynthesis of secondary metabolitesâ€� (<a href="#F1">Figure 1</a>). This is consistent with our observation that Septemberâ€"November is the time of the year when <i>C. camphora</i> harbors the highest oil contents (<a href="#B30">Qin et al., 2015</a>), and it is also the harvesting time of our samples. Furthermore, 116 and 18 expressed unigenes were identified to be highly homologous with 49 and 11 known enzymes in the terpenoid backbone biosynthesis and monoterpenoid biosynthesis pathways, respectively.</p> <p>By comparing the transcriptome profiles of leaves of linalool and leaves with low oil contents, 3,689 unigenes were identified as DEGs. Among them, expression levels of <i>AACT-like</i> and <i>DXS-like</i>, catalyzing the first steps toward IPP and DMAPP in the MVA and MEP pathways, were up-regulated in the leaves of linalool type. <i>DXS</i> genes are believed to be a significant control point in the MEP pathway (<a href="#B41">Wolfertz et al., 2004</a>; <a href="#B33">Rohdich et al., 2006</a>), and manipulation of gene expression levels of <i>DXS</i> in <i>Arabidopsis</i> altered the levels of tocopherols, carotenoids, chlorophylls, abscisic acid, and various other isoprenoids (<a href="#B13">EstÃ©vez et al., 2001</a>). However, the contribution of each pathway to the generation of the common precursors for varied terpenoid biosynthesis is still unknown, because crosstalk between the two pathways has been documented in many species including chamomile, <i>Arabidopsis</i>, tobacco, and snapdragon (<a href="#B1">Adam and Zapp, 1998</a>; <a href="#B5">Bick and Lange, 2003</a>; <a href="#B18">Hemmerlin et al., 2003</a>; <a href="#B22">Laule et al., 2003</a>; <a href="#B12">Dudareva et al., 2005</a>). In addition, the expression level of <i>GGPPS-like</i> in the â€œterpenoid backbone biosynthesisâ€� pathway was up-regulated in the comparison, indicating increased supply of precursors for diterpenes biosynthesis. Although the percentage of diterpenes in leaf extracts of linalool-type camphor trees is not high, their oil yield is much higher than trees of low oil yield. Therefore, higher <i>GGPPS</i> expression level may simply be correlated with a general increase in oil content. Interestingly, it has been documented that GPP, the precursor for monoterpene, is constitutively and ubiquitously expressed in <i>Arabidopsis</i> (<a href="#B40">Van Schie et al., 2013</a>), and our results also showed that the level of <i>GPPS</i> was indifferent between the groups regardless of different accumulation of monoterpenes.</p> <p>Noticeably, <i>LIS-like</i> shared a high sequence identity (&amp;gt;99%) with some of the identified linalool synthases in the Lauraceae family. The conserved DDxxD and NSE/DTE motifs for Mg<sup>2+</sup> or Mn<sup>2+</sup> ion binding as well as the predicted D562 active site cleft were all properly distributed in the LIS-like protein (<a href="#FS4">Supplementary Figure S4</a> and <a href="#TS7">Supplementary Table S7</a>). However, caution should be used when assigning protein functions to terpenoid synthase genes predicted based on sequence similarities as it has been reported that point mutations alone were sufficient to significantly change the kinetic activity of a linalool synthase gene in <i>Cinnamomum osmophloeum</i> (<a href="#B23">Lin et al., 2014</a>). Additionally, alternative splicing has been shown to alter linalool biosynthesis, as reported in <i>Camellia sinensis</i> (<a href="#B24">Liu et al., 2018</a>). Higher levels of monoterpene production could potentially lower production of other terpenes and isoprenoid derivatives due to the competition for common precursors. Indeed, four unigenes in the â€œcarotenoid biosynthesis pathwayâ€� and two unigenes in the â€œsteroid biosynthesis pathwayâ€� were down-regulated in the comparison (<a href="#TS6">Supplementary Table S6</a>). Similar examples can be seen in tobacco plants producing monoterpene where lower levels of Î²-caryophyllene were detected in the flowers of these plants (<a href="#B25">LÃ¼cker et al., 2004</a>). Additionally, both transgenic tomato and <i>Arabidopsis</i> plants producing a phytoene synthase and the strawberry linalool/nerolidol synthase separately had significant growth retardation due to the severe inhibition of gibberellin biosynthesis (<a href="#B14">Fray et al., 1995</a>; <a href="#B2">Aharoni et al., 2004</a>).</p> <p>Other genes of interest emerged from this study, which include several members of the WRKY family. <i>WRKY21-like</i>, <i>WRKY47-like</i>, and <i>WRKY65-like</i> were all up-regulated in the leaves with high oil content (<a href="#TS6">Supplementary Table S6</a>). WRKY transcription factors have been reported to regulate the expression of genes involved in terpene biosynthesis in many species, including <i>Arabidopsis</i>, cotton, <i>Artemisia annua</i>, <i>Catharanthus roseus</i>, and others (<a href="#B42">Xu et al., 2004</a>; <a href="#B26">Ma et al., 2009</a>; <a href="#B28">Mao et al., 2011</a>; <a href="#B39">Suttipanta et al., 2011</a>). Moreover, it was suggested that the linalool-type <i>C. osmophloeum</i> contains a W-box element (T)(T)TGAC(C/T) in the promoter region of the linalool synthase gene, enabling regulation by WRKY transcription factors (<a href="#B23">Lin et al., 2014</a>). Based on these previous studies, we suspect that members of the WRKY family might be involved in the regulation of terpene synthase genes of <i>C. camphora</i> and therefore could potentially play critical roles in the quantitative difference of oil contents between different chemotypes.</p> <p>All in all, this study identified a cohort of genes in the terpenoid backbone biosynthesis and monoterpenoid biosynthesis pathways that were commonly up-regulated in leaves with high linalool contents compared to leaves with low oil contents. The knowledge obtained from this study could facilitate manipulation of camphor tree essential oil production through metabolic engineering of essential oil biosynthesis. Additional studies should focus on the isolation of monoterpene synthases genes from different chemotypes of <i>C. camphora</i>. Additionally, functional analyses are needed to demonstrate the impact of these genes on terpenoid production.</p> </div> <div class="data-availability" title="data-availability"> <div class="title" tagx="title" title="title">
Data Availability Statement</div> <p>Raw reads yielded from Illumina sequencing have been uploaded to the NCBI Sequence Read Archive (<a href="https://www.ncbi.nlm.nih.gov/sra">https://www.ncbi.nlm.nih.gov/sra</a>) and accession numbers for the six samples are as below: N-1 (<a href="SRR11362618">SRR11362618</a>); N-2 (<a href="SRR11362617">SRR11362617</a>); N-3 (<a href="SRR12576812">SRR12576812</a>); LI-1 (<a href="SRR11362614">SRR11362614</a>); LI-2 527 (<a href="SRR11362613">SRR11362613</a>); and LI-3 (<a href="SRR12576811">SRR12576811</a>).</p> </div> <div class="authorcontributions" title="sec"> <div class="title" tagx="title" title="title">
Author Contributions</div> <p>HZ, JH, and ZJ designed the experiments. BZ, JZ, XJ, and JH performed the experiments. HZ and JH analyzed the data. JH wrote the manuscript. All authors read and approved the final manuscript.</p> </div> <div class="conflictofinterest" title="sec"> <div class="title" tagx="title" title="title">
Conflict of Interest</div> <p>The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.</p> </div> </div> <div class="back" title="back"> <div class="fn-group" title="fn-group"> <div class="fn-type-financial-disclosure" title="financial-disclosure"> <p><b>Funding.</b> This work was supported by the National Natural Science Foundation of China (grant no. 31560242) and the Jiangxi Provincial Education Department Fund (grant no. GJJ190975).</p> </div> </div> <div class="ack" title="ack"> <p>We thank HuiJuan Cheng and Yiqun Zhang for their assistance in sample harvesting and essential oil extraction. We would also like to thank Yanbo Wang and reviewers for their help in improving the manuscript.</p> </div> <div class="supplementary material" title="supplementary material"> <div class="title" tagx="title" title="title">
Supplementary Material</div> <p>The Supplementary Material for this article can be found online at: <a href="https://www.frontiersin.org/articles/10.3389/fgene.2020.598714/full#supplementary-material">https://www.frontiersin.org/articles/10.3389/fgene.2020.598714/full#supplementary-material</a></p> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Supplementary Figure 1</span> <p>BUSCO analysis of the final transcriptomic assembly.</p>  <div class="media" title="media"><a href="Data_Sheet_1.docx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Supplementary Figure 2</span> <p>Relationship between the samples.</p>  <div class="media" title="media"><a href="Data_Sheet_1.docx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Supplementary Figure 3</span> <p>GO enrichment histogram of DEGs.</p>  <div class="media" title="media"><a href="Data_Sheet_1.docx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Supplementary Figure 4</span> <p>Amino acid sequence alignment of identified LIS-like in <i>C. camphora</i> with linalool synthase CoLIS-LL in <i>C. osmophloeum</i> (accession number: <a href="JX028232">JX028232</a>).</p>  <div class="media" title="media"><a href="Data_Sheet_1.docx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Supplementary Table 1</span> <p>Sequences of the gene-specific primers used in quantitative RT-PCR.</p>  <div class="media" title="media"><a href="Data_Sheet_2.xlsx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Supplementary Table 2</span> <p>Raw transcriptomic data.</p>  <div class="media" title="media"><a href="Data_Sheet_2.xlsx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Supplementary Table 3</span> <p>GO enrichment analysis of <i>C. camphora</i> unigenes.</p>  <div class="media" title="media"><a href="Data_Sheet_2.xlsx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Supplementary Table 4</span> <p>KEGG enrichment of <i>C. camphora</i> unigenes.</p>  <div class="media" title="media"><a href="Data_Sheet_2.xlsx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Supplementary Table 5</span> <p>KEGG enrichment of DEGs.</p>  <div class="media" title="media"><a href="Data_Sheet_2.xlsx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Supplementary Table 6</span> <p>Putative unigenes associated with terpenoid biosynthesis.</p>  <div class="media" title="media"><a href="Data_Sheet_2.xlsx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Supplementary Table 7</span> <p>Sequences of putative unigenes associated with terpenoid biosynthesis.</p>  <div class="media" title="media"><a href="Data_Sheet_2.xlsx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="B1" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Adam</span><span class="given-names" tagx="given-names" title="given-names">K.-P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zapp</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span> (<span class="year" tagx="year" title="year">1998</span>). <span class="mixed-article-title" title="mixed-article-title">Biosynthesis of the isoprene units of chamomile sesquiterpenes.</span><span class="source" tagx="source" title="source"><i>Phytochemistry</i></span><span class="volume" tagx="volume" title="volume">48</span><span class="fpage" tagx="fpage" title="fpage">953</span>â€"<span class="lpage" tagx="lpage" title="lpage">959</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/S0031-9422(97)00992-8">10.1016/S0031-9422(97)00992-8</a></span></span></li> <li tag="ref"><a name="B2" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aharoni</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Giri</span><span class="given-names" tagx="given-names" title="given-names">A. P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Verstappen</span><span class="given-names" tagx="given-names" title="given-names">F. W. A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bertea</span><span class="given-names" tagx="given-names" title="given-names">C. M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sevenier</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2004</span>). <span class="mixed-article-title" title="mixed-article-title">Gain and loss of fruit flavor compounds produced by wild and cultivated strawberry species.</span><span class="source" tagx="source" title="source"><i>Plant Cell</i></span><span class="volume" tagx="volume" title="volume">16</span><span class="fpage" tagx="fpage" title="fpage">3110</span>â€"<span class="lpage" tagx="lpage" title="lpage">3131</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1105/tpc.104.023895">10.1105/tpc.104.023895</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15522848">15522848</a></span></span></li> <li tag="ref"><a name="B3" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bairoch</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Apweiler</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span></span> (<span class="year" tagx="year" title="year">2000</span>). <span class="mixed-article-title" title="mixed-article-title">The SWISS-PROT protein sequence database and its supplement TrEMBL in 2000.</span><span class="source" tagx="source" title="source"><i>Nucleic Acids Res.</i></span><span class="volume" tagx="volume" title="volume">28</span><span class="fpage" tagx="fpage" title="fpage">45</span>â€"<span class="lpage" tagx="lpage" title="lpage">48</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/28.1.45">10.1093/nar/28.1.45</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10592178">10592178</a></span></span></li> <li tag="ref"><a name="B4" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Battaile</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Loomis</span><span class="given-names" tagx="given-names" title="given-names">W. D.</span></span></span> (<span class="year" tagx="year" title="year">1961</span>). <span class="mixed-article-title" title="mixed-article-title">Biosynthesis of terpenes: II. The site and sequence of terpene formation in peppermint.</span><span class="source" tagx="source" title="source"><i>BBA</i></span><span class="volume" tagx="volume" title="volume">51</span><span class="fpage" tagx="fpage" title="fpage">545</span>â€"<span class="lpage" tagx="lpage" title="lpage">552</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/0006-3002(61)90612-6">10.1016/0006-3002(61)90612-6</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/13865784">13865784</a></span></span></li> <li tag="ref"><a name="B5" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bick</span><span class="given-names" tagx="given-names" title="given-names">J. A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lange</span><span class="given-names" tagx="given-names" title="given-names">B. M.</span></span></span> (<span class="year" tagx="year" title="year">2003</span>). <span class="mixed-article-title" title="mixed-article-title">Metabolic cross talk between cytosolic and plastidial pathways of isoprenoid biosynthesis: unidirectional transport of intermediates across the chloroplast envelope membrane.</span><span class="source" tagx="source" title="source"><i>Arch. Biochem. Biophys.</i></span><span class="volume" tagx="volume" title="volume">415</span><span class="fpage" tagx="fpage" title="fpage">146</span>â€"<span class="lpage" tagx="lpage" title="lpage">154</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/s0003-9861(03)00233-9">10.1016/s0003-9861(03)00233-9</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12831836">12831836</a></span></span></li> <li tag="ref"><a name="B6" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zheng</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhong</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">L.-A.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2018</span>). <span class="mixed-article-title" title="mixed-article-title">Transcriptome analysis and identification of genes related to terpenoid biosynthesis in <i>Cinnamomum camphora</i>.</span><span class="source" tagx="source" title="source"><i>BMC Genomics</i></span><span class="volume" tagx="volume" title="volume">19</span>:<span class="issue" tagx="issue" title="issue">550</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1186/s12864-018-4941-1">10.1186/s12864-018-4941-1</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30041601">30041601</a></span></span></li> <li tag="ref"><a name="B7" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tholl</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dâ€™Auria</span><span class="given-names" tagx="given-names" title="given-names">J. C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Farooq</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pichersky</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gershenzon</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span> (<span class="year" tagx="year" title="year">2003</span>). <span class="mixed-article-title" title="mixed-article-title">Biosynthesis and emission of terpenoid volatiles from <i>Arabidopsis</i> flowers.</span><span class="source" tagx="source" title="source"><i>Plant Cell</i></span><span class="volume" tagx="volume" title="volume">15</span><span class="fpage" tagx="fpage" title="fpage">481</span>â€"<span class="lpage" tagx="lpage" title="lpage">494</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1105/tpc.007989">10.1105/tpc.007989</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12566586">12566586</a></span></span></li> <li tag="ref"><a name="B8" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gao</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">The Litsea genome and the evolution of the laurel family.</span><span class="source" tagx="source" title="source"><i>Nat. Commun.</i></span><span class="volume" tagx="volume" title="volume">11</span>:<span class="issue" tagx="issue" title="issue">1675</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/s41467-020-15493-5">10.1038/s41467-020-15493-5</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32245969">32245969</a></span></span></li> <li tag="ref"><a name="B9" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheng</span><span class="given-names" tagx="given-names" title="given-names">A.-X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lou</span><span class="given-names" tagx="given-names" title="given-names">Y.-G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mao</span><span class="given-names" tagx="given-names" title="given-names">Y.-B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lu</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">L.-J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">X.-Y.</span></span></span> (<span class="year" tagx="year" title="year">2007</span>). <span class="mixed-article-title" title="mixed-article-title">Plant terpenoids: biosynthesis and ecological functions.</span><span class="source" tagx="source" title="source"><i>J. Integr. Plant Biol.</i></span><span class="volume" tagx="volume" title="volume">49</span><span class="fpage" tagx="fpage" title="fpage">179</span>â€"<span class="lpage" tagx="lpage" title="lpage">186</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1744-7909.2007.00395.x">10.1111/j.1744-7909.2007.00395.x</a></span></span></li> <li tag="ref"><a name="B10" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Conesa</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">GÃ¶tz</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">GarcÃ­a-GÃ³mez</span><span class="given-names" tagx="given-names" title="given-names">J. M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Terol</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">TalÃ³n</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Robles</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span> (<span class="year" tagx="year" title="year">2005</span>). <span class="mixed-article-title" title="mixed-article-title">Blast2GO: a universal tool for annotation, visualization and analysis in functional genomics research.</span><span class="source" tagx="source" title="source"><i>Bioinformatics</i></span><span class="volume" tagx="volume" title="volume">21</span><span class="fpage" tagx="fpage" title="fpage">3674</span>â€"<span class="lpage" tagx="lpage" title="lpage">3676</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/bti610">10.1093/bioinformatics/bti610</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16081474">16081474</a></span></span></li> <li tag="ref"><a name="B11" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Davidson</span><span class="given-names" tagx="given-names" title="given-names">N. M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oshlack</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span> (<span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Corset: enabling differential gene expression analysis for de novo assembled transcriptomes.</span><span class="source" tagx="source" title="source"><i>Genome Biol.</i></span><span class="volume" tagx="volume" title="volume">15</span>:<span class="issue" tagx="issue" title="issue">410</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1186/s13059-014-0410-6">10.1186/s13059-014-0410-6</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25063469">25063469</a></span></span></li> <li tag="ref"><a name="B12" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dudareva</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Andersson</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Orlova</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gatto</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reichelt</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rhodes</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2005</span>). <span class="mixed-article-title" title="mixed-article-title">The nonmevalonate pathway supports both monoterpene and sesquiterpene formation in snapdragon flowers.</span><span class="source" tagx="source" title="source"><i>Proc. Nat. Acad. Sci. U.S.A.</i></span><span class="volume" tagx="volume" title="volume">102</span><span class="fpage" tagx="fpage" title="fpage">933</span>â€"<span class="lpage" tagx="lpage" title="lpage">938</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.0407360102">10.1073/pnas.0407360102</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15630092">15630092</a></span></span></li> <li tag="ref"><a name="B13" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">EstÃ©vez</span><span class="given-names" tagx="given-names" title="given-names">J. M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cantero</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reindl</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reichler</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">LeÃ³n</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span> (<span class="year" tagx="year" title="year">2001</span>). <span class="mixed-article-title" title="mixed-article-title">1-Deoxy-D-xylulose-5-phosphate synthase, a limiting enzyme for plastidic isoprenoid biosynthesis in plants.</span><span class="source" tagx="source" title="source"><i>J. Biol. Chem.</i></span><span class="volume" tagx="volume" title="volume">276</span><span class="fpage" tagx="fpage" title="fpage">22901</span>â€"<span class="lpage" tagx="lpage" title="lpage">22909</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1074/jbc.M100854200">10.1074/jbc.M100854200</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11264287">11264287</a></span></span></li> <li tag="ref"><a name="B14" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fray</span><span class="given-names" tagx="given-names" title="given-names">R. G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wallace</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fraser</span><span class="given-names" tagx="given-names" title="given-names">P. D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Valero</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hedden</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bramley</span><span class="given-names" tagx="given-names" title="given-names">P. M.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">1995</span>). <span class="mixed-article-title" title="mixed-article-title">Constitutive expression of a fruit phytoene synthase gene in transgenic tomatoes causes dwarfism by redirecting metabolites from the gibberellin pathway.</span><span class="source" tagx="source" title="source"><i>Plant J.</i></span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">693</span>â€"<span class="lpage" tagx="lpage" title="lpage">701</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1046/j.1365-313X.1995.08050693.x">10.1046/j.1365-313X.1995.08050693.x</a></span></span></li> <li tag="ref"><a name="B15" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grabherr</span><span class="given-names" tagx="given-names" title="given-names">M. G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haas</span><span class="given-names" tagx="given-names" title="given-names">B. J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yassour</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Levin</span><span class="given-names" tagx="given-names" title="given-names">J. Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thompson</span><span class="given-names" tagx="given-names" title="given-names">D. A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Amit</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2011</span>). <span class="mixed-article-title" title="mixed-article-title">Full-length transcriptome assembly from RNA-Seq data without a reference genome.</span><span class="source" tagx="source" title="source"><i>Nat. Biotechnol.</i></span><span class="volume" tagx="volume" title="volume">29</span><span class="fpage" tagx="fpage" title="fpage">644</span>â€"<span class="lpage" tagx="lpage" title="lpage">652</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/nbt.1883">10.1038/nbt.1883</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21572440">21572440</a></span></span></li> <li tag="ref"><a name="B16" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guo</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Geng</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liang</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Deng</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">The chemical composition of essential oils from <i>Cinnamomum camphora</i> and their insecticidal activity against the stored product pests.</span><span class="source" tagx="source" title="source"><i>Int. J. Mol. Sci.</i></span><span class="volume" tagx="volume" title="volume">17</span>:<span class="issue" tagx="issue" title="issue">1836</span>. <span class="pub-id"><a href="https://dx.doi.org/10.3390/ijms17111836">10.3390/ijms17111836</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27827929">27827929</a></span></span></li> <li tag="ref"><a name="B17" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guo</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cui</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Deng</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2017</span>). <span class="mixed-article-title" title="mixed-article-title">Molecular differentiation of five <i>Cinnamomum camphora</i> chemotypes using desorption atmospheric pressure chemical ionization mass spectrometry of raw leaves.</span><span class="source" tagx="source" title="source"><i>Sci. Rep.</i></span><span class="volume" tagx="volume" title="volume">7</span>:<span class="issue" tagx="issue" title="issue">46579</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/srep46579">10.1038/srep46579</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28425482">28425482</a></span></span></li> <li tag="ref"><a name="B18" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hemmerlin</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hoeffler</span><span class="given-names" tagx="given-names" title="given-names">J.-F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meyer</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tritsch</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kagan</span><span class="given-names" tagx="given-names" title="given-names">I. A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grosdemange-Billiard</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2003</span>). <span class="mixed-article-title" title="mixed-article-title">Cross-talk between the cytosolic mevalonate and the plastidial methylerythritol phosphate pathways in tobacco bright yellow-2 cells.</span><span class="source" tagx="source" title="source"><i>J. Biol. Chem.</i></span><span class="volume" tagx="volume" title="volume">278</span><span class="fpage" tagx="fpage" title="fpage">26666</span>â€"<span class="lpage" tagx="lpage" title="lpage">26676</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1074/jbc.M302526200">10.1074/jbc.M302526200</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12736259">12736259</a></span></span></li> <li tag="ref"><a name="B19" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiang</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Song</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cao</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yao</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tang</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">GCÃ—GC-TOFMS Analysis of essential oils composition from leaves, twigs and seeds of Cinnamomum camphora L. Presl and their insecticidal and repellent activities.</span><span class="source" tagx="source" title="source"><i>Molecules</i></span><span class="volume" tagx="volume" title="volume">21</span><span class="fpage" tagx="fpage" title="fpage">423</span>â€"<span class="lpage" tagx="lpage" title="lpage">423</span>. <span class="pub-id"><a href="https://dx.doi.org/10.3390/molecules21040423">10.3390/molecules21040423</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27043503">27043503</a></span></span></li> <li tag="ref"><a name="B20" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiang</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xiao</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xiong</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span></span> (<span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Transcriptome analysis for leaves of five chemical types in <i>Cinnamomum camphora</i>.</span><span class="source" tagx="source" title="source"><i>Yi chuan</i></span><span class="volume" tagx="volume" title="volume">36</span><span class="fpage" tagx="fpage" title="fpage">58</span>â€"<span class="lpage" tagx="lpage" title="lpage">68</span>. <span class="pub-id"><a href="https://dx.doi.org/10.3724/sp.j.1005.2014.00058">10.3724/sp.j.1005.2014.00058</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24846919">24846919</a></span></span></li> <li tag="ref"><a name="B21" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lange</span><span class="given-names" tagx="given-names" title="given-names">B. M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ghassemian</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span> (<span class="year" tagx="year" title="year">2003</span>). <span class="mixed-article-title" title="mixed-article-title">Genome organization in Arabidopsis thaliana: a survey for genes involved in isoprenoid and chlorophyll metabolism.</span><span class="source" tagx="source" title="source"><i>Plant Mol. Biol.</i></span><span class="volume" tagx="volume" title="volume">51</span><span class="fpage" tagx="fpage" title="fpage">925</span>â€"<span class="lpage" tagx="lpage" title="lpage">948</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1023/a:1023005504702">10.1023/a:1023005504702</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12777052">12777052</a></span></span></li> <li tag="ref"><a name="B22" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Laule</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">FÃ¼rholz</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chang</span><span class="given-names" tagx="given-names" title="given-names">H.-S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heifetz</span><span class="given-names" tagx="given-names" title="given-names">P. B.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2003</span>). <span class="mixed-article-title" title="mixed-article-title">Crosstalk between cytosolic and plastidial pathways of isoprenoid biosynthesis in <i>Arabidopsis thaliana</i>.</span><span class="source" tagx="source" title="source"><i>PNAS</i></span><span class="volume" tagx="volume" title="volume">100</span><span class="fpage" tagx="fpage" title="fpage">6866</span>â€"<span class="lpage" tagx="lpage" title="lpage">6871</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.1031755100">10.1073/pnas.1031755100</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12748386">12748386</a></span></span></li> <li tag="ref"><a name="B23" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lin</span><span class="given-names" tagx="given-names" title="given-names">Y.-L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">Y.-R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">W.-K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chang</span><span class="given-names" tagx="given-names" title="given-names">S.-T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chu</span><span class="given-names" tagx="given-names" title="given-names">F.-H.</span></span></span> (<span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Characterization of S-(+)-linalool synthase from several provenances of <i>Cinnamomum osmophloeum</i>.</span><span class="source" tagx="source" title="source"><i>Tree Genet Genomes</i></span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">75</span>â€"<span class="lpage" tagx="lpage" title="lpage">86</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1007/s11295-013-0665-1">10.1007/s11295-013-0665-1</a></span></span></li> <li tag="ref"><a name="B24" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">G.-F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">J.-J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">He</span><span class="given-names" tagx="given-names" title="given-names">Z.-R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">F.-M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yan</span><span class="given-names" tagx="given-names" title="given-names">Y.-F.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2018</span>). <span class="mixed-article-title" title="mixed-article-title">Implementation of CsLIS/NES in linalool biosynthesis involves transcript splicing regulation in <i>Camellia sinensis</i>.</span><span class="source" tagx="source" title="source"><i>Plant Cell Environ.</i></span><span class="volume" tagx="volume" title="volume">41</span><span class="fpage" tagx="fpage" title="fpage">176</span>â€"<span class="lpage" tagx="lpage" title="lpage">186</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/pce.13080">10.1111/pce.13080</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28963730">28963730</a></span></span></li> <li tag="ref"><a name="B25" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">LÃ¼cker</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schwab</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van Hautum</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Blaas</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van der Plas</span><span class="given-names" tagx="given-names" title="given-names">L. H. W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bouwmeester</span><span class="given-names" tagx="given-names" title="given-names">H. J.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2004</span>). <span class="mixed-article-title" title="mixed-article-title">Increased and altered fragrance of tobacco plants after metabolic engineering using three monoterpene synthases from lemon.</span><span class="source" tagx="source" title="source"><i>Plant Physiol.</i></span><span class="volume" tagx="volume" title="volume">134</span><span class="fpage" tagx="fpage" title="fpage">510</span>â€"<span class="lpage" tagx="lpage" title="lpage">519</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1104/pp.103.030189">10.1104/pp.103.030189</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14718674">14718674</a></span></span></li> <li tag="ref"><a name="B26" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ma</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pu</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lei</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ma</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guo</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2009</span>). <span class="mixed-article-title" title="mixed-article-title">Isolation and characterization of AaWRKY1, an Artemisia annua transcription factor that regulates the amorpha-4,11-diene synthase gene, a key gene of artemisinin biosynthesis.</span><span class="source" tagx="source" title="source"><i>Plant Cell Physiol.</i></span><span class="volume" tagx="volume" title="volume">50</span><span class="fpage" tagx="fpage" title="fpage">2146</span>â€"<span class="lpage" tagx="lpage" title="lpage">2161</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/pcp/pcp149">10.1093/pcp/pcp149</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19880398">19880398</a></span></span></li> <li tag="ref"><a name="B27" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ma</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yuan</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">X. E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lu</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span> (<span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">Genome-wide identification and characterization of novel genes involved in terpenoid biosynthesis in <i>Salvia miltiorrhiza</i>.</span><span class="source" tagx="source" title="source"><i>J. Exp. Bot.</i></span><span class="volume" tagx="volume" title="volume">63</span><span class="fpage" tagx="fpage" title="fpage">2809</span>â€"<span class="lpage" tagx="lpage" title="lpage">2823</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/jxb/err466">10.1093/jxb/err466</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22291132">22291132</a></span></span></li> <li tag="ref"><a name="B28" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mao</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meng</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zheng</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span> (<span class="year" tagx="year" title="year">2011</span>). <span class="mixed-article-title" title="mixed-article-title">Phosphorylation of a WRKY transcription factor by two pathogen-responsive MAPKs drives phytoalexin biosynthesis in <i>Arabidopsis</i>.</span><span class="source" tagx="source" title="source"><i>Plant Cell</i></span><span class="volume" tagx="volume" title="volume">23</span><span class="fpage" tagx="fpage" title="fpage">1639</span>â€"<span class="lpage" tagx="lpage" title="lpage">1653</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1105/tpc.111.084996">10.1105/tpc.111.084996</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21498677">21498677</a></span></span></li> <li tag="ref"><a name="B29" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oâ€™Leary</span><span class="given-names" tagx="given-names" title="given-names">N. A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wright</span><span class="given-names" tagx="given-names" title="given-names">M. W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brister</span><span class="given-names" tagx="given-names" title="given-names">J. R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ciufo</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haddad</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McVeigh</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Reference sequence (RefSeq) database at NCBI: current status, taxonomic expansion, and functional annotation.</span><span class="source" tagx="source" title="source"><i>Nucleic Acids Res.</i></span><span class="volume" tagx="volume" title="volume">44</span><span class="fpage" tagx="fpage" title="fpage">D733</span>â€"<span class="lpage" tagx="lpage" title="lpage">D745</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gkv1189">10.1093/nar/gkv1189</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26553804">26553804</a></span></span></li> <li tag="ref"><a name="B30" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qin</span><span class="given-names" tagx="given-names" title="given-names">Z.-H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">J.-F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shi</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lu</span><span class="given-names" tagx="given-names" title="given-names">K.-C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tang</span><span class="given-names" tagx="given-names" title="given-names">F.-C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">H.-L.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">Content and main compositions analysis of essential oil in branches and leaves of <i>Cinnamomum camphora</i> in different months (in Chinese).</span><span class="source" tagx="source" title="source"><i>Guangxi For. Sci.</i></span><span class="volume" tagx="volume" title="volume">44</span><span class="fpage" tagx="fpage" title="fpage">428</span>â€"<span class="lpage" tagx="lpage" title="lpage">430</span>. <span class="pub-id"><a href="https://dx.doi.org/10.19692/j.cnki.gfs.2015.04.021">10.19692/j.cnki.gfs.2015.04.021</a></span></span></li> <li tag="ref"><a name="B31" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reumann</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Babujee</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ma</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wienkoop</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Siemsen</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Antonicelli</span><span class="given-names" tagx="given-names" title="given-names">G. E.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2007</span>). <span class="mixed-article-title" title="mixed-article-title">Proteome analysis of <i>Arabidopsis</i> leaf peroxisomes reveals novel targeting peptides, metabolic pathways, and defense mechanisms.</span><span class="source" tagx="source" title="source"><i>Plant Cell</i></span><span class="volume" tagx="volume" title="volume">19</span><span class="fpage" tagx="fpage" title="fpage">3170</span>â€"<span class="lpage" tagx="lpage" title="lpage">3193</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1105/tpc.107.050989">10.1105/tpc.107.050989</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17951448">17951448</a></span></span></li> <li tag="ref"><a name="B32" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Robinson</span><span class="given-names" tagx="given-names" title="given-names">M. D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McCarthy</span><span class="given-names" tagx="given-names" title="given-names">D. J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smyth</span><span class="given-names" tagx="given-names" title="given-names">G. K.</span></span></span> (<span class="year" tagx="year" title="year">2010</span>). <span class="mixed-article-title" title="mixed-article-title">edgeR: a Bioconductor package for differential expression analysis of digital gene expression data.</span><span class="source" tagx="source" title="source"><i>Bioinformatics</i></span><span class="volume" tagx="volume" title="volume">26</span><span class="fpage" tagx="fpage" title="fpage">139</span>â€"<span class="lpage" tagx="lpage" title="lpage">140</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btp616">10.1093/bioinformatics/btp616</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19910308">19910308</a></span></span></li> <li tag="ref"><a name="B33" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rohdich</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lauw</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kaiser</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Feicht</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">KÃ¶hler</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bacher</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2006</span>). <span class="mixed-article-title" title="mixed-article-title">Isoprenoid biosynthesis in plants â€" 2C-methyl-d-erythritol-4-phosphate synthase (IspC protein) of <i>Arabidopsis thaliana</i>.</span><span class="source" tagx="source" title="source"><i>FEBS J.</i></span><span class="volume" tagx="volume" title="volume">273</span><span class="fpage" tagx="fpage" title="fpage">4446</span>â€"<span class="lpage" tagx="lpage" title="lpage">4458</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1742-4658.2006.05446.x">10.1111/j.1742-4658.2006.05446.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16972937">16972937</a></span></span></li> <li tag="ref"><a name="B34" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sacchettini</span><span class="given-names" tagx="given-names" title="given-names">J. C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Poulter</span><span class="given-names" tagx="given-names" title="given-names">C. D.</span></span></span> (<span class="year" tagx="year" title="year">1997</span>). <span class="mixed-article-title" title="mixed-article-title">Creating isoprenoid diversity.</span><span class="source" tagx="source" title="source"><i>Science</i></span><span class="volume" tagx="volume" title="volume">277</span><span class="fpage" tagx="fpage" title="fpage">1788</span>â€"<span class="lpage" tagx="lpage" title="lpage">1789</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1126/science.277.5333.1788">10.1126/science.277.5333.1788</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9324768">9324768</a></span></span></li> <li tag="ref"><a name="B35" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sando</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Takaoka</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mukai</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yamashita</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hattori</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ogasawara</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2008a</span>). <span class="mixed-article-title" title="mixed-article-title">Cloning and characterization of mevalonate pathway genes in a natural rubber producing plant, <i>Hevea brasiliensis</i>.</span><span class="source" tagx="source" title="source"><i>Biosci. Biotechnol. Biochem.</i></span><span class="volume" tagx="volume" title="volume">72</span><span class="fpage" tagx="fpage" title="fpage">2049</span>â€"<span class="lpage" tagx="lpage" title="lpage">2060</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1271/bbb.80165">10.1271/bbb.80165</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18685207">18685207</a></span></span></li> <li tag="ref"><a name="B36" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sando</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Takeno</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Watanabe</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Okumoto</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuzuyama</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yamashita</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2008b</span>). <span class="mixed-article-title" title="mixed-article-title">Cloning and characterization of the 2-C-methyl-D-erythritol 4-phosphate (MEP) pathway genes of a natural-rubber producing plant, <i>Hevea brasiliensis</i>.</span><span class="source" tagx="source" title="source"><i>Biosci. Biotechnol. Biochem.</i></span><span class="volume" tagx="volume" title="volume">72</span><span class="fpage" tagx="fpage" title="fpage">2903</span>â€"<span class="lpage" tagx="lpage" title="lpage">2917</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1271/bbb.80387">10.1271/bbb.80387</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18997428">18997428</a></span></span></li> <li tag="ref"><a name="B37" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simao</span><span class="given-names" tagx="given-names" title="given-names">F. A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Waterhouse</span><span class="given-names" tagx="given-names" title="given-names">R. M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ioannidis</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kriventseva</span><span class="given-names" tagx="given-names" title="given-names">E. V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zdobnov</span><span class="given-names" tagx="given-names" title="given-names">E. M.</span></span></span> (<span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs.</span><span class="source" tagx="source" title="source"><i>Bioinformatics</i></span><span class="volume" tagx="volume" title="volume">31</span><span class="fpage" tagx="fpage" title="fpage">3210</span>â€"<span class="lpage" tagx="lpage" title="lpage">3212</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btv351">10.1093/bioinformatics/btv351</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26059717">26059717</a></span></span></li> <li tag="ref"><a name="B38" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simkin</span><span class="given-names" tagx="given-names" title="given-names">A. J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guirimand</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Papon</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Courdavault</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thabet</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ginis</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2011</span>). <span class="mixed-article-title" title="mixed-article-title">Peroxisomal localisation of the final steps of the mevalonic acid pathway in planta.</span><span class="source" tagx="source" title="source"><i>Planta</i></span><span class="volume" tagx="volume" title="volume">234</span>:<span class="issue" tagx="issue" title="issue">903</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1007/s00425-011-1444-6">10.1007/s00425-011-1444-6</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21655959">21655959</a></span></span></li> <li tag="ref"><a name="B39" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suttipanta</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pattanaik</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kulshrestha</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Patra</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Singh</span><span class="given-names" tagx="given-names" title="given-names">S. K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yuan</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span></span> (<span class="year" tagx="year" title="year">2011</span>). <span class="mixed-article-title" title="mixed-article-title">The transcription factor CrWRKY1 positively regulates the terpenoid indole alkaloid biosynthesis in <i>Catharanthus roseus</i>.</span><span class="source" tagx="source" title="source"><i>Plant Physiol.</i></span><span class="volume" tagx="volume" title="volume">157</span><span class="fpage" tagx="fpage" title="fpage">2081</span>â€"<span class="lpage" tagx="lpage" title="lpage">2093</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1104/pp.111.181834">10.1104/pp.111.181834</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21988879">21988879</a></span></span></li> <li tag="ref"><a name="B40" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Van Schie</span><span class="given-names" tagx="given-names" title="given-names">C. C. N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haring</span><span class="given-names" tagx="given-names" title="given-names">M. A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schuurink</span><span class="given-names" tagx="given-names" title="given-names">R. C.</span></span></span> (<span class="year" tagx="year" title="year">2013</span>). â€œ<span class="mixed-article-title" title="mixed-article-title">Prenyldiphosphate synthases and gibberellin biosynthesis</span>,â€� in <span class="source" tagx="source" title="source"><i>Isoprenoid Synthesis in Plants and Microorganisms: New Concepts and Experimental Approaches</i></span>, <span class="role" tagx="role" title="role">eds</span><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bach</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rohmer</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span> (<span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">New York</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">Springer</span>), <span class="fpage" tagx="fpage" title="fpage">214</span>â€"<span class="lpage" tagx="lpage" title="lpage">220</span>.</span></li> <li tag="ref"><a name="B41" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wolfertz</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sharkey</span><span class="given-names" tagx="given-names" title="given-names">T. D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boland</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">KÃ¼hnemann</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span></span> (<span class="year" tagx="year" title="year">2004</span>). <span class="mixed-article-title" title="mixed-article-title">Rapid regulation of the methylerythritol 4-Phosphate pathway during isoprene synthesis.</span><span class="source" tagx="source" title="source"><i>Plant Physiol.</i></span><span class="volume" tagx="volume" title="volume">135</span><span class="fpage" tagx="fpage" title="fpage">1939</span>â€"<span class="lpage" tagx="lpage" title="lpage">1945</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1104/pp.104.043737">10.1104/pp.104.043737</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15286290">15286290</a></span></span></li> <li tag="ref"><a name="B42" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">Y.-H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">J.-W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">J.-Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">X.-Y.</span></span></span> (<span class="year" tagx="year" title="year">2004</span>). <span class="mixed-article-title" title="mixed-article-title">Characterization of GaWRKY1, a cotton transcription factor that regulates the sesquiterpene synthase gene (+)-delta-cadinene synthase-A.</span><span class="source" tagx="source" title="source"><i>Plant Physiol.</i></span><span class="volume" tagx="volume" title="volume">135</span><span class="fpage" tagx="fpage" title="fpage">507</span>â€"<span class="lpage" tagx="lpage" title="lpage">515</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1104/pp.104.038612">10.1104/pp.104.038612</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15133151">15133151</a></span></span></li> <li tag="ref"><a name="B43" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">H.-X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zeng</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span></span> (<span class="year" tagx="year" title="year">2005</span>). <span class="mixed-article-title" title="mixed-article-title">A geraniol-synthase gene from <i>Cinnamomum tenuipilum</i>.</span><span class="source" tagx="source" title="source"><i>Phytochemistry</i></span><span class="volume" tagx="volume" title="volume">66</span><span class="fpage" tagx="fpage" title="fpage">285</span>â€"<span class="lpage" tagx="lpage" title="lpage">293</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.phytochem.2004.12.004">10.1016/j.phytochem.2004.12.004</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15680985">15680985</a></span></span></li> <li tag="ref"><a name="B44" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ye</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fang</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zheng</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2006</span>). <span class="mixed-article-title" title="mixed-article-title">WEGO: a web tool for plotting GO annotations.</span><span class="source" tagx="source" title="source"><i>Nucleic Acids Res.</i></span><span class="volume" tagx="volume" title="volume">34</span><span class="fpage" tagx="fpage" title="fpage">W293</span>â€"<span class="lpage" tagx="lpage" title="lpage">W297</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gkl031">10.1093/nar/gkl031</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16845012">16845012</a></span></span></li> <li tag="ref"><a name="B45" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xiao</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cao</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2019</span>). <span class="mixed-article-title" title="mixed-article-title">Chemical constituents and chemotypes of fresh leaf essential oil of wild species belonging to Sect. Camphor (Trew.) Meissn. in southeastern China.</span><span class="source" tagx="source" title="source"><i>J. Essent.</i></span><span class="volume" tagx="volume" title="volume">22</span><span class="fpage" tagx="fpage" title="fpage">1115</span>â€"<span class="lpage" tagx="lpage" title="lpage">1122</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1080/0972060X.2019.1662331">10.1080/0972060X.2019.1662331</a></span></span></li> <li tag="ref"><a name="B46" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zulak</span><span class="given-names" tagx="given-names" title="given-names">K. G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bohlmann</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span> (<span class="year" tagx="year" title="year">2010</span>). <span class="mixed-article-title" title="mixed-article-title">Terpenoid biosynthesis and specialized vascular cells of conifer defense.</span><span class="source" tagx="source" title="source"><i>J. Integr. Plant Biol.</i></span><span class="volume" tagx="volume" title="volume">52</span><span class="fpage" tagx="fpage" title="fpage">86</span>â€"<span class="lpage" tagx="lpage" title="lpage">97</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1744-7909.2010.00910.x">10.1111/j.1744-7909.2010.00910.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20074143">20074143</a></span></span></li> </ul> </div> </div>  </body></html>