<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Sci Rep</span><span class="iso-abbrev" title="iso-abbrev">Sci Rep</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Scientific Reports</span></div><span class="issn-epub" tagx="issn" title="issn-epub">2045-2322</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Nature Publishing Group UK</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/8115310">8115310</a></span><span class="publisher-id" title="publisher-id">89557</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1038/s41598-021-89557-x">10.1038/s41598-021-89557-x</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Molecular docking analysis and evaluation of the antibacterial and antioxidant activities of the constituents of <i>Ocimum cufodontii</i></div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aliye</span><span class="given-names" tagx="given-names" title="given-names">Muhdin</span></span><a href="#Aff1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dekebo</span><span class="given-names" tagx="given-names" title="given-names">Aman</span></span><a href="#Aff1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tesso</span><span class="given-names" tagx="given-names" title="given-names">Hailemichael</span></span><a href="#Aff1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abdo</span><span class="given-names" tagx="given-names" title="given-names">Teshome</span></span><a href="#Aff1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eswaramoorthy</span><span class="given-names" tagx="given-names" title="given-names">Rajalakshmanan</span></span><a href="#Aff1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Melaku</span><span class="given-names" tagx="given-names" title="given-names">Yadessa</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">yadessamelaku2010@gmail.com</span></span><a href="#Aff1" /></span><span class="citation_author_institution" id="Aff1">[], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.442848.6</span><span class="institution-id" title="institution-id">0000 0004 0570 6336</span><span class="institution" title="institution">Department of Applied Chemistry, School of Applied Natural Science, </span><span class="institution" title="institution">Adama Science and Technology University, </span></div></span></div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2021-5-5</span></span><span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2021-5-5</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2021</span></span><span class="volume" tagx="volume" title="volume">11</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">10101</span><span class="history" title="history"><span class="received" title="received">received: 2020-10-16</span><span class="accepted" title="accepted">accepted: 2021-4-26</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , </span><span class="license" title="license"> <div tag="license_ref">
https://creativecommons.org/licenses/by/4.0/</div><span class="license-p" title="license-p"><b>Open Access</b> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>.</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p id="Par1"><i>Ocimum cufodontii</i> ((Lanza) A.J.Paton) has been traditionally used in Ethiopia against bacteria<i>.</i> The extracts of the leaves and roots of <i>O. cufodontii</i> after silica gel column chromatography furnished compounds <b>1â€"5</b>, compounds <b>3</b> and <b>4</b> are new natural products. The oil from the hydro-distillation of the leaves, after analyzed with GCâ€"MS, has led to the identification of <i>Î²</i>-caryophyllene as a principal component, suggesting the essential oil as medicine and spices to enhance the taste of food. The constituents of <i>O. cufodontii</i> were assessed for their antibacterial activity against <i>E. coli, K. pneumonia, S. typhymurium</i> and <i>S. aureus.</i> The best activity was displayed against <i>S. aureus</i> by the hexane extract of the roots, compound <b>4</b>, and the essential oil with an inhibition zone of 17, 15, and 19Â mm, respectively. Molecular docking analysis revealed that compound <b>1</b> has better docking efficiency and forms hydrophobic interactions with five amino acids (ARG192, PHE196, GLU185, GLU193, and LYS189). This suggests that the compounds may act as potential inhibitors of DNA gyrase. The constituents were also assessed for their antioxidant activities using DPPH, ferric thicyanate and ferric reducing power assay. The hexane extracts of the roots inhibited the DPPH radical and peroxide formation by 90.5 and 83%, respectively, suggesting the potential of the extract as an antioxidant. Furthermore, the hexane extract of the roots of <i>O. cufodontii</i> exhibited the maximum reducing power compared with the EtOAc and methanol extracts. Hence, the activity displayed herein indicated as the plant has great potential as a remedy for diseases caused by bacteria and radicals.</p> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Subject terms</div> <div class="kwd" title="kwd">
Medicinal chemistry</div> <div class="kwd" title="kwd">
Chemistry</div> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="issue-copyright-statement" value="Â© The Author(s) 2021" /> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="sec"> <div class="title" tagx="title" title="title">
Introduction</div> <p id="Par2">Since ancient time, plants of the genus <i>Ocimum</i> (Lamiaceae) are commercially significant because of their nutritional, aromatic, ornamental, and medicinal importance<sup><a href="#CR1">1</a>,<a href="#CR2">2</a></sup>. <i>Ocimum</i> has been a key traditional medicine in various parts of the world to treat fevers, stomachache<sup><a href="#CR3">3</a></sup>, headaches, coughs, diarrhea and as stimulants and an emetic<sup><a href="#CR2">2</a>,<a href="#CR4">4</a></sup>. A lot of research is being done to test the biological effects of some species of the genus <i>Ocimum</i>. Scientific sources revealed <i>Ocimum</i> has antidiabetic, antipyretic, wound healing, antioxidant, hypolipidemic, bactericidal, and larvicidal activities<sup><a href="#CR4">4</a>â€"<a href="#CR9">9</a></sup>. The genus <i>Ocimum</i> is strongly aromatic because of essential oils which comprise monoterpenes, sesquiterpenes, and phenylpropanoids<sup><a href="#CR10">10</a></sup>. It is also recognized as a safe and rich source of phenolic antioxidants and flavonoids<sup><a href="#CR11">11</a></sup>. The essential oils from plants of this genus are important in pharmaceutical, flavoring, perfumery, fragrance, and cosmetic industries<sup><a href="#CR12">12</a>,<a href="#CR13">13</a></sup>.</p> <p id="Par3">The genus <i>Ocimum</i> comprises over 150 recognized species of herbs and shrubs widely distributed in various parts of the world<sup><a href="#CR14">14</a></sup>. <i>O. cufodontii</i>, <i>O. basilicum</i>, <i>O. sanctum</i>, <i>O. citriodorum,</i> and <i>O. gratissimum</i> are among few <i>Ocimum</i> species commonly found in Ethiopia. <i>O. cufodontii</i> is a popular hedge plant in Ethiopia, traditionally been used to relieve pain, fever, and inflammation besides its use as antibacterial agent<i>. O. cufodontii</i>, locally named in Ethiopia as â€œ<i>Gororsisa</i>â€� in Afan Oromo, is distributed in Borana, Harar, Arsi, and Bale<sup><a href="#CR15">15</a></sup>. The plant is also found in East Africa including Somalia and Kenya. Despite the extensive traditional use of this plant against various life-threatening diseases, there is no scientific study on the roots and leaves of <i>O. cufodontii</i>. Hence, in view of the attributed medicinal significance of the plant and its availability as an indigenous resource, the present work was undertaken to explore the chemical constituents and some biological activities of the leaves and roots of <i>O. cufodontii</i>. Therefore, this paper describes for the first time the molecular docking analysis, antibacterial, and antioxidant activities of the extracts and constituents of the leaves and roots of <i>O. cufodontii</i>.</p> </div> <div class="materialsandmethods" title="sec"> <div class="title" tagx="title" title="title">
Materials and methods</div> <div class="generalexperimentalprocedure" title="sec"> <div class="title" tagx="title" title="title">
General experimental procedure</div> <p id="Par4">Melting points of the isolated compounds were done using melting point apparatus (BUCHI, Germany). Analytical TLC was run on a 0.25Â mm thick layer of silica gel GF254 (Merck) on aluminium plate. Detection of spots on TLC was done either by spraying with vanillin or by observation under UV lamp. Purification of compounds was done using column chromatography over silica gel (230â€"400 mesh). The UV and NMR spectra were measured using UVâ€"Vis spectrophotometer (PG instruments, UK) and Bruker Avance 400 spectrometer operating at 400Â MHz, respectively. Mass spectra were run on a JMS HX-110 double focussing mass spectrophotometer, using MeOH as a solvent and glycerol as a matrix on the target. Samples were ionized by bombardment with xenon (gas) atom. Methanol (99.99% GC grade, United Kingdom), <i>n</i>-hexane (99% Ranchem industry and trading), Ethyl acetate (98% UNI-CHEM chemical reagents), Ethanol (100% GC grade Mumbai, India) and DPPH (Sigma) were purchased from Addis Ababa (NEWAY PLC, chemicals).</p> </div> <div class="plantmaterialcollectionandidentification" title="sec"> <div class="title" tagx="title" title="title">
Plant material collection and identification</div> <p id="Par5">The roots and leaves of <i>O. cufodontii</i> were collected on October 15, 2017 from Merti, Arsi Zone, Oromia region, Ethiopia. <i>O. cufodontii</i> is a wild growing plant collected after getting permission from Merti district administration office, Arsi Zone of Oromia Regional State. The collection of the plant material comply with institutional, national and international guidelines including IUCN Policy. The plant was identified by Mr. Shambel Alemu at the Biology Department and specimen stored in the National Herbarium of Addis Ababa University with a voucher number MA001. The collected plant material was air dried and grinded to a uniform size using an electric grinder.</p> </div> <div class="extraction" title="sec"> <div class="title" tagx="title" title="title">
Extraction</div> <p id="Par6">Ground roots of <i>O. cufodontii</i> (300Â g) were successively extracted on maceration with each 1.5 L of <i>n</i>-hexane, EtOAc and MeOH for 72Â h following the standard procedure in the literature with slight modifications<sup><a href="#CR16">16</a></sup>. Each extracts was filtered and concentrated under vacuum using rotary evaporator at 40Â Â°C to furnish 1.5, 1.2 and 3%, respectively. Likewise, the leaves (300Â g) were also extracted following same procedure as above to afford 2, 1.5 and 4% of the <i>n</i>-hexane, EtOAc, and MeOH extract respectively.</p> </div> <div class="isolationofcompounds" title="sec"> <div class="title" tagx="title" title="title">
Isolation of compounds</div> <p id="Par7">The EtOAc extract (3Â g) of the leaves of <i>O. cufodontii</i> was adsorbed and fractionated over silica gel (150Â g) column chromatography with <i>n</i>-hexane:EtOAc of increasing polarities as eluent to furnish 58 fractions, each 50Â mL. Fraction eluted with <i>n</i>-hexane:EtOAc (7:3) gave compound <b>1</b> (40Â mg).</p> <p id="Par8">The <i>n</i>-hexane extract (3.0Â g) of the roots of <i>O. cufodontii</i> was also fractionated over silica gel (150Â g) column chromatography with increasing polarity of EtOAc in <i>n</i>-hexane as eluent to give 24 fractions, each 100Â mL. Fraction 4 eluted with <i>n</i>-hexane:EtOAc (85:15) furnished compound <b>2</b> (32Â mg) as yellow crystals. Fraction 10 eluted with <i>n</i>-hexane:EtOAc (4:1) is identified as compound <b>3</b> (32Â mg).</p> <p id="Par9">The EtOAc extract (3.0Â g) of the roots of <i>O. cufodontii</i> was adsorbed and subjected to silica gel (150Â g) column chromatography with increasing polarities of EtOAc in <i>n</i>-hexane as eluent to give 41 fractions, each 50Â mL. Fraction 7 eluted with <i>n</i>-hexane:EtOAc (9:1) afford compound <b>4</b> (30Â mg). On the other hand, fraction 37 and 38 eluted with <i>n</i>-hexane:EtOAc (3:7) were merged to give compound <b>5</b> (23Â mg).</p> </div> <div class="extractionofessentialoilfromtheleaves" title="sec"> <div class="title" tagx="title" title="title">
Extraction of essential oil from the leaves</div> <p id="Par10">The ground leaves of <i>O. cufodontii</i> (60Â g) were hydro-distilled in a Clevengerâ€™s apparatus for 4Â h. The less dense yellowish oil separated using separatory funnel was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, and kept in refrigerator until analysis. Experiments were conducted in triplicate.</p> </div> <div class="gcâ€" title="sec"> <div class="title" tagx="title" title="title">
GCâ€"MS analyses of the essential oil of the leaves of <i>O. cufodontii</i></div> <p id="Par11">GCâ€"MS analysis of the essential oil was carried out by following the method described in Hema<sup><a href="#CR17">17</a></sup>. A GCâ€"MS instrument from Agilent Technologies (Santa Clara, CA, USA) equipped with a 6890Â N network GC system, 5975 inert mass selective detector, 7683B series auto sampler injector (10 Î¼L in size), G1701DA GC/MSD Chem Station and HP5MS column (30Â m lengthâ€‰Ã—â€‰0.25Â mm internal diameterâ€‰Ã—â€‰0.25Â Î¼m film thickness) coated with 5% phenyl 95% methyl poly siloxane was used for analyzing the samples. 2 Î¼L essential oil solutions in chloroform was injected through auto sampler and analyzed with HP5MS column. Column temperature was programed as follows: 55â€"120Â Â°C at 20Â Â°C/min, 120â€"150Â Â°C at 1.5Â Â°C/min, 150â€"250Â Â°C at 20Â Â°C/min, 250Â Â°C (10Â min) and 3Â min solvent delay. The mass spectra transfer line temperature was 280Â Â°C. The carrier gas was helium (1Â mL/min) with a split ratio equal to 100:1. The mass spectra were recorded in electron ionization mode at 70Â eV with scanning from 50 to 500 amu (atomic mass unit) at 0.5Â s with the mass source being set at 230Â Â°C. The relative % amount of each component was calculated by comparing its average peak area to the total area Identification of the constituents was done with the aid of NIST 2005 library of mass spectra.</p> </div> <div class="antibacterialactivity" title="sec"> <div class="title" tagx="title" title="title">
Antibacterial activity</div> <p id="Par12">The antibacterial activity of the extracts and isolated compounds were tested using Mueller Hinton agar medium following previously reported protocol with slight modification<sup><a href="#CR15">15</a></sup>. The bacterial strains including <i>Escherichia coli, Klebsella pneumonia, Salmonella typhymurium</i> and <i>Staphylococcus aureus</i> were the American type culture collection obtained from Oromia Public Health Research, Capacity Building and Quality Assurance Laboratory Center, Adama, Ethiopia. Wells have been made in nutrient agar plate using cork borer (6Â mm diameter) and inoculums containing 10<sup>6</sup>Â CFU/mL of bacteria have been spread on the solid plates with a sterile swab moistened with the bacterial suspension and each 50Â Âµg/mL of the extracts and isolated compounds were filled in the wells with the help of micropipette. Plates have been left for some time till the samples diffuse in the medium with the lid closed and incubated at 37Â Â°C for 24Â h. After overnight incubation, the plates were observed for the zone of inhibition (ZI) and the diameter of the inhibition zone was measured using scale. Then the samples were analyzed in triplicates and expressed as meanâ€‰Â±â€‰SD. Ciprofloxacin and DMSO were used as positive and negative controls, respectively.</p> </div> <div class="moleculardockingstudiesoftheisolatedcompounds" title="sec"> <div class="title" tagx="title" title="title">
Molecular docking studies of the isolated compounds</div> <p id="Par13">AutoDock Vina with previously reported protocol<sup><a href="#CR17">17</a></sup> was used to dock the proteins (PDB ID: 6F86) and isolated compounds (<b>1â€"5</b>) into the active site of proteins<sup><a href="#CR18">18</a>,<a href="#CR19">19</a></sup>. The chemical structures of compounds <b>1â€"5</b> were drawn using ChemOffice tool (Chem Draw 16.0) assigned with proper 2D orientation, and energy of each molecule was minimized using ChemBio3D. The energy minimized ligand molecules were then used as input for AutoDock Vina, in order to carry out the docking simulation. The crystal structure of receptor molecule <i>E.Â coli</i>Â gyraseB (PDB ID: 6F86) were downloaded from protein data bank. The protein preparation was done using the reported standard protocol<sup><a href="#CR20">20</a></sup> by removing the co-crystallized ligand, selected water molecules and cofactors. Next, the target protein file was prepared by leaving the associated residue with protein by using Auto Preparation of target protein file Auto Dock 4.2 (MGL tools1.5.6). The graphical user interface program was used to set the grid box for docking simulations. The grid was set so that it surrounds the region of interest in the macromolecule. The docking algorithm provided with Auto Dock Vina was used to search for the best docked conformation between ligand and protein. During the docking process, a maximum of nine conformers were considered for each ligand. The conformations with the most favorable (least) free binding energy were selected for analyzing the interactions between the target receptor and ligands by Discovery studio visualizer and PyMOL. The ligands are represented in different color, H-bonds and the interacting residues are represented in ball and stick model representation.</p> </div> <div class="antioxidantactivity" title="sec"> <div class="title" tagx="title" title="title">
Antioxidant activity</div> <div class="2,2-diphenyl-1-picrylhydrazyl(dpph)radicalscavengingassay" title="sec"> <div class="title" tagx="title" title="title">
2,2-Diphenyl-1-picrylhydrazyl (DPPH) radical scavenging assay</div> <p id="Par14">The radical scavenging activity of the samples were estimated using 2,2-diphenyl-2-picrylhydrazyl (DPPH) assay<sup><a href="#CR21">21</a></sup>. Briefly, the methanol solution of the hexane extract was serially diluted with 0.004% DPPH in methanol to furnish 200, 100, 50, and 25Â Âµg/mL of the extract. After 30Â min incubation in an oven at 37Â Â°C, absorbance at 517Â nm was measured using UVâ€"Vis spectrophotometer. Ascorbic acid and 0.004% DPPH in methanol were used as positive control and blank, respectively. The percentage inhibition of the extracts was calculated using the following formula. % Inhibitionâ€‰=â€‰(A<sub>control</sub>Â âˆ’Â A<sub>extract</sub>)/A<sub>control</sub>â€‰Ã—â€‰100 where Acontrol is the absorbance of 0.004% DPPH in methanol and Aextract is the absorbance of DPPH solution plus sample. The DPPH radical scavenging activity of the samples was also expressed as IC<sub>50</sub>, the concentration of the test compound to give a 50% decrease of the absorbance from that of the control solution. The radical scavenging activity of the other extracts and the isolated compounds of the leaves and the roots of <i>O. cufodontii</i> were also evaluated following the same. Ascorbic acid and quercetin were used as positive controls.</p> </div> <div class="ferricthiocyanatemethod" title="sec"> <div class="title" tagx="title" title="title">
Ferricthiocyanate method</div> <p id="Par15">The lipid peroxidation inhibitory potential of the samples was evaluated using ferricthiocyanate method<sup><a href="#CR22">22</a></sup>. In brief, the hexane extract (0.1Â mg) of the leaves of <i>O. cufodontii</i>, linoleic acid (100 ÂµL), EtOH (5Â mL) and phosphate buffer (5Â mL, 0.05Â M, pHâ€‰=â€‰7) in water were mixed and incubated at 40Â Â°C in an oven. After 24Â h, 0.1Â mL was taken and added in to a vial containing 75% aqueous EtOH (7Â mL), 30% of NH<sub>4</sub>SCN (0.15Â mL) and 0.15Â mL of 0.02Â M FeCl<sub>2</sub> in 3.5% HCl. The absorbance of the red coloured solution was measured at 500Â nm using UVâ€"Vis spectrophotometer. This was repeated every 24Â h until the control gave its maximal absorbance value. Likewise, the lipid peroxidation inhibitory activity of the EtOAc extract of the leaves, the hexane and EtOAc extracts of the roots, and isolated compounds were assessed following the same procedure. Absorbance of the blank and ascorbic acid was measured in the same fashion. The percentage inhibition using ferricthiocyante method has been calculated according to the following formula. </p><div class="disp-formula" title="disp-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M1">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$${\text{Percentage inhibition}} = 100 - \left( {\frac{{{\text{As}}}}{{{\text{Ab}}}} \times 100} \right)\% ,$$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
Percentage inhibition</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
100</div> <div class="mo_UNKNOWN">
-</div> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mtext_UNKNOWN">
As</div> <div class="mtext_UNKNOWN">
Ab</div> </div> <div class="mo_UNKNOWN">
Ã—</div> <div class="mn_UNKNOWN">
100</div> </div> </div> <div class="mo_UNKNOWN">
%</div> <div class="mo_UNKNOWN">
,</div> </div></div><div class="graphic" position="anchor" /> </div> </div>where As is absorbance of the sample and Ab is absorbance of the blank<sup><a href="#CR23">23</a></sup>.<p /> </div> <div class="ferric-reducingantioxidantpower(frap)assay" title="sec"> <div class="title" tagx="title" title="title">
Ferric-reducing antioxidant power (FRAP) assay</div> <p id="Par16">The ferric reducing antioxidant power assay of the extracts and isolated compounds were measured according to Birasuren et al.<sup><a href="#CR24">24</a></sup>. The FRAP reagent was prepared by mixing acetate buffer (25Â mL, 0.3Â M), TPTZ (2,4,6-tripyridyl-s-triazine) solution (2.5Â mL), and FeCl<sub>3</sub>âˆ™6H<sub>2</sub>O solution (2.5Â mL), and then heated to 37Â Â°C before use. A properly diluted sample (0.1Â mL, 100Â Âµg/mL) was mixed with FRAP reagent (4.0Â mL) to form a mixture. This was incubated at 37Â Â°C for 10Â min in the dark, and the absorbance was measured at 593Â nm against a blank (distilled water). The results were expressed as aqueous solution of ferrous sulfate (FeSO<sub>4</sub>âˆ™7H<sub>2</sub>O), and derived from a calibration curve of the standards. Likewise, the FRAP value of ascorbic acid was obtained by the same procedure.</p> </div> </div> <div class="statisticsdataanalysis" title="sec"> <div class="title" tagx="title" title="title">
Statistics data analysis</div> <p id="Par17">The antibacterial and antioxidant assay data generated by triplicate measurements were reported using meanâ€‰Â±â€‰standard deviation. Analyses were performed using GraphPad Prism version 5.00 for Windows, (GraphPad Software, San Diego California USA, <a href="http://www.graphpad.com">www.graphpad.com</a>â€�). Groups were analyzed for significant differences using a linear model of variance analysis (ANOVA) test for comparisons, with level significance at pâ€‰<div class="â€‰0.05_UNKNOWN" href="#MOESM1">
Supplementary Information).</div></p> </div> </div> <div class="resultsanddiscussion" title="sec"> <div class="title" tagx="title" title="title">
Results and discussion</div> <div class="characterizationofisolatedcompounds" title="sec"> <div class="title" tagx="title" title="title">
Characterization of isolated compounds</div> <p id="Par18">In the present work, five compounds from the roots and leaves of <i>O. cufodontii</i> were isolated and characterized (Fig.Â <a href="#Fig1">1</a>). The structure elucidations of these compounds are described herein. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1</span></a></div>   <p>Structures of compounds 1â€"5 drawn by ChemDraw Pro 16.0 Suite (PerkinElmer, USA).</p>   </div> <p /> <p id="Par19">Compound <b>1</b> (40Â mg) was isolated as a white solid melting at 295â€"296Â Â°C. TLC showed a spot at Rf 0.47 with <i>n</i>-hexane:EtOAc (7:3) as eluent which was similar to the one reported for betulinic acid<sup><a href="#CR25">25</a></sup>. The mass spectrum of <b>1</b> displayed a molecular ion peak (Mâˆ’H)<sup>+</sup> at m/z 455 corresponding to the formula C<sub>30</sub>H<sub>48</sub>O<sub>3</sub>. The <sup>1</sup>H-NMR (400Â MHz, CDCl<sub>3</sub>) spectrum of <b>1</b> showed signals attributable to an exo-methylene group at <i>Î´</i> 4.64/4.51 (2H, <i>d</i>, <i>J</i>â€‰=â€‰2.0, H-29) which together with an allylic methyl at <i>Î´</i> 1.61 (3H, <i>s</i>, Me) indicated an isopropenyl function. The spectrum also showed signals for five tertiary methyl groups at <i>Î´</i> 1.18, 0.74, 0.67, 0.87 and 0.67, one isopropenyl moiety at <i>Î´</i> 1.61, 4.51 and 4.64, indicating a lupane-type skeleton. The signals at <i>Î´</i> 1.18 (3H, <i>s</i>) and <i>Î´</i> 0.74 (3H, <i>s</i>) were due to methyl protons at C-23 and C-24, respectively. In addition, the spectrum also showed a multiplet at <i>Î´</i> 2.93 (1H, <i>m</i>, H-19) and 1.50 (1H, <i>m</i>, H-18). These spectral features are in close agreement to those reported for betulinic acid<sup><a href="#CR26">26</a></sup>. The <sup>13</sup>C-NMR spectrum of compound <b>1</b> displayed the presence of 30 carbon signals of which six methyl, six quaternary, one carboxylic acid, eleven methylene, and six methine, confirming that compound <b>1</b> is betulinic acid<sup><a href="#CR27">27</a></sup>.</p> <p id="Par20">Compound <b>2</b> (32Â mg) was isolated as a yellow solid from <i>n</i>-hexane extract of the root of <i>O. cufodontii</i>. The MS spectrum revealed the molecular ion peak <i>m/z</i> 345 [Mâ€‰+â€‰H]<sup>+</sup> attributed to the molecular formula C<sub>21</sub>H<sub>29</sub>O<sub>4</sub>. The UVâ€"Vis spectrum (MeOH) showed absorption maxima at 140, 240 and 290Â nm suggesting a quinone skeleton<sup><a href="#CR28">28</a></sup>. The <sup>1</sup>H-NMR spectral data revealed the presence of two methoxy signals <i>Î´</i> 3.90 and 3.91. The absence of methine proton signals in the olefinic region justify that the carbons in this regions are all quaternary. The spectrum also displayed signals due to methylene protons at <i>Î´</i> 1.33, 1.47, 1.94 and 1.98. Four methyl signals are evident at <i>Î´</i> 0.92, 0.99, 1.11, 1.17, 1.59 and 1.69. The <sup>I3</sup>C-NMR spectral data together with DEPT-135 revealed the presence of twenty one carbon resonances: two carbonyls (<i>Î´</i> 189.7 and <i>Î´</i> 184.2), two oxygenated quaternary carbons (<i>Î´</i> 157.1 and 150.1), two aliphatic methine carbon (<i>Î´</i> 33.0 and 41.2), three non-oxygenated quaternary carbons (<i>Î´</i> 140.4, <i>Î´</i> 135.6 and 45.9), two methoxy carbons (<i>Î´</i> 60.7 and <i>Î´</i> 62.7), four methylene carbons (<i>Î´</i> 18.9, <i>Î´</i> 35.9, <i>Î´</i> 33.2 and <i>Î´</i> 41.2), and six methyls at <i>Î´</i> 18.6, 20.3, 20.5, 21.7, 26.0 and 26.1. The spectral data of <b>2</b> agreed well with the quinone whose structure is depicted in Fig.Â <a href="#Fig1">1</a>.</p> <p id="Par21">Compound <b>3</b> (38Â mg) was isolated as a red crystals melted at 85â€"86Â Â°C. The MS spectrum revealed the molecular ion peak at <i>m/z</i> 330.2 [Mâˆ’H] <sup>+</sup> attributed to the molecular formula C<sub>19</sub>H<sub>25</sub>O<sub>4</sub>N. The UVâ€"Vis spectrum (Methanol) showed absorption maxima at 340 and 540Â nm, indicating the presence of conjugated chromophore in the structure. The <sup>1</sup>H-NMR spectrum (400Â MHz, CDCl<sub>3</sub>) showed signals due to four methyls at <i>Î´</i> 1.12 (6H, <i>d</i>, <i>J</i>â€‰=â€‰6.94Â Hz, H-5â€³, 6â€²) and <i>Î´</i> 1.08 (6H, <i>d</i>, <i>J</i>â€‰=â€‰6.77Â Hz, H-6â€², 7â€²). Another methyl group is observed at <i>Î´</i> 2.32 (3H, <i>s</i>, H-6a). The signals due to two methylene protons were apparent at <i>Î´</i> 3.12 (2H, <i>t</i>, <i>J</i>â€‰=â€‰6.84, H-1â€³) and <i>Î´</i> 2.67 (2H, <i>t</i>, <i>J</i>â€‰=â€‰6.84, H-2â€³). Signals due to three olefinic protons were evident at <i>Î´</i> 7.07 (1H, <i>s</i>, H-3â€²), 7. 04 (1H, <i>d, J</i>â€‰=â€‰8.42Â Hz, H-3), and 7.35 (1H, <i>d</i>, <i>J</i>â€‰=â€‰8.42Â Hz, H-4). The <sup>13</sup>C-NMR spectrum (Table <a href="#Tab1">1</a>) together with DEPT-135 NMR spectrum revealed the presence of seven quaternary, five methine, two methylene, and five methyl groups. The presence of three carbonyls are also evident at <i>Î´</i> 213.6 (C-3â€³), 181.2 (C-1â€²) and 182.3 (C-4â€²). The two methylenes were observed at <i>Î´</i> 24.9 (C-1â€³) and <i>Î´</i> 38.5 (C-2â€³). The spectrum also displayed signals due to quaternary carbons at <i>Î´</i> 147.0 (C-2), 144.7 (C-6), 134.9 (C-2â€²), and <i>Î´</i> 128.4 (C-5). The signal due to two methine group on aromatic ring were evident at <i>Î´</i> 128.5 (C-3) and <i>Î´</i> 137.0 (C-4). The data generated enable us to propose the structure of compound <b>3</b> as shown in Fig.Â <a href="#Fig1">1</a>. <div class="table-wrap_UNKNOWN" id="Tab1"><span class="label" tagx="label" title="label">Table 1</span> </div></p><p><sup>1</sup>H (400Â MHz, CDCl<sub>3</sub>) and <sup>13</sup>C NMR (100Â MHz, CDCl<sub>3</sub>) data of compounds 3.</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="left">No</th> <th align="left"><i>Î´</i><sub>H</sub> NMR data (<i>Î´</i> in ppm, J in Hz)</th> <th align="left"><i>Î´</i><sub>C</sub> NMR data (<i>Î´</i> in ppm)</th> </tr> </thead> <tbody> <tr> <td align="left">6a</td> <td align="left">2. 3 (3H, <i>s</i>)</td> <td align="left">19.7</td> </tr> <tr> <td align="left">2</td> <td align="left">â€"</td> <td align="left">147.0</td> </tr> <tr> <td align="left">3</td> <td align="left">7.04 (1H, <i>d</i>, <i>J</i>â€‰=â€‰8.42Â Hz)</td> <td align="left">128.5</td> </tr> <tr> <td align="left">4</td> <td align="left">7.35 (1H, <i>d, J</i>â€‰=â€‰8.42Â Hz)</td> <td align="left">137.0</td> </tr> <tr> <td align="left">5</td> <td align="left">â€"</td> <td align="left">128.5</td> </tr> <tr> <td align="left">6</td> <td align="left">â€"</td> <td align="left">144.7</td> </tr> <tr> <td align="left">1â€²</td> <td align="left">â€"</td> <td align="left">181.2</td> </tr> <tr> <td align="left">2â€²</td> <td align="left">â€"</td> <td align="left">134.9</td> </tr> <tr> <td align="left">3â€²</td> <td align="left">7.04 (1H, <i>s</i>)</td> <td align="left">140.1</td> </tr> <tr> <td align="left">4â€²</td> <td align="left">â€"</td> <td align="left">182.3</td> </tr> <tr> <td align="left">5â€²</td> <td align="left">2.70 (1H, <i>septet</i>)</td> <td align="left">40.7</td> </tr> <tr> <td align="left">6â€²</td> <td align="left">1.08 (3H, <i>d</i>, <i>J</i>â€‰=â€‰6.77Â Hz)</td> <td align="left">21.4</td> </tr> <tr> <td align="left">7â€²</td> <td align="left">1.08 (3H, <i>d</i>, <i>J</i>â€‰=â€‰6.77Â Hz)</td> <td align="left">21.4</td> </tr> <tr> <td align="left">1â€³</td> <td align="left">3.19 (2H, <i>t</i>, <i>J</i>â€‰=â€‰6.81Â Hz)</td> <td align="left">24.9</td> </tr> <tr> <td align="left">2â€³</td> <td align="left">2.67 (2H,t)</td> <td align="left">38.5</td> </tr> <tr> <td align="left">3â€³</td> <td align="left">â€"</td> <td align="left">213.6</td> </tr> <tr> <td align="left">4â€³</td> <td align="left">2.98 (1H, <i>septe</i>t)</td> <td align="left">26.9</td> </tr> <tr> <td align="left">5â€³</td> <td align="left">1.13 , 6H, <i>d</i>, <i>J</i>â€‰=â€‰6.94Â Hz)</td> <td align="left">18.2</td> </tr> <tr> <td align="left">6â€³</td> <td align="left">1.13 , 6H, <i>d</i>, <i>J</i>â€‰=â€‰6.94Â Hz)</td> <td align="left">19.7</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p>Î´ in ppm and J in Hz.</p> </div>  <p /> <p id="Par22">The structure of compound <b>3</b> was further confirmed using 2D NMR. The COSY spectrum clearly indicates a correlation between the proton at <i>Î´</i> 7.04 and <i>Î´</i> 7.34. Also observed correlation in the COSY spectrum is the proton at <i>Î´</i> 2.98 with 1.12 (H-5â€² with H-6â€², 7â€²) and <i>Î´</i> 2.67 with 1.08 (H-4â€³ with H-5â€³, 6â€³). The HSQC spectrum showed the proton signals at <i>Î´</i> 7.34, 7.07, 7.04, 3.19, 2.98, 2.70, 2.67, 2.30, 1.12 and 1.08 correlate with the carbon signals at <i>Î´</i> 137.0, 140.1, 128.5, 24.9, 26.9, 40.7, 38.5, 19.7, 21.4 and 18.2, respectively. In the HMBC spectrum, the proton signal at <i>Î´</i> 1.08 correlates with the carbon <i>Î´</i> 40.7, 213.6 and 24.9. The signal at <i>Î´</i> 2.7 correlates with the carbon signals at <i>Î´</i> 24.9 and 213.6 (Fragment 1). The spectrum also revealed a correlation between the proton at <i>Î´</i> 1.12 with the carbon at <i>Î´</i> 26 and 181 (Fragment 2). While the proton at <i>Î´</i> 2.98 correlates with the carbon at <i>Î´</i> 181.0. </p><div class="graphic" position="anchor" id="MO2" /> <p /> <p id="Par23">Correlations are also observed in the HMBC spectrum due to the proton at <i>Î´</i> 7.04 with the carbon at <i>Î´</i> 140.2 and 128. The proton at <i>Î´</i> 7.34 correlates with the carbon at <i>Î´</i> 147, 134, and 19. Therefore, compound <b>3</b> is a novel compound isolated for the first time from the roots of <i>O. cufodontii</i> with its structure depicted in Fig.Â <a href="#Fig1">1</a>.</p> <p id="Par24">Compound <b>4</b> (29Â mg) was isolated as a brown solid from the EtOAc extract of the root of <i>O. cufodontii</i>. The mass spectrum revealed the molecular ion peak at <i>m/z</i> 330.3 [M]<sup>+</sup> attributed to the molecular formula C<sub>20</sub>H<sub>28</sub>O<sub>3</sub>N. The UVâ€"Vis (Methanol) showed absorption maxima at 240 and 340Â nm, signifying the presence of conjugated chromophore in the structure. In the <sup>1</sup>H-NMR spectrum (400Â MHz, CDCl<sub>3</sub>) of <b>4,</b> signals due to two methyl groups at <i>Î´</i> 1.21 (3H, d, H-4â€²a) and 1.25 (3H, <i>d</i>, 4â€²b) along with the signal due to methine proton at <i>Î´</i> 1.21 (1H, <i>septet</i>, H-4â€²) suggests the presence of isopropyl group. The presence of two methyl groups on quaternary carbons are evident at <i>Î´</i> 1.04, 0.87 (6H, <i>s</i>, H-3â€³a, 3â€³b). The protons at <i>Î´</i> 7.21 (1H, <i>d</i>, <i>J</i>â€‰=â€‰7.6Â Hz, H-4), <i>Î´</i> 7.08 (1H, <i>d</i>, <i>J</i>â€‰=â€‰0.8Â Hz, H-7) and <i>Î´</i> 7.04 (1H, <i>d</i>, <i>J</i>â€‰=â€‰7.6Â Hz, H-5) indicate the presence of a 1,3-disubsituted aromatic ring. Three methylene protons are evident at <i>Î´</i> 1.40 (2H, <i>m</i>, H-1â€²), <i>Î´</i> 1.98 (2H, <i>m,</i> H-2â€²) and <i>Î´</i> 3.00 (2H, <i>m</i>, H-1â€³). The spectrum also displayed signal due to methine proton and methyl protons at <i>Î´</i> 2.90 (1 H, <i>m,</i> H-3) and 2.40 (3 H, <i>s,</i> H-2a), respectively.</p> <p id="Par25">The proton decoupled <sup>13</sup>C-NMR spectrum with the aid of DEPT-135 revealed the presence of seven quaternary, five methine, three methylene, and five methyl groups. The spectrum (Table <a href="#Tab2">2</a>) revealed the presence of two carbonyl carbons at <i>Î´</i><sub>C</sub> 210.5 (C-3â€²) and 203.5 (C-2â€³). The three methylene groups are apparent at <i>Î´</i> 40.3 (C-2â€²), 22.0 (C-1â€²) and 29.7 (C-1â€³). The sp2 quaternary carbons at <i>Î´</i> 140.5, 138.3, 137.9 and 121.1 were attributed to C-2, C-8a, C-6 and C-8, respectively. The other oxygenated sp3 quaternary carbon appeared at <i>Î´</i> 81.9 (C-3â€³). Two methine carbons are also evident at <i>Î´</i> 27.3 (C-4â€²) and 55.7 (C-3). The data generated led us to the identification of compound <b>4</b> as shown in Fig.Â <a href="#Fig1">1</a>. <div class="table-wrap_UNKNOWN" id="Tab2"><span class="label" tagx="label" title="label">Table 2</span> </div></p><p><sup>1</sup>H (CDCl<sub>3</sub>, 400Â MHz) and <sup>13</sup>C NMR (CDCl<sub>3</sub>, 100Â MHz) data of compounds 4.</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="left">No</th> <th align="left"><sup>1</sup>H-NMR data of compound 4</th> <th align="left"><sup>13</sup>C-NMR data of compound 4</th> </tr> </thead> <tbody> <tr> <td align="left">2</td> <td align="left">â€"</td> <td align="left">140.5</td> </tr> <tr> <td align="left">2a</td> <td align="left">2.40 (3 H, <i>s,</i> H-2a)</td> <td align="left">17.8</td> </tr> <tr> <td align="left">3</td> <td align="left">2.90 (1 H, <i>m,</i> H-3)</td> <td align="left">55.7</td> </tr> <tr> <td align="left">4</td> <td align="left">7.21 (1H, <i>d</i>, <i>J</i>â€‰=â€‰7.6Â Hz, H-4),</td> <td align="left">141.2</td> </tr> <tr> <td align="left">5</td> <td align="left">7.04 (1H, <i>d</i> , <i>J</i>â€‰=â€‰7.6Â Hz, H-5)</td> <td align="left">130.8</td> </tr> <tr> <td align="left">6</td> <td align="left">â€"</td> <td align="left">137.9</td> </tr> <tr> <td align="left">7</td> <td align="left">7.08 (1H, <i>d</i>, <i>J</i>â€‰=â€‰0.8Â Hz, H-7)</td> <td align="left">127.6</td> </tr> <tr> <td align="left">8</td> <td align="left">â€"</td> <td align="left">138.3</td> </tr> <tr> <td align="left">8a</td> <td align="left">â€"</td> <td align="left">129.1</td> </tr> <tr> <td align="left">1â€²</td> <td align="left">1.40 (2 H, <i>m</i>, H-1â€²)</td> <td align="left">22.0</td> </tr> <tr> <td align="left">2â€²</td> <td align="left">1.98 (2 H, <i>m</i>, H-2â€²)</td> <td align="left">40.3</td> </tr> <tr> <td align="left">3â€²</td> <td align="left">â€"</td> <td align="left">210.5</td> </tr> <tr> <td align="left">4â€²</td> <td align="left">1.25 (1H, <i>septet</i>, H-4â€²)</td> <td align="left">27.3</td> </tr> <tr> <td align="left">4â€²a</td> <td align="left">1.21 (3H, <i>d</i>, <i>J</i>â€‰=â€‰7.2Â Hz, H-4â€²a)</td> <td align="left">18.9</td> </tr> <tr> <td align="left">4â€²b</td> <td align="left">1.25 (3H, <i>d</i>, <i>J</i>â€‰=â€‰7.2Â Hz, H-4â€²b)</td> <td align="left">18.2</td> </tr> <tr> <td align="left">1â€³</td> <td align="left">3.00 (2 H, <i>d</i>, <i>J</i>â€‰=â€‰2.4Â Hz,H-1â€³)</td> <td align="left">29.7</td> </tr> <tr> <td align="left">2â€³</td> <td align="left">â€"</td> <td align="left">203.5</td> </tr> <tr> <td align="left">3â€³</td> <td align="left">â€"</td> <td align="left">81.9</td> </tr> <tr> <td align="left">3â€³a</td> <td align="left">1.04 (3H, <i>s,</i> H-3â€³a)</td> <td align="left">21.4</td> </tr> <tr> <td align="left">3â€³b</td> <td align="left">0.87 (3H, <i>s</i>, H-3â€³b)</td> <td align="left">21.5</td> </tr> </tbody> </table>  <p /> <p id="Par26">Compound <b>5</b> (23Â mg) was isolated as a white powder from the ethyl acetate extract of the root of <i>O. cufodontii</i>. The UVâ€"Vis spectrum showed absorption band at 340Â nm indicating the presence of conjugated chromphore in the compound. The <sup>1</sup>H-NMR spectrum (CDCl<sub>3</sub>, 400Â MHz) revealed the presence of the three aromatic protons at <i>Î´</i> 7.90 (1H), <i>Î´</i> 7.75 (1H), and 7.63 (1H). The spectrum showed the presence of a methoxy group at <i>Î´</i> 3.85 (3H, <i>s</i>). The signal observed at <i>Î´</i> 3.41 (1H, m) is due to the proton on carbon bearing oxygen<i>.</i> The <sup>13</sup>C-NMR spectrum (CDCl<sub>3</sub>, 100Â MHz) show 21 carbon signals which are in agreement with the number of carbon atoms in this compound. The signals due to aromatic protons were evident at 148.4, 145.8, 139.9, 135.9, 124.5, and 116.0. The signals due to methoxy group and an oxygenated aliphatic carbon were observed at <i>Î´</i> 61.9 and 77.2, respectively. The spectrum also showed signals in the aliphatic regions at<i> Î´</i> 14.1, 22.2, 23.4, 23.6, 27.2, 29.7, 30.71, 30.72, 30.73, 38.4, 38.9, 41.8, and 44.0. The data generated led us to the identification of compound 5 as a known compound whose structure is depicted in Fig.Â <a href="#Fig1">1</a>.</p> </div> <div class="essentialoilanalysis" title="sec"> <div class="title" tagx="title" title="title">
Essential oil analysis</div> <p id="Par27">Hydro-distillation of the air dried ground leaves of <i>O. cufodontii</i> gave a yellow liquid essential oil with a yield of 2.66% w/w. The yield obtained herein is superior compared with the essential oil obtained from <i>O. gratissimum</i><sup><a href="#CR21">21</a></sup>. The presence of such huge quantities of essential oil from the leaves of <i>O. cufodontii</i> may justify the uses of this species as spices by the local people. The GCâ€"MS analysis of the essential oil has led to the identification of <i>Î²</i>-caryophyllene as a principal component with 31.3%. Such significant amount of <i>Î²</i>-caryophyllene was also reported from the essential oil of <i>O. campechianum</i><sup><a href="#CR29">29</a></sup>. The other constituents identified in the essential oil of <i>O. cufodontii</i> are <i>Z</i>,<i>Z</i>,<i>Z</i>-1,4,6,9-nonadecatetraene (30.64%), methyl-10,12-pentacosadiynoate (10.97%), 8,11-Octadecadiynoic acid, methyl ester (9.21%), 6,9,12-Octadecatrienoic acid, and phenylmethyl ester, (<i>Z</i>,<i>Z</i>,<i>Z</i>)-(5.57%). The presence of huge quantities of <i>Î²</i>-caryophyllene indicates the essential oils of <i>O. cufodontii</i> to be used as medicine and spices to enhance the taste of food.</p> </div> <div class="antibacterialactivity" title="sec"> <div class="title" tagx="title" title="title">
Antibacterial activity</div> <p id="Par28">The findings of the in vitro antibacterial activity of the extracts and isolated compounds from the leaves and roots of <i>O. cufodontii</i> were presented in Table <a href="#Tab3">3</a>. <div class="table-wrap_UNKNOWN" id="Tab3"><span class="label" tagx="label" title="label">Table 3</span> </div></p><p>Inhibition zone (in mm) displayed by constituents of the leaves and roots of <i>O. cufodontii</i> at dose of 50Â Âµg/mL.</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="left">Samples</th> <th align="left"><i>S. aureus</i></th> <th align="left"><i>E. coli</i></th> <th align="left"><i>K. phnemonea</i></th> <th align="left"><i>S. typhmurium</i></th> </tr> </thead> <tbody> <tr> <td align="left">HLE</td> <td align="left">12.01 (0.20)</td> <td align="left">8.00 (0.10)</td> <td align="left">6.00 (0.30)</td> <td align="left">7.02 (0.10)</td> </tr> <tr> <td align="left">EALE</td> <td align="left">8.00 (0.10)</td> <td align="left">6.02 (0.09)</td> <td align="left">8.01 (0.09)</td> <td align="left">6.00 (0.10)</td> </tr> <tr> <td align="left">MLE</td> <td align="left">8.02 (0.10)</td> <td align="left">6.01 (0.19)</td> <td align="left">6.02 (0.10)</td> <td align="left">6.00 (0.30)</td> </tr> <tr> <td align="left">HRE</td> <td align="left">17.09 (0.30)</td> <td align="left">8.01 (0.10)</td> <td align="left">7.02 (0.10)</td> <td align="left">9.03 (0.09)</td> </tr> <tr> <td align="left">EARE</td> <td align="left">13.00 (0.20)</td> <td align="left">6.01 (0.30)</td> <td align="left">8.03 (0.08)</td> <td align="left">10.01 (0.10)</td> </tr> <tr> <td align="left">MRE</td> <td align="left">15.04 (0.10)</td> <td align="left">6.02 (0.30)</td> <td align="left">6.03 (0.10)</td> <td align="left">6.01 (0.20)</td> </tr> <tr> <td align="left">EO</td> <td align="left">19.04 (0.10)</td> <td align="left">6.01 (0.20)</td> <td align="left">8.03 (0.50)</td> <td align="left">8.00 (0.10)</td> </tr> <tr> <td align="left">1</td> <td align="left">12.02 (0.20)</td> <td align="left">9.02 (0.30)</td> <td align="left">8.03 (0.10)</td> <td align="left">10.03 (0.30)</td> </tr> <tr> <td align="left">3</td> <td align="left">14.01 (0.30)</td> <td align="left">10.02 (0.20)</td> <td align="left">7.01 (0.10)</td> <td align="left">9.00 (0.10)</td> </tr> <tr> <td align="left">4</td> <td align="left">15.02 (0.10)</td> <td align="left">11.01 (0.30)</td> <td align="left">9.02 (0.40)</td> <td align="left">13.03 (0.10)</td> </tr> <tr> <td align="left">DMSO</td> <td align="left">0</td> <td align="left">0</td> <td align="left">0</td> <td align="left">0</td> </tr> <tr> <td align="left">Ciprofloxacin</td> <td align="left">19.30 (0.10)</td> <td align="left">19.01 (0.67)</td> <td align="left">22.00 (0.10)</td> <td align="left">21.50 (0.30)</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p>Samples were analyzed in triplicates and expressed as mâ€‰Â±â€‰SD.</p> <p><i>HLE</i> hexane leaves extract, <i>EALE</i> EtOAc leaves extract, <i>MLE</i> MeOH leaves extract, <i>HRE</i> hexane root extract, <i>ERE</i> EtOAc root extract, <i>MRE</i> MeOH root extract, <i>EO</i> Essential oil.</p> </div>  <p /> <p id="Par29">As clearly depicted in Table <a href="#Tab3">3</a>, the extracts and isolated compounds from the roots and leaves of <i>O. cufodontii</i> show differences in the inhibition of growth of bacterial strains. The absolute values of the diameter zones of inhibition (DZI) varied from 6.00 to 19.04â€‰Â±â€‰0.10Â mm. The hexane extract of the leaves of <i>O. cufodontii</i> displayed DZI of 12 0.01Â mm while the EtOAc and MeOH extract showed DZI of each 8.00Â mm indicating that the active constituents are likely residing in the non-polar fraction of the leaves of <i>O. cufodontii</i>. Likewise the hexane, EtOAc and MeOH extract of the roots exhibited DZI of 17.09, 13.00 and 15.04Â mm against <i>S. aureus</i>, respectively. The results displayed by the roots extracts are superior compared with that of the leaves of <i>O. cufodontii.</i> Compounds 1, 3 and 4 were also evaluated for their antibacterial activity and better inhibition was observed against <i>S. aureus</i> with DZI of 12.02, 14.01, and 15.02Â mm, respectively. The latter exhibited good activity with both Gram positive and gram negative bacteria. Therefore, the antibacterial activity displayed by the extracts of roots of <i>O. cufodontii</i> reveals the presence of compound <b>3</b> and <b>4</b>.</p> <p id="Par30">Moreover, the essential oil extracted from the leaves displayed significant antibacterial activity against <i>S. aureus</i> with DZI of 19.04â€‰Â±â€‰0.1Â mm. The activity demonstrated by the hexane extract of the leaves might be due to the presence of essential oils in the non-polar fraction of <i>O. cufodontii.</i> Under the same conditions, the EO exhibited similar antibacterial activity with chloramphenicol used as positive control for same strain. The antibacterial activity of the essential oil extracted from the leaves of <i>O. cufodontii</i> was also compared with the literature reported for essential oil from <i>O. gratissimum</i> and the results are almost similar<sup><a href="#CR30">30</a></sup>. The zone of inhibition of the extracts and isolated compounds of the roots extract of <i>O. cufodontii</i> show that the plant has good potential as a remedy for diseases caused by gram negative bacterial pathogens. Furthermore, results obtained from the present work corroborate the traditional use of the roots of <i>O. cufodontii</i> against bacteria.</p> </div> <div class="antioxidantactivity" title="sec"> <div class="title" tagx="title" title="title">
Antioxidant activity</div> <p id="Par31">DPPH assay is a simple method to assess antioxidants activities by measuring absorbance at 517Â nm due to the formation of DPPH radical<sup><a href="#CR29">29</a>,<a href="#CR31">31</a></sup>. The extracts and isolated compounds from the leaves and roots of <i>O. cufodontii</i> change the purple colored DPPH radical to the yellow-colored diphenylpicrylhydrazicine. The extracts also reduce the absorbance of the DPPH radical at 517Â nm indicating their potential as radical scavengers. The activity increases in a dose dependent manner. The hexane extract of the roots of <i>O. cufodontii</i> reduced the DPPH radical by 90.50% at 200Â Âµg/mL (Table <a href="#Tab4">4</a>). This is superior to the activity displayed by the other extracts. This is evident from the low IC<sub>50</sub> (4.4Â Âµg/mL) value observed for the hexane extract while the IC<sub>50</sub> value of the hexane extract of the leaves, the EtOAc extract of the leaves and the roots were 6.0, 48 and 122Â Âµg/mL, respectively. This is in agreement with the high radical scavenging activity reported in the literature for the hexane extracts of various non-pungent peppers compared with the EtOAc and methanol extracts<sup><a href="#CR32">32</a></sup>. Ascorbic acid and quercetin used as positive control scavenged the DPPH radical by 93.02 and 89.01% at 100Â Âµg/mL, respectively while their corresponding IC<sub>50</sub> values were 3.45 and 4.41Â Âµg/mL. The result was comparable to the activity displayed by the hexane extract of the roots of <i>O. cufodontii.</i> Literature reports have shown that <i>O. gratissimum, O. americanum, O. minimum</i> and <i>O. citriodorum</i> had radical scavenging activity of 81.1, 77.4, 70.1 and 60.6%, respectively<sup><a href="#CR33">33</a></sup>. This indicates that the activity displayed by the hexane extract of the roots of <i>O. cufodontii</i> is superior than the radical scavenging properties of its sister species. <div class="table-wrap_UNKNOWN" id="Tab4"><span class="label" tagx="label" title="label">Table 4</span> </div></p><p>DPPH radical scavenging and lipid peroxidation inhibitory activities of extracts and constituents of the roots and leaves of <i>O. cufodontii</i>.</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="left" rowspan="2">Samples</th> <th align="left" colspan="5">%DPPH inhibition at</th> <th align="left" rowspan="2">Anti-lipid peroxidation<br /><br />% inhibition</th> </tr> <tr> <th align="left">200 (Âµg/mL)</th> <th align="left">100 (Âµg/mL)</th> <th align="left" colspan="2">50 (Âµg/mL)</th> <th align="left">25 (Âµg/mL)</th> </tr> </thead> <tbody> <tr> <td align="left">HLE</td> <td align="left">65.02 (0.04)</td> <td align="left">57.90 (0.20)</td> <td align="left" colspan="2">52.81 (0.03)</td> <td align="left">51.72 (0.09)</td> <td align="left">70.01 (0.08)</td> </tr> <tr> <td align="left">EALE</td> <td align="left">68.32 (0.01)</td> <td align="left">53.21 (0.09)</td> <td align="left" colspan="2">50.73 (0.07)</td> <td align="left">48.50 (0.11)</td> <td align="left">52.02 (0.12)</td> </tr> <tr> <td align="left">HRE</td> <td align="left">90.50 (0.03)</td> <td align="left">85.01 (0.05)</td> <td align="left" colspan="2">61.22 (0.08)</td> <td align="left">50.90 (0.08)</td> <td align="left">83.02 (0.01)</td> </tr> <tr> <td align="left">EARE</td> <td align="left">52.44 (0.01)</td> <td align="left">49.23 (0.01)</td> <td align="left" colspan="2">48.81 (0.02)</td> <td align="left">45.92 (0.20)</td> <td align="left">57.01 (0.70)</td> </tr> <tr> <td align="left">1</td> <td align="left">32.30 (0.03)</td> <td align="left">28.52 (0.06)</td> <td align="left" colspan="2">24.00 (0.10)</td> <td align="left">19.21 (0.09)</td> <td align="left">22.01 (0.19)</td> </tr> <tr> <td align="left">3</td> <td align="left">39.22 (0.31)</td> <td align="left">35.81 (0.90)</td> <td align="left" colspan="2">28.22 (0.05)</td> <td align="left">16.80 (0.40)</td> <td align="left">76.02 (0.80)</td> </tr> <tr> <td align="left">4</td> <td align="left">37.90 (0.27)</td> <td align="left">32.70 (0.50)</td> <td align="left" colspan="2">26.90 (0.10)</td> <td align="left">22.82 (0.20)</td> <td align="left">69.01 (0.10)</td> </tr> <tr> <td align="left">Ascorbic acid</td> <td align="left">93.02 (0.02)</td> <td align="left">80.02 (0.04)</td> <td align="left" colspan="2">60.02 (0.05)</td> <td align="left">50.09 (0.02)</td> <td align="left">87.08 (0.02)</td> </tr> <tr> <td align="left">Quercetin</td> <td align="left">89.01 (0.06)</td> <td align="left">73.00 (0.10)</td> <td align="left" colspan="2">64.03 (0.21)</td> <td align="left">64.01 (0.08)</td> <td align="left">85.02 (0.07)</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p><i>HLE</i> hexane leaves extract, <i>EALE</i> EtOAc leaves extract, <i>HRE</i> hexane root extract, <i>EARE</i> EtOAc root extract; Samples were reported as meanâ€‰Â±â€‰SEM; Ascorbic acid and quercetin used as positive control were measured at 100, 50, 25 and 12Â Âµg/mL.</p> </div>  <p /> <p id="Par32">Compounds <b>1</b>, <b>3</b> and <b>4</b> inhibited DPPH radical by 32.30, 39.22, and 37.90%, respectively. As revealed from Table <a href="#Tab4">4</a>, the DPPH radical scavenging activities of the isolated compounds are inferior compared with the activity shown by extracts. This is likely ascribed to the presence of phenols and flavonoids in the extracts of the roots and leaves of <i>O. cufodontii.</i> The presence of flavonoids was further confirmed by the formation of yellow orange color on treating the extracts of the roots and leaves of <i>O. cufodontii</i> with NaOH followed by HCl<sup><a href="#CR34">34</a></sup>. Moreover, the extracts furnished a bluish green color on treatment with FeCl<sub>3</sub> supporting the presence of phenols in the extracts<sup><a href="#CR35">35</a></sup>.</p> <p id="Par33">The lipid peroxidation inhibitory potential of the extracts and isolated compounds were assessed using ferricthiocyante methods with the results presented in Table <a href="#Tab4">4</a>. As observed from Table <a href="#Tab4">4</a>, the hexane extracts of the leaves and the roots inhibited peroxide formation by 70.01 and 83.02%, respectively. The activity displayed by the latter extract is better than the activity displayed by the isolated compounds. The activity displayed by the hexane extract of the roots of <i>O. cufodontii</i> is comparable with the standard drug demonstrating the lipid peroxidation inhibitory potential of the hexane extract. This indicates the potential use of the extracts of the leaves and roots of <i>O. cufodontii</i> as remedy against diseases caused by radicals.</p> </div> <div class="ferric-reducingantioxidantpower(frap)assay" title="sec"> <div class="title" tagx="title" title="title">
Ferric-reducing antioxidant power (FRAP) assay</div> <p id="Par34">FRAP assay is used to evaluate the antioxidant potential of extracts based on the reduction of the Fe<sup>3+</sup>-TPTZ (2,4,6-tripyridyl-s-triazine) complex to the ferrous form at a pH around 3.6<sup><a href="#CR24">24</a></sup>. The ferric-TPTZ complex can be monitored at 593Â nm<sup><a href="#CR36">36</a></sup>. In view of this, the ferric-reducing antioxidant power of the extracts and isolated compounds was assessed and the results are depicted in Table <a href="#Tab5">5</a>. <div class="table-wrap_UNKNOWN" id="Tab5"><span class="label" tagx="label" title="label">Table 5</span> </div></p><p>Ferric-reducing potential of extracts and isolated compounds of <i>O. cufodontii</i>.</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="left">Samples</th> <th align="left">mmol Fe(II)/mg of samples</th> </tr> </thead> <tbody> <tr> <td align="left">HLE</td> <td align="left">0.090 (0.004)</td> </tr> <tr> <td align="left">EALE</td> <td align="left">0.087 (0.003)</td> </tr> <tr> <td align="left">HRE</td> <td align="left">1.120 (0.007)</td> </tr> <tr> <td align="left">EARE</td> <td align="left">0.900 (0.006)</td> </tr> <tr> <td align="left">1</td> <td align="left">0.050 (0.009)</td> </tr> <tr> <td align="left">3</td> <td align="left">0.070 (0.009)</td> </tr> <tr> <td align="left">4</td> <td align="left">0.060 (0.001)</td> </tr> <tr> <td align="left">Ascorbic acid</td> <td align="left">1.250 (0.002)</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p>Samples were measured in triplicates and expressed as Mâ€‰Â±â€‰SD; Ascorbic acid was used as positive control.</p> </div>  <p /> <p id="Par35">The higher absorbance of the extract indicates a higher ferric reducing power. As clearly seen from Table <a href="#Tab5">5</a>, the FRAP values for HLE, EALE, HRE and EARE were 0.090, 0.087, 1.120 and 0.900Â mmol Fe(II)/mg extract. This shows that the hexane extract of the roots of <i>O. cufodontii</i> exhibited the maximum reducing power. Because the root extract of <i>O. cufodontii</i> possesses antioxidant activity, it can be vital to human health.</p> </div> <div class="moleculardockingstudies" title="sec"> <div class="title" tagx="title" title="title">
Molecular docking studies</div> <p id="Par36">Compounds with antimicrobial activities disable bacteria by targeting key components of bacterial metabolism including cell-walls, DNA-directed RNApolymerase, protein synthesis, enzymes and DNAgyrase. The latter controls the topology of DNAduring transcription and replication by introducing transient breaks to both DNA strands<sup><a href="#CR37">37</a>â€"<a href="#CR39">39</a></sup>. Since DNAgyrase is pivotal for bacterial survival, it is essential to exploits bacterial DNAgyrase as a key target of antibacterial agent<sup><a href="#CR39">39</a></sup>. In view of this, the molecular docking study was carried out to examine the binding interactions of isolated compounds with the pocket of DNAgyrase and compared with ciprofloxacin. All the synthesized compounds exhibited well established bonds with one or more amino acids in the active pocket of the enzyme. The compounds also displayed minimum binding energy ranging from âˆ’Â 6.1 to âˆ’Â 6.9Â kcal/mol (Table <a href="#Tab6">6</a>) with compound <b>3</b> shown to have comparable binding score and amino acid interactions compared to ciprofloxacin. The binding affinity of the synthetic compounds along with their bonding interactions of ligands (<b>1â€"5</b>) were summarized in Table <a href="#Tab6">6</a>. <div class="table-wrap_UNKNOWN" id="Tab6"><span class="label" tagx="label" title="label">Table 6</span> </div></p><p>Molecular docking value of compounds isolated from root extracts of <i>O. cufodontii</i>.</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="left" rowspan="2">Compounds</th> <th align="left" rowspan="2">Affinity (kcal/mol)</th> <th align="left" rowspan="2">H-bond</th> <th align="left" colspan="2">Residual interactions</th> </tr> <tr> <th align="left">Hydrophobic/Pi-Cation</th> <th align="left">Van dar Waals</th> </tr> </thead> <tbody> <tr> <td align="left"><b>1</b></td> <td align="left">âˆ’Â 6.3</td> <td align="left">Arg-76</td> <td align="left">Ile-78</td> <td align="left">Asp-73, Glu-50, Ile-94, Asp-49, Ieu-98, Val-120</td> </tr> <tr> <td align="left"><b>2</b></td> <td align="left">âˆ’Â 6.1</td> <td align="left">â€"</td> <td align="left">Ile-78, Ile-94, Glu-50</td> <td align="left">Asp-73, Arg-76, Asn-46, Gly-77, Thr-165, Asp-49,</td> </tr> <tr> <td align="left"><b>3</b></td> <td align="left">âˆ’Â 6.9</td> <td align="left">Asn-46, Arg-76</td> <td align="left">Ile-78, Glu-50</td> <td align="left">Asp-43, Ala-47, Gly-77, Pro-79, Val-43, Arg-136, Val-167, Thr-165</td> </tr> <tr> <td align="left"><b>4</b></td> <td align="left">âˆ’Â 6.6</td> <td align="left">Asn-46</td> <td align="left">Glu-50, Ile-78</td> <td align="left">Asp-73, Asp-49, Ala-47, Gly-77, Pro-79, Ile-94, Arg-76, Val-43, Thr-165, Val-167</td> </tr> <tr> <td align="left"><b>5</b></td> <td align="left">âˆ’Â 6.6</td> <td align="left">Asp-73</td> <td align="left">Arg-76</td> <td align="left">Asn-46, Ala-47, Glu-50, gly-77, Ile-78, Pro-79, Thr-165</td> </tr> <tr> <td align="left"><b>Ciprofloxacin</b></td> <td align="left">âˆ’Â 6.9</td> <td align="left">Asp-73, Asn-46, Arg-76</td> <td align="left">Ile-78, Ile-94, Glu-50, Gly-77</td> <td align="left">Ala-47, Pro-79, Thr-165</td> </tr> </tbody> </table>  <p /> <p id="Par37">The extracted natural product molecules <b>1â€"5</b> showed better docking efficiency with DNA gyrase B within the binding pocket. In comparison to ciprofloxacin, all the isolated compounds have shown similar residual amino acids binding interactions with Ala-47, Glu-50, Gly-77, Ile-78, Pro-79, Ile-94, Thr-165 (Hydrophobic) and Asp-73, Arg-76, Asn-46 (Hydrogen bonds). The, isolated compound <b>1</b> forms additional residual interaction with Val-120 in the active site of the target protein with the docking score âˆ’Â 6.3Â kcal/mol. Compound <b>2</b> has the docking score âˆ’Â 6.1Â kcal/mol with no hydrogen bond or additional residual interactions within the binding pocket. The compounds <b>3</b> and <b>4</b> each formed additional residual Van dar Waals interaction with residual amino acids Val-43 and Val-167. Compound <b>3</b> with docking score âˆ’Â 6.9Â kcal/mol has two hydrogen bonds (Asn-46, Arg-76) and similar residual amino acid interaction within the binding pocket like standard inhibitor Ciprofloxacin. The compounds <b>4</b> and <b>5</b> both have docking score of âˆ’Â 6.6Â kcal/mol and forms one hydrogen bond with Asn-46 and Asp-73, respectively. The Molecular docking study of these compounds displayed moderate (âˆ’Â 6.1) to better (âˆ’Â 6.9) docking score within binding pocket toward <i>E. coli</i> DNA gyrase B (6F86). Overall, in silico molecular docking analysis of the isolated compounds matches with in vitro analysis against <i>S. aureus.</i> Among compounds analyzed, compound <b>3</b> shown comparable residual interactions and docking score of ciprofloxacin. Hence, compound <b>3</b> have a better antibacterial agent than the other isolated compounds in this investigation. The binding affinity, H-bonds, and residual amino acid interactions of the five compounds are summarized in Table <a href="#Tab5">5</a> and their binding interactions are shown in Figs.Â <a href="#Fig2">2</a>, <a href="#Fig3">3</a>, <a href="#Fig4">4</a>, <a href="#Fig5">5</a> and <a href="#Fig6">6</a>. Ribbon model shows the binding pocket structure of DNA gyrase B with compounds. Hydrogen bonds between compounds and amino acids are shown as green dashed lines while hydrophobic interaction are shown as pink lines. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2</span></a></div>   <p>The 2D and 3D intermolecular contact between compound <b>1</b> and DNA gyrase B (PDB ID: 6F86). Chemical structures were drawn by ChemDraw Pro 16.0 Suite (PerkinElmer, USA) and analyzed by theÂ Discovery studio visualizer (BIOVIA Discovery studio 2020 Client).</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 3</span></a></div>   <p>The 2D and 3D intermolecular contact between compound <b>2</b> and DNA gyrase B (PDB ID: 6F86). Chemical structures were drawn by ChemDraw Pro 16.0 Suite (PerkinElmer, USA) and analyzed by theÂ Discovery studio visualizer (BIOVIA Discovery studio 2020 Client).</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 4</span></a></div>   <p>The 2D and 3D intermolecular contact between compound <b>4</b> and DNA gyrase B (PDB ID: 6F86). Chemical structures were drawn by ChemDraw Pro 16.0 Suite (PerkinElmer, USA) and analyzed by theÂ Discovery studio visualizer (BIOVIA Discovery studio 2020 Client).</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 5</span></a></div>   <p>The 2D and 3D intermolecular contact between compound <b>5</b> and DNA gyrase B (PDB ID: 6F86). Chemical structures were drawn by ChemDraw Pro 16.0 Suite (PerkinElmer, USA) and analyzed by theÂ Discovery studio visualizer (BIOVIA Discovery studio 2020 Client).</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 6</span></a></div>   <p>The 2D and 3D intermolecular contact between ciprofloxacin and DNA gyrase B (PDB ID: 6F86). Chemical structures were drawn by ChemDraw Pro 16.0 Suite (PerkinElmer, USA) and analyzed by theÂ Discovery studio visualizer (BIOVIA Discovery studio 2020 Client).</p>   </div> <p /> </div> </div> <div class="conclusion" title="sec"> <div class="title" tagx="title" title="title">
Conclusion</div> <p id="Par38">In conclusion, two novel compounds <b>3</b> and <b>4</b> have been isolated and characterized from the roots extracts of <i>O. cufodontii</i>. The <i>n</i>-hexane extracts of the roots, the essential oil of the leaves of <i>O. cufodontii,</i> and compound <b>4</b> have shown comparable results to ciprofloxacin. The radical scavenging activity and anti-lipid peroxidation potential displayed by the <i>n</i>-hexane extracts of the roots of <i>O. cufodontii</i> also suggests the potential of the plant as an antioxidant. The in silico molecular docking studies revealed that all the isolated compounds <b>1â€"5</b> have close binding energy to the standard drug and may be considered as a good inhibitor of DNA gyrase. Hence, the activity displayed herein indicates that the plant has a great potential as a remedy for diseases caused by bacteria and radicals. Furthermore, the results obtained in the present study may help substantiate the traditional use of the roots of <i>O. cufodontii</i> with modern scientific based medical treatment.</p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary Information</div> <div class="sec" title="sec"> <p> </p><div class="local-data" title="local-data"> <div class="media" title="media"><a href="41598_2021_89557_MOESM1_ESM.docx">LINK</a> <p>Supplementary Information.</p>  </div> </div> <p /> </div> </div> </div> <div class="back" title="back"> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p><b>Publisher's note</b></p> <p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p> </div> </div> <div class="supplementaryinformation" title="sec"> <div class="title" tagx="title" title="title">
Supplementary Information</div> <p>The online version contains supplementary material available at 10.1038/s41598-021-89557-x.</p> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>We are grateful to Adama Science and Technology University for covering the cost of the reagents and sample analysis. We acknowledge Dr. Alemu Disassa (Assistant Professor of Applied Linguistic and Communication) for language editing.</p> </div> <div class="title" tagx="title" title="title">
Author contributions</div> <p>The chemistry part was done by Dr. Y.M. and M.A. while the bioassay were done by Professor A.D., Dr. T.A. and Dr. H.T. The molecular docking analysis was contributed by Dr. R.E. The manuscript was prepared by Dr. Y.M. and M.A., and reviewed by the group.</p> <div class="title" tagx="title" title="title">
Competing interests</div> <p id="Par39">The authors declare no competing interests.</p> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="CR1" /><span class="label" tagx="label" title="label">1.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Makri</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kintzios</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="mixed-article-title" title="mixed-article-title"><i>Ocimum</i> sp. (Basil): Botany, Cultivation, pharmaceutical properties, and biotechnology</span><span class="source" tagx="source" title="source">J. Herbs Spices Med. Plants</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">13</span><span class="issue" tagx="issue" title="issue">3</span><span class="fpage" tagx="fpage" title="fpage">1</span></span></li> <li tag="ref"><a name="CR2" /><span class="label" tagx="label" title="label">2.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roberto</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">James</span><span class="given-names" tagx="given-names" title="given-names">ES</span></span></span><span class="mixed-article-title" title="mixed-article-title">Chemical characterization of basil (<i>Ocimum</i> Spp.) found in the markets and used in traditional medicine in Brazil</span><span class="source" tagx="source" title="source">Econ. Bot.</span><span class="year" tagx="year" title="year">2000</span><span class="volume" tagx="volume" title="volume">54</span><span class="issue" tagx="issue" title="issue">2</span><span class="fpage" tagx="fpage" title="fpage">207</span><span class="lpage" tagx="lpage" title="lpage">216</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/BF02907824">10.1007/BF02907824</a></span></span></li> <li tag="ref"><a name="CR3" /><span class="label" tagx="label" title="label">3.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tan</span><span class="given-names" tagx="given-names" title="given-names">PV</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nyasse</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dimo</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mezui</span><span class="given-names" tagx="given-names" title="given-names">C</span></span></span><span class="mixed-article-title" title="mixed-article-title">Gastric cytoprotective anti-ulcer effects of the leaf methanol extract of <i>Ocimum suave</i> (Lamiaceae) in rats</span><span class="source" tagx="source" title="source">J. Ethnopharmacol.</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">82</span><span class="fpage" tagx="fpage" title="fpage">69</span><span class="lpage" tagx="lpage" title="lpage">74</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0378-8741(02)00142-3">10.1016/S0378-8741(02)00142-3</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12241979">12241979</a></span></span></li> <li tag="ref"><a name="CR4" /><span class="label" tagx="label" title="label">4.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Simon, J. E., Quinn, J., Murray, R. G. Basil: A source of essential oils. <i>Adv. New Crops Res.</i> 484â€"489 (1990).</span></li> <li tag="ref"><a name="CR5" /><span class="label" tagx="label" title="label">5.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kadhim</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sosa</span><span class="given-names" tagx="given-names" title="given-names">AA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hameed</span><span class="given-names" tagx="given-names" title="given-names">LH</span></span></span><span class="mixed-article-title" title="mixed-article-title">Evaluation of anti-bacterial activity and bioactive chemical analysis of <i>Ocimum basilicum</i> using Fourier transform infrared (FT-IR) and gas chromatographymass spectrometry (GC-MS) techniques</span><span class="source" tagx="source" title="source">J. Pharmacogn. Phytother.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">8</span><span class="issue" tagx="issue" title="issue">6</span><span class="fpage" tagx="fpage" title="fpage">127</span><span class="lpage" tagx="lpage" title="lpage">146</span><span class="pub-id"><a href="https://dx.doi.org/10.5897/JPP2015.0366">10.5897/JPP2015.0366</a></span></span></li> <li tag="ref"><a name="CR6" /><span class="label" tagx="label" title="label">6.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Makonnen</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Debella</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abebe</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Teka</span><span class="given-names" tagx="given-names" title="given-names">F</span></span></span><span class="mixed-article-title" title="mixed-article-title">Analgesic properties of some Ethiopian medicinal plants in different models of nociception in mice</span><span class="source" tagx="source" title="source">Phytother. Res.</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">17</span><span class="fpage" tagx="fpage" title="fpage">1108</span><span class="lpage" tagx="lpage" title="lpage">1112</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/ptr.1306">10.1002/ptr.1306</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14595598">14595598</a></span></span></li> <li tag="ref"><a name="CR7" /><span class="label" tagx="label" title="label">7.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Makonnen</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Debella</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zerihun</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abebe</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Teka</span><span class="given-names" tagx="given-names" title="given-names">F</span></span></span><span class="mixed-article-title" title="mixed-article-title">Antipyretic properties of the aqueous and ethanol extracts of the leaves of <i>Ocimum suave</i> and <i>Ocimum lamiifolium</i> in mice</span><span class="source" tagx="source" title="source">J. Ethnopharmacol.</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">88</span><span class="fpage" tagx="fpage" title="fpage">85</span><span class="lpage" tagx="lpage" title="lpage">91</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0378-8741(03)00175-2">10.1016/S0378-8741(03)00175-2</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12902056">12902056</a></span></span></li> <li tag="ref"><a name="CR8" /><span class="label" tagx="label" title="label">8.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Runyoro</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ngassapa</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vagionas</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aligiannis</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Graikou</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chinou</span><span class="given-names" tagx="given-names" title="given-names">I</span></span></span><span class="mixed-article-title" title="mixed-article-title">Chemical composition and antimicrobial activity of the essential oils of four <i>Ocimum</i> species growing in Tanzania</span><span class="source" tagx="source" title="source">Food Chem.</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">119</span><span class="fpage" tagx="fpage" title="fpage">311</span><span class="lpage" tagx="lpage" title="lpage">316</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.foodchem.2009.06.028">10.1016/j.foodchem.2009.06.028</a></span></span></li> <li tag="ref"><a name="CR9" /><span class="label" tagx="label" title="label">9.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hassan</span><span class="given-names" tagx="given-names" title="given-names">KA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Deogratius</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nyafuono</span><span class="given-names" tagx="given-names" title="given-names">JF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Francis</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Engeu</span><span class="given-names" tagx="given-names" title="given-names">OP</span></span></span><span class="mixed-article-title" title="mixed-article-title">Wound healing potential of the ethanolic extracts of <i>Bidens pilosa</i> and <i>Ocimum suave</i></span><span class="source" tagx="source" title="source">Afr. J. Pharm. Pharmacol.</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">5</span><span class="fpage" tagx="fpage" title="fpage">132</span><span class="lpage" tagx="lpage" title="lpage">136</span><span class="pub-id"><a href="https://dx.doi.org/10.5897/AJPP10.266">10.5897/AJPP10.266</a></span></span></li> <li tag="ref"><a name="CR10" /><span class="label" tagx="label" title="label">10.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paton</span><span class="given-names" tagx="given-names" title="given-names">AW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Woodrow</span><span class="given-names" tagx="given-names" title="given-names">MC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Doyle</span><span class="given-names" tagx="given-names" title="given-names">RM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lanser</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paton</span><span class="given-names" tagx="given-names" title="given-names">JC</span></span></span><span class="mixed-article-title" title="mixed-article-title">Molecular characterization of a Shiga toxigenic <i>Escherichia coli</i> O113:H21 strain lacking eae responsible for a cluster of cases of hemolytic-uremic syndrome</span><span class="source" tagx="source" title="source">J. Clin. Microbiol.</span><span class="year" tagx="year" title="year">1999</span><span class="volume" tagx="volume" title="volume">37</span><span class="fpage" tagx="fpage" title="fpage">3357</span><span class="lpage" tagx="lpage" title="lpage">3361</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/JCM.37.10.3357-3361.1999">10.1128/JCM.37.10.3357-3361.1999</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10488206">10488206</a></span></span></li> <li tag="ref"><a name="CR11" /><span class="label" tagx="label" title="label">11.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Juliani, H. R. &amp;amp; Simon, J. E. Antioxidant activity of basil. In <i>Trends in New Crops and New Uses</i> (eds. Janick, J. &amp;amp; Whipkey, A.) 575â€"579 (ASHS Press, 2002).</span></li> <li tag="ref"><a name="CR12" /><span class="label" tagx="label" title="label">12.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ozkan</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">Glandular and eglandular hairs of salvia recognita Fisch. and Mey. (Lamiaceae) in Turkey</span><span class="source" tagx="source" title="source">Bangladesh J. Bot.</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">37</span><span class="fpage" tagx="fpage" title="fpage">93</span><span class="lpage" tagx="lpage" title="lpage">95</span><span class="pub-id"><a href="https://dx.doi.org/10.3329/bjb.v37i1.1571">10.3329/bjb.v37i1.1571</a></span></span></li> <li tag="ref"><a name="CR13" /><span class="label" tagx="label" title="label">13.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tansi</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Muto</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Critelli</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Iovine</span><span class="given-names" tagx="given-names" title="given-names">G</span></span></span><span class="mixed-article-title" title="mixed-article-title">Neogene-Quaternary strike-slip tectonics in the central Calabrian Arc (Southern Italy)</span><span class="source" tagx="source" title="source">J. Geodyn.</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">43</span><span class="fpage" tagx="fpage" title="fpage">393</span><span class="lpage" tagx="lpage" title="lpage">414</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jog.2006.10.006">10.1016/j.jog.2006.10.006</a></span></span></li> <li tag="ref"><a name="CR14" /><span class="label" tagx="label" title="label">14.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sarwat</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Srivastava</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Naved</span><span class="given-names" tagx="given-names" title="given-names">T</span></span></span><span class="mixed-article-title" title="mixed-article-title">Diversity Analyses in <i>Ocimum</i> Species: Why and How?</span><span class="source" tagx="source" title="source">Int. J. Pharmacogn. Phytochem. Res.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">8</span><span class="issue" tagx="issue" title="issue">2</span><span class="fpage" tagx="fpage" title="fpage">252</span><span class="lpage" tagx="lpage" title="lpage">262</span></span></li> <li tag="ref"><a name="CR15" /><span class="label" tagx="label" title="label">15.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paton</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">A synopsis of <i>Ocimum</i> L. (Labiatae) in Africa</span><span class="source" tagx="source" title="source">Kew Bull.</span><span class="year" tagx="year" title="year">1992</span><span class="volume" tagx="volume" title="volume">47</span><span class="fpage" tagx="fpage" title="fpage">405</span><span class="lpage" tagx="lpage" title="lpage">437</span></span></li> <li tag="ref"><a name="CR16" /><span class="label" tagx="label" title="label">16.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Kalbessa, A., Dekebo, A., Tesso, H., Abdo,T., Abdissa, N., &amp;amp; Melaku, Y. Chemical constituents of root barks of <i>Gnidia involucrata</i> and evaluation for antibacterial and antioxidant activities. <i>J. Trop. Med.</i> 1â€"9 (2019).</span></li> <li tag="ref"><a name="CR17" /><span class="label" tagx="label" title="label">17.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hema</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kumarave</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gomathi</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sivasubramaniam</span><span class="given-names" tagx="given-names" title="given-names">C</span></span></span><span class="mixed-article-title" title="mixed-article-title">Gas chromatography-Mass spectroscopic analysis of <i>Lasonia inermis</i> leaves</span><span class="source" tagx="source" title="source">N. Y. Sci. J.</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">3</span><span class="issue" tagx="issue" title="issue">11</span><span class="fpage" tagx="fpage" title="fpage">141</span><span class="lpage" tagx="lpage" title="lpage">143</span></span></li> <li tag="ref"><a name="CR18" /><span class="label" tagx="label" title="label">18.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Zeleke, D., Eswaramoorthy, R,. Belay, Z. &amp;amp; Melaku, Y. Synthesis and antibacterial, antioxidant, and molecular docking analysis of some novel quinoline derivatives. <i>J. Chem.</i> 1â€"9 (2020).</span></li> <li tag="ref"><a name="CR19" /><span class="label" tagx="label" title="label">19.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Trott</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Olson</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span></span><span class="mixed-article-title" title="mixed-article-title">AutoDock Vina: Improving the speed and accuracy of docking with a new scoring function, efficient optimization, and multithreading</span><span class="source" tagx="source" title="source">J. Comput. Chem.</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">31</span><span class="issue" tagx="issue" title="issue">2</span><span class="fpage" tagx="fpage" title="fpage">455</span><span class="lpage" tagx="lpage" title="lpage">461</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19499576">19499576</a></span></span></li> <li tag="ref"><a name="CR20" /><span class="label" tagx="label" title="label">20.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Narramore</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stevenson</span><span class="given-names" tagx="given-names" title="given-names">CEM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maxwell</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lawson</span><span class="given-names" tagx="given-names" title="given-names">DM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fishwick</span><span class="given-names" tagx="given-names" title="given-names">CWG</span></span></span><span class="mixed-article-title" title="mixed-article-title">New insights into the binding mode of pyridine-3-carboxamide inhibitors of <i>E. coli</i> DNA gyrase</span><span class="source" tagx="source" title="source">Bioorgan. Med. Chem.</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">27</span><span class="issue" tagx="issue" title="issue">16</span><span class="fpage" tagx="fpage" title="fpage">3546</span><span class="lpage" tagx="lpage" title="lpage">3550</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.bmc.2019.06.015">10.1016/j.bmc.2019.06.015</a></span></span></li> <li tag="ref"><a name="CR21" /><span class="label" tagx="label" title="label">21.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Braca</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tommasi</span><span class="given-names" tagx="given-names" title="given-names">ND</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bari</span><span class="given-names" tagx="given-names" title="given-names">LD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pizza</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Politi</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morelli</span><span class="given-names" tagx="given-names" title="given-names">I</span></span></span><span class="mixed-article-title" title="mixed-article-title">Antioxidant principles from <i>Bauhinia terapotensis</i></span><span class="source" tagx="source" title="source">J. Nat. Prod.</span><span class="year" tagx="year" title="year">2001</span><span class="volume" tagx="volume" title="volume">64</span><span class="fpage" tagx="fpage" title="fpage">892</span><span class="lpage" tagx="lpage" title="lpage">895</span><span class="pub-id"><a href="https://dx.doi.org/10.1021/np0100845">10.1021/np0100845</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11473417">11473417</a></span></span></li> <li tag="ref"><a name="CR22" /><span class="label" tagx="label" title="label">22.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gao</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cai</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yao</span><span class="given-names" tagx="given-names" title="given-names">W</span></span></span><span class="mixed-article-title" title="mixed-article-title">In vitro and in vivo antioxidant activity of aqueous extract from <i>Choerospondia saxillaris</i> fruit</span><span class="source" tagx="source" title="source">Food Chem.</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">106</span><span class="fpage" tagx="fpage" title="fpage">888</span><span class="lpage" tagx="lpage" title="lpage">895</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.foodchem.2007.05.068">10.1016/j.foodchem.2007.05.068</a></span></span></li> <li tag="ref"><a name="CR23" /><span class="label" tagx="label" title="label">23.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nagatsu</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">Investigation of anti-oxidative compounds from oil plant seed</span><span class="source" tagx="source" title="source">FABAD J. Pharm. Sci.</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">29</span><span class="fpage" tagx="fpage" title="fpage">203</span><span class="lpage" tagx="lpage" title="lpage">210</span></span></li> <li tag="ref"><a name="CR24" /><span class="label" tagx="label" title="label">24.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Birasuren</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">NY</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jeon</span><span class="given-names" tagx="given-names" title="given-names">HL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">MR</span></span></span><span class="mixed-article-title" title="mixed-article-title">Evaluation of the antioxidant capacity and phenolic content of <i>Agriophyllum pungens</i> seed extracts from Mongolia</span><span class="source" tagx="source" title="source">Prev. Nutr. Food Sci.</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">18</span><span class="issue" tagx="issue" title="issue">3</span><span class="fpage" tagx="fpage" title="fpage">188</span><span class="lpage" tagx="lpage" title="lpage">195</span><span class="pub-id"><a href="https://dx.doi.org/10.3746/pnf.2013.18.3.188">10.3746/pnf.2013.18.3.188</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24471131">24471131</a></span></span></li> <li tag="ref"><a name="CR25" /><span class="label" tagx="label" title="label">25.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Igoli</span><span class="given-names" tagx="given-names" title="given-names">JO</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gray</span><span class="given-names" tagx="given-names" title="given-names">IA</span></span></span><span class="mixed-article-title" title="mixed-article-title">Friedelanone and other triterpenoids from <i>Hymenocardia acida</i></span><span class="source" tagx="source" title="source">Int. J. Phys. Sci.</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">3</span><span class="issue" tagx="issue" title="issue">6</span><span class="fpage" tagx="fpage" title="fpage">156</span><span class="lpage" tagx="lpage" title="lpage">158</span></span></li> <li tag="ref"><a name="CR26" /><span class="label" tagx="label" title="label">26.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nazma</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mohammad</span><span class="given-names" tagx="given-names" title="given-names">SR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mohammad</span><span class="given-names" tagx="given-names" title="given-names">SI</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mohammad</span><span class="given-names" tagx="given-names" title="given-names">AR</span></span></span><span class="mixed-article-title" title="mixed-article-title">Chemical and biological investigations of <i>Dillenia indica</i> Linn</span><span class="source" tagx="source" title="source">J. Bangladesh Pharmacol. Soc.</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">4</span><span class="fpage" tagx="fpage" title="fpage">122</span><span class="lpage" tagx="lpage" title="lpage">125</span></span></li> <li tag="ref"><a name="CR27" /><span class="label" tagx="label" title="label">27.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Siddique</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hafeez</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Begum</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Siddique</span><span class="given-names" tagx="given-names" title="given-names">BS</span></span></span><span class="mixed-article-title" title="mixed-article-title">Oleanderol, a new pentacyclic triterpene from the leaves of Nerium oleander</span><span class="source" tagx="source" title="source">J. Nat. Prod.</span><span class="year" tagx="year" title="year">1998</span><span class="volume" tagx="volume" title="volume">51</span><span class="issue" tagx="issue" title="issue">2</span><span class="fpage" tagx="fpage" title="fpage">229</span><span class="lpage" tagx="lpage" title="lpage">233</span><span class="pub-id"><a href="https://dx.doi.org/10.1021/np50056a006">10.1021/np50056a006</a></span></span></li> <li tag="ref"><a name="CR28" /><span class="label" tagx="label" title="label">28.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Scott</span><span class="given-names" tagx="given-names" title="given-names">AI</span></span></span><span class="source" tagx="source" title="source">Interpretation of Ultraviolet Spectra of Natural Products</span><span class="year" tagx="year" title="year">1964</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Pergamon</span></span></li> <li tag="ref"><a name="CR29" /><span class="label" tagx="label" title="label">29.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Scalvenzi</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Larvicidal activity of <i>Ocimum campechianum, Ocotea quixos</i> and <i>Piper aduncum</i> essential oils against <i>Aedes aegypti</i></span><span class="source" tagx="source" title="source">Parasite</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">26</span><span class="fpage" tagx="fpage" title="fpage">23</span><span class="pub-id"><a href="https://dx.doi.org/10.1051/parasite/2019024">10.1051/parasite/2019024</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30994444">30994444</a></span></span></li> <li tag="ref"><a name="CR30" /><span class="label" tagx="label" title="label">30.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ajose</span><span class="given-names" tagx="given-names" title="given-names">O</span></span></span><span class="mixed-article-title" title="mixed-article-title">Antibacterial activity of <i>Ocimum gratissimum</i> L. against some selected human bacterial pathogens</span><span class="source" tagx="source" title="source">JPRI</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">20</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">8</span></span></li> <li tag="ref"><a name="CR31" /><span class="label" tagx="label" title="label">31.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gulcin</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huyut</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elmastas</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aboul-Enein</span><span class="given-names" tagx="given-names" title="given-names">HY</span></span></span><span class="mixed-article-title" title="mixed-article-title">Radical scavenging and antioxidant activity of tannic acid</span><span class="source" tagx="source" title="source">Arab. J. Chem.</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">3</span><span class="fpage" tagx="fpage" title="fpage">43</span><span class="lpage" tagx="lpage" title="lpage">53</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.arabjc.2009.12.008">10.1016/j.arabjc.2009.12.008</a></span></span></li> <li tag="ref"><a name="CR32" /><span class="label" tagx="label" title="label">32.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bae</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jayaprakasha</span><span class="given-names" tagx="given-names" title="given-names">GK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Crosby</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jifon</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span></span><span class="mixed-article-title" title="mixed-article-title">Influence of extraction solvents on antioxidant activity and the content of bioactive compounds in non-pungent peppers</span><span class="source" tagx="source" title="source">Plant Foods Hum. Nutr.</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">67</span><span class="fpage" tagx="fpage" title="fpage">120</span><span class="lpage" tagx="lpage" title="lpage">128</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s11130-012-0290-4">10.1007/s11130-012-0290-4</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22569831">22569831</a></span></span></li> <li tag="ref"><a name="CR33" /><span class="label" tagx="label" title="label">33.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hakkim</span><span class="given-names" tagx="given-names" title="given-names">LF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gowri</span><span class="given-names" tagx="given-names" title="given-names">SC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Girija</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="mixed-article-title" title="mixed-article-title">Chemical composition and antioxidant property of holy basil (<i>Ocimum sanctum</i> L.) leaves, stems and inflorescence and their in vitro callus cultures</span><span class="source" tagx="source" title="source">J. Agric. Food Chem.</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">55</span><span class="fpage" tagx="fpage" title="fpage">9109</span><span class="lpage" tagx="lpage" title="lpage">9117</span><span class="pub-id"><a href="https://dx.doi.org/10.1021/jf071509h">10.1021/jf071509h</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17924700">17924700</a></span></span></li> <li tag="ref"><a name="CR34" /><span class="label" tagx="label" title="label">34.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sasi</span><span class="given-names" tagx="given-names" title="given-names">KR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Balasubramanian</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Govindaraj</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krishnaveni</span><span class="given-names" tagx="given-names" title="given-names">T</span></span></span><span class="mixed-article-title" title="mixed-article-title">Preliminary studies on phytochemicals and antimicrobial activity of solvent extracts of <i>Coriandrumsativum</i> L. roots (Coriander)</span><span class="source" tagx="source" title="source">J. Pharmacogn. Phytochem.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">2</span><span class="issue" tagx="issue" title="issue">6</span><span class="fpage" tagx="fpage" title="fpage">74</span><span class="lpage" tagx="lpage" title="lpage">78</span></span></li> <li tag="ref"><a name="CR35" /><span class="label" tagx="label" title="label">35.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Harborne, J. <i>Phytochemical Methods A Guide to Modern Techniques of Plant Analysis</i>, 3rd Edn. 1â€"307 (Chapman and Hall, 1998).</span></li> <li tag="ref"><a name="CR36" /><span class="label" tagx="label" title="label">36.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Banothu</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Neelagiri</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Adepally</span><span class="given-names" tagx="given-names" title="given-names">U</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lingam</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bommareddy</span><span class="given-names" tagx="given-names" title="given-names">K</span></span></span><span class="mixed-article-title" title="mixed-article-title">Phytochemical screening and evaluation of in vitro antioxidant and antimicrobial activities of the indigenous medicinal plant <i>Albizia odoratissima</i></span><span class="source" tagx="source" title="source">Pharm. Biol.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">55</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">1155</span><span class="lpage" tagx="lpage" title="lpage">1161</span><span class="pub-id"><a href="https://dx.doi.org/10.1080/13880209.2017.1291694">10.1080/13880209.2017.1291694</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28219296">28219296</a></span></span></li> <li tag="ref"><a name="CR37" /><span class="label" tagx="label" title="label">37.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ziraldo</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hanke</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Levene</span><span class="given-names" tagx="given-names" title="given-names">SD</span></span></span><span class="mixed-article-title" title="mixed-article-title">Kinetic pathways of topology simplification by Type-II topoisomerases in knotted supercoiled DNA</span><span class="source" tagx="source" title="source">Nucleic Acids Res.</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">47</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">69</span><span class="lpage" tagx="lpage" title="lpage">84</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gky1174">10.1093/nar/gky1174</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30476194">30476194</a></span></span></li> <li tag="ref"><a name="CR38" /><span class="label" tagx="label" title="label">38.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dorman</span><span class="given-names" tagx="given-names" title="given-names">CJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dorman</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span></span><span class="mixed-article-title" title="mixed-article-title">DNA supercoiling is a fundamental regulatory principle in the control of bacterial gene expression</span><span class="source" tagx="source" title="source">Biophys. Rev.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">8</span><span class="issue" tagx="issue" title="issue">3</span><span class="fpage" tagx="fpage" title="fpage">209</span><span class="lpage" tagx="lpage" title="lpage">220</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s12551-016-0205-y">10.1007/s12551-016-0205-y</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28510224">28510224</a></span></span></li> <li tag="ref"><a name="CR39" /><span class="label" tagx="label" title="label">39.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schoeffler</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Berger</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span></span><span class="mixed-article-title" title="mixed-article-title">DNA topoisomerases: Harnessing and constraining energy to govern chromosome topology</span><span class="source" tagx="source" title="source">Q. Rev. Biophys.</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">41</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">41</span><span class="lpage" tagx="lpage" title="lpage">101</span><span class="pub-id"><a href="https://dx.doi.org/10.1017/S003358350800468X">10.1017/S003358350800468X</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18755053">18755053</a></span></span></li> </ul> </div> </div>  </body></html>